* 1305433
* Recombination, Genetic Interactions, and Observable Evolutionary Dynamics
* MPS,PHY
* 09/01/2013,08/31/2017
* Daniel Fisher, Stanford University
* Continuing Grant
* Krastan Blagoev
* 08/31/2017
* USD 420,000.00

The goal of this project is to develop quantitative understanding of the
evolutionary dynamics of large populations with abundant selection and
recombination by analyzing a variety of models that build in various of the key
features. The interplay between multiple beneficial mutations accumulating in
one lineage and recombination that breaks apart and brings together collections
of such mutations will be studied in several situations: re-assortment, the
exchange of chromosomes between individuals without recombination as occurs for
influenza viruses; transfer of small segments of a genome as in lateral gene
transfer between bacteria; occasional mating between otherwise asexually
evolving populations, as in yeast; and various approximations of the spectrum of
degrees of linkage that arise from the linear structure of chromosomes.
Epistatic interactions between mutations together with changing linkage leads to
a balance between selection on individual genes and selection on genomes: this
will be explored for various models of the interactions focusing on the
statistical properties of the rare, anomalously good combinations of mutations
that dominate the future evolution. The models of epistasis will be informed by,
and inform, experiments on laboratory evolution of budding yeast carried out by
collaborators. Mating within or between asexually evolved populations that have
been "barcoded", should provide crucial information about the fitness of parents
and their offspring, and the theory developed will enable predictions about the
benefits of sex that can be tested in future experiments. Going beyond well-
mixed populations in which all individuals compete directly, other forms of
competition that could be realized in the laboratory (such as selection of the
best half of a population) will be studied. Spatial structure and the interplay
between local competition and mixing on longer time scales will also be
investigated, focusing on the effects of recombination on rates of evolution and
dynamics of diversity. Collaborations with several groups carrying out
laboratory evolution experiments with yeast or bacteria will be further
developed to obtain statistical information needed for the modeling, and to
propose and plan future experiments. Analysis of deep sequencing data of
bacterial populations will be carried out with collaborators to infer aspects of
the evolutionary history, including recombination, and to develop plans for the
most informative future sampling and sequencing. The students and postdocs
involved will greatly benefit from mentoring by several faculty from different
fields. Beyond the research, the PI is developing new courses on evolution for
quantitative scientists and biologists: lecture notes will be made available on
the internet. The PI will also become further involved in educating the general
public about evolution, including the future challenges, as well as the
successes, of evolutionary theory.