* 1312250
* Integrated Bayesian Deterministic and Probabilistic Modeling of Phenotype-Genotype Networks in the Metabolic System
* MPS,DMS
* 08/01/2013,07/31/2017
* Rachael Hageman Blair, SUNY at Buffalo
* Continuing Grant
* Junping Wang
* 07/31/2017
* USD 185,000.00

This project develops a comprehensive Bayesian methodology for mathematical
modeling, which integrates across diverse data types, accounts for model
uncertainty, and incorporates existing prior knowledge about the biological
system. The Bayesian models integrate two well-established modeling paradigms:
(1) deterministic mathematical models of cellular metabolism and (2) causal
graphical models of phenotypes and genotypes. These modeling paradigms are
widely used, but suffer major limitations: deterministic models do not reflect
the individual variation in fluxes that result from allelic variation of
enzymes, and graphical modeling techniques currently ignore all prior
information about the biochemical pathway. This project overcomes these
limitations by incorporating diverse forms of data and variables, which span the
genome, transcriptome, metabolome, and clinical phenotypes. Applications focus
on genome-scale metabolic reconstructions of breast and skin cancer cell
metabolism to search out novel molecular targeted therapies, side-effects, and
unravel the complex mechanisms underlying the disease. The methodology will be
implemented in a publicly available software package, which is designed for
users with a minimal background in mathematics and statistics. Understanding how
genetics influences metabolic and regulatory function is important for the
prevention and treatment of disease. Genetics explains in part why some people
are more responsive to drug treatments, while others have no response or may
experience deadly side effects. Advances in high-throughput phenotyping
technologies have made large-scale measurements of molecular traits possible.
Mathematical models are widely used to elucidate networks of molecular traits
from high-throughput data, and have become important tools for systems biology.
Despite this progress, integrating diverse types of data remains a major
challenge that has limited our ability to take full advantage of the wealth of
post-genomics data for knowledge and discovery. This project addresses this
challenge and represents a bold new direction in systems biology, which can be
generalized to model different biological systems. A broader impact of this
project is the software development, which aims to bridge the gap between
computational and experimental biology by putting accessible tools in the hands
of the biologist.