* 1330813
* Collaborative Research: RUI: Developing Integrated Metabolic Regulatory Models (iMRMs) for the Investigation of Metabolic and Regulatory Diversity of Sequenced Microbes
* BIO,MCB
* 10/01/2013,09/30/2017
* Nathan Tintle, Dordt University, Incorporated
* Standard Grant
* Edward Crane
* 09/30/2017
* USD 248,652.00

Intellectual Merit&lt;br/&gt;Recent breakthroughs in genetic sequencing
technology have set the stage for an unprecedented wide understanding of all
microbial life. Previously, automated methods for the creation of genome-scale
metabolic models (MMs) have been developed and applied by this group. These
methods have since been applied to thousands of microbial genomes in freely
available online software systems. While the MMs are yielding unprecedented
insights into microbial metabolism, the next great challenge is to capture gene
regulatory information in order to more accurately model the metabolic response
of an organism to its environment. Anticipating this need, ongoing efforts by
the group have set the stage for integrating a wide range of alternative data
sources with the thousands of MMs. The group is now uniquely positioned to
develop enhanced MMs through the integration of regulatory information into the
models, yielding integrated metabolic, regulatory models (iMRMs). To date,
little methodological effort has been directed toward the wide-scale development
of iMRMs due, in part, to the lack of sufficient and integrated data for most
organisms. This project will (1) develop new and improved iMRMs by addressing
methodological weaknesses in current approaches, (2) develop methods to utilize
iMRMs to predict conditions for wet-lab experiments to generate and test novel
biological hypotheses, (3) develop novel approaches to use thousands of models
to better understand metabolic and regulatory diversity, and (4) fully
incorporate undergraduate and high school students in all aspects of the
research. Thus, the project will substantially advance the capacity to construct
integrated metabolic regulatory models through the development and evaluation of
methods for the incorporation of regulatory data, produce tools for researchers
to assess the diversity of existing gene expression data sets for their
organisms of interest, validate proposed methods via targeted wet lab
experiments, and address fundamental questions about metabolic and regulatory
diversity across the microbial tree of life.&lt;br/&gt;&lt;br/&gt;Broader
Impacts&lt;br/&gt;All methodological advancements will be integrated into an
open-source software environment for modeling microbial life. At least 21
undergraduate and at least 60 high school students will be integrally involved
in the research, providing them with training, experience and exposure to the
interdisciplinary field of quantitative approaches for predictive systems
biology.