* 1330320
* Role of Dot1 and H3K79 methylation in gene regulation
* BIO,MCB
* 12/15/2013,11/30/2016
* Brian Strahl, University of North Carolina at Chapel Hill
* Continuing Grant
* Karen Cone
* 11/30/2016
* USD 750,000.00

Intellectual Merit: How our DNA is packaged into cells and is accessed for
processes such as gene expression remains a fundamental question in modern
biology. A major contributor to DNA packaging and its accessibility are a class
of proteins called histones. These proteins wrap DNA into repeating units called
nucleosomes, which are the fundamental building blocks of chromatin. Remarkably,
histones are decorated with an array of distinct chemical modifications or
'tags' that control their function and the underlining DNA they package. While
much has been learned about the modification types and where on the histones
these modifications reside, a great deal less is known about how histone
modifications contribute to chromatin biology. One modification in particular,
methylation of lysine 79 on histone H3 (H3K79), is mediated by the histone
methyltransferase Dot1 and contributes to gene silencing, cell cycle control and
DNA damage response in yeast. The mammalian counterpart of yeast Dot1, DOT1L, is
essential for regulation of embryonic development, cardiac function and
promotion of leukemia. Despite an established role of Dot1 and H3K79 methylation
in cellular growth and development, the mechanisms whereby Dot1 contributes to
chromatin organization and gene expression are poorly understood. Thus, the
long-term goal of this project is to uncover the fundamental mechanisms by which
Dot1 functions in cellular biology. This goal will be accomplished by addressing
the following key questions: 1) How is Dot1 recruited to chromatin to methylate
H3K79? 2) Once on chromatin, do other histones and their modifications
contribute to Dot1-mediated H3K79 methylation? 3) How does H3K79 methylation
actually signal in chromatin to regulate chromatin organization and gene
transcription? Our hypothesis is that Dot1 and H3K79 methylation contribute to
gene expression through multiple, regulated steps that include i) recruitment
and binding of Dot1 to chromatin, ii) functional interplay of histone tails that
stimulate methylation followed by iii) the recruitment and/or exclusion of
effector proteins that "read" H3K79 methylation. Because Dot1 and H3K79
methylation are conserved across species, we will address the foregoing
questions by taking advantage of the highly tractable model organism
Saccharomyces cerevisiae, using a combination of biochemical and genetic
experiments. These studies will provide fundamental insights into how histone
methylation contributes to chromatin organization and gene
expression.&lt;br/&gt;&lt;br/&gt;Broader Impacts: This project will provide
opportunities for undergraduate students, graduate students, and post-doctoral
trainees to experience scientific discovery, learn modern technologies and
develop critical thinking, planning, and communication skills. The proposed
experiments involve a variety of classical genetics, molecular biology,
biochemical, and proteomic approaches making them excellent vehicles for
training new scientists. Projects involving yeast are particularly amenable to
undergraduate participation because yeast cells are easy to grow and manipulate.
There will also be numerous opportunities for participation by underrepresented
minorities through my involvement with the UNC Post-Baccalaureate Research
Education Program (PREP), for which I am a preceptor. As our studies will
produce high-impact findings, we will disseminate our discoveries to the
scientific community and the general public through open access journal
publications in addition to local and national on-line news outlets. Along with
my plans of partnering with the Kenan Fellows for Curriculum &amp; Leadership
Development Program, which pairs K-12 teachers with university mentors to
develop new teaching modules at local schools, this proposal will have a
significant and concrete impact on scientific education at multiple levels.