* 1339393
* Deleterious Mutations and the Response to Strong Selection in a Self-fertilizing Crop
* BIO,IOS
* 06/15/2014,05/31/2018
* Peter Morrell, University of Minnesota-Twin Cities
* Standard Grant
* Anne W. Sylvester
* 05/31/2018
* USD 1,161,444.00

PI: Peter L. Morrell (University of Minnesota - Twin
Cities)&lt;br/&gt;&lt;br/&gt;CoPIs: Kevin P. Smith (University of Minnesota -
Twin Cities) and Justin C. Fay (Washington
University)&lt;br/&gt;&lt;br/&gt;Through examination of the prevalence of
deleterious mutations, the project will address a longstanding issue of both
theoretical and practical importance. Important crops such as barley, rice,
soybeans, and wheat are inbreeding and deleterious mutations could be of greater
consequence on yield in these crops, as the mutations will more frequently occur
in the homozygous state in inbreds than in hybrid cultivars. Once identified,
deleterious mutations can be eliminated through marker-assisted selection using
genomics tools that are independent of the cost and inaccuracy of phenotyping to
increase yield and reduce disease prevalence in crop plants. The barley system
to be used in this study is ideally suited to translate our understanding of the
nature and importance of deleterious mutations to application within a
nationally organized and primarily public sector plant breeding community. USDA
regional genotyping laboratories serve barley and wheat breeding programs across
the country, using contemporary genotyping platforms to perform selection with
informative SNPs. Several large collaborative research projects in barley are
developing next generation mapping populations, a centralized trait and genotype
database and analysis tools, and other resources that can be employed to further
explore the role of deleterious mutations in barley. The public nature of the
national barley breeding community will make application of this research in
commercially relevant breeding lines broadly visible to the agricultural
research community. This provides a uniquely open research system where results
can serve as a model for other plant species that are used for food, fuel, and
other natural products. Finally, the project will provide interdisciplinary
training opportunities for a postdoctoral scholar, a graduate student, and an
undergraduate in crop genomics, evolutionary genetics, and computational
biology. Through involvement with outreach programs at both universities, the
project will engage students from populations underrepresented in the
sciences.&lt;br/&gt;&lt;br/&gt;The research aims to characterize the potentially
conflicting effects of self-fertilization and strong selection on the proportion
of deleterious mutations harbored in individual plant genomes. Using
resequencing of barley exomes from wild accessions, landraces (local cultivars),
and modern elite (high yielding) inbred lines, the project will determine the
extent to which selection for yield has affected the proportion of deleterious
mutations present across genomes. Given sufficient population sizes, active
selection of progeny that are more productive than either parent clearly has the
potential to reduce the proportion of deleterious mutations present in elite
lines. Nevertheless, strong selection for alleles that improve inbred lines or
adaptation to agronomic environments may also increase the frequency of linked
deleterious mutations. Genome-level comparisons will permit the identification
of loci subject to artificial selection and estimation of the degree to which
barley breeding programs have eliminated deleterious mutations across the genome
while simultaneously contaminating the genome with deleterious mutations through
hitchhiking at agriculturally important loci. All data generated in this project
will be accessible through long-term repositories such as the NCBI SRA,
GrainGenes and the Triticeae Toolbox (T3) database. Germplasm used in this
project will be available upon request and through the USDA-ARS National Plant
Germplasm System and Grin-Global.