* 1339362
* NutriNet: A Network Inspired Approach to Improving Nutrient Use Efficiency (NUE) in Crop Plants
* BIO,IOS
* 09/01/2014,08/31/2020
* Gabriel Krouk, New York University
* Standard Grant
* Diane Jofuku Okamuro
* 08/31/2020
* USD 2,885,527.00

PI: Gloria Coruzzi (New York University)&lt;br/&gt;&lt;br/&gt;CoPIs: Dennis
Shasha (New York University), Stephen Moose (University of Illinois at Urbana-
Champaign), Sandrine Ruffel and Gabriel Krouk (INRA, Montpellier,
France)&lt;br/&gt;&lt;br/&gt;Senior Personnel: Manpreet Katari (New York
University) and W. Richard McCombie (Cold Spring Harbor
Laboratory)&lt;br/&gt;&lt;br/&gt;Improving nutrient use efficiency (NUE) in crop
plants is critical to ameliorating the impacts of future climate change and to
sustainably increasing global crop yields to meet projected food and energy
demands. The NutriNet project seeks to identify and compare biologically
connected gene networks whose collective expression patterns are predictive of
phenotypic variation in NUE in Arabidopsis and maize. This cross-species network
inspired approach may be readily applied to many other economically important
traits, and adapted to other crops. The advantages of the NutriNet approach
include: i) exploiting detailed datasets for gene and protein interactions in
Arabidopsis, to inform analysis of data poor crop species, and ii)
identification of robust network modules that can be applied in molecular
breeding programs. Proof-of-principle studies will demonstrate both conserved
and species-specific features of network modules (but not necessarily candidate
genes) regulating nitrogen assimilation and remobilization. The new knowledge
generated in this project will consist of gene discovery, elucidation of
regulatory circuits, and a better understanding of the molecular basis for
nutrient physiology that drives crop productivity. As a practical deliverable,
network-inspired molecular breeding tools will be developed that are expected to
perform better than candidate gene approaches in selecting genotypes with
improved NUE. The NutriNet team links expertise in systems biology, plant
physiology, and crop genomics, to increase the fundamental understanding of crop
utilization of nutrients. The project offers multidisciplinary training to
postdoctoral scientists, graduate and undergraduate students in New York and
Illinois. High school students will be introduced to systems biology through co-
mentorship by biologists and computer scientists at NYU. In addition, because of
the broad public interest in nutrient-efficient crops, the project team will
engage audiences through outreach activities at the Illinois Corn Breeders'
school to leverage the pioneering efforts and long history of the University of
Illinois in concert with breeders to understand crop responses to nutrients and
breeding for nitrogen utilization.&lt;br/&gt;&lt;br/&gt;Recent advances in
genome sequencing, functional genomics, and computational tools enable a systems
level understanding of key physiological and developmental processes including
NUE in the model plant Arabidopsis thaliana. However, translating this "network
knowledge" from Arabidopsis to crops to potentially enhance agriculturally
important phenotypes in crop species remains challenging. The goal of this
project is to develop network-connected gene modules that can be used to predict
the outcome of NUE in crops, by exploiting Arabidopsis network knowledge. The
project approaches this goal by developing novel data sets and analytical
methods as follows: 1) integrating phenotypic variation for NUE with new and
existing data for nutrient-responsive gene expression profiles which allows for
the development of a training set that exploits the power of genetic diversity
from both Arabidopsis and maize; 2) using a split-root experimental design to
identify evolutionarily conserved gene mechanisms that function in root-shoot
N-signaling that may control root foraging for nutrients in the soil; 3)
defining network modules predictive of NUE traits using a bioinformatics
pipeline to combine Arabidopsis "network knowledge" with maize transcriptome
data to generate NutriNet modules that will be validated using and tested for
their ability to predict NUE based on gene expression; and, 4), using
information derived from NutriNet modules to select individual genotypes that
possess optimal NutriNet configurations from diverse germplasm pools which will
then be evaluated for improved NUE traits in the lab (Arabidopsis) and field
(maize). A comparative analysis of lab-to-field results will directly assess the
"translation" of network knowledge from Arabidopsis to maize to serve as a
general proof-of-principle, which can be applied to other networks and species.
All data and biological resources will be available upon request and accessible
through long-term data and germplasm repositories.