* 1344299
* INSPIRE Track 1: Epigenomic analysis of single cells with controlled perturbations
* BIO,MCB
* 10/01/2013,09/30/2016
* Joseph Ecker, The Salk Institute for Biological Studies
* Continuing Grant
* Karen Cone
* 09/30/2016
* USD 799,999.00

This INSPIRE award is partially funded by the Genetic Mechanisms and Cellular
Dynamics and Function Programs in the Division of Molecular and Cellular
Biosciences, by the Plant Genome Research Program in the Division of Integrative
Organismal Systems, and by the Division of Emerging Frontiers in the Directorate
for Biological Sciences, and also by the Biotechnology, Biochemical, and Biomass
Engineering Program in the Division of Chemical, Bioengineering, Environmental
and Transport Systems in the Directorate for
Engineering.&lt;br/&gt;&lt;br/&gt;Epigenetic modifications to the DNA and
protein components of chromatin are important determinants of gene expression,
and dynamic epigenetic changes are thought to underlie the ability of organisms
to rapidly respond to environmental perturbations. Much of what is understood
about such regulation has been obtained through so-called ensemble approaches,
which determine epigenetic modifications from cells isolated from mixed tissue
types; thus, the data represent an average of epigenetic and gene expression
profiles of multiple cells. Recent technical advances have made it possible to
obtain DNA sequence information and gene expression profiles for single cells,
but it is not yet possible to obtain epigenetic information from single cells.
The ability to monitor dynamic epigenetic profiles in single cells would
revolutionize the field of epigenetics by enabling hitherto impossible studies
of stochasticity, stability, and heritability of epigenetic responses. This
project seeks to develop and implement an innovative pipeline for controlled
hormone and stress perturbation of plant cells, followed by single-cell analysis
of genome-wide DNA methylation profiles. An integrated micro- and nano-fluidic
device will be built to enable single-cell isolation from specific cell types
and subsequent chromatin sorting. Then DNA methylation information will be
extracted from the isolated cells via new kinetic analysis of real-time
sequencing data produced by ultrasensitive FRET-based single molecule
sequencing. If successful, this new technology will enable analysis of
epigenomic profiles of single cells, thus making it possible to address
fundamental questions about epigenetic regulation in a revolutionary new way,
not only in plants, but also in mammalian and microbial cells.
&lt;br/&gt;&lt;br/&gt;In addition to the scientific impact, the project will
have broad educational impacts by offering opportunities for cross-disciplinary
training at the interfaces of molecular biology and nano-engineering.
Postdoctoral and graduate students will be involved directly in carrying out the
research, and the technology developed in the project will be incorporated into
hands-on laboratory training modules for both graduate and undergraduate
students.