* 1349449
* CAREER:Resources for mixed model association mapping of complex traits
* BIO,DBI
* 06/01/2014,02/29/2020
* Alkes Price, Harvard University
* Continuing Grant
* Peter McCartney
* 02/29/2020
* USD 599,475.00

The goal of this project is to devise new statistical methods for identifying
genetic variants associated to disease resistance and other traits in plants,
livestock, and model organisms such as fruit flies and mice. Specifically, the
research will focus on mixed model association methods, a class of methods in
which association statistics are computed by considering all genetic variants
simultaneously in order to increase power to detect true associations and to
avoid false-positive associations. Although mixed model association methods have
previously been applied to studies in these species, existing methods are based
on statistical assumptions that are unlikely to hold in real data sets,
particularly in the large whole-genome sequencing data sets of the future. The
new methods produced by this research will address those limitations. The
investigator will use the research activities in this project to help teach and
train numerous students, including those from underrepresented groups. The
project will release publicly available, open-source software implementing the
new methods. Making these statistical methods and software widely available to
other researchers is likely to lead to new scientific discoveries and insights
when the methods are applied to large whole-genome sequencing data sets.

The research will produce new mixed model association methods and software for
application to complex trait association studies in plants, livestock and model
organisms that will address limitations of existing methods, increasing power
while controlling false-positive associations. Specifically, the methods
developed will account for general genetic architectures, moving beyond the
"infinitesimal" genetic architecture with all markers associated that underlies
existing mixed model association methods; will account for non-randomly
ascertained traits; and will enable pooled analyses of rare variants in a mixed
model framework. All of these new features are expected to increase statistical
power to detect true associations. The broader impacts of this project will
include: enhanced computational infrastructure for conducting mixed model
association mapping via the continued release of open-source software; multi-
disciplinary synergy from research, collaborations and conferences that will
bring together plant, livestock, model organism and human geneticists; and
increased participation of underrepresented groups via the HSPH Summer Program
in Quantitative Sciences. Software produced by this project will be available at
http://www.hsph.harvard.edu/alkes-price/software/.