* 1350677
* CAREER: Comparative Network Biology to Study the Evolution of Regulatory Networks
* BIO,DBI
* 08/01/2014,07/31/2020
* Sushmita Roy, University of Wisconsin-Madison
* Standard Grant
* Peter McCartney
* 07/31/2020
* USD 506,579.00

Central to how living cells accomplish diverse biological functions are
regulatory networks that control what genes need to be activated under different
environmental conditions. As evolution is the ultimate tinkerer of living
systems systematic comparisons of how regulatory networks evolve to drive
species-specific differences are critical to understand cellular functions.
Through advances in genomics, it is now possible to measure the activity levels
of almost all genes for many species. This provides a unique opportunity to
systematically compare these gene activity levels across multiple organisms and
link changes in activity to changes in the networks of individual species.
However, this is challenging because, first, such comparisons require the
regulatory networks to be known in not one but multiple species including those
that are poorly characterized, and second, computational methods to compare
molecular datasets across species other than DNA sequence are in their infancy.
This project will address these challenges by developing novel computational
methods to identify and compare regulatory networks across multiple species.

The overarching goal of this project is to establish a comparative network
biology framework to study evolutionary dynamics of regulatory networks. In Aim
1, algorithms for regulatory network reconstruction that combine probabilistic
models of evolution with network reconstruction will be developed, that take
into account both sequence and expression determinants of regulatory networks.
The network reconstruction algorithms will incorporate structural constraints of
regulatory networks such as modular organization, where sets of genes are
coordinately regulated to achieve specific cellular functions. In Aim 2, methods
to identify and correlate regulatory network patterns of divergence to
phenotypic states will be developed. In Aim 3, methods from Aims 1 and 2 will be
applied to published and newly generated expression datasets in yeast and land
plant phylogenies to study the evolution of stress response, carbon metabolism
and plant-microbe relationships. Network predictions will be interpreted and
tested by yeast and plant collaborators in the laboratory. This research project
intersects several computational (graph theory, algorithms, machine learning)
and biological (gene regulation and evolution) areas of research, providing an
excellent training opportunity for scientists at the interface of these areas.
The planned outreach activities include developing a Network biology course, and
network biology boot camp for a broader audience including high school teachers.
The PI will organize summer research internships in Network biology for
undergraduate students from underrepresented and minority groups through the
Computational Biology and Biostatistics Summer Research Opportunities (CBB SROP)
at UW Madison, and participate in general public outreach from the Town Center
at the Wisconsin Institute for Discovery (a public venue for regular school
field trips and Science festivals). Software tools and resources developed by
this proposal will be made publicly available through dedicated websites.