* 1353897
* IDBR: Type A: Whole-genome epigenetic profiling of single-cells
* BIO,DBI
* 08/01/2014,07/31/2018
* Robert Riehn, North Carolina State University
* Standard Grant
* Robert Fleischmann
* 07/31/2018
* USD 524,766.00

Non-Technical Description: An award is made to North Carolina State University
to develop a novel nanofluidic instrument for whole-genome epigenetic profiling
of single cells. Epigenetic modification, and in particular cytosine
methylation, is a modification of genetic material that enables control over
gene activity which can be inherited through cell division and over multiple
generations. Patterns of epigenetic modification can change as a response to the
environment, during differentiation and development, or as part of disease
onset, for example in cancer. In order to fully understand the dynamic character
of cytosine modification patterns, in addition to identifying the the
differences between individuals, and across tissue types within an individual,
it is important to also identify the variability across cells within a tissue.
To perform such mapping, an integrated device and associated hardware, will be
developed and tested on sets of cultured cells that will have high and low
epigenetic variability. The device will include a nanofluidic control system to
conduct measurements. The project will train graduate students in
bioinstrumentation, and will expose undergraduates to micro-and nanofluidic
experiments. Additional educational activities involve a course for elementary
school teachers.

Technical Description: Current techniques for the determination of epigenetic
patterns typically are based on methyl cytosine sensitive sequencing. It yields
an overwhelming quantity of data for single-cells and whole genome for a tissue
sample of 1000 cells. Differentially methylated regions are then found by coarse
graining this large set of data. The instrument developed in this work will map
methyl cytosine patterns with 250 bp-1000 bp resolution using a fluorescently
labeled methylation-sensitive protein, and reading its binding location through
a hybrid optic-electronic readout. All steps from single cell handling to
detection are performed within one single device. Dissemination activities will
include development of a webinar, provision of access to the finished device to
the user community (a few devices will be fabricated and distributed to users),
and training of undergraduate students in the basic principles of operation.