* 1355033
* Population genetics-based codon models
* BIO,DEB
* 09/15/2014,08/31/2019
* Michael Gilchrist, University of Tennessee Knoxville
* Continuing Grant
* Amanda Ingram
* 08/31/2019
* USD 520,000.00

Mammals, viruses, bacteria, and even cancer tumors evolve. Understanding their
evolutionary history can be critical to understanding their biology and
potential for future change. Scientists currently use models to help infer this
history, but most of these models ignore the reality of natural selection. This
research proposal will develop models that allow scientists to estimate these
histories while incorporating selection for certain amino acids into the models.
These new models include parameters reflecting processes from population
genetics, and will be used to infer the strength of selection on codon usage and
amino acid sequence, sensitivity of protein function to different amino acid
properties such as size and polarity, and mutation rates. These models fit
empirical data far better than traditional models, and will more accurately
infer evolutionary histories and processes. While the preliminary work uses
amino acid sequences, the final project will use DNA sequences, allowing even
greater resolution of evolutionary events. Given the importance of understanding
evolutionary history and processes in fields ranging from medicine to forensics
and from agriculture to basic taxonomy, the improvements in methods developed
here will have great utility to both science and society.

This research will create models linking mutation, drift, codon selection, and
amino acid selection for use in phylogenetic inference. Rather than symmetric
transition matrices derived from empirical estimates as used in the past, the
models developed here will allow for twenty different transition matrices (one
for each possible optimal amino acid) which allow for different rates of gain
and loss between pairs of amino acids but which are generated from just a few,
realistic parameters. The models developed here will be evaluated both for fit,
using measures such as AIC, and adequacy, comparing them with each other and
with standard models. Performance will be evaluated using multiple gene datasets
from groups of organisms, ranging from a handful of genes to entire genomes, as
well as from simulated genetic datasets. Preliminary analyses using a
simplification of the proposed models show a dramatic improvement in model fit
compared to traditional models. They also better match and predict empirical
data, which is a standard test of model adequacy. Models will be incorporated
into existing phylogenetics software via a hackathon including software
developers. Outreach will include work with local teachers and development of a
learning module for their high school classes.