* 1355990
* ABI Innovation: Sequence Optimization for Synthetic Biology
* BIO,DBI
* 06/01/2014,05/31/2019
* Steven Skiena, SUNY at Stony Brook
* Standard Grant
* Peter McCartney
* 05/31/2019
* USD 499,881.00

A grant is awarded to Stony Brook University to generalize and expand
computational approaches for designing gene sequences into a coherent set of
tools for three areas of molecular biology: 1) sequence design for optimized
gene expression; 2) signal identification and location; and 3) self-assembly.
This project will improve our understanding of several sequence phenomena that
occur in genomes, such as codon-pair bias, how RNA secondary structure and tRNA
usage modulates gene expression, and how to use these results to develop gene
design software to optimize gene expression.&lt;br/&gt;&lt;br/&gt;A new approach
to identify the locations of critical DNA or RNA sequence signals (e.g. binding
domains, secondary structures, and mRNAi targets) couples large-scale synthesis
with sophisticated designs based on ideas from combinatorial group testing. This
project will improve the efficiency and generality of this procedure, and will
develop new homology-based software tools to identify candidate signals prior to
employing combinatorial search approaches. New array-based oligo synthesis
technologies provide access to tens or even hundreds of thousands of short
custom sequences, but even greater power will become enabled if oligonucleotides
can be designed to self-assemble into chromosome-sized sequences. Recombination
systems in yeast provide the platform to conduct such experiments. This project
will develop algorithms for the design of large-scale, self-assembling
sequences, to best exploit new array-based synthesis technologies, and test how
they work in practice.&lt;br/&gt;&lt;br/&gt;This collaboration between
computational and life sciences researchers will advance both disciplines,
through new results in combinatorial algorithm design and discrete optimization
as well as fundamental discoveries regarding gene expression, signal detection,
and self-assembly. Broader impacts of this project will result in software and
experimental tools to advance broad areas of molecular biology, including the
design of vaccines. Software and results of this project will be available from
the website http://www.cs.stonybrook.edu/~skiena/dna.