* 1355071
* Collaborative Research: Bayesian Model Checking for Phylogenetics in the Post-Genomic Era
* BIO,DEB
* 08/01/2014,07/31/2019
* Jeremy Brown, Louisiana State University
* Standard Grant
* Simon Malcomber
* 07/31/2019
* USD 418,252.00

Diagrams of evolutionary relationships (phylogenetic trees) for species and
genes are widely employed in biological research, including the fields of
medicine, epidemiology, forensics, conservation, evolutionary biology and
agriculture. This research project will explore new ideas and develop new
software tools to improve the accuracy by which phylogenetic relationships are
determined; in this way the research will contribute to improved understanding
and decision-making for a broad range of scientific disciplines and practical
applications. Results from this research will be broadly disseminated, including
in-person and online training opportunities to familiarize researchers in the
relevant disciplines with these newly developed computer-based analytical tools.
Further, the research activities will involve the participation and training of
a postdoctoral scholar, a graduate student, and several undergraduates at
Louisiana State University (LSU) and the University of Hawaii at Manoa. This
project will be incorporated into a seminar series at LSU focused on increasing
awareness of computational biology among undergraduate
students.&lt;br/&gt;&lt;br/&gt;Phylogenetic trees are now routinely inferred
from enormous genome-scale data sets, revealing extensive variation in apparent
phylogenetic signal across loci. However, no general tools currently exist to
objectively and quantitatively assess how much of this variation is due to
biological processes and how much is caused by methodological error.
Distinguishing between true variation and error is the problem to be studied in
this project, as resolving this issue is essential for robustly resolving the
Tree of Life and for understanding genomic evolution. The goal of this work is
to give researchers the tools to identify and avoid situations where
phylogenetic inferences are unreliable. These tools will be implemented in open-
source software (RevBayes and R), and will be easily extensible to many types of
phylogenetic inference beyond those in this project. This research will
implement suites of existing, alternative statistical approaches employing
Bayesian posterior prediction to rigorously assess absolute fit of phylogenetic
models to evolutionary data, and how this fit impacts the reliability of
inference. Simulations comparing performance of alternative models will focus on
three types of inferences: (i) estimation of individual gene trees, (ii)
estimation of species trees from many genes, and (iii) comparative analysis of
continuous traits. These approaches will be applied to exemplar empirical
questions, including the placement of turtles among amniotes using several
recently published genome-scale data sets. These data contain surprising and
massive heterogeneity in phylogenetic signal regarding the placement of turtles,
and thus form an excellent case study.