* 1356078
* ABI Innovation: Creating Complete and Accurate Alternative Splicing Repertoires from RNA-seq Data
* BIO,DBI
* 08/01/2014,07/31/2018
* Liliana Florea, Johns Hopkins University
* Standard Grant
* Jennifer Weller
* 07/31/2018
* USD 630,122.00

The project aims to produce a suite of bioinformatics tools to accurately and
comprehensively identify alternative splicing variation and present this
information to researchers through a web accessible database. Alternative
splicing is an intrinsic property of eukaryotic genes. A gene can be spliced in
multiple ways to produce different variants depending on the tissue,
developmental stage, or disease versus normal condition, with each variant
having a distinct function. Creating a repertoire of alternative splicing
variations of genes is therefore critical for understanding the biology of a
species. Recently, fast and cost-effective next generation sequencing has made
it possible to explore the repertoire of genes and their splicing variations in
a wide variety of species. However, interpreting the short sequencing reads into
gene and alternative splicing annotations has significant challenges, and
existing tools cannot always reconstruct alternative splicing patterns with high
accuracy. The project seeks to produce, first, a bioinformatics tool to identify
alternative splicing variations from sequencing reads in more detail and more
accurately than previously possible. Secondly, the methods will be applied to
build a comprehensive catalog of alternative splicing events in several plant
species, taking advantage of the growing amounts of data being generated. The
project will produce open source software that can be used by biologists for
both research and education, and will contribute to the larger effort to recruit
students from diverse backgrounds into interdisciplinary science, by creating
research opportunities for undergraduate and high-school students through summer
internships.&lt;br/&gt;&lt;br/&gt;Alternative splicing (AS) is a widespread
mechanism with an important role in creating gene and functional diversity. Next
generation sequencing of cellular RNA (RNA-seq) has made it possible to explore
the repertoire of splice variants in a cell type or species. However, current
bioinformatics tools have difficulty in capturing AS variation in detail, and
have not scaled up with the increasing number of samples in a typical
experiment. To answer these needs, the first aim is to develop a first-of-its-
kind next-generation tool for reconstructing gene and splice variants from short
RNA-seq reads, simultaneously from large numbers of samples. It will capture AS
variation both more fully and more accurately than current programs, including
fine variations and hard-to-detect types of variations. The approach is to
combine linear programming techniques, with statistical models of RNA-seq
"noise", and with scalable splice graph representations of genes, and exploits
similarity between expression profiles of related samples to increase
efficiency. The second aim is to produce the first comprehensive catalog of AS
events in plant species, with web-based visualizations and analysis
capabilities, building on the splice graph model as a compact representation of
genes. All software will be open source and available without restrictions for
all, for research and education. Additionally, the project will create new
teaching materials and opportunities for training and recruiting the next
generation of scientists, in particular from among under-represented groups and
Baltimore inner-city high-schools students, through Summer internships. All
software, teaching materials, and the database will be accessible from
http://ccb.jhu.edu/people/florea/research/ .