* 1356065
* Collaborative Research: ABI Innovation: Towards high-performance flexible transcription factor-DNA docking
* BIO,DBI
* 08/01/2014,07/31/2018
* Srinivas Aluru, Georgia Tech Research Corporation
* Continuing Grant
* Peter McCartney
* 07/31/2018
* USD 156,722.00

Molecular interactions between proteins and DNA play crucial roles in many
fundamental biological processes such as DNA modification and gene regulation.
Transcription factors are a special group of proteins that interact with
specific DNA sequences in the genome to regulate gene expression. Knowledge of
transcription factor-DNA interactions at the structural level can help elucidate
the fundamental mechanisms of protein-DNA recognition. This project aims to
develop novel computational algorithms for flexible protein-DNA docking. The
focus of this project is to model transcription factor-DNA complex structures.
Computational docking, by filling the gap in the complex structure landscape due
to the limitation of experimental methods, has become a cost-efficient
alternative to experimental approaches to understanding complex models. A better
understanding of protein-DNA interaction can help stimulate innovations in drug
designs. The techniques developed in this project can be readily applied to
other types of computational docking studies. This project will provide students
an interdisciplinary and collaborative research environment. The intellectual
challenges and educational opportunities from this project will help prepare the
graduate students and postdoc involved to become independent researchers in
related fields.

Computational docking represents a grand challenge in structural
bioinformatics. One major bottleneck of protein-DNA docking is the sampling of
the enormous search space. Compared to rigid-docking that assumes rigid protein
and DNA structures, sampling of flexible protein-DNA docking is much more
difficult. In flexible docking, in addition to exploring the relative positions
of the protein and DNA molecules, the flexibility of protein and DNA needs to be
also considered since protein and DNA molecules undergo conformational changes
upon interaction. Efficient sampling algorithms and sped-up computation will be
critical for achieving higher accuracy in protein-DNA docking. Novel algorithms
for efficient sampling of transcription factor-DNA conformations will be
developed to simulate the molecular recognition mechanism. The results of this
research project will be published in international peer-reviewed journals and
presented at scientific meetings to ensure broad dissemination to the scientific
community. http://guolab.uncc.edu