* 1356669
* ABI innovation: Integration of flux balance analyses with data mining and 13C-labeling experiments to decipher microbial metabolisms
* BIO,DBI
* 07/01/2014,06/30/2018
* Yixin Chen, Washington University
* Standard Grant
* Jennifer Weller
* 06/30/2018
* USD 486,510.00

Microorganisms play important roles in ecology, biogeochemical cycles, human
diseases, bioremediation and bioenergy. Currently, high-throughput sequencing is
being used to map the genomes of microbial species. However, the DNA sequence of
a microbe does not provide a complete understanding of its functioning. To
bridge the knowledge gap between genotype and phenotype, metabolic flux analysis
is an important phenomic tool to investigate in vivo enzymatic activities.
Analysis of metabolic fluxes can identify bottleneck pathways in the
biosynthesis of desirable products, decipher the function of unknown genes,
discover new enzymes, and reveal the mechanisms of diseases. In this project, a
user-friendly metabolic flux analysis platform will be developed to provide a
robust cyberinfrastructure for biologists, helping them analyze large amounts of
phenomic data efficiently. In addition, this platform includes an open source
database for storing and disseminating flux analysis data on diverse microbial
metabolisms. This database can assist metabolic flux analyses of new
microorganisms, enabling the systems biology community to benefit from the fast
advancement in "Big Data" technology. Ultimately, this platform can be a
springboard for future development of high throughput methodologies to analyze
diverse biological systems. The broader impact of this project not only includes
educational efforts for high school students (especially minority and under-
represented groups) to promote their inquiry and interests in STEM fields, but
also development of a wiki-styled website and discussion forum for flux analysis
technologies and applications. &lt;br/&gt;&lt;br/&gt;Current systems biology
studies (for example, transcriptomics) rely on model organisms for genome
annotation and have limited power to reveal novel metabolic pathways. In
addition, post-transcriptional and post-translational regulations hinder the
possible phenotypic information that can be determined from genomic approaches.
Thereby, it is of great value to build a new metabolic flux analysis platform to
decipher microbial metabolisms and metabolic regulations. This project has three
tasks to develop novel capabilities for metabolic flux analysis. The first task
will be to build a comprehensive carbon-fate map for 13C-assisted pathway
identification and to provide effective computational algorithms for
13C-metabolic flux analysis. Thereby, this flux analysis platform can precisely
quantify the pathway activities using the labeling information from 13C-tracer
experiments. The second task will be to build a database of published microbial
fluxomic results, which can then be analyzed via data mining approaches to
predict pathway linkages and novel microbial metabolisms in new species. The
third task will be to create new approaches to integrate genome-scale flux
balance analysis with both 13C-labeling and data mining results to precisely
characterize microbial metabolisms. Once the new model platform is developed, it
will be tested using case studies. Based on available experimental data on
Shewanella metabolic flux analyses, the model applicability will be validated
and improved. Ultimately, this new platform can be widely used by the systems
biology society to provide new insights into diverse microbial species. The
project web link is: http://tang.eece.wustl.edu/