* 1361375
* Computational Cell Motility Model Educed from Single-Cell and High-Throughput Phenotype Analysis
* MPS,DMS
* 08/15/2014,07/31/2019
* Richard Superfine, University of North Carolina at Chapel Hill
* Continuing Grant
* Junping Wang
* 07/31/2019
* USD 1,053,451.00

The mechanical properties and mechanical sensing of cells is critical to the
identity and homeostasis of the cell. A wide range of studies have shown
correlations between a variety of mechanical properties of cells and their
ability to differentiate, to modulate gene expression, and to direct motility.
These processes are critical for during the development of the embryo and appear
to be central in cancer. While correlations are being established between the
mechanical measurements, biochemistry and motility, and cancer behavior, there
is a striking lack of integrated understanding of mechanics in these processes.
In this project, the investigators develop an integrated computational model
that will incorporate biochemistry, cell structure, and cell motility dynamical
behavior with the ability to compute mechanical properties as measured by
experimental techniques. This will allow the research team to understand first,
the underlying biological phenomena, and second, how these tools can connect
biochemistry and genomics with cell mechanics. Further, to fully characterize
the role of mechanics in cellular processes, this project will bring high-
throughput methods to cell biophysics. This will allow the investigators to
connect genomics and proteomics to the mechanical properties of cells across the
spectrum of cancers and cell/tissue models. The results of this project will
provide guidance for the use of experimental tools in diagnosis of cancer and in
developing strategies for treatment. &lt;br/&gt;&lt;br/&gt;Models of cell
motility have established six principle biophysical constituents/phenomena: a.
lamellipodia extension, b. flow of cortical actin, c. membrane tension, d.
traction, e. a central region consisting of an active gel, and f. a viscoelastic
nucleus. Finally, cells sense their mechanical environment and forces that are
applied to them, and these cues can direct their motility. These phenomena are
not independent, and an understanding of the role of the leading edge of cell
protrusions have to be combined with cell retraction at the rear, global shape
distortions, cell-matrix forces, and the global fate and trafficking of actin in
the cell. There is no current computational model that combines these features
to develop a self-consistent understanding of the primary mechanisms of
motility. Correspondingly, a wide range of cell biophysical characterization
techniques have been developed ranging from probe-based methods of atomic force
microscopy (AFM), and active and passive bead based methods, traction force
measurements and global cell mechanical methods. Understanding the cell
structures and processes that are being probed by these methods depends on
biochemical interventions and modeling, and this project employs an integrated
computational model to interpret multiple mechanical assays on single cells or
on cell populations. The computational model that is developed will be tested
against a battery of mechanical and structural studies on selected cancer cell
motility models, with the investigators' laboratory performing simultaneous
multi-mechanical measurements on individual cells, and performing high-
throughput mechanical studies on cell populations. This unique integrated
computational/experimental approach will allow cell mechanical studies to be
integrated with genomic/proteomic methodologies.