* 9614074
* Empirical Potential Functions for Prediction of Protein     Folding
* BIO,DBI
* 08/01/1997,07/31/2000
* Gordon Crippen, Regents of the University of Michigan - Ann Arbor
* Continuing Grant
* THOMAS QUARLES
* 07/31/2000
* USD 203,423.00

One of the grand challenges in biophysics is the prediction of protein folding
from the amino acid sequence. The investigator and coworkers have determined an
empirical mathematical function that can recognize the native conformation for
many proteins compared to many thousands of decoy conformation constructed from
pieces of other protein crystal structures. They are now improving this function
by testing different functional forms, training against more natives, and
challenging it with wider classes of nonnative folds. If the function can be
made smooth enough, it could quide a search toward the native structure as part
of a structure prediction algorithm. In the current excitement over the human
genome project, it is easy to forget that gene sequences are like blueprints for
a house, and we are more concerned with the house than the blueprints. At the
moment, we have no idea about the biological and medical role that about a third
of the sequences play once they are translated into proteins, the three-
dimensional molecular machines that make cells work. How proteins function
depends critically on how these long chains fold up into particular shapes. This
project is concerned with making sense of those thousands of mysterious
sequences by trying to predict their three-dimensional structures. Workers in
this laboratory have developed computer methods to choose the correct structure
for a given sequence out of a lineup of many incorrect ones, as in a huge
multiple choice exam. They are now moving toward answering an essay exam, where
the sequence is given, and the structure must not merely be recognized, but
constructed. Until this can be done, much of the value of the massive sequencing
projects cannot be realized.