* 9630191
* Interactions Among Sequences, Structures and Proteins       Involved in Viral Replication
* NONE,NONE
* 02/01/1997,10/31/2000
* Anne Simon, University of Massachusetts Amherst
* Continuing grant
* Susan Porter Ridley
* 10/31/2000
* USD 319,754.00

9630191 Simon Virus-encoded RNA-dependent RNA polymerases (RdRp) are responsible
for a two-stage replication process: the RdRp synthesizes complementary minus-
strands that in turn become templates for plus-strand synthesis. A number of
different structural elements have been implicated in replication including
pseudoknots, stem-loops and tRNA-like structures. Less understood viruses whose
genomic RNA(s) end in a 3'-terminal hydroxyl group such as turnip crinkle virus
(TCV) will be utilized as a model to study sequences and structures involved in
replication. TCV is among the smallest and simplest of the plant RNA viruses and
is associated with subviral sat-RNAs and DI RNAs, making possible the use of
these simple, non-protein-coding RNAs as templates for replication studies.
Protoplast and in vitro systems have been developed to study replication of
genomic and subviral RNAs and virus-encoded RdRp subunits have been expressed in
insect cells using baculovirus vectors. A 29 base sequence at the 3' end of
plus-strand satC (356 bases), which includes a stable hairpin and the CCUGCCC
motif found at the ends of all TCV-associated RNAs, is required and sufficient
for minus-strand synthesis in vitro. Mutagenesis studies indicated that the
stability of the stem, and not the sequence of the stem or loop, is important
for promoter function. Minus-strand satC was found to have two independent
promoters, one within the 3' terminal 49 nucleotides and a second near positions
310 to 334. The following experiments will be performed: (1) The
sequence/structural requirements for plus-strand and minus-strand promoters of
satC will be determined using further mutagenesis and an in vitro selection
strategy (SELEX), where a large pool of random polynucleotide sequences are
"selected" for further amplification if they comprise a functional promoter. (2)
Cis-signals involved in replication in vivo and transcription in vitro for satD
will be determined using deletions and site-directed mutagenesis. (3) The
importa nce of the sequence and location of the CCUGCCC 3'-terminal motif in
relation to the 3' hairpin for satC and TCV will be examined. In addition, we
will determine if deletion of the 3' end of TCV and satC also result in the
formation of "new" RNAs in vivo with replacement of the CCUGCCC terminal motif
as was found for satD. (4) Binding studies using the TCV-encoded RdRp subunits
p28 and p88 to promoter sequences associated with satC and satD will be
performed using gel retardation analysis, filter binding, and hydroxyl radical
footprinting. (5) The effect of coat protein on replication of TCV subviral RNA
plus and minus strands in protoplasts will be determined using wild-type TCV and
TCV with a coat protein open reading frame deletion (TCV(CP). In addition, using
RdRp-active extracts from protoplasts inoculated with TCV(CP, we will assay for
the effect of different amounts of highly purified TCV(CP on in vitro
transcription of satC and satD plus- and minus -strands. %%% The process by
which a plant RNA virus can reproduce itself by copying its gene is very
complex. Turnip crinkle virus RNA replication can be carried out in a test tube.
These studies will focus on the contribution of specific sequences of the RNA
interacting with the enzymatic activities that produce new viral RNA copies.
These studies may lead to understanding how to manipulate and control RNA virus
activities in plant and animal systems. ***