* 9630604
* Molecular-Genetic Study of Aspartate Aminotransferase Genes/Isoenzymes in Arabidopsis thaliana
* BIO,MCB
* 08/15/1996,07/31/1999
* Gloria Coruzzi, New York University
* Continuing Grant
* Hector E. Flores
* 07/31/1999
* USD 330,000.00

9630604 Coruzzi A combined molecular-genetic and biochemical approach is being
used to study the function of distinct isoenzymes of aspartate aminotransferase
(AspAT) in the model genetic plant Arabidopsis thaliana. A major goal is to
identify the in vivo role played by individual AspAT isoenzymes localized in the
cytosol, chloroplast, mitochondria or peroxisome. Arabidopsis mutants deficient
in either of the two major AspAT isoenzymes (cytosolic AAT2 or chloroplastic
AAT3) have been isolated and partially characterized. Additional Arabidopsis
mutants defective in the other "minor" AspAT isoenzymes, (mitochondrial AATl and
peroxisomal AAT) will be created using a dominant-negative approach. Arabidopsis
mutants defective in each AAT isoenzyme will be subject to physiological and
metabolic analysis to define the in vivo role of the "missing" isoenzyme. In
addition to these genetic studies, the cloned genes encoding each AspAT
isoenzyme are also under investigation. RFLP mapping has been used to determine
which ASP cloned genes co-segregate with an isoenzyme mutation. Preliminary
results demonstrate that the mutants are deficient in cytosolic AAT2 are
affected in the ASP2 gene for cytosolic AAT2. The phenotypic analysis of the
aat2 mutant suggests that ASP2 encodes a key isoenzyme controlling the
assimilation of nitrogen into aspartate used for intercellular nitrogen
transport. Gene expression studies have also shown that ASP2 plays a key
regulatory role as its expression is regulated by light and/or metabolic
control. The specific aims of this project are to: 1) Demonstrate by in vitro
organelle uptake that ASPl and ASP3 encode mitochondrial and peroxisomal forms
of AspAT. 2) Determine the cell-specific expression pattern of each ASP gene by
in situ hybridization and/or by using promoter-GUS fusions. 3) Elucidate the
mechanism of metabolic regulation for ASP2. 4) Map all the ASP genes. 5)
Determine whether the aat3 mutation deficient in chloroplast AAT3 is in an ASP
structural or re gulatory gene. 6) Define the molecular lesion in each aat2 and
aat3 mutant. 7) Isolate deletion alleles of aat2, aat3 and/or regulatory
mutants. 8) Create mutants in mitochondrial AATl or peroxisomal AAT using anti-
sense or dominant-negative transgenic technology. 9) Perform a detailed
characterization of the aat mutants including genetic, physiological and
biochemical tests. 10) Create transgenic plants which ectopically express
specific ASP genes defined to be key by mutant analysis. %%% This project is a
molecular genetic analysis that focuses on an important family of related
enzymes, aspartate aminotransferase, (AspAT) involved in nitrogen metabolism in
higher plants. AspAT functions to 1) convert assimilated nitrogen into aspartate
for intercellular nitrogen transport to developing organs (e.g. seed), 2) act as
the catalyst in innumerable transamination reactions to donate nitrogen for the
synthesis of nitrogen containing compounds, 3) maintain a balance between
organic nitrogen and carbon, 4) shuttle reducing equivalents between subcellular
compartments. These molecular-genetic studies in Arabidopsis will help define
which particular AspAT isoenzyme is involved in each of the above processes.
Moreover, the engineering of the ASP genes in transgenic plants has potential
for improving nitrogen-use in transgenic crop plants. Finally, these studies
will also serve as a model for ways in which to dissect steps in other complex
multi-isoenzyme pathways in plants. ***