* 9601845
* Acquisition of a Multi-Processor Computer for Computational Physics and Structural Biology
* BIO,DBI
* 10/01/1996,09/30/1998
* David Cowburn, Rockefeller University
* Standard Grant
* Lee C. Makowski
* 09/30/1998
* USD 312,990.00

Project Summary This proposal seeks funding for a new multi-processor computer
for structural biology and computational physics at The Rockefeller University.
Structural biologists need increasingly faster computers with memories in excess
of 256 MB because of the size of data sets from X-ray crystallography, NMR
spectroscopy, and electron microscopy that need to be processed and fitted by
complicated models. In addition, the main subject of structural biology,
understanding the relation between structure and function, frequently depends on
large computer simulations such as molecular dynamics and protein structure
modeling. Likewise, calculations in computational physics also require more
computing power and memory than currently available in order to simulate larger
systems at a higher resolution and to explore larger parameter spaces. The
projects of the eleven major users include crystallographic, NMR, and electron
microscopic studies of structure, dynamics and function of proteins. The
proteins under study are involved in signal transduction, chromosomal
replication, transcription, and transmission of signals through cell membranes.
Additional work by the major users includes protein identification and
characterization by mass spectrometry, simulations of protein, DNA, and
chromosome structure and dynamics, and modeling of neuron function and visual
cortex architecture. The researchers are currently using small workstations,
many of which are more than three years old and have less than 64 MB of memory.
Speed and memory of these workstations are increasingly restricting. The
existing and future needs can best be met by a multi-processor computer such as
the Silicon Graphics Power Challenge XL system with 12 R10000 processors and
1.25 GB of memory, which we propose to purchase. A multi-processor server
computer is more suited to a small group of independent users than is a large
number of client workstations because of the speed deriving from fast processors
and from coarse parallelization, because memory and software can be shared among
several users, and because of the savings in support and maintenance. The use of
a locally located multi-processor computer is also more appropriate than the use
of national supercomputer centers because of the faster and more flexible
networking, sustained availability of computing power, smaller backlog, and
access to local consultants. In comparison with the current equipment, the new
server computer will provide a four-fold increase in computation speed and will
allow effective execution of very large jobs by avoiding swapping to hard disks.
The server will also free the current client workstations for their original
purpose, interactive and graphical work, both of which are impeded by the large
jobs running in the background. As a result, current computational bottlenecks
in protein structure refinement in X-ray crystallography, NMR, and electron
microscopy will be eliminated. The server will thus help in the structural
determination of eukaryotic RNA polymerase system, DNA polymerase complex, E.
coli RNA polymerase, voltage-gated potassium channel, Abl SH(32) dual domain,
and other proteins. It will also allow many simulations that are currently out
of reach. For example, NMR and molecular dynamics will be used to determine
motions of protein domains and large loops on the nanosecond time scale;
proteins will be identified rapidly by mass spectrometry and database searching,
which will be useful in the analysis of sequences produced by genome projects;
protein structure modeling by homology will be improved by simultaneous
optimization of the model and the alignment, which will allow a significantly
larger fraction of protein sequences determined by genome projects to be modeled
with useful accuracy; molecular orbital calculations of retinal and its
environment will elucidate the opsin shift mechanism in visual pigments; the
experimental data on the condensation of eukaryotic chromosomes will be analyzed
statistically and the models for this process will be proposed; and more
realistic neural models will be solved that may result in better understanding
of the function of the brain. The server will also provide graduate students and
postdoctoral fellows at The Rockefeller University an opportunity to program and
exploit the latest generation of mini supercomputers. This will be made easier
and encouraged by the existing infrastructure for networking and consulting. In
addition, the new server computer will facilitate planned recruitment of new
faculty in the areas of structural biology and computational physics. And
finally, it will stimulate the growing interest of all the faculty in structure-
function studies in biology and in the problems at the interface between physics
and biology.