* 9604653
* Regulation of Middle Transcription of Phage Mu
* BIO,MCB
* 03/15/1997,07/31/2001
* Martha Howe, The University of Tennessee, Memphis - The Health Science Center
* Continuing Grant
* Susan Porter Ridley
* 07/31/2001
* USD 315,000.00

9604653 Howe This investigator's model system for regulation of gene expression
is the middle promoter Pm of the Mu virus which infects Escherichia coil K12.
The middle transcript is made by the host RNA polymerase, but only in the
presence of the Mu activator protein Mor and DNA replication. Studies during the
previous grant period revealed that Mor activates Pm by binding as a dimer
upstream of and overlapping the -35 region, interacting with both the alpha and
sigma subunits of RNA polymerase, and thereby generating a DNA distortion which
facilities open complex formation and transcription initiation. In the absence
of DNA replication, Mor does not bind and open complexes do not form. The
current project includes experiments (i) to define at a molecular level the
specific roles of the sigma and alpha subunits of RNA polymerase, by using
footprinting and in vitro transcription assays with wild-type and mutant forms
of Pm, alpha, and sigma; (ii) to identify the amino acids in Mor which are
required for Pm binding and interaction with alpha and sigma, by assaying the
effect of targeted alanine substitutions in Mor on these processes; (iii) to
determine the origin and role of the DNA distortion by mutational analysis of
nearby sequences and identification and characterization of host proteins which
may participate in distortion formation; and (iv) to determine whether there is
a host "repressor" protein which binds to non-replicating Pm DNA, thereby
preventing Mor binding, by identification of Pm-binding proteins and
characterization of Pm and host mutations conferring replication independence.
This research will define the molecular mechanism by which the Mu middle genes
are regulated---that is, how they are kept silent prior to the time in
development when they are needed and how they are turned on at the appropriate
time. Not only will these results provide an understanding of these novel
mechanisms for turning genes on and off, but those mechanisms can serve as
models for studying and understanding the regulatory mechanisms occurring at
other important viral, bacterial, animal cell and human genes. Since much of any
organism's ability to maintain an appropriate internal milieu, respond to
environment stimuli, and carry out the programmed changes we call development
depend on its ability to regulate expression of specific genes, this research is
of great importance. In addition the experiments will serve as an excellent,
state-of-the-art, training experience in genetics, biochemistry, and molecular
biology for the graduate students who participate in them.