* 9630127
* Biosynthesis of Iron Sulfur Clusters
* BIO,MCB
* 09/01/1996,08/31/2000
* Limin Zheng, Virginia Polytechnic Institute and State University
* Continuing Grant
* Thomas E. Smith
* 08/31/2000
* USD 330,000.00

9630127 Dean Iron sulfur (Fe-S) clusters are found in numerous proteins that
have important redox, catalytic or regulatory properties. Such clusters are
known to act as electron carriers and environmental sensors, or to be involved
in substrate binding and catalysis, yet very little is known about how they are
formed biologically. Because free intracellular Fe and S are extremely toxic and
are unlikely to exist in the cell in appreciable quantities to permit
spontaneous Fe-S cluster formation, it is likely that biological Fe-S cluster
formation occurs through a controlled process catalyzed by a specific set of
enzymes. Three enzymes associated with nitrogen fixation (NifS, NifU and ORF6)
are likely to have functions related to the mobilization and activation of Fe
and S for Fe-S cluster formation. Recent sequence comparisons have shown that
these enzymes are among the most conserved throughout nature. This project will
analyze the function of these enzymes by determining how they work together to
form biologically active Fe-S clusters. This goal will be accomplished by
determining the interaction among these proteins and by optimizing conditions
which leads to their ability to catalyze Fe-S clusters in vitro. %%% Iron(Fe)
and sulfur(S) are essential elements for all free-living organisms. In some
cases, Fe and S are brought together to form structures called Iron-Sulfur
clusters. These clusters are important biological entities because they are
involved, for example, as electron carriers in important life sustaining
processes such as photosynthesis, nitrogen fixation, and respiration. Because
iron and sulfur are extremely toxic, the cell must be able to form Fe-S clusters
through a controlled enzymatic process. Three enzymes have been identified that
are likely involved in the biological formation of Iron-Sulfur clusters. In this
project, purified samples of these enzymes will be used to determine how they
work together to form such clusters. *** Project Summary Iron-Sulf ur clusters
(Fe-S clusters) are found in numerous proteins that have important redox,
catalytic or regulatory properties. Fe-S clusters are known to act as electron
carriers, environmental sensors, or to be involved in substrate binding and
catalysis. However, very little is known about how they are formed biologically.
Because free Fe and S are toxic and are unlikely to exist in the cell in
appreciable quantities, it is likely that biological Fe-S cluster formation
occurs through a controlled process catalyzed by a specific set of enzymes.
Previous work using the nitrogenase system has indicated that three nif-specific
gene products from the nitrogen-fixing organism Azotobacter vinelandii are good
candidates to participate in the formation of the Fe-S cores required for
nitrogenase activation. These genes are nif~, nifU, and a gene designated orf~.
NifS has been shown to be a cysteine desulfurase which is able to catalyze Fe-S
cluster formation in vitro, whereas the specific functions of NifU and Orf6 have
not yet been established. However, the phenotype of nifU mutants and the
organization of conserved cysteine residues contained within NifU and Orf6
indicate they could have roles in the activation of Fe for Fe-S cluster
assembly, act as intermediate Fe/S carriers, or provide a scaffolding site for
Fe-S cluster formation. The goals of the present project are to determine how
gi~o~ge~ F~S clusters are formed biologically and to find out whether or not
the'~'mo'lecular mechanism for nit'rogenase~Fe-S cluster fo~madon represents a
global mechanism for formation of Fe-S clusters found in other proteins. The
reason for suspecting that NifS, NifU and Orf6 activities might represent
general mec~anisms for Fe-S cluster formation is that homologs to all three
proteins have been identified in other prokaryotic and eucaryotic organisms that
do not fix pitrogen. The potential Ioles of NW, NifS and Orf6 in nitrogenase
specific Fe-S cluster formation will be explored by the following experiments.
First, co nditions for p~rotein-protein interactions between and among NifU,
NifS, Orf6, and an apo-form of the mtrogenase Fe protein will be examined. This
will be accomplished primarily by chemical~DI~ eLues and affinity chromatography
using the purified proteins. Second, the potential interaction among NifU, NifS,
Orf6, and the apo-Fe protein with other proteins whose identity is not yet known
will be determined. This will be done using affinity chromatography and
nitrogenase-derepressed crude extracts. Third, conditions for the optimum in
vitro catalytic formation of Fe-S clusters will be determined using various
combinations of purified NifU, NifS, orf6 and an apo-form of the nitrogenase Fe
protein. Fe-S cluster assembly will be monitored by activation of nitrogenase Fe
protein activity and by various spectroscopic techniques including Electron
Paramagnetic Resonance, Resonance Raman, Magnetic Circular Dichroism, and
~s'sb'auer spectroscopies. Fourth, crude extracts will be used to determine if
proteins other than NifU, NifS, and Orf6 might participate in nitrogenase Fe-S
cluster formation. Fifth, the possible formation of Fe-S cluster intermediates,
both in the presence and absence of the apo-form of the nitrogenase Fe protein,
will be monitored by a series of time-course freeze-quench experiments where
assembly is monitored by various spectroscopic techniques. The second major goal
of the project is to determine if homologs to NifU, NifS, and Orf6 are contained
within the A. vinelandii genome and, if so, do they encode housekeeping
functions for general Fe-S cluster assembly. A nifS homolog has already been
identified. In the present project the chromosomal region encoding the NifS-like
protein, which is suspected to also include nipJ-like and orl~like genes, will
be cloned and characterized. The functions of these genes will be tested using
gene directed mutagenesis techniques. If, as is expected, one or more of these
genes is essential for viability, genetic constructions will be perfo~med to
permit their conditional expression. 3