* 9504393
* Statistical and Computational Problems in Genetics and      Molecular Biology
* BIO,DBI
* 09/15/1995,07/31/2002
* Michael Waterman, University of Southern California
* Continuing Grant
* Sylvia Spengler
* 07/31/2002
* USD 1,010,540.00

This proposal addresses a variety of statistical and computational problems
that arise in genetics and molecular biology. An important aim of the research
project is the training and development of post-doctoral research fellows in
this interdisciplinary area. The problems described here are part of our ongoing
collaborations with geneticists and molecular biologists from USC and elsewhere.
All of them have significant computational and statistical components. We are
studying statistical and algorithmic aspects of physical mapping. Among these
are optical mapping, by which ordered restriction maps are produced using
fluorescence microscopy. The long term aim is the development of an online
program that produces such an ordered map. We are analyzing the efficiency of
various physical mapping strategies based on variants of sequence-tagged-sites
schemes. In particular, we are studying inhomogeneous clone and anchor
distributions, the effects of false, ambiguous or missing anchors, and
strategies that allow for mixtures of clone walking and random steps. We are
developing statistical models for the evolution of genome rearrangements and
methods for reconstructing phylogenetic trees from such data. This is motivated
in part by data on rearrangements occurring in the promoter of GFAP in the
brain. A related project addresses other aspects of phylogenetic tree
reconstruction, in particular the effects of dependence among sites when DNA
sequence data is used to build the tree. Motivated by data on rRNA promoters, we
are also developing algorithms for fitting more detailed evolutionary models
when the underlying tree is known. We are investigating the statistics of
pattern counts in molecular sequences. These a re frequently used to
characterize biological features such as coding regions, regulatory sequences,
splice sites and so on. Our focus is on the development of useful computational
algorithms for pattern counts when the underlying sequence structure must be
estimated from the data. A further project addresses the estimation of deletion
rates in mitochondrial genomes in humans. A PCR assay is used to detect whether
or not a cell line contains any mitochondria with the deletion. The data
comprise the results of the assay on several cell lines. We are developing
analogs of the Luria-Delbruck method of fluctuation analysis for estimating the
rate of deletions. The results are used to study the role of the deletion in
Kearns-Sayre Syndrome patients. We are developing Monte Carlo likelihood methods
for inference in genetics, aimed primarily at the study of mitochondrial and
nuclear gene variation in humans, and its use in the study of human evolution.
One focus is on the estimation of recombination rates, in particular between
human disease loci and a variety of marker loci.