* 9505829
* RNA-Protein Interactions in a Double-Stranded RNA Virus of  Saccharomyces Cerevisiae
* BIO,MCB
* 02/01/1996,01/31/1999
* Jeremy Bruenn, SUNY at Buffalo
* Standard Grant
* Susan Porter Ridley
* 01/31/1999
* USD 161,000.00

9505829 Bruenn Many of the double-stranded RNA viruses of lower eucaryotes are
non-infectious viruses that persist indefinitely by replicating in tandem with
their host cells, their spread mediated entirely by sexual and asexual
reproduction. They have adapted to this lifestyle in some cases by packaging
extra RNA not essential to the virus but encoding products of selective value to
their host cells. These products are cellular toxins that kill cells not
harboring the virus (the killer phenotype). Several such systems are known among
the fungal dsRNA viruses, the best studied of which is the Saccharomyces
cerevisiae virus (ScV). A number of characteristics make this system very
attractive for studying nucleic acid-protein interactions. First, the host cells
are readily amenable to molecular and classical genetics. This makes it possible
to use the killer phenotype for screening and selection of viral mutants.
Second, the virus separately encapsidates its RNAs, making many experiments
simpler than those in the dsRNA viruses of higher eucaryotes, many of which have
as many as 12 unique dsRNAs encapsidated in each viral particle. Third,
extensive analysis of the single essential large viral dsRNA (4580 bp) has
mapped many viral functions. There are only two viral proteins, one the major
capsid polypeptide and product of the cap gene, and the second a multifunctional
protein the result of frameshifting between the two open reading frames (cap and
pol) to produce a cap-pol fusion protein. The cap-pol fusion protein has the
enzymatic and packaging activities of the virion. Among the cis-acting sites is
the viral binding site (packaging site) which has (for RNA) a very high affinity
for its recognition protein (the cap-pol fusion protein). This research will
define the interaction between cap-pol and this binding site by mapping all the
elements of specificity in both the RNA and the protein. Analysis will include
in vitro mutagenesis of both cap-pol and the binding site, domain swapping
experiment s between related RNA binding sites, domain swapping experiments
between related cap-pol proteins, optimization of the RNA binding site, and
optimization of the cap-pol binding domain. The minimal size of the optimal RNA
biding site is known (19 nucleotides) as is the region of the viral protein
necessary for binding (73-200 amino acids). This is one of the few systems in
which both in vivo and in vitro systems for selection and analysis of altered
binding sites and binding protein exist. It promises to contribute significantly
to understanding RNA-protein interactions. %%% Many of the double-stranded RNA
viruses of lower eucaryotes are non-infectious viruses that persist indefinitely
by replicating in tandem with their host cells, their spread mediated entirely
by sexual and asexual reproduction. They have adapted to this lifestyle in some
cases by packaging extra RNAs not essential to the virus but encoding products
of selective value to their host cells. These products are cellular toxins that
kill cells not harboring the virus (the killer phenotype). Several such systems
are known among the fungal dsRNA viruses, the best studied of which is the
Saccharomyces cerevisiae virus (ScV). A number of characteristics make this
system very attractive for studying nucleic acid-protein interactions. First,
the host cells are readily amenable to molecular and classical genetics. This
makes it possible to use the killer phenotype for screening and selection of
viral mutants. Second, the virus separately encapsidates its RNAs, making many
experiments simpler than those in the dsRNA viruses of higher eucaryotes. Third,
extensive analysis of the single essential large viral dsRNA (4580 bp) has
mapped many viral functions. There are only two viral proteins, the major capsid
polypeptide and the cap-pol fusion protein, which has the enzymatic and
packaging activities of the virion. This research seeks to understand the nature
of the interaction between cappol and its viral RNA binding site by mapping all
th e elements of specificity in both the RNA and the protein. This is one of the
few systems in which both in vivo and in vitro systems for selection and
analysis of altered binding sites and binding protein exist. It promises to
contribute significantly to understanding RNA-protein interactions. ***