* 9506913
* NMR Studies of RNA Aptamers for Biological CoFactors
* BIO,MCB
* 08/01/1995,07/31/1999
* Juli Feigon, University of California-Los Angeles
* Continuing grant
* Kamal Shukla
* 07/31/1999
* USD 270,000.00

9506913 Feigon Aptamer is a name coined for DNA or RNA oligonucleotides which
have been selected from a large pool of oligonucleotides containing a region of
random nucleotide sequence for binding to a specific target molecule. The
isolation process involves repeated cycles of selection for, and enrichment of,
oligonucleotides with an affinity to a specific target, followed by
amplification of these sequences using the polymerase chain reaction (PCR).
Olionucleotides with the selected characteristics, i.e. binding to a specific
molecule, are finally identified through cloning and sequencing. The cloned
sequences are usually screened for a common consensus and possible secondary
structural motif. This methodology has been used to identify a number of
different classes of RNA aptamers that bind specifically to common metabolic
substrates and cofactors. These include ATP, GTP, riboflavin, FAD, FMN,
cyanocobalamin and the amino acids arginine and citrulline. Based on the
consensus sequences and further experiments such as deletion analysis, minimal
binding sequences have in most cases been identified and a secondary structure
proposed. However, to date, no three dimensional structure for RNA aptamers have
been determined. Multidimensional, multinuclear NMR spectroscopy will be used to
determine the three-dimensional structures of aptamers that bind biological
cofactors, in the absence and presence of bound cofactor. RNA samples for NMR
studies will be synthesized by in vitro transcription using T7 RNA polymerase.
These will initially be investigated by 1H NMR spectroscopy to screen for
sequences which give good NMR spectra. Once well-behaved samples are obtained,
partially or uniformly 15N and/or 13C-labeled samples will be synthesized, to
facilitate complete resonances assignment. In favorable cases, complete three-
dimensional structures will be determined using metric matrix distance geometry
and refinement by molecular dynamics. %%% NMR spectroscopy will be used to study
the structures of RNA oligonucleotides that bind biological cofactors. These
RNAs contain a wide variety of proposed secondary structural motifs, including
hairpin loops, internal loops, pseudoknots, and intramolecular G-quartets. Thus,
determination of their structures will lead to a greater understanding of the
principles of nucleic acid folding. Very few RNA structures of any kind have
been solved so far, so any information on tertiary structures of RNA should
greatly add to the knowledge base of these secondary structure motifs.
Determination of the specific cofactor-RNA interactions will also help elucidate
some of the factors governing sequence specific recognition of RNA by proteins
and other ligands. The structures of the specific ligand binding sites are also
relevant to "RNA world" models of the origin of life, in which the evolution of
intermediary metabolism occured at the time of the RNA world. Finally,
determination of these structures should be useful in the design of new
catalysts which need these co-factors for their function. ***