* 9506301
* Control of RNA Processing in Drosophila
* BIO,MCB
* 06/15/1995,05/31/1999
* Lillie Searles, University of North Carolina at Chapel Hill
* Continuing Grant
* DeLill Nasser
* 05/31/1999
* USD 440,000.00

9506301 Searles The goal of this research is to determine the role of the
nuclear protein encoded by the Drosophila melanogaster suppressor of sable su(s)
gene in RNA processing control. The proposed experiments will test the
hypothesis, based on previous work, that su(s) functions to aid the splicing
machinery in discriminating between authentic and cryptic 5' splice donor sites.
The following are specific objectives of this proposal. (1) Since purified,
recombinant su(s) protein,expressed in a baculovirus system, binds to RNA,
several biochemical techniques will be employed to determine whether su(s)
binding affects the ability of a given pre-mRNA to be spliced. The sites of
su(s) protein binding on RNAs will be mapped. If su(s) binds specifically to
donor site regions, a splicing complex assembly assay will be used to determine
if su(s) binding to these sites interferes with Ul snRNP binding. Other
experiments will examine the RNA binding specificity of endogenous su(s) protein
and define the basis for poor splicing activity of su(s) mutant nuclear
extracts. (2) To determine if the region of su(s) required for in vitro RNA
binding activity is required for su(s) function in vivo, deletions will be
introduced into RNA binding domain, and transformants carrying these deletion
derivatives will be analyzed. If the deletions affect su(s) function in vivo, an
effort will be made to identify specific amino acid residues required for the
RNA binding activity. As a first step toward the identification of other
functional dmains, the Musca domeslica su(s) homolog will be cloned and its
sequence compared to the available D. melanogaster and D. virilis sequences. (3)
To determine whether the intracellular distributions of splicing components and
inefficiently spliced pre-mRNAs are altered in su(sJ mutants, larval salivary
gland cells will be immunostained or in situ hybridized with appropriate probes
and examined by con focal microscopy. (4) Natural RNA targets of su(s)
regulation will be identified. Antibodies that recognize su(s) strongly stain a
relatively small number of polytene chromosomes bands in salivary gland nuclei.
The strong sites of su(s) staining, which are likely to contain genes regulated
by su(s), will be mapped on polytene chromosomes. Subsequently, an effort will
be made to identify and analyze specific transcripts that are subject to
regulation by su(s). This analysis of su(s) function has the potential to
advance the understanding of mechanisms that regulate splice site selection in
higher eukaryotes. %%% Eukaryotic genes contain varying numbers of regions of
sequence which are named "introns". They are spliced out of the mRNA transcript
and, in some cases, have alternative end points and so can result in a gene at
the DNA level encoding more than one protein as a final product. Thus it becomes
important to understand how the splice sites in a transcript are identified and
utilized. During the tenure of the previous award, experimental evidence
suggested that the nuclear protein encoded by the gene from Drosophila,
suppressor of sable su(s) functions to aid the splicing machinery to
discriminate between authentic and cryptic 5' splice donor sites. Further work
will be undertaken to provide data to support this hypothesis. This analysis of
su(s) function has the potential to advance the understanding of mechanisms that
regulate splice site selection in higher eukaryotes. ***