* 9507352
* DNA Helicases in Yeast DNA Replication and Repair
* BIO,MCB
* 08/01/1995,07/31/1998
* Judith Campbell, California Institute of Technology
* Continuing Grant
* Marcia Steinberg
* 07/31/1998
* USD 240,000.00

; R o o t E n t r y F qD @ C o m p O b j b W o r d D o c u m e n t O b j e c t
P o o l `S qD `S qD C D E F G H I J K L F Microsoft Word 6.0 Document MSWordDoc
Word.Document.6 ; h.&lt;hB&gt; f s f M EU V v Ph Vh " I F Pj j` t v v Ph
h.&lt;hB&gt;P F V V Ph " F F | t5 ~ u ~ t) D @ P B 7 F P D @ P " n F V ^ f M EU
H&gt; &lt;&lt; &gt;&lt; f M 9507352 Campbell A working model for these studies
is that assembly of a replication fork at a yeast chromosomal origin of
replication occurs in multiple stages: recognition, unwinding, priming of DNA
synthesis by DNA polymerase alpha primase, and recruitment of pol delta and/or
pol epsilon. In a complicated, multicomponent process such as DNA replication,
even the most exhaustive characterization of the individual proteins cannot
describe the dynamics of the actual physiological events. In order to do that,
interaction between the components, no matter how many there are, must be
studied. The experimental theme of this project is therefore to design both
biochemical and genetic experiments to characterize the dynamic interactions
between the replication proteins, especially the helicases that unwind and the
polymerases that form the primosome and elongation apparatus. Ultimately we
would like to reconstitute an unwinding and initiation event at an origin with
purified proteins. The experiments described center on DNA helicases. In
particular, the P.I. has identified a new yeast DNA replication m utant, dna2,
using a screen for mutants defective in DNA replication in permeabilized cells.
The genetic and biochemical experiments proposed will test the mode of
involvement of the Dna 2 helicase in yeast DNA replication. In particular, the
interaction of the helicase with the three essential DNA polymerases and the
essential origin recognition protein, ORC, will be studied. Interestingly, the
Dna2 helicase copurifies with a nuclease whose specificity is similar to the
HeLa MFl nuclease, that is involved in maturation of Okazaki pieces during
lagging strand DNA synthesis in the SV40 replication system. A gene encoding a
nuclease with similar specificity in yeast, called YKL510, suppresses the dna2
mutation when present on a high copy number plasmid. This suggests that YKL510
encodes the nuclease that is associated with Dna2p il vitro and that YKLS 10 can
interact with Dna2p in vivo. If the interpretation of these preliminary
experiments is correct, then they also point to a role for Dna2 helicase in
replication. A second portion of the proposal will test the hypothesis that the
nuclease associated with Dna2 is YKLS 10, that they do associate in vivo, and
that YKLS 10 is involved in processing Okazaki pieces or some other aspect of
replication in yeast. %% In the past five years there has been a revolution in
the study of cell biology. Molecular approaches have been possible in
elucidating the cell cycle, the process by which a new cell arises from a parent
cell. Fundamental to the cell cycle are two processes; duplication of the genome
(DNA replication) and segregation of the genome to daughter cells. This proposal
focuses on DNA replication. The P.I. has discovered a new, essential enzyme for
this process. Studies of the replication enzyme (the DNA helicase) described in
this proposal will enhance our understanding of how normal cells transmit the
genetic information contained in their DNA from one generation to the next. ***
@ ....()()))()() Oh +' 0 $ H l D h R:\WWUSER\TEMPLATE\NORMAL.DOT marcia
steinberg jack cohen @ S 5 @ M S u m m a r y I n f o r m a t i o n ( B 5 @ # qD
@ 2~ Microsoft Word 6.0 4 e = e " " " " " " " L L L L L d n L @ C x | | | | | |
|