* 9513538
* Isotope Enriched Active Site Structure of EF-Tu by Endor
* BIO,MCB
* 03/01/1996,02/29/2000
* Shan Chen, University of Chicago
* Continuing Grant
* Kamal Shukla
* 02/29/2000
* USD 304,000.00

9513538 Makinen A new approach combining angle selected electron nuclear double
resonance (ENDOR) spectroscopy with molecular biological techniques to
selectively enrich proteins biosynthetically with isotopes will be applied to
determine the catalytically competent active site structure of the bacterial
elongation factor Tu (EF-Tu) in GTP hydrolysis. EF-Tu will be isolated from E.
coli JM109 into which the tufB gene of Thermus thermophilus has been cloned to
overproduce the thermostable protein. The perdeuterated protein will be isolated
from bacteria grown on deuterated algal hydrolyzate as the culture medium.
Auxotrophic strains also suitably engineered to overproduce thermostable EF-Tu
will be used for growth on deuterated medium to incorporate site specifically
isotopically enriched amino acids. Since there is only one tyrosine, one lysine,
one cysteine, one valine, and one proline within an 8 angstrom radius of the
metal ion site in the protein, auxotropic strains specific for these amino acids
will be chosen. Although other sites in the protein will be also isotopically
enriched for each amino acid according to its primary sequence, they are too
distant from the metal ion to have prominent spectral contributions. In
preliminary studies we have already determined that the vanadyl (VO2+) ion
specifically substitutes for Mg2+ and supports hydrolysis of GTP catalyzed by
EF-Tu. This cation will be employed as the paramagnetic probe for angle selected
ENDOR to determine metal-to-nucleus distances by ENDOR and to assign positions
of ENDOR active nuclei with respect to magnetic axes defined by the g tensor of
VO2+). The catalytically competent structure of the EF-Tu:VO2+:GTP complex in
solution will be generated by rapid freeze-quenching of the solution for
structural characteriztion by ENDOR spectroscopy. The active site structure of
other ternary complexes, namely the EF-Tu:VO2+:GDP, EF-Tu:VO2+:GDPCP and the EF-
Tu:VO2+GDPCP complexes, where the latter two nucleotides represent the
nonhydrolyzable (,( -imido and (,(-methylene analogs of GTP, respectively, will
be also determined. The methods will be also extended to determine active site
structure of EF-Tu:VO2+:GTP complexed to amino acyl-tRNA. %%% The bacterial
elongation factor Tu(EF-Tu) is an essential component in the biosynthesis of
proteins and shows amino acid sequence homology with a variety of G-proteins in
cells of higher organisms. In order to carry out its function, EF-Tu binds and
hydrolyzes GTP. The structure of the catalytically active form of EF-Tu with GTP
in the active site is not known. Determination of the structure of the protein-
GTP complex will be accomplished by a series of electron nuclear double
resonance (ENDOR) studies designed to measure interatomic distances between a
catalytically active metal ion and magnetic nuclei on side-chains of nearby
amino acid residues. The structural information will be applicable to
understanding structure-function relationships of a wide variety of other
homologous G-proteins. ***