* 9513602
* Flow Cytometry:  Digital Processing of Molecular Information
* BIO,DBI
* 03/01/1996,02/28/1997
* David Galbraith, University of Arizona
* Standard Grant
* Lee C. Makowski
* 02/28/1997
* USD 50,000.00

This project involves the design, implementation, and evaluation of flow
cytometric instrumentation and operating procedures for use in the
cytoenzymological and histochemical analysis of eukaryotic cells. It is divided
into two subprojects. The first involves the development of a second-generation
digital data acquisition and pulse processing system for the real-time analysis
and sorting of cells based on pulse shape features. The second subproject
involves the development of novel gene expression systems for use in the
molecular and cellular analysis in flow of transgenic eukaryotic cells. We then
intend to employ these techniques for the selection through flow sorting of
transformed plant nuclei according to the level of transgene expression,
ultimately directed at the construction of cDNA libraries representing the
developmentally-regulated genes of tissues that are microscopically-small. The
significance of this work is as follows. In the first case, our previous work
has shown that important information about eukaryotic cells is contained within
the waveform shapes produced by these cells produced when they are subjected to
analysis using flow cytometry. Present flow cytometric instrumentation cannot
extract this shape-based information. The capability of the proposed second
generation machine to analyze and sort cells based on pulse shape information
will provide important knowledge about cellular substructure. This will lead to
improved understanding of the basic cellular biology of living cells. In the
second case, the ability of scientists to analyze the expression of genes within
different cell types is hampered when these cells are interspersed within other
cell types. The molecular methods proposed in this project should provide
general means for isolation of transcripts from any desired cell type. This
should greatly facilitate advances in the study of developmental- and cell type-
specific gene expression.