* 9513248
* Subunit Arrangement within a Replicative Complex
* BIO,MCB
* 06/15/1996,05/31/1999
* Charles McHenry, University of Colorado at Denver
* Continuing Grant
* David Rockcliffe
* 05/31/1999
* USD 300,000.00

9513248 McHenry The DNA polymerase III holoenzyme is the replicative complex of
E. coli, responsible for the synthesis of the majority of the chromosome. The
replicative complexes of all cellular systems are closely related. They all
consist of a special replicative polymerase, structurally identical sliding
clamps with which the polymerase associates and a clamp setting apparatus. In E.
coli, these components correspond to the DNA polymerase III ((((), the (
"sliding clamp", and the 5-protein DnaX clamp setting apparatus. The holoenzyme
contains 10 different subunits, all available in 100 mg quantities from
overproducing strains. This replicative complex exhibits many properties that
distinguish it from simpler polymerases. These properties include a high rate of
elongation, the ability to form an ATP-dependent highly processive clamp on the
DNA template, and the ability to function as an asymmetric dimer with
distinguishable leading and lagging strand polymerases. Using photoreactive
nucleotides placed at unique positions within long primers, my laboratory has
located the ( catalytic subunit (3'-nucleotides 1-13), the ( (DnaX)-complex
(contact at nucleotide 18) and the processivity factor (nucleotide 22) within
the initiation complex. In this proposal, experiments are outlined to extend
this analysis in order to identify subunit contacts with the template,
particularly in front of the primer and regions within the template with short
primers of ca. 10 nucleotides such as those used to prime DNA replication.
Placement of photoreactive nucleotides at specific locations in templates of
designed sequence will permit the replicative complex to be "walked" into
specific positions by the addition of a limited number of dNTPs. This will
permit changes in the composition and the contacts within the replicative
complex to be determined as it progresses from initiation to elongation. The
positioning of subunits relative to others within the replicative complex will
be determined by chem ical protein-protein cross-linking. Initial focus will be
on the "zero-length" cross-linker EDC. These carbodiimide-induced intersubunit
cross-links will be analyzed at the sequence level to permit regions of subunit-
subunit contact to be localized. Fluorescence energy transfer will be used to
determine the distance between subunits. Primary focus will be on the distances
between subunits that do not cross-link to one another. Concurrent determination
of the distances of these subunits to the primer terminus and the ( subunit will
permit triangulation and placement of the studied components within the
replicative complex. These experiments are supported by our demonstrated ability
to locate the ( subunit 65 A from the 3'-antepenultimate nucleotide in primers,
a position verified by photocross-linking. Additionally, pilot experiments will
be conducted collaboratively using electron microscopy to examine the structure
of the replicative complex. Initial focus will be upon a direct visual test of
the dimeric polymerase hypothesis. This proposed work will yield the first
detailed structure of a prototypical replicative complex at the level of subunit
organization. %%% The 10-protein apparatus that duplicates chromosomes prior to
cell division is closely conserved between bacterial and higher cells. The
purpose of this proposed study is to determine the physical arrangement and the
contacts of the components of a prototypical DNA replicative machine. This will
be accomplished by (i) placing analogs of DNA bases at specific points in a
model chromosome that chemically attach to proteins when flashed with light to
determine the linear arrangement of the replication proteins relative to DNA
both in a static and an active elongation mode; (ii) direct chemical protein-
protein cross-linking to determine which proteins touch one another, (iii)
fluorescence energy transfer, a technique that uses a spectroscopic (light)
ruler to determine the distances between the components that do not contact one
another and (iv) electron microscopy to directly visualize relative positions of
proteins and DNA. These complementary methods will permit a description of the
geometry of a prototypical replicative complex and permit the design of specific
experiments to test the function of protein subunits implied by their
position(s) within the complex. ***