* 9513222
* Mechanisms of Chromosome Partitioning in Caulobacter
* NONE,NONE
* 05/01/1996,04/30/1999
* James Gober, University of California-Los Angeles
* Continuing grant
* Philip Harriman
* 04/30/1999
* USD 285,000.00

Gober 95-13222 The bacterium Caulobacter crescentus expresses polarity during
its cell cycle that is generated by a simple developmental program. Cell
division yields two dissimilar daughter cells: a motile swarmer cell and a
sessile stalked cell. These two cell types differ in their relative programs of
gene expression and DNA replication. The progeny stalked cell reinitiates DNA
replication immediately after cell division, whereas replication is silenced for
a defined period of time in the swarmer cell. The Caulobacter developmental
cycle allows the experimental analysis of the basic mechanisms underlying cell
cycle controlled gene expression, the regulation of DNA replication and the
molecular basis of positional information. The objectives of this research are
to genetically and biochemically determine the molecular basis of chromosomal
partitioning and the coupling of this event to cell division in Caulobacter. The
ease at which synchronized populations can be obtained plus, the intrinsic
asymmetry of the Caulobacter predivisional cell make it an ideal system for
tracking the subcellular location of newly replicated chromosomes and the
proteins that direct them during the partitioning process. Specifically, the
role of the cellular homologues of plasmid partitioning gene products (parA and
parB) in chromosomal partitioning will be determined. In bacterial cells the
newly replicated chromosomes efficiently partition to the poles of the
predivisonal cell, ensuring that each daughter cell inherits its own copy of the
chromosome Partitioning of unit- or low-copy number plasmids may provide an
experimental framework with which to dissect chromosome partitioning. Plasmid
partitiomng is accomplished in part, through the action of, ParA and ParB, two
plasmid-specific gene products. Both ParA and ParB are necessary for efficient
plasmid partitioning. ParB functions as a DNA binding protein and specifically
binds the plasmid centromere, which is often located immediately downstream of
the parB cod ing sequence. ParA is homologous to a large family of ATPases and
plays an undefined role in partitioning. In several species of bacteria, the
chromosomal homologues of parA and parB have been identified. The parA and parB
homologues of Caulobacter have been cloned and sequenced. These two genes are
expressed as an operon and immediately following the parB coding region lies a
DNA sequence containing a number of inverted and direct repeats that could serve
as the centromere during partitioning. Genetic experiments indicate that parA is
essential for cell viability and parB is required for efficient partitioning of
chromosomal DNA. The parAB operon is expressed under cell cycle control, with
peal~ expression occurring at the latter stages of DNA replication. Furthermore,
ParA is concentrated at the poles of the Caulobacter cell. ParA and ParB may
have several possible roles in chromosome partitioning. First, they may directly
participate in chromosome movement by functioning to tether or direct the
chromosomes to the locomotor apparatus. Alternatively, they may function in
regulating a cell cycle checkpoint that coordinates chromosome partitioning and
cell division. In order to dissect the roles of ParA and ParB in chromosome
partitioning in Caulobacter, the following experimental approaches are being
taken: (1) Genetic analysis of the Caulobacter parA and parB genes will be
performed to determine their role in chromosome partitioning and cell division.
In addition, the proteins that specifically interact with ParA and ParB during
cell division and chromosome partitioning will be identlfied using a genetic
suppressor analysis and a two-hybrid screen. Par protein interactions will also
be assayed utilizing in vitro techniques including co-immunoprecipitation (2)
The function of ParB DNA binding activity and its role in chromosome
partitioning will be determined. (3) The mechanisms responsible for the cell
cycle regulation of parA andparB expression and the subcellular location of ParA
and ParB du ring the cell division cycle will be defined. To determine whether
ParA or ParB function in a cell cycle checkpoint, the effect of conditional parA
and parB mutations on the coordination of cell division, the timing of
initiation of DNA replication and the expression of developmentally regulated
genes will be tested. This research will contribute to a basic understanding of
an importnt component of bacterial reproduction.