* 9513589
* U1trastructural Analysis of RNA Synthesis and Processing
* BIO,MCB
* 03/01/1996,02/28/2002
* Ann Beyer, University of Virginia Main Campus
* Continuing Grant
* Joanne Tornow
* 02/28/2002
* USD 444,940.00

9513589 Beyer This research involves the details of the basic cellular machinery
involved in eukaryotic gene expression, specifically at the level of pre-mRNA
splicing and 3' end formation. An unusual in vivo ultrastructural approach, the
"Miller chromatin spreading procedure", will be used to analyze co-
transcriptional events on specific genes after their microinjection into Xenopus
laevis oocyte nuclei. One can visualize multiple copies of specific genes on
extrachromosomal plasmid vectors and gain global information concerning
structural aspects of RNA transcription and processing at the resolution of the
individual gene and the individual transcript. Studies will concentrate on the
very early stages of splice site recognition and spliceosome assembly on
constitutively spliced introns. There is little understanding of how splice
sites are paired across typical long introns. The ultrastructural approach has
revealed that splice sites are recognized and paired within seconds of their
synthesis, even for long introns of 10-20 kilobases. The ultrastructure is
consistent with the possibility that, once synthesized, 5' splice sites are
somehow tethered near the site of RNA emergence from the polymerase in order to
be paired with the next recognized 3' splice site. Experiments are designed to
test the involvement of various cis and transacting factors in the early intron
recognition phenomenon regardless of whether or not tethering is involved, and
also to test the possibility of a tethering mechanism. The second specific
objective is to use this ultrastructural assay to determine the mechanism of
transcript release at the end of a gene, and to distinguish the relative
contributions of polyadenylation signals and termination/pause signals to this
process. Genes will be visualized in a transcriptionally active state for
determination of the gene length and density of the transcripts at the 3' end of
the gene, yielding direct visual information on the site(s) of transcript
release, the mode of transcript release (cleavage or termination), the
efficiency of transcript release, and the possible role of transcriptional
pausing in transcript release. Two alternative models for transcript release
have been proposed, both of which accommodate the known dependence of transcript
release on a functional poly(A) signal, (i.e., co-transcriptional RNA cleavage
at the poly(A) site vs alteration of the polymerase to a termination-competent
form). The results of this study will allow a clear distinction between these
models. %%% The information for the synthesis of a protein is encoded in blocks
of sequences of RNA interspersed within RNA sequences that have no clearly
defined function. A group of proteins are responsible for the excising the
blocks of amino acid specifying sequences from the rest of the RNA, and joining
them in a linear chain for delivery to the protein synthesizing complex. These
studies involve an ultrastructural approach to visualize the separate steps of
the process, and to identify intermediates as a means of confirming postulated
mechanisms of processing the RNA. A second objective is to use the same assay to
determine how the RNA transcript is released from the end of a DNA gene
sequence, and the role of different genetic signals on this process. Genes will
be ultrastructurally observed in an actively transcribed state to allow
mechanistic information to be obtained on transcription mechanisms. * * *