* 9520465
* A Molecular Biologist's Computational Workbench
* BIO,DBI
* 09/15/1995,08/31/1996
* Elizabeth Kutter, Evergreen State College
* Standard Grant
* James H. Beach
* 08/31/1996
* USD 53,000.00

This grant will provide support for the infrastructure of the molecular biology
database research program at the evergreen State College over the next year,
maintaining a full-time programmer for months and providing limited travel and
software upgrading. Activities which will be enhanced by this support include:
1. Completion of publication of the analysis of the bacteriophage T4 genome.
With its approximately 300 genes and long years of study, T4 is the most complex
virus for which the entire sequence is known. Many interesting phenomena have
already been observed, from both spliced- and folded-out introns to nested genes
and complex protein machines to a 26-amino acid dual-function protein gene
product to genes highly homologous with eukaryotic proteins. The computer
support under this grant will facilitate interaction among the members of the T4
community, and also work with various outside groups using the T4 data set as a
model system for genoming analysis. A major activity will be to explore
available tools for analyzing the still-uncharacterized half of the genes
involved in the transition from host to viral metabolism after infection by
large virulent phages. This work will also help to identify the most useful
candidate programs for a Molecular Biologist's Computational Workbench. 2.
Support of collaborative efforts for object-oriented molecular biology database
specification, research and prototype development. This collaboration involves
the Evergreen team (Drs. Cushing and Kutter, Tom Stidham and several students),
Tom Marr and colleagues at Cold Spring Harbor and Tim Hunkapillar and David Yee
at the University of Washington. Here, the major current work involves
development of a metadatabase for the storage of complex multiple database
sequence comparisons, particularly important in facilitating comparing search
results across a range of search programs to tease out weak but significant
relationships from very large amounts of data . The metadatabase will be built
upon the framework existing in the Genome Topographer, an integrated database
for genomic analysis and viewing being developed by Tom Marr; we have been
collaborating with them for the last year on its expansion to application to
prokaryotic systems. In a parallel effort Tim Hunkapillar and David Yee will b
implementing the sequence comparison metadatabase in Sybase so that these two
database approaches can be compared in functionality and usability; we are
working together closely on this project. 3. Provide support to students in the
"Student Originated Software" and "Molecule to Organism" programs who are
involved in the above projects. The infrastructure thus supported makes it
possible for molecular biology and computer scientist faculty and students to
work together, sharing problems and solutions in facilitating analysis of the
complex ways that genomic information is organized and expressed.