* 1803055
* Unraveling regulatory networks in biological nutrient removal (BNR) microbiomes
* ENG,CBET
* 09/01/2018,08/31/2023
* Timothy Donohue, University of Wisconsin-Madison
* Standard Grant
* Mamadou Diallo
* 08/31/2023
* USD 330,000.00

A large fraction of the energy cost in operating a wastewater treatment plant is
supplying oxygen to the microorganisms performing the treatment process. These
costs would decrease if treatment plants could be operated under low-oxygen
conditions. It remains unknown how microorganisms adapt to low-oxygen conditions
and whether treatment plants can be effectively operated with minimal oxygen
aeration. This project will reveal how key microorganisms in wastewater
treatment systems regulate their internal gene networks in response to plant
operations with minimal aeration. Other impacts of the project include
educational activities related to mentoring high school, undergraduate, and
graduate students, and advancing the understanding of gene regulation in natural
and engineered environments. If successful, this project will provide
transformative information that can help protect the Nation's water security
through advanced treatment processes to remove both reactive nitrogen and
phosphorus.&lt;br/&gt;&lt;br/&gt;This proposal seeks to develop and test a
framework to study transcriptional regulation in complex microbiomes.
Specifically, the framework will be used to investigate transcriptional
regulatory networks (TRNs) in an ammonia-oxidizing bacterium and a polyphosphate
accumulating organism identified as major contributors to a biological nutrient
removal (BNR) microbiome. This microbiome is maintained in a high-rate BNR
bioreactor operated using cycles of anaerobic and micro-aerobic conditions. The
proposed framework involves: (i) the use of metagenomic data to assemble genomes
of key organisms; time-series metatranscriptomic data to identify gene
expression patterns in the organisms of interest; (ii) chromatin
immunoprecipitation (ChIP) followed by genome-scale DNA sequencing (ChIP-seq) to
identify genes directly targeted by a specific transcription factor (TF); and
(iii) an in silico phylogenomic approach to predict sets of co-regulated genes
in assembled genomes. The project consists of four Tasks. Task 1 focuses on
improving genome assembly by using novel tools, obtaining long DNA sequence
reads with an emerging sequencing technology, and creating enrichments of the
organisms of interest. Task 2 relates to the use ChIP-seq to experimentally
identify genes in Nitrosomonas strain UW-LDO-1 targeted by ANR. Task 3 will use
ChIP-seq to target RegA, ANR, and DNR - three TFs hypothesized to control
aerobic and anoxic respiration in Accumulibacter strain UW-TNR-IC. Task 4
consists of an in silico reconstruction of large-scale TRNs of Nitrosomonas UW-
LDO-1 and Accumulibacter UW-TNR-IC. The effective use of this integrative
systems biology methodology is transformational because it is the first attempt
at assembling a TRN in uncultured microbes using a combination of experimental
and computational approaches. This project will provide the first analysis of
how respiration is regulated at the transcription level in two key
organisms.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.