* 1816027
* AF: Small: Algorithmic Techniques for High-throughput Analysis of Long Reads
* CSE,CCF
* 10/01/2018,09/30/2022
* Srinivas Aluru, Georgia Tech Research Corporation
* Standard Grant
* Mitra Basu
* 09/30/2022
* USD 424,992.00

The field of DNA sequencing has rapidly evolved from Sanger sequencing (700-1000
base pairs, or bp, in length) to massively parallel high-throughput sequencing
of short reads (100-200 bp) to the more recent advances in generating long (&gt;
5,000 bp) and ultra-long (&gt; 50,000 bp) reads. There is currently an urgent
need to develop efficient algorithms for analyzing long-read datasets in the
context of the myriad biological applications they enable. Long read
technologies sustain high error rates but with more attractive error
distribution characteristics that often permit probabilistic guarantees on the
quality of results. This project will result in fundamental research advances in
developing bioinformatics algorithms for long-read sequencing, along with
distributable open-source software products to facilitate their immediate
adoption by the life sciences community. The award will also support
interdisciplinary training, and undergraduate participation in
research.&lt;br/&gt;&lt;br/&gt;The project seeks to advance mapping, assembly,
and biological applications of long-read sequencing through the design of
provably efficient algorithms, formal characterization of the quality of
results, development of methods that scale to larger datasets, and methods that
are robust to changes brought about by continued developments in long read
sequencing. Problems addressed include (i) split-read mapping of ultra-long
reads to a reference genome, (ii) development of data structures based on bloom
filters to achieve space optimization and perfect statistical sensitivity, (iii)
algorithms for mapping long reads to a collection of reference genomes
represented by compact graph-based structures, (iv) algorithms for partitioning
long reads to facilitate identification of haplotypes in diploid assemblies, and
(v) long-read inspired alignment free algorithms for genome-to-genome
comparison, as well as important biological applications enabled by these. The
research will emphasize utilization of real datasets, relevance to practically
encountered problems and applications, and independent validation by
collaborators and other experts.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.