* 1817623
* Design Principles for Complex and Dynamically-Regulated CRISPR-Cas Gene Expression Programs in Bacteria
* BIO,MCB
* 09/01/2018,08/31/2022
* James Carothers, University of Washington
* Standard Grant
* Anthony Garza
* 08/31/2022
* USD 520,000.00

This project seeks to develop new tools to regulate bacterial gene expression to
support biosynthetic processes. Dramatic successes in metabolic engineering have
been achieved through laborious efforts to optimize the expression of multiple
genes to produce high-value chemicals. This project will develop programmable
CRISPR-Cas systems to simultaneously target multiple genes for activation and
repression. This will rapidly implement complex genetic programs without
extensive genome engineering. Key project goals include the development of
reliable control of synthetic transcriptional activators in bacteria. The
implementation of these control strategies will help to dynamically regulate
gene expression. The sophisticated control systems developed in this project
will be useful for practical biosynthesis, bacterial engineering, and basic
research in bacteria. These findings will be incorporated into educational
materials and courses taught to chemistry and engineering students. This project
will also provide opportunities for underrepresented students at the high school
and undergraduate levels to participate in laboratory
research.&lt;br/&gt;&lt;br/&gt;The goal of this proposal is to develop new tools
to systematically and dynamically regulate multi-gene expression programs, and
to identify regulatory architectures that can improve the output of biosynthetic
pathways. CRISPR-Cas transcriptional regulatory circuits will be used to
simultaneously up- and down-regulate both endogenous and heterologous genes
using a combination of CRISPRi-based gene repression and CRISPRa-based synthetic
activation. There are currently very few synthetic transcriptional activators
available for bacterial systems, and a key component of this proposal will be to
engineer new activation domains. The CRISPR-Cas system will be controlled by
inducible and/or dynamically-responsive promoters to regulate the timing of
multi-gene expression programs. To identify transcriptional programs that
improve biosynthetic yields, RNA-based biosensors for metabolic products will be
used to enable high-throughput screening of libraries of engineered strains.
Combining CRISPR-Cas tools for sophisticated control of multi-gene expression
programs with the ability to screen many variants will enable a systematic
exploration of a large space of dynamically-responsive regulatory architectures
to optimize bacterial biosynthesis.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.