* 1817551
* Experimental and theoretical investigations of gene regulation by chromosomal topological domains in E. coli
* BIO,MCB
* 09/01/2018,08/31/2022
* Jie Xiao, Johns Hopkins University
* Standard Grant
* Bianca Garner
* 08/31/2022
* USD 969,000.00

Bacteria are one of the simplest forms of life on Earth. Despite their relative
simplicity, each single bacterial cell is still a sophisticated living system
capable of precisely responding to changes in its environment. Over the last 70
years details describing of how this regulation occurs have been uncovered,
however there is still gaps in our understanding. New research is uncovering
evidence of a novel mechanism employed by bacterial cells to globally control
the regulation of genes using the structure of chromosomal DNA. This mechanism
was long thought to be missing from bacterial life forms. Investigators will
explore this previously unknown regulatory mechanism using an integrated
experimental and computational modeling approach. Successful completion of the
project goals will contribute to the understanding of the evolution of gene
regulation in bacterial systems. As part of the broader impacts of the project,
investigators will engage in interdisciplinary training of graduate students,
research experience for high school students, and summer programs for elementary
students within inner city communities in the Baltimore
area.&lt;br/&gt;&lt;br/&gt;The objective of the project is to investigate the
role of chromosomal topological domains in regulating gene expression in E.
coli. The project will combine single-cell imaging, biochemical analysis, and
theoretical modeling of these topological domains to uncover their global
regulatory effect on gene expression. The classic dogma of bacterial genetic
information processing focuses on the regulation of a gene's transcriptional
activity only through the mechanism of DNA and/or RNA sequence recognition by
proteins such as RNA polymerase and transcription factors. In recent years an
increasing number of studies have shown that the supercoiling state of
chromosomal DNA is another fundamental factor impacting gene expression in
bacteria. The topological organization of chromosomal DNA into individual
domains, between which the diffusion of supercoiling is prohibited, must play an
important role in gene regulation. However, knowledge of how chromosomal
topological domains are organized in E. coli, and how they impact gene
expression, remains elusive. In this project the two principal investigators
will combine their expertise in the experimental analyses of E. coli chromosomal
organization and theoretical modeling of bacterial gene regulatory networks to
address three specific aims: (1) Investigate whether and how the expression of
multiple genes within the same topological domain is coordinated using single
cell mRNA and protein production imaging methods; (2) Model gene regulation by
chromosomal topological domains at the genome scale; and (3) Determine the
correlation between chromosomal topological organization and gene expression
profile. Results of the proposed work will result in the development of a
topologically-informed gene regulatory network model.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.