* 1818344
* Simulating a minimal cell: Integrating experiment and theory
* BIO,MCB
* 08/15/2018,07/31/2023
* Zaida Luthey-Schulten, University of Illinois at Urbana-Champaign
* Standard Grant
* David Rockcliffe
* 07/31/2023
* USD 1,500,000.00

All cells share a universal, minimal set of biological processes essential for
life. The search for this set led to the construction of the minimal bacterial
cell JCVI-syn3A. With 493 genes in a genome of 543 kbp, JCVI-syn3A has a genome
smaller than that of any independently-replicating cell found in nature, a
robust morphology, and can divide every two hours in a stress-free laboratory
growth medium. Nearly all genes in this minimal cell are essential, and the cell
is small enough that a complete description of all cellular functions can be
attempted over biological relevant length, time, and concentrations scales by
exploiting graphics processing unit (GPU) computing. Recent successes in GPU
computing, and 3D imaging have made it now possible to build a whole-cell
computational model of this minimal bacterial cell and to investigate what are
the physical rules of life. In this project the investigators will study
hitherto uncharacterized genes in the minimal cell whose functions have not been
identified, and use this information to construct a whole-cell computational
model encompassing all cellular functions. The outcome of this project will
allow the research team to predict cellular behavior under a variety of
perturbations, and thus explain how a complete cell works. The educational
broader impacts include the training of students and postdoctoral investigators,
and outreach to the broader community through workshops and YouTube/VR platforms
facilitating the public dissemination of the science&lt;br/&gt;&lt;br/&gt;This
project aims to comprehensively characterize the minimal bacterial cell JCVI-
syn3A through multimodal experiments, and to integrate the heterogeneous data
into a multi-scale, predictive whole-cell computational model of the minimal
cell using novel simulation methods developed during this project. In
particular, the principal investigators have proposed two major aims. Major aim
1: Characterization of the minimal cell and its cellular networks. 1a: The PIs
will probe the function of the remaining genes of unknown, but essential
function (as determined by transposon insertion experiments) using CRISPRi-based
expression modulation and study the corresponding change in cellular phenotype.
1b: Investigators will refine and expand the existing metabolic model for JCVI-
syn3A by developing a defined growth medium as a basis for all subsequent
experiments; studying various aspects of cellular composition and functionality;
and expanding the steady-state metabolic model to a kinetic model. 1c: Visual
proteomics of JCVI-syn3A cells will be obtained using cryo-electron tomography
(CET) to extract cell-wide abundance and spatial distribution of large
macromolecular complexes. Major aim 2: Researchers will integrate the
heterogeneous experimental data into a whole-cell computational model using the
GPU-based Lattice Microbes software that can treat the spatially heterogeneous
environment of the cell. 2a: The team will engage in methodological development
of hybrid methods that will allow handling of species with vastly different
concentration ranges and dynamic behavior. To bridge these scales, investigators
will develop hybrid stochastic-deterministic methodologies that couple Reaction
Diffusion Master Equations (RDME) with Brownian dynamics (BD) and ordinary
differential equation (ODE) descriptions of cellular components. 2b: They will
integrate the metabolic network with models of ribosome assembly, transcription,
translation, mRNA/protein decay, DNA replication, cell growth and division. 2c.
Using their constructed, spatially resolved model the investigators will examine
the sensitivity of the cellular phenotype to the assignment of kinetic
parameters and validate the whole cell model at each stage of development
through comparisons to diverse biochemical, genetic and structural experiments
such as CET.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.