* 1826734
* RII Track-2 FEC: Single Cell Genome-to-Phenome: Integrating Genome and Phenome Analyses of Individual Microbial Cells in Complex Microbiomes
* O/D,OIA
* 08/15/2018,07/31/2024
* Ramunas Stepanauskas, Bigelow Laboratory for Ocean Sciences
* Cooperative Agreement
* Andrea Johnson
* 07/31/2024
* USD 6,589,587.00

Non-technical description&lt;br/&gt;The majority of biological diversity on
Earth is found in microscopic, single-celled microbes, which are responsible for
most of the cycling of carbon, nitrogen and other elements in the biosphere.
However, less than 1% of microbes have been successfully isolated for detailed
study in laboratory cultures, using traditional microbiology tools. Although
newer ?meta-omics? research tools circumvent the need for laboratory cultivation
by analyzing the molecular composition of entire microbial communities, these
tools often blend information from highly diverse organisms, making it difficult
to directly link the function of specific microbes (the ?phenome?) to their DNA
code (the ?genome?). This project will tackle this technological weakness
through a new combination of approaches that directly links functional and
genomic properties of individual microbial cells. This new approach combines the
project team?s ability to sequence the DNA of individual cells with the
utilization of specific compounds that microbes incorporate in their cells in
proportion to specific biological activities, such as respiration, growth, and
uptake of nutrients. The researchers will validate this new approach in
environments ranging from the Gulf of Maine to ocean crust and deep continental
subsurface. This project will leverage genome research facilities at the Bigelow
Laboratory Single Cell Genomics Center in Maine, expertise in the synthesis of
fluorescent organic probes at the University of New Hampshire, continental
geomicrobiology infrastructure at the Desert Research Institute in Nevada, and
ongoing oceanographic research programs in Maine and New Hampshire. The project
will contribute to the development of novel analytical capabilities for the
national Research and Development (R&amp;D) community, creation of new jobs, and
training of skilled workforce in all three
jurisdictions.&lt;br/&gt;&lt;br/&gt;Technical Description &lt;br/&gt;This award
is for a multi-jurisdictional project that will capitalize on existing pilot
results to achieve a breakthrough in microbial genome-to-phenome methodology at
the most fundamental level of biological organization: the single cell. The
overarching objective is to tie cell-specific genome data to expressed functions
such that rates of environmental processes can be coupled to specific microbial
lineages. The research team will: a) develop a new, integrated workflow for
genome-to-phenome analyses of individual cells that takes advantage of
fluorescent compounds to measure phenotypes; b) validate and utilize this
workflow in studies of globally significant environments; c) share novel
research tools with the research community through publications and core
facility services; d) improve sustainable research infrastructure at
collaborating EPSCoR jurisdictions; and e) create a multifaceted workforce
development program to ensure expertise in these new research approaches. The
project will establish and verify this workflow through three case studies that
leverage the team?s expertise and access to environments of differing microbiome
complexity: the coastal ocean, the deep dark ocean, and the marine and
terrestrial subsurface. To further expand the societal impact and technological
potential of the project, an added pilot study will focus on characterization of
individual microbes responsible for degradation of an environmental contaminant
(polyacrylamide/acrylamide). The overarching broader impact will be the
development of a new tool: a pipeline for linking phenotype information to
single microbial cells. This will have broad applicability across research
communities and industrial applications and address a societal need to
understand the critical function of microbes in the environment. The project
will leverage the Bigelow Laboratory?s capabilities in single cell genomics and
flow cytometry, New Hampshire?s expertise in the marine biogeochemistry of
polymers and the synthesis of labeled organic molecules, and the Desert Research
Institute?s (DRI) expertise and infrastructure in studying deep subsurface
environments. New faculty members will be mentored, including a new senior staff
member to be hired at Bigelow, and a new faculty member currently at the
University of New Hampshire.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.