* 1833182
* NSF EAGER:  A cross-kingdom comparison of single cell transcriptomes
* BIO,IOS
* 07/01/2018,06/30/2021
* David Jackson, Cold Spring Harbor Laboratory
* Standard Grant
* Diane Jofuku Okamuro
* 06/30/2021
* USD 299,412.00

Cells are the building blocks of all living organisms and different cell types
within an organism are defined by which of their genes they express. However,
the precise gene expression profiles that define specific cell types are largely
unknown. In addition, it is not known how cells come to express the set of genes
that define one cell type versus another or whether the rules that specify cell
types differ between plants and animals. This last question is especially
interesting because being multicellular evolved independently in these two
different kingdoms. The project will address these important questions by
sequencing and analyzing mRNA from individual maize cells of different types,
making these sequences and their methods and analytical procedures available to
the wider plant research community. The project will also contribute to outreach
and education by training undergraduate and high school students in genomics and
computational biology as part of the Cold Spring Harbor Laboratory Undergraduate
Research Program, which has a focus in quantitative biology, as well as the CSHL
Partners For The Future Program, both of which train members of groups
underrepresented in science. Students from each of these programs will be
engaged in this project in both wet lab experiments and in bioinformatics and
validation experiments, providing cross-disciplinary training in developmental
biology and computational biology.

Living organisms develop and function by complex interactions between different
specialized cell types. Using an optimized protocol to protoplast developing
maize inflorescences, this team will isolate hundreds of thousands of single
cells rapidly from different regions of this developing organ. The project will
apply expertise in cell biology, genomics and single cell gene expression
analysis to identify cell specific expression signatures in maize and to ask how
cell type specification compares between plants and animals. A transformative
new technology, single cell RNA sequencing (scRNA-seq), offers for the first
time the opportunity to profile single plant cells and understand plant
development at a fundamentally new level. This method has the power to
understand cell type specific networks, to understand transcriptional
heterogeneity within cell types and to identify novel regulators of cell fate.
The project will test different scRNA-seq technologies and develop informatics
tools that can be widely adopted by the plant community, and will be
transformative in giving unique insights into single cell transcriptomes across
different kingdoms of life. Therefore, this project has the potential to uncover
the rules by which cell types are specified across kingdom wide scales. Data
generated in this project will be openly available for reanalysis and
integration with other datasets. The project will also develop and adapt single
cell analysis methods that will be broadly applicable for the plant community.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.