* 1838345
* RoL: FELS: EAGER:   Connecting RNA Molecular Kinetics to Developmental Regulation
* BIO,MCB
* 09/01/2018,08/31/2021
* Reed Sorenson, University of Utah
* Standard Grant
* Manju Hingorani
* 08/31/2021
* USD 299,995.00

Understanding how cells perceive and respond to developmental signals is
critical to understanding how an organism develops its overall shape and
structure. One important way that cells respond is through changes in gene
expression, which is typically monitored by measuring mRNA abundance. However,
changes in mRNA abundance have not been a highly predictive tool for
understanding growth and development. This project will address whether specific
mRNA decay characteristics improve predictive power by investigating RNA decay
rates in the plant Arabidopsis thaliana in response to signals that drive
development. Information learned through this study will provide deeper
understanding of developmental signals and their cellular responses, which will
yield better predictions of how targeted genetic changes might lead to desired
traits in agriculture and medicine. Project participants will be involved in
local K-12 science education. Undergraduate scientists, including those from
under-represented groups, will be trained in modern molecular and cell biology
techniques. &lt;br/&gt;&lt;br/&gt;This project will investigate a rule of life
that high rates of RNA flux, or fast RNA turnover, provide predictive power when
analyzing transcriptome data because high-flux RNAs are expected to have the
fastest abundance-level response to an inductive signal. The project will follow
mRNA decay rate responses to inductive signals with a focus on vascular
development, an important emergent property of plants. mRNA decay rates will be
analyzed genome-wide using mathematical modeling approaches to identify two mRNA
decay rate parameters, flux and the beta parameter, and mRNAs whose decay rates
change over the time course of vascular development. The project will test
whether changes in mRNA decay rates are necessary by imposing constitutive fast
or slow decay, while controlling for protein abundance, and comparing induced
vascular development responses of wild type and decay-rate-modified tissues. The
new experimental tool for engineering stability (or instability) into RNAs has
the potential to also impact synthetic biology approaches. Elucidation of the
role of RNA flux in fine control of gene expression as a rule of life might lead
to better forecasting of the post-transcriptional regulatory potential of genes
responsible for controlling emergent properties such as development of leaf
vascular patterning.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.