* 1844331
* ERA-CAPS: BARley yield associated Network
* BIO,IOS
* 08/15/2018,07/31/2023
* Gary Muehlbauer, University of Minnesota-Twin Cities
* Standard Grant
* Gerald Schoenknecht
* 07/31/2023
* USD 778,920.00

Agriculture faces the 'Grand Challenge' of feeding a growing population with
less land, fewer resources, a changing climate, and increasing environmental
quality demands from society. Thus, crops that produce more per unit area with
fewer resources and are both resilient and adaptable will be key components in
addressing this Grand Challenge. An important first step towards robust
agricultural systems is to develop a deep understanding of the genetic control
of yield. In barley, yield is determined by many components and the ability to
increase yield through standard breeding approaches has slowed. The primary goal
of this ERA-CAPS project is to identify genes that control yield-related traits
in the small grain temperate cereal crop barley, and to begin to understand how
they contribute to this complex phenotype. The three partners in this project
will separately focus on seed, spike (inflorescence) and tiller (seed bearing
stem) traits. The project will benefit from bringing together a collection of
genetic resources and expertise from the three partners. Long-term community
integration will be assured by sharing the resources, tools and information with
the broader barley genetics and breeding community and by training the next
generation of molecular breeders and scientists.&lt;br/&gt;&lt;br/&gt;In barley,
yield is determined by many components and the ability to increase yield through
standard breeding approaches has slowed. Barley grain yield is impacted by three
components including: seed (size and number), spike (inflorescence) and tillers
(vegetative branches). The overall goal of this ERA-CAPS funded project is to
identify genes that control morphological and developmental components of yield
in barley, and to begin to understand how they contribute to this complex
phenotype. The research leverages a previously exome captured sequenced
germplasm panel, which will be phenotyped for yield-related traits. These data
will provide the opportunity to gain an increased understanding of GxE
interactions for key traits. The team will conduct transcriptome sequencing of
six trait-related tissues per genotype and analyze how gene expression relates
to trait development. To enable accurate quantification of transcript read
depth, a reference transcript dataset will be developed by using deep paired-end
Illumina RNA-seq and PacBio ISO-seq data from six tissues from cv. Morex. This
will be used to quantify transcript abundance from RNA-seq data collected from
the same six tissues sampled across the population. RNA-seq derived SNP alleles
will supplement the exome capture SNPs and both SNP datasets and transcript read
depth variation will be used for association analysis with yield-related traits.
Gene co-expression networks will be constructed using reference transcript
dataset from Morex and RNA-seq data from the six tissues sampled from the
germplasm panel. Both datasets will be integrated to identify candidate genes or
key regulators of yield related traits. Functional characterization of candidate
genes will be initiated by identifying deleterious alleles from TILLING
populations available in each of the three partner labs. The project will
provide global community resources of broad impact including: a reference
transcript dataset, additional SNPs derived from the landrace collection, and
gene co-expression networks for exploring key regulatory genes and their
relationship to yield-related traits.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.