* 1850692
* Bacteria as biosensors of marine carbon and energy flow: Quantitative links between substrates, transcriptomics, and metabolism
* GEO,OCE
* 08/01/2019,01/31/2024
* Scott Gifford, University of North Carolina at Chapel Hill
* Standard Grant
* Daniel J. Thornhill
* 01/31/2024
* USD 525,015.00

The formation and flux of organic material is the foundation of ocean
ecosystems, which in turn, substantially influences the global carbon cycle. As
such, a fundamental goal in the ocean sciences is increasing our ability to
identify marine organic matter's sources, transformations, and sinks, as well as
how these components may change due to anthropogenic activities. Understanding
these components is especially important in estuarine and coastal systems given
these ecosystems are critical zones of organic carbon transformations. However,
the dissolved organic carbon (DOC) pool in these systems consists of numerous
different compounds from a multitude of sources that can turn over at vastly
different rates (minutes to millennia). This makes it difficult to identify
which DOC compounds support microbial growth, limiting the incorporation of
microbial metabolism into predictive ecosystem models. Novel approaches are
therefore needed to identify the DOC substrates driving microbial metabolism in
ocean ecosystems. This project is premised on the idea that the bacterial
cellular system is the ultimate chemical sensor of the organic environment and
that the information recorded in the cell's active gene pool (transcripts) can
be leveraged to make insights into DOC composition when the relationships
between organic substrate availability, gene activity, and metabolism are known.
This project identifies substrate-transcript relationships for a model marine
bacterium, as well as the growth and metabolic outcomes of substrate
availability. These insights are used to identify the biologically active DOC
substrates in coastal environments when the model organism is added directly to
coastal samples, and to interpret both historical and current environmental RNA
and DNA data sets. This work provides novel insights into the substrates driving
the ocean's carbon cycle and how marine bacterial cellular systems are
regulated. Bioassays are developed that can be applied in many different aquatic
environment settings. The project trains graduate and undergraduate students
directly involved in the research and minority undergraduates will be recruited
to use research modules for hands-on study of cell cultivation, bioinformatics,
and microbial metabolism. High school students will be engaged through a module
developed for an aquatic microbiology field trip and subsequent sample and data
analysis.&lt;br/&gt;&lt;br/&gt;Bacterial processing of dissolved organic carbon
(DOC) mediates the flux of gigatons of carbon in the ocean, yet a significant
hurdle to incorporating bacterial metabolism into ocean models is the inability
to quantify the DOC substrates supporting bacterial metabolism and their
transformation. Metatranscriptomics (sequencing of community mRNAs) has the
potential to be a sensitive method for surveying bacterioplankton responses to
the DOC pool and making insights into its composition but is currently limited
by insufficient knowledge as to how transcriptional patterns relate to substrate
availability. This project will identify carbon substrates supporting microbial
metabolism and their transformation in estuarine-coastal ecosystems by
elucidating the relationships between transcript abundances and carbon substrate
availability. It aims to bridge the gap between model organism and environmental
-omic studies by creating quantitative inventories of transcripts in response to
defined substrates, and then using these calibrated transcriptional signals to
interpret environmental DOC bioassays and metatranscriptomes. The first
component of the project will establish genome-wide transcript-substrate
relationships in a model marine bacterium in response to individual,
environmentally-relevant carbon substrates. The second component will determine
the extent to which transcription and metabolism are altered when the bacterium
is exposed to complex mixtures of defined and undefined substrates, revealing
the potential for transcription to identify individual substrates within a
complex DOC pool and how metabolic processing may shape the DOC pools labile and
refractory components. Finally, these calibrated transcriptional responses will
be used to identify the DOC substrates driving bacterial metabolism in an
estuarine-coastal system via DOC drawdown bioassays in which the model organism
is added to natural seawater samples, as well as community wide bacterioplankton
responses to the extant DOC pool via
metatranscriptomics.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.