* 1850492
* CRII: III: A Scalable Probabilistic Model Selection Method for Deep Learning in Gene-Protein Network Inference and Integration
* CSE,IIS
* 09/01/2019,08/31/2023
* Rui Li, Rochester Institute of Tech
* Standard Grant
* Sylvia Spengler
* 08/31/2023
* USD 171,922.00

Detailed characterization of interactions between individual genes and proteins
has been one of the focuses of biological research, since these interactions
control cellular processes involving complicated cascades of biochemical
reactions and signaling pathways. Dysregulation of any component of these
pathways can lead to a broad spectrum of human pathologies including cancer,
cardiovascular disease, neurodegenerative conditions, and metabolic diseases.
Deep learning techniques, which is attributed to the most recent advances in
artificial intelligence and machine learning, have emerged and many studies have
tried to construct and analyze gene/protein interactions at a genome/proteome-
wide scale to describe their global characteristics. However, the heterogeneous
nature of the biological data and their continuous evolution pose a unique
challenge to the architecture design of deep neural networks. The state-of-the-
art solutions heavily rely on expertise, heuristics, and experimentation, and
are time-consuming and not scalable. This research proposes to enable deep
neural networks to automatically go through a qualitative growth to accommodate
richer information from new heterogenous data as they are accumulating. This
project will provide powerful novel computational tools to discover and target
gene-protein interactions driving regulatory diseases such as cancer, diabetes
and neurological disorders, while gaining insight into the fundamental
principles behind cellular information processing.&lt;br/&gt;&lt;br/&gt;This
project will focus on developing scalable probabilistic model selection
approaches to infer deep neural network architectures for gene-protein
interaction network inference and integration. The project will design efficient
inference algorithms for the proposed model selection approach to enable its
translation into a deployable tool for use by biologists. The researchers will
develop novel principled model selection methods to infer the most plausible
architectures of deep neural networks warranted by the heterogenous biological
data. By modeling the hypothesis space of neural network architectures as
stochastic processes, the proposed method enables neural network architectures
to evolve according to the biological data; to design and implement efficient
techniques to make use of the inference computationally tractable. The project
proposes to evaluate the marginal likelihoods for preference to the alternatives
approximately based on variational methods. The investigator will compare the
performance of the deep neural networks whose architectures are learned with the
proposed model selection method with the state-of-the-art methods on networks
and functional annotations of eukaryotic organism S. cerevisiae and human cells
by treating protein function prediction as a multi-label classification problem,
and measuring the performance with two complementary approaches: cross-
validation and temporal holdout validation.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.