* 1856744
* RESEARH-PGR:  Genomics of strain- and host-specific performance in the legume-rhizobia symbiosis
* BIO,IOS
* 08/01/2019,07/31/2023
* Katy Heath, University of Minnesota-Twin Cities
* Standard Grant
* Kan Wang
* 07/31/2023
* USD 1,768,277.00

The plants and animals we see around us, including agriculturally important crop
plants, live in close association with millions of microorganisms (microbes).
Some of these microbes cause disease but others are beneficial. Identifying the
genes and genetic pathways that determine the outcome of host-microbe
associations provides targets for manipulating them for human benefit. This
project uses an innovative approach based on high-throughput genome sequencing
to efficiently identify bacterial and host genes that shape the outcome of host-
microbe associations. The new methods are being deployed on one of the most
important plant-microbe associations, that between leguminous plants (soybeans,
beans, peas, alfalfa) and rhizobial bacteria. When rhizobia grow in association
with these plants, they convert atmospheric nitrogen into a plant useable form,
thereby "feeding" the plant and lessening the need for nitrogen fertilizer. The
results will provide new empirical tools that advance how we study these systems
and the identification of genes and bacterial strains that can be exploited to
increase the benefits rhizobial bacteria provide in agricultural systems. Teams
of student writers and designers, as well as local high school teachers, will be
actively involved in the project to communicate the results using print,
graphics, and animation.&lt;br/&gt;&lt;br/&gt;The symbiosis between leguminous
plants and rhizobial bacteria is a classic system for understanding how plant
and microbial genomes function in a coordinated manner. The benefits of host-
rhizobia interactions depend on genomic variation found in both partners (i.e.
inter-genome or genome-by-genome variation). Neither the reasons for nor the
consequences of this inter-genome variation are well understood. Moreover, very
little is known about the genetic basis of genome by genome variation in this,
or any other, host-microbe system. The research under this award will fill this
void through a novel two-species genotype-phenotype association study, two-
species co-expression networks, and functional validation of candidate genes. A
similar approach will be used to evaluate the inter-genome variation between
Medicago and Aphanomyces, an economically important pathogen. The data and code
used for analyses will be made available in a timely manner through public data
repositories. Analyses of these data will uncover, in unprecedented detail, the
genetic basis and mechanisms responsible for variation in inter-genome effects
on phenotype in host-symbiont and host-pathogen systems. To extend the reach of
the project and actively disseminate the results, students from other
disciplines (such as writers and graphic designers) will develop new
communication materials that will be used in local high school
curricula.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.