* 8912312
* REU: Ribosomal RNA Sequences in Marine Yeasts:  A Model for Identification and Quantification of Marine Eukaryotes
* GEO,OCE
* 11/15/1989,10/31/1992
* Jack Fell, University of Miami
* Continuing Grant
* Phillip R. Taylor
* 10/31/1992
* USD 228,997.00

The ability to distinguish species of marine microorganisms (bacteria
protozoans, fungi and microalgae) is a major problem in understanding the
processes of primary production and organic recycling. Developments in molecular
biology are providing an array of tools to genetically characterize and quantify
marine microbes. The first step in applying these techniques to problems in
biological oceanography is to develop species- specific genetic markers. There
has been extensive study on the 5S, small (SSU) and large (LSU) subunits of
ribosomal RNA (rRNA). Studies of conserved regions of rRNA, which evolve slowly,
have been instrumental in developing phylogenetic lines of evolution. In
addition to conserved regions, there are variable regions in LSU that evolve
rapidly. Initial investigations, in Dr. Fell's laboratory and elsewhere, suggest
that nucleotide sequences in the variable regions are sufficiently conserved to
distinguish organisms at the level of species and genus. This research will
amplify preliminary studies of the variable regions of LSU rRNA using marine
yeasts as a model for other marine eukaryotes. The yeasts are uniquely
appropriate as they can be grown in culture and their nucleotide sequences can
be compared with known mating genetic markers. Nucleotide sequencing will use
the established techniques of Sanger and Lane. Recognition of nucleotide
sequences that are species specific, in combination with techniques such as the
polymerase chain reaction and the use of molecular probes and hybridizations,
provide powerful tools for the amplification of nucleic acids and the
identification and quantification of marine microorganisms from single cell
material.