* 1612170
* NSF Postdoctoral Fellowship in Biology FY 2016
* BIO,IOS
* 07/01/2016,06/30/2019
* David Mitchell, Mitchell                David
* Fellowship Award
* Diane Jofuku Okamuro
* 06/30/2019
* USD 216,000.00

This action funds an NSF National Plant Genome Initiative Postdoctoral Research
Fellowship in Biology for FY 2015. The fellowship supports a research and
training plan in a host laboratory for the Fellow who also presents a plan to
broaden participation in biology. The title of the research and training plan
for this fellowship to David Mitchell is "A Genome-wide Investigation of Light
Intensity Effects on the Maize RNA Structurome". The host institution for the
fellowship is the Pennsylvania State University and the sponsoring scientists
are Dr. Philip C. Bevilacqua and Dr. Sarah M. Assmann.

As maize is a vital food and fuel crop, efforts have been made to increase maize
yields by increasing planting density. Dense planting increases the canopy cover
and blocks sunlight from reaching the lower leaves, thus stimulating shade
avoidance responses which limits crop yield. Information obtained from this
research can contribute to genetic engineering of maize to improve yield under
high planting densities. Powerful tools used and developed in this research will
revolutionize how RNA structure and function are studied. Through this project,
the researcher will gain novel expertise in plant biology techniques and in
computational and bioinformatics analysis. Furthermore, for six weeks annually
underrepresented high school students will be mentored in laboratory research
projects.

This research will adapt Structure-seq to maize to perform high-throughput,
genome-wide in vivo investigation of gene expression, RNA secondary structures,
and RNA-protein interactions in response to low-light stress. By applying new
structure-probing chemical reagents that allow Structure-seq to probe all four
RNA bases, this research will better enable Structure-seq to distinguish RNA-RNA
from RNA-protein interactions. It will examine the use of in vivo crosslinking
via photoactivatable nucleoside analogues to provide direct proximity
information on these interactions. Furthermore, it will modify the Structure-seq
RNA structure prediction algorithm to incorporate constraints from proximity
information, sequence complementarity, and known protein binding motifs. Data
generated from this research will be uploaded upon processing to the appropriate
community databases, including the NCBI Sequence Reads archive (SRA) for RNA
structure data (http://www.ncbi.nlm.nih.gov/sra) and GEO for RNA-seq data
(www.ncbi.nlm.nih.gov/geo).