* 1613203
* Collaborative Research: RUI: Broadening the Application of Programmed Evolution for Metabolic Engineering
* BIO,MCB
* 09/01/2016,08/31/2021
* Laurie Heyer, Davidson College
* Standard Grant
* David Rockcliffe
* 08/31/2021
* USD 630,042.00

Design considerations during metabolic engineering have been based on incomplete
understanding of the evolutionary forces acting upon populations of engineered
bacteria. This project inverts the metabolic engineering paradigm by harnessing
evolution instead of fighting it. The investigators developed an evolutionary
approach to metabolic engineering that enables bacteria to integrate their
growth environment and their engineered metabolism. The approach is called
Programmed Evolution because a population of bacteria is programmed with DNA
software to compute solutions to a metabolic pathway optimization problem, and
evolution is used to direct the bacterial population toward optimal solutions.
The goal of this project is to expand Programmed Evolution by developing a new
method for new riboswitch discovery, using the new riboswitches optimally
express the enzymes necessary for e the production of new compounds, and
developing mathematical models and computational tools to support both
processes. These approaches will reduce the cost of producing useful compounds
in metabolically engineered bacteria for applications in energy,
pharmaceuticals, and bioremediation. This project will increase diversity for
science education, contribute to a competitive and scientifically literate
workforce, and has the potential to improve American economic competitiveness.
This project responds to the call in Vision and Change for more authentic
undergraduate research experiences as undergraduates pose research questions,
develop testable hypotheses, collect and analyze data, and communicate results.
As they learn how to program the evolution of bacteria for metabolic
engineering, undergraduate research students will learn how to program the
course of their own futures as science literate citizens, educators, and
research scientists.&lt;br/&gt;&lt;br/&gt;Programmed Evolution is a modular
system for the optimization of orthogonal metabolic pathways in bacteria. It
uses combinatorics, fitness, and biosensor modules that can be developed and
tested separately, used in combinations, and shared among research groups. A key
component of Programmed Evolution is the riboswitch that transduces metabolic
output into fitness gene expression and selective advantage. Most riboswitches
used in metabolic engineering incorporate RNA aptamers discovered by the in
vitro process of Systematic Evolution of Ligands by Exponential enrichment.
However, aptamers discovered in vitro rarely function in vivo. The investigators
propose to develop Cell-based Exponential enrichment as a new in vivo method of
discovering riboswitches that function predictably in bacterial cells. The new
method introduces genetic variation in a riboswitch to produce a library,
applies negative and positive selection, and characterizes the phenotype and
genotype of new riboswitches. The significance of the new approach derives from
its potential to advance knowledge of naturally occurring riboswitches and to
discover new riboswitches for applications in energy, pharmaceuticals and
bioremediation.