* 1615789
* Development of Statistical Methods for Analyzing Whole Genome Bisulfite Sequencing Experiment Data to Identify Differentially Methylated Regions
* MPS,DMS
* 06/01/2016,05/31/2020
* Tieming Ji, University of Missouri-Columbia
* Standard Grant
* Junping Wang
* 05/31/2020
* USD 239,670.00

DNA methylation is a chemical modification to DNA that imparts information on
how and when genes should be turned on or off. As such DNA methylation plays a
vital role in many biological processes. There are many instances in which DNA
methylation between samples is different. Examples are: 1) different organs
(liver vs brain), 2) health vs disease (cancer exhibits methylation patterns
that are different from those of healthy tissue), and 3) environmental responses
(plants that endure heat and drought stress). These differences are known as
differential methylation. Recently, technology has been developed to
simultaneously query the millions of methylation sites in DNA.  This technology,
however, creates computational and statistical challenges. A pressing question
in the field of biology is how to analyze the massive amount of data generated
in this way so as to draw biologically relevant, and statistically sound
conclusions, without requiring expensive computing equipment. The output of this
project will provide statistical methods for detecting differential methylation.
This information can then be used by specialists to understand cell function,
develop therapeutic interventions, or tackle questions associated with
environment such as climate change. Freely available statistical tools that can
be used by all scientists interested in analyzing differential methylation will
be developed. This interdisciplinary research project between the mathematical
and biological sciences also supports the training of students in these fields.
&lt;br/&gt;&lt;br/&gt;DNA methylation is an epigenetic modification that directs
gene expression and chromatin conformation. The most common form of DNA
methylation in vertebrates is an addition of a methyl group to a cytosine base
that is directly followed by a guanine, which is referred to as a CpG site.
There are approximately 30 million CpG sites in the human genome with an even
larger combination of states of methylation. A differentially methylated region
is a region in the genome where mean methylation levels of CpGs are different
between two sample groups, such as disease versus normal. Bisulfite sequencing
methods are typically used for measuring DNA methylation at CpG sites. Expansion
of this technology to permit simultaneous query of all CpG sites, known as whole
genome bisulfite sequencing, has created computational and statistical
challenges. These include improving the capability to distinguish signals from
noise in very large datasets, a current focus of much modern statistics. The
project team will develop statistical methods to detect differential methylation
between sample groups. Specifically, the methods in this research project
improve existing approaches by recognizing and accounting for correlations among
methylation sites based on their genomic locations. Such model assumptions
ensure that statistical results are biologically meaningful and interpretable.
In addition, the methods will borrow information across the whole genome to
improve estimation and statistical testing reliability. Bayesian models with
theoretical justifications will be developed and implemented using efficient and
scalable algorithms to ensure their applicability to a wide variety of high-
throughput methylation datasets. Methods will be implemented into software tools
and will be freely available for biology and statistics researchers.