* 1616827
* Is Abiotic Stress Tolerance Achieved by Network Restructuring or Invention of New Genetic Modules?
* BIO,MCB
* 08/15/2016,07/31/2021
* Aaron Smith, Louisiana State University
* Continuing Grant
* Stephen DiFazio
* 07/31/2021
* USD 900,000.00

The project involves exploration of how plants adapt naturally to harsh
environments, in order to understand stress tolerance mechanisms and inform
development of food and energy crops that can better tolerate changing
environments. The outcomes have broad societal relevance, as crop production is
frequently constrained by environmental stresses, such as salinity. This project
will provide a learning platform for diverse teams to design new computational
and molecular tools for studying naturally existing genetic variation. These
teams will consist of biological and computer science graduate students,
undergraduates, including underrepresented minorities, high school students and
instructors, and international collaborators. The project also supports career
development for two beginning investigators. Data generated from this project
will be used to design an interactive visual data analysis interface with
contributions from high school student programmers. Other activities include
plant nutrient awareness events for the public and K-12 students, a summer
workshop to train high school teachers and students in PCR and DNA
fingerprinting, and discussions with local high school students on ethical
issues related to genetic engineering. DNA sequence datasets and bioinformatics
programs resulting from this project will be shared publicly, which will enable
scientific activities ranging from basic research exploring mechanisms in
genetics and evolution to generating markers and resources for crop
improvement.&lt;br/&gt; &lt;br/&gt;This project will investigate how a species
achieves stress adaptation compared to a closely related stress sensitive
species by analyzing genomic signatures that lead to transcriptomic responses to
stress, followed by targeted transgenics and molecular phenotyping. The overall
analysis will be tested in a conceptual framework that highlights stress
adaptation achieved by modifications in existing core stress response systems,
and/or recruitment of new genetic components. Extremophile plants represent
emerging models for understanding genetic mechanisms governing plant survival
under extreme abiotic stresses. Schrenkiella parvula and Eutrema salsugineum,
the two most salt-tolerant species among known wild relatives of Brassica crops,
will be used as model genomes, and multiple salt stresses will be applied to
study adaptation compared to the stress sensitive model A. thaliana. Genomes of
both S. parvula and E. salsugineum show extensive overall synteny with the A.
thaliana genome, which enables comparative studies benefiting from the wealth of
genetic information available on A. thaliana. Inter-species comparative RNAseq
methods are planned to uncover the dynamics of abiotic stress responsive gene
regulation caused by excessive Na+, K+, or Li+ salts. Transgenic plants in all
test species will be created targeting representative genes in major stress
responsive units identified. Molecular level phenotyping will be achieved via
ionomic and metabolomic profiling of wild-type and transgenic lines collected
from control and stress-treated samples. Finally, descriptive genetic modules
will be developed with a core data set linking genomic restructuring that
enables improved stress adaptation in known and novel stress responsive
pathways, in both stress-adapted and stress-sensitive species. The work will
provide a strategic case study that can be expanded to other organisms, to
evaluate genomic organization via targeted genetic studies assisted by
transgenic plants, transcriptomic regulation, and metabolomes used for molecular
phenotyping.&lt;br/&gt;&lt;br/&gt;This project is co-funded by the Genetic
Mechanisms Cluster in the Division of Molecular and Cellular Biosciences in the
Biological Sciences Directorate and by the NSF EPSCoR Program.