* 1616997
* Engineered Selective Pressure: Challenges and Opportunities
* BIO,MCB
* 09/01/2016,08/31/2020
* Jeff Hasty, University of California-San Diego
* Standard Grant
* David Rockcliffe
* 08/31/2020
* USD 1,167,762.00

Rapid advances in the forward engineering of complex genetic circuits in living
cells proved that the young discipline of Synthetic Biology has the potential to
transform modern biotechnology. A predictive understanding of living systems is
a prerequisite for designed manipulation in bioengineering and informed
intervention in medicine. Such an understanding requires quantitative
measurements, mathematical analysis, and theoretical abstraction. The advent of
powerful measurement technologies and computing capacity has positioned biology
to drive the next scientific revolution. Synthetic Biology provides a natural
platform for the development and testing of models and general network
principles using quantitative data from engineered genetic circuits. One of the
major fundamental problems of Synthetic Biology is the propensity of synthetic
circuits to evolve and obliterate their intended functionality. Thus, it is
important to develop a quantitative approach to make genetic networks more
robust and stable which also will make them more useful in therapeutic
applications. The overall goal of this project to explore the challenges and
exploit the opportunities that accompany the inevitable selective pressure that
arises in synthetic biology through a combination of cutting-edge experimental
and computational tools. Successful accomplishment of this project will lead to
significant advances in understanding the interaction between synthetic circuits
and host genome in the evolutionary context. This project will provide ample
opportunities for cross-disciplinary training of the new generation of
quantitative biologists. Furthermore, to broaden the impact of this project
beyond academia, its participants will expand a highly successful elementary
school science program that that foster collaboration between researchers and
teachers at partner elementary schools to improve hands-on science
education.&lt;br/&gt;&lt;br/&gt;To reach the overall goal of the project, the
recently developed "synchronized lysis circuit" will be used as the major focus.
This synthetic gene circuit functions by triggering death of around 90% of the
bacteria at a threshold population density, leaving the remaining 10% of the
cells to grow back to threshold and restart the cycle. This generates a distinct
phenotype characterized by population cycling that can be monitored across a
wide range of length scales using micro- and bench-top chemostats. It also
introduces a strong selective pressure that can lead to rapid evolution. The
investigators will develop quantitative measurement technology and computational
modeling that will lead to a quantitative statistical characterization of the
circuit-host evolution process. Sequencing of the circuit and mutated sections
of the host genome along the time course of the experiments will be used to
reconstruct the evolutionary path the systems have taken. Mathematical modeling
will guide experiments and help extract key parameters characterizing
evolutionary dynamics of the synchronized lysis circuit. Using the gained
knowledge, the researchers will engineer a "lysis circuit stabilizer" module
that kills mutant bacteria losing lysis efficiency, and explore ways to use
lysis circuit as a tool in directed evolution of synthetic
circuits.&lt;br/&gt;&lt;br/&gt;This award was co-funded by the Systems and
Synthetic Biology (SSB) program in the Molecular and Cellular Biosciences (MCB)
Division in the Biological Sciences Directorate and the Biotechnology and
Biochemical Engineering (BBE) program of the Division of Chemical,
Bioengineering, Environmental and Transport Systems (CBET) in the Engineering
Directorate.