* 1616737
* RUI: Dynamics of Genomic Mosaicism in Non-Host Associated Escherichia Populations
* BIO,MCB
* 08/15/2016,07/31/2021
* Aaron Best, Hope College
* Continuing Grant
* Anthony Garza
* 07/31/2021
* USD 775,316.00

This project aims to address whether Escherichia from watersheds can serve as a
reservoir for horizontal gene transfer to host-associated strains such as
Escherichia coli, which is a known source of human infections. Watersheds can
become contaminated with Escherichia via treated sewage, livestock farming, wild
animals, fowl, and sediment runoff from urban and agricultural environments.
Several studies revealed that Escherichia may adapt and persist in these
environments, and eventually may be able to pass the genes for new traits to
host-associated strains such as Escherichia coli. In addition to examining this
idea, the data from this study will allow the broader scientific community to
evaluate existing water quality monitoring methods. The research for this
project will be integrated into authentic research experiences for undergraduate
students in STEM fields through course-based research experiences (CREs). In
particular, undergraduates in first year and upper level laboratories will
actively participate in all aspects of the project.&lt;br/&gt;&lt;br/&gt;This
project focuses on two related scientific goals: 1) characterizing genomic
diversity of environmentally derived Escherichia isolates to understand
variation in naturally occurring populations over time; and 2) linking profiles
of 16S community surveys to observed variation in Escherichia isolate genomes
and to environmental changes in a watershed. Remarkable genetic diversity has
been characterized by global studies of E. coli through analysis of
representative fully sequenced genomes, and targeted studies of pathogen sub-
types through hundreds of draft genome sequences of clinical isolates. The
preponderance of genomic data is from clinical and host-associated environments.
This project produces 720 draft genome sequences of environmentally derived
Escherichia isolates. The longitudinal data enable testing hypotheses about
genome structure and genetic exchange in naturally occurring, non-host-
associated populations through comparative genomics of genome content,
arrangement and single nucleotide polymorphisms. The data address long term
persistence of Escherichia populations in non-host environments, potentially
serving as reservoirs for genetic exchange and renewal of fecal indicator
bacteria in bodies of water. Oligotyping will estimate diversity in the
Escherichia populations; water samples will produce coupled isolate genome
sequences and microbial community 16S rRNA sequences. It is asserted that single
nucleotide changes in 16S rRNA represent diversity of genome structure; the data
will be used to test this assertion. Fine scale sampling produces longitudinal
16S rRNA microbial community profiles used to understand changes in Escherichia
and other microbial populations. Work linking 16S rRNA variation to genome
variation enables prediction of population changes based on 16S rRNA surveys
alone. High quality recreational and drinking water sources are essential for
society, and as water quality monitoring shifts to molecular based approaches,
the types of datasets produced here become critical to ascertaining effective
and broadly applicable monitoring techniques. A third goal fully incorporates
undergraduate students in STEM fields through individual and course-based
research experiences (CREs). Over 100 students in courses will gain first-hand
knowledge of authentic research. The effectiveness of these experiences will be
assessed through convergent mixed methods approaches designed to gauge student
attitudes toward STEM fields, comparing effects of individual research
experiences and CREs on first year and upper level students.