* 1616955
* Physiologic state modulation by conditional translational complexes
* BIO,MCB
* 09/01/2016,08/31/2021
* Nitin Baliga, Institute for Systems Biology
* Standard Grant
* Anthony Garza
* 08/31/2021
* USD 1,199,383.00

The process of gene expression, encompassing transcription and translation, is
closely interconnected with cellular physiology, and is carried out by all
organisms on Earth. Yet, the process of translation and its impact on cellular
physiology is generally thought to be subordinate to transcriptional regulation
and post-translational signaling. This notion continues despite the fact that
both transcription and signaling depend on proteins, and hence on translation.
This project seeks to identify a universal aspect of gene expression and
physiology that has been overlooked. This project will test the novel idea that
proteins needed under particular environmental conditions are preferentially
translated when exposed to such conditions. The proposed studies may uncover a
new molecular mechanism for gene regulation and will greatly advance our
understanding of the unifying, central role of translation in controlling a
myriad life processes. In addition, the project will contribute to training of
K-12 students and teachers, and high school science curriculum development.
&lt;br/&gt;&lt;br/&gt; The proposed research will address whether functional
diversity of components of the translation system and their transcriptionally
generated modular expression regulate large-scale physiological state
transitions in organisms. The underlying hypothesis is that environment-
dependent physiological cell states are generated by the conditional production,
assembly, and activity of distinct ribosomal complexes with variable subunit
compositions. This hypothesis is based on the intriguing organization of
translational machinery genes within gene regulatory networks of
phylogenetically diverse organisms. Specifically, ribosomal subunits and other
translation system proteins are conditionally co-regulated as multiple distinct,
yet overlapping modules with un-correlated expression patterns across
environmental shifts. The proposed research will attempt to observe conditional
association of certain ribosomal subunits, and whether this association directs
the translation complex to preferentially translate transcripts encoding
functions for a particular environment-relevant physiological state. Protein
(using SWATH mass spectrometry) and mRNA (with RNA-seq) compositions of
ribosomal complexes will be characterized across environmental shifts (e.g.,
aerobic to anaerobic) that effect large physiological state transitions.
Additionally, the proposed research will predictably manipulate the
physiological state of each organism by engineering environment-responsive
regulation or knock outs of conditional ribosomal subunits. Altered regulation
of specific conditionally expressed ribosomal subunits should manifest an
inappropriate physiological state transition relative to the environmental
shift. Generality of the hypothesis will be assessed by performing studies using
model microorganisms from the three domains of life - H. salinarum (archaeon),
E. coli (bacterium), and S. cerevisiae (eukaryote). These proposed activities
will demonstrate whether variable translation complexes drive environment-
dependent physiological transitions.