* 1616411
* Identification of RNA editing factors in the primitive eukaryote, Physarum
* BIO,MCB
* 09/15/2016,08/31/2020
* Jonatha Gott, Case Western Reserve University
* Standard Grant
* Karen Cone
* 08/31/2020
* USD 600,000.00

The goal of this research is to identify factors responsible for RNA editing.
RNA editing involves specific alterations of RNA molecules, relative to the
genes from which they were copied, and affects expression of genes in
mitochondria, the organelles responsible for energy generation in the cell.
Whereas, RNA editing is widespread in many organisms, including humans, the
underlying mechanisms are not well understood. This research will fill that
knowledge gap and provide new insights how RNA editing occurs. The project will
also provide training opportunities at many levels through concerted efforts to
involve high school, undergraduate, and graduate students. One such effort will
be to continue ongoing work with a high school teacher and students to develop
experiments that can be done in an inner city school setting. Specific outreach
to women and minority students will help broaden participation in science.
Students will take part directly in the research, gaining valuable experience by
learning how to optimize methods, master state-of-the-art techniques, and
present the results of their work as part of the project's dissemination of the
research outcomes to the broader scientific community. &lt;br/&gt;&lt;br/&gt;RNA
editing occurs in many organisms and has been most extensively studied in humans
and other mammals, plants, and human pathogens. Observed sequence changes
include base conversions, base substitutions, the deletion of encoded
nucleotides, and the insertion of additional residues. The only organism known
to carry out all of these forms of editing is the primitive eukaryote Physarum
polycephalum. The sequence of virtually all of the mRNAs, rRNAs, and tRNAs
encoded in Physarum mitochondria differs from that of the mitochondrial genome
in highly specific and reproducible ways. Many RNAs contain so many non-encoded
nucleotides that their genes escaped identification for years. Previous work has
shown that "extra" nucleotides are added to these RNAs during their synthesis
and that the specificity of editing depends on template elements that flank
editing sites; however, the underlying mechanism is still mysterious. The major
aim of this project is to identify the factors that are required for these site-
specific changes to mitochondrial RNAs. Previous work on two different forms of
editing has narrowed the list of potential candidates for each type and these
will be investigated further. The expression of each of these proteins will be
knocked down individually and mitochondrial RNAs will be analyzed to determine
whether RNA editing levels are reduced. Once editing factors are identified,
their interactions with other mitochondrial proteins will be examined. Long-term
goals for the project include the development of an in vitro system assembled
from defined components for use in mechanistic studies and the definition of the
specific roles played by both template elements and auxiliary factors.
Successful completion of these studies will significantly enhance current
understanding of how RNA sequence changes can be targeted in an exquisitely
specific manner.