* 1616247
* The quantitative landscape of the mitotic checkpoint: from genes to function
* BIO,MCB
* 07/15/2016,06/30/2020
* Silke Hauf, Virginia Polytechnic Institute and State University
* Continuing Grant
* Steve Clouse
* 06/30/2020
* USD 539,751.00

Regulation of the cell cycle is critical for the growth and development of all
organisms and the information needed to control this process is encoded in the
DNA of the genome. The genome not only specifies which biomolecules are being
produced, but also specifies their quantities, which is important since too
little or too much of a biomolecule can lead to cellular malfunction. Despite
this importance, it is still incompletely understood how the information
regulating levels of biomolecules is specified in the genome sequence. This
project will examine one particular regulatory pathway, the "mitotic
checkpoint", which controls the inheritance of chromosomes during cellular
division. This pathway only functions properly if the proteins that make up the
pathway are present at the optimal concentrations. By combining laboratory
experiments and computational analyses, this project will identify critical
regions on the genome that control the quantity of proteins in the mitotic
checkpoint and thereby control checkpoint function. This research will enhance
our ability to interpret and purposefully modify genome sequences, with benefits
for gene technology and synthetic biology. Furthermore, the project will provide
training opportunities for graduate and undergraduate students that prepare them
for careers in biotechnology or basic research and enhance their skills to work
at the interface of traditional disciplines. &lt;br/&gt;&lt;br/&gt;Critical
transitions in the eukaryotic cell cycle are guarded by checkpoints. The mitotic
checkpoint prevents the precocious separation of duplicated chromosomes and
thereby protects genome integrity. To function properly, the mitotic checkpoint
needs precise concentrations of the proteins that act in this signaling pathway.
Yet, hardly anything is known about how transcription and translation of the
checkpoint genes is quantitatively controlled to yield these levels. This work
will determine how checkpoint protein levels are kept within their permissive
range and why checkpoint functionality is restricted to this range.
Cas9-mediated genome engineering in fission yeast will be used to map the
genetic regions that are critical to control checkpoint protein levels. This
experimental analysis will be complemented by a bioinformatic analysis of these
genes. Checkpoint functionality assays will be used to address how different
checkpoint protein levels affect the activity and critical dynamics of the
checkpoint. These results will be combined into a predictive computational model
for checkpoint activity that will shed light on how this checkpoint operates.
Overall, this project will provide new insights into the dynamics of checkpoint
signaling and into the relation between genotype, quantitative gene expression
control and phenotype.