* 1617369
* III: Small: High-Throughput Annotation of Cellular Functions of Intrinsic Disorder in Proteins
* CSE,IIS
* 10/01/2016,09/30/2020
* Lukasz Kurgan, Virginia Commonwealth University
* Standard Grant
* Sylvia Spengler
* 09/30/2020
* USD 500,000.00

One of fundamental problems in molecular biology is to decipher functions of
millions of uncharacterized protein sequences that are rapidly generated by
high-throughput genome sequencing. The sequence-to-structure-to-function
paradigm was used for decades to determine functions of proteins. However,
recent research has broadened this paradigm by adding new players, proteins with
intrinsic disorder (ID). They are highly abundant and cannot be solved with the
structure-driven approach. While there are many widely used computational
methods that accurately predict ID in protein sequences, methods for the
prediction of the many functions of ID are lacking. This project will develop a
family of novel, accurate, and high-throughput computational methods that
predict all major functions of ID in protein sequences. It will produce putative
functional annotations on an unprecedented scale of thousands of species,
addressing the problem of high rate acquisition of raw sequence data and
contributing to the increase of the rate of scientific discovery. These results
will advance our understanding of fundamental biological processes and human
health given the high prevalence of ID in human diseases and attractiveness of
proteins with ID as drug targets. This project will also contribute to training
of STEM students and researchers via short workshops and undergraduate and
graduate level lectures and mentoring, focusing on demonstrating relevance and
value of education and research in the emerging areas of protein
bioinformatics.&lt;br/&gt;&lt;br/&gt;This project will conceptualize, design,
rigorously test and deploy predictors of all major functions of intrinsic
disorder including protein-RNA, protein-DNA, protein-protein, protein-ligand and
protein-lipid interaction, flexible linkers and spacers, regions that host post-
translational modifications, and moonlighting regions. These methods will
provide fast and accurate predictions in the absence of sequence similarity when
the commonly used sequence alignment-based approaches fail. The design will
include a novel, hybrid function-specific feature extraction, empirical feature
selection, and optimization of predictive models generated with modern machine
learning algorithms. The inputs to these predictive models will be quantified
and aggregated from a comprehensive set of empirically selected and sequence-
derived structural and biophysical characteristics of proteins. The resulting
methods will be benchmarked and made freely available to the broad research
community via a web-based, server-side portal. The results will be also
deposited into relevant public databases.