* 1623887
* EAGER: Revealing the function of the epitranscriptome in plant pathogen defense
* BIO,MCB
* 07/01/2016,06/30/2018
* Brian Gregory, University of Pennsylvania
* Standard Grant
* Karen Cone
* 06/30/2018
* USD 299,982.00

This project will explore the role of RNA modification in plant defense against
pathogens. In eukaryotes, including plants and animals, modification of RNAs by
methylation is widespread and essential for viability, but the function of such
modifications is largely unknown. Preliminary research using the model plant,
Arabidopsis thaliana, uncovered a surprising possible link between regulation of
the plant pathogen response and levels of RNA methylation. This project aims to
test the link directly by conducting a series of genome-wide analyses. The
research will produce large, complex datasets that will be analyzed with tools
that leverage high performance computing capabilities. These tools will be used
in both the laboratory and the classroom to provide students and postdocs with
interdisciplinary training in RNA biology, computational science, and
bioinformatics. The outcome of the research has the potential to transform our
understanding of the molecular basis for plant responses to pathogen attack and
could thereby contribute to future development of pathogen resistant plants.
&lt;br/&gt;&lt;br/&gt;The most prevalent internal covalent modification of
eukaryotic protein-coding messenger RNA (mRNA) is methylation of adenosine at
the N6 position [N6-methyladenosine (m6A)]. This is an essential modification,
but its function is not well understood. Transcriptome-wide studies in
Arabidopsis revealed variation in m6A levels in mRNAs from genes known to be
involved in regulating plant responses to fungal or bacterial infection. This
observation led to the hypothesis that fungal and bacterial infection may
regulate the plant's so-called epitranscriptome by reprogramming m6A
modification, which in turn alters RNA secondary structure and RNA-protein
interactions. To test this idea, RNA will be isolated from normal and
m6A-deficient plants treated with compounds that mimic fungal (chitin) or
bacterial (flg22) infection, and the RNA will be analyzed comprehensively to
measure effects on m6A levels and locations, global RNA secondary structure,
RNA-protein interactions, RNA abundance, and splicing. All of the data, web-
accessible genome analytical tools, and data management systems will be made
readily accessible to the broader research community through a project website
and through long-term genomics repositories. This research will provide
unprecedented new insights at the intersection of two traditionally disparate
fields, i.e., RNA-mediated regulation and plant-pathogen interactions, and thus
should set the stage for future developments at this interdisciplinary
interface.