* 1643825
* Genome Assembly and Analysis of the Bloom Forming Southern Ocean Salp, Salpa Thompsoni
* GEO,OPP
* 09/15/2017,08/31/2021
* Ann Bucklin, University of Connecticut
* Standard Grant
* Maria Vernet
* 08/31/2021
* USD 250,039.00

The Antarctic salp, Salpa thompsoni, is a gelatinous zooplankton that is an
important member in the Southern Ocean pelagic ecosystem. Field studies have
documented rapid population growth under favorable environmental conditions,
resulting in dense blooms of salps that substantially change the pelagic
ecosystem in regards to both structure and processes. Because this zooplankton
can proliferate rapidly and it is not readily consumed by upper trophic levels,
its periodic dominance has the potential to drastically chance ecosystem
energetics as well as change material export to the deep ocean. Completion of a
comprehensive reference genome for the Antarctic salp will enable the
identification of genes and gene networks underlying physiological responses and
allow detection of potential processes driving natural selection and the
species? adaptation strategies to the Antarctic Environment. Comparative genomic
analysis will add the dimension of time to inferences about organismal
adaptation and allow consideration of their potential to adapt to future
environmental changes, and will allow examination of novel aspects of genomic
evolution found only in the invertebrate class Tunicata. The completed salp
reference genome will provide a valuable foundational resource for other
scientists working on this species as well as the genomic basis for function and
adaptation in the Antarctic. &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;The
primary goal of this effort is to examine the rapid genome evolution
characteristic of this tunicate species and examine the genomic bases of the
species? potential for adaptation, and specifically the role of flexible gene
networks for successful responses to changing environmental conditions. The
primary hypothesis driving this research is that predicted S. thompsoni
orthologs (i.e., genes of the same function that share a common ancestor) that
show evidence of rapid evolution are indicative of positive selection, and
further that these genes and associated gene networks provide the basis for
rapid adaptation of the Antarctic salp to environmental variation associated
with a changing ocean. The proposed genome assembly strategy will allow further
refinements and scaffolding of the current, highly fragmented genome assembly
using the methods developed during previous work. Specimens of S. thompsoni now
archived at UConn will be analyzed to improve the salp genome assembly,
increasing overall scaffold length, and decreasing the number of total contigs.
High-quality reference assemblies will be obtained with two high-output paired-
end sequencing runs (Illumina) on a single individual, coupled with three runs
on the Oxford Nanopore long-read sequencer. The same sequencing strategy will be
performed on a sub-sampling of tissues from the same specimen to produce a very
high quality reference transcriptome, which will allow for high quality gene
models and near-complete gene predictions in the genome assembly. Comparisons
with available genomic data for Urochordate and Cephalochordate species will
increase the number of orthologs analyzed. Orthologous genes will be tested for
evidence of rapid selection in the salp lineage, and the results will be
compared to published expression profiles and ontology functions for the salp.
All data will be made publicly available via existing web portals; a project
website will be developed to disseminate research results for access by the both
research and educational communities. Website design will use a local instance
of jbrowse that will offer annotations, downloadable data files, and tracts of
previously-published datasets.