* 1645875
* Systems genetics of symbiotic quality in legume-rhizobium mutualism
* BIO,IOS
* 04/15/2017,03/31/2023
* Katy Heath, University of Illinois at Urbana-Champaign
* Continuing Grant
* Courtney Jahn
* 03/31/2023
* USD 717,607.00

Non-technical:&lt;br/&gt;&lt;br/&gt;Mutually-beneficial interactions between
plants and microbes are integral aspects of Earth's ecosystems. These mutualisms
are important for stabilizing pristine natural communities, restoring degraded
environments, and plant productivity in agricultural systems. Rhizobial bacteria
(rhizobia) benefit leguminous host plants (peas, beans, soybean) by fixing
atmospheric nitrogen into usable forms, essentially fertilizing their plant
hosts. As a result, legumes are high in nitrogen and provide an abundant source
of plant-derived protein. Rhizobial strains, however, are diverse in nature and
vary quite a bit in partner quality, or the level of benefits they provide to
the plant host. This project focuses on a genetic model system (the legume
Medicago truncatula and its partner bacterium Sinorhizobium meliloti) to
investigate the range of partner quality variation and its underlying molecular
basis in rhizobia, integrating various approaches including whole-genome DNA
sequencing, gene expression, and metabolomics. These techniques provide insight
into the genetic determinants and cellular-level metabolism that characterizes
highly-beneficial interactions, versus low-quality interactions that are less
beneficial for plant hosts. Understanding what determines partner quality at a
mechanistic level is critical foundational knowledge that informs efforts to
engineer the plant microbiome as part of plant improvement programs and
sustainable crop production. Moreover a basic understanding of what determines
partner quality is important for understanding how mutualisms evolve through
time and how crucial mutualisms persist in current environments. This project
provides biological and computational training at the undergraduate and graduate
level, provides internships for local community college students, and generates
online outreach and analysis
tools.&lt;br/&gt;&lt;br/&gt;Technical:&lt;br/&gt;&lt;br/&gt;Mutualists in nature
vary widely in partner quality, often with important implications for natural
and managed systems, yet the molecular basis of mutualist partner quality
variation is not well-understood. Recently studies have demonstrated mutualism
decline in response to environmental changes, resulting from dis-association of
partners, ecological shifts towards less-mutualistic species, or evolutionary
decreases in mean partner quality. A predictive understanding of mutualisms is
needed to understand their coevolution, persistence, and potential decline. This
project will integrate natural genetic variation with physiological and
molecular traits into symbiosis networks to uncover the genetic and molecular
architecture underlying the mutualism between the rhizobium Ensifer
(Sinorhizobium) meliloti and its host plant Medicago truncatula. The central
hypothesis is that naturally-occurring variation can be used to better resolve
the genetic regulation of plant and rhizobium metabolic pathways that contribute
to high-quality symbiosis. This project integrates rhizobium genome-wide
nucleotide variation with both rhizobia and host plant phenotypic and molecular
traits to address the molecular basis of rhizobium partner quality (how much
rhizobia benefit their plant hosts), a trait relevant to both natural and
managed ecosystems. Network analysis of transcriptomic and metabolomic data will
result in comparative models of high and low quality symbiosis, enabling gene-
to-trait predictions for the ecologically- and evolutionarily- important
symbiosis between leguminous plants and nitrogen-fixing rhizobium bacteria.
Hypotheses generated by network analysis will be validated in both rhizobia and
host plant systems for physiological and molecular phenotypes underlying
important partner quality traits.