* 1645795
* Bilateral NSF/BIO-BBSRC: Synthetic gene circuits to measure and mitigate translational stress during heterologous protein expression
* BIO,MCB
* 08/01/2016,07/31/2021
* Philip Farabaugh, University of Maryland Baltimore County
* Standard Grant
* Clifford Weil
* 07/31/2021
* USD 683,395.00

In this project, an interdisciplinary team of biologists and physicists will
establish novel technologies to improve the ability of a cell to make
recombinant proteins (rPs) at higher efficiency and with greater accuracy. A
majority of the biotechnology industry is dedicated to the production of rPs for
use as pharmaceuticals, vaccines and food additives. These proteins are made in
living cells using the normal process of protein synthesis, whereas the
industrial production of these same proteins requires unusually high levels of
the protein - such that the desired protein accounts for the majority of all
protein mass in the cell. At such a high production level, proteins are made
with errors, when incorrect amino acids are inserted into the growing protein
chain. This project will investigate how these errors accumulate, and will
create genetic circuits that can sense any significant increase in the frequency
of protein synthesis errors and can deploy mechanisms to restore normal highly
accurate protein synthesis. The research team consists of the lab at University
of Maryland Baltimore County, working with a model bacterium E.coli, and the lab
at the University of Aberdeen, UK, working with baker`s yeast. The two teams
will also collaborate with a major biotechnology company in the United Kingdom
to insure that the products of the project will be directly applicable to use in
industrial scale protein production.&lt;br/&gt;&lt;br/&gt;Amino acids in
proteins are encoded by three nucleotide codons in mRNA; translation involves
three nucleotide base pairs between the mRNA codon and a tRNA anticodon. Errors
occur when a decoding tRNA forms only two base pairs with the codon. For each
correct (cognate) tRNA there are about fourteen incorrect (near-cognate) tRNAs
that could induce such an error. The decoding process is normally highly
accurate, with near-cognate tRNAs introducing incorrect amino acids at
frequencies from about one in 10,000 to as low as one in 1,000,000.
Mistranslation can also result from frameshifting in which a codon overlapping
the correct codon is selected for decoding. The frequency of these
mistranslation errors increase sharply during translation of codons served by
low abundance tRNA, or during ribosomal pausing caused by depletion of charged
tRNAs. The high-level expression of biotechnological proteins can creates
exactly these conditions and thus increasing translational error frequencies.
Indeed, many reports describe biotechnological protein expression generating a
range of undesirable mistranslation events, compromising product yield and
quality, and thus the safety and efficacy of biologics. In this project we will
pursue a better understanding of the system-wide causes of translational error
through the design and application of novel reporters of mistranslation, capable
of producing either reporter enzyme activities, or regulatory transcription
factors. We combine these experimental approaches with global mathematical
modelling of translation and tRNA competition to predict when system stress will
stimulate mistranslation. We then use synthetic biology gene circuits to
transcriptionally couple the output from these new mistranslation sensors to
control rP expression, to reduce mistranslation and increase rP product quality.
Our industrial partner will test these synthetic gene circuits to maximize the
opportunities for realizing the impact of this research on biotechnology. This
research will reveal for the first time the role of rPs in triggering
translation system stress, and identify novel ways in which stress can be
ameliorated.