* 1645589
* Elucidation of translational regulatory mechanisms of plant immune responses
* BIO,IOS
* 04/15/2017,03/31/2022
* Xinnian Dong, Duke University
* Continuing Grant
* Michael Mishkind
* 03/31/2022
* USD 925,000.00

In the past two decades, significant progress has been made in understanding how
plants control the production of messenger RNAs encoding defense proteins when
challenged by pathogens, but very little is known about whether and how
translation of these messenger RNAs is regulated to produce proteins. With the
development of a sequencing-based technology for measuring the rate of messenger
RNA translation, it becomes clear that defense protein synthesis is tightly
regulated in plants. This project aims to understand how this regulation is
carried out and to identify messenger RNA features and important molecules
involved in this regulation. The success of this project will not only fill a
gap in the knowledge of the plant immune mechanisms, but also revolutionize
strategies of engineering disease resistant crops by more precisely control
defense proteins production. This new strategy will enable engineering broad-
spectrum disease resistance without compromising plant fitness, leading to
reduced use of pesticides and lightening of selective pressure for resistant
pathogens. This project will also result in training of scientists from high
school students to postdoctoral researchers in performing frontier research.
&lt;br/&gt;&lt;br/&gt;The long-term goal of this project is to elucidate the
translational regulatory mechanisms of plant immune responses. Studies of TBF1,
a key plant immune transcription factor (TF), showed that its translation is
controlled by two upstream open reading frames (uORFs). These uORFs normally
block ribosomes from reaching the major ORF. Upon immune induction, this
inhibitory effect is rapidly alleviated, leading to TBF1 translation. This
suggests that de-repressing translation of pre-existing immune TF mRNA may be a
strategy for rapid response to pathogen challenge. Global translatome profiling
using ribosome footprinting showed that during pattern-triggered immunity (PTI),
changes in translation did not correlate well with transcription. Genes with
altered translational efficiency (TE) during PTI belong to diverse functional
groups and contain novel regulators of this immune response. Moreover, an mRNA
consensus (uR-motif) was found to be significantly enriched in the 5' leader
regions of transcripts with increased TE during PTI. This project aims to
generate more translatome profiles and the first plant tRNAome datasets,
essential for studying global translational changes during plant defense
responses, and elucidate how tRNA level changes, mRNA sequence features (e.g.,
uORFs and R-motif), and trans-acting factors regulate translation. The project
will lead to insight into how plants selectively translate defense-related
proteins and identify new regulators of different immune responses. The project
will also involve training of two postdoctoral researchers and enable
continuation in outreach activities, including hosting lab tours for local high
school and community college students, and mentoring for the SROP Program for
minority undergraduate summer research.