* 1645219
* IOS EDGE: Nanoscale Probes and Infrastructure for Real-Time and Single-Cell Genomics across Metazoa
* BIO,IOS
* 07/15/2017,06/30/2021
* Weihong Tan, University of Florida
* Standard Grant
* Edda Thiels
* 06/30/2021
* USD 1,000,000.00

The genetic material, or genome, first and foremost operates at the level of
specific cells, and practically any animal tissue or embryo consists of
thousands of highly diverse cells. How and why the same genome leads to such
enormous diversity of cell types and functions are unanswered questions of
modern biology. Yet, cell-specific approaches to link cause and effect are
virtually absent for a majority of animal groups. This interdisciplinary project
addresses these bottlenecks experimentally by developing novel genomic
approaches and chemical labeling tools for genome-wide characterization of
expression, classification, and mapping of thousands of individual cells in
parallel. This information is used to (i) achieve a nearly complete census of
cell types within a given organism, focusing on animal models critical to
understanding mechanisms of learning and memory, such as Aplysia, and
regeneration, such as Pleurobrachia, and (ii) generate nanoscale probes that
selectively mark specific cells for genome editing, regardless of any advance
knowledge about the cells' molecular diversity. Several communities are
benefiting from the proposed research, including comparative neurobiology,
development, biological oceanography, and the emerging field of synthetic
biology. The project also affords cross-disciplinary training opportunities for
trainees from the undergraduate to postdoctoral level and educational outreach
activities in marine and comparative biology aimed at a diverse K-12 student
body. &lt;br/&gt;&lt;br/&gt;The grand challenge in our understanding of the
genomes-to-phenomes relationships is our general inability to manipulate genome
operation at the level of specific individual cells at any given location and at
any given time. These obstacles are more dramatic for most invertebrates, when
researchers study development or neuronal functions with little information
about the cellular composition of target organs. Here, microfluidics for massive
parallel single-cell capture and sequencing are integrated with novel cell
selection technologies, such as aptamer-based-Cell-SELEX, for quantitative gene
expression analyses and imaging of individual cells in intact tissues. Aplysia
(and, once single-cell tools are validated, Pleurobrachia and/or related
ctenophore species) are used to achieve nearly complete genome-wide
classification of the majority of cell types in their neural systems and
effector organs. The read-out(s) to measure/control gene expression in
identified neurons are: scRNA-seq data with both normalized and absolute
quantification of expression levels for target genes, and q-RT-PCR. Controls are
neurons in which target genes are not active or silenced. First, unique
resources for a diversity of cell adhesion molecules and other surface
macromolecular structures critical to design and characterize cell-specific
probes are generated. Then, using tools of chemical evolution, a high-throughput
system to manufacture cell-specific aptamer-/molecular beacon-based fluorescent
probes at a large scale is tested. Finally, hybrid nanoscale probes (e.g. made
by coupling cell-specific fluorescent markers with nucleic acid analogues) are
tested for their ability to self-deliver molecular constructs into target cells
without direct injection, electroporation, or the need to make transgenic
animals. This project is co-funded by the Chemistry of Life Processes program in
the Division of Chemistry.