* 1650040
* CAREER: Uncovering Quantitative Design Principles of RNA Regulators For Synthetic Biology
* BIO,MCB
* 08/01/2016,07/31/2021
* Julius Lucks, Northwestern University
* Continuing Grant
* David Rockcliffe
* 07/31/2021
* USD 574,088.00

Nontechnical Description: Cells have an amazing ability to process information,
make decisions, and change their state in response to changing environments.
This ability is encoded within the cellular DNA genome, which is converted into
RNA and protein molecules through the basic processes of gene expression. Among
the many functions these RNAs and proteins perform is regulating their own
expression. In fact RNAs are now known to regulate almost all aspects of gene
expression, and play central roles in controlling some of life's most basic
processes. A key question in biotechnology and synthetic biology is then: How
can RNA molecules be designed to control the expression of target genes inside
cells to facilitate applications ranging from using cells as chemical factories
all the way to using them as environmental sensors? As with many biomolecules,
RNA function is intimately related to its structure. In this work, the
investigator builds off of his development of a new small synthetic RNA
mechanism called Small Transcription Activating RNAs, or STARs. STARs are
hypothesized to allow the construction of unique gene expression control
techniques since they represent a brand new function for sRNAs. This project
seeks to use STARs as a test-bed for uncovering the principles for engineering
RNA molecules to precisely control gene expression inside cells. This work will
help realize the potential of RNA as a powerful substrate for cellular
engineering. It will also shed new light on our scientific understanding of the
fundamental sequence/structure/function relationship that underlies RNA's
central role in natural biological systems. The projected studies and research
training activities are thus expected to have a broad impact on society, ranging
from the science of cellular gene regulation and the engineering science of RNA
gene regulation that can directly connect to a broad array of biotechnological
applications. This project will also cultivate the next generation of highly
trained graduate students and teachers of synthetic biology who will be
introduced to the broad, interdisciplinary nature of biotechnology research.
Moreover, this program will actively engage the broader community to help create
an informed public that is equipped to make important decisions about the future
of synthetic biology.&lt;br/&gt;&lt;br/&gt;Technical Description: The overall
goal of this project is to build an integrated research and education program
focused on uncovering quantitative design principles that link small RNA
sequence, structure, and function, and to use these principles to design
synthetic RNAs that can precisely regulate gene expression. The education plan
focuses on integrating research into the training of the next generation of
synthetic biology students and teachers, and informing and exciting the broader
public about synthetic biology.&lt;br/&gt;&lt;br/&gt;Trans-acting bacterial
small RNAs (sRNA) exert regulation via direct RNA-RNA interactions with target
messenger RNAs (mRNAs) that cause structural changes in the target. These
changes in turn regulate many aspects of gene expression including
transcription, translation and mRNA degradation. The central hypotheses of this
project are that: 1) quantitative sRNA structure/function design principles can
be discovered and used to rationally optimize and expand the functionality of
RNA regulators, and 2) circuit-level design rules can be discovered and used to
engineer new synthetic RNA genetic modules that control the timing and pattern
of gene expression. The central goal of this CAREER project is to use Small
Transcription Activating RNAs (STARs) as a test-bed for uncovering the
quantitative design principles of RNA regulators for synthetic biology. This
will be pursued using a multi-faceted approach that includes using cutting-edge
RNA structure measurement technologies to elucidate the molecular level design
principles of STARs. Once molecular-level principles are learned, the project
will focus on using these techniques to learn the design rules for integrating
STARs into decision making regulatory networks. In addition, new experiments and
computational tools that can quantitatively model the kinetics of gene
expression mediated by STARs will be used to understand how different aspects of
STAR function propagate through RNA networks. Successful completion of these
studies will forward the broader goal of creating a quantitative discipline of
biological design that can be used to program cellular systems to solve
important problems in sustainable energy, and biomanufacturing. In addition, the
work may provide insight to mechanisms of native small RNA regulation of
naturally occurring biological systems.&lt;br/&gt;&lt;br/&gt;Educational
activities include developing a long term plan for the continuation of the Cold
Spring Harbor Synthetic Biology summer course as a training center for a broad
range of students and teachers across the globe; developing quantitative
curricular materials for training future generations of synthetic biologists;
and performing hands-on activities aimed towards families with school-age
children to excite and inform them about synthetic biology.