* 1650331
* EAGER: Site-specific engineering of DNA methylation states in plant genomes
* BIO,MCB
* 09/01/2016,08/31/2019
* Daniel Voytas, University of Georgia Research Foundation Inc
* Continuing Grant
* Karen Cone
* 08/31/2019
* USD 299,738.00

This project will engineer chemical features of plant genomes as a prelude to
understanding how those features control plant traits. In plant genomes, many
cytosine residues are chemically modified by addition of methyl groups. The
presence of DNA methylation is often correlated with changes in gene expression,
but how such changes influence specific traits is not well understood. This
research will develop and implement methods for adding or removing DNA
methylation at specific sites in the Arabidopsis thaliana genome, thereby paving
the way for analyzing the effects of DNA methylation on gene function. Research
goals will be achieved in collaboration with high school and graduate students,
who will obtain valuable experience in state-of-the-art genome engineering
technology. This kind of technology could have broad impact in both basic and
applied plant biology by providing a way to control gene expression states
without changing the DNA sequence.&lt;br/&gt;&lt;br/&gt;Epigenetic modifications
to genomes, such as methylation of cytosines in DNA, are widely known to be
associated with changes in phenotypic traits. However, there is currently no
systematic method to directly test the causality of these observations. This
project aims to develop a set of universal tools for site-specific engineering
of DNA methylation states in plant genomes. To modulate DNA methylation states
at specific regions of the genome, enzymes that control DNA methylation will be
tethered to targets of interest using the CRISPR-dCas9 system. The approach will
first be developed in the yeast, Saccharomyces cerevisiae, as it has a small,
unmethylated genome, which will permit easy evaluation of both on- and off-
target changes in methylation states. Once optimized for yeast, the methods will
be deployed in the model plant, Arabidopsis thaliana, to target site-specific
methylation and de-methylation. The tools developed through this research will
make it possible to test hypotheses relating differential methylation states and
phenotypes to enable "reverse epigenetics" approaches for predictive studies of
gene function and for gene discovery. &lt;br/&gt;&lt;br/&gt;This award was co-
funded by the Genetic Mechanisms and the Systems and Synthetic Biology Programs
in the Division of Molecular and Cellular Biosciences in the Biological Sciences
Directorate.