* 1651614
* CAREER: Conservation of cohesin-containing cis regulatory modules in the human and mouse lineages
* BIO,DBI
* 04/15/2017,03/31/2022
* Alan Boyle, Regents of the University of Michigan - Ann Arbor
* Continuing Grant
* Peter McCartney
* 03/31/2022
* USD 979,984.00

The mouse has been used as a flexible genetic system for exploring how diseases
occur, including in humans. However, the treatments that work for mice having a
condition almost never work in other animals with that condition. This
apparently occurs because, even when gene pathways seem similar across species,
their regulation, or network wiring, is different. Some of these changes may be
caused by retrotransposon elements, whose presence has been shown to be
correlated with changes in the cohesin-associated regulatory modules of human
and mouse cells. Modifying sequencing technologies with special reagents has
allowed us to start mapping regulatory regions in both human and mouse; results
show that there is a high rate of change at regulatory sites, with only about
half of the sites showing conserved sequence between human and mouse cell lines
and, even where conservation is found, the expected transcription factors don't
bind under similar conditions. A further complication is that most gene networks
are regulated by multiple transcription factors (TFs) that must bind in a
particular order and combination; often this binding is at sites quite distant
from the genes. Looping of the DNA at distant sites back around to bring it
close, and 'locking' it in place with a regulatory protein, is one of the
factors this research aims to dissect. Understanding what these regulatory
element TF combinations are, what mechanisms lead to change, and how their
regulatory actions can be modulated will have important applications in such
areas as the treatment of disease, including better targeted therapies. This
research requires strong computational skills as well as understanding the
biological states of cells and the technical details of the assays; in addition
to mentoring the graduate students who will take leading roles in carrying out
the research, high school students will be recruited to participate in
computational summer boot camps and undergraduates will be recruited to paid
summer internships for immersion in the research methods and questions, with a
strong emphasis on bringing more women and under-served minorities into active
research experiences. &lt;br/&gt;&lt;br/&gt;In this project the PI will use
recently developed methods to explore the co-binding of transcription factors
with CTCF and the cohesin complex in matched human and mouse cells. These sites
represent anchor regions for 3D interactions in the genome. Initial work
suggests that there is extensive turnover of these sites beyond the already high
turnover of cis-regulatory modules in the two genomes. This finding stands in
sharp contrast to previously published work showing, at low resolution, there is
a high level of conservation of the physical 3D structure between the two
species. The PI has previously developed computational methods demonstrating a
complex co-localization of transcription factors and shown that these patterns
are conserved in mouse. These patterns were shown to maintain similar regulatory
properties even with underlying sequence divergence. Here the PI will explore
these conserved patterns in terms of conservation at cohesin sites in the two
species. As an extension of this, the project will focus on the effect of
retrotransposons in the expansion of cohesin sites in the two lineages. Finally,
the project will integrate 3D structural information from ChIA-PET and HiC data
to determine the effects of the gains and losses of cohesin sites on the
conservation of specific loops and topological domains. This work extends from a
new way of considering conservation, that of conservation of regulatory patterns
instead of considering only sequence context. Because of this, the PI is able to
study the entire mouse and human genomes in comparison instead of only the small
fraction of the genomes comparable by sequence. As a result, the project will
provide new insights into the regulatory structure of the two genomes and shed
light on the aspects of gene regulatory control through looping and domain
structure that would previously have been impossible to consider. Progress from
this project will be available at http://boylelab.org/.