* 1656006
* RII Track-1: Microbiomes of Aquatic, Plant, and Soil Systems across Kansas
* O/D,OIA
* 09/01/2017,08/31/2023
* Kristin Bowman-James, University of Kansas Center for Research Inc
* Cooperative Agreement
* Andrea Johnson
* 08/31/2023
* USD 20,000,000.00

Non-technical Description&lt;br/&gt;Microbiome science, the study of the
combined genetic material from the set of all microorganism taxa present in a
particular environment, is increasingly seen as important for addressing
ecological questions, including societal challenges such as sustaining
agricultural production while maintaining soil and water quality. This project
will undertake a large survey of plant, soil and aquatic microbiomes and their
environmental characteristics to catalog these communities and assess the
ability of microbiomes to influence crop production, soil condition and water
quality. Kansas is an ideal natural laboratory for this project due to the large
gradients in precipitation and land use across the state. The project team will
integrate its research and educational activities to improve Science,
Technology, Engineering, and Mathematics (STEM) education capacity in both urban
and rural areas, among mainstream, economically disadvantaged, and first-
generation college students, enhance the participation of Native Americans and
other under-represented groups, and to expand the workforce in microbial, plant,
and soil science, genomics, bioinformatics and
ecology.&lt;br/&gt;&lt;br/&gt;Technical Description &lt;br/&gt;This project is
synergistic with the 2016 White House Initiative to improve understanding of
microbiomes. Major project goals are to: develop a mechanistic understanding of
microbiome-mediated ecosystem functions, predict ecosystem responses to changes
in precipitation and land-use patterns, and identify ways to select for and
utilize microbiomes to produce desired characteristics such as increased
agricultural productivity or drought tolerance, efficient nutrient utilization,
and enhanced soil quality. Advanced molecular techniques, including massively
parallel sequencing of DNA, amplicon sequencing, and metatranscriptomic
analyses, will be utilized in conjunction with large-scale field and greenhouse
experiments. Databases will be created to link metagenomic data with
environmental parameters across the state and identify genotypes that may confer
desired responses under varying conditions. Data will be analyzed using standard
statistical techniques and advanced methods such as structural equation
modeling. Results will be used to identify practices to maximize plant
productivity, water quality, and soil health.