* 1657380
* CRII: III: RUI: Computational Approaches for Inferring the Evolutionary Histories of Cancer Genomes
* CSE,IIS
* 03/01/2017,02/28/2022
* Layla Oesper, Carleton College
* Standard Grant
* Sylvia Spengler
* 02/28/2022
* USD 142,783.00

Cancer results from the accumulation of genomic alterations that occur during
the individual's lifetime and cause the uncontrolled growth of a collection of
cells into a tumor. These mutations occur as part of an evolutionary process
that may have begun decades before a patient?s diagnosis. Better understanding
of the history of a tumor's evolution over time may yield important insight into
how and why tumors develop, as well as which mutations drive their growth.
Recent advances in DNA sequencing technologies have revolutionized how all
aspects of the human genome are measured and have the potential to shed light on
the molecular underpinnings of cancer and many other human diseases. However,
realizing the full potential of these technological advances will require novel
algorithmic methods specifically designed to analyze this data. For example, DNA
sequencing data only captures information about the tumor at the time of
sequencing, rather than how it evolved to its current state. While many
algorithms have been developed in recent years to infer information about tumor
evolution from DNA sequence data, this area of computational development is
relatively young and many unsolved challenges remain. This project will focus on
the development of computational approaches that enable improved inference of
the evolutionary histories of cancer genomes, while simultaneously expanding
undergraduate research participation in the field of computational biology.
&lt;br/&gt;&lt;br/&gt;The evolutionary history of a tumor can be described as a
rooted tree whose vertices represent different tumor populations that existed
during the history of the tumor. While many computational methods aim to infer
this history from sequencing data, there is much room for improvement. For
instance, the incorporation of multiple data signals (e.g., single nucleotide
variants and copy number aberrations) has proven difficult and different methods
may produce different results when run on the same dataset. This project will
develop computational approaches that directly address this and other
limitations. Rather than proposing the advent of another method to directly
infer the evolutionary history of a tumor, this project will develop consensus
methods that, given a collection of potential tumor history trees, infer a
single consensus tree. Furthermore, this project will investigate both
theoretical and practical limitations to inferring the evolutionary history of
tumors. This will include analysis of when large-scale events such as whole-
genome duplications are theoretically detectable as well as simulation studies
to investigate how practical considerations, such as: (i) sequencing coverage,
(ii) number and distribution of sequenced samples, and (iii) noise in the
sequenced data, limit or alter the ability to infer the evolutionary history of
a tumor. This project will be completed with the help of undergraduate student
researchers from a wide array of backgrounds, thus broadening student
participation in computational biology and computer science. Additionally, the
PI will coordinate a cross-institutional undergraduate workshop on computational
biology that will provide a venue for both students and faculty at baccalaureate
institutions to interact and collaborate.