* 1661380
* Collaborative Research: ABI Innovation: A Graph Based Approach for the Genome Wide Prediction of Conditionally Essential Genes
* BIO,DBI
* 09/15/2017,08/31/2022
* Haiyuan Yu, Cornell University
* Standard Grant
* Reed Beaman
* 08/31/2022
* USD 639,600.00

How does one identify, and characterize at the genome scale, the set of genes
that is essential for an organism to grow and thrive under particular
conditions? Predicting such sets of genes is a fundamental goal in
bioinformatics; this project aims to create methods and tools for making
accurate lists of such functional genes. The approach combines phenotype
prediction with knowledge about the functional biological networks in cells to
infer new knowledge. The network analysis methods developed here can be easily
transferred and applied to a large variety of datasets to answer a wide range of
questions from inferring gene-phenotype associations to detecting communities on
social networks, extensions highly relevant to the network science community.
Moreover, the project's state-of-the-art analysis of temporal gene expression
data using state-space models and dimensionality reduction techniques is
universally applicable to any groups of genes - e.g. tissue specific vs
universally expressed genes. In addition to advancing functional genomics
knowledge in the study organism, yeast, the tools will have an impact on
research in fields like personal genomics research, by providing a large-scale
system-level identification and molecular characterization of phenotypes.
Finally, this project provides new and innovative tools for education in
bioinformatics. &lt;br/&gt;&lt;br/&gt;In more technical terms, this project's
major goal is to develop new mathematical models and methods that, given a set
of genes or an entire genome, can infer their phenotypes and suggest whether or
not these genes are necessary for the organism survival. Specifically,
information will be integrated on two levels: phenotypic and molecular. At the
phenotypic level the structure of biological networks will be used to assign
phenotypic attributes to genes and identify sets of genes that share similar
essential phenotypes. At the molecular level, the resulted phenotype predictions
will be refined by identifying groups of essential genes governed by similar
activity patterns. The integration of the information on these two levels will
result in a comprehensive gene-phenotype characterization and a refined group of
conditionally essential genes. The resulting predictions will be validated
experimentally in two yeast systems. All the tools and datasets associated with
this project will be made freely available through genopheno.gersteinlab.org.