* 1661391
* Collaborative Project: ABI Innovation: Computational Identification &amp; Screening for Deleterious Mutants
* BIO,DBI
* 06/01/2017,05/31/2021
* Robert Jernigan, Iowa State University
* Standard Grant
* Peter McCartney
* 05/31/2021
* USD 883,059.00

When interpreting the effect of changes in genome sequences on their protein
products, what are the criteria for separating the beneficial from the neutral
and destructive mutations? It is not practical to experimentally test all
possible changes; it is possible to develop computational rules and methods to
accurately predict the effect of most changes, reserving experimental testing
for borderline cases. The rules that constrain functional protein sequences, the
structures into which they fold, the stability of the structures and the
transitions that occur can be organized into an informatics framework that helps
distinguish functional from non-functional mutations. The stability and dynamic
transitions of protein structures over many mutations are the focus of this
research, for which new evaluation methods will be developed and tested. Once
the assessment methodologies are validated they will be integrated, along with
the raw data and assessment results, within a single, efficient computational
platform. The ability to interpret new data and compare it to existing results
will not only allow discrimination of beneficial, neutral and deleterious
mutations, but provides a resource for a protein-sequence-structure-based
understanding of evolution. Informed selection rules will enable better
decisions about the importance of saving individual members of endangered
species, as well as how different environments affect selection in individual
species. &lt;br/&gt;&lt;br/&gt;There is a large body of data in protein
sequences and structures that can aid in understanding evolution, which is not
currently being used. This project will utilize this data in systematic ways to
shed new light on evolution. Protein structures are often modeled in different
computational ways; the evaluation of these structure models is critical for
expanding the set of reliable structures, since there are huge numbers of
possible mutant sequences. Changes to the protein sequences and structures are
not fully understood. Part of the difficulty lies in understanding the
interactions within their densely packed structures, which have significant
interdependences. Developing new ways to evaluate the effects of dense packing
on protein structures is one of the project?s aims. Focusing on the physically
interacting clusters within protein structures together with their sequence
variants provides rich information about the correlations among amino acid
substitutions. This rich data will then significantly advance the ability to
distinguish between the important and the unimportant mutations. Progress has
been seen with these approaches in the evaluations of predicted structure models
at the CASP competitions for protein structure prediction. These new approaches
lend themselves directly to the evaluation of the stabilities of different
protein mutants. The evaluation procedures are derived directly from the
available sets of protein structures and carefully tested against other known
structures. In one important innovation these now include entropies that account
for known changes in the structures of individual proteins, i.e. their dynamics.
Capturing these tendencies for changes significantly improves the evaluation of
the stabilities of proteins. Applications to sets of mutants show that
unfavorable mutants are either more stable or less stable than the normal cases.
These changes in stability directly affect the ways in which the proteins can
move to carry out their functions; evaluating these changes significantly aids
the understanding of protein mutations. This project will yield a uniform way to
reliably assess the effects of protein mutations. This ability will
significantly aid in the understanding of many aspects of evolution that remain.