* 1661338
* ABI Innovation: Tools for reconstructing mobile genetic elements from shotgun metagenomic data
* BIO,DBI
* 07/01/2017,06/30/2021
* Ilana Brito, Cornell University
* Continuing Grant
* Peter McCartney
* 06/30/2021
* USD 1,029,606.00

Bacteria may acquire novel DNA from other bacteria in their environment through
the process of horizontal gene transfer (HGT). This transferred DNA often
encodes new functions that expand an organism's niche, change its relationships
with its host or provide a competitive edge against other organisms within its
environment. For example, antibiotic resistance genes are frequently observed to
be transferred between organisms. New high-throughput DNA sequencing
technologies now allow researchers to simultaneously sequence data from millions
of different bacterial species in a single sample, called metagenome sequencing.
Thousands of publicly available metagenomic sequence datasets exist for samples
from very diverse ecologies, including: gut microbiomes from humans, animals and
insects, soil and root communities, surfaces in built environments, hydrothermal
mats and vents, wastewater treatment plants, freshwater sources and marine
ecosystems. Using these datasets, the research team will develop a suite of
computational tools for examining the mobile gene content of these datasets, to
identify those genes that may shape the response of microbial communities to
stress. Graduate students will carry out the research under the direction of the
PI, while undergraduates will be trained to use the resulting methods through a
series of hackathons aimed at building the computational science community
focused on microbial community research. Recruiting efforts will be aimed at
measurably increasing the diversity in computational biology throughout the
greater Finger Lakes region.

Functional characterization of microbial communities is often performed using
metagenomic shotgun data, obtained by sequencing microbial samples in their
entirety. Yet, despite its importance, few methods exist to distinguish genes
that are inherited versus those genes which have been mobilized and transferred
between organisms in this type of short-read sequence data. By providing new
computational methods for reconstructing mobile genetic elements in short-read
data, researchers can tap into the huge resource of existing metagenomic data to
examine the role of HGT in shaping the functions of natural microbial
communities. This research will be transformative for all fields of
microbiology, including clinical microbiology, microbiome research,
environmental microbiology, and microbial engineering.