* 1661331
* ABI INNOVATION: Molecular Network Inference via Nonparametric Functional Dependencies
* BIO,DBI
* 07/01/2017,06/30/2023
* Mingzhou Song, New Mexico State University
* Standard Grant
* Reed Beaman
* 06/30/2023
* USD 814,942.00

Despite sharing a common genome, cells and tissues making up the same organism
can have distinct biological function. One explanation is that the molecular
networks in which genes interact are wired differently across cell types within
the organism. The purpose of this project is to identify the functional
molecular networks in major cell and tissue types in two mammals, humans and
mice. Using gene expression, or transcriptome, data with newly developed
statistical inference methodologies, this research will uncover the connections
currently obscured by the complex gene interaction patterns that result from the
concurrent activity of many networks. There are rich 'omics data resources
profiling these dynamic molecular rewiring processes, including the FANTOM5
promoter-level gene expression atlases of mammalian primary cells and tissues,
and tissue-specific human and mouse epigenome profiles from the ENCODE project.
Using these public data sets and innovative computational methods, the project
is expected to decipher epigenetically rewired molecular networks across a
number of types of tissues during differentiation. The resulting high-resolution
networks can then be compared between the organisms, showing how functional
networks have evolved between human and mouse. Therefore, the project is
expected to be transformative in the understanding of complex biological
mechanisms, implementing a strategy demonstrating the causality-by-functionality
principle. &lt;br/&gt;&lt;br/&gt;Network inference is becoming a bottleneck in
modern biology as high-throughput data backlogs have surpassed the capacity of
human interpretation. To address this challenge, the goal of this project is to
design a nonparametric functional dependency framework for network inference
using the causality-by-functionality principle, apply it in characterizing
mammalian molecular networks, and evaluate the performance in a subnetwork
underlying cerebellar development. With the latest excellent mammalian
transcriptome and epigenome resources, the project will reveal combinatorial
functional effects to understand causal molecular interactions, detect tissue-
specific network rewiring in mammalian cell differentiation, characterize
pathway evolution between human and mouse at the promoter level, and validate a
three-layer inferred gene subnetwork for cerebellar development. The project
outcomes are expected to include molecular networks involving transcription
factors, chromatin remodelers, and noncoding RNAs; second-order network rewiring
across major cell and tissue types in mammalian cell differentiation; and
pathway evolution between human and mouse in response to an ongoing debate on
transcriptome divergence between the two species. The inferred cerebellar
subnetwork after validation will contribute to new biology in brain development
research. The network inference methodologies are expected to substantially
expand the landscape of discoverable complex nonlinear interactions in dynamic
networks, and the resulting software will be an essential informatics tool for
discoveries in biology. Educational activities planned at New Mexico State
University will enhance biological informatics workforce development and prepare
students for knowledge-based career opportunities. Project information is
available at https://www.cs.nmsu.edu/~joemsong/funchisq .