* 1661543
* ABI Development:   Gene function visualization and inference across all plants
* BIO,DBI
* 01/01/2018,12/31/2020
* Eva Huala, Phoenix Bioinformatics Corporation
* Continuing Grant
* Peter McCartney
* 12/31/2020
* USD 1,261,211.00

Addressing the food needs of an increasing global population in the 21st century
will require significant improvements in agriculture akin to the Green
Revolution of the previous century. To address this goal it is of critical
importance to understand as much as possible about the functions of the tens of
thousands of genes present in each plant species, in order to select those
functions of most relevance to engineering favorable traits such as drought
tolerance or increased grain quality. Much of what is currently known about
plant gene function comes from experiments in model plant systems such as
Arabidopsis thaliana, but because of evolutionary relatedness this information
can be used to predict the functions of genes in crop species such as maize and
rice, and to provide targeted hypotheses that can be followed up by further
experiments. This project will increase both the efficiency and accuracy of gene
function prediction in plants by organizing and integrating high quality data
from a number of sources and presenting it in a user-friendly format that makes
clear the evolutionary relationships between genes. The team will develop
software for visualizing the evolutionary relationships as a gene family tree,
providing an organizing framework for disparate experimental observations from a
number of experimental systems. The project will provide software for use, and
for data sharing, by a broad community of both researchers and students. The
project?s inclusion of local community college students will provide training
and increase participation in computer science, genomics and plant biology by
groups that are underrepresented in these fields.&lt;br/&gt;&lt;br/&gt;The
project output will be a powerful and intuitive software tool for inferring
plant gene function from phylogenetic relationships and experimental evidence.
To achieve this goal the project aims are to: 1) assemble sets of phylogenetic
trees using selected model organism and plant genomes, along with a suite of web
services to ensure easy access to the data; 2) follow user-driven design
principles to design and build the user interface, in order to visualize the
phylogenetic trees and associate the genes within each tree with high quality
functional information including Gene Ontology annotations, gene expression
patterns, protein domains, sequence variations and mutant phenotypes; 3) provide
an analysis platform that streamlines workflows and includes personal workspaces
for users to add their own data and make it public when their work is completed,
thus enriching the knowledgebase; and 4) produce workshops, course materials and
online content to train scientists at all academic levels on how to use the
software. The results of the project will be available at
http://www.phoenixbioinformatics.org/treevision/.&lt;br/&gt;