* 0900671
* Collaborative Research: RNA-guided DNA recombination through assembly graphs
* MPS,DMS
* 09/01/2009,08/31/2014
* Natasa Jonoska, University of South Florida
* Continuing Grant
* Mary Ann Horn
* 08/31/2014
* USD 558,143.00

Genome rearrangements occur on an evolutionary scale, as well as on a
developmental scale, in a wide range of eukaryotes. Even in cancer cells,
dramatic DNA rearrangements are frequently observed in somatic cell lineages.
This project uses certain species of ciliates as a model system to study gene
rearrangements. These organisms undergo massive recombination during
differentiation of an archival germline micronucleus into a somatic macronucleus
capable of gene expression. The objective of this project is to increase the
understanding of this process of assembly, starting from the basic principles of
molecular DNA - RNA - enzyme interactions, through general knowledge of the
possible pathways and intermediate steps that lead to the final product. Through
mutual reinforcement, the project interlaces biological experiments with
mathematical findings. The theoretical model utilizes spatial graphs with
4-valent vertices as a physical representation of the DNA at the time of
recombination, and smoothing of the vertices models the actual recombination.
Specific aims include: (a) examine the interactions of the template RNA and the
involvement of possible cutting and splicing enzymes at the moment of
recombination, (b) identify steps in the process by pinpointing putative
intermediate molecules during unscrambling a variety of genes, and theoretically
analyze sets of pointers leading to these intermediates (c) characterize
theoretically and confirm experimentally all possible gene unscrambling pathways
for a subset of genes, and (d) develop mathematical techniques based on methods
from knot theory and formal languages to study the theoretical
model.&lt;br/&gt;&lt;br/&gt;Our understanding of genome-wide DNA rearrangements
is still very limited. These processes appear on an evolutionary scale, where
they can lead to species-specific differences and even the creation of species
boundaries. On a developmental scale, genome rearrangements appear in a wide
range of differentiating eukaryotic cells. The findings of this project will
offer insight into the process of gene descrambling in ciliates and may also
extend more generally to other model systems. This project seeks to learn from
both theory and experiments how a mechanism for RNA-templated DNA recombination
sculpts the assembly of genetic information in ciliates. This mechanism of
template-mediated DNA rearrangement can be visualized as a scaffold for
controlling and programming certain types of genetic information within a cell.
Such control of information processing could facilitate biomolecular computing
in vivo, as well as methods for epigenetic control of development.&lt;br/&gt;