* 0916102
* III: Small: Computational Infrastructure for the Identification of Copy Number Variations from SNP Microarrays
* CSE,IIS
* 09/01/2009,08/31/2013
* Thomas LaFramboise, Case Western Reserve University
* Standard Grant
* Sylvia Spengler
* 08/31/2013
* USD 497,603.00

It was recently discovered that copy number variations (CNVs) in human genome
are quite common, and have important implications on phenotype. Currently, the
primary platforms for large-scale detection and characterization of CNVs are SNP
(single nucleotide polymorphism) microarrays. The current state-of-the-art in
computational identification of CNVs from microarray data relies mostly on
model-based approaches (e.g., Hidden Markov Models). However, such methods
require extensive training data, which may not be always available. Furthermore,
since these methods use common CNVs to train their models, they are not as
successful in identifying rare CNVs, which are believed to make up a substantial
proportion of all CNVs in the human population. The objective of this project is
to develop optimization based algorithms and software for the identification and
genotyping of CNVs, with a view to enabling fast and accurate identification of
different types of CNVs (rare and common), without the requirement of training
data.

The proposed framework develops a novel computational approach by explicitly
formulating CNV identification as a series of optimization problems that
incorporate multiple factors, including sensitivity to noise, rarity/commonality
of CNVs, genotypic specificity, and parsimony. This formulation enables
development of efficient algorithms that treat identification of rare and common
CNVs as different problems with different objective functions. Availability of
the resulting software to the community will enable more efficient and accurate
identification of CNVs in large samples, facilitating advances in understanding
the role of CNVs in a range of complex phenotypes, including HIV, autism,
schizophrenia, mental retardation, and many others. Furthermore, the
computational innovations introduced by this project are likely to find
applications in next generation sequencing.