* 0919658
* Codon Choice and Gene Expression in Saccharomyces Cerevisiae
* BIO,MCB
* 08/01/2009,07/31/2013
* Elizabeth Grayhack, University of Rochester
* Continuing Grant
* Arcady Mushegian
* 07/31/2013
* USD 542,617.00

Intellectual Merit: Translation of mRNA into protein is a fundamental process
requiring massive energy expenditure and a large fraction of the cell's
components. Translation in all kingdoms is regulated both by signal transduction
pathways, and by codon-mediated signals. Translation of different genes occurs
with stunning differences in efficiency in that steady state expression levels
of proteins vary by as much as six orders of magnitude relative to their mRNA
abundance. Moreover, the genetic code itself plays a major role in modulating
the efficiency of translation: The 61 codons that specify insertion of the
twenty amino acids into proteins include many synonymous codons that specify
insertion of the same amino acid, and these synonymous codons are used with
vastly different efficiencies. There is a wealth of evidence that the particular
choice of codons used to encode a protein influences its translation efficiency,
as well as the accuracy of amino acid insertion, and protein folding. Although
the identities of a set of 25 optimal codons that result in rapid and accurate
translation in yeast have been known for over a quarter of a century, there has
been no systematic description of how codons affect expression. Crucial
questions that remain unresolved include the identity and properties of codons
or codon combinations that cause reduced expression in eukaryotes, whether it is
the arrangement and/or location of these codons within a gene that results in
reduced expression, and how or why usage of these codons might affect
expression. The goal of this project is to systematically determine how codon
usage impacts gene expression in a eukaryote, the yeast Saccharomyces
cerevisiae. In particular, codons and codon combinations that impair expression
will be identified, the parameters of codon density, arrangement, and location
that are important to impair expression will be defined, and the mechanisms by
which codons and codon combinations modulate gene expression will be examined.
The results obtained from this research are likely to yield crucial new insights
into the perplexing problem of how translation efficiency is modulated to such a
large extent by codon choice. This is an essential first step to begin to
understand the importance of this regulation for the survival and biology of
organisms. &lt;br/&gt;&lt;br/&gt;Broader Impacts: Thorough understanding of the
codon-based factors that contribute to expression of proteins in yeast will
provide a framework to understand the fundamental processes by which the genetic
code modulates gene expression and may uncover physiologically important
regulatory processes that depend on the particular choice of codons used in
encoding some genes. These studies also have a practical benefit of guiding
pharmaceutical, biochemical and structural biologists in their efforts to design
synthetic genes to produce high levels of protein. Moreover, the project offers
ideal opportunities to expose both undergraduate and high school students to
scientific research, since the proposed work uses both robust methodology
(involving yeast genetics and luciferase assays) and a non-hazardous organism
(baker's yeast), both of which are optimal for hands-on research experience for
undergraduates and high school students. Undergraduate students will be
encouraged to join the laboratory, and high school students will be involved in
collaboration with a colleague with a Masters of Education.