* 0921038
* Collaborative Research and RUI:  Stochastic Dynamic Network Models of Gene Regulation under Environmental Stress
* MPS,DMS
* 09/01/2009,08/31/2014
* Kam Dahlquist, Loyola Marymount University
* Standard Grant
* Mary Ann Horn
* 08/31/2014
* USD 246,123.00

This project is a collaboration between a biologist and a mathematician and
their undergraduate students in stochastic dynamic modeling of gene regulatory
networks under environmental stress. All organisms must respond to changes and
stresses in their environment to survive and reproduce. Such environmental
stresses include changes in nutrient or oxygen availability, changes in
osmolarity or pH, the presence of reactive oxygen species or other damaging
agents, and sudden or large changes in temperature. Organisms respond to
environmental stresses through characteristic programs of gene expression. Among
the most interesting and challenging problems in understanding this
environmental stress response is the dynamic behavior of gene regulatory
networks within the cell. The careful regulation of these networks is a
fundamental activity of the organism. The objectives of this project are (1) to
identify the network of transcription factors that regulate the response to cold
shock in budding yeast, Saccharomyces cerevisiae, through a combination of
mining of publicly available data, the genetic screening of systematic yeast
deletion strains and the analysis of in-house microarray data, and a Bayesian
approach to network reconstruction based on our model; (2) to analyze the model,
comparing it to deterministic chemical kinetic and dynamic Bayesian network
models in development and use in the research community; (3) to develop models
of the additional exogenous perturbations of multiple temperature shifts and the
resultant affect on growth rate for integration into the stochastic dynamic
network model; (4) to test the model predictions experimentally using qRT-PCR
and DNA microarrays on both total RNA and transcriptionally active mRNA, in both
wild type and gene deletion strains, improving the model through successive
rounds of simulation and experiment; and (5) to develop and analyze a general
mathematical modeling framework suitable for studying a wide variety of gene
regulatory networks.

The invention of high-throughput genomics methods has transformed 21st century
biology from a ?one gene at a time? approach to the analysis of whole systems.
Baker?s yeast, Saccharomyces cerevisiae, is an ideal model organism to study
because it grows quickly and the expression of all 6000 genes can be measured in
a single experiment. While this ability to measure the expression of all the
genes at once is a significant first step towards understanding fundamental
cellular processes and the defects that lead to disease, it is still only a
?parts list?. Just as listing the numbers and kinds of boards, nails, bricks,
and mortar it takes to build a house does not explain how the house is put
together, simply measuring all of the genes does not explain how cells function
and respond to environmental stresses. Instead, to understand cell function, we
need to understand the rules that govern which genes are expressed under what
circumstances and how the genes interact with each other. In short, we need to
understand how this complex gene regulatory network changes over time. In this
project, we will build a mathematical model that can be used to make testable
predictions about cell function. Our approach offers the potential to synthesize
a number of seemingly disparate techniques of gene regulatory analysis currently
being used. The research program will also yield biological insight into the
overall regulatory mechanism of the response to cold shock in yeast, which is
poorly understood. Our work will determine the particular regulatory factors
involved, the extent of environmental stress response pathway overlap, and will
provide a measure of the direct and indirect effects of individual factors. The
mathematical techniques we develop should provide a fruitful framework for the
integration of dynamic modeling and statistical analysis of gene expression data
and should be broadly applicable to the biology of complex organisms. In
particular, a model that explicitly deals with the indirect effects of genes in
a regulatory network should provide insight into the causes of complex diseases
such as cancer where multiple genes and environmental effects are involved.

