* 0922545
* The Origin and Evolution of Y Chromosomes in Caricaceae
* BIO,IOS
* 09/15/2009,08/31/2014
* Ray Ming, University of Illinois at Urbana-Champaign
* Continuing Grant
* scott jackson
* 08/31/2014
* USD 3,141,352.00

&lt;br/&gt;PI: Ray R. Ming (University of Illinois at Urbana-
Champaign)&lt;br/&gt;&lt;br/&gt;CoPIs: Paul H. Moore (Hawaii Agriculture
Research Center), Richard C. Moore (Miami University) and Qingyi Yu (Texas
A&amp;M University)&lt;br/&gt;&lt;br/&gt;Sex expression in papaya (Carica papaya
L.; Family Caricaceae) is controlled by loci found in the male specific region
of the Y chromosome (MSY). Unlike the sex chromosomes of other plants or
animals, the MSY of papaya is quite small and is restricted to a region about 8
Mb in size. In addition, sex reversal mutants in papaya indicate that the two
sex determination genes are confined to a smaller region of about 1 Mb that is
currently being sequenced. Thus the small size of the sex determination region
in papaya offers the opportunity to dissect the early events in sex chromosome
evolution and the identification of sex determination genes feasible. The long-
term goal of this project is to understand the evolutionary mechanisms governing
the formation and divergence of sex chromosomes. To accomplish this objective,
300 bacterial artificial chromosomes (BACs) from the sex specific regions of six
species in three genera of Caricaceae will be sequenced and subjected to
detailed functional and comparative genome analyses. The specific objectives are
to: 1) identify the sex determination genes in C. papaya. The focus will be on
candidate genes in the 1 Mb deleted region of the male to female sex reversal
mutants to identify the two sex determination genes; 2) elucidate the origin of
sex chromosomes in Caricaceae. A comparative genomic approach using chromosome
sequences of six species from three genera with diverse breeding systems to
determine whether the sex chromosomes of papaya share a similar evolutionary
history with the homologous regions in these sister species; 3) determine the
forces that have shaped genome evolution of papaya sex chromosomes. Competing
evolutionary hypotheses concerning the deterioration of the MSY will be
distinguished by conducting population genomics analyses across the entire
region and the corresponding region on the X chromosome; and 4) engineer true
breeding hermaphrodite papaya varieties to validate the function of the sex
determination genes. Candidate gene(s) promoting stamen development will be
transformed into female plants of leading papaya cultivars to produce
hermaphrodite varieties without the Yh chromosome. &lt;br/&gt;&lt;br/&gt;This
project will significantly advance genomic tools and knowledge for tropical
trees, an under-explored node of the angiosperms that is of large and growing
importance to US agriculture. The knowledge gained may enable the engineering of
true breeding hermaphrodite papaya varieties and improve papaya fruit
productivity while lowering the cost. With respect to training and outreach, the
four institutions participating on this project, University of Illinois at
Urbana-Champaign (UIUC), Hawaii Agriculture Research Center (HARC), Texas
A&amp;M University (TAMU) and Miami University (MU) each has strong teaching,
training, and outreach programs with a successful history of engaging high
school, undergraduate, and graduate students, including at all levels and
minority groups that are under-represented in the sciences. College and high
school students will be trained through summer intern programs at UIUC, HARC,
TAMU, and MU. Graduate students and postdoctoral scientists will be exposed to
new genomic tools and techniques. Knowledge of plant genomics will be
communicated to the general public as well through participation in outreach
programs such as the Annual Hawaii State Farm Fair.