* 0924023
* Discovery, Revision and Validation of Maize Genes by Proteogenomics
* BIO,IOS
* 09/15/2009,08/31/2013
* Laurie Smith, University of California-San Diego
* Continuing Grant
* Diane Jofuku Okamuro
* 08/31/2013
* USD 3,801,162.00

&lt;br/&gt;PI: Steven P. Briggs (University of California - San
Diego)&lt;br/&gt;&lt;br/&gt;CoPI: Vineet Bafna and Laurie G. Smith (University
of California - San Diego)&lt;br/&gt;&lt;br/&gt;Collaborators: W. Joan Chen (San
Diego State University), Laura J. Olsen (University of Michigan - Ann Arbor),
Steven Rodermel and Patrick Schnable (Iowa State University), and Frank
Hochholdinger (Universität Tübingen, Tübingen, Germany)&lt;br/&gt;&lt;br/&gt;The
most fundamental goal of genome science is to discover all of the protein-coding
genes, and then to discern the abundance, location, and exact chemical
composition of every protein made during the life cycle of an organism; this is
called the proteome. A complete and accurately annotated proteome provides the
foundation for studies of systems biology and molecular evolution, as well as
for hypothesis-driven research. Recent progress in proteogenomics (using
proteomic information to annotate the genome) has established it as a data-
driven method that complements nucleotide (DNA and RNA)-based annotation
strategies. Genome-wide, quantitative proteomics also makes possible the
creation of a protein atlas that reveals the anatomical distribution of the
proteome and protein sub-cellular locations. This project has two research aims.
Aim 1 is to create an Atlas of Maize Proteins. The atlas includes the identity
and relative amount of 40,000-50,000 proteins in each of 37 different tissues
and stages of maize development. The atlas also includes the protein composition
of the plasma membrane, chloroplast, mitochondrion, and peroxisome along with
information about the protein changes caused by abiotic and biotic stress. Aim 2
provides proteogenomic discovery, revision, and confirmation of 40,000-50,000
maize gene models, including the identification of exons, the definition of
translation start sites and exon borders, and the determination of the correct
exon reading frames. This project enhances genome-enabled maize research and
breeding by increasing the completeness and accuracy of maize genome annotation.
Furthermore, investigations of maize physiology, development, cell functions,
and breeding benefit from knowledge of the anatomical and sub-cellular
distribution of maize proteins provided by the Atlas of Maize Proteins.
Interdisciplinary educational and outreach opportunities are provided to post-
docs, graduate students, undergraduates, high school students and Cal State
researchers, with an emphasis on involvement of under-represented
minorities.&lt;br/&gt;&lt;br/&gt;This project will provide interdisciplinary
educational and outreach opportunities for post-docs, graduate students,
undergraduates, high school students and San Diego State University researchers,
with an emphasis on involvement of under-represented minorities. All project
participants in San Diego including post-docs, graduate students and
undergraduates are receiving unique, interdisciplinary training made possible by
the collaboration this project involves between investigators with expertise in
mass spectrometry, bioinformatics, maize developmental and cell biology, and
plant responses to stress. High school students are participating in the
research via a module developed for BioBridge, a UC San Diego outreach program
that brings hands-on learning activities into San Diego public schools.
Researchers at San Diego State University will receive training and education in
proteomics and bioinformatics through workshops. Access to the biological
materials used in the project is provided by the Germplasm Resources Information
Network (GRIN, http://www.ars-grin.gov/). Access to the project results,
including data and software, is provided by websites and publications by the
investigators (http://briggs.ucsd.edu/; http://www-cse.ucsd.edu/~vbafna/). The
long-term repository for project data is Tranche
(https://proteomecommons.org/index.jsp) and Gramene (http://www.gramene.org).