* 0925802
* Doctoral Dissertation Improvement: Patterns of Genetic Diversity and Signatures of Natural Selection at the Intercellular Adhesion Molecule-1 (ICAM-1) and CD36 Loci
* NONE,NONE
* 09/01/2009,08/31/2010
* Sarah Tishkoff, University of Pennsylvania
* Standard Grant
* Carolyn Ehardt
* 08/31/2010
* USD 15,000.00

Understanding the scope of human genetic variation and how evolutionary forces
like natural selection, genetic drift, and mutation have shaped the human genome
is key to describing the genetic history that underlies all human populations.
Identifying patterns of nucleotide variation that have resulted from natural
selection may also help identify genetic variants that influence disease
susceptibility and will further clarify the context of modern human evolution.

Malaria, resulting from infection by Plasmodium falciparum, causes millions of
deaths each year, and is one of the strongest selective forces in recent human
evolution. Because genetic variants at malaria resistance candidate genes may be
targets of natural selection, nucleotide diversity at these loci can be examined
to identify variants that affect susceptibility to malarial disease. African
populations are particularly informative for studying patterns of genetic
variation at malaria susceptibility candidate genes because they contain a high
level of genetic diversity and are likely to have previously unknown genetic
variants. In addition, due to high malarial endemicity in certain regions of
Africa, it is likely that evidence of local adaptation to malaria will be
detected within diverse African populations.

The aim of this project is to examine patterns of nucleotide variation, using
DNA re-sequencing technology, at two malaria resistance candidate genes,
intercellular adhesion molecule-1 (ICAM-1) and CD36, in a diverse group of
African and non-African human populations (39 total populations). This project
will: 1) identify and characterize patterns of nucleotide diversity at two loci
that have been suggested to influence susceptibility to malarial disease, 2)
conduct statistical analyses to identify signatures of natural selection and
examine the effects of demography, and 3) test for correlations between genetic
diversity and climate, subsistence strategy, ethnicity, and the prevalence of P.
falciparum infection.

These data will add to our limited knowledge of genetic diversity and the
demographic history of African populations. These data will also add to our
knowledge of how demography and natural selection have shaped patterns of
variation within the human genome. Ultimately, this research will provide a
clearer picture of human genetic adaptation and the co-evolution of human and P.
falciparum genomes, and will contribute to the training of a doctoral student
from an underrepresented group in anthropological genetics.