* 0929046
* Arabidopsis 2010: High-resolution Mapping of Regulatory DNA
* BIO,MCB
* 12/01/2009,11/30/2013
* Christine Queitsch, University of Washington
* Continuing Grant
* Karen Cone
* 11/30/2013
* USD 2,011,482.00

The genome of an organism encodes not only genes and their RNA and protein
products, but also the integrated programs that define when, where, and to what
extent different genes are activated or silenced. At the DNA level, gene
regulatory signals are encoded by regulatory elements that comprise clustered
recognition sites for DNA binding proteins. However, the location and function
of the vast majority of Arabidopsis regulatory sequences is currently obscure.
In this project, novel high-throughput epigenomic technologies--Digital DNaseI
Mapping and Digital Genomic Footprinting--will be applied to map and
characterize regulatory DNA across the A. thaliana genome at nucleotide
resolution. These technologies are capable of mapping the locations of
regulatory DNA sequences, and delineating the specific sites of regulatory
factor binding within such regions. Because gene regulatory programs vary widely
both between different cell types and within a cell type during differentiation,
the project will encompass multiple developmental stages and tissues of a
reference strain. As sessile organisms, plants integrate many cues into
appropriate developmental and stress responses, most of which rely on major re-
programming gene regulatory responses. Regulatory DNA involved in such responses
will therefore be mapped through study of standard stress conditions. At the
population level, most phenotypic variation is likely to derive from non-coding
genetic variation. By systematically extending maps of regulatory DNA across
both diverse A. thaliana accessions and related species, the project will expose
relationships between genotypic variation and gene regulatory programs on a
genome-wide scale. The resulting data will provide unprecedented insight into
endogenous and environmentally-responsive plant regulatory programs, and will
significantly accelerate the identification of functional non-coding variation
underlying relevant phenotypic variation. &lt;br/&gt;&lt;br/&gt;Broader impacts.
This project has the potential to change fundamentally the landscape of gene
regulation research in A. thaliana and in plants generally, both as it applies
to basic mechanisms and in its application to solve diverse quantitatively
varying phenotypes. The availability of comprehensive, high-resolution
regulatory DNA maps for A. thaliana stages, tissues, treatments, accessions, and
evolutionarily related species will immediately bring A. thaliana to the
forefront of regulatory genomics, and will provide a powerful attraction for
bringing dynamic new investigators to the field. The comprehensive annotation of
A. thaliana regulatory regions and transcription factor binding sites targeted
under this project will be of use to the entire plant biology community, and
will develop significant data resources that will potentiate experimental
approaches to determining gene function. The project will foster the advancement
of plant regulatory genomics through rapid dissemination of data to the public
domain via genomic databases as well as relevant analytical tools to assist in
its utilization by diverse investigators. The project will also encompass a
significant educational component aimed at training next-generation leaders in
plant regulatory genomics, and recruitment and training of talented
undergraduate and graduate scientists from diverse backgrounds.