* 0929338
* Arabidopsis 2010:   Nitrogen Networks in Plants
* BIO,MCB
* 07/15/2009,06/30/2015
* Gloria Coruzzi, New York University
* Continuing Grant
* Gregory W. Warr
* 06/30/2015
* USD 3,096,615.00

This project exploits and develops systems biology approaches to identify the
regulatory networks mediating plant adaptation to nitrogen (N) nutrient changes
in the environment. In the previous cycle, transcriptome datasets analyzed and
integrated in the context of an Arabidopsis multinetwork identified the first
regulatory networks and components (transcription factors and micro RNAs) that
regulate plant metabolism and development in response to sensing nitrogen
signals in their environment. These discoveries now fuel a new round of the
systems biology cycle of high throughput experimentation, modeling, and
hypothesis generation. This project exploits new technology (deep-
transcriptomics) and new biology discovered during the previous cycle, namely
the prevalence of the "RNA world" in controlling plant adaptation to the
environment. In this project, large-scale experimentation will be used to gather
genome-wide quantitative evidence for the role of RNA-based network modules in
mediating Nresponses (Aim 1). Validation of the role of miRNA-TF motifs in
mediating root adaptation to Nenvironments will be tested using two
complementary approaches. The first is a unique experimental set up (split-root)
designed to identify and test components involved in adapting root growth in
response to local vs. systemic N-signals (Aim 2). The second addresses how these
RNA-based network motifs and modules evolve across micro-evolutionary time to
enable populations to adapt to changes in N-nutrient acquisition in natural
environments (Aim 3). To integrate the exhaustive data, models and pipeline
analysis tools will be developed to encompass RNA data from wild-type, mutants
and ecotypes (Aim 4). In total, this project will provide a complete view of the
"RNA plant" that will merge exciting new advancements in i) biotechnology (deep
sequencing) with ii) new biology (smRNAs, mRNAs) to create iii) new regulatory
nodes and edges in networks that can be used to generate hypotheses for testing.
The data, approaches and tools developed in this project will be applicable to
(and made available for) the community to enable a systems approach to answer a
wide range of problems in plant biology. The results obtained will also enable
the derivation and testing of hypotheses for regulatory components that can be
manipulated to effect changes in N-use efficiency in plants, an issue that
affects energy-use and the environment.&lt;br/&gt;&lt;br/&gt;The aims of this
project integrate across the four goals of the Arabidopsis 2010
project:&lt;br/&gt;1. Metabolic biology, particularly relevant to energy capture
and use: Identify regulatory components responding to N-metabolite signals
(N-use efficiency) (Aim 1).&lt;br/&gt;2. Adaptation to the environment: Identify
network components (mir-TFs) that mediate root adaptation to changes in local
N-environments using a unique split-root system (Aim 2).&lt;br/&gt;3. Multi-
scale analysis of genome evolution and genetic systems: Examine how N-regulatory
networks evolve in ecotypes to mediate root adaptation to N in natural
environments (Aim 3).&lt;br/&gt;4. Development of resources (including
informatic tools) for genome-wide experimental approaches to determine gene
function: Develop informatic pipelines for analysis and integration of the RNA-
plant into next-generation networks and for comparative ecotype analysis (Aim
4).&lt;br/&gt;&lt;br/&gt;Broader impacts of research&lt;br/&gt;A. Applications
to Agriculture: Modification N-use efficiency in plants. Patents filed from this
project relate to improving nitrogen use efficiency, which has implications for
energy and the environment.&lt;br/&gt;B. Development of informatic tools: To
interrogate the RNA-world and interpret it in the context of next-generation
networks within and across ecotypes, and to associate genomes with
phenotypes.&lt;br/&gt;C. Assign function to unknown genes: Network analysis
associated many genes of unknown function with regulatory networks mediating
response of lateral root development to the N-environment.&lt;br/&gt;D. Training
in Systems Biology: Postdocs and students will be trained in Systems Biology by
comentorship between biologists (Coruzzi, Crawford &amp; Birnbaum) and
Math/Computer scientists (Shasha &amp; Tranchina) from The Courant Institute of
Math &amp; Computer Science.&lt;br/&gt;E. Collaborations: In addition to the
biologists, mathematician, and Computer Scientist listed above, this project
also involves international collaborators (Rodrigo Gutierrez, Chile) and high-
throughput technology collaborations (McCombie, CSHL).