* 0929100
* Arabidopsis 2010:   Functional Analysis of Ubiquitin-Protein Ligase (E3) Families in Arabidopis
* BIO,MCB
* 08/01/2009,07/31/2015
* Richard Vierstra, University of Wisconsin-Madison
* Continuing Grant
* Engin Serpersu
* 07/31/2015
* USD 4,675,380.00

Intellectual Merit:&lt;br/&gt;The goal of this project is to define the network
of Arabidopsis proteins responsible for covalent attachment of the small protein
ubiquitin (Ub) to specific cellular targets. The attachment of Ub to other
proteins is highly selective and plays an integral role in numerous
developmental and metabolic processes in plants. Although addition of Ub was
originally discovered as a mechanism to promote protein degradation, recently
other functions have become apparent, including roles in DNA repair, lysosomal
catabolism, intracellular trafficking, and the regulation of chromatin structure
and transcription. Several enzymes are needed for attachment of Ub to targets,
but among these the Ub-protein ligases (or E3s) are the crucial enzymes that
control both target selectivity and the nature of the Ub linkage. Genome
analysis now shows that Arabidopsis may express over 1,500 E3 components, making
E3s one of the largest functional protein groups in plants (~5% of the
proteome). The diversity of E3s is unique to the plant kingdom, and the large
size of this family indicates it presents a versatile mechanism for controlling
cellular processes. This versatility makes it of significant interest relative
to plant evolution and adaptation to the environment, as well as holding
potential for future plant genetic engineering. This project will expand
successful collaborative efforts to define E3 proteins families and their
targets in Arabidopsis, using the large battery of methods, reagents and
mutants, as well as a variety of new genomic and structure-based approaches. The
project will continue to identify and annotate the Arabidopsis E3 protein
families and define their biochemical activities. Their importance to
Arabidopsis physiology, development and survival will be analyzed by reverse
genetic analyses and further studied by expression profiling, localization using
GFP-E3 fusions, and strategies to identify partners or substrates (yeast-two-
hybrid and co-immunoprecipitation). The project will continue to develop protein
microarrays as a high-throughput method to unravel the network of proteins
involved in E3 action. Further substrates of ubiquitylation will be identified
and then confirmed genetically and by global protein stability assays. The
relationships between E3s and their partner E2s will be assessed by substrate-
specific in vitro assays and genetic analyses. 3-D models for how targets are
recognized by representative E3s will be developed by X-ray crystallographic
analyses. Finally, the project will transfer knowledge gained from Arabidopsis
to other plant species to help define E3 evolution, to understand why plants are
so reliant on this post-translational modification, and to identify E3 orthologs
in crops, which can then be exploited to improve agricultural productivity.
&lt;br/&gt; &lt;br/&gt;Broader Impact: &lt;br/&gt;The program will provide high
level training for postdoctoral researchers and graduate students. A mentoring
plan is in place to involve trainees specifically in design of the research, and
to develop career opportunities. Students and postdocs will present their work
at annual project meetings of the investigators, exposing them to the
interdisciplinary aspects of the work. In addition to training graduate students
and postdoctoral fellows, under-represented minority undergraduates will also be
part of the discovery process through a summer research program partnering with
the University of West Alabama. This project aims to train, in a highly
interdisciplinary context, the next generation of professionals to continue
these efforts, and to expose under-represented minorities to plant research.