* 0940979
* EAGER:   Exploring the potential of pentatricopeptide repeat proteins for the site-directed modulation of RNA metabolism
* BIO,MCB
* 08/15/2009,07/31/2012
* Alice Barkan, University of Oregon Eugene
* Standard Grant
* Karen Cone
* 07/31/2012
* USD 299,758.00

This award is funded under the American Recovery and Reinvestment Act of 2009
(Public Law 111-5).&lt;br/&gt;&lt;br/&gt;Intellectual Merit&lt;br/&gt;The
ability to modulate specific steps in gene expression via "designer" regulatory
elements has broad application in basic research and biotechnology. RNA binding
proteins can modulate RNA-mediated events, but most classes of RNA binding
protein are poor candidates for engineering applications due to the difficulty
of predicting their RNA binding properties from their sequence or structure. In
this context, the unusual RNA recognition mechanism of the Pumilio Homology
Domain (PUM-HD) has attracted considerable attention: the PUM-HD consists of
eight helical repeats that recognize a contiguous 8-9 nucleotide RNA segment via
a one-repeat, one-nucleotide mechanism. This modular recognition mechanism
suggested a "code" for RNA recognition by the PUM-HD, and offers the possibility
of engineering the PUM-HD to bind novel RNA targets. This project focuses on a
less well-known protein class, the pentatricopeptide repeat (PPR) proteins,
which offer greater promise for the design of novel RNA binding specificities
and for the engineering of gene regulatory systems. PPR proteins are predicted
to adopt a helical repeat solenoid structure that is reminiscent of the PUM-HD.
Current data suggest that PPR tracts, like the PUM-HD, bind RNA via a modular
recognition mechanism whose rules should be definable. However, the highly
variable number of repeats in PPR proteins and the remarkable diversity of their
natural RNA ligands and physiological functions support the view that PPR tracts
provide a much more malleable platform for RNA binding. Furthermore, the ability
of PPR proteins to bind single-stranded RNA along an unusually long interface
imparts an unusual repertoire of biochemical activities, which predict specific
effects on gene expression when PPR proteins are targeted to specific sites.
This project will (i) test the premise that the unusual properties of PPR
proteins can be exploited to modulate gene expression at diverse steps in
diverse organisms, and (ii) make substantial progress towards understanding the
"code" for sequence-specific RNA binding by PPR
proteins.&lt;br/&gt;&lt;br/&gt;Broader Impacts&lt;br/&gt;Development of the PPR
motif as a platform for engineering applications will require the ability to
design PPR proteins to recognize a wide variety of RNAs, and knowledge about the
steps in gene expression that can be modulated by PPR/RNA interactions. This
project addresses both of these issues, and will thereby provide the foundation
for the development of new tools for the manipulation of gene expression in both
prokaryotes and eukaryotes. Successful outcomes will provide the basis for the
design of powerful selection schemes for the directed evolution of PPR proteins
to recognize new RNA ligands. Furthermore, the results will provide mechanistic
insights into this large and poorly-understood family of RNA binding proteins,
which plays numerous essential roles in organellar gene expression (and thus in
energy transduction) in all eukaryotes. The interdisciplinary nature of this
research and the diversity of the organisms and assays to be employed will
provide a rich educational experience for an undergraduate student. The same
student will participate throughout the two-year project, such that the student
can take ownership of one aim and follow it through from beginning to end.