* 0940390
* EAGER: Viral tagging: Combining flow cytometry and genomics to explore virus-host interactions
* GEO,OCE
* 08/01/2009,07/31/2012
* Matthew Sullivan, University of Arizona
* Standard Grant
* David Garrison
* 07/31/2012
* USD 206,825.00

The investigator will develop a high-throughput viral-tagging method to
investigate virus-host interactions in natural marine communities. In this
approach, cultured or wild viruses are tagged by fluorescent labeling of their
nucleic acid and allowed to adsorb to cells, thereby labeling the cells for
differential sorting of viral-tagged cells by flow cytometry. Preliminary
experiments using this method to detect, sort, and identify ocean microbes with
attached viruses (i.e., viral-tagged cells) have been accurate and specific,
showing distinct fluorescent shifts for viral-tagged target host cell
populations and greatly reduced tagging of non-host cells.
&lt;br/&gt;&lt;br/&gt;This project will develop and rigorously evaluate this
viral-tagging method for &lt;br/&gt;use with wild populations of viruses and
microbes. First, additional systematic characterization of viral-tagging
methodology with cultured virus-host pairs will be needed before application to
wild populations. The investigator will use representatives of the three major
types of marine DNA viruses with ecologically important host cell lineages to
evaluate the effects of variable cell physiological states, experimental
conditions, and host specificity on viral-tagging efficiency and signal. In
addition, cryopreservation is critical for use of this method at remote sites.
The investigator will test various cryopreservation techniques for impact on
viral-tagging signatures and on downstream molecular biology. Parallel
challenges face single-cell genomic researchers, making intellectual cross-
fertilization desirable and likely. Finally, in experiments using cultured hosts
as "hooks" to capture wild viral diversity, one must enrich for viral DNA after
sorting viral-tagged cells. Based on stable isotope probing protocols, the
investigator will test using cultured "bait" microbes grown on a 13C-labeled
substrate to create heavy host DNA, thus allowing density-gradient separation of
host from viral DNA. Light viral DNA is then linker-mediated amplified and
pyrosequenced to reveal the host-specific viral metagenome.
&lt;br/&gt;&lt;br/&gt;Broader impacts: An experimentally validated viral-tagging
method offers to alleviate three bottlenecks in understanding virus-microbe
interactions. Specifically, the investigator anticipates that the method would
allow a researcher to perform high-throughput experiments that could (1) map the
in situ host range of a virus, (2) define the natural range of viruses that
infect a given host, and (3) enable targeted metagenomic sequencing to further
uncover the myriad ways that viruses impact microbial metabolism in the wild.
Such advances would be useful across fields of viral and microbial ecology, as
well as to biogeochemists and ecological modelers seeking population-scale
datasets. A post-doctoral researcher will receive significant methodological
training in advanced flow cytometry and molecular biological methods, as well as
the opportunity to gain experience in grant and paper writing, teaching and
cross-discipline and cross-cultural collaboration through collaboration with a
Fulbright scholar and a systems administrator to enable high throughput data
analysis pipelines to be established.