* 0945771
* EAGER:  Single Molecule Tools for Evaluating Histone Modifications in Single Living Cells
* BIO,MCB
* 09/01/2009,08/31/2012
* Ann Kirchmaier, Purdue University
* Standard Grant
* Karen Cone
* 08/31/2012
* USD 298,298.00

This award is funded under the American Recovery and Reinvestment Act of 2009
(Public Law 111-5).&lt;br/&gt;&lt;br/&gt;Intellectual Merit: The transformative
accomplishment of this research will be the development of single molecule
strategies to detect, quantify, and map multiple targets in a single nucleosome
simultaneously at any locus within a living cell. These single molecule tools
will enable researchers in both basic and applied sciences to uncover dynamic
aspects of chromatin modifications directing gene expression that cannot be
revealed by current approaches. The ultimate goal is to develop transformative
technologies to advance chromatin biology and gene regulation studies in any
cell type into the single living cell realm. Near-term objectives supporting our
ultimate goal are: 1) To identify any desired chromosomal locus by optimizing
nuclear delivery and locus-specific targeting in single living cells; and 2) To
quantify multiple histone modifications and histone variants within individual
or neighboring nucleosomes simultaneously at targeted loci. This
interdisciplinary proposal brings together a chromatin biologist and a
biophysicist to devise revolutionary single cell platforms to evaluate multiple
active or repressive histone modifications simultaneously in individual
nucleosomes. This high-risk approach will set the precedence for understanding
nuclear delivery, distribution, and targeting of multiple chromatin probes to
ensure efficient signal to noise ratio for the quantification of epigenetic
targets. Once developed, these powerful tools will be applicable to multiple
areas of research to profoundly impact an array of disciplines and fields
supported by the NSF's Molecular and Cellular Biosciences programs. Deliverables
include: (i) an optimized strategy for nuclear probe delivery and targeting,
(ii) identification and quantification of multiple modifications within a single
nucleosome and the extent to which individual modifications are required for
others to occur, (iii) elucidation of histone modifications present within
single nucleosomes at any defined locus in a living cell.
&lt;br/&gt;&lt;br/&gt;Broader Impacts: The power of this approach comes from its
sensitivity to detect single molecules in single cells; its modular design that
promotes customization; and its applicability to a wide range of basic and
applied research questions related to gene regulation, DNA replication and
repair, and chromosome dynamics in cells. Educational impact will include the
cross-disciplinary training of a postdoctoral researcher and a graduate student
in the fields of Biochemistry and Biophysics, opening new avenues for career
development. The project also has the capacity to support multidisciplinary
research and curriculum development. Training of undergraduates will be achieved
through the involvement of sophomore/junior level students from the Discovery
Undergraduate Research Internship Program, the Biochemistry undergraduate
research program, the Summer Research Opportunity Program or the Summer
Undergraduate Research Fellowship program at Purdue.