* 0959427
* QuasiNovo:   An Information Theoretic Approach to De Novo Peptide Sequencing
* BIO,DBI
* 04/15/2010,03/31/2014
* Ian Dryden, University South Carolina Research Foundation
* Standard Grant
* Peter McCartney
* 03/31/2014
* USD 643,747.00

The University of South Carolina is awarded a grant to develop tools for peptide
identification. Tandem mass spectrometry has become increasingly important in
high-throughput protein identification. The most popular approach to identifying
peptides from mass spectra uses the experimental spectrum to query a database of
known peptides. A major weakness of all database approaches is that they are
unable to identify peptides that are not contained in the database. This is a
particularly important limitation in the case of microbial peptides. It is well
known that only 1%-10% of all microbes found in the environment can be cultured.
Thus there are many bacteria that have not been previously identified. Indeed,
even among culturable organisms many remain uncharacterized due to extreme
diversity. Only de novo sequencing offers the possibility of identifying novel
peptides.&lt;br/&gt;&lt;br/&gt;The goal of this project is to make significant
improvements in the accuracy of de novo peptide sequencing. This will be
accomplished through a systematic study of the use of amino acid usage models in
fragmentation and cross-correlation peptide scoring functions. Preliminary
results strongly support the hypothesis that a scoring function that considers
amino acid usage patterns will be better able to distinguish between candidate
peptides. This in turn will lead to much higher accuracy in peptide prediction.
In addition, a Bayesian model for exploring the uncertainty of candidate
peptides produced by de novo sequencing will be developed. A major difference
between this model with that developed by other groups is that the proposed
approach will incorporate the concept of proteome signature as a prior. Existing
models for de novo sequencing do not expressly indicate amino acid usage, and
thus implicitly assume flat priors for amino acid usage. The proposed research
will be integrated into undergraduate and graduate courses taught in computer
science and statistics at the University of South Carolina. The research will
also be leveraged to support training and education through the development of
training workshops to be presented at annual SC-INBRE Bioinformatics Core
statewide meetings. The target audience of the annual workshops will be
undergraduate and graduate students as well as faculty and staff in
bioinformatics-related disciplines (biology, computer science, statistics, etc.)
in South Carolina.