
* 9501570
* Bayesian Methods for Multiple Sequence Alignment and RelatedStatistical Problems
* DMS,STATISTICS
* 09/01/1995,06/27/1997
* Jun Liu,CA,Stanford University
* Continuing Grant
* Joseph M. Rosenblatt
* 08/31/1999
* USD 63,000.00

9501570 Liu Abstract (technical) Multiple sequence alignment is crucial in
research on the structure and function of gene products, through promoting the
detection and description of sequence motifs; aiding efforts at protein
modeling, structure prediction and engineering; and shedding light on molecular
evolution and phylogeny construction in molecular systematics. We recently
presented new local alignment algorithms (Lawrence et al. 1993, 1994) based on
Bayesian modeling and Gibbs sampling. In this project, we propose to develop
several extensions of our original Bayesian model to accommodate various
practical situations, and to investigate several statistical problems arisen
from our study of the alignment problem. On the applied side, we wish to
accomplish the following: (1) relax the existence assumption and develop
Bayesian models to simultaneously detect and align the common multiple motifs in
the sequences; (2) combine the idea of Markovian structure modeling of sequence
data (e.g., hidden Markov Model) with the block-based method of Lawrence et. al.
(1993) to enhance the alignment of multiple motifs; (3) modify and apply the
methodology developed previously and in this project to several other important
biological problems: the detection of subtle sequence signals in noncoding DNA
region; the prediction of protein secondary structures; and the combination of
threading and alignment. On the theoretical side, we attempt to (4) understand
the nature and potential of predictive updating in connection with the
bootstrap, the Bayesian bootstrap, and general nonparametric problems; (5)
develop model selection criteria that are useful for determining pattern width
in alignment problems; (6) study the influence of prior specifications on the
results and frequency properties of these results in Bayesian classification
problems; and finally, (7) study convergence properties of several sampling
algorithms. During the award period, I will develop my own styles on teaching
introductory probability and statistics courses with an emphasis on statistical
thinking, and get involved in a program designed to attract talented
undergraduates into our field. I will also make an effort to develop new
graduate level courses on current research areas such as image processing,
missing data, Markov Chain Monte Carlo, genetics and biology. My guideline for
directing graduate students is: independence, connections, and creativity.
9501570 Liu Abstract (nontechnical) A wealth of data concerning life's basic
molecules, proteins and nucleic acids, has emerged from the biotechnology
revolution. The human genome project has accelerated the growth of these data.
Multiple observations of homologous protein or nucleic acid sequences from
different organisms are often available. However, since mutations and sequence
errors misalign these data, multiple sequence alignment has become an essential
and valuable tool for understanding the structures and functions of these
molecule, aiding efforts at protein modeling, structure prediction and
engineering; and shedding light on molecular evolution and phylogeny
construction in molecular systematics. Traditional algorithms for finding such
alignment are either too computationally expensive so as to limit their
applications or too heuristic so that the sensitivity to subtle patterns is
lost. We recently presented new local alignment algorithms (Lawrence et al.
1993, 1994) based on a probabilistic model of the sequences and a stochastic
simulation technique called the Gibbs sampler. In this project, we propose to
develop several extensions of our original probabilistic model to accommodate
various practical situations, and to investigate several statistical problems
arisen from our study of the alignment problem. On the applied side, we wish to
accomplish the following: (1) relax the existence assum ption and develop more
general mixture models to simultaneously detect and align common patterns in
multiple homologous biological sequences; (2) investigate the connection between
our method and other successful approaches such as the hidden Markov modeling;
(3) modify and apply the methodology developed previously and in this project to
several other important biological problems: the detection of subtle sequence
signals in noncoding DNA region; the prediction of protein secondary structures;
and the combination of threading and alignment. On the theoretical side, we
attempt to (4) understand the nature and potential of the predictive updating, a
new iterative simulation method, in connection with the bootstrap, the Bayesian
bootstrap, and general nonparametric problems; (5) develop model selection
criteria that are useful for determining pattern width in alignment problems;
(6) study the influence of prior specifications on the final alignment results;
and finally, (7) study convergence properties of several stochastic simulation
algorithms. During the award period, I will develop my own styles on teaching
introductory probability and statistics courses with an emphasis on statistical
thinking, and get involved in a program designed to attract talented
undergraduates into our field. I will also make an effort to develop new
graduate level courses on current research areas such as image processing,
missing data, Markov Chain Monte Carlo, genetics and biology. My guideline for
directing graduate students is: independence, connections, and creativity.
