* 9408042
* Exon-Trap Sequencing Using Transposons in Arabidopsis
* BIO,MCB
* 08/01/1994,07/31/1997
* Robert Martienssen, Cold Spring Harbor Laboratory
* Continuing Grant
* DeLill Nasser
* 07/31/1997
* USD 450,000.00

Abstract 9408042 An in vivo gene-trapping technique has been developed in
Arabidopsis using the transposable elements Ac and Ds. Our modified Ds element
(DsG) contains a Beta-glucuronidase reporter gene that is equipped with splice
acceptor signals to allow multi- frame gene fusions (an "exon-trap"). When this
Ds element inserts into a chromosomal gene, the pattern of chromosomal gene
expression is mimiced by the Beta-glucuronidase reporter gene. Because
expression depends on gene fusion, insertions that result in reporter gene
expression will also result in gene disruption. Using a novel selection scheme,
a collection of 5,000 plants will be generated that will each carry an insertion
at a different location in the genome. Insertions into genes will be identified
by staining the progeny of each plant for reporter gene expression: 15-20% of
DsG insertions result in reporter gene expression, so that the proposed
collection should include insertions into 500- 1,000 genes. We will use this
collection to develop an exon-trap sequencing technique to enable the rapid
sequencing of a portion of the chromosomal gene identified by each exon-trap
tag. The chromosomal exon corresponding to each insertion will be amplified by
RACE PCR, and its sequence determined. As many of the exons as possible will be
mapped via anchored YAC contigs and recombinant inbred lines. A database of DsG
insertion lines will be developed, each characterized by expression pattern,
mutant phenotype and map location, as well as partial exon sequence. This
database can be integrated with existing genomic and cDNA sequence databases.
*** A major goal in the characterization of the Arabidopsis genome is to assign
a function to genes identified by cDNA and genomic sequencing, gene-finding
algorithms, and map location. Homology searches are expected to fulfill this
goal for those genes that have similarity to sequences from other organisms.
However, a majority of gene products will remain an onymous in these large
collections of sequenced genes. In the proposed study, a function will be
assigned to many of these genes by determining their patterns of expression, and
by observing the mutant phenotype that results from gene disruption. This
proposal will form the basis for establishing a large database of insertion
lines, as other laboratories use the "starter" lines to generate collections of
their own. This will allow a function to be assigned to many of the genes
identified by DNA sequence alone, as well as providing a genetic resource of
widespread use. %%%