* 0321678
* Oryza Map Alignment Project
* BIO,IOS
* 10/01/2003,09/30/2009
* Rod Wing, University of Arizona
* Cooperative Agreement
* Diane Jofuku Okamuro
* 09/30/2009
* USD 9,743,546.00

This project will map and align twelve wild genomes of rice to the public
finished rice genome sequences. These genome sequences were derived from Oryza
sativa ssp. japonica and ssp. indica, the major cultivated rice varieties grown
throughout Asia and India. The Oryza Map Alignment Project (OMAP) resource will
contain representatives of all extant wild rice genomes, making it possible to
describe the genome rearrangements that happened as wild rice underwent
domestication to yield the cultivated rice grown today. The OMAP resource will
also make it possible to explore the evolution of gene clusters across the rice
genus to learn about the diversity of genes in a cluster, or the sequences
regulating their expression. This kind of research will not only impact rice
genomics, but also the ways in which breeders can target and select genes for
agronomically important traits present in some wild rice varieties but absent
from cultivated rice. The OMAP resource will be the first of its kind, and will
be likely to serve as a model for future resources for other plants, and for
animals.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Specific objectives
&lt;br/&gt;&lt;br/&gt;1. Construct DNA fingerprint and Bacterial Artificial
Chromosome (BAC)-end sequence physical maps from the 12 wild genomes of rice.
&lt;br/&gt;&lt;br/&gt;2. Align the 12 physical maps with the sequenced reference
subspecies japonica and indica. &lt;br/&gt;&lt;br/&gt;3. Construct high-
resolution physical maps of rice chromosomes 1, 3 and 10 across the 12 wild
genomes using a combination of anchoring strategies.&lt;br/&gt;&lt;br/&gt;4.
Provide convenient bioinformatics research and educational tools to rapidly
access and explore the integrated rice genome map
resource.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Availability of project
outcomes&lt;br/&gt;&lt;br/&gt;All data derived from this project will be placed
in public databases as soon as it is produced and checked for quality.
&lt;br/&gt;&lt;br/&gt;1. All raw fingerprint data will be available in a
downloadable format that can be used by researchers on local Fingerprint Contig
(FPC) software.&lt;br/&gt;&lt;br/&gt;2. All assembled FPC contigs will be
viewable on the Internet with webFPC and Gramene
(www.gramene.org).&lt;br/&gt;&lt;br/&gt;3. All BAC-end sequences, quality and
trace files will be deposited in Genbank at weekly intervals.
&lt;br/&gt;&lt;br/&gt;4. All global alignment and SNP data with the reference
japonica sequence will be available on Gramene. &lt;br/&gt;&lt;br/&gt;5. All
reconstruction data for chromosomes 1,3 and 10 will be made available on the
project web site at Cold Spring Harbor Laboratory, Purdue University and the
University of Arizona (www.omap.org).&lt;br/&gt;&lt;br/&gt;6. All repeat
analysis will be made available through the Purdue University and Cold Spring
Harbor Laboratory web sites &lt;br/&gt;&lt;br/&gt;7. All software developed in
the course of the project at Cold Spring Harbor Laboratory, the University of
Arizona, and Purdue University will be released to the community under an open
source license approved by the Open Source Initiative
(www.opensource.org).&lt;br/&gt;&lt;br/&gt;