* 0331312
* QSB: Integrated Genomic and Metabolic Analysis of Arabidopsis Thaliana Physiology
* BIO,MCB
* 10/01/2003,09/30/2007
* John Quackenbush, University of Maryland, College Park
* Continuing Grant
* Nara Gavini
* 09/30/2007
* USD 498,514.00

It is presently clear that the analysis of a biological system requires the
integration of all fingerprints of cellular function: maps of gene expression,
total protein production and in vivo enzymatic activity. Information of all
profiles of a systematically perturbed cellular system can provide insight about
the function of unknown genes, the relationship between gene and metabolic
regulation and even the reconstruction of the gene regulation network. In this
context, this project will attempt an integrated analysis of the Arabidopsis
thaliana physiology, in which the response of the plant to external
perturbations will be simultaneously measured both at the genomic and metabolic
level. Specifically, the project comprises of the following specific aims: (a)
optimize and use a "homemade" experimental setup to apply various perturbations
in the growth environment (i.e. air and media composition) of young and healthy
plant liquid cultures grown under constant light and monitor the short-term
(one-day) plant response, (b) measure the average (over the entire plant) gene
and metabolic profile of the harvested plants, using full-genome DNA microarrays
and Gas Chromatography-Mass Spectrometry (GC-MS) respectively; measurements of
the tracer distribution in the GC-MS measured metabolite pools after the
introduction of labeled substrates, when possible, are expected to increase the
information about in vivo enzymatic activity, (c) analyze each of the profiles
in the context of the known A. thaliana metabolic network stoichiometry and
regulation and compare the results to unravel similarities and differences
between genomic and metabolic response to external stresses and (d) develop a
software module to be incorporated to the TIGR TM4 DNA microarray analysis
software package, which will allow the visualization of metabolic pathway
activity at the transcriptional level. It will also enable the clustering of
known genes based on a common metabolic characteristic and the identification of
unknown genes that show similarities to the "metabolic clusters'" expression
profiles under the monitored experimental conditions. &lt;br/&gt;&lt;br/&gt;The
broader impacts of this project relate to the development of a methodology for
the systematic integration of physiological data from both genomic and metabolic
levels in complex cellular systems. Integrated analyses provide enormous
opportunities in deciphering the language used by the cell to communicate
changes in the cellular environments to gene expression and vice versa.
Obtaining such detailed information about cellular physiology will open new
dimensions in metabolic engineering (cellular modifications), biotechnological
applications, disease prognosis and diagnosis, pharmacology, gene and other
medical therapies. The educational activities to be undertaken include
undergraduate and graduate teaching, graduate advising and various opportunities
in undergraduate research involvement and mentoring. In addition any software
modules developed from this project will be incorporated to the TIGR TM4 DNA
microarray analysis application package. TIGR TM4 is open -source software,
available to any member of the research community through the TIGR web
page.&lt;br/&gt;