* 0342172
* Modelling of Graphs, Networks and Trees for Genomic Applications:  High-Dimensional Model Search
* MPS,DMS
* 07/01/2004,06/30/2010
* Joseph Nevins, Duke University
* Continuing Grant
* Gabor Szekely
* 06/30/2010
* USD 1,910,000.00

This research concerns the development of theory, methods and computational
&lt;br/&gt;tools for statistical modelling motivated by applications in
functional &lt;br/&gt;genomics. The over-arching theme is that of "High-
Dimensional Model Search," &lt;br/&gt;i.e. the development of tools and methods
for generating, evaluating and &lt;br/&gt;interpreting sets of relevant
candidate models of a biological problem &lt;br/&gt;involving many variables.
Research includes the investigation of models and &lt;br/&gt;computational
methods for very large-scale graphical models, especially &lt;br/&gt;Gaussian
graphical models and very sparse model structures, as &lt;br/&gt;representatives
of the structure and patterns of association between &lt;br/&gt;biological
variables from observed and experimental data. Contexts of gene &lt;br/&gt;and
protein expression studies generate multiple motivating problems and
&lt;br/&gt;data for case studies. In problems of realistic dimension (thousands)
any &lt;br/&gt;one such graphical model is high-dimensional in both the variable
domain &lt;br/&gt;(number of nodes) and numbers of parameters; but, the
fundamental challenge &lt;br/&gt;is that the number of candidate models for a
given problem is astronomical. &lt;br/&gt;In fitting and evaluating candidate
models, it is typical that many &lt;br/&gt;candidates have plausible support
based on fit to the available &lt;br/&gt;observational or experimental data and
in the context of relevant biological &lt;br/&gt;information. Hence the need for
methods to identify, explore and evaluate &lt;br/&gt;many plausible models, and
to interpret and characterize their similarities &lt;br/&gt;and differences in
both statistical and substantive biological terms. &lt;br/&gt;Additional classes
of regression models, including nonlinear statistical &lt;br/&gt;classification
and prediction tree models, represent flexible frameworks for &lt;br/&gt;the
exploration, combination and utilization of multiple forms of biological
&lt;br/&gt;data in predictive phenotyping for prognosis, diagnosis and
discovery. In &lt;br/&gt;genomic contexts, high-throughput molecular information
such as gene &lt;br/&gt;expression data leads to very, very many potential
predictor variables to be &lt;br/&gt;assessed, so that, again, the challenges to
statistics include dealing with &lt;br/&gt;and exploring spaces of very high-
dimensional models. These central research &lt;br/&gt;challenges -- variable
selection and model uncertainty that, with large &lt;br/&gt;numbers of
variables, are simply unanswerable based on current mathematical &lt;br/&gt;and
statistical methods -- form the core focus of this project. Motivated by
&lt;br/&gt;and integrated with a number of genomic studies that provide data,
&lt;br/&gt;collaborators and application contexts in functional genomics in
several &lt;br/&gt;areas of application, this project involves model development
and innovation &lt;br/&gt;in methods of stochastic search over complex and very
high-dimensional &lt;br/&gt;statistical model spaces. This includes associated
model theory, methods, &lt;br/&gt;and computational algorithm development that
involve distributed cluster-&lt;br/&gt;based implementations, as well as
feedback application in a number of &lt;br/&gt;biomedical studies.
&lt;br/&gt;&lt;br/&gt;Modern biomedicine now has access to data of rapidly
escalating scales and &lt;br/&gt;complexities based on innovations and advances
in, in particular, genome &lt;br/&gt;technologies. This includes high-throughput
genomic data from DNA microarray &lt;br/&gt;gene expression studies in both
laboratory and human observational disease &lt;br/&gt;and exposure studies,
related large-scale molecular information from &lt;br/&gt;proteomic and
metabolic profiling technologies, genetic and sequence &lt;br/&gt;information
that is quickly expanding towards genome-scale sequence data, &lt;br/&gt;and, of
course, traditional forms of clinical, environmental and demographic
&lt;br/&gt;data. Advances in both basic biology and the use of such information
to &lt;br/&gt;inform and aid in human health studies requires very substantial
&lt;br/&gt;advances in the capacity to analyse and interpret these data sets of
ever-&lt;br/&gt;increasing scale and complexity. Bringing multiple forms of such
data to bear &lt;br/&gt;in defining predictive phenotypes is at the core of the
emergent arena of &lt;br/&gt;clinico-genomics, for example. The challenges to
modern mathematical and &lt;br/&gt;computational modellers are those of very
high-dimensional mathematical &lt;br/&gt;model specification and analysis,
coupled with the need for computational &lt;br/&gt;tools to search across
astronomical numbers of candidate models, an endeavor &lt;br/&gt;that is well
beyond the current capacity to implement, evaluate and &lt;br/&gt;understand.
This challenge defines the core agenda of this research project: &lt;br/&gt;the
development of statistical and computational tools for very
high-&lt;br/&gt;dimensional statistical models. The research is coupled with an
integrated &lt;br/&gt;collection of genomic studies that provide data,
biological collaborators &lt;br/&gt;and applications in functional genomics in
several areas of cancer and &lt;br/&gt;cardiovascular biology, pathway studies
in cell cycle regulation and &lt;br/&gt;oncogenesis, cancer proteomics,
transcription regulation and other areas. At &lt;br/&gt;the core of the research
lies innovation in methods of stochastic search -- &lt;br/&gt;i.e., simulation
techniques -- over complex and very high-dimensional &lt;br/&gt;statistical
model spaces, and the associated model theory and methods. The
&lt;br/&gt;inherent intellectual merit of the research lies in the advances and
&lt;br/&gt;innovative new methods in computation and statistical modelling, as
well as &lt;br/&gt;specific biological applications. The broader impact of the
research lies in &lt;br/&gt;the applicability of the resulting methods and tools
to applications in these &lt;br/&gt;specific areas, in other related
biomedical/genomic applications, and in &lt;br/&gt;other fields of
science.&lt;br/&gt;