* 2209128
* NSF Postdoctoral Fellowship in Biology
* BIO,IOS
* 07/01/2022,06/30/2025
* Benjamin Stone, Stone, Benjamin W
* Fellowship Award
* Kan Wang
* 06/30/2025
* USD 216,000.00

This action funds an NSF Plant Genome Postdoctoral Research Fellowship in
Biology for FY 2022. The fellowship supports a research and training plan in a
host laboratory for the Fellow who also presents a plan to broaden participation
in biology. The title of the research and training plan for this fellowship to
Dr. Benjamin Stone is "Genomic predictability of ecological speciation: insights
from replicate hybrid zones". The host institution for the fellowship is the
University of South Carolina and the sponsoring scientist is Dr. Carolyn
Wessinger.&lt;br/&gt;&lt;br/&gt;Identifying the evolutionary mechanisms
underlying patterns of genetic differentiation between species as they diverge
is a primary goal of speciation genomics. It is important to understand the
genomics of speciation in a variety of biological systems, because doing so
allows us to compare evolutionary processes across taxonomic groups and assess
the degree to which evolution may be repeatable and predictable across the tree
of life. However, our understanding these processes is currently limited, as
much of our knowledge stems from controlled experimental crosses in genetic
model systems. This project uses whole-genome sequence data to identify genomic
features of adaptive divergence and reproductive isolation across natural hybrid
zones of non-model plants in the genus Penstemon. The data and insights
generated through this project have applications in evolutionary biology,
agriculture management, and conservation biology, and include the development of
a comparative framework which addresses the predictability of genomic features
of adaptive trait divergence and speciation. Training objectives include the
development and integration of new skills in the quantification and analysis of
phenotypic and genomic variation in natural populations, as well as the
development of mentoring and project management skills through the mentorship of
undergraduate researchers and the completion of mentoring training programs.
Broader impacts include broadening diversity in STEM by utilizing well-
established programs at the host institution that target undergraduate
researchers from historically underrepresented groups in STEM. This includes the
recruitment of undergraduate mentees to collaborate on the work, and building
mentee research expertise through supervised independent
projects.&lt;br/&gt;&lt;br/&gt;Adaptive radiation, or the proliferation of
ecologically diverse species within a lineage, is a key mechanism of biological
diversification. Despite the importance of this process, our understanding of
the genomic underpinnings of adaptive trait divergence and reproductive
isolation is limited, especially in non-model systems. This limitation inhibits
our ability to determine the degree to which genomic processes of species
differentiation are predictable and generalizable across the tree of life. This
project integrates complementary methodologies (admixture mapping and genomic
cline analysis) across natural replicate hybrid zones in a non-model adaptive
radiation to understand whether the genomic basis of adaptive trait divergence
involves predictable features. These goals are pursued in two primary
Objectives. Objective 1 integrates phenotypic and genomic data to map QTLs for
adaptive floral divergence in separate hybrid zones involving secondary contact
between distinct species pairs. Objective 2 identifies genomic regions
associated with barriers to gene flow in hybrid zones and determines whether
these regions correspond to identified floral QTLs. This project generates novel
molecular (population-level whole-genome resequencing data) and phenotypic data
for focal taxa in the genus Penstemon, and integrates complementary
methodologies across these data sets to contribute to a more phylogenetically
diverse perspective of the genetic underpinnings of ecological adaptations and
their role in reproductive isolation. Phenotypic data and scripts for data
analysis will be available on the Dryad Digital Repository. Genome assemblies
will be archived in the NCBI Genome Assembly repository, and raw reads will be
deposited in the NCBI Sequence Read Archive.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.