* 2209002
* NSF Postdoctoral Fellowship in Biology FY 2022: Determining functional correlates to the emergence and maintenance of antibiotic resistance genes in host-associated microbiomes
* BIO,DBI
* 01/01/2023,12/31/2024
* Taylor Reiter, Reiter, Taylor E
* Fellowship Award
* Daniel Marenda
* 12/31/2024
* USD 0.00

This action funds an NSF Postdoctoral Research Fellowship in Biology for FY
2022, Integrative Research Investigating the Rules of Life Governing
Interactions Between Genomes, Environment and Phenotypes. The fellowship
supports research and training of the fellow that will contribute to the area of
Rules of Life in innovative ways. Microbes are everywhere, and participate in
broad number of biological processes such as producing and degrading
antibiotics, and aiding in host metabolism. As most microbes have not yet likely
been discovered, continued investigation of these organisms will lead to better
understanding of the ecological importance and technological potential of
microbes. Many recent discoveries in microbiology have relied on DNA and RNA
sequencing approaches to identify new microbes. While sequencing is a excellent
approach, there exists a large amount of sequencing data that is currently being
unused. This project will develop tools to help recover this data in microbial
communities, improving our understanding of the role microbes play in different
ecosystems. The fellow will also engage in multiple training efforts to improve
computational literacy of biologists from the undergraduate to post doctoral
level.&lt;br/&gt;&lt;br/&gt;The overarching goal of this project is to improve
gene annotation, abundance estimation, and functional characterization of
metagenomes and metatranscriptomes. This project will first extend current
approaches that rely on assembly graphs to achieve ortholog-level annotation and
abundance estimation in meta-omic sequencing data. Assembly graphs contain all
the sequences in a meta-omic library and thus do not suffer the same data loss
as current analysis techniques. Meta-omic sequencing experiments also frequently
lack power given small numbers of profiled communities from a given environment.
Transfer learning overcomes this problem by first training a model on a large
microbial dataset, then re-using this learned information to predict patterns in
a smaller dataset. This project will build a transfer learning model that will
be trained on publicly available bacterial isolate RNA sequencing data and will
be used to discover patterns in metatranscriptomes. These approaches will be
used to study antimicrobial resistance in pig gut microbiomes. While
antimicrobial resistance can often be detected through sequencing, the
occurrence of antimicrobial resistance genes is likely underestimated because
these sequences are shared across many genomes which makes the resultant
sequencing data complex. The tools developed in this project will improve the
detection of complex sequences like antimicrobial resistance genes. The project
will also investigate how host traits like rearing environment and antibiotic
exposure impact the occurrence and emergence of antimicrobial resistance in pig
gut microbiomes when exposed to various antibiotics. The tools developed in this
study will improve annotation and functional characterization of diverse
microbial ecosystems, generating predictive value in other systems with meta-
omic sequencing data. Lastly, this proposal aims to broaden participation in
science using open science principles for the development and dissemination of
research products, and through training endeavors that facilitate entry into the
field of data-intensive biology.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.