* 2211645
* CIF: Small: Multiview Graph Learning with Applications to Single Cell Gene Expression Networks
* CSE,CCF
* 07/01/2022,06/30/2025
* Tapabrata Maiti, Michigan State University
* Standard Grant
* James Fowler
* 06/30/2025
* USD 593,145.00

Modern data analysis involves large sets of structured data, where the structure
carries critical information about the nature of the data. The relationships
between entities, such as features or data samples, are usually described by a
graph structure. While many real-world data are intrinsically graph-structured,
e.g. social and traffic networks, there are still a large number of applications
where the graph topology is not readily available. For instance, gene
regulations in biological applications or neuronal connections in the brain are
not usually observed. Inferring the underlying structure is an essential task
for such data. Most of the existing work on graph learning focuses on learning a
single graph structure, assuming that the relations between the observed data
samples are homogeneous. However, in many real-world applications, there are
different forms of interactions between data samples, such as single-cell RNA
sequencing (scRNA-seq) across multiple cell types. This project aims to address
the multi-view graph-learning problem for heterogeneous data with a focus on
gene regulatory network (GRN) inference from scRNA-
seq.&lt;br/&gt;&lt;br/&gt;This project will introduce a multi-view framework to
learn graphical structures from heterogeneous data. First, a new approach for
learning signed graphs will be introduced. Signed graphs are commonly
encountered in biological networks, where the positive and negative edges
correspond to activating and inhibitory relationships, respectively. This
framework will take the nonlinear nature of interactions between nodes into
account through graph signal kernels. Second, a comprehensive framework for
multi-view graph learning in two settings will be considered: i) multiple views
of the same data and ii) heterogeneous data with unknown cluster information. In
the first case, a joint learning approach where both individual graphs and a
consensus graph are learned will be developed. In the second case, a unified
framework that merges classical spectral clustering with graph signal smoothness
will be developed for joint clustering and multi-view graph learning. The graph-
learning algorithms will address some of the challenges encountered in gene
regulatory network inference, such as non-Gaussian, nonlinear nature of gene
expression data, changes in gene expression due to cell-cycle heterogeneity, and
high sparsity due to low amounts of mRNA in individual cells. This project will
provide interdisciplinary training to a diverse population of students at all
levels. The research outcomes will be incorporated into a new online graduate
course as part of the growing interest in data science. Finally, the proposed
research will be incorporated into outreach efforts targeting K-12 female
students and disseminated to the broader research community through
publications, workshops and code sharing.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.