* 2217515
* Developing Random Field based novel approaches for spatial transcriptomics
* BIO,DBI
* 08/01/2022,07/31/2025
* Xiaobo Zhou, The University of Texas Health Science Center at Houston
* Standard Grant
* Jean Gao
* 07/31/2025
* USD 704,556.00

The overall goal of the project is to develop Random Field based approaches to
spatially analyze and understand tissue architecture and heterogeneity as well
as the communication between different cells and stromal components. The
knowledge will gain insights into the potential molecular mechanism related to
cell evolution, tissue development, disease progression, and drug resistance.
The developed tools will significantly improve our understanding of tissue
heterogeneity and development. The characterization and use of subclones and
spatial architecture for function analysis will provide a completely different
approach to study tissue heterogeneity. Graduate and undergraduate students will
work under this project and gain experience in leading-edge research. An
undergraduate summer research program on "Computational Systems Biology‚Äù will be
organized. &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Systematic analysis methods used to
analyze spatial transcriptome data are still in their infancy. Current analysis
methods of spatial transcriptome data focus on transcriptional profiling. Novel
methods are needed to identify genomic variants and integrate genetic and
transcriptional variations. To address the challenges, a hybrid model based on
Variational Graph AutoEncoder (VGAE) will be developed to characterize the
spatial relationship between the spots and subclones. Hidden Markov Models (HMM)
will be used to infer the copy number variations (CNVs) from spatial
transcriptomic data. The new approach of identifying subclone and CNVs from
spatial transcriptomics through VGAE and HMM is coined as CVAM. A Random Field
based computational toolset called SPAT (spatial architectural analysis) is
proposed to study the architectural heterogeneity with spatial transcriptomics
and scRNA-seq data. SPAT can identify single gene-based spatial biomarkers,
analyze spatial distribution patterns of signaling networks, and explore cell-
cell interaction using spatial transcriptomics and scRNA-seq data. Specifically,
a Gromov-Wasserstein distance and Random Field-based approach will be designed
to characterize signaling between adjacent cells and stromal cells. Finally the
new computational tools and results will be validated through biological
experiments. Software prototypes and the variants and gene biomarkers with
spatial patterns will be made publicly available to the research community via a
project website at https:/ccsm.uth.edu/NSFSPA.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.