* 2218477
* Engineering autoregulation of the epitranscriptome to track and control stress responses
* BIO,MCB
* 07/15/2022,06/30/2025
* Andrew Ellington, University of Texas at Austin
* Standard Grant
* Adrienne Cheng
* 06/30/2025
* USD 788,204.00

Environmental stressors of biological systems are highly complex. It has been
postulated that environmental stressors can result in RNA modifications in
biological cells. The study of such modifications is known as
epitranscriptomics. In this project, an interdisciplinary team of researchers,
including graduate students and undergraduate students, are using
epitranscriptomics to study how chemically induced changes in RNA molecules
within biological cells could be used to record environmental stresses. The
study makes use of computational tools, biochemical characterization,
traditional genetics methods as well as tools in Next Generation Sequencing.
These studies contribute to the general understanding of how cells sense, record
and respond to their environments and are important for understanding how to
design early warning systems for virus detection for example. The project
provides research opportunities to students from low-income underrepresented
communities through the “Raising Future Scientists” program at the University of
Texas at Austin. Emphasis is given on promoting participation of undergraduate
students as a way to encourage their interest in graduate programs in science
and engineering. The project also contributes to the advancement of the public-
school synthetic biology curriculum.&lt;br/&gt;&lt;br/&gt;This proposal uses the
epitranscriptome as a mechanism to sense, record, and respond to the
complexities of the environment. The goal is to utilize RNA modifications in new
classes of modular devices that combine sensory RNA motifs, called “RNA
Sensors,” with activating protein domains, or “Protein Actuators”, to enable
engineered regulation of cellular responses. The research combines integrated
systems approaches involving computational tools, molecular RNA-protein
biochemical characterization strategies, traditional genetics methods, and
novel, in vivo Next Generation Sequencing-based tools to accomplish three
objectives: (i) establish the epitranscriptome as a sensory module in bacteria,
(ii) rationally re-engineer 8-OG recognition proteins as actuating modules, and
(iii) couple sensing and actuation of the epitranscriptome with CRISPR-systems
for signaling, memory, and stress response control capabilities. Results from
this research are broadly applicable in the field of synthetic biology and could
be used for understanding specific environmental exposures for risk assessment
and interventions.&lt;br/&gt;&lt;br/&gt;This proposal is jointly funded by the
Systems and Synthetic Biology program in the Division of Molecular and Cellular
Biosciences and the Cellular and Biochemical Engineering program in the Division
of Chemical Bioengineering Environmental and Transport
Systems.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.