* 2218236
* Collaborative Research: EAGER/Tools4Cells: Translating single cell data into an ultra-high resolution spatial map using fluorescent marker genes
* BIO,MCB
* 06/15/2022,05/31/2024
* Mao Li, Donald Danforth Plant Science Center
* Standard Grant
* Richard Cyr
* 05/31/2024
* USD 89,575.00

Recent technical advances in molecular biology now make it possible to determine
the entire population of messenger RNA transcripts within each individual cell
of a multicellular organism. This technology, known as single-cell RNA
sequencing (scRNA-seq), has the potential to be broadly applied in plants to
better understand their development, evolution, and stress responses. In this
project, the plant root will be used as a model organ to construct an ultra-
high-resolution 3D-model displaying gene expression data from individual cells
embedded in this model. Using this method, a user can locate cells labeled by a
fluorescent marker in a plant organ and determine the expression levels of
thousands of genes in both labeled and unlabeled cells. This tool can also be
used to combine fluorescent images from different reporter genes to understand
the similarity and differences of gene expression for both the marker gene and
other genes expressed in the same sample. The Broader Impacts of the work
include the intrinsic merit of the research results, which will be disseminated
to the broad research community via the Plant Cell Atlas (PCA). These results
will include protocols for collecting image data, a computational pipeline for
constructing 3D images, and a method to annotate and assign cell types in a
conceptual model of plant roots. The computational pipeline for image analysis
and machine learning will be deposited to public repository with detailed
documentation and user manuals and peer-reviewed publications. Research training
will be provided to graduate students and a post-doc and, through a
collaboration with Virginia State University, training workshops will be
developed for advanced genomic data analysis for VSU
students.&lt;br/&gt;&lt;br/&gt;Connecting spatial location of individual cells
and gene expression patterns within each cell is the frontier of plant cell
biology research. Currently available scRNA-seq protocols do not preserve
spatial locations of each cell, whereas spatial transcriptome approaches using
physical slices of embedded tissues have limited resolution. The goal of this
EAGER project is to establish a new approach for spatial transcriptome analysis
in plants. One major resource from the plant research community is a large
number of transgenic reporter gene lines (e.g. promoter-GFP lines) that have
been accumulated for the past several decades. This project will leverage this
large reporter gene resource to perform a proof of principle study using the
same GFP marker lines for both imaging and scRNA-seq experiments. Using the
meristematic region of plant roots as our model system, scRNA-seq data for
selected promoter-GFP marker lines will be generated and machine learning models
will be applied to accurately predict GFP+ and GFP- cells. Fluorescent imaging
and sematic labeling will be used to merge and model 3D root images and GFP
expression. Finally, a machine learning method will be developed to map the
scRNA-seq data to the 3D root model. Results from this new approach will be
compared with existing data and will be validated in planta. Together, this work
will provide a powerful new approach to develop 3D expression models for any
plant species. Results from this method can be used to address questions related
to asymmetrical gene expression in development and stress responses in roots, as
well as in other tissues or organs in plants. This project is jointly funded by
the Divisions of Molecular and Cellular Sciences (Cellular Dynamics and Function
program) together with Integrative Organismal Systems (Physiological Mechanisms
and Biomechanics program) , both in the Biological Sciences
Directorate.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.