* 2220726
* EDGE CMT: Genetic basis of plant root growth traits and their response to environment
* BIO,IOS
* 01/01/2023,12/31/2025
* John McKay, Colorado State University
* Continuing Grant
* Christopher Balakrishnan
* 12/31/2025
* USD 1,500,000.00

Predicting how complex phenotypes emerge from the interaction of genetic
variation and developmental environment (i.e., genotype by environment
interaction) has become increasingly feasible in model systems. The ability to
make this kind of prediction is important in many fields, for example:
understanding of disease risk, increase the potential for mitigating the effects
of a changing environment, and improve the efficiency of breeding for advanced
agricultural varieties. However, accurate prediction does not mean we have
achieved mechanistic understanding at the individual or population level.
Addressing this challenge of a mechanistic understanding requires the ability to
replicate many genotypes across environments, a difficulty that has hindered
studies of complex traits in humans and vertebrate model systems. Crops are
ideal for generating the data needed for dissecting mechanisms of genotype by
environment interactions and are the original model systems for quantitative
genetics. This research provides an opportunity to develop and test methods for
studying genotype by environment interactions using maize field trials, a system
where such replication is highly feasible. To extract mechanistic understanding
of pathways from these data, this project will develop statistical methods and
software that will be useful for a broad range of species. The results, methods,
and concepts addressed in our research could also be extended by facilitating a
shared learning experience that help traditionally unrepresented students
challenge barriers and build collaborative relationships with researchers with
experience in bridging fundamental science with pragmatic application in
agriculture.&lt;br/&gt;&lt;br/&gt;This main goal of this project is to provide
mechanistic understanding how genes interact with variable environments to
produce complex phenotypes, specifically root system architecture and gene
expression in nodal root tissues. The phenomenon of a genotype producing
different phenotypes in response to different environmental conditions is termed
phenotypic plasticity and is a ubiquitous aspect of biology. This project will
use maize root system architecture traits as a model system for mechanistic
understanding of the genetics of complex traits. The research will refine and
validate field-based high-throughput phenotyping (HTP) of plant root system
architecture under agriculturally and ecologically relevant well-watered and
controlled-drought conditions. These HTP root phenotypes will be collected
across the lifecycle to understand the polymorphisms underlying differences in
growth trajectories. Gene expression analysis of root tissue will allow eQTL
mapping and test the relative role of cis and trans polymorphisms on expression
of core genes. Phenotyping will be performed on bi-parental recombinant inbred
populations of maize. Linking phenotypes to causal polymorphisms in these
populations will involve the development of new computational tools for
extracting the maximum amount of information from empirical studies of Genotype
x Environment (GxE) interactions and time-series data. Together, these novel
data and analysis methods will be used to identify polymorphisms underlying
genotype x environment interactions (GxE) and genotype x time interactions
(GxT). The functional roles of candidate genes and the utility of simple, multi-
gene models of GxE will be tested using single and higher order mutants in maize
and Arabidopsis thaliana. This project has four specific aims : 1)Perform
empirical studies of QTL x E x Time in maize roots in field studies, 2) Perform
whole genome transcript abundance analysis of root tissue to allow eQTL mapping
and test predictions of the role of cis effects on core gene expression, 3) Use
mutants, CRISPR edits and overexpression of large effect size GxE QTL to test
models of plasticity to soil moisture in both dicot and monocot species, 4)
Develop QTL mapping methods based on multivariate linear models to handle QTL x
E x Time data.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.