* 2221237
* Simulating a growing minimal cell: Integrating experiment and theory
* BIO,MCB
* 08/15/2022,07/31/2026
* Zaida Luthey-Schulten, University of Illinois at Urbana-Champaign
* Continuing Grant
* Bianca Garner
* 07/31/2026
* USD 499,999.00

All cells share a universal, minimal set of biological processes essential for
life. The search for this set led to the construction of the minimal bacterial
cell JCVI-syn3A. With 493 genes in a genome of 543 kbp, JCVI-syn3A has a genome
smaller than that of any independently-replicating cell found in nature, a
robust morphology, and can divide every two hours in a stress-free laboratory
growth medium. Nearly all genes in this minimal cell are essential, and the cell
is small enough that a complete description of all cellular functions can be
attempted over biological relevant length, time, and concentrations scales by
exploiting graphics processing unit (GPU) computing. Recent successes in GPU
computing, and 3D imaging have made it now possible to build a whole-cell
computational model of this minimal bacterial cell and to investigate what are
the rules of life allowing this cell to grow and divide. In this project the
investigators construct a whole-cell model coupling all the cellular functions.
The outcome of this project will allow the research team to predict cellular
behavior under a variety of perturbations, and thus explain how a complete cell
works. The educational broader impacts include the training of students and
postdoctoral investigators, and outreach to the broader community through
workshops and YouTube/VR platforms facilitating the public dissemination of the
science.&lt;br/&gt;&lt;br/&gt;This research project addresses key components
needed to make a more complete computational model of a growing minimal cell.
Approximately âˆ¼90 out of the 493 genes encode proteins of unclear cellular
context, of which 30 were determined to be essential from transposon bombardment
experiments. Extension of the current 3D spatial model to encompass the full
cell cycle requires new hybrid stochastic-deterministic algorithms to be
implemented in the GPU-based Lattice Microbe software and imaging experiments.
MINFLUX microscopy will be used to provide data at the enhanced spatial and
temporal resolutions needed to construct accurate models of cell growth,
division, and DNA replication. The structural and functional characterization of
proteins of unclear function, which recent work indicates are required for
consistent morphology and division of Syn3A, are needed to extend the model and
promise to reveal novel interactions and biochemical reactions. To achieve these
goals, the cellular processes of Syn3A will be experimentally measured. The
function of unknown genes associated with cell shape and growth will be
identified. The organization of transcriptional units and MINFLUX imaging
studies of cell division and initiation of DNA replication will be determined.
The integration of new heterogeneous data into the existing whole-cell
computational models will occur through the develop and implementation growth
and cell division kinetic models. In addition, the structure of the 30 essential
proteins will be identified. Finally, the gene expression kinetic model will be
informed with measured transcriptional units to determine the overall impacts on
protein production and the cell. The planned research will be conducted with
postdocs and graduate students at the University of Illinois at Urbana-Champaign
(UIUC), Johns Hopkins University (JHU), the Synthetic Biology group at JCVI, and
collaborators at TU Dresden, Leiden University, and University of
Groningen&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.