* 2225632
* Post-transcriptional genome regulation in bacteria with next generation CRISPR-Cas tools
* BIO,MCB
* 08/15/2022,07/31/2025
* James Carothers, University of Washington
* Standard Grant
* Anthony Garza
* 07/31/2025
* USD 689,062.00

This project seeks to develop new tools to regulate bacterial gene expression
for biosynthesis. Dramatic successes in metabolic engineering have been achieved
through laborious efforts to optimize gene expression for the production of
high-value chemicals. Programmable, DNA-targeting CRISPR-Cas tools can be used
to rapidly implement complex genetic programs, but in bacteria these systems
have limitations in their ability to precisely control individual genes. To
improve the ability to precisely control cellular behavior, the investigators
will develop RNA-targeting CRISPR-Cas systems that act post-transcriptionally
and may overcome the limitations of the DNA-targeting systems. The sophisticated
control systems developed in this project will be useful for practical
biosynthesis, bacterial engineering, and basic research in bacteria. These
findings will be incorporated into educational materials and courses taught to
chemistry and engineering students. This project will also provide opportunities
for underrepresented students at the high school and undergraduate levels to
participate in laboratory research.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Bacterial
metabolic pathways perform complex chemical transformations with high
specificity to produce biosynthetic products. Introducing heterologous genes
allows metabolism to be diverted to new synthetic targets, but optimizing the
function of these engineered strains is challenging. The goal of this proposal
is to develop a new class of bacterial RNA-targeting tools to systematically
regulate multi-gene expression programs, and to identify regulatory
architectures that can improve the output of biosynthetic pathways. Previously,
DNA-targeting CRISPR-Cas transcriptional regulatory circuits have been
successfully assembled into sophisticated multi-gene regulatory programs.
Despite their enormous potential, DNA-targeting CRISPR-Cas systems have
important limitations in their ability to precisely up- or down-regulate
individual bacterial gene targets. For this project, the investigators will
create dCas13-based RNA-targeting tools, which act post-transcriptionally and
may overcome the limitations of DNA-targeting CRISPR-Cas systems. The immediate
goal of the project is to develop new capabilities for gene repression and
activation at the translational level. First, the investigators will use Agile
BioFoundry (ABF) capabilities to learn global rules for dCas13-mediated
translational regulation of gene expression and metabolic flux in bacteria.
Next, they will systematically compare dCas13-mediated translational regulation
to dCas9-mediated transcriptional control to determine whether these systems can
produce distinct functional effects. Finally, they will demonstrate the utility
of this knowledge by implementing translational control systems in Design-Build-
Test-Learn (DBTL) cycles applied to engineer aromatic biosynthesis in a non-
model microbe that can be used for industrial bioproduction. These systems will
further expand the toolkit for exploring the large space of regulatory
architectures to optimize bacterial biosynthesis.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.