* 2226183
* AI-powered multi-scale modeling of microbiome-host interactions
* BIO,DBI
* 09/01/2022,08/31/2025
* Lei Xie, CUNY Hunter College
* Continuing Grant
* Jean Gao
* 08/31/2025
* USD 521,558.00

Prediction of organismal characteristics based on genetic signatures of an
individual&lt;br/&gt;organism is a fundamental problem in biology. Recent
advancements in the field indicate that &lt;br/&gt;microbiome, a community of
microorganisms found in all environments on earth from human&lt;br/&gt;bodies to
soils, plays an essential role in determining the function of ecosystems.
Microbiome&lt;br/&gt;generates a variety of metabolites that serve as messengers
for the microbiome to communicate&lt;br/&gt;with its surroundings. It remains
unclear how these messengers communicate. This research&lt;br/&gt;intends to
decipher the molecular language of microbiome-environment interactions,
thereby&lt;br/&gt;resolving a major enigma in life sciences. Understanding the
fundamental rules governing&lt;br/&gt;microbiome-environment interactions will
facilitate the solution of pressing problems in&lt;br/&gt;agriculture,
environment, energy, and human health/well-being. By modulating the function of
the&lt;br/&gt;microbiome in plants, for instance, agriculture could become more
productive while requiring less&lt;br/&gt;pesticides and fertilizers. Utilizing
the microbiome's photosynthetic energy is a promising&lt;br/&gt;alternative
clean energy solution. Maintaining gut microbiome stability may improve
human&lt;br/&gt;immunity. This multidisciplinary research involving biology,
chemistry, and computer&lt;br/&gt;science will create opportunities for
underrepresented students in life sciences to pursue&lt;br/&gt;careers in
computer science and bioinformatics, fields in which racial and ethnic
minorities are&lt;br/&gt;woefully
underrepresented.&lt;br/&gt;&lt;br/&gt;Metabolite-protein interactions (MPIs)
play a major mechanistic role in maintaining microbiome&lt;br/&gt;communities
and mediating microbiome-host interactions. Exploration of the global landscape
of&lt;br/&gt;MPIs and the host gene expressions regulated by the MPI would fill
critical knowledge gaps in&lt;br/&gt;causal environment-genotype-phenotype
associations. However, our understanding of MPIs and&lt;br/&gt;their regulatory
pathways is limited due to the transient and low-affinity nature of MPIs.
Existing&lt;br/&gt;experimental techniques for determining MPIs and their
functional roles are time-consuming,&lt;br/&gt;biased to certain molecules, or
applicable on a relatively small scale. The rapid growth of
multiple&lt;br/&gt;omics data provides us with new opportunities to predict MPIs
across entire microbiomes and host&lt;br/&gt;genomes. However, due to noisiness,
biasness, incompleteness, and heterogeneity of omics&lt;br/&gt;data sets, their
potential has not been fully appreciated for developing accurate, robust,
and&lt;br/&gt;interpretable predictive models for MPIs and their functions. The
project will develop an&lt;br/&gt;innovative AI-powered multi-scale modeling
framework to predict biological functions of&lt;br/&gt;microbiome metabolites by
integrating diverse data from chemical genomics, structural
genomics,&lt;br/&gt;and functional genomics. Specifically, this research will
develop a novel end-to-end meta-learning&lt;br/&gt;method following a sequence-
structure-function paradigm to predict genome-wide
microbiome&lt;br/&gt;metabolite-host protein interactions, and innovative domain
adaption methods to translate cell line&lt;br/&gt;screens to host systemic
responses to microbiome metabolites by disentangling intrinsic
biological&lt;br/&gt;signals from both biological and technical confounders. An
immediate outcome will be a bespoke&lt;br/&gt;platform to infer novel biological
relationships from noisy, biased, and incomplete data, which&lt;br/&gt;will have
broad applications in addressing many fundamental biological problems. Another
far-reaching&lt;br/&gt;benefit will be the discovery of novel genetic,
molecular, and cellular mechanisms of&lt;br/&gt;biological processes. Completing
this project will provide a powerful computational framework
to&lt;br/&gt;correlate molecular interactions to physiological processes, and
establish causal environment microbiome-genotype-phenotype associations. The
results of the project can be found
at&lt;br/&gt;https://github.com/XieResearchGroup/.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.