* 2226633
* Collaborative Research: MFB: Deciphering RNA-based regulatory logic with interpretable machine learning
* BIO,MCB
* 09/01/2022,08/31/2025
* SHU-BING QIAN, Cornell University
* Standard Grant
* Wilson Francisco
* 08/31/2025
* USD 605,000.00

RNA transcripts, single-stranded ribonucleic acid products synthesized by
transcription of DNA, contain multiple, complex, and overlapping codes that
dictate their biochemical processing. Two RNA processes, RNA splicing and 5’UTR
regulation, play key roles in the fundamental transfer of information from DNA
to functional RNA and protein products. Understanding the regulatory logic of
these RNA-based codes is required for the rational design of RNA transcripts in
biotechnology. Despite decades of genetics, biochemistry, and bioinformatics
research, understanding RNA-based regulatory logic remains elusive. Recent
applications of “off-the-shelf” machine learning methods to limited datasets
have provided limited insights into the underlying regulatory logic, hindering
rational RNA transcript design. In this collaborative project, “interpretable-
by-design” machine learning algorithms that can explain how they arrive at their
predictions will be designed, deployed, and trained on massively parallel
reporter assays and interpretability will be demonstrated by experimental
validation. The project will have broader impacts through the development of
generalizable experimental and machine learning approaches that can be applied
to other biomolecular systems, its potential impact for biotechnology,
recruitment, participation, and professional development for trainees, with an
emphasis on supporting students and researchers from diverse backgrounds
underrepresented in the sciences, and development of curricula for undergraduate
students in computer science and biology. &lt;br/&gt;&lt;br/&gt;This Molecular
Foundations of Biotechnology (MFB) project is focused on two applications: (1) A
comprehensive understanding of the splicing code in determining exon skipping.
During RNA splicing, introns are removed, and exons are ligated to form the
mature RNA transcript. It is currently unknown how exactly an exon’s sequence
determines whether it would be included or not. Using a massively parallel
reporter assay dataset with &gt;350,000 constructs and analyzed using an
interpretable neural network the investigators have derived multiple insights
into the splicing code. In this project, experimental RNA binding protein
binding data will be incorporated to capture non-additive effects among sequence
features and elucidate the effect of secondary structure. (2) A comprehensive
understanding of the role of the 5’UTR code in translation initiation and RNA
stability. The 5’UTR sequence plays important roles in regulating translation
and stability, yet our understanding of this 5’UTR code is far from complete. In
this project, interpretable neural networks will be designed to decipher the
5’UTR code, based on preliminary analysis, as well as information about
microRNAs, and bidirectional 5’UTR scanning. This project will demonstrate that
interpretable machine learning can be used to decipher RNA-based regulatory
logic, a critical step forward for basic research with direct and generalizable
implications for biotechnology applications.&lt;br/&gt;&lt;br/&gt;This project
is jointly supported by the Division of Molecular and Cellular Biosciences
(MCB), the Division of Chemistry (CHE), and the Division of Information and
Intelligent Systems (IIS).&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.