* 2226663
* MFB: Novel Graph Neural Networks to Understand, Predict, and Design Allosteric Transcription Factors
* MPS,CHE
* 09/01/2022,08/31/2025
* Yao Xie, Georgia Tech Research Corporation
* Standard Grant
* Tingyu Li
* 08/31/2025
* USD 1,485,925.00

In this Molecular Foundations for Biotechnology (MFB) project, Professors Corey
J. Wilson, Matthew J. Realff, and Yao Xie at the Georgia Institute of Technology
are leveraging both novel experimental and machine learning strategies to
understand, predict, and design allosteric communication in a family of proteins
called transcription factors that regulate gene expression in living systems.
Protein allostery is an important protein function which enables communication
between different parts of a functional protein that are widely separated. Our
lack of understanding of the mechanism of allostery prevents scientist and
engineers from designing this critically important function. By combining the
tools of molecular biology and artificial intelligence, this project aims to
decipher structure/activity patterns for naturally occurring and engineered
transcription factors at the molecular level, specifically at the level of
individual amino acids. Understanding the rules that govern allosteric
communication would, in principle, enable investigators to design new
transcription factors for a variety of high-impact applications such as
manipulating the composition of bacteria in the gut. This project involves a
blend of biochemistry, biophysics, engineering, and machine learning approaches
that will facilitate student engagement across traditional disciplinary
boundaries. In addition to diverse student involvement, the broader impacts of
this project will include the development of innovative pedagogical modules in
the areas of machine learning and biological engineering. This project will
contribute to the development of a diverse and engaged STEM (science,
technology, engineering and mathematics) workforce, building a firm foundation
for a lifetime of contributions to research, education, and their integration.
&lt;br/&gt;&lt;br/&gt;Protein allostery is a vitally important protein function
that has proven to be a vexing problem to understand at the molecular level. The
goal of this project is to decipher the underlying molecular mechanisms by which
the allosteric signal traverses the scaffold across several naturally occurring
and engineer transcription factors with alternate allosteric controls from the
broader LacI/GalR family of protein homologues. In general, allosteric
communication involves networks of non-neighboring amino acid positions;
therefore, traditional pairwise computational approaches (e.g., molecular
mechanics simulations, and related computer-aided protein design strategies) are
of limited use in understanding and designing allosteric networks a priori.
Accordingly, this project seeks to develop novel machine learning approaches and
complementary experimental strategies to accelerate scientific progress and
transform the nature of studying and designing allosteric communication. This
project has the potential to lead to a paradigm shift with regard to the origins
and construction (design) of allosteric networks in a single fold. Moreover, the
machine learning approaches developed in this project can in principle be
applied to other complex network problems beyond the designated protein systems
â€“ e.g., distillation column sequences, communication systems, and power grid
systems.&lt;br/&gt;&lt;br/&gt;This project is jointly supported by the Division
of Chemistry, the Division of Chemical, Bioengineering, Environmental and
Transport Systems, and the Division of Information and Intelligent
Systems.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.