* 2226816
* MFB: Targeting the Dark Proteome by Machine-learning-guided Protein Design
* MPS,CHE
* 09/01/2022,08/31/2025
* Sagar Khare, Rutgers University New Brunswick
* Standard Grant
* Catalina Achim
* 08/31/2025
* USD 1,500,000.00

Drs. Sagar Khare, Jean Baum, Adam Gormley, Guillaume Lamoureux, and Sijian Wang
from Rutgers University will develop targeted protein editors guided by novel
machine learning (ML) approaches. The ability to precisely edit genomes has
transformed modern biotechnology and medicine; however, technology for the in-
situ precision editing of proteins – the workhorses of biology – has been
lacking. Targeting with such methods proteins contain functionally-important but
structurally-disordered segments would be particularly useful but it is
challenging. The interdisciplinary research team will develop and apply novel
machine learning algorithms to the design of new editor proteins that contain a
binding domain and an enzymatic domain to ensure the selective recognition and
modification, respectively, of a chosen target protein in the presence of
thousands other proteins present in the cell. Iterative Design-Build-Test-Learn
cycles will involve a close interplay of biochemical experiments and model
development to obtain robust, generalizable and interpretable ML models that
predictively enable selective protein editing. The project lies at the interface
of chemistry, biophysics, robotics, computer science, and statistics, and will
therefore offer unique training and research opportunities for students from
both quantitative and biological and chemical science backgrounds. Teams of
undergraduates (recruited from various Research Experiences for Undergraduates
programs at Rutgers) will use gamified versions of computational protein design
algorithms to develop and refine protein editors. Hands-on protein design
sessions will be held at affinity group conferences to reach students from
groups underrepresented in STEM and to present them with opportunities to learn
about, and participate in, research.&lt;br/&gt;&lt;br/&gt;New ML approaches for
protein modeling have recently demonstrated that sequence and structure data can
be leveraged to correctly learn complex sequence-structure relationships and
design novel proteins. However, abundant functional and biophysical data, e.g.,
protein-peptide binding data, remain largely unexploited. New ML models that go
beyond sequence representations and towards semantically richer representations
based on molecular structure are needed. These representations can capture more
specific sequence-function and sequence-energetics relationships. The research
team will use a Design-Build-Test-Learn pipeline driven by ML models that are
trained on both experimental data and molecular structure and energetics. The
team will apply these methods to the design of PDZ domains that bind with high
affinity and specificity to the C-termini of target intrinsically disordered
proteins and selectively cleave at an internal sequence site in the targeted
protein. ML models will be pre-trained on available sequence and experimental
biophysical data on PDZ and protease domains and used for generating new
proteins. Design validation and iterative improvements to the ML models will be
carried out by experimental characterization using high-throughput assays, deep
sequencing, biomolecular NMR and robotics-enabled assays of binding and
cleavage. Successful designed proteins will be fused to obtain PDZ-protease
protein editors. The developed methodology will be general and broadly
applicable to allow in situ protein editing of a variety of biologically
relevant targets, especially with intrinsically disordered proteins or regions.
The novel reagents will help illuminate the so-called “dark” proteome.
&lt;br/&gt;&lt;br/&gt;This project is supported by the Division of Chemistry
(CHE) and the Division of Mathematical Sciences (DMS) in the Mathematics and
Physical Sciences (MPS) Directorate and by the the Division of Information and
Intelligent Systems (IIS) in the Computer and Information Science and
Engineering (CISE) Directorate at the National Science
Foundation.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.