* 2239526
* CAREER Exploiting missense histone mutations to explore mechanisms of gene regulation
* BIO,MCB
* 02/01/2023,01/31/2028
* Jennifer Spangle, Emory University
* Continuing Grant
* Stephen DiFazio
* 01/31/2028
* USD 558,912.00

Each cell within an organism contains identical genetic information, but the
precise regulation of this information dictates gene expression and biological
function. Part of regulating gene expression is accomplished by the packaging of
the genome into chromatin. The nucleosome forms the repeating chromatin unit, a
complex of DNA and two copies each of histones H3, H4, H2A, and H2B. Histone
variants and histone posttranslational modifications (PTMs) are key regulators
of nucleosome accessibility and chromatin remodeling, supporting changes to gene
expression. This gene regulation is fundamentally important to how organisms
function and respond to environmental cues and is highlighted by the prevalence
of histone mutations associated with human disease. This project will leverage
computational and wet-lab research with yeast and human model systems to examine
mechanisms by which naturally occurring histone mutations dysregulate gene
expression. This work will advance our understanding of the role of histone
variants in nucleosome integrity, chromatin modification, and gene expression.
The project itself provides opportunities to engage a diverse group of
scientific trainees as part of “Team Histone,” assembled of high school,
undergraduate, graduate, and post-doctoral trainees to build scientific agency
and provide research and educational experiences to underrepresented groups
within STEM fields. &lt;br/&gt;&lt;br/&gt;Genetic studies demonstrate the
importance of conserved and essential histone amino acids for maintaining
nucleosome structure and function as well as proper regulation of gene
expression. However, how most missense histone mutations impact histone
structure and function is currently unknown. This project identified a series of
recurrent histone missense mutations in human disease. In silico modeling
suggests that some of these histone mutations impart local structural changes
that alter transcription, and studies in yeast have defined diverse growth and
cellular phenotypes. The major aims of this project are to use mammalian and
yeast models, along with molecular, cellular, and computational biology to
define the mechanism(s) by which the changes alter histone/nucleosome function
and delineate the biological outcome of mutant histone expression in model
organisms and cells. By exploiting histone mutations to explore histone protein
function, nucleosome remodeling, and regulation of gene expression, this project
will broadly advance the understanding of how histone proteins contribute to the
maintenance of appropriate nucleosome structure and function, to contribute to
cell and organismal fitness.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.