* 2240098
* CAREER: Parallel evolution in the microbiome: novel methods for detecting repeated, rapid adaptations in metagenomic data
* BIO,DBI
* 01/15/2023,12/31/2027
* Nandita Garud, University of California-Los Angeles
* Continuing Grant
* Reed Beaman
* 12/31/2027
* USD 776,213.00

Microbiomes experience an influx of new mutations daily, some of which are known
to confer essential phenotypes to their hosts or environments. Despite their
importance, the vast majority of the adaptive mutations in the microbiome are
undetectable, due in large part to the limited power of current approaches,
which focus on detecting extreme allele frequency changes in a single sample.
These approaches miss mutations that experience subtle frequency changes due to
complex forces like pleiotropy and spatial structure. However, the microbiome
has attributes that might permit us to understand the importance of these
mutations even though they cannot be detected by classical approaches: namely,
mutations are so frequent that the same mutation can often be found in the
microbiomes of different hosts. Some mutations display dynamics characteristic
of adaptation, rising in frequency in parallel across multiple host microbiomes.
This parallelism is unlikely to occur by chance, and is indicative of
adaptation. Development of new statistical methods that leverage data from many
individuals simultaneously may have significantly more power to detect a much
broader range of adaptations than have been found by identifying extreme allele
frequency changes in one host at a time, hence unlocking the potential to
uncover essential adaptive phenotypes that are currently unexplored. Via
research and educational efforts at the undergraduate and graduate levels,
students will acquire interdisciplinary skills from biology, computer science,
and statistics to perform evolutionary genomics research in the
microbiome.&lt;br/&gt;&lt;br/&gt;The project will develop new statistical
methods to quantify the pace and targets of adaptation in the microbiome.
Specifically, the project will develop statistical methods to (1) identify
adaptive single nucleotide variants via parallelism across hosts, (2) identify
adaptive gene gain and loss events via parallelism, and (3) detect adaptive
events that have spread across multiple host microbiomes. To demonstrate their
utility, the methods will be applied to several temporally sampled datasets from
humans to obtain insights into the targets and pace of adaptation in
microbiomes. Finally, a wholistic approach for recruitment, training and
retention of underrepresented STEM students will be piloted via educational
opportunities at the undergraduate and graduate levels. Results from the project
will be made available at https://garud.eeb.ucla.edu/nsf-
career/.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.