* 2243562
* Ideas Lab: Collaborative Research: Integrating cross-kingdom lncRNA genetic and functional interactions to build predictive network models
* BIO,DBI
* 04/15/2023,03/31/2026
* Karen McGinnis, Florida State University
* Standard Grant
* Jean Gao
* 03/31/2026
* USD 1,325,000.00

Although most well-known genes in cells include instructions for creating
proteins as a functional product via an RNA intermediate, scientists are
discovering an increasing number of genes that create non-protein coding RNA
that are essential for the health and life cycle of organisms. Long non-coding
RNAs (lncRNAs) are relatively recently recognized as important, and the exact
role of most remains poorly understood. Detailed study of each lncRNA would
provide essential information about possible functions, but would be time,
labor, and resource intensive. Computational analyses can provide an efficient
way to analyze many different molecules simultaneously, classify the molecules
based on different shared features, and use these classifications and other
information to predict function. This project will develop and use computational
tools to integrate many different data types generated by previous studies and
predict functions of lncRNAs in plant and animal cells. These predictions will
allow for hypotheses about how the loss or increase of specific lncRNAs might
impact a cell or whole organism. These hypotheses will then be tested through
experimentation as a way to assess the efficacy of the predictions. Project
resources will also support public education activities, K-12 teacher training
on these emerging scientific topics, and student participation in modern
computer and molecular genomics research.&lt;br/&gt;&lt;br/&gt;A central
hypothesis of this project is that a multimodal, causal network analysis
approach, constrained by physiological and molecular data from data-rich
experiments, can be utilized to functionally annotate lncRNAs and to build a
predictive model for future experiments. The ultimate goal of this project is to
develop network-based models that can predict lncRNA function based on genetic,
molecular, and genomic interactions, which can then be applied to datasets from
any species. Through this award, researchers will construct and test networks
that depict and predict lncRNA functions. Publicly available high-throughput
sequencing from human cell lines, associated -omics data, and literature-based
functional information will be used to infer a fully directional gene-regulatory
network (GRN). Machine learning models will be applied to predict essential
lncRNAs based on network topologies and connectivity. A similar approach will be
applied to available datasets in the model plant, Arabidopsis thaliana.
Predictions from both networks will be tested by manipulating ncRNA expression
and assaying the impact on the network using Nanostring
technology.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.