* 0108231
* Phylogenetics of Diploid and Allopolyploid Hibiscus Sect. Furcaria (Malvaceae)
* BIO,DEB
* 09/01/2001,08/31/2003
* Randall Small, University of Tennessee Knoxville
* Standard Grant
* James E. Rodman
* 08/31/2003
* USD 99,992.00

0108231 Small One of the primary foci of the field of plant systematics is to
understand the genealogical relationships among present day species, and to
understand the processes that gave rise to those species. This project is
focused on a group of species in the genus Hibiscus in the plant family
Malvaceae (the Mallow family). This group of Hibiscus, taxonomically classified
as Hibiscus section Furcaria, includes over 100 species that are distributed
worldwide. Ten of these species are known to be diploid (having the "normal" two
sets of chromosomes), while the remainder are polyploid having four, six, eight
or ten sets of chromosomes. These polyploids are hypothesized to have originated
through hybridization between two different species followed by chromosome
doubling in the offspring. In this group the polyploid species are more numerous
than the diploid species, perhaps due to the genetic diversity provided by the
extra sets of chromosomes. The goal of this project is to first elucidate
relationships among the diploid species in this group. Subsequently selected
polyploid species will also be analyzed - each polyploid will contain genes from
each of its parents, some of which are likely to be diagnostic, which will allow
the identification of the hybridizing parents of each polyploid species.
Ultimately these data will be used to address the classification of the species
into taxonomic groups, to study the geographic patterns inherent in those
relationships, and the evolution of morphological characters. The data to be
employed in these analyses include nuclear ribosomal genes and chloroplast gene
markers, as well as DNA sequences from nuclear-encoded genes of the alcohol
dehydrogenase gene family. Molecular biology has provided researchers access to
a wealth of genetic data, in particular DNA sequences, for analyzing
relationships among species. Plant systematists have relied primarily on
sequences from two sources: chloroplast DNA and nuclear ribosomal DNA, both of
which are genetically tractable and easy to isolate. There is a vast resource of
DNA sequences that have remained relatively untapped, however, in the rest of
the plant nuclear genome. These types of sequences have been largely ignored by
systematists primarily because they are more genetically complex and thus more
difficult to isolate and characterize. This additional experimental effort,
however, can be outweighed by the quantity and quality of data available from
these sequences. Previous work by Dr. Randall Small on cotton and related
species (genus Gossypium, also in the Mallow family) has shown that these types
of sequences can be reliably isolated and can be exceptionally useful for
systematic studies. These studies were expedited by the wealth of knowledge of
cotton genetics because of its importance as a crop plant. To facilitate the
transfer of such knowledge from well-known groups like Gossypium to less well
characterized wild species groups (such as Hibiscus section Furcaria) the
current project will use an understanding of the alcohol dehydrogenase gene
family developed in Gossypium to address species relationships Hibiscus section
Furcaria. Thus, this project provides a direct test of the transferability of
information gleaned from a genetically well characterized group (Gossypium) to a
less well understood group (Hibiscus section Furcaria).

