* 0108955
* Function and Regulation of the Bradyrhizobium Japonicum nolA Gene
* BIO,MCB
* 02/01/2002,01/31/2005
* Gary Stacey, University of Missouri-Columbia
* Standard Grant
* Joanne Tornow
* 01/31/2005
* USD 403,572.00

Recent research has identified several large families of prokaryotic global
regulators. Results from the PI's laboratory have shown that members of the
LysR, two-component, and MerR families of transcriptional regulatory proteins
regulate Bradyrhizobium japonicum nod gene expression. Regulation of nod gene
expression in B. japonicum may represent an interesting and useful model system
in which to explore the interactions between these global regulatory systems.
The focus of this project is the MerR orthologue, NolA. Recent results show that
this protein is a central player in the repression of nod gene expression in
response to a variety of novel signal molecules. NolA mediates its repressive
effects by induction of NodD2, which represses nod gene expression. Results from
the PI's laboratory indicate that NolA expression is responsive to the level of
intracellular, tetrameric Nod signal produced in B. japonicum. Therefore, NolA
is a component of a unique feedback, regulatory loop that controls nod gene
expression. Nol1A expression is induced by a cell density factor (CDF, a
glycosylated flavonoid) and a plant-produced signal that is destroyed by
chitinase treatment. The objectives of this research project are to 1. confirm
the chemical structure of the CDF, explore its activity, and survey other
bacteria for the presence of such a novel, quorum sensing molecule; 2. identify
other genes involved in NolA action using a combination of genomic and mutagenic
approaches and 3. isolate and chemically characterize the chitin-like plant
inducer of nolA expression. These studies should clarify the role of NolA in nod
gene expression. The results obtained, when added to other on-going research,
should give a more complete picture of nod gene expression in B. japonicum and
may serve as a paradigm for other complex regulatory systems where global
regulatory networks impinge on the expression of a single operon. The B.
japonicum-soybean symbiosis is an important model system for exploring the cell
signaling pathways between host and symbiont. The ultimate goal of this work is
a clearer understanding of the integration of regulatory mechanisms involved in
the global regulation of cellular metabolism.