* 0110004
* Linking Rice, Arabidopsis and the Grasses to Explore Natural Genetic Variation
* BIO,IOS
* 09/01/2001,08/31/2006
* Ray Wu, Cornell Univ - State: AWDS MADE PRIOR MAY 2010
* Standard Grant
* Diane Jofuku Okamuro
* 08/31/2006
* USD 1,368,685.00

To date, the map-based cloning and characterization of genes from crop species
has largely been limited to monogenic traits, such as those providing disease
resistance, that are controlled by single genes causing clear phenotypic
effects. Most agronomically-important traits, such as yield and yield
components, however, are polygenic traits controlled by multiple genes and
influenced by the environment. Over the past decade, the application of
molecular markers and genetic linkage maps has allowed specific regions of the
genome controlling polygenic traits, called quantitative trait loci (QTLs), to
be identified in all of the major crop species. In spite of the wealth of
positional information for these regions, however, few genes underlying
quantitative traits have actually been cloned. As a result, many questions
concerning the nature and function of these genes, as well as the molecular
mechanisms behind the genetic variation of agronomically-important traits,
remain unanswered. The genes underlying QTLs derived from a wild rice ancestor
(O. rufipogon) that are associated with natural variation for flowering time,
grain weight, and plant height in elite cultivated material will be identified
and characterized. Three QTLs that map to rice chromosome 1, one of the first
rice chromosomes to be completely sequenced, will be targeted. The specific
objectives are to fine-map major QTLs for flowering time, grain weight, and
plant height, then to isolate candidate genes that are located within those
regions and, finally, to demonstrate phenotypic complementation via
transformation with genes underlying the QTLs. The sequence variation from
orthologs of the genes underlying the target QTLs in wild and cultivated Oryza,
other grasses, and Arabidopsis will be compared to gain an evolutionary
perspective and insight into functionally-important conserved domains. This work
will generate significant new information about the identity and functional
significance of genes associated with agronomically-important traits in the
grasses and will test a multi-faceted strategy for rapid identification of genes
underlying QTLs when genomic sequence is available in the target region.