* 0110675
* Probing Chromosome Structure with Phage Mu
* BIO,MCB
* 11/15/2001,10/31/2004
* Norman Higgins, University of Alabama at Birmingham
* Continuing Grant
* Patrick P. Dennis
* 10/31/2004
* USD 328,791.00

Two related methods have been developed to probe the structure of bacterial
chromosomes inside living cells. One technique is called Muprinting, and it uses
PCR reactions to generate a high-resolution picture of the transposition target
sites in a bacterial population infected with phage Mu. By comparing the PCR
profiles of naked DNA in vitro and protein bound DNA in vivo, it is possible to
detect the location of sequence-specific DNA binding. Proteins detected with
Muprinting include repressors and RNA polymerases. A new method recently
developed called Mu-screening allows the identification of the transposition
targets that are used efficiently in vivo during replicative Mu transposition.
It is now possible to identify hot spots and cold spots for transposon
insertion, and to follow changes in these structure when cells are placed in
different physiological environments. Two primary objectives will be pursued.
First, Mu-screening will be used in conjunction with gene micro-arrays
containing 4115/4238 ORFs in E. coli to identify the most efficient
transposition targets in the genomes of E. coli and Salmonella enterica serovar
typhimurium. Second, Mu transposition hotspots will be cloned and subjected to
in vivo and in vitro Muprint analysis to precisely identify the regions where
proteins are bound in living cells. Ultimately, the proteins bound to Mu
hotspots will be characterized by biochemical and genetic methods. New methods
to systematically investigate chromosome structure inside living cells are
required to address the challenging problems in modern molecular evolution. With
the stunning advances in the ability to efficiently acquire entire sequences of
virtually any organism, new methods to understand chromosome function and
structure are now urgent. Information gained from Mu-screening and Muprinting
technology promises to provide insight into the area of chromosome evolution and
into the question of structure in bacterial chromatin. Micro-array-driven
structure analysis complements gene expression studies and should stimulate new
genetic and biochemical models of chromosome function.