* 0112896
* ITR/AP: Computational Analysis of Proteins: From Structure to Sequence to Function
* BIO,EF
* 10/01/2001,09/30/2004
* Alexander Tropsha, University of North Carolina at Chapel Hill
* Continuing Grant
* Kamal Shukla
* 09/30/2004
* USD 301,619.00

Accurate determination of protein structure and function is the focus of
proteomics, the next step in the genomic revolution. Traditional knowledge
discovery in molecular biology has followed a research paradigm corresponding to
the information flow in functional genomics, that is from gene sequence to
protein structure to function. This computational bioinformatics proposal
exploits the information content of the protein structural and sequence
databases following a research paradigm formulated in the title of this
proposal,&lt;br/&gt;namely from structure to sequence to function prediction.
The important postulate of this research is that spatial nearest neighbor
residue motifs defined by the means of computational geometry (Delaunay
tessellation) provide unique determinants of protein structure and function. The
major goals of this proposal include: (1) development and validation of
multibody statistical potentials on the basis of non-redundant subsets of the
protein crystallographic database, (2) prediction of 3D protein structure by the
means of novel implementation of the chain growth algorithm using Monte Carlo
sampling, statistical potentials, and spatial constraints defined by specific
residue motifs, and (3) determination of sequence and structure specific amino
acid residue motifs for known proteins and large scale prediction of protein
structural and functional classes for genomic sequences. The successful
implementation of this proposal will have a broad impact on functional genomics
and proteomics.