* 0113459
* RUI:  The Role of AlgZ in the Twitching Motility of Pseudomonas Aeruginosa
* BIO,MCB
* 09/01/2001,09/30/2004
* Patricia Baynham, Thomas More College
* Continuing Grant
* Patrick P. Dennis
* 09/30/2004
* USD 240,000.00

Pseudomonas aeruginosa and some other common soil bacteria utilize a flagella-
independent mode of movement called twitching motility (also called gliding
motility). P. aeruginosa is an excellent model organism in which to study this
form of motility due to its advanced genetics and newly completed genome
sequence database. In twitching motility, bacteria use fibrous polar appendages
(type 4 pili) to move across a surface. It is thought that the basis of this
movement is the extension and retraction of these pili. Twitching motility may
be assayed by stab-inoculation of an agar plate and then observation of
spreading growth of the bacteria at the plate-agar interface. Dr. Cynthia
Whitchurch, a collaborator on this project, has performed video microscopy of P.
aeruginosa twitching motility. This may be viewed at
http://www.cmcb.uq.edu.au/cmcb/PUBS/twitch.html. Twitching motility is highly
complex and involves dozens of genes at different chromosomal loci. Although
many genes required for twitching have been identified, an understanding of the
mechanism and regulation of twitching motility is elusive. A gene first
identified as a regulator required for alginate production, algZ, is required
for twitching motility. algZ encodes a ribbon-helix-helix DNA binding protein
that was first identified and cloned by the principle investigator. The deletion
of algZ in an environmental P. aeruginosa isolate results in a loss of twitching
motility. The goal of this project is to define the role of algZ in twitching
motility. These studies will define the phenotype of the algZ deletion strain.
The strain will be analyzed to determine whether the defect appears to be in
pili expression, processing, export, or function. Electron microscopy, whole
cell ELISA, phage sensitivity, and video microscopy will be used. Biochemical,
genetic, and genomic approaches will identify genes that are candidates for
algZ-dependence and involvement in twitching motility. The biochemical approach
involves a modified SELEX (systematic evolution of ligands by exponential
enrichment) technique utilizing purified AlgZ and genomic DNA. A transposable
promoter probe in an algZ-inducible strain is used in the genetic approach. The
genomic approach involves the determination of the consensus for AlgZ binding to
DNA (via DNA footprinting and mutagenesis) and then interrogation of the P.
aeruginosa genome database for possible AlgZ targets. The candidate gene(s) will
then be tested to confirm algZ-dependence by construction of a candidate gene
fusion(s) in an algZ inducible strain. The expression of the candidate gene(s)
may then be monitored in response to algZ induction. Lastly, studies will be
undertaken in which the expression or interruption of a candidate gene(s) (or
some combination of these) will be able to restore the algZ deletion strain to a
twitching phenotype. This will confirm that all algZ-dependent genes involved in
twitching motility have been identified. These studies will elucidate the
genetic control of twitching motility. Undergraduate students will be involved
in all aspects of these studies. This will not only increase current knowledge
regarding the coordinate regulation of alginate production and twitching
motility, but also the ways to better use and control environmental bacteria
with regard to bioremediation.