* 0114866
* Arabidopsis 2010:  Essential Gene Functions in Arabidopsis Seed Development
* BIO,IOS
* 09/01/2001,08/31/2007
* Allan Dickerman, Oklahoma State University
* Continuing Grant
* Chris T. Amemiya
* 08/31/2007
* USD 2,330,000.00

0114866&lt;br/&gt;Meinke &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;PROJECT
SUMMARY&lt;br/&gt;&lt;br/&gt;This project will establish a foundation for
identifying every Arabidopsis gene with an essential function during seed
development. This information is central to the Arabidopsis 2010 Project and to
identifying the minimal gene set required for growth and development of a model
plant. The project will focus on genes that give a visible seed phenotype when
disrupted by mutation. The network of genes chosen for analysis is therefore not
defined by shared biochemical or molecular features but rather by the fact that
they are essential. Arabidopsis contains an estimated 500 such EMB genes
required for normal seed development and another 200 genes required for normal
seed pigmentation. The approach outlined here is to coordinate the collection,
analysis, and presentation of information on more than half of these genes based
on cloning of mutant alleles. Information on essential genes and the
corresponding mutants will be obtained from a variety of public and private
sources and synthesized for efficient use by the community. The resulting
insights into biological functions of these diverse genes will complement
research in other labs on the biochemical activities of specific gene products.
The result will be an integrated view of essential gene functions within a
cellular, organismal, and evolutionary context. &lt;br/&gt;&lt;br/&gt;Project
objectives are to: (1) use a combination of forward and reverse genetics to
approach saturation for cloned EMB genes; (2) standardize phenotypic
characterization of the corresponding mutants; (3) collect related information
on seed pigment mutants; (4) make seeds for all mutant lines available through
ABRC; (5) understand the functions of these genes in growth and development; (6)
determine through expression studies and comparative sequence analysis why these
genes are essential; (7) integrate this information into a simple but robust
database with a format compatible with TAIR; (8) establish a model for public
release of private sector resources; and (9) develop outreach and training
programs that complement research efforts in Arabidopsis functional genomics.
Project deliverables include public access to synthesized information and seed
stocks for 500 mutants defective in 300 different EMB genes; similar information
for another 100 pigment mutants defective in 75 genes; valuable expression data
for genes active in young seeds (wild type and selected mutants); and a database
that should serve as a model for analyzing and presenting information on large
collections of mutants. The project web page (www.seedgenes.org) will be linked
through TAIR (www.arabidopsis.org). Release of the first version of the database
is scheduled for March 2002. Profiles of the first 100 essential genes examined
in detail will be presented at that time. &lt;br/&gt;&lt;br/&gt;This project
builds upon an existing collaboration between Oklahoma State University and
Syngenta that has over the past 4 years resulted in identification of 350 tagged
emb mutants and preliminary sequence data for about 200 of these mutants. This
represents a significant corporate investment into a project that would be
difficult and expensive to duplicate with federal funding. The current proposal
includes substantial public release of private materials with minimal
restrictions and provides a mechanism for collecting and analyzing community-
wide information on essential genes. Such efforts are needed at the start of the
2010 Project to focus attention on genes with important biological functions.
&lt;br/&gt;&lt;br/&gt;&lt;br/&gt; &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;