* 0114783
*  Arabidopsis 2010: Expression Profiling of Plant Disease Resistance Pathways
* BIO,MCB
* 09/01/2001,08/31/2006
* Shauna Somerville, Duke University
* Continuing Grant
* Eve Ida Barak
* 08/31/2006
* USD 3,765,723.00

&lt;br/&gt;&lt;br/&gt;Plants respond to pathogen attack through a variety of
signaling pathways consisting of a large number of regulatory as well as
effector genes. During the past several years, many defense-related genes have
been identified through genetic analysis conducted in Arabidopsis thaliana.
Importantly, Arabidopsis exhibits all of the major kinds of defense responses
present in other plants. Although a relatively large number of Arabidopsis
defense-related genes have been identified, progress in developing overall
explanatory models of plant-pathogen interactions is currently limited by two
major experimental roadblocks. First, most of the phenotypic tests that have
been used to characterize pathogen-host interactions do not have sufficient
discriminatory power to assign defense-related genes to specific signal response
pathways. To circumvent this limitation, large-scale transcript profiling
analyses will be carried out on appropriately selected Arabidopsis defense-
related mutants, including double and triple mutants. Combining genetic
epistasis analysis with genomic technologies should lead to the development of a
much more detailed model of how the various defense-related genes function and
interact in combating pathogen attack. The second factor limiting the
understanding of the plant defense response is that it is simply not possible to
analyze the overwhelming volume of current data using conventional methods. The
large volume of microarray data that will be generated in the near future
compounds this problem. Moreover, data from different laboratories are not
directly comparable because standardized experimental conditions are not
employed. To help mitigate the problems associated with the analysis of large
data sets, a sophisticated web-accessible plant-microbe interaction database
(PMIDB) will be created to provide a common repository and standardized format
for experimental data. The specific aims of the project are
to:&lt;br/&gt;&lt;br/&gt;1) Use transcript-profiling analysis to identify
Arabidopsis defense-related genes and construct a &lt;br/&gt;custom microarray
(pathoarray) enriched for defense-related genes (A list the genes identified so
far can be found at
http://genetics.mgh.harvard.edu/ausubelweb/nsf2010/NSF_2010.html). These custom
pathoarrays will be made available to the Arabidopsis community at a nominal
cost with the expectation that the experimental results generated using these
pathoarrays will be deposited in PMIDB.&lt;br/&gt;&lt;br/&gt;2) Use the
pathoarrays from Aim 1 and Arabidopsis defense-related mutants to define the
expression signatures resulting from the activation of defense pathways.
&lt;br/&gt;&lt;br/&gt;3) Create a web-accessible plant-microbe interaction
database (PMIDB). This database, which will be accessible at
http://genetics.mgh.harvard.edu/ausubelweb/nsf2010/NSF_2010.html, will be
developed during the next four years and will contain standardized experimental
procedures for analyzing host defense responses, a list of all the pathogenesis-
related mutants and their phenotypes, a list of defense-related genes with links
to various sequence databases, and expression profiles of different plant-
pathogen interactions and different defense-related mutants.
&lt;br/&gt;&lt;br/&gt;This 2010 Project provides a unique opportunity for
applying genomic approaches to genetic analysis of plant defense mechanisms.
Understanding the mechanisms of plant defense is of interest not only to basic
science but also to development of agriculture and protection of the
environment. &lt;br/&gt;&lt;br/&gt;