* 0114992
* Arabidopsis 2010:  Determination of Biological Functions of the NPH3/RPT2 Family
* BIO,IOS
* 09/01/2001,08/31/2006
* Emmanuel Liscum, University of Missouri-Columbia
* Continuing Grant
* Michael Mishkind
* 08/31/2006
* USD 900,000.00

Because of their sessile nature, plants have evolved a wide range of strategies
to respond to changes in their environment. For a number of these responses
plants have co-opted (or co-evolved) systems that are shared with other taxa.
However, many of the environmental (both biotic and abiotic) response systems
plants utilize are apparently specific to plants and are often comprised of
novel molecular components. With the recent completion of the Arabidopsis genome
sequencing project we are now in a unique position to ask fundamental questions
about the functions of these latter set of regulatory proteins in plants. Of the
~11,000 protein families formed by the 25,498 predicted genes of Arabidopsis,
only ~150 families appear to be plant-specific. One of these plant-specific
families is the NPH3/RPT2 family. This family, which is comprised of 32 novel
proteins (for more information about these proteins/genes, including GenBank
accession numbers, see
http://www.biosci.missouri.edu/liscum/nph3-rpt2figs.html), was first identified
with the cloning of NPH3 (NONPHOTOTROPIC HYPOCOTYL 3) and RPT2 (ROOT
PHOTOTROPISM 2) - genes identified in screens for mutants with altered
phototropic responses. NPH3 and RPT2 function in early phototropic signaling and
appear to act as modular scaffold proteins to recruit or activate enzymatic
components of the transduction chains, including the respective photoreceptors
themselves, mediating phototropic responses. Given the high degree of similarity
in primary sequence and secondary structure between all members of the NPH3/RPT2
family it seems appropriate to hypothesize that these plant-specific proteins
may act as scaffolds for sensor-containing signaling complexes utilized in a
variety of abiotic and/or biotic responses. The major goals proposed for this
project are to determine mRNA and protein expression patterns, and biological
functions for each member of the NPH3/RPT2 family. The expression studies are
proposed as a means of developing logical and focused analyses of loss- and
gain-of-function mutants. In such a way we expect to be able to determine what
biological processes require the function of each member of the NPH3/RPT2
family. All project data will be publicly available through a project database
(http://www.biosci.missouri.edu/liscum/NPH3-RPT2family.html), as well as The
Arabidopsis Information Resource (TAIR) (http://www.arabidopsis.org/home.html).
Given the plant-specific and novel nature of the NPH3/RPT2 family, it is likely
that elucidation of biological functions for the family will represent an
important step in understanding fundamental aspects of plant growth and
development.&lt;br/&gt;&lt;br/&gt;