* 0114666
* Arabidopsis 2010: Collaborative Project on the Functional Genomics of Arabidopsis beta-Glucosidase and beta-Galactosidase Gene Families
* BIO,MCB
* 09/15/2001,08/31/2005
* Ming-Che Shih, University of Iowa
* Continuing Grant
* Parag R. Chitnis
* 08/31/2005
* USD 915,000.00

Widely distributed in animals, plants and microbes, O-glycoside hydrolases (EC
3.2.1.-) are enzymes that catalyze the cleavage of chemical bonds between two or
more carbohydrates or between a carbohydrate and a non-carbohydrate moiety.
Responding to the challenge of the 2010 Project to identify the function of all
Arabidopsis thaliana genes within the next decade, this collaborative research
will focus on approximately 75 members of two related families of glycoside
hydrolases (http://www.biol.vt.edu/faculty/esen/glycosidase.lab.html and
http://afmb.cnrs-mrs.fr/~pedro/CAZY/). Family 1 includes beta -glucosidases (EC
3.2.1.21) and myrosinases (EC 3.2.3.1), which function in higher plants in
chemical defense against herbivores and pathogens, lignin biosynthesis, and
plant growth and development. Family 35 contains the beta -galactosidases (EC
3.2.1.23), which play key roles in fruit ripening, flower senescence,
mobilization of carbohydrate reserves, and galactolipid turnover. To date, the
precise biochemical roles of only three of these Arabidopsis hydrolases are
known with certainty. &lt;br/&gt;&lt;br/&gt;The purpose of this
multidisciplinary collaborative research is to assign biological functions to
beta-glucosidases and beta-galactosidases encoded by the Arabidopsis genome.
After using phylogenetic analysis to identify subfamilies that contain closely
related enzymes, cDNAs encoding each target hydrolase will be obtained. Each
hydrolase will then be overexpressed in foreign host (e.g. yeast and bacteria)
cells and purified to ascertain its biological function by measuring its
enzymatic activity toward a wide range of natural glycosidic substrates isolated
primarily from Arabidopsis and related crucifers for this purpose. In parallel
studies, three-dimensional structures of selected subfamily representatives will
be determined by homology modeling and x-ray diffraction, providing novel
insights into how these hydrolases recognize and bind their substrates. This
information will be of paramount importance in future research to alter the
substrate specificity of Family 1 and Family 35 hydrolases for biotechnological
purposes, including biomass conversion and improvements in anti-herbivore
defenses and fruit ripening.&lt;br/&gt;