* 0115109
* Arabidopsis 2010:  Functional Genomics of Quantitative Traits: Expression Level Polymorphisms of QTLs Affecting Disease Resistance Pathways in Arabidopsis
* NONE,NONE
* 09/01/2001,08/31/2006
* Richard Michelmore, University of California-Davis
* Continuing grant
* Susan Porter Ridley
* 08/31/2006
* USD 2,354,230.00

Arabidopsis 2010: Functional Genomics of Quantitative Traits. Quantitative
differences in the expression of genes involved in disease resistance responses
will be investigated using a functional genomics approach that involves a novel
application of quantitative trait locus (QTL) analysis to microarray data.
Regulatory QTLs controlling natural variation in induced gene expression
patterns (i.e., expression level polymorphisms, ELPs) through QTL analysis of
microarray data for ELPs from genetically segregating populations will be
identified. Dissection of regulatory networks using genetic analysis of natural
allelic variation will provide an efficient method for searching for regulatory
loci at the systems biology level and avoids unnatural traumatic perturbations
to gene regulation that are caused by extreme mutations. QTL dissection of
natural variation is complementary to mutant analysis as it is likely to reveal
different aspects of the regulatory network controlling disease resistance than
mutant analysis because qualitatively inherited resistance genes do not account
for all the aspects of complex pathways. This project aims to: 1) develop
integrated molecular and statistical approaches for the dissection of
quantitatively inherited traits, 2) determine if expression level polymorphisms
involved in the variation of disease resistance pathways in Arabidopsis thaliana
are due to regulatory QTLs, structural QTLs, or both, and 3) characterize
individual genes at the molecular level that encode the regulatory QTLs. This
will be accomplished by surveying accessions for natural variation in ELPs in
response to induction of defense related pathways by a salicylic acid analog
(dichloroisonicotinic acid) and jasmonic acid using Affymetrix chips for the
preliminary global screen and spotted microarrays to confirm reproducible ELPs.
Recombinant inbred lines derived from crosses between polymorphic accessions
will then be phenotyped for ELPs using targeted DNA microarrays designed with
novel applications of statistical methods. QTLs associated with ELPs will be
mapped by employing an innovative application of established QTL mapping
methodologies, including composite interval mapping and permutation thresholds.
This approach will allow the identification of regulatory QTLs, a subset of
which will be cloned using a combination of candidate gene and extreme allele
approaches. Ultimately, these approaches will enable massively parallel QTL
analysis for mRNA, protein and metabolite levels relative to the whole plant
phenotypes. The data generated from this project will be available at
(www.niblrrs.ucdavis.edu) and (www.genomics.purdue.edu). This project will also
provide multidisciplinary training at the interface of quantitative and
molecular genetics, statistics, and genomics for postdoctoral researchers,
graduate students, undergraduate students, and high school students.