* 0119441
* Enzymatic Properties and Biological Roles of Novel Beta-Hydroxyacid Dehydrogenases in Plants and Bacteria
* BIO,MCB
* 09/01/2001,08/31/2003
* John Hawes, Indiana University
* Standard Grant
* Parag R. Chitnis
* 08/31/2003
* USD 200,000.00

The beta-hydroxyacid dehydrogenases are a structurally and mechanistically
related family of enzymes with a predominance of homologues specific to bacteria
and plants. The only two known mammalian family members include beta-
hydroxyisobutyrate dehydrogenase and 6-phosphogluconate dehydrogenase. Two of
the bacterial family members have recently been identified as isozymes of
tartronate semialdehyde reductase (TSR) with differences in substrate
specificity. TSR is part of a pathway for glycerate biosynthesis from
glyoxylate, hydroxypyruvate, and other substrates in bacteria and plants. Most
bacterial and plant genomes display multiple homologues of TSR. The Escherichia
coli genome contains 4 homologues, of which two have been cloned and
enzymatically characterized. The Arabidopsis thaliana genome contains 6
homologues of TSR, none of which have been cloned or identified in terms of
substrate specificity and enzymatic properties. Most of the bacterial beta-
hydroxyacid dehydrogenase genes contain apparent binding sites for cAMP-CRP and
are likely regulated by catabolite repression. These enzymes may serve
alternative metabolic pathways, such as glycerate biosynthesis, in anaerobiosis
and in conditions of nutrient deprivation when TCA cycle activity is inadequate.
This hypothesis is fitting with the organization of surrounding genes and cAMP-
CRP sites observed in E. coli and Haemophilus influenzae loci containing beta-
hydroxyacid dehydrogenase genes. This project will examine the enzymatic
properties and substrate specificities of cloned and over-expressed beta-
hydroxyacid dehydrogenases of A. thaliana, to determine what biological
functions the plant TSR isozymes may serve. The identities and relative
quantities of these and other co-expressed proteins will be determined by a
proteomic approach, utilizing isotope-coded affinity tags, in order to
understand more clearly what metabolic pathways are functioning together under
conditions of specific substrate limitation and/or anaerobiosis in E. coli and
H. influenzae. &lt;br/&gt;It is not clear, biologically, why certain bacteria
and plants should require multiple isoforms of TSR, but this genetic capacity
may reflect a previously unrecognized importance of glycerate biosynthesis in
bacteria and plants. This project will expand our understanding of this recently
discovered family of enzymes, and their apparently specific metabolic roles in
plants and bacteria. This research will also bring new information about what
enzymes and other proteins are predominantly expressed in two different
bacterial species when grown under various limiting and anaerobic
conditions.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;