* 0129848
* Theoretical Condensed Matter Physics
* MPS,DMR
* 01/01/2002,12/31/2005
* Eric Siggia, Cornell University
* Continuing grant
* G. Bruce Taggart
* 12/31/2005
* USD 453,000.00

This grant is co-funded by the Divisions of Materials Research, Molecular and
Cellular Biology, and Physics. The grant supports theoretical research in
biological physics.

Many problems in biology have a component that involves statistical mechanics
either directly because the question is on the molecular scale, or indirectly
because a probablistic model is being used to decode a genomic sequence.

The internal Watson-Crick base pairing of RNA defines its secondary structure
which is better delineated energetically than the corresponding elements in
proteins. A kinetic Monte Carlo code has recently been written to follow the
formation of secondary structure for RNA's up to ~ 400 bases in length and
includes non-nested configurations (pseudoknots). The code will be applied to
the ends of the molecule as in recent experiments.

Protein trafficing in cells involves numerous steps where proteins are sorted
and targeted, e.g., in the Golgi substrate as processed in successive cisternae
by different enzymes. In collaboration with a laboratory at NIH, models which
allow for Golgi enzyme localization in a connected membrane system will be
devised and tested. These are alternatives to the prevailing ideas that either
the substrate lives in fixed compartments and the enzymes flux through the
compartments via an active process of sorting and vesicle transport, or the
enzymes are fixed and the substrates are actively transported.

There are now multiple fluorescent proteins that can be expressed in live cells
(sometimes as chimeras with native proteins), which allow for measurement of
their number in one cell as a function of time. In conjunction with an
experimental group at Rockefeller, we are modeling the stochastic noise (due to
a small number of molecules) in transcription and translation. More generally we
will collaborate on experiments and theory to assess whether stochastic effects
impact the fitness of bacteria.

The regulation of gene transcription by short, e.g., 10-30 bp, motifs to which
proteins bind and integrate environmental signals is important to many areas of
biology. An algorithm has been devised which finds improbable strings of bases,
based on the frequency of shorter strings present in an iteratively constructed
'dictionary.' It yielded many significant motifs when applied to yeast, and we
intend to apply the algorithm to other organisms, as well as extend it to
recognize fuzzy patterns. The comparison of related species is another way to
recover regulatory motifs and we intend to apply a probablistic alignment code
both to fit the evolution rate in noncoding sequence and assign probabilities to
the regions which are more conserved than chance and thus functional. An
algorithm to relate gene expression data directly to sequence and assign
statistical significance has been tried on yeast and will be extended to other
multicellualr organisms. %%% This grant is co-funded by the Divisions of
Materials Research, Molecular and Cellular Biology, and Physics. The grant
supports theoretical research in biological physics.

Many problems in biology have a component that involves statistical mechanics
either directly because the question is on the molecular scale, or indirectly
because a probablistic model is being used to decode a genomic sequence. This
research will investigate a variety of problems of current biological interest
using techniques from theoretical condensed matter physics. ***

