* 0132675
* Chromosome Homeostasis: Identification and Functional Analysis of Chromosomal Replicons in the Amitotic Tetrahymena Macronucleus
* BIO,MCB
* 05/01/2002,04/30/2006
* Geoffrey Kapler, The Texas A&M University System  HSC Research Foundation
* Continuing Grant
* Susan Porter Ridley
* 04/30/2006
* USD 385,000.00

The regulation of replication initiation is poorly understood in eukaryotes,
largely due to the difficulty in identifying cis-acting regulatory determinants.
A thorough genetic dissection of replicons (replication initiation sites and
their cis-acting regulatory determinants) has been achieved in a limited number
of experimental organisms. Prior research on Tetrahymena thermophila focused on
the 21 kb rDNA minichromosome, which is amplified 5000-fold during a single S
phase in the developing macronucleus, yet is replicated just once during each
subsequent vegetative cell division. The goal of this new research project is to
identify and dissect additional chromosomal replicons in Tetrahymena. By
comparing the genetic organization of replicons that are not amplified to the
rDNA, insights into cell cycle control should be obtained. These investigations
will also serve as an entry point to explore how genic balance is maintained in
the ciliate macronucleus, which lacks traditional mitotic segregation machinery.
The initial stage of the project involves the mapping of replication origins in
small (~100 kb), non-rDNA macronuclear chromosomes by 2D gel electrophoresis,
and cloning of DNA that spans these sites. These regions will be subjected to
comparative sequence and functional genomic analysis, to identify conserved
elements and determine the physical and possibly functional relationship of
replication origins to neighboring genes. The genetic organization of these
replicons will then be dissected using two complementary strategies. The first
approach will employ a novel method in which chromosome breakage sequence (Cbs)
elements will be targeted to specific positions in the germline (micronuclear)
genome. Programmed DNA fragmentation at these Cbs sites in the developing
macronucleus will be used to map the physical limits of the replicons under
examination. As a variation to this approach, marker rescue will be used to
restore function to defective replicons. In a complementary approach,
restriction fragments from size-selected (100 kb) chromosomes will be inserted
into an engineered selectable vector to generate plasmid libraries. These
libraries will be transformed into Tetrahymena to identify artificial
minichromosomes capable of supporting autonomous
replication.&lt;br/&gt;&lt;br/&gt;DNA replication is a highly regulated process,
assuring that chromosomes are duplicated once and only once per cell division.
Deviations from strict cell cycle control can result in selective amplification
(over-replication) of small DNA segments. The high degree of complexity of
chromosomal replicons in higher eukaryotes, and the absence of natural or
artificial minichromosomes in these systems has hampered research in this area.
The amenability of Tetrahymena to reverse genetic analysis will be exploited
here to determine how complex eukaryotic replicons are organized and to explore
how global cell cycle regulatory mechanisms can be differentially imposed upon
replication origins. A bonus of this project is the diversity of experimental
approaches, which will provide students with unique learning opportunities at
the interface of new (genomics) and more traditional (molecular genetic)
technologies.&lt;br/&gt;