* 0140140
* Computer Simulation of Protein Structure and Dynamics
* MPS,PHY
* 09/01/2002,08/31/2007
* Sebastian Doniach, Stanford University
* Continuing Grant
* Kathleen McCloud
* 08/31/2007
* USD 300,000.00

Research will focus on two areas of computational biology - (1)application of
methods which explore conformation space for biomolecules, and (2) development
of methods to help in protein structure determination both for soluble proteins
and for membrane proteins.&lt;br/&gt;&lt;br/&gt;Understanding protein folding is
important in a number of contexts such as protein design and the more general
issue of finding general rules relating the folding process to the primary
sequence of the protein. A major problem for computer simulation of protein
folding is the fact that the dynamics takes place over many decades of time,
from sub-picoseconds to milliseconds or even seconds. The approach we adopt is
to constrain the motions of a protein so that it moves from an initial state,
such as an unfolded state, to a final folded state in a fixed number of time
steps. This "trajectory annealing" approach has the capability to explore a
number of possible pathways by which a protein may fold while keeping the system
at a physiological temperature. We will continue our present studies of
trajectory annealing and apply them to two small proteins whose folding
properties have been extensively sampled experimentally. The trajectory
annealing simulations offer the possibility of interpreting those experiments in
terms of changes in probability of various pathways for folding which are very
difficult to observe experimentally.&lt;br/&gt;&lt;br/&gt;As a result of the
tremendous progress in large-scale DNA sequencing projects, rapid growth in
accumulation of biological sequence information has put strong pressure on the
structural biology community to produce structural information for new genes
with high throughput. Experimentally, large scale X-ray crystallography and NMR
measurements now aim to determine all (1000 to 10000) available protein folds
within a few decades or even years. However there remain many soluble proteins
which have difficulties in crystallizing and for which NMR methods may be
ineffective. Structural homology is a very powerful tool by which we can try to
assign functions in silico to new genes which bear only remote if any
association with known genes in terms of sequence homology. We will build on our
current computational methods of ab initio structure prediction by adding
additional physical information contained in small angle x-ray scattering (SAXS)
data. This will improve our ability to find structural homologs of a given
protein with proteins of known structure but for which there is little or no
sequence homology. We will also extend our methods of reconstructing low
resolution electron density maps from small-angle scattering data to x-ray
scattering from membrane proteins bound in small vesicles. Some 25% of gene
sequences in the database code for expression of membrane proteins. Although
structures for several thousand soluble proteins have been determined, only a
handful of structures for membrane proteins are known owing to the difficulties
of crystallizing proteins which are naturally stabilized by the hydrophobic
environment of the s urrounding lipids. By embedding a membrane protein in small
lipid vesicles, we believe we can generate SAXS data which contains information
about the shape and size of the protein in addition to data on the vesicle, and
also about the x-ray interference pattern between the scattering from the
vesicle and the scattering from the protein. By making a preparation in which
vesicle sizes can be varied, we believe we can extend our present reconstruction
methods to sort out the protein contributions to the scattering from the vesicle
contributions and hence obtain low resolution structural information on the
membrane protein.