* 0408768
* YIA-PGR: Assessment of the use of Oligonucleotide Microarrays for Single Nucleotide Polymorphism Mutation Detection in Maize
* BIO,IOS
* 09/29/2003,09/30/2006
* Nathan Springer, University of Minnesota-Twin Cities
* Continuing Grant
* Diane Jofuku Okamuro
* 09/30/2006
* USD 327,757.00

Large-scale genome sequencing projects in plants have yielded a wealth of
sequence information that can be used to form hypotheses regarding gene
function. However, in many plant species it is difficult to test these
hypotheses due to a lack of reverse-genetics resources. This project will
develop an Ethane methylsulfonate (EMS)-mutagenized population of maize plants
and screen it using two approaches to SNP mutation detection: Targeted Induced
Local Lesions IN Genomes (TILLING), which utilizes mismatch-specific
endonuclease digestion to detect SNPs, and a prototype oligonucleotide
microarray system, which uses a hybridization-based approach. &lt;br/&gt;
&lt;br/&gt;EMS is a chemical mutagen that causes primarily C-&gt;T to A-&gt;G
transition mutations. Bioinformatic analysis can be used to predict whether
these transitions result in nonsense, missense, or silent mutations. An
oligonucleotide microarray can be designed to screen for the informative
mutations by incorporating oligonucleotides identical to the wild-type parental
sequence and oligonucleotides identical to the transition mutation. DNA from
mutagenized individuals will be labeled and hybridized to the oligonucleotide
microarrays to identify individuals containing a specific mutation. The
advantages of this screening method are increased efficiency (more genes at
lower cost), precise identification of the mutation when it is detected, and the
characterized genetic background being used. &lt;br/&gt;&lt;br/&gt;The approach
employed in this project is made possible by the use of a maskless array
synthesizer (MAS) device, which allows rapid construction of inexpensive
oligonucleotide microarrays that can easily be customized. A single microarray
can be used to screen for over 100,000 potential mutations if eight
oligonucleotides are used to screen for each SNP. &lt;br/&gt;&lt;br/&gt;The
outcomes of this project will form the foundation of a publicly available
reverse-genetic resource for maize that will include mutants, screening
protocols, and bioinformatic tools. These resources will be valuable to
researchers interested in developing reverse-genetics resources for other plant
species as well as researchers interested in other SNP detection applications
for plant and animal genomic DNA. &lt;br/&gt;