* 0415486
* Genes for Phylogenies: Identifying Genes with Intrinsically Good Properties for Phylogenetic Estimation
* BIO,DEB
* 12/01/2003,06/30/2005
* Gavin Naylor, Florida State University
* Standard Grant
* James E. Rodman
* 06/30/2005
* USD 59,465.00

The study of phylogenetic relationships among species of vertebrate animals
relies heavily at present on analyses of mitochondrial gene sequences, and less
so on analyses of nuclear genes. Dr. Gavin Naylor of Iowa State University is
exploring several candidate nuclear genes, presumed to be single-copy or from
small families (duplications) of genes, in order to find sources of reliable
characters for phylogenetic analysis of vertebrates. From initial surveys of
over 7000 genes in gene-sequence databases, ten or so candidates will be
selected, amplification primers constructed to extract and amplify the segments,
and the genes then sequenced for several model taxa of fishes, birds, and
mammals. In comparing mutational differences between gene sequences from
different organisms, such properties as nucleotide base compositional evenness,
the proportion of sites free to vary, the among-site rate variation, and codon
usage will be evaluated in tests using a known phylogenetic tree for the model
taxa. Not all genes are equally likely to yield informative results in
phylogenetic analyses, but there has been little explicit testing of properties
of candidate genes, to determine beforehand useful gene sources. The work to be
carried out will identify genes with properties that predispose them to be
phylogenetically useful and will provide empirical tests of their utility.
&lt;br/&gt; As sequencing technology improves and as more genomes are sequenced
in their entirety, bioinformatics will turn to comparative genomics to mine
information that is buried in the comparison of whle-genomes. This endeavor will
be greatly enhanced with an accurate understanding of the phylogenetic
relationships among the genomes being compared, and in turn robust, reliable
phylogenies will depend upon data drawn from several nuclear as well as
mitochondrial genes.&lt;br/&gt;&lt;br/&gt;