* 0419648
* Arabiodopsis 2010:  Network Analysis of Disease Resistance Signaling
* BIO,IOS
* 09/01/2004,08/31/2009
* Jane Glazebrook, University of Minnesota-Twin Cities
* Continuing Grant
* Michael Mishkind
* 08/31/2009
* USD 1,336,000.00

Arabidopsis 2010 Project - Network Analysis of Disease Resistance
Signaling&lt;br/&gt;&lt;br/&gt;PI - Jane Glazebrook&lt;br/&gt;Co-PI - Fumiaki
Katagiri&lt;br/&gt;&lt;br/&gt;Department of Plant Biology, University of
Minnesota&lt;br/&gt;&lt;br/&gt;ABSTRACT&lt;br/&gt;&lt;br/&gt;This project will
determine the function of a network of genes involved in disease resistance. The
genes studied will be chosen from among those found to be expressed at increased
levels after pathogen infection. As several thousand genes satisfy this
condition, genes will be prioritized based on other characteristics, such as
rapid expression changes or annotation suggesting functions in signaling,
secondary metabolism, or unknown function. There are four Specific
Aims:&lt;br/&gt;1. Obtain global expression profiles for Arabidopsis responses
to P. syringae infection.&lt;br/&gt;2. Identify genes that contribute to
resistance to P. syringae.&lt;br/&gt;3. Identify genes comprising the defense
signaling circuitry controlling activation of defense responses.&lt;br/&gt;4.
Model the defense signaling circuitry using gene expression profiles of
regulatory mutants.&lt;br/&gt;A description of the project, including a list of
genes under study named by Arabidopsis Gene Index numbers, allele numbers of
mutants under study, project status and preliminary results, will be available
to the public at: http://www.cbs.umn.edu/labs/glazebrook/NSF2010/.&lt;br/&gt;The
function of selected genes in disease resistance will be determined using
mutants with defects in the selected genes. These mutants will be tested for
enhanced susceptibility to pathogen attack and altered expression profiles
following infection. Based on the expression profile data, a model of the
topology of the signal transduction network controlling activation of defense
responses will be constructed.&lt;br/&gt;Results will be shared at
http://www.cbs.umn.edu/labs/glazebrook/NSF2010/ as soon as they are judged to be
reliable. Such results will include data from global expression profiling
experiments (expected within 18 months from project start, and also posted at
TAIR), pathogen susceptibility data from mutant lines (beginning 18 months from
project start), expression profiles from mutant lines (beginning 24 months from
project start), and network topology models (prior to the end of the project).
In addition, project results will be shared at conferences and in publications
in the scientific literature.&lt;br/&gt;The project addresses the 2010 program
objective of determining functions for networks of genes involved in a
particular biological process. In this case, the biological process is
activation of a defense response to pathogen attack. &lt;br/&gt;The project will
have broader impacts in the area of integration of research and education. These
include participation of undergraduates, including those from under-represented
groups, involvement of graduate students and post-doctoral fellows in laboratory
teaching of undergraduates, development of a mentoring program for
undergraduates involved in laboratory research, and integration of research
results into a bioinformatics course. Improved understanding of plant disease
resistance mechanisms has potential applications in crop
improvement.&lt;br/&gt;&lt;br/&gt;