* 0420015
* Arabidopsis 2010: The Function of Subtilase Genes in Plant Development
* BIO,IOS
* 10/01/2004,09/30/2009
* Martin Spalding, Iowa State University
* Continuing Grant
* Thomas P.  Jack
* 09/30/2009
* USD 958,373.00

Abstract &lt;br/&gt;&lt;br/&gt;This research will determine the function of
subtilase genes in Arabidopsis. Subtilase genes encode subtilisin-like serine
proteases, proteins that play important roles in plant growth and development.
In mammalian systems, some subtilases are called "hormone convertases" because
they cut other proteins releasing bioactive peptides that serve as hormones
and/or function in cell-to-cell signaling. The properties of some plant
subtilases suggest they may play similar roles in plant development and
physiology. &lt;br/&gt;&lt;br/&gt;The function of nearly 60 subtilase genes in
Arabidopsis will be investigated through collaboration with European partners in
The Arabidopsis Subtilase Consortium (TASC). A listing of all the genes in the
subtilase gene family (and their subfamily identification) is found at
http://csbdb.mpimp-golm.mpg.de/csbdb/dbcawp/psdb/main/mgenes.html. This research
will focus on the role of ten subtilase genes that are highly regulated during
shoot regeneration in tissue culture. Three of these subtilase genes appear to
influence shoot development in that the expression of these genes is controlled
by a major quantitative trait locus (QTL) that determines shoot regeneration
efficiency. Shoot regeneration efficiency varies in Arabidopsis and in some
important economic crops where it can impede the successful application of
genetic engineering technologies. Another seven subtilase genes are
differentially regulated during shoot or root regeneration.
&lt;br/&gt;&lt;br/&gt;A proteomics approach (the study of a large number of
proteins using protein expression profiling) will be used to understand the
function of the subtilases. The function of these genes in vivo will be examined
by quantitatively analyzing 2D difference gel electrophoresis (DIGE) profiles of
T-DNA insertion mutants and transgenics with induced expression of subtilase
genes. Subtilase target sites will be determined by analyzing the products from
the limited digestion of common substrate proteins by the recombinant enzyme.
Optimal subtilase target sites will be determined by designing and testing the
hydrolysis of synthetic peptides related to the cleavage sites for common
substrates.&lt;br/&gt;&lt;br/&gt;Research results will shared with the public
and other members of the scientific community through http://csbdb.mpimp-
golm.mpg.de/csbdb/dbcawp/psdb.html. Information at this site is edited by the
Altmann lab and will be updated on a monthly basis. This research is significant
to the goals of the 2010 project in that it contributes to our understanding of
a group of important genes that influence plant growth and development. This
research makes a unique contribution in that it uses new tools in proteomics to
address gene function.&lt;br/&gt;&lt;br/&gt;Broader
impact&lt;br/&gt;&lt;br/&gt;This project has broader impact because it
contributes knowledge to the role of proteolysis and peptide signaling in plant
development. Little is known about bioactive plant peptides from genomics
because peptides are often embedded in other proteins and are not released until
cleaved by proteases, such as subtilases. &lt;br/&gt;&lt;br/&gt;The project also
has broader impact because it provides lab research training for Iowa State
University and Des Moines Area Community College undergraduates. Some of the
undergraduate trainees will also have the opportunity to conduct research during
the summer in international labs of partners associated with TASC. To encourage
ISU minority undergrads to participate in this project, we will host a meeting
each fall in the Roy J. Carver Co-Laboratory for the local chapter of Minorities
in Agriculture, Natural Resources and Related Sciences
(MANRRS).&lt;br/&gt;&lt;br/&gt;