* 0420267
* Arabidopsis 2010:  Deriving the Gene Circuitry and Network Motifs of the Arabidopsis Defense Response
* BIO,MCB
* 09/01/2004,08/31/2008
* Terence Speed, University of California-Berkeley
* Standard Grant
* Richard Rodewald
* 08/31/2008
* USD 800,000.00

In order to determine the gene function for a network of genes, the circuitry of
plant defense against pathogen will be derived from temporal and spatial large-
scale mRNA profiling of powdery mildew infection of Arabidopsis. Laser capture
microdissection will be utilized to isolate RNA from distinct populations of
plant cells (including infected epidermal cells and neighboring epidermal and
mesophyll cells) at a sampling rate and with sufficient time points to capture
the underlying biological processes. To analyze this mRNA expression data, novel
statistical approaches will be developed. For example, time-alignment algorithms
will be used to synchronize data among experiments and evaluate altered temporal
patterns of response including time delays and period compression/expansion.
Modeling the circuitry of plant defense allows for the rigorous assessment of
the regulatory factors and downstream products impacting the progression of
infection. Continuous representations will be used to derive the circuitry of
response, allowing one to uncover network nodes and subnodes associated with
distinct functional responses. Experimental testing of predicted gene function
will focus on genes in subnodes of salicylic acid (SA)-impacted networks
elucidated using an SA biosynthetic mutant. In addition, defining these networks
and nodes allows for discovery and assessment of network motifs such as feed-
forward loops used to solve a common biological "problem" in diverse
circumstances and species. This spatially and temporally resolved expression
data, the statistical and computational approaches and tools, and the identified
defense circuitry and network motifs will all be of significant value to the
Arabidopsis community. Previous datasets collected from whole leaf samples and
few time points were unable to resolve much of the complexity of the defense
response. Numerous predictions of gene function and regulation will emerge from
these efforts. Model-driven experiments will focus on genes in SA-impacted nodes
and subnodes. These genes are not known a priori, but will be determined from
the modeling efforts. Likely genes include: transcription factor(s) involved in
the induction of ICS1 and PR1 clusters, a putative SA glucosyltransferase,
candidate genes for the second enzyme involved in the synthesis of SA from
isochorismate, and a putative transcriptional repressor responsible for the SA-
dependent repression of the PDF1.2 cluster. The expression data will be
deposited in public databases such as the Integrated Microarray Database System
being developed as part of the NSF Arabidopsis 2010 award to X. Dong and co-PIs
and will be freely available for use. In addition, developed algorithms and
computational tools will be available for download and will be included as tools
in Bioconductor, a free microarray analysis platform. Information about this
project is available at http://plantbio.berkeley.edu:16080/~wildermuth/.

Broader Impacts This research will result in a more comprehensive understanding
of plant-pathogen interactions by providing a formal mathematical framework for
molecular genetic and biochemical data. The strategies elucidated for this
plant-pathogen interaction are likely applicable to other host-pathogen
interactions. In particular, identified functional control modules are likely to
be shared across pathosystems. The informatic and statistical methodologies and
tools developed for the analysis and modeling efforts will be made widely
available to both the plant and general scientific communities. In addition, the
intimate collaboration and cross-training of young mathematicians, engineers,
and experimental biologists yields truly interdisciplinary scientists uniquely
positioned to address biological questions using quantitative and systems-based
approaches.