* 0421799
* Plant RNA Binding Proteins: Tools for Functional Genomics and Application to Chloroplast Biogenesis
* BIO,IOS
* 07/01/2004,09/30/2010
* Maureen Hanson, University of Oregon Eugene
* Continuing Grant
* Diane Jofuku Okamuro
* 09/30/2010
* USD 4,181,531.00

Plant RNA Binding Proteins: Tools for functional genomics and application to
chloroplast biogenesis

PI: Alice Barkan, University of Oregon coPI: Maureen Hanson, Cornell University

Interactions between proteins and RNAs underlie every aspect of plant
development and function. Accordingly, plant genomes encode >700 proteins that
harbor predicted RNA binding motifs, but few of these proteins have been
studied. To dissect RNA-protein interaction networks in plants, it is necessary
to catalog plant RNA-binding proteins, to identify the RNAs with which they
interact, and to determine how those interactions influence the fate of the RNA
and downstream processes. This project develops tools to address each of these
issues, and applies them to a set of 40 RNA binding proteins. The experimental
set of 40 proteins emphasizes the CRM and PPR protein families, which are
largely specific to plants and which together contain ~500 members, most of
which are predicted to be targeted to chloroplasts or mitochondria. Project
goals are to: (i) Develop a relational database of predicted plant RNA binding
proteins that will integrate data from rice, maize, and Arabidopsis. The
database will feature cross-referenced orthologs and paralogs, annotated with
gene models, intracellular targeting predictions, experimental data and
literature citations; (ii) Use an approach that couples coimmunoprecipitation
with microarray technology to identify RNAs with which each of the 40 proteins
in the test set is associated in vivo; (iii) Pinpoint aspects of RNA metabolism
that are influenced by these 40 proteins, through the analysis of molecular
defects resulting from reduced expression of their genes in maize and rice; (iv)
Test an approach to increase the efficiency of cloning maize mutations caused by
insertions of Mu transposons and, if successful, apply it to an existing set of
Mu-induced mutants with defects in chloroplast RNA metabolism; (v) Increase
community awareness of plant research through outreach programs at both sites.
Expected outcomes include (i) a comprehensive relational database of RNA-binding
proteins in plants, which will be of value for a broad range of plant biology
researchers; (ii) elucidation of the general properties of two recently
recognized plant protein families, the PPR and CRM families, through the
identification of specific RNA substrates and physiological roles for
representative family members; (iii) continuation of a reverse-genetic screening
service for chloroplast biogenesis genes; (iv) tutorials on rice transformation
and genetics for college and high school students, respectively, and K-12
outreach programs in Ithaca and Eugene. Access to tools and data is available
through the project's Photosynthetic Mutant Library and database at
http://chloroplast.uoregon.edu/.