* 0421651
* Global Analysis of Functional Units in Plant Chromosomes: DNA Replication, Domain Structure, and Transcription
* BIO,IOS
* 10/01/2004,09/30/2011
* George Allen, North Carolina State University
* Continuing Grant
* Diane Jofuku Okamuro
* 09/30/2011
* USD 5,245,054.00

It is well known that higher organisms, including plants, begin the process of
replicating their DNA at many different places in their genomes. However, as yet
we know very little about how these starting points ("replication origins") are
defined, or how they relate to other chromosomal features. An interdisciplinary
team from three leading research institutions will pool their efforts and
expertise to investigate this question and attain a deeper understanding of
spatial and biochemical parameters affecting genome function.

Specific objectives: (1) A primary goal of their research project is to
construct maps of replication origins in large regions of representative plant
chromosomes, as well as to construct and overlay maps of key chromosomal
features such as matrix attachment regions, regions enriched in modified
histones or methylated DNA. (2) A parallel effort will map DNA replication
activity and gene activity (transcription) in the same chromosomal regions under
several different developmental conditions, which will lead to a better
understanding of the interrelationship of chromosome structure and function.

Outcomes:

When completed, the project will help scientists understand how chromosome
functions are spatially organized and coordinated in structural and functional
domains, and help define chromosomal features that play key roles in these
functions. The proposed experiments will combine powerful genomic technologies
with molecular, cellular and biochemical tools. The team will focus on the short
arm of rice chromosome 1 and the full length of Arabidopsis chromosome 4, thus
providing extensive data for model organisms representing each of the two major
groups of crop plants. Comparing chromosomal features for these plant species
with data for animals and other organisms will help identify chromosomal
features important enough to be broadly conserved in evolution, as well as
highlight features that may be unique to the plant kingdom. New genomic
information and tools that are generated (including the carefully designed set
of sequences to be used as mapping landmarks on genomic microarrays) will be
made available to other members of the scientific community via the web as well
as through conventional publications. Data will be submitted to the Gene
Expression Omnibus homepage (http://www.ncbi.nlm.nih.gov/geo/) and ArrayExpress,
a public repository (http://www.ebi.ac.uk/arrayexpress/) . All sequence
annotation for the chromosomal elements will be submitted to GenBank. The basic
information on chromosome structure generated by this project will help support
a new generation of genetic engineering technologies, which may include new
methods for directing gene expression, targeted gene insertion, and construction
of artificial chromosomes. In addition, the project will provide training in
state-of-the-art functional genomic and bioinformatics techniques. Trainees at
the undergraduate, graduate and postdoctoral levels will participate in the
research, and workshops will be organized for scientists in the southeastern US,
including faculty and students at historically black and minority institutions.
The team will also work with local educators to develop teaching tools useful
for presenting genomics concepts to high school students.