* 0425931
* QSB: Charting the Regulatory Space of Hematopoiesis by Microfluidic Digital PCR
* BIO,MCB
* 09/01/2004,09/30/2005
* Ellen Rothenberg, California Institute of Technology
* Standard Grant
* Parag R. Chitnis
* 09/30/2005
* USD 475,095.00

All the cells of a developing organism carry the same instructions for making
proteins, yet the execution of the genetic program is systematically varied to
produce the specialized tissues that comprise the fully-formed adult. While the
character of each cell reflects the expression of many thousands of genes, this
expression pattern is itself determined by a smaller network of regulatory
genes, or 'transcription factors.' The causal basis for cellular differentiation
can be traced back to the activity of these regulatory networks. For example, in
response to diffusive signals called cytokines, self-renewing hematopoietic stem
cells give rise to daughter cells that commit to progressively more specialized
fates, ultimately becoming terminally differentiated cells, such as
erythrocytes, macrophages, and the B and T cells of immunity. This project will
develop and apply a new technology - a digital RT-PCR assay using silicone
elastomer microfluidic chips - to take a focused look at the behavior of genetic
regulatory networks in individual developing blood cells. This research will
profile the activity of networks of genes implicated as hematopoietic regulators
with single-cell resolution. A database of such profiles will reveal how network
activity varies within ostensibly uniform populations of progenitor cells and
support comparisons between distinct, developmentally-staged populations. RT-PCR
readings will be collated with flow cytometry data to facilitate population
analysis that encompasses both surface marker phenotype and internal regulatory
network state. Sibling assay studies will be conducted to explore the
possibility that metastable regulatory network states bias lineage choice
responses to instructive signaling from cytokines.&lt;br/&gt;&lt;br/&gt;Broader
Impact: This project will further creation of new, interdisciplinary approaches
to biological investigations by integrating mainstream developmental genetics,
advanced 'lab-on-a-chip' technology, and a quantitative, engineering-oriented
approach to the analysis of complex systems. Software developed for the project,
including a powerful new program for designing multiplexed PCR primers, will be
made available to the research community on an open source basis. In addition,
this project will provide opportunity for cross-disciplinary education and
training of graduate students; undergraduate students will be involved through a
CalTech- sponsored summer research fellowship program (SURF). Valuable research
experience will also be available to underrepresented groups through
investigator involvement with the Pasadena City College Biotech Program that
sponsors research science internships by providing opportunity for interns to
work on this cross-disciplinary project.&lt;br/&gt;