* 0445825
* SGER: Proteomic Analysis of Multi-Gene Families
* BIO,IOS
* 09/01/2004,02/28/2007
* Karl Wilson, SUNY at Binghamton
* Standard Grant
* Diane Jofuku Okamuro
* 02/28/2007
* USD 200,000.00

Plants have gene families encoding so many proteins that traditional antibody-
based methods for studying protein level expression become nearly impossible.
Since information about protein content with respect to developmental stage,
subcellular localization and environmental conditions can give important clues
regarding function, a method for identifying proteins of a multi-gene family and
measuring their relative levels in plant tissues would be valuable. The approach
to be developed in this proof of concept project makes use of antibodies that
recognize common epitopes in a gene family as a tool to isolate family members
for proteomic analysis. This project is justified for support as a Small Grant
for Exploratory Research because it involves preliminary work on untested and
novel ideas.

The procedure starts with peptide digests and differential labeling of two sets
of peptides with coded mass tags. After mixing the two sets, immunoaffinity
chromatography is employed to isolate the epitope-bearing peptides. These are
resolved by reverse phase chromatography in-line to a tandem mass spectrometer
where relative peak intensities of the peptide from different sets are
determined and parent proteins are identified. To test the method, we will start
with known mixtures of peptides prepared from expressed clones. Situations in
which mixtures contain peptides of varying affinity to antibody or where a few
are highly abundant will be simulated. This work will be carried out using
protein extracts prepared from Arabidopsis thaliana and rice seedling leaves.

Although the proof of concept work will focus on the subtilisin protease gene
family, this proteomics approach will be directly applicable to many other
multi-gene families. Homology within multi-gene families extends across plant
species. Therefore, this new approach can be used to uncover orthologous genes
in crop plants that share the same tissue specificity and subcellular
localization. Those orthologs that also show parallel changes in protein level
with respect to developmental stage and responses to hormonal cues and
environmental stresses are likely to share similar functions. Thus,
understanding of the function of an individual protein in a well-studied model
species can be multiplied rapidly toward understanding the function of orthologs
in other plant species. Conversely, an ortholog can be identified for deeper
study in a model species, and the results applied back to other plant species to
increase crop productivity or to develop new sources of fiber and specialty
chemicals. The project will provide research opportunities for graduate and
undergraduate students. It will also provide data for discussions and problem-
solving sessions in biochemistry and proteomics courses taught by the Principal
Investigators.