* 1902892
* Scalable Model-Based Reconstruction of Network Evolution
* MPS,DMS
* 06/01/2019,05/31/2023
* Curtis Brandt, University of Wisconsin-Madison
* Continuing Grant
* Zhilan Feng
* 05/31/2023
* USD 724,239.00

The availability of full genomes across individuals from many populations and
many species offers rich information about past evolutionary history. By
comparing the genes of different individuals, one can detect which individuals
are most closely related and reconstruct the history of population splits and
speciation, as visualized in a phylogenetic tree. Challenges arise however
because of genealogical differences between individuals within each species,
current or ancestral. This project focuses on the detection of species
convergences: when species hybridize, or when individuals from one species
migrate to another, or when strains recombine. The history of a group of species
is then best described by a network, where a backbone tree represents speciation
and extra branches describe gene flow from one population into another. Current
methods to estimate phylogenetic networks cannot analyze data sets with more
than a few dozen species. Based on novel theoretical foundations, the PIs will
develop statistical methods and software that will scale to hundreds of species
and thousands of genetic loci. These new methods will also be particularly
valuable to advance knowledge in bacterial and virus evolution, where
recombination is prevalent. The project will support graduate and undergraduate
students, who will gain training beyond traditional disciplinary boundaries with
involvement in the larger community of campus researchers interested in networks
in data science.&lt;br/&gt;&lt;br/&gt;Through the mathematical analysis of
coalescent processes on phylogenetic networks, the PIs will determine the
maximal substructures of these networks that can be theoretically identified
from various data types, such as from gene trees, or genetic distances between
pairs of individuals, using one or more individuals per populations. Theory will
also be developed to determine the amount of data necessary to reconstruct the
phylogenetic network with accuracy. These theoretical findings will guide the
development of new statistical methods and software to estimate phylogenetic
networks from data, with a focus on the use of genetic distances to devise fast
algorithms that can handle hundreds of species. These fast reconstruction
methods will allow the deployment of a cross-validation method to learn from
data the appropriate complexity of the network, that is, the appropriate number
of gene flow events. The proposed research will advance knowledge of the
evolutionary history in many groups where gene flow and recombination is
suspected, such as the early radiation of mammals or land plants, and the
evolutionary history of the herpes virus family.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.