* 1903139
* Collaborative Research: Novel Statistical Tools for Metagenomics and Metabolomics Data
* MPS,DMS
* 06/15/2019,05/31/2023
* Jun Liu, Harvard University
* Continuing Grant
* Zhilan Feng
* 05/31/2023
* USD 350,000.00

Accumulating evidence suggests that disrupting intestinal microbial ecosystems
can cause many serious diseases including chronic diseases such as coronary
heart disease (CHD), neurobehavioral diseases such as autism, inflammatory
diseases such as inflammatory bowel disease (IBD) and etc. For example, the
developing infant intestinal microbiome has been implicated in central
myelination and the maturation and function of microglia (CNS immune cells), a
core deficiency in Autism Spectrum Disorders. Thus, the study of the gut
microbial distributions and their metabolites are very important to find new
therapeutic targets for many diseases. However, compared to the huge amounts of
medical research on human cells, our understanding of the microbial ecosystem is
very limited: the biodiversity of them is barely studied, not to mention their
interactions with the human host. This project will develop a suite of
statistical theory and methods as well as computational tools to facilitate the
understanding of the intestinal microbial ecosystem. The proposed methods are
fast, efficient, and highly accurate. They can be widely applied to any
metagenomic and metabolomic investigations. Graduate students will be trained by
participating in research activities.&lt;br/&gt; &lt;br/&gt;The main goal of
this project is to extend our knowledge of intestinal microbial ecosystem by
developing novel quantitative methods for microbial species and their
metabolites detection, identification, and quantification in various diseases.
The sensitivity and specificity of our methods permit accurate detection of
microbial species at very low coverage levels. This is a translational
technology that should find substantial use in biomedical researches and drug
developments. More specifically, the PIs shall develop methods for identifying
microbial species especially unknown species, reducing error in metabolomic
analysis, estimating microbial and metabolite distributions, quantifying
microbial or metabolites distributional differences that are associated with
diseases, integrating metagenomic and metabolomic analysis together to study the
microbial ecosystem, building analytical models to link metabolite profiling
with species profiling to understand how metabolites interact with genetic
contents and eventually affect cell metabolism.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.