* 1906389
* NSF Postdoctoral Fellowship in Biology FY 2019: Deciphering CLE Peptide Signaling Pathways in Sunflower (Helianthus annuus)
* BIO,IOS
* 07/01/2019,06/30/2022
* Daniel Jones, Jones, Daniel Scott
* Fellowship Award
* Diane Jofuku Okamuro
* 06/30/2022
* USD 216,000.00

This action funds an NSF National Plant Genome Initiative Postdoctoral Research
Fellowship in Biology for FY 2019. The fellowship supports a research and
training plan in a host laboratory for the Fellow who also presents a plan to
broaden participation in biology. The title of the research and training plan
for this fellowship to Dr. Daniel Scott Jones is "Deciphering CLE peptide
signaling pathways in sunflower (Helianthus annuus)". The host institution for
the fellowship is the University of North Carolina in Chapel Hill and the
sponsoring scientist is Dr. Zachary Nimchuk. &lt;br/&gt;&lt;br/&gt;Unlike
animals, plants continue to develop and generate new organs throughout their
lifespan. To support this continual growth, plants maintain a functional set of
stem cells that can divide and become any other type of cell within the plant as
needed; giving rise to all cells found in stems, leaves, roots, flowers, and
seeds. Even slight changes in the number of stem cells maintained in the shoot
of a plant can have a direct impact on the number and size of organs it
produces. Understanding how stem cell identity is controlled in plants is
directly linked with our ability to influence agronomically important traits
such as fruit size, grain yield or even overall crop health and performance.
This project aims to uncover key genes regulating stem cell identity in
sunflower, an important seed and oil crop. During the course of this project,
tools will be developed to further the use of sunflower as a model research
system for studying species within the sunflower family (Asteraceae), one of the
largest plant families with great economic significance/potential. Support
provided during this fellowship will also enable research opportunities
specifically catered toward first-generation college students at the host
institution. Training objectives include acquiring new skills in comparative
genomics, development, epigenetics and bioinformatics. &lt;br/&gt;
&lt;br/&gt;Understanding how conserved signaling pathways mediate cell-cell
communication to coordinate diverse developmental forms is a central question
within the biology. CLE (CLAVATA3/Endosperm surrounding region-related) peptide
signaling represents a conserved pathway regulating stem cell identity and
organogenesis across highly divergent species. The main objectives of this
project are to: 1) identify CLE signaling components expressed during
inflorescence development in sunflower; 2) define signaling outputs in sunflower
shoot and root meristems in response to CLE peptides via RNA-seq and ATAC -seq;
3) implement CRISPR-Cas9 mutagenesis in sunflower and functionally analyze key
CLE pathway components; 4) complete an intensive scientific training program,
developing new skills in comparative and large-scale genomics while producing
foundational data from which to build an independent research program. All
genomic datasets generated during the course of this study will be deposited
into the publicly accessible Gene Expression Omnibus-GEO
(http://www.ncbi.nlm.nih.gov/geo/) and will be freely shared with the sunflower
community via the Institut National de la Recherche Aronomique (INRA) Sunflower
Bioinformatics Resource site (https://www.heliagene.org/). Additionally, all
CRISPR-Cas9 constructs, optimized for use in sunflower, will be made available
by request and through Addgene, the nonprofit global plasmid repository
(https://www.addgene.org/), for ease of distribution.
&lt;br/&gt;&lt;br/&gt;Keywords: sunflower, RNA-seq, ATAC-seq, transcriptome,
stem cell, intercellular signaling, inflorescence,
flower&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.