* 1907088
* NSF Postdoctoral Fellowship in Biology FY 2019:  Leveraging Quantitative Genetic Mapping Data for Discovery of Novel Regulatory DNA in Solanum lycopersicum
* BIO,IOS
* 08/01/2019,07/31/2023
* Grace Mason, Mason, Grace Alexandra
* Fellowship Award
* Diane Jofuku Okamuro
* 07/31/2023
* USD 285,000.00

This action funds an NSF National Plant Genome Initiative Postdoctoral Research
Fellowship in Biology for FY 2019. The fellowship supports a research and
training plan in a host laboratory for the Fellow who also presents a plan to
broaden participation in biology. The title of the research and training plan
for this fellowship to Grace A. Mason is "Leveraging Quantitative Genetic
Mapping Data for Discovery of Novel Regulatory DNA in Solanum lycopersicum". The
host institutions for the fellowship are the University of California,
Davis/University of Minnesota, Twin Cities and the sponsoring scientists are
Drs. Siobhan Brady and Chad Myers. &lt;br/&gt;&lt;br/&gt;Traditionally, crop
improvement relies solely on selective breeding for desired traits. While
successful, this laborious methodology reduces breeders' ability to quickly
predict traits based on genetic variation as well as improve available breeding
resources. To allow for faster and more systematic breeding, this project aims
to systematically test genetic variation within the tomato genome for their
effects on gene expression and on selected root architecture phenotypes.
Potential regulatory variants will then be tested first for the ability to
influence gene expression and then for its effect on plant root architecture
phenotypes, traits that are generally understudied and could be of potential use
to tomato breeders. The broader impacts of the project include providing hands-
on training in basic research for local high school and undergraduate students
from diverse backgrounds as well educating the general public about tomato
breeding and how modern technologies can allow for non-transgenic crop
improvement. Training objectives include acquiring skills in the use of
quantitative genetics and computational science and tool development in an
economically relevant crop plant.&lt;br/&gt;&lt;br/&gt;Variation in regulatory
DNA is thought to be a major factor for phenotypic variation in wild species as
well as domestication of cultivated crop plants. Enhancers, a type of regulatory
DNA, regulate target gene expression in a spatiotemporal manner and can be
relatively distal to their target genes, thus making them difficult to
characterize. In animal systems, hundreds of enhancers are well-characterized
and have been linked to target genes. In plants, despite recent efforts to
identify genome-wide regulatory elements, only thirteen enhancers in four plant
species have been directly linked to target genes. Of these, only variation in
the tb1 enhancer in Zea mays (corn) is demonstrably linked to domestication and
crop improvement. To address this disparity, the overall goal of this project is
to discover allele-specific plant enhancers and link them to their target genes.
Specifically, regulatory enhancer elements will be identified in Solanum
lycopersicum (tomato) using novel computational methods for genetic mapping data
with a specific focus on root architecture traits. The research will take
advantage of several diverse genetic resources, including an extensive catalog
of domesticated tomato accessions, existing quantitative trait data sets, as
well as a genetically divergent set of isogenic introgression lines. Large-scale
enhancer assay techniques will be used to yield allele-specific regulatory
regions linked to root architecture phenotypes. While there has been intense
effort to characterize the regulatory landscape in several cultivated plants,
these experiments will be the first to directly test plant-based association
mapping variants using large-scale sequencing technologies. Sequencing data from
this project will be archived and publicly available through the National Center
for Biotechnology Information (NCBI) GEO and Probe databases. Additionally, all
CRISPR-Cas9 constructs will be made available by request and through Addgene,
the nonprofit global plasmid repository (https://www.addgene.org/). Software and
analysis scripts will be made available through
GitHub.&lt;br/&gt;&lt;br/&gt;Keywords: tomato, gene expression, genetic
variation, regulatory DNA, enhancer element, root architecture, quantitative
genetics&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.