* 1909122
* FET: AF: Small: Spatial Stochastic Modeling and Simulation with application in the Caulobacter Cell Cycle control
* CSE,CCF
* 10/01/2019,09/30/2024
* John Tyson, Virginia Polytechnic Institute and State University
* Standard Grant
* Mitra Basu
* 09/30/2024
* USD 499,906.00

In the past decade a lot of data have been accumulated about protein
localization and gene-protein interactions during the asymmetric division cycle
of a free-living bacterial cell (Caulobacter crescentus). An asymmetric cell
division produces two daughter cells with different cellular fates, whereas
symmetric cell divisions give rise to daughter cells of equivalent fates. For
example, stem cells divide asymmetrically to give rise to two distinct daughter
cells: one copy of the original stem cell as well as a second daughter
programmed to differentiate into a non-stem cell. In this project a
comprehensive computational model of this control system is built. Mathematical
modeling is a well-established method for exploring hypotheses about molecular-
regulatory mechanisms in living cells. These models integrate experimental
evidence into a computer representation of molecular interactions. Computer
simulations then serve to test the hypotheses against known experimental facts
and to predict the outcome of novel experimental studies. It helps the
scientific community to gain deeper insight into the molecular mechanisms
controlling the asymmetric division cycle by building mathematical models of
gene expression and protein localization within the cell. This project also
provides training for graduate students and undergraduate researchers in
computational cell biology and mathematical modeling.&lt;br/&gt;&lt;br/&gt;In
this project two types of models are built: deterministic and stochastic.
Deterministic models (based on differential equations) of the dynamics of
protein synthesis, degradation, stoichiometric and catalytic interactions, and
spatial redistribution are useful in describing the average behavior of a
population of bacterial cells. To describe the precise spatial distribution and
temporal dynamics of specific proteins in single cells (as measured by modern
microscopic studies of fluorescently labeled proteins), detailed stochastic
models of chromosome movement, gene expression, and protein dynamics are built.
Simulation results of the model are compared with wet-lab experimental data. To
collect simulation results, accurate yet efficient simulation methods, with
corresponding numerical analysis, are also developed.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.