* 1913165
* Role of histone modifying enzymes in regulating alternate active versus silent gene expression in plants
* BIO,MCB
* 08/01/2019,07/31/2024
* Vitaly Citovsky, SUNY at Stony Brook
* Standard Grant
* Stephen DiFazio
* 07/31/2024
* USD 746,979.00

This project addresses a poorly studied, yet fundamental, question of how
enzymes that chemically modify chromosome-associated proteins can elicit
opposing effects on gene expression, i.e., activating or silencing the gene. A
combination of genetic and genomic approaches will be applied in the model
plant, Arabidopsis thaliana, to test the idea that these alternate outcomes
arise from coordinating the action of distinct enzymes. The research is
integrated into educational activities including: research experiences for
undergraduate students from two predominantly undergraduate and minority-serving
institutions--Hunter College and Lehman College--in New York City; training for
high school teachers through a series of seminars and summer workshops offered
through Stony Brook's Center for Science and Mathematics Education; and
mentoring of a postdoctoral researcher for an independent career in academia,
industry, or the Federal government. &lt;br/&gt;&lt;br/&gt;Post-translational
histone modifications are central to gene regulation in all eukaryotic
organisms, determining the active or inactive state of the chromatin. These
modifications are dynamic, effected by writers and erasers, i.e., histone
modifying enzymes that add or remove specific functional groups. Moreover, one
histone modification often promotes generation of another, a process termed
histone crosstalk, that shapes the histone modification landscape and its
transcriptional outcomes. In plants, one major class of important, yet poorly
characterized, erasers are histone deubiquitinases, implicated in diverse
aspects of the plant life cycle. The study focuses on histone deubiquitinase
OTLD1, which is known to deubiquitylate histone H2B, associate with the
chromatin mainly by binding to histone H3, and directly interact with another
histone modifying enzyme, the histone lysine demethylase KDM1C. OTLD1 is found
at the promoter regions of target genes and, fascinatingly, OTLD1 can both
repress and activate its direct targets. These findings identify a gap in our
understanding: how does deubiquitylation of the same type of histone by the same
histone deubiquitinase elicit two opposing effects on transcription of the
direct target genes and what is the biological significance of this ability?
This question will be addressed by two specific aims: (i) define how association
of OTLD1 and KDM1C with different histones variants affects the resulting
histone modifications and transcriptional regulation of the target genes, and
(ii) discover whether KDM1C and/or OTLD1 are down-regulated via the
ubiquitin/proteasome system and study the effects of this regulation on
OTLD1/KDM1C-mediated histone modifications and their transcriptional outcomes.
The results are expected to advance understanding of new junction in chromatin-
level processes leading to transcriptional repression or
activation.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.