* 1914596
* Collaborative Research: Uncovering How Riboswitches Exploit Out-of-Equilibrium RNA Folding Pathways to Make Genetic Decisions
* MPS,PHY
* 08/15/2019,07/31/2023
* Alan Chen, SUNY at Albany
* Continuing Grant
* Krastan Blagoev
* 07/31/2023
* USD 154,491.00

This work will uncover new principles by which riboswitch RNAs exploit out-of-
equilibrium RNA folding and ligand binding to make genetic decisions. The tools
developed will also be applicable to understanding the role of co-
transcriptional folding and assembly of ribozymes, regulatory RNA structures
within non-coding and coding RNAs, and assembly of RNA-protein complexes such as
the ribosome and spliceosome. These studies will also contribute understanding
of the physical principles of out-of- equilibrium co-transcriptional RNA
folding, help address long-standing questions about how dynamic RNA structures
coordinate genetic processes, and shed light on how natural RNAs exploit out-of-
equilibrium mechanisms to efficiently fold on extremely rugged free-energy
landscapes. Since co-transcriptional RNA folding happens every time an RNA is
synthesized, the broader impacts of this research include developing general
principles and techniques that can be used to understand a wide array of
fundamental cellular processes from gene expression to regulation. The study of
riboswitches also has several broader impacts towards societal goals, since they
can be used as biosensors within new molecular diagnostics, and they are
important targets for new classes of antibiotics. Broader impacts of integrated
research and education will come from a multi-pronged plan including conducting
demonstrations of riboswitch diagnostics to school-age groups, mentorship of
undergraduate researchers, and delivering hands-on tutorials of computational
RNA folding approaches to broader scientific
communities.&lt;br/&gt;&lt;br/&gt;The overarching goals of this proposal are to:
(i) Uncover detailed mechanisms of how ligand binding bifurcates out-of-
equilibrium RNA cotranscriptional folding pathways to enact genetic decisions in
riboswitch RNAs; and (ii) Develop and apply new hybrid experimental-
computational frameworks that can reconstruct RNA cotranscriptional folding
pathways at the secondary and tertiary structure levels. The education plan
focuses on integrating this research into hands-on demonstration activities
targeted towards school age children, undergraduate researcher mentorship, and
hands-on training tutorials for the broader scientific community. The post-
genomic era has ushered in a new appreciation that RNAs play central roles in
regulating, maintaining and defending the genomes of all organisms. However, a
critical knowledge gap remains: we have relatively little understanding of the
dynamic folding pathways that RNAs undergo as they are being synthesized during
transcription, thus hindering our fundamental understanding of how RNA
structures enact critical cellular functions such as catalysis, gene expression
regulation, and cellular sensing. To address this gap, the PIs recently
innovated and validated a hybrid experimental-computational approach that uses
high-throughput RNA structure chemical probing data with computational
algorithms to generate two and three-dimensional models of RNA cotranscriptional
folding pathways. One central objective of this proposal is to extend this
approach to incorporate more complex RNA structures and interactions such as
pseudoknots relevant to a broad range of functional cellular RNAs. The second is
to uncover biophysical principles of how out-of-equilibrium RNA fluctuations
during cotranscriptional folding influence RNA function. The latter will be
pursued through the use of riboswitch RNAs as model systems, which make ligand-
mediated genetic decisions, use the dynamic formation of broadly utilized RNA
structures to do so, and have broader impact relevance for fundamental biology
and biotechnologies.&lt;br/&gt;&lt;br/&gt;This project is being jointly
supported by the Physics of Living Systems program in the Division of Physics
and the Molecular Biophysics program in the Division of Molecular and Cellular
Biosciences.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.