* 1921641
* URoL:  Epigenetics 2:  Reverse Engineering Human Epigenetic Machinery in Yeast
* BIO,EF
* 09/01/2019,08/31/2024
* Jef Boeke, New York University Medical Center
* Standard Grant
* Anthony Garza
* 08/31/2024
* USD 2,999,996.00

DNA is the blueprint of life that provides instructions for producing various
functional proteins within a cell. But DNA in living cells exists in a protein
environment/packaging system called "chromatin" that controls when functional
proteins are produced, or when the instructions from DNA to produce proteins are
silenced. The way chromatin controls the production of proteins can be
influenced by age of the organism, environment that the organism is living in,
and whether the organism is diseased. Much is still unknown about how chromatin
regulates the production of proteins, but a better understanding could lead to
advances in our knowledge of how organisms adapt to an environment or how better
to treat certain diseases. This project will leverage novel molecular biology
techniques to better understand key components of the complex human chromatin
organization by progressively rewriting (or rebuilding) the system within
budding yeast, a much simpler laboratory model. This will enable the dissection
of the complicated regulatory circuits that control the production of functional
proteins from DNA in humans, other animals, and other multicellular organisms.
Two postdoctoral researchers and one graduate student will be trained in state-
of-the-art molecular biology methods and analyses. Results from the research
will be disseminated broadly by a yeast art program to be exhibited on the New
York City subway, by partnering with the New York City "biobus" program, and by
presentations at a new "epigenome engineering" meeting.
&lt;br/&gt;&lt;br/&gt;The epigenome equips many eukaryotes with the ability to
generate stable and distinct cell types from a single identical genome. This
multipurpose ability stems from interconnected chromatin organizing pathways
acting at multiple length scales across chromosomes to regulate gene expression,
maintain cell identity, and adapt to environmental changes. What rules lead to
one cell's chromatin organization versus another? Advances in genome-wide
methodologies have revealed the patterns underlying chromatin architectures, but
inferring causal effects remains difficult. Mechanistic biochemical approaches
that require complex protein purifications and use minimal chromatin substrates
have limited ability in recapitulating living systems. Multigene knockout
studies can lead to cellular dysfunction and pleiotropy, making it difficult to
decouple overlapping phenotypic effects. In this research project, human
epigenetic pathways will be reconstituted within the eukaryote budding yeast
(Saccharomyces cerevisiae) at four levels to provide a bottom-up approach for
unraveling principles of chromatin organization and inherited gene expression.
The four levels are:&lt;br/&gt;(1) Histones: expanding the ability to generate
diverse human histone variant architectures.&lt;br/&gt;(2) Heterochromatin:
human pathways such as Polycomb Group Proteins will be imported into budding
yeast to generate the repressive histone Post-Translational Modifications (PTMs)
H3K27me, H3K9me, H2AK119ub and CpG methylation.&lt;br/&gt;(3) Euchromatin: the
COMPASS complex in yeast will be replaced by the human pathway that generates
the activating histone PTM H3K4me.&lt;br/&gt;(4) Topologically Associating
Domain (TAD) structures: human-like TAD structures will be engineered using
human cohesin and CTCF complexes.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.