* 1921412
* URoL:  Epigenetics 1:  Control by Noncoding RNA
* BIO,EF
* 08/01/2019,07/31/2022
* Xiaojun Tian, Arizona State University
* Standard Grant
* Junping Wang
* 07/31/2022
* USD 299,999.00

An organism's genetic information is coded by its DNA sequence, however the
activity of genes such as whether they are expressed or silenced, is controlled
by "epigenetic marks" on chromatin or modifications of DNA and histones.
Epigenetic marks are stable and can be inherited from generation to generation.
They also can be disrupted, leading to activation of normally silenced genes and
the silencing of normally active genes, the result of which can be developmental
abnormalities and other complex diseases. It is well understood that the
interactions among DNA, histone proteins and other regulatory factors such as
histone modification enzymes, transcription factors, and regulatory noncoding
RNAs determine the epigenetic marks including DNA and histone modifications and
thus change the 3D packing of nucleosome and gene activities. This project
provides new integrated mathematical modeling frameworks for epigenetics to
advance our fundamental knowledge on the coupling of the different layers of
epigenetics. Researchers involved in the project, including graduate,
undergraduate and high school students, will receive solid interdisciplinary
training in mathematical modeling of epigenetics-regulated biological systems,
including model construction/validation, sensitivity analysis, and
deterministic/stochastic simulation.&lt;br/&gt;&lt;br/&gt;Noncoding RNAs serve
as binding scaffolds for RNA-binding proteins to recruit epigenetic enzymes and
thus functions as key regulators that couple epigenetic and genetic regulations.
Despite the recognized importance of noncoding RNAs in the regulation of
epigenetics, and the abundance of accumulated data in this area, quantitative
tools are still lacking to characterize different modes of epigenetic control by
noncoding RNAs. The lack of quantitative models of noncoding RNA mediated
epigenetic control is largely due to the multiscale coupling of different layers
of epigenetics, including histone modification, DNA methylation, and 3D
chromosome folding. It is important to determine how these multiscale
interactions determine the 3D organization of the chromatin and gene activities.
To achieve this goal, a general mathematical framework will be developed to
study the nonlinearity of various modes of epigenetic control by noncoding RNAs,
and the emergent properties of various recurring regulatory motifs that couples
epigenetic and genetic regulation through noncoding RNAs will be investigated.
This research will not only provide valuable tools that are currently lacking to
dissect the regulation of epigenetics by noncoding RNAs and critical
implications for understanding the associated rich nonlinear dynamics and
emergent properties, but also lay the foundation for the modeling of a host of
biological systems that are regulated by a multiscale epigenetics-genetic
regulatory network.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.