* 1928344
* EAGER: Characterization of a putative Xanthomonas-Pseudomonas disease complex of Cotton
* BIO,IOS
* 07/01/2019,06/30/2023
* Rebecca Bart, Donald Danforth Plant Science Center
* Standard Grant
* Diane Jofuku Okamuro
* 06/30/2023
* USD 376,284.00

In agricultural settings, plants are hosts to complex microbial communities.
Competition is common within these communities as diverse microbes compete for a
limited supply of nutrients, but there are also opportunities for collaboration.
In the context of the "arms race" that defines host-pathogen interactions,
pathogen-pathogen collaboration represents a potentially powerful mechanism for
disease organisms to overcome host resistance strategies, with severe
implications for agriculture. As an example, in 2001, cotton bacterial blight
(CBB), a disease for which there are resistant cotton varieties, re-emerged
across the southern USA. Surprisingly, the disease was observed on CBB-resistant
cotton plants. Closer examination revealed the CBB pathogen as well as a second
bacterial pathogen on these diseased cotton plants. The co-occurrence led to the
hypothesis that the two bacterial pathogens are collaborating to suppress host
resistance responses and cause disease. This project tests that hypothesis using
genomics and molecular biology approaches. If confirmed, this research will have
revealed a new frontier of pathogen strategies to overcome host resistance that
will need to be considered in the future.&lt;br/&gt;&lt;br/&gt;CBB is caused by
Xanthomonas citri pv. malvacearum (Xcm). Xcm and a second pathogen, Pseudomonas
syringae (Ps), were co-isolated from cotton plants that are resistant to Xcm.
Xcm and Ps may, therefore, be forming a disease complex that suppresses the
resistance triggered by Xcm. This research aims to reveal the putative
mechanism(s) of collaboration between Xcm and Ps. The first aim will establish
experimental systems and assays, adapted from classic plant pathology
techniques, to carefully define points of potential interaction between Xcm and
Ps. The second aim will seek potential molecular mechanisms of collaboration
between Xcm and Ps. For example, Ps may use its repertoire of type III effectors
to repress the resistance response normally triggered by Xcm. This hypothesis
will be tested by comparing plant defense gene expression after inoculation with
Xcm, Ps and co-inoculation with Xcm and Ps. RNA sequencing and differential gene
expression analyses will reveal the relevant virulence mechanisms employed by
both pathogens and the response triggered within cotton. The final aim will
directly test the roles of specific bacterial genes in any interaction by
constructing genetic knockouts in Xcm and/or Ps and conducting pathogenicity
assays. This project breaks important new ground by considering the effect of
multiple bacterial plant pathogens attacking the same host simultaneously.
Understanding the rules that define these complex interactions is required to
meet the agricultural needs of an increasing population and changing
environment.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.