* 1929737
* NSF/MCB-BSF Evolution of gene expression: from static patterns to dynamic systems
* BIO,MCB
* 09/01/2019,08/31/2024
* Patricia Wittkopp, Regents of the University of Michigan - Ann Arbor
* Standard Grant
* Anthony Garza
* 08/31/2024
* USD 648,800.00

Each individual's genome gives rise to a large array of cellular phenotypes
depending on which genes are expressed. Gene expression is thus an essential
step in converting genotypes into phenotypes. The regulation of gene expression
is responsive to the environment, allowing changes in phenotypes that can help
an organism adjust to a variety of living conditions. To date, most studies of
genetic variation affecting gene expression have been conducted within a single
environment, making it difficult to know how genetic changes affect the
environment-dependent aspects of gene regulation. This project will fill this
knowledge gap by examining differences in regulatory networks that exist among
strains and species of yeast. Experimental and computational advances developed
during the course of this work will help other researchers who study gene
expression and the evolution of regulatory networks. A museum exhibit will also
be developed during the course of this work and will help communicate the
importance of evolutionary biology to the general public.
&lt;br/&gt;&lt;br/&gt;Mutations in either cis-regulatory DNA sequences or trans-
regulatory factors that interact with these cis-regulatory sequences can alter
gene expression. Over the last 15 years, genomic studies of gene expression have
investigated the relative contributions of cis- and trans-regulatory changes to
expression differences within and between species, but these studies have
focused almost exclusively on levels of gene expression assayed at one point in
time, under one set of conditions. Consequently, the impact of cis- and trans-
regulatory changes on dynamic aspects of gene expression, such as how expression
changes in different environments or throughout the cell cycle, remains less
understood. The proposed project will fill this knowledge gap by identifying the
genetic mechanisms responsible for variation in dynamic gene expression within
and between species. To achieve this goal, the researchers will work at multiple
evolutionary scales, employ both genomic and single-gene methods, integrate
cutting-edge empirical and computational tools, and harness the complementary
resources and expertise of two labs that have been studying gene expression for
more than a decade. More specifically, a panel of budding yeast strains, species
and hybrids characterized in the Wittkopp lab will be profiled under a wide
array of conditions using a setup available in the Barkai lab. Data analysis
will proceed through tight collaboration: the Barkai lab will perform the
initial processing and quality controls of the data, and establish a data
structure amendable for high-throughput analysis and visualization. Both labs
will then contribute to the establishment of a bioinformatics platform that will
integrate the data with existing genomics and functional genomics data. Relying
on its expertise in evolutionary theory, the Wittkopp lab will perform in-depth
biological analysis of the data to identify recurrent principles governing the
variation between networks and define specific predictions exemplifying these
principles. Both labs will contribute to follow-up experiments designed to
validate specific predictions.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.