* 1929053
* TRTech-PGR: Improvement of the genome sequence assembly of Chinese Spring wheat
* BIO,IOS
* 09/01/2019,08/31/2020
* Jan Dvorak, University of California-Davis
* Standard Grant
* Gerald Schoenknecht
* 08/31/2020
* USD 219,174.00

Common bread wheat is globally one of the two most important crops and is third
in economic importance in this country. A reference-quality genome sequence for
wheat is an essential resource for advancing wheat genetics and breeding. The
long-awaited sequence of the genome of bread wheat cultivar Chinese Spring (CS)
was recently published by an international consortium of scientists. The CS
cultivar has served as a genetic model since the 1950s, and the sequence of its
genome is expected to serve as a reference for the wheat genome. Considering the
large size of the wheat genome, which is five times the size of the human
genome, the full sequence was a remarkable accomplishment. To fully leverage the
value of the genome, this project continues improving the published genome by
making use of recent technological advances that will further improve the
sequence by filling gaps and resolving inconsistencies in the assembly. The
sequence will be used to analyze the structure and evolution of the wheat genome
and will be provided to others to identify and catalogue all of the genes in the
sequence. The sequence will be made publicly available and will serve an
excellent training ground for young scientists in genomics.

The International Wheat Genome Sequencing Consortium (IWGSC) recently published
the assembly of the bread wheat cv Chinese Spring (CS) genome sequence (IWGSC
RefSeq v1.0), to serve as a reference for the wheat genome. The sequence was a
whole-genome-shotgun, short-read assembly with remaining gaps that could be
filled by making use of long-read sequencing technology and of Bionano optical
maps. To improve the sequence, the IWGSC RefSeq v1.0 pseudomolecules will be
disassembled into scaffolds, and optical maps will be used to detect and resolve
chimeric scaffolds, to anchor unassigned scaffolds, and to correct ambiguities
in positions and orientations of scaffolds and super-scaffolds. A long-read
assembly of the CS genome (Falcon Triticum 1.0) will be used to close gaps.
Genetic and optical maps will be used to reconstruct the pseudomolecules of the
21 wheat chromosomes. The pseudomolecules will be employed in the analyses of
structure and evolution of the wheat genome. The sequence will be made publicly
available through a project website, and will be released to the NCBI and IWGSC
databases. In collaboration with the IWGSC, genes will be reannotated in the new
sequence. Awareness about the sequence will be disseminated on GrainGenes and
via publications and presentations in scientific meetings. The project will
afford excellent training opportunities for postdoctoral scholars and young
scientists carrying out the bioinformatics work.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.