* 1940829
* EAGER:  Lambda-Red-enhanced homologous recombination in plants
* BIO,IOS
* 12/01/2019,11/30/2023
* Jose Alonso, North Carolina State University
* Standard Grant
* Gerald Schoenknecht
* 11/30/2023
* USD 299,988.00

The ability to edit the information in an organism's genome is transforming not
only the way scientists address basic biological questions, but also how
solutions to deal with critical societal problems such as human diseases, food
security, biofuel production, etc. are sought. The predominant genome editing
technologies currently in use in plants and animal systems require the
introduction of breaks in the genome at the precise locations where the changes
or edits are wanted. Although highly efficient, current genome editing
procedures are still laborious and not free of risks due, among other things, to
undesired breaks in the genome. Although high-efficiency genome editing
approaches that do not rely on the introduction of chromosomal breaks in the
genome already exist, they have only been proven to work in bacterial systems.
Using the current understanding of how DNA-break-free genome editing approaches
work and what their requirements are, this project aims to modify components of
an existing bacterial system and adapt them to function in plant cells. In
addition to the benefits of developing a new DNA-break-free genome editing
approach for plants, the generated knowledge would inform similar developments
in animal systems. This project will provide an ideal training platform for
researchers and undergraduate students to experience first-hand how basic
biological knowledge can be translated into potentially transformative new
technologies. To further inform the public and motivate young scientists, the
project will lead workshops at a local museum on principles of plant biology and
genetics. &lt;br/&gt;&lt;br/&gt;The main goal of this project is to develop a
Lambda-Red-based homologous recombination (HR) system for eukaryotic cells. The
implementation of such system would not only provide an alternative to the
current genome editing approaches, but may also offer important advantages, such
as the elimination of the need to induce potentially dangerous dsDNA breaks in
the genome, very short sequence homology requirements, high efficiency, and the
potential to generate large arrays of modifications, from single nucleotide
changes to large insertions and deletions. Importantly, the molecular mechanisms
behind the Lambda-Red system have been thoroughly investigated, thus providing
critical clues on the requirements that need to be fulfilled in order to test
the feasibility of Lambda-Red-mediated HR in a eukaryotic system. These
requirements include: 1) an actively replicating genome, 2) a linear DNA
fragment that can serve as the repair template, and 3) active recombineering
enzymes targeted to the nucleus of the cell. To fulfill these requirements,
codon-optimized, nuclear-localized recombineering enzymes will be expressed in
plant cells together with a virus-derived replicon that can serve as the source
for repair template DNA. A codon-optimized, nuclear-localized meganuclease will
be also expressed to linearize the viral replicons to generate the required
linear repair template. The work, if successful, will open doors to new means of
precise genome manipulation for the benefits of agriculture and
beyond.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.