* 1942898
* CAREER: Unraveling the metabolic interface between pathogenic Pseudomonas syringae bacteria and host plants
* BIO,IOS
* 03/01/2020,02/28/2025
* Jeffrey Anderson, Oregon State University
* Standard Grant
* Courtney Jahn
* 02/28/2025
* USD 1,139,270.00

The overall goal of this project is to develop an integrated program of research
and education that advances knowledge of how bacterial pathogens infect plants,
and introduces undergraduates and under-served high school students to the
excitement of discovery in STEM. Bacterial diseases of plants cause significant
losses in overall yield and marketability of many important US crops. Although
plants have an immune system that can provide effective resistance against
infection, bacteria have evolved sophisticated counter-measures that can
effectively suppress these host defenses. As a result, an important determinant
of infection outcomes is how rapidly both bacteria and host plant can deploy
their respective virulence and defense strategies. This project will investigate
chemical signaling events that occur between plants and bacteria, with a focus
on how pathogenic bacteria perceive plant-derived metabolites to start an
infection, as well as how plants may interfere with this perception process.
Bacterial diseases in crops are frequently controlled by non-specific
antimicrobials such as copper sprays. Knowledge gained from this project could
lead to the development of novel chemical inhibitors of bacterial diseases, as
well as engineered crops that are more disease resistant. Research objectives of
this project will be integrated into three main educational activities: 1) on-
campus summer STEM camps for students from under-served rural and urban high
schools in Oregon and Washington, 2) training of students in a functional
genomics course and 3) summer-long laboratory research internships for high
school students. &lt;br/&gt;&lt;br/&gt;The plant pathogen Pseudomonas syringae
must rapidly deploy its type III secretion system (T3SS) at early stages of
infection to be virulent. Specific plant-derived organic acids and amino acids,
together with simple sugars, induce T3SS-encoding genes in P. syringae, yet how
these signals are perceived is poorly understood. In previous work the
investigator identified a transcription factor SetA that is required for maximal
sugar-induced expression of T3SS genes in P. syringae. The project’s first
objective is to determine how SetA regulates expression of T3SS master regulator
hrpL and to identify the intracellular metabolite signal(s) that may regulate
SetA activity. The investigator also identified a two-component system, AauS-
AauR, that directly links detection of host-derived amino acid signals to
regulation of T3SS-encoding genes. A second objective is to determine how host
signals activate AauS and how the downstream response regulator AauR regulates
T3SS-encoding genes. Proposed methods include biochemical approaches to assess
protein-metabolite, protein-DNA and protein-protein interactions that regulate
the functions of SetA and AauS-AauR. Using metabolomics, the investigator also
discovered that specific T3SS-inducing metabolites decrease in abundance in
leaves during a host defense response. The project’s third objective is to
determine the molecular mechanism(s) that regulate the abundance of these
metabolites, and how these host signals are perceived by P. syringae through
AauS/AauR- and SetA-independent mechanisms.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.