* 1945854
* EAGER:  Single-cell, spatial transcriptomics of plant-fungal interactions using a maskless array technology
* BIO,MCB
* 09/01/2019,08/31/2023
* Kirk Czymmek, Donald Danforth Plant Science Center
* Standard Grant
* Stephen DiFazio
* 08/31/2023
* USD 299,396.00

The goal of this proposal is to develop new technology to allow detection of RNA
molecules produced by single, living cells. Cells contain thousands of RNAs, and
the RNAs produced from one cell may differ from the RNAs in a neighboring cell.
Detecting these differences on a cell-by-cell basis is not easy, especially for
plant cells. This project will create a method that transcends past technical
limitations by preserving spatial information, thereby making it possible to
tell which RNAs come from which cells. The new method will be put to the test by
identifying RNAs in single plant cells that are undergoing infection by a fungal
pathogen. This is a high-risk project that involves a complicated experimental
design that will be challenging to implement. If successful, the results will
provide plant biologists with a revolutionary new tool for studying cell-
specific gene-to-RNA-to-trait relationships. This could lead to new discoveries
in understanding plant diseases, which in turn could advance efforts to breed
disease resistant plants. The project will be led by a postdoctoral scientist,
who will receive multidisciplinary training in plant biology, microscopy,
genomics, and computational science. &lt;br/&gt;&lt;br/&gt;Several methods have
been established in plants for transcriptional analysis in tissue sections, cell
types or even single cells. These methods typically require cell sorting,
transgenic plants, protoplast production, or other damaging or laborious
processes. Most of these technologies lose most or all spatial resolution during
implementation, while those that offer high spatial resolution for RNA
localization lack breadth in the number of transcripts characterized,
particularly for more challenging RNAs like small RNAs. Plant small RNAs
function in many more pathways than other organisms, yet there are no techniques
for their single- cell analysis other than fluorescent in situ hybridization
microscopy typically performed with just one or a few small RNAs (or mRNAs) at a
time. A new approach, spatial transcriptomics, has been described that overcomes
many of these limitations for mRNA analysis; RNA-seq libraries are made using
microarrays with high positional resolution within sectioned materials. Yet,
spatial transcriptomics has not been developed for small RNAs or cleaved mRNAs,
and the implementations described to date still suffer from various
shortcomings. This project will substantially advance this method, using high-
resolution arrays (14 micron grids with 1 micron linear gaps) built on a
maskless array synthesis instrument, to enable near single-cell analysis of
diverse types of RNAs in plants and potentially other organisms. The test bed
for this work is a system in which such high-resolution analyses would be highly
informative, namely plant-pathogen interactions, for which localized and
systemic transcriptional responses will be measured. &lt;br/&gt;&lt;br/&gt;This
award is jointly funded by the Genetic Mechanisms and the Cellular Dynamics and
Function programs in the Division of Molecular and Cellular Biosciences in the
Biological Sciences Directorate.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.