* 1948117
* CRII: III: Deep Learning Methods for Protein Inter-residue Distance Prediction
* CSE,IIS
* 06/15/2020,05/31/2023
* Badri Adhikari, University of Missouri-Saint Louis
* Standard Grant
* Sylvia Spengler
* 05/31/2023
* USD 163,535.00

Proteins are the gears of life. They do most of the work in our cells. Insulin
and hemoglobin are two examples. Similar to mechanical gears, proteins function
because of their unique and precise three-dimensional structure. For example,
hemoglobin carries oxygen in our body because of its precise structure. Learning
the precise structures of proteins helps us to better understand life processes
and disease mechanisms. This knowledge also enables the design of novel drugs.
Determining the structures of a protein in a lab can take months to years. These
experiments sometimes fail with no results. As an alternative, artificial
intelligence (AI) based computer algorithms are often used to predict
structures. Computationally predicting protein structures has been one of the
most significant and fundamental challenges during the past 50+ years. Recently,
specific types of AI methods, known as convolutional neural networks, have been
found to be most effective for solving this problem. However, even the best AI
methods are inadequate for many proteins. It is currently an open question which
types of AI algorithms are best suited for solving this problem. This project
will push forward the current advances in AI-based methods for predicting
structures of proteins. The main novelty of this project will be in developing
AI-based methods particularly suitable for protein structure-related tasks and
investigating what input variables drive the predictability. The project will
increase the understanding of which kinds of AI algorithms are effective for
solving fundamental biological problems, beginning with protein structure
prediction. Ultimately, this will lead to a better understanding of many
incurable diseases and enable a much faster development of therapeutics that
will positively contribute to health and welfare of individuals, communities,
and the nation.&lt;br/&gt;&lt;br/&gt;Current progress in the field of protein
structure prediction comes from deep learning methods and the availability of a
large number of protein sequences obtained from high throughput sequencing. Deep
learning methods can predict inter-residue contacts and distances, the key
information for structure prediction, much more accurately. Despite recent
advances, the overall problem remains far from being solved. While many variants
of convolutional neural networks and residual networks have been investigated
for solving the problem of contact and distance prediction, the potential of
many other types of deep learning methods such as capsule networks have not been
explored. The work scope is set to achieve high accuracy protein structure
prediction using novel methods for protein inter-residue distance prediction,
improved feature engineering, and improved selection of high-quality multiple
sequence alignments. This work will use datasets at various scales - small,
medium, and large - that are representative of the protein structure database.
The research will develop novel methods for protein distance prediction using
existing and novel variants of capsule networks, deep learning methods for
predicting the quality of multiple sequence alignments, and investigate what
variables drive the accuracy of deep learning models. These research efforts
will increase our understanding of the applicability and limitations of diverse
deep learning algorithms in solving the protein folding problem. The findings
will also be helpful in solving similar problems in computational
biology.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.