* 1949036
* EAGER: IIBR Informatics: Deep learning tools for the identification of RNA modifications from direct RNA sequencing data
* BIO,DBI
* 04/01/2020,03/31/2023
* Anireddy Reddy, Colorado State University
* Standard Grant
* Jean Gao
* 03/31/2023
* USD 300,000.00

The role of mRNA modifications as a regulatory process that affects gene
expression at multiple levels is not well studied or understood. Current
experimental tools for determining RNA modifications are laborious, noisy, and
often do not provide exact locations of modified bases. Sequencing using Oxford
Nanopore technology offers multiple advantages over Illumina sequencing
including long reads and the ability to directly sequence RNA without the need
for amplification, leading to reduced bias in coverage and the potential ability
to uncover modified bases. The potential for discovering modified bases is still
unfulfilled due to the lack of tools for this task. This project seeks to to
make it significantly easier to identify RNA modifications globally and to help
uncover the biological roles of the over 150 different types of RNA
modifications. The challenge in the proposed research is that of the relatively
small number of known modified bases, requiring clever design of sufficiently
large labeled datasets, and necessitating the use of deep learning training
algorithms that can succeed despite the relatively smaller datasets. The project
draws upon recent developments in deep learning for tasks with few available
labeled training examples to develop novel ways in which deep learning
architectures for base calling can be applied to calling of modified RNA
bases.&lt;br/&gt;&lt;br/&gt;The proposed work will be transformative for
research into RNA modifications and will enable the use of nanopore sequencing
as a one-stop-shop for this purpose. Furthermore, it has the potential of
leading to improved methods for the detection of targets of RNA-binding
proteins, as several novel methods for this task are based on detecting modified
RNA bases. Oxford Nanopore does not release the code for their production base
calling software as open-source, limiting the ability of the research community
to extend their methods to handle modifications. The tools designed as part of
this work will provide a flexible open-source alternative, enabling progress on
base calling of nanopore data.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.