* 1953324
* Uncovering the regulatory DNA logic of the Drosophila innate immune response
* BIO,MCB
* 05/01/2020,04/30/2022
* Zeba Wunderlich, University of California-Irvine
* Standard Grant
* Stephen DiFazio
* 04/30/2022
* USD 999,999.00

Many biological processes, from early development to the immune response, rely
on the “non-coding” part of the genome, about which researchers have much to
learn. An organism’s DNA can be separated into two categories: coding DNA, which
has the directions for how to build proteins – the molecular machines of the
cell, and non-coding DNA, which has many functions including the instructions
for when, where, and how much protein should be built. The majority of the human
genome is non-coding DNA, and its proper functioning is essential for life. This
project aims to decipher some of the rules that govern function of non-coding
DNA in the innate immune response to infection and to compare these rules to
non-coding DNA needed for other biological processes. Success will give
scientists specific information about how non-coding DNA orchestrates the immune
response and a deeper understanding about how a large part of the genome
functions in general. Both graduate and undergraduate students will participate
and be trained in the work. This project also includes a broader impacts plan to
improve quantitative literacy among biology undergraduate and graduate students,
which will better equip them for modern jobs that increasingly require
quantitative skills.&lt;br/&gt;&lt;br/&gt;Virtually every biological process,
from development to immunity, is controlled by a gene regulatory network (GRN),
a network of signaling molecules and transcription factors that control gene
expression. Years of work have uncovered how the sequence of regulatory DNA,
e.g. promoters and enhancers, can specify a GRN’s function. This work largely
has focused on developmental GRNs, which are uniquely constrained by evolution:
selection favors GRNs that drive a stereotyped developmental program in spite of
perturbations. To decipher which principles of developmental GRNs translate to
lesser studied non-developmental GRNs, which have diverse functions and are
subject to different evolutionary pressures, this project will study the
Drosophila innate immunity GRN. In Aim 1, allele-specific RNA-seq will be used
to characterize the natural variation in transcriptional response to infection
between D. melanogaster lines. These data will allow the identification of genes
whose expression variation is driven by changes in regulatory DNA. Comparisons
of the pattern of expression variation using different infection conditions will
be made. In Aim 2, in vivo and in vitro functional genomics assays will be used
to identify immune-responsive enhancers and to generate sequence-to-function
models of these enhancers. Additional experiments will validate and refine the
models of enhancer function. The project also includes a plan to implement
problem-based learning of numerical literacy components into a large,
undergraduate class, which will improve the quantitive skills of biology
students and be disseminated for use by others.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.