* 2107108
* III: Small: Learning Multi-scale Sequence Features for Predicting Gene to Microbiome Function
* CSE,IIS
* 09/01/2021,08/31/2024
* Gail Rosen, Drexel University
* Standard Grant
* Sylvia Spengler
* 08/31/2024
* USD 492,871.00

Microbial communities play vital roles in health and the environment. In human
health, they are referred to as our microbiomes; for example, healthy gut
microbiomes can help digest and efficiently convert food to nutrients to be
taken up in our gut. However, what constitutes “unhealthy” (dysbiotic)
microbiomes and how they can affect or be affected by the body (or environment)
is unknown. If we can understand microbes’ interactions with each other and the
body, then we can design better treatments, therapies, and medicines (e.g. pre-
and pro-biotics) to manipulate microbiomes. To understand the ``rules'' for
microbial ecosystems, we must first solve the genotype-to-phenotype problem,
i.e. identify microbial genetic changes which correlate to changes in microbiome
functioning/traits. Most researchers have simply focused on predicting
environmental or disease phenotypes by solely using microbiome community
structure (ie: a population census of species in a community) and do not
consider detailed DNA/RNA differences. It is not surprising that most studies
have yielded modest prediction accuracy and little understanding of how
microbiomes function. Attributing which “configurations” of organisms and/or
genes contribute to a particular “microbiome state” can help us predict disease,
understand how the environment may change microbial ecosystems, and be able to
predict future changes of these systems (e.g. perturbations due to a chemical,
temperature, etc.). Methods that can learn pertinent features at multiple scales
(genome-, organism-, and community-level) simultaneously, are needed to
interpret both the “species census” and microbial genetic changes (mutations
that may lead to speciation and/or functional evolution) that influence
community structure. Our educational activities will bring cutting edge research
and topics to undergraduate and graduate education in Bioinformatics-related
courses, which are part of Machine Learning and Bioinformatics Masters programs
and a Bachelor’s bioinformatics minor at Drexel University. In addition, we plan
to organize a Drexel College of Engineering-wide high school extracurricular
program for mentoring of science projects for underserved public schools.
&lt;br/&gt;&lt;br/&gt;A unified algorithm is needed to learn microbiome features
on multiple levels to be able to predict microbiome functioning, thereby
identifying biological processes (a.k.a harnessing data to understand the rules
of life, NSF Big10 goals) that result in important “states” (e.g. disease or
healthy). Doing so will transform our understanding of how large- and small-
scale changes influence microbiome phenotypes. Current approaches are highly
limited. Phenotype prediction based on 16S rRNA surveys is usually conducted
solely on microbial operational taxonomic units (OTUs), which rarely capture the
mutations that signify overall phenotypic changes. Phenotype prediction using
metagenomes may perform better than 16S surveys, but many downstream analyses
(feature selection, statistical tests) are needed to interpret (e.g. infer
subcommunities relevant to phenotype) this classification. Therefore, we propose
to develop a recurrent neural network (RNN) that can learn both community-level
changes in the microbiome and genetic changes that relate to microbiome
phenotypes. While most neural networks can ``learn'' features, it is usually
difficult to get this information back out of the network (i.e.:
interpretation). We will also use the recent advances in attention-based RNNs
that will help us interpret which multi-scale features are most important to
phenotype prediction. We will make our algorithms and software available to the
microbiome community, whose potential applications include improving
agriculture, environmental monitoring, personalized medicine, among
others.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.