* 2109868
* NSF Postdoctoral Fellowship in Biology FY 2021: Mapping the Relationship Between Genetic Pleiotropy, Gene Function, and Adaptation
* BIO,IOS
* 07/01/2021,06/30/2024
* Megan Ruffley, Ruffley, Megan R
* Fellowship Award
* Diane Jofuku Okamuro
* 06/30/2024
* USD 216,000.00

This action funds an NSF Plant Genome Postdoctoral Research Fellowship in
Biology for FY 2021. The fellowship supports a research and training plan in a
host laboratory for the Fellow who also presents a plan to broaden participation
in biology. The title of the research and training plan for this fellowship to
Megan Ruffley is "Mapping the Relationship Between Genetic Pleiotropy, Gene
Function, and Adaptation" The host institution for the fellowship is the
Carnegie Institution for Science and the sponsoring scientists are Drs. Sue Rhee
and Moises Exposito-Alonso.&lt;br/&gt;&lt;br/&gt;The impact of climate change
and human consumption on ecosystems has been, and is predicted to be, the
greatest threat to an inhabitable earth. Society relies on plants not only for
food, material goods, and health products, but also for sequestration of a
majority of carbon in the atmosphere and for the oxygen needed to sustain life.
One major question arising from these concerns is how will plants respond to the
changing climate. Will they migrate to a new suitable habitat, adapt to the
changing climate, or face extinction? Genetic and trait data have increasingly
been used to predict how natural populations of plants may respond to the
changing climate. In this project, modern genomic approaches and the most
extensive genetic and trait dataset to date for plants will be used to measure
the relationship between mutational effect and adaptation in different
environments. Evolutionary theory predicts that mutations affecting multiple
traits will most likely slow down adaptation because physiological and genetic
constraints create trade-offs between traits. Identifying when large effect
mutations are constraining adaptation by affecting multiple traits will enable
the success breeding of plant traits from genome editing technology without
unintended consequences that could lead to maladapted organisms in future
climates. Additionally, a better understanding of mutational effects will
enhance the power to predict adaptation, which has broad implications for
agriculture and conservation biology. Training objectives include acquiring new
skills in comparative functional genomics and quantitative genetics at the
Carnegie Institution for Science. Support through this fellowship will provide
new opportunities for the Fellow to educate the public on why plant science is
so important, especially in the face of climate change, and what society can all
do to reduce its carbon footprint. &lt;br/&gt;&lt;br/&gt;Pleiotropy, the
phenomenon that a genetic mutation can affect multiple traits, is often
considered a critical barrier to adaptation. The origin of this idea dates back
to the early 20th century and Fisherâ€™s Geometric Model which is still used by
geneticists to explain the cost of complexity of organisms or aging. However,
recent advances in constructing a genotype-to-phenotype map from experimental
data in Saccharomyces cerevisiae suggest that pleiotropy is limited within
modules of functionally related phenotypes and may not always be detrimental to
adaptation. Little is known about the role of pleiotropy in complex organisms
such as plants in nature. This project will study pleiotropy with the 1001
Genomes (1001genomes.org) and Phenomes (arapheno.1001genomes.org) of Arabidopsis
thaliana to understand its functional causes and ultimate consequences to
adaptation in natural environments. Using an unprecedented publicly available
dataset of 1850 phenotypes spanning over 10 million SNPs for up to 1135
individuals of the plant Arabidopsis thaliana, the Fellow will: 1) devise
metrics to quantify pleiotropic effects and their modularity across the genome
and phenome; 2) study the functional context of pleiotropic mutations using
public genome-wide histone modification and cytosine methylation data along with
curated datasets of gene annotations in cellular processes, molecular function,
and localization; and, 3) test whether pleiotropic variants contribute to or
constrain adaptation in semi-natural outdoor experiments with direct measures of
fitness. All data and computational tools and resources generated during the
course of this study will be freely available to the broader research community
through public repositories. &lt;br/&gt;&lt;br/&gt;Keywords: Arabidopsis
thaliana, machine learning, sequence analysis, pleiotropy, gene function, gene
annotation&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.