* 2114191
* Collaborative Research: Genome editing approaches to unravel microRNA roles in stochastic multistable networks
* BIO,MCB
* 07/15/2021,06/30/2025
* Herbert Levine, Northeastern University
* Standard Grant
* Bianca Garner
* 06/30/2025
* USD 896,200.00

One of the fundamental questions in biology is to understand the roles of the
gene regulatory networks driving cellular decisions; cellular decisions drive
everything from an organism's development to a cell's fate as healthy or
diseased. MicroRNAs (miRNAs) are small RNA molecules that bind to the mRNA of
target genes, acting as regulators of gene expression. Previous studies have
demonstrated the critical roles of miRNAs in a variety of biological processes
such as cell growth and cell differentiation. However, what is still not well
understood concerns possible synergistic effects from multiple miRNA molecules
targeting different binding sites of the same mRNA and concerns how miRNA
interactions operate within a complex gene regulatory network. To address these
issues, an interdisciplinary platform that combines genome editing, live-cell
imaging, and mathematical modeling will be developed in this project. The
broader impacts of the project from the University of Texas at Dallas side will
include support for the International Genetically Engineered Machine (iGEM) team
and developing custom educational modules for local schools (Plano ISD) and
summer camps, organizing public educational events at the interface of the
biological and physical sciences, and the recruitment of underrepresented
minorities. From the Northeastern University side, the group will take advantage
of the investigators' participation in the NSF Center for Theoretical Biological
Physics ongoing diversity efforts to recruit undergraduates from under-
represented to work on this project, and spearhead an effort to create a
modeling and computational track for undergraduate Bioengineering majors.
Finally, both groups will be directly involved in reaching out to local
biomedical groups to create more appreciation for the types of rapid progress
that can be made by combining advanced tools such as CRISPR with state-of-the-
art computational methodology including both mechanistic studies and machine
learning approaches. Lying at the heart of intricate relationships that
determine the epithelial-mesenchymal transition (EMT) and mesenchymal-epithelial
transition (MET) phenotypes is a core regulatory unit that consists of
transcription factors and microRNAs. The project will focus on miRNAs targeting
the master transcription factor (TF) families of EMTs, SNAIL and ZEB during the
cellular decision process of EMT in multiple cell lines. The team will first
perform CRISPR-based screens and custom genome and base editing modifications on
miRNA binding sites that are located at the 3'-UTR of the transcription factor
families SNAIL and ZEB. The effects of binding site modifications in EMT and
isolated respective clones will be evaluated. Second, the team will prepare and
optimize an RNA imaging platform in live cells and measure time-series data and
population distributions for miRNA, mRNA and protein levels of corresponding
genes. Using this data, the team will develop stochastic kinetic models of miRNA
regulation and infer the combinatorial effects of multiple miRNA species binding
to multiple sites of the same mRNA. Third, the team will integrate the kinetic
models for each miRNA interaction into full transcription factor-miRNA network
models for different cell lines. The models will be refined by calibrating model
predictions with experimental observations on the distributions of gene
expression and the distribution of cells in various EMT states. This project
brings together investigators who have extensive experience in genome
editing/systems biology (Bleris), epithelialâ€“mesenchymal networks (Levine), and
systems biology/mathematics (Lu). This award reflects NSFs statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.