* 2120918
* Increasing Undergraduate Engagement in Bioinformatics
* EDU,DUE
* 10/15/2021,09/30/2024
* Neil Lamb, HudsonAlpha Institute for Biotechnology
* Standard Grant
* Thomas Kim
* 09/30/2024
* USD 299,927.00

This project aims to serve the national interest by building a regional network
of faculty from smaller colleges and universities that offer entry-level
bioinformatics and protein modeling experiences to their students. Historically,
bioinformatics programs and experiences have been at large, research-intensive
universities and inaccessible to students from smaller and less-resourced
institutions. However, technological advances in molecular biology - including
genomics, transcriptomics, and proteomics - have led to an explosion in publicly
accessible biological databases containing oceans of information. With the
emergence of user-friendly tools to organize and analyze this information,
students from all types of post-secondary institutions can participate in
authentic bioinformatics research. The Characterizing Our DNA Exceptions (CODE)
program will incorporate skill development, active learning, and research-driven
coursework, bringing students directly into the world of research. Students
participating in CODE will use computational modeling and database analysis to
explore the significance of DNA variants identified in clinical and agricultural
studies. They will become producers of knowledge, gaining insight into the
biological significance of these DNA variants. Along the way, they will gain
confidence working with informatics tools, and an awareness of related career
pathways - key components for increasing their scientific identity and
persisting in a STEM major. &lt;br/&gt;&lt;br/&gt;This proposal will build a
regional network of 50-60 college faculty facilitators from up to 40 post-
secondary institutions across the Southeastern US. The project's primary goal is
to increase access to research-based learning experiences, particularly for
students at small institutions with fewer undergraduate research opportunities.
Working individually or in small collaborative teams, students will select from
a list of clinical or agricultural DNA variants identified as part of a
HudsonAlpha research project or from a publicly available genome database. They
will use student friendly analysis software to analyze the sequence in the
context of its location within the genome and compare it with similar segments
from other organisms, followed by molecular modeling to study how the DNA
variant may alter the protein structure. Student work will be documented in a
written report, uploaded to an open-access repository library, and presented at
an annual CODE symposium. Facilitators will be trained to mentor undergraduate
students using this variant analysis and molecular modeling workflow, which can
easily be integrated into existing and new courses. This approach will introduce
a broad range of students to bioinformatics in a mentored project-based format
to capture interest, build confidence, and encourage continued participation in
science education and a STEM career path. Quantitative and qualitative
evaluations will be conducted to determine whether participating in a CODE
project increases awareness and interest in bioinformatics, science self-
efficacy, scientific identity, and intention to enter or remain in a STEM major.
Evaluation data will measure shifts in student awareness and interest in
bioinformatics, self-efficacy, and scientific identity - all factors that
positively impact STEM persistence. This projectâ€™s success will provide a
sustainable model for low-cost, authentic undergraduate bioinformatics research
projects for institutions of all sizes. The NSF IUSE: EHR Program supports
research and development projects to improve the effectiveness of STEM education
for all students. Through the Engaged Student Learning track, the program
supports the creation, exploration, and implementation of promising practices
and tools.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.