* 2122902
* MCA - The role of mRNA structure in allosteric control of ribosomal frameshifting
* BIO,MCB
* 09/01/2021,08/31/2024
* Peter Cornish, University of Missouri-Columbia
* Standard Grant
* Ishita Mukerji
* 08/31/2024
* USD 316,680.00

Every protein in the cell is built from a sequence of amino acid building blocks
by a large macromolecular machine: the ribosome. The specific arrangement of the
amino acids in each protein is set as the ribosome reads the instructions, a
message transcribed from DNA. Occasionally, the ribosome makes a mistake and
slips out of register, changing the sequence of the product it manufactures.
This slippage can result by chance, or it can be specifically engineered by the
host. The latter results from a specific region within the message that causes
slippage at the same place each time with a defined efficiency. The PI will
investigate this mechanism of slippage with emphasis on how the ribosome steps
through the manufacturing process, one amino acid at a time, and is made prone
to slip. Further, the PI will develop methods to design novel sequences to
either change the slippage frequency or to turn the process on or off. These
fundamental details will help us understand how viruses exploit slippage during
their reproduction. A molecular view of viral reproduction is key to the
development of strategies to prevent viral transmission. As part of this
project, the PI will develop a plan to enhance awareness of RNA biology through
outreach efforts as well as provide research opportunities. The PI will
strengthen outreach programs aimed at high-school aged youth in hopes of
maintaining interest once these students reach a university.
&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Protein synthesis is accomplished through a
complex and highly coordinated process that requires a substantial amount of
cellular energy. The central component in protein synthesis is the ribosome,
which is a two-subunit macromolecular complex composed of both RNA and protein
components. The ribosome binds to an mRNA molecule and faithfully translates the
genetic code into an amino acid sequence. Large-scale conformational motions
between and within its subunits facilitate this function. On occasion,
translation fidelity can become compromised due to a failure of the ribosome.
For example, the ribosome, when encountering specific structured sequences
within the mRNA, can shift reading frames in the -1 direction instead of its
normal forward progression of 3 nucleotides per amino acid incorporated. This
suggests the hypothesis that aberrant changes in the conformation of the
ribosome contribute to frameshifting. To investigate this process in detail, the
following research will be performed: (i) determine the correlation between
ribosomal conformational motions and -1 frameshifting using single molecule
FÃ¶rster resonance energy transfer (smFRET); (ii) determine the mechanism of RNA
unwinding by the ribosome via smFRET and (iii) develop tailored and aptamer-
controlled mRNA frameshifting elements by directed evolutionary approaches.
Outcomes from these studies will be evaluated to provide a comprehensive
understanding of -1 ribosomal frameshifting in both bacteria and
eukaryotes.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.