* 2124230
* Collaborative Research: SCH: Integrated Analysis of Single-Cell and Spatially Resolved Omics Data
* CSE,IIS
* 09/01/2021,08/31/2025
* Donald Geman, Johns Hopkins University
* Standard Grant
* Balakrishnan Prabhakaran
* 08/31/2025
* USD 749,997.00

What genes are expressed in a cell shapes a cell's function and behavior. The
set of genes that are expressed in a cell at a given time is highly regulated
and can be impacted by many factors, including the cell's spatial context.
Consequently, a fundamental question for understanding the function and behavior
of cells in both health and disease is how such gene expression levels may be
different in healthy versus diseased cells. Recent technological advances,
including single-cell RNA-seq and spatially resolved omics, now allow us to
measure what genes are expressed in what cells simultaneously for hundreds to
thousands of genes and thousands to millions of cells while preserving the
spatial arrangement of cells within the original tissue. Such technological
advances and the resulting data generated present the opportunity to develop a
deeper understanding of how different patterns of gene expression define
different cell-types or cell-states, how these different cell-types and cell-
states are organized within tissues, and how gene expression and spatial
organizational patterns may be different in health versus diseased tissues.
However, as these datasets are huge and complex, new analytical tools are
required to extract relevant biological insights in an interpretable manner.
This project aims to develop and test these tools, with a particular emphasis on
building simple interpretations to facilitate potential clinical translations to
cost-effective procedures. The PIs plan to integrate the research with
education, train students, especially young women, by involving them in the
research, and share the developed curricula materials and computer code with the
public.&lt;br/&gt;&lt;br/&gt;Cellular identity and heterogeneity are shaped by a
multitude of intrinsic molecular and spatial-contextual factors. Profiling cells
and their molecular states within their native spatial context provides a
modality to connect cellular organization and function. Recent technological
advances are enabling such genome-wide molecular profiling of the transcriptome
and proteome in a spatially resolved manner at single-cell and near-single-cell
resolution. However, new statistical methods and computational tools are still
needed to model and analyze these high-dimensional spatially resolved omics
measurements in order to extract relevant biological insights in an
interpretable manner. This research aims to deduce from the abundant information
that these data modalities provide, a compressed, but interpretable, summary,
based on minimal sets of genes that capture the underlying cellular
transcriptional heterogeneity. The scientific approach is based on statistical
spatial modeling and information theory, designing algorithms for approximating
the entropy of groups of random variables, and using integer programming methods
to search for small subsets of variables that represent best the whole
collection initially observed. The research development and validation will be
based on multiple datasets, some of them being publicly available and some
shared by collaborators at Weill-Cornell Medicine. These datasets will involve
three of the main spatially resolved transcriptomic modalities recently
introduced (VISIUM, CODEX, MERFISH), and will contain traditional imaging
modalities using immunohistochemistry as well. Understanding how some of these
modalities can be used to enhance partial observation in other modalities is
also part of the research program.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.