* 2131923
* Massively Parallel Experiments to Develop a Predictive Biophysical Model of Transcription Rate across Cellular Conditions
* BIO,MCB
* 09/01/2021,08/31/2024
* Howard Salis, Pennsylvania State Univ University Park
* Standard Grant
* David Rockcliffe
* 08/31/2024
* USD 608,569.00

This project seeks to develop new models that quantitatively predict how
bacterial gene expression is regulated across cellular and environmental
conditions. The developed models enable researchers to rationally engineer
microbial organisms with new sensing and metabolic capabilities that are
suitable targeted environments. In bacteria, the first step in gene expression,
transcription, is catalyzed by an RNA polymerase enzyme and a sigma factor
protein. Bacteria contain multiple sigma factors with distinct DNA binding
properties and use signaling pathways to modify the abundances of these sigma
factors in response to cell state, causing large state-dependent changes in
transcription rate across hundreds of genes. In this project, thousands of
experiments are conducted to systematically measure how DNA nucleotide sequences
control sigma factor-specific transcriptional initiation frequencies and the
sites at which transcription is initiated. These measurements are used to create
biophysical models of transcription initiation that accept arbitrary DNA
sequence inputs, calculate the strengths of the relevant interactions, and then
predict the sigma-specific frequencies of transcription initiation at each
potential start site. These predictions enable the automated design of
engineered genetic systems (sensors, genetic circuits, and metabolic pathways)
with targeted transcriptional profiles that adapt and respond to changing
cellular and environmental conditions. The biophysical models of transcription
will also be combined with a web-based interface, visual animations, and
tutorials to facilitate interactive and experiential student learning. This
project also provides opportunities for underrepresented undergraduate students
to participate in laboratory research.&lt;br/&gt;&lt;br/&gt;This project
integrates systematic design, massively parallel experiments, next-generation
sequencing, and machine learning to develop predictive statistical thermodynamic
models of bacterial transcription initiation (a Promoter Calculator).
Transcription initiation rate measurements and transcriptional start site
mapping is conducted on thousands of systematically designed promoter sequences
in well-defined in vitro assays that each contain RNA polymerase and a single
sigma factor. From multiple sets of measurements and using different sigma
factors, sigma-specific models are trained and validated to accurately predict
sigma-specific and site-specific transcription initiation rates. Model
predictions are combined and validated in several ways, including by comparison
to in vivo transcriptomic measurements across a range of environmental
conditions and by engineering genetic systems with rationally designed state-
dependent transcriptional profiles. New types of in vitro assays using naked
genome templates are also devised to enhance precision in measurement and to
ultimately understand transcriptional interactions at natural promoters. The
developed model of transcriptional initiation will enable Synthetic Biologists
to predict and control transcription rates for diverse biotech applications
(e.g. engineering biosensors, genetic circuits, metabolic pathways, and
genomes), while facilitating system-wide, quantitative analysis and debugging of
genetic system function.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.