* 2133740
* URofL:EN: Does re-wilding lead to re-wiring of gene expression and species interaction networks?
* BIO,EF
* 01/01/2022,12/31/2026
* Daniel Bolnick, University of Connecticut
* Continuing Grant
* Hector Munoz-Avila
* 12/31/2026
* USD 2,999,999.00

Evolution is at the root of many socially important problems and solutions.
Infectious diseases evolve resistance to drugs or escape vaccines. Tumors evolve
to exploit their host’s body and tolerate chemotherapy. Agricultural pests
evolve resistance to pesticides. Threatened species must adapt to environmental
variability and land use change or else risk extinction. To solve such problems
rooted in evolution, biologists need to be able to forecast future evolutionary
change. Although biologists have a deep understanding of the forces that cause
adaptive evolution, evolutionary forecasting remains a major challenge. In the
laboratory, if we subject initially similar populations to the same
environmental stress, they often evolve the same solutions using the same genes,
demonstrating that forecasts are possible. However, sometimes experimental
evolution leads to entirely different outcomes. Why is evolution predictable in
some situations, but not others? One hypothesis proposes that evolution is more
predictable for traits built by simple genetic networks (few interacting genes)
than for complex genetic networks (many genes working synergistically). This
project seeks to test this hypothesis. As part of an ecological restoration, the
research team reintroduced native stickleback fish into 8 recently-fishless
lakes in Alaska, beginning the largest evolution experiment yet attempted in a
natural setting. Tracking evolution in these lakes in coming generations will
let the investigators test whether genetic networks evolve, and whether simpler
genetic networks evolve more predictably. The results of this experiment will
yield new tools for forecasting evolutionary change using gene network data,
which can then be applied to evolutionary problems of public interest. To
achieve this aim the team must also develop new tools in computer science and
statistics to analyze how networks change through time. These new computational
tools will have broad applicability to measure changes in any type of network of
concern to impacts national health and security.&lt;br/&gt;&lt;br/&gt;In 2019,
researchers reintroduced native stickleback fish into 8 lakes where they had
been extirpated by invasive species. The team will track the evolution of these
experimentally-created populations as they adapt to their new habitats, annually
monitoring diet, morphology, parasites and microbiota, immunity, genotypes, and
transcriptomes in every source and experimental population in each year. To
analyze these data they will develop new computational and mathematical models
in network theory to measure temporally changing network structures. Using these
new tools they will test for plastic and evolutionary changes in genetic,
transcriptomic, and ecological networks in the focal populations. Comparing
network changes between experimental populations they can test the
predictability of network structure evolution, and its relation to changing
ecological networks. This convergent research deeply integrates biology
(genetics, evolution, ecology, microbiology) with computer science and
statistics to promote new advances at the interface of life and data science. To
educate the public, the team will work with the Big Biology Podcast to produce
six shows about the intersection of network data science, evolution, genetics,
and conservation. Each podcast will be supplemented with ‘virtual field trip”
videos, interviews, and lesson plans for K-12 biology, math, and computer
science classes.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.