* 2135169
* Improving FAPROTAX, a popular tool for predicting metabolic phenotypes in microbiome surveys
* BIO,DBI
* 02/01/2022,01/31/2025
* Stilianos Louca, University of Oregon Eugene
* Standard Grant
* Reed Beaman
* 01/31/2025
* USD 387,240.00

FAPROTAX (Functional Annotation of Prokaryotic Taxa) is a popular database and
computational tool for predicting the metabolic functions encoded by bacterial
and archaeal (`prokaryotic') taxa in microbiome surveys. FAPROTAX has been used
in &gt;700 studies (as of November 2021) in a broad spectrum of environments.
With this award, FAPROTAX will be revised to resolve many of its current
limitations and substantially expand its applicability scope. FAPROTAX2 will be
the only tool for conducting broad phenotypic predictions in microbiome surveys
based on both experimental and genomic evidence, thus providing a valuable and
unique resource for microbiome research. FAPROTAX2 will allow rapid insight into
the metabolic functional potential of microbial communities without the need for
expensive metagenomic sequencing or culturing. Hence, FAPROTAX2 will help tackle
a multitude of environmental, industrial, agricultural and human health issues
related to the microbiota, such as the bioremediation of soil pollutants,
habitat restoration and conservation, wastewater treatment, landfill management,
biofuel production, agricultural soil enhancement, and treating human diseases.
As a side-product, the proposed project will also deliver an unprecedentedly
comprehensive database of metabolic phenotypes across thousands of prokaryotic
species, thus facilitating investigations of microbial evolution and providing
valuable training/testing data for algorithms that predict prokaryotic
phenotypes from genomes. This project will also provide student training in
bioinformatics, programming, database design and management, and microbiology.
&lt;br/&gt;&lt;br/&gt;FAPROTAX predicts the metabolic functions encoded by
bacterial and archaeal (“prokaryotic”) 16S rRNA phylotypes in marker-gene
microbiome surveys, based on published experimental evidence (e.g., from
cultures) across thousands of taxa, covering &gt;80 metabolic functions of
environmental, industrial and medical interest. The specific improvements to be
completed under this funding include incorporation of additional experimental
evidence from more literature sources, redefinition of metabolic functions to be
consistent with MetaCyc (a major metabolic pathway database), inclusion of
phenotypic information from thousands of available prokaryotic genomes, added
support for calculating prediction uncertainties, and increased flexibility when
balancing sensitivity versus specificity, depending on the user's objectives.
The proposed revision, FAPROTAX2, will be the only tool for conducting broad
phenotypic predictions in marker-gene microbiome surveys based on both
experimental and genomic evidence. These improvements will improve the ability
to study the metabolic functions of microbial communities and their role in
ecosystem processes. The final results of the project will be accessible from
www.loucalab.com/archive/FAPROTAX.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.