* 2142336
* CAREER: Leveraging Microfluidics for High-Throughput in Vitro Investigations of Transcriptional Regulation
* BIO,MCB
* 01/15/2022,12/31/2026
* Polly Fordyce, Stanford University
* Continuing Grant
* Ishita Mukerji
* 12/31/2026
* USD 323,417.00

All biological processes depend on the precise regulation of when genes are
transcribed and how many RNA transcripts are produced. This regulation is driven
primarily by transcription factor (TF) proteins that bind specific DNA sequences
in the genome and then recruit additional ‘effector’ proteins to either activate
or repress gene transcription. While scientists have ‘cracked’ the DNA code
specifying the sequences of RNA and protein molecules produced from protein-
coding DNA, it remains unclear how the TF/DNA ‘regulatory’ code that governs the
strength and timing of gene expression. It is not known how closely related TFs
with apparently similar DNA binding preferences recognize different sites in the
genome to regulate distinct transcriptional programs. There is little known
about the web of interactions between TFs and the ‘effector’ proteins required
to activate transcription. This project will utilize new microfluidic
technologies that enable accurate measurement of 1000s of protein/DNA and
protein/protein binding interactions simultaneously and at low cost. This
project will apply these technologies to better understand how TFs find and bind
their DNA targets, how bound TFs recruit ‘effector’ proteins, and the degree to
which the ‘regulatory code’ relies on thermodynamics. The PI will expand a
hands-on microfluidics device laboratory to provide inquiry-based summer
research experiences to community college students, increasing training
opportunities for student populations traditionally underrepresented in STEM.
&lt;br/&gt;&lt;br/&gt;Regulated gene expression is central to biology, sculpting
the transformation from embryo to animal and enabling cells to respond
dynamically to environmental changes. At a molecular level, this regulation is
accomplished primarily by transcription factor (TF) proteins that bind DNA
regulatory elements and then recruit additional protein cofactors to either
activate or repress transcription. Biological diversity is simply too vast for
us to ever measure TF binding and transcription in all organisms and all tissues
under all conditions of interest. This project will provide in vitro
measurements that quantify thermodynamic and kinetic constants of reconstituted
macromolecular interactions at unprecedented scale by using multiple novel
microfluidic platforms capable of measuring affinities and kinetics for up to
one million protein/DNA or protein/protein interactions in parallel that were
developed in the PI’s laboratory. In this project, these in vitro technologies
will be used to develop quantitative and predictive models of how TFs find and
bind their genomic targets and how bound TFs recruit cofactors to regulate gene
expression. Using cutting-edge in silico tools, these measurements will then be
integrated with existing in vivo data sets to develop quantitative models that,
in turn, will be directly tested in vivo by quantifying TF and cofactor binding
and gene expression for sequence variants. This project is funded by the
Molecular Biophysics and Genetic Mechanisms Clusters in the Division of
Molecular and Cellular Biosciences.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.