* 2143622
* CAREER: Environmental monitoring of antibiotic resistance using targeted long-read sequencing
* ENG,CBET
* 06/15/2022,05/31/2027
* Amy Pickering, University of California-Berkeley
* Continuing Grant
* Mamadou Diallo
* 05/31/2027
* USD 452,983.00

The World Health Organization has identified antibiotic resistance as one of the
top ten global public health threats facing humanity. Antibiotic resistance (AR)
can spread across humans, animals, and the environment through the transfer of
whole bacteria carrying resistance as well as horizontal gene transfer between
bacteria. Solutions and interventions that can prevent the global spread of AR
are urgently needed. However, a fundamental understanding of the pathways and
mechanisms of transfer of antibiotic-resistant bacteria (ARB) and antibiotic-
resistance genes (ARGs) in community settings has remained elusive. The
overarching goal of this CAREER project is to develop and validate a field
compatible method for large-scale environmental monitoring and identification of
the transmission pathways of ARB and ARGs. To advance this goal, the Principal
Investigator proposes to explore the utilization of targeted long-read DNA
sequencing to develop a low-cost and culture-independent metagenomics platform
for global monitoring of AR at the population level that would not require
costly equipment or access to high performance computing resources. The
successful completion of this project will benefit society by laying the
foundation for more efficient and cost-effective environmental surveillance
platforms to monitor the global spread of antibiotic resistance in low-resource
settings thereby enabling surveillance in communities where it is often most
needed. Further benefits to society will be achieved through student education
and training including the mentoring of a graduate student and an undergraduate
student at the University of California, Berkeley.
&lt;br/&gt;&lt;br/&gt;Advances in long-read metagenomics (e.g., Nanopore
Sequencing) are providing new opportunities to develop more efficient and cost-
effective assays/platforms that could identify antibiotic resistant genes (ARGs)
in their genomic context with the goal of unraveling the mechanisms and pathways
of antibiotic resistance (AR) transfer including horizontal gene transfer
between bacterial hosts. However, a critical limitation of current long-read AR
sequencing is that most of the sequencing is wasted on non-informative reads,
which in turns limits the depth and coverage of low-abundance ARGs in
environmental samples. To address this critical limitation of existing AR
genomic sequencing tools/protocols, the Principal Investigator (PI) of this
CAREER project proposes to develop and validate a targeted sequencing approach
to enable the enrichment of ARGs and their flanking regions with the goal of
facilitating the detection of low-abundance ARG targets and their genomic
context (e.g., bacterial hosts or plasmid carriers) in metagenomic samples. The
successful completion of this project has potential for transformative impact.
By combining CRISPR-based enrichment with portable long-read sequencing (i.e.,
using a Cas9-guide RNA complex to cleave genes of interest followed by the
ligation of long-read sequencing adapters to the cleaved gene sites), the PI
hopes to build the foundation for more efficient and cost-effective metagenomics
surveillance platforms to monitor the global spread of antibiotic resistance. To
implement the educational and training goals of this CAREER project, the PI will
partner with the UC-Berkeley DCAC (Destination College Advising Corps) to
develop and implement a classroom research module for undergraduate and high-
school students that employs portable DNA sequencing technologies to detect ARGs
of clinical importance in soil samples collected by the students in their
communities. In addition, the PI plans to tailor the AR sequencing classroom
research module to develop and disseminate a citizen science platform that will
enable teachers and students worldwide to contribute to a global AR surveillance
effort through the development of an open-source database of soil metagenomes in
collaboration with the PARE (Prevalence of Antibiotic-Resistance in the
Environment) project.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.