* 2145736
* CAREER: Learning Mechanisms from Single Cell Multi-Omics Data
* BIO,DBI
* 09/01/2022,08/31/2027
* Xiuwei Zhang, Georgia Tech Research Corporation
* Continuing Grant
* Jean Gao
* 08/31/2027
* USD 564,510.00

This award is funded in whole or in part under the American Rescue Plan Act of
2021 (Public Law 117-2).

Cells are the fundamental units of life. Understanding how cells differentiate
into various cell types and how cells change during evolutionary process has
been a long-standing scientific problem. Recent sequencing and imaging
technologies can provide large scale data for each single cell in a tissue,
including the amount of mRNA products of a gene, the amount of protein products
of certain genes, the chromatin 3D structure, and the spatial location of each
cell. The project will develop computational methods to learn mechanisms in cell
differentiation and development from complex and high dimensional data. The
project will provide open-source tools for the community and can be integrated
with educational activities and outreach, such as courses on topics of
algorithms in computational biology. Opportunities for high school and
undergraduate students especially those from underrepresented groups will be
provided to participate in cutting-edge research in computational biology.

During recent years, single cell technologies have progressed in terms of both
“modality” and “scale”. In terms of modality, multi-omic technologies allow
researchers to profile each single cell from multiple aspects, including
transcriptome, genome, chromatin accessibility and protein abundance. In terms
of scale, more cells, tissues, individuals, and species are profiled with single
cell RNA-sequencing (scRNA-seq) technology. Computational integration methods
have been developed to integrate single cell data from different modalities or
different batches. The goal of the project is to develop integration tools with
a strong emphasis on mechanisms, and to learn molecular mechanisms from the
large-scale, multi-modality single cell data. The specific aims are: (1) develop
methods to integrate paired and unpaired single cell multi-omics data from the
same tissue, and in particular, propose a method to learn consensus cell
identity considering cross-modality relationships; (2) develop deep learning
methods to integrate scRNA-seq data from multiple individuals or species. The
method aims to remove technical batch effects but preserve biological variation
between data matrices and infer the genes which are associated with each meta
feature like age and race; (3) develop methods to learn cell-specific gene
regulatory networks (GRNs) for cells in a temporal or spatial context. When
spatial locations of cells are available, the project will implement a method to
learn both cell-specific GRNs and cell-cell interactions. This research has the
potential to make a significant step forward in understanding mechanisms in
cells and diseases using large-scale, multi-modality single cell data. The
results of the project can be found at the PI’s website:
https://xiuweizhang.wordpress.com/.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.