* 2151777
* Collaborative Research: Unraveling Structural and Mechanistic Aspects of RNA Viral Frameshifting Elements by Graph Theory and Molecular Modeling
* MPS,DMS
* 05/01/2022,04/30/2026
* Tamar Schlick, New York University
* Continuing Grant
* Zhilan Feng
* 04/30/2026
* USD 254,000.00

Programmed ribosomal frameshifting is indispensable to many viruses, including
HIV and SARS-associated coronaviruses, to translate overlapping reading frames
on the mRNA so that essential viral proteins can be produced. Because modulation
of frameshifting has been shown to dramatically influence viral viability, the
RNA frameshifting element (FSE) has been an attractive anti-viral drug target.
However, the complex aspects of frameshifting must be understood before
therapeutic strategies can succeed. Following a 2020 NSF RAPID award, the
Schlick mathematics/computational biology lab, in collaboration with the
Laederach experimental RNA group, will combine graph theory applications to RNA
(RAG: RNA-As-Graphs) with biophysical studies and biomolecular
modeling/simulation to unravel structures and mechanisms of the RNA FSE of SARS-
CoV-2 and related viruses. The collaborative research program will be the basis
for interdisciplinary training of students and postdoctoral fellows, including
women and minorities, in mathematics, computer science, biology, physics,
chemistry, and engineering, through computer program development, data analysis,
and biological interpretations. Students and postdocs will learn to analyze,
process, and visualize biological data; devise and validate models; develop
simulation algorithms and coarse-grained models; and collect and interpret
structural/functional patterns to yield new mathematical and biophysical
relationships.&lt;br/&gt;&lt;br/&gt;The project will describe conformations and
structural transitions of the FSE of SARS-CoV-2 from phylogenetic and
biophysical viewpoints by exploiting global representation of mathematical RNA
graphs. Specifically, the researchers will gain insight into the evolutionary
path of the FSE of coronaviruses by computing and validating experimentally RNA
secondary-structure conformational landscapes of the FSE of SARS-CoV-2
relatives; probe frameshifting mechanisms by determining the SARS-CoV-2 FSE's
transition pathway; and identify and test experimentally structure-altering
mutations to transform the FSE into complex intertwined motifs by RAG inverse
folding and genetic algorithms to hamper frameshifting. This unique approach
applied to frameshifting elements in coronaviruses including SARS-CoV-2 using
novel mathematical graph-theory tools and biophysical models will yield crucial
insights into the structure, mechanisms, and evolutionary trends in related
viruses to explain the relationship between viral structure and frameshifting
efficiency/viral viability. By looking at structure from a global graph theory
point of view, patterns can be discerned and related more easily than sequence
or atomic-based models. The determined structures, mechanisms, and structure-
altering mutations define gene therapy and anti-viral targets for therapeutic
interventions.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.