* 2151106
* Collaborative Research: EAGER: Development of an Artificial Chromosome System in Chlamydomonas Based on CENH3 Tethering
* BIO,IOS
* 05/01/2022,04/30/2024
* R. Kelly Dawe, University of Georgia Research Foundation Inc
* Standard Grant
* Gerald Schoenknecht
* 04/30/2024
* USD 93,680.00

Some of the most pressing problems in agriculture and biotechnology can only be
tackled by large scale changes in crop or host genomes where multiple genes must
be introduced and inherited together for successful trait improvement. For
example, desirable crop traits such as disease resistance, improved yields and
resistance to climate change may require multiple genes introduced into one
strain or cultivar, a task that is not practical with current transgenic
methods. Artificial chromosomes have the potential for solving this problem, but
the technology for creating and deploying plant artificial chromosomes remains
inadequate. Under this proposal a green alga and established research organism
that is related to land plants, Chlamydomonas reinhardtii, will be used to
accelerate the design and implementation of artificial chromosomes. Because of
its easy cultivation and rapid generation time (4 hours versus 3+ months for
most plants) Chlamydomonas artificial chromosomes can be developed much faster
than in land plants. The lessons learned from successful implementation of
artificial chromosomes in Chlamydomonas can be used to fast-track the creation
of artificial chromosomes in land plants and will have important immediate
benefits by enabling the modification and improvement of algae as sources of
biomass, biofuel, and high value pharmaceuticals and nutraceuticals which cannot
be easily or cheaply produced using existing
biotechnology.&lt;br/&gt;&lt;br/&gt;This proposal aims to address the most
significant challenge in any effort to design a functional synthetic chromosome
in eukaryotes, the centromere. Centromeres in all eukaryotes are defined by the
presence of the histone H3 variant CENP-A/CENH3. The Dawe laboratory has
demonstrated the feasibility of activating new plant centromeres using a simple
tethering approach based on the DNA binding domain of the LexA repressor to its
operator, LexO, which serves as a synthetic centromere organizing center. In
this method a host strain expressing a native CENH3 sequence fused to LexA is
generated. Next, an array of LexO binding sites is incorporated into a small
synthetic chromosome containing telomeric sequences and selectable markers on
both arms. Preliminary data from the Dawe laboratory shows that in plants the
LexA-CENH3 protein binds to the LexO array and recruits additional CENH3 to
create functional centromeres. Under this proposal a similar method will be
adopted for the alga Chlamydomonas as a potentially game-changing advance in
synthetic biology. This proposal combines the established expertise of the Dawe
laboratory in plant artificial chromosomes and the Umen laboratory in algal
genetics and molecular biology to test and establish an artificial chromosome
system in Chlamydomonas. Under this proposal 1. Chlamydomonas centromeres will
be fully sequenced and validated using long-read sequencing methods, and
formally validated/defined using chromatin immunoprecipitation with a centromere
marker protein, CenH3. 2. Transgenic Chlamydomonas strains expressing its native
CenH3 paralogs as fusion proteins to LexA will be created and validated. 3.
Chlamydomonas artificial chromosomes of different sizes and configurations
(e.g., linear, circular) and containing LexO arrays for centromere nucleation
will be built and tested for centromere assembly, mitotic/meiotic segregation,
stability, and gene expression.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.