* 2153787
* Structural characterization of the mechanism leading to recognition of Alu elements by the Z-RNA-binding domain of ADAR1
* BIO,MCB
* 07/15/2022,06/30/2026
* Beat Vogeli, University of Colorado at Denver
* Standard Grant
* Ishita Mukerji
* 06/30/2026
* USD 1,000,000.00

DNA represents the repository of our genetic makeup, but its many copies into
RNA constitute the actual blueprint for life. These RNA copies start off as
exact DNA transcripts, but they later get altered in precise ways that end up
determining the fate of the cell. The swap of a very common RNA building block
(adenosine or A) to a less common one (inosine or I). Such editing of cellular
RNAs helps a cell distinguish between self and non-self RNAs. If that mechanism
is corrupted, the immune system can be compromised, or auto-immune diseases may
arise and misediting is also observed in cancerous cells. The biological pathway
leading to such modifications is complex because not every A needs to or should
be turned into an I. A single protein enzyme called ADAR1 is responsible for
selecting the modification site, as well as for catalyzing the chemical reaction
that leads to ‘editing of A to I’. The catalysis part is understood, but how
ADAR1 recognizes a particular site in an RNA molecule remains a mystery. This
project is grounded in the hypothesis that a region in ADAR1 is key for
recognizing where exactly editing should occur on RNA. That region is known to
selectively bind to a rare left-handed form of RNA. This proposal will use
advanced NMR methods to monitor the transition from A-form to Z-form RNA,
combined with techniques such as isothermal calorimetry and circular dichroism
to localize where on natural RNA this part of long ADAR1 binds. The project will
then delineate the signature for recognition by ADAR1, through pinpointing the
steps leading to a fully assembled complex. This project will in corporate
research into course modules and train graduate, undergraduate, and high school
students from underrepresented communities.&lt;br/&gt;&lt;br/&gt;RNA editing of
cellular RNAs helps a cell distinguish between self and non-self RNAs. Editing
of adenosines into inosines (A-to-I) is generally catalyzed by the ‘adenosine
deaminase acting on RNA-1’ protein (ADAR1) at primate-specific Alu
retrotransposons. A-to-I editing is augmented upon infection, primarily through
the interferon-induced longer isoform of ADAR1 that comprises a Z-DNA/Z-RNA
binding domain named ‘Zα’ at its N-terminus. Z-RNA in the form of repeats of
cytosine and guanosine (CpG) in a left-handed double-helical conformation has
been found in cells, but the prevalence of such structures and their exact role
are unknown. In addition, many —if not most— regions proposed to adopt a Z
conformation do not resemble regular (CpG)n. How these local Z-RNA conformations
are generated within A-form helices, stabilized, and regulated by Zα of ADAR1
(acting in synergy with the downstream Zβ domain), as well as their exact role
in the function of these RNAs, remain unknown. This project will test the
hypothesis that the binding of Zα to Alu elements plays an essential role during
the editing process with the overall goal of achieving structural and dynamic
characterization of the formation of A-Z RNA junctions and their recognition by
Zα(-Zβ) across transcriptomes. This project is supported by the Molecular
Biophysics Cluster of the Division of Molecular and Cellular
Biosciences.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.