* 2153205
* CRII:III:Development of deep learning methods for high-resolution 3D genome structure spatial reconstruction
* CSE,IIS
* 05/01/2022,04/30/2024
* Oluwatosin Oluwadare, University of Colorado at Colorado Springs
* Standard Grant
* Sylvia Spengler
* 04/30/2024
* USD 175,000.00

An increase in the amount and resolution of data from different individuals and
cell types, from multiple cells, single cells, or subcellular localizations, has
consequently increased the complexity of these datasets. This increase means we
need more sophisticated and automated approaches to infer patterns and
structures from these datasets. The cell's three-dimensional (3D) chromosome and
genome organization structure is one of the vital structures deducible in our
data-rich society. Reconstructing the 3D organization of the genome is a
complicated and challenging task. Nevertheless, it is necessary to understand
many cellular activities, such as gene expression, gene stability, and
regulation. To improve the understanding of chromosome organization within a
cell, genomic technologies based on chromosome conformation capture techniques,
particularly Hi-C, were developed. This development greatly improved cellular
study and led to the development of multiple 3D chromosome structure
reconstruction methods over the years. Although many 3D chromosome structure
reconstruction methods have been proposed, detailed insight into the structural
architecture of the chromosome and genome at a high resolution (&lt;=5kb) is
lacking. This project aims to develop an advanced and scalable algorithm for
high-resolution 3D genome reconstruction from Hi-C data that provides enough
detail to explain biological activities such as gene-gene interaction at a
refined scale. This project will be developed as open-source tools and software.
Also, the multidisciplinary nature of this project provides a mechanism for
training and providing hands-on learning and mentoring opportunities in teaching
and research to students at both undergraduate and graduate
levels.&lt;br/&gt;&lt;br/&gt;Specifically, the ultimate objective of this
project is to develop computational and machine learning-based frameworks to
elucidate the interplay between the hierarchical organization within the genome
and its functions through high-resolution(&lt;=5kb) 3D structure reconstruction.
This project is expected to develop algorithms and computational tools to
advance 3D genome organization research in the following aspects. First, a
robust, reliable, and flexible high-resolution 3D chromosome and genome
structure reconstruction algorithm will be developed. This algorithm will use
graph convolutional neural networks (GCNNs) as the core method for spatial
structure reconstruction. Second, it will develop a novel noninstance-based
approach for 3D structure reconstruction capable of high-resolution 3D genome
structure model prediction using a generalization approach, where a trained
model can be used to reconstruct multiple chromosomes or used for prediction
across resolutions. The proposed noninstance-based approach for 3D structure
utilizes a node embedding algorithm for the graph node feature representation
corresponding to each chromosomal locus. These features are trained with a GCNN
to generate predictions for chromosome and genome 3D structure coordinates
corresponding to each chromosomal locus. Overall, this project will investigate
the GCNN complexity and model depth questions to provide a new approach for
chromosome and genome 3D structure reconstruction at a high resolution. The
successful implementation of this project will give a unique insight into the 3D
organization of chromosomes and genomes and motivate the future development of
noninstance-based methods for chromosome 3D structure
prediction.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.