* 1707751
* Theoretical Modeling of Protein-Driven Chromosomal Dynamics and Biological Function
* MPS,PHY
* 08/01/2017,07/31/2021
* Andrew Spakowitz, Stanford University
* Continuing Grant
* Krastan Blagoev
* 07/31/2021
* USD 455,497.00

This research project aims to enhance the use of theoretical and simulation
approaches to address critical biological problems that have both fundamental
and practical impact. The models used in this project draw from the fields of
polymer and soft-materials physics and infuse novel physical concepts and
understanding into problems involving living cells and organisms. The engagement
of both in vitro and in vivo experimental measurements will provide guidance on
the implementation of quantitative theory into practical biological problems.
Results from this program will be widely disseminated through publications,
conferences, and an online repository of simulation tools as well as
descriptions, press releases, and educational tools for use by the broad
scientific community and the general public. The lab of the PI is an ideal
environment for training quantitative biologists who are capable of tackling a
diverse range of fundamental and applied problems in living systems, and the
lab's commitment to diversity and educational development has led to alumni of
the group who are making major contributions to both academic and industrial
research. This program develops and implements laboratory science and
engineering teaching modules for the education of high school students who are
being treated for childhood cancer and other illnesses. This program is piloted
at the Hospital School at the Lucile Packard Children's Hospital, and the PI
engages undergraduate and graduate students at Stanford to develop the program,
resulting in an exciting opportunity for students to enrich their educational
experience with broader outreach. Ongoing efforts aim to broadly disseminate
this program nationally as a general laboratory science curriculum for hospital-
school education.&lt;br/&gt;&lt;br/&gt;Prokaryotic and eukaryotic cells have the
daunting task of compactly packaging their immense genome while simultaneously
maintaining gene regulatory control and the ability for chromosomal DNA to be
replicated and segregated in a robust manner. Chromosomal DNA within living
cells is manipulated and accessed by a host of proteins that perform numerous
functions that are orchestrated throughout the cell&lt;br/&gt;cycle. The
physical manipulation of chromosomal DNA is central to its function, and
achieving a fundamental quantitative framework for predicting and interpreting
chromosomal organization and dynamics would be instrumental to our understanding
of living systems. This research program aims to tackle three major biological
processes involving chromosomal DNA in prokaryotic cells: regulation,
segregation, and organization. These efforts are delineated into three
complementary thrusts that establish fundamental physical insight into the
orchestrated function of proteins that influence chromosomal organization and
dynamics. In Thrust 1, the PI addresses the impact of DNA supercoiling on gene
regulation. This effort focuses on two prototypical regulatory proteins (Lac
repressor and Ã« repressor) that serve as examples of regulatory proteins that
engage DNA at multiple sites to form a regulatory complex. Cellular control of
supercoiling elicits major changes in gene regulation, and this thrust will
directly address how topoisomerases and regulatory proteins orchestrate their
function. Thrust 2 aims to address segregation of DNA through the ParABS
system's highly conserved prokaryotic segregation apparatus. The mysterious
function of this system is unlike motor or cytoskeletal proteins in that there
is no obvious force generating function. The efforts will aim to resolve how
temporal correlations in chromosomal dynamics result in directed motion in a
burning-bridge mechanism for the ParABS system. In Thrust 3, the PI will tackle
the interplay between RNA polymerase and condensin in dictating chromosomal
organization. Active forces generated by RNA polymerase displace proteins along
DNA, and the resulting impact of these nonequilibrium processes on chromosomal
organization will be addressed in this thrust, providing new physical insight
into the formation and maintenance of organized domains within the
cell.&lt;br/&gt;&lt;br/&gt;This project is being jointly supported by the
Physics of Living Systems program in the Division of Physics and the Genetic
Mechanisms Cluster in the Division of Molecular and Cellular Biosciences.