* 1714778
* RUI: Transcription Factor Discovery in the Extremophile Thermus Thermophilus
* BIO,MCB
* 07/15/2017,06/30/2021
* Michael Van Dyke, Kennesaw State University Research and Service Foundation
* Standard Grant
* Arcady Mushegian
* 06/30/2021
* USD 253,790.00

With this award, the Genetic Mechanisms Cluster of the Division of Molecular and
Cellular Biosciences: Biological Sciences Directorate is funding Dr. Michael Van
Dyke from Kennesaw State University to identify orphan transcription factors and
their potential biological roles. Transcription factors are the primary
regulators of gene expression in most organisms, controlling those genes that
are turned on or off in response to intrinsic factors or external cues.
Presently, the targets of most transcription factors are not known. However,
using a novel selection method that allows the screening of trillions of DNA
sequences, it is now possible to go from putative transcription factor to
understanding its biological regulatory roles in relatively short order. This
project will explore potential transcription factors in Thermus thermophilus, a
model extremophile organism whose proteins are widely used in many aspects of
biotechnology. Knowledge derived from this project will be applicable to a host
of industries, ranging from agribusiness to waste treatment. In addition, this
project will train a mix of undergraduate and graduate (MS) students in the
biochemistry of protein-DNA interactions as well as bioinformatics and regulon
development, thereby providing future scientists in these fields.

This research project will be undertaken to better understand transcriptional
regulatory networks in the model extremophile T. thermophilus HB8. Binding
specificity of putative transcription factors will be determined by the
combinatorial approach Restriction Endonuclease Protection, Selection, and
Amplification, followed by massively parallel semiconductor sequencing and
Multiple Expectation Maximization for Motif Elicitation analysis. Potential
binding sites will then be mapped to the Thermus thermophilus HB8 genome using
Find Individual Motif Occurrences and functionally validated both in vitro
(electrophoretic mobility shift assays) and in vivo (quantitative Polymerase
Chain Reactions). Information from this study will provide a better
understanding of regulons within a model eubacterial extremophile and will
complement ongoing studies, for example, the Structural-Biological Whole Cell
Project, which seeks to understand the molecular constituents of life at an
atomic level. Methods developed by this project should also be generally
applicable to characterizing orphan transcription factors and eliciting regulons
in other organisms.