* 1714417
* AF:  Small:  Fast and accurate computational tools for large-scale evolutionary inference:  a phylogenetic network approach
* CSE,CCF
* 08/15/2017,07/31/2021
* Kevin Liu, Michigan State University
* Standard Grant
* Mitra Basu
* 07/31/2021
* USD 428,748.00

A grand challenge in science is reconstructing the "Tree of Life", which is the
phylogeny, or evolutionary history, of all species on Earth. The notion of a
Tree of Life reflects Darwin's view of evolution as "tree-like", where
bifurcating speciation results in an ancestral species giving rise to two
genetically isolated descendant species. However, recent studies have challenged
this view. "Non-tree-like" evolution due to inter-species gene flow - where DNA
is shared between species existing at the same time - has significantly shaped
the evolution of a far greater diversity of species than was ever thought
possible, including humans and Neandertal, mice, and butterflies. In these
cases, the phylogeny cannot be described by a tree, but is instead a more
general structure known as a phylogenetic network. Our understanding of
evolution and biology is at a crossroads. How frequently is the traditional
assumption of tree-like evolution violated in the Tree of Life? What is the
evolutionary role of gene flow? Applications include understanding the spread of
antibiotic resistance among bacteria, which costs the U.S. over $35 billion and
a loss of 23,000 lives annually, and pesticide resistance in weeds, mice, and
other pests, which costs the U.S. billions of dollars annually. Phylogenetics,
or the discipline which seeks to reconstruct evolutionary histories using
biomolecular sequences and other biological data, can shed new light into these
questions. Two ingredients are necessary for phylogenetic reconstruction and
analysis: suitable biological data for the organisms under study, and
computational methods capable of efficiently and accurately analyzing the data.
Today, biological data abounds due to recent biotechnological advances, and
large-scale datasets are common. However, computational methods have not kept
pace. New computational frameworks are needed for fast and accurate phylogenetic
network inference and analysis in the era of "big data". To address these
challenges, this project will create new computational frameworks for fast and
accurate network-based phylogenetic inference using large-scale genomic sequence
datasets and evolutionary analysis of continuous biological data. The new
methodologies will be validated using a comprehensive performance study. More
broadly, this project will enable student training, scientific outreach, open-
source software development, and scientific research that may yield new
biological and biomedical discoveries.

Phylogenies are typically inferred using computational analysis of biomolecular
sequence data, and phylogenetic comparative methods are used for evolutionary
analysis of continuous biological data (e.g., trait data). Today, "big data"
challenges abound due to rapid advances in sequencing and related
biotechnologies. In particular, large-scale datasets with hundreds of genomes
are now common. The state of the art of phylogenetic inference therefore faces
two critical scalability challenges: (1) the number of organisms in a study, and
(2) greater evolutionary divergence reflecting the complex interplay of tree-
like and non-tree-like evolution. For discrete sequence data, state-of-the-art
methods address the second challenge, but are not scalable beyond inputs with a
few dozen genomes; for continuous data, scalable approaches are needed to
address the second challenge in the context of phylogenetic uncertainty and
adaptive evolution.

The proposed research creates new computational approaches that address both
challenges for discrete sequence data and continuous data. The first objective
is to create a novel computational framework for scalable phylogenetic network
inference using large-scale genomic sequence data. The framework makes use of
the multi-species network coalescent model to account for genetic drift,
incomplete lineage sorting, and gene flow as well as traditional substitution-
based models of sequence evolution. The framework builds on the PI's work on
large-scale phylogenetic tree inference by adapting divide-and-conquer
algorithms to the more general case of networks, resulting in accurate and
efficient inference. The second objective is to develop novel stochastic models
and methods for analyzing continuous character evolution on phylogenetic
networks. The new models will generalize widely-used non-neutral models of
continuous character evolution that assume tree-like evolution, and will be used
to create new methods for phylogenetic inference using heterogeneous large-scale
inputs. The third objective is to validate the new computational methodologies
using new empirical and synthetic benchmarks. The empirical benchmarks include
mouse, plant, and fungal datasets that have been produced through ongoing
collaborations.