* 1714264
* mRNA selection by eIF4E isoforms and their sequestering factors.
* BIO,MCB
* 08/01/2017,07/31/2023
* Myon Hee Lee, East Carolina University
* Continuing Grant
* Stephen DiFazio
* 07/31/2023
* USD 635,248.00

This project explores how "undecided" stem cells progress to a defined cellular
identity by changing the proteins they make within the cell. During the early
development of an organism, cells must decide their path to become an organ that
performs a specialized function. The messenger RNAs (mRNAs) within each cell are
blueprints for the proteins to be made (translation). Such mRNAs are used
selectively to introduce only the appropriate specialized functions. This
project will define the mechanisms that translation factors use to carefully
select mRNAs to decode at appropriate times and places, yielding useful
information about how cells determine their final fate. High school,
undergraduate, Masters, and PhD students will gain hands-on experience and
knowledge of the most contemporary bioinformatics, recombinant, molecular and
biochemical techniques, making them competitive for entering the workforce while
pursuing science careers. &lt;br/&gt;&lt;br/&gt;Protein synthesis is highly
regulated in early animal development, occurring differently in each cell/tissue
type to create functional organs with appropriate architecture and cellular
activities. Transcriptional regulation of genes during development is well
studied by many labs. However, transcriptional patterns often don't match the
spatial and temporal protein requirements. The actual appearance of proteins is
dictated largely by mRNA selection/translational control, whose mechanisms are
more obscure. Over many years the Keiper lab has studied mRNA selectivity by
unique forms of the eIF4 translation factors. The hypothesis is that isoforms of
eIF4E and eIF4G positively and selectively recruit dormant mRNPs to ribosomes
for efficient protein synthesis. This project focusses on two germ line eIF4E's
(IFE-1 and IFE-3) in C. elegans, a simple worm that is an ideal
genetic/transgenic model for reproductive development. The first goal of this
project is to use resolved polysome RNA Seq, a technology developed in the PIs
lab, to identify all RNAs that rely on individual IFEs for efficient
translation. The second uses CRISPR/Cas9 to fluorescently tag each IFE to
determine its localization in vivo, and allow characterization of its storage
and retrieval complexes by MALDI-tof proteomics. The result: a blueprint of
dynamic and "whole genomic" protein synthesis, which promotes fertility,
reproduction, embryo and organ development