* 1714352
* Establishing a generalizable model for predictable antisense RNA repression
* BIO,MCB
* 09/01/2017,08/31/2021
* Tae Seok Moon, Washington University
* Standard Grant
* David Rockcliffe
* 08/31/2021
* USD 425,000.00

RNA is mostly known as a messenger to convey genetic information, while small
RNA plays important roles in gene regulation. Such regulatory RNAs have been
found to exist naturally in many organisms, but they have been used in a
relatively limited number of bacteria as a tool to regulate gene expression. In
this project, the investigators will develop generalizable RNA regulators that
can be used to understand and engineer diverse biotechnologically important
bacteria. Specifically, design principles for antisense RNA (asRNA), one of the
common regulatory RNAs in bacteria, will be determined to provide biologists and
bioengineers with predictable RNA tools for gene regulation. Additionally, this
project is broadly impactful by cultivating the next generation. Multiple
graduate students will be trained to perform the interdisciplinary project.
Moreover, our outreach program will introduce state-of-the-art synthetic biology
RNA tools to high school teachers, and provide them with teaching kits that will
be used in their classroom, broadening student participation in science and
engineering.&lt;br/&gt;&lt;br/&gt;The long-term goal of this project is to
determine quantitative design principles for asRNAs and provide a generalizable
model that enables predictably tunable asRNA-mediated gene repression in diverse
organisms. asRNA is one of several types of small regulatory RNAs, and it
represses gene expression by binding to its target messenger RNA. Although this
binding occurs following the simple RNA base-pairing rule, predictably tunable
asRNA-mediated repression is still challenging. The most important design
parameters for asRNA in Escherichia coli DH10B have been identified by using
three target reporter genes in plasmids. A predictive model for asRNA-mediated
repression, which relates repression efficiency to the identified parameters by
a multiple linear regression analysis, will be developed. To demonstrate the
generalizability of this model, three specific aims are proposed. First, the
model developed with plasmid reporters in E. coli DH10B will be validated by
expanding it to chromosomal gene targets. This task will allow for testing of
forward-designed asRNAs in different genetic contexts. Second, the predictive
model will be tested in three different bacterial species. This task will test
the hypothesis that asRNA design principles are consistent among diverse
bacteria. Third, the model will be further validated by building and testing
additional asRNAs. A user-friendly online version of the model will also be
developed to provide a way to collaboratively advance this RNA-based technology.