* 1715589
* Differential effects of genomic context on the binding specificity of paralogous transcription factors
* BIO,MCB
* 09/01/2017,01/31/2021
* Raluca Gordan, Duke University
* Continuing Grant
* Stephen DiFazio
* 01/31/2021
* USD 721,834.00

Cells often use similar proteins to perform very different cellular functions.
Focusing on regulatory proteins that dictate which genes are expressed in each
cell (among the large repertoire of &gt;20,000 human genes), this project will
decipher mechanisms that allow similar proteins to interact with distinct
regions of the genome in order to perform distinct functions. This is
significant because most human proteins, including the transcription factors
studied in this project, are closely-related members of large families. Over
time, these proteins diversified in function, often with little change in the
protein sequence. Our understanding of how closely-related transcription factors
can bind to different regions of the genome is very limited. To address this gap
in knowledge, this project will generate high-quality transcription factor-DNA
binding data and models, and will analyze the data to study genetic mechanisms
by which transcription factors target specific genomic regions. The data and
models will be made available to the scientific community, and are expected to
become a valuable resource for future studies of differences among related
transcription factors. This research will expose postdoctoral fellows, graduate,
undergraduate, high school, and middle school students to the genetic mechanisms
used by human cells to regulate gene expression. Trainees will participate in
data generation and analysis, as well as dissemination of results. Thus, they
will be introduced to molecular and computational biology through practical
studies of protein-DNA interactions. Trainees will also experience the power of
interdisciplinary research, as they will combine computational modeling and
biological experiments to address fundamental questions about human cells. In
addition, by using biological questions as motivation to teach computational
techniques, this project aims to attract female students of various age groups
to STEM fields.&lt;br/&gt;&lt;br/&gt;Interactions between proteins and DNA are
critical for many cellular processes, including the regulation of gene
expression. This project focuses on how closely-related transcription factor
proteins identify their unique DNA binding sites across the human genome, in
order to regulate the expression of target genes. Interactions between
transcription factors and DNA occur over short regions, oftentimes &lt;10
nucleotides. However, the DNA binding sites always occur within a specific
genomic context, which is very likely to influence the interactions with
transcription factors. This project will investigate three mechanisms by which
genomic context can influence binding of closely-related factors: homotypic
clustering (i.e. the presence of additional binding sites for factors from the
same protein family), heterotypic clustering (i.e. the presence of binding sites
for factors from other families), and the presence of repetitive DNA sequence
elements (which were recently shown to affect protein-DNA binding). The project
will use high-throughput cell-free assays to quantitatively measure the binding
levels of 21 human transcription factors (from 9 distinct proteins families) for
tens of thousands of DNA sites in their native genomic sequence contexts. For
each protein family, DNA sequences with different arrangements of binding sites
and different repeat elements will be selected from the human genome and
synthesized de novo on glass slides. Binding of related transcription factors to
the selected genomic sequences will be measured quantitatively, on the slides,
using genomic-context protein-binding microarray (gcPBM) assays. gcPBM data are
quantitative, highly reproducible, and can detect differences in binding between
closely related factors, which makes the gcPBM assay ideal for this project. The
generated data will be used to develop computational models of protein-DNA
binding specificity, and to identify differences in specificity between related
factors. All models will take into account characteristics of the genomic
context (homotypic clustering, heterotypic clustering, and repetitive DNA
elements). The models will be validated computationally and experimentally.
Finally, the project will assess the extent to which different mechanisms can
explain differential genomic binding in the cell. By identifying genetic
mechanism that contribute to the differential DNA binding of closely-related
transcription factors, this project represents a significant step forward in
understanding how these regulatory proteins select distinct genomic targets and
perform distinct functions in human cells.