* 1715207
* Functional Effects of Chromosome-Nuclear Envelope Interactions
* BIO,MCB
* 08/01/2017,07/31/2021
* Igor Sharakhov, Virginia Polytechnic Institute and State University
* Standard Grant
* Charles Cunningham
* 07/31/2021
* USD 907,000.00

The research goal of this project is to uncover the fundamental mechanisms
through which interactions between chromosomes and the periphery of the cell
nucleus regulate gene expression. The function of a cell, including gene
expression, is determined not only by the letters of the DNA code, but also by
how the DNA (contained in the chromosomes) is packaged tightly and intricately
inside the tiny volume of the cell nucleus. This complex nuclear architecture -
organization of the genome in 3D - has a profound effect on how the genes work
and how the cell functions. However, the effects of chromosome attachments to
the nuclear periphery on 3D genome organization and key cell functions are
largely unknown. This project will employ a tight integration between
computation and experiment: computational models can explore vast amounts of
spatial chromosomal configurations and many relevant parameter values to
identify a small subset of the most promising ones, which will be tested
experimentally. The research will establish a set of general principles that
translate the 'language' of 3D chromosome organization into the 'language' of
gene expression, which if of fundamental importance to biology. The project also
integrates research and education with emphasis on outreach among middle-school
students and mentoring undergraduate and graduate students. Specifically, the
research team will host a hands-on exhibit of 3D interactive models of
chromosomes at Kids' Tech University (KTU), which is a semester-long enrichment
program. Virginia Tech undergraduate and graduate students will have an
opportunity to investigate chromosomes in a Multi-Screen Immersive 3D Display
where researchers can be virtually present inside the cell nucleus. In addition,
the project will provide the broad scientific and educational communities with a
unique approach for visualizing the complex hierarchy of chromosome structure
and interactions in 3D, based on versatile and powerful, freely available,
visualization software already well-established in the structural biology
community.&lt;br/&gt;&lt;br/&gt;The project will study nuclear organization in
fruit fly giant polytene chromosomes using confocal microscopy and regular
interphase chromosomes using the powerful Hi-C technique. Experimental analyses
of nuclear architecture will be complemented and guided by computer simulation.
Specifically, the project will develop computational models of chromosome
organization in Drosophila nuclei for polytene and regular (non-polytene)
interphase chromosomes, including information-rich visualization tools. It will
also explore by simulation, and test experimentally, the hypothesis that
chromosome-nuclear envelope attachments in Drosophila nuclei affect the
frequency of long-range intra-/inter- chromosomal contacts. Finally, the
research will provide a direct connection to cell physiology by exploring the
effects of chromosome-nuclear envelope detachments on gene expression via re-
patterning of long-range intra-/inter-chromosomal contacts. Successful
completion of this highly interdisciplinary work will reveal the extent and
mechanism of how the functional organization of the genome in the nuclear space
is regulated by chromosome-nuclear envelope interactions - a common dynamic
feature of living cells whose effect on the nuclear architecture is unclear.