* 1716491
* Telomere Roles in Fungal Genome Evolution
* BIO,MCB
* 07/15/2017,06/30/2023
* John Carmen, University of Kentucky Research Foundation
* Standard Grant
* Clifford Weil
* 06/30/2023
* USD 737,507.00

This project will examine the roles that telomeres, the DNA sequences that form
the ends of chromosomes, play in chromosome evolution and in making new genes.
The project will study chromosome ends in the filamentous fungus Magnaporthe
oryzae - a devastating pathogen of rice and wheat plants. Of the more than 20
known genes that control fungus interactions with the host plants, at least 50%
are found close to the telomeres. It has long been a puzzle as to how (and why)
these genes find their ways to the chromosome ends. Recent work has shown that
the M. oryzae chromosome ends undergo rearrangements at spectacularly high
frequencies and this can lead to the capture and duplication of internal
sequences at the chromosome ends. This, in turn, causes the formerly
subtelomeric sequences to be relegated to the chromosome interior. This project
will use DNA sequencing and related data analysis methods to test the hypothesis
that telomere dynamics is a major driver of chromosome evolution. This project
largely involves DNA sequencing and associated data analysis techniques
(bioinformatics). Individuals with the relevant cross-training in biology and
computer science are rare and collaborations between individuals from the two
domains are challenged by differences in their training and skills. Recognizing
a national need for individuals with dual domain expertise, most of the
objectives for this project will be completed through research experiences for
teams of biology and computer science undergraduates from the partner
institutions of the Univ. of Kentucky, Northern Kentucky Univ., Eastern Kentucky
Univ. and Western Kentucky Univ., with specific efforts being made to enhance
participation for student populations traditionally underrepresented in STEM
research.&lt;br/&gt;&lt;br/&gt;Despite intense interest in telomere biology and
the importance of the associated chromosome regions (subtelomeres) in cellular
biology, organismal adaptation and evolution, the organization of chromosome
ends and their dynamics is poorly understood due to poor representation in
genome assemblies. The central hypothesis driving this project is that the
telomere regions are the main factories of fungal genome evolution, whereby
recurrent bouts of telomere crisis provide the fuel for raw materials
recruitment and telomere rescue pathways act to generate novel sequences. It has
long been proposed that fungal genomes might evolve through such "Adaptive
Telomere Failure" (ATF) but, until now, no examples had been identified. This
project will greatly expand knowledge on fungal genome structure and reveal the
full impact of ATF on genome organization and evolution. Additionally, it is
suspected that enhanced rates of nucleotide substitution/indel polymorphism in
the subtelomeric regions may further accelerate the neo-functionalization of
terminally captured sequences. This project will test a number of predictions
related to these hypotheses, while seeking to gain an improved understanding of
subtelomere-internal genome sequence exchange, as it relates to fungal genome
structure and host adaptation. Specific objectives include: 1. Generate end-to-
end genome assemblies for the main host-specialized forms of M. oryzae and
characterize the structural and genic diversity of the subterminal chromosome
regions. 2. Assess the impact of telomere dynamics on M. oryzae genome evolution
by identifying and characterizing internalized relics of former telomeres; 3.
Examine genome reorganization following spontaneous or experimentally-induced
telomere crisis events; and 4. Accomplish project objectives through "bite-
sized" undergraduate research projects that foster collaboration between
biologists and computer scientists.