* 1716431
* Mechanisms of epigenetic gene regulation by the Drosophila COMPASS-like complex
* BIO,MCB
* 08/01/2017,07/31/2023
* Andrew Dingwall, Loyola University Stritch School of Medicine
* Standard Grant
* Manju Hingorani
* 07/31/2023
* USD 938,838.00

Developmental processes in eukaryotic cells are controlled by DNA elements
called enhancers. The molecular nature of enhancers is not well understood,
although current evidence suggests they differ from other DNA elements by having
a distinct structure within the chromosomes. Structural differences can arise
from enzymatic modification of the histone proteins responsible for packaging
the DNA in the nucleus. This project focuses on studying the enzymes that modify
enhancer elements and how their action helps turn on critical genes at the right
time and location during development. The work will be carried out using a
multi-disciplinary approach that combines genetic, biochemical and high-
throughput bioinformatics tools in the fruit fly, Drosophila melanogaster, as
the model organism. Because the mechanisms governing how, when and where genes
are turned on and off during development are conserved across evolution, the
results should have far-reaching impact, from yeast to humans. The project will
have broad educational impact by providing students with training in multiple
areas, including protein structure modeling, biochemistry, bioinformatics,
genetics and advanced microscopy imaging. Students also learn how to develop
scientific hypotheses, independently carry out experiments and interpret
results, prepare oral and written summaries and publish their work, skills that
provide a strong technical knowledge base for diverse scientific
careers.&lt;br/&gt;&lt;br/&gt;The metazoan COMPASS related coactivator complexes
catalyze the methylation of histone H3 on Lysine 4 (H3K4), epigenetic marks
associated with controlling eukaryotic gene transcription. The Cmi/Trr COMPASS-
like complex in Drosophila is responsible for monomethylating H3K4 and regulates
enhancer activity in cooperation with transcription factors important for normal
development. Trr provides histone methyltransferase activity, while Cmi contains
plant homeodomain Zn fingers in two conserved clusters that co-evolved over 1.5
billion years with chromosome compaction in nucleosomes. Despite the
conservation and importance in transcription control, it is not well understood
how the COMPASS-like complexes are able to prime and maintain enhancer
activities during development. This project focuses on the role of the conserved
PHD finger domains in chromatin recognition and key cellular signaling pathways
that depend on precise enhancer control. Taking advantage of the versatile
Drosophila genetic model system, this project tests broad hypotheses (1) that
the clustered PHD domains found in Cmi and homologous vertebrate proteins
contribute essential epigenetic histone reader functions that drive the proper
priming and regulation of gene enhancers and (2) the Drosophila COMPASS-like
coactivator complex regulates the timing of enhancer utilization to integrate
key developmental signals. One aim of this project explores the combined
functions of the finger domains in the conserved PHD cluster that are essential
for proper enhancer regulation. Structural modeling studies of the PHD finger
domains and targeted mutagenesis, combined with in vitro and in vivo
measurements of chromatin association, will define the histone recognition and
binding properties of Cmi and mammalian PHD finger domains and further elucidate
the mechanisms of enhancer control by the COMPASS-like complexes. A second aim
incorporating developmental, genetic and molecular analyses (ChIP-seq, RNA-seq,
chromatin capture) will expand our understanding of target gene regulation in
vivo, help correlate specific enhancer epigenetic marks with COMPASS-like
complex functions, and provide significant new insights regarding conserved
mechanisms of chromatin regulation.