* 1724993
* RESEARCH-PGR: Leveraging the Medicago HapMap to characterize genome-by-genome interactions in nitrogen-fixing symbiosis
* BIO,IOS
* 08/15/2017,07/31/2019
* Katy Heath, University of Minnesota-Twin Cities
* Standard Grant
* Gerald Schoenknecht
* 07/31/2019
* USD 550,549.00

Nitrogen is required for the synthesis of amino acids and proteins, essential
components of the human diet. Nitrogen is also one of the most important
nutrients required by plants, and it is normally applied as fertilizer in
agricultural systems. However, legumes have the ability to establish a symbiotic
association with specialized nitrogen-fixing soil bacteria through the formation
of root nodules where the bacteria fix atmospheric nitrogen; the fixed nitrogen
is used by the plant for protein synthesis. The symbionts have mechanisms that
recognize molecular signals from each other, which in turn initiate processes
that lead to the formation of root nodules. However, there is significant
genetic variation among bacterial strains and among legumes in their abilities
to interact symbiotically. The objective of this proposal is to identify
bacterial and plant genes and gene variants responsible for the most efficient
and effective interactions. This objective will be accomplished by identifying
the bacterial strains that outcompete other strains in a large mixture in their
ability to form nodules in roots of Medicago truncatula, the barrel medic. These
experiments will be carried out with a large collection of barrel medic
accessions representing most of the variation in the species. Thus, the results
are expected to identify the most competitive strains and plant accessions.
Quantitative analysis of the DNA sequences of the most competitive types is
expected to reveal the identity of the bacterial and plant genes that are the
most effective in the symbiotic interaction. The project will train several
undergraduate students.&lt;br/&gt;&lt;br/&gt; &lt;br/&gt;&lt;br/&gt;Biological
N-fixation mediated by legume-rhizobium symbiosis contributes N to both natural
and agro-ecosystems, and results in legumes having N-rich leaves and seeds that
make them some of the most important sources of dietary protein. For this
reason, this symbiosis is a critical component of sustainable agricultural
systems. The legume-rhizobia symbiosis represents an ideal system to study the
coordination of function between a bacterial and a plant genome. Although both
partners benefit from this relationship, not all partnerships are equally
beneficial. The outcome of host-rhizobia interactions depends on the genotypes
of the partners. This means that there is not a single rhizobium strain that
confers the greatest benefit to all plant genotypes, and that different rhizobia
strains are favored by different plant genotypes. The objective of this proposal
is to identify the genomic basis that underlies G x G variation in mutualism,
which could enable manipulation of the interaction to maximize biological
N-fixation. The PIs have developed a select and resequence assay to identify
strains with the greatest fitness in a diverse multistrain community. The PIs
are proposing to refine and evaluate the robustness of the performance of this
assay that will satisfy various biological and statistical challenges. The
Broader Impacts of this project include the direct benefits to the agricultural
sector and the environment that will result from more efficient nitrogen-
fixation systems and the training of undergraduate students.