* 1736253
* RII Track-2 FEC: Using Biophysical Protein Models to Map Genetic Variation to Phenotypes
* O/D,OIA
* 08/01/2017,07/31/2021
* Daniel Weinreich, Regents of the University of Idaho
* Cooperative Agreement
* Subrata Acharya
* 07/31/2021
* USD 6,000,000.00

Non-technical description&lt;br/&gt;One of the great challenges in modern
biology is understanding how changes in amino acids that are the building blocks
of proteins lead to changes in the characteristics of a living organism. This
Research Infrastructure Improvement Track-2 Focused EPSCoR Collaborations (RII
Track-2 FEC) award addresses this challenge by using computer simulations,
mathematical modeling, and experiments to determine how amino acid changes
modify the way that proteins interact with other molecules. This information, in
turn, will elucidate how these changes modify the characteristics of organisms.
This research will contribute to basic scientific knowledge and will benefit
society by advancing biotechnology, agriculture, and human health. The work on
protein binding, for example, will improve the ability to predict which
pathogens will evade drug therapy or which molecules might interfere with
protein function. The project will provide resources to support early-career
faculty and mentor them in building robust, interdisciplinary, collaborative
research programs. It will also provide collaborative opportunities to students
and postdocs and prepare them for cutting-edge scientific careers. The research
is team-based, involving scientists and students in Idaho, Vermont, and Rhode
Island. Findings will be shared with the public via interactive animations, a
website for dissemination of results, and presentations for diverse audiences in
all three states including local science centers, schools, and regional Tribal
communities. The project will promote a diverse, vibrant, and sustainable
workforce and provide opportunities for research, education, and
outreach.&lt;br/&gt;&lt;br/&gt;Technical description&lt;br/&gt;The research will
utilize protein biophysical tools to develop genome to phenome analyses that
predict the impact of mutations and combinations of mutations on a broad range
of systems. The central scientific hypothesis for the project is that protein
biophysical models provide an efficient framework for predicting how mutations
influence protein stability, affinity for substrates and partners, and the
mappings to higher-level phenotypes. To test this hypothesis, the research team
will computationally predict the effect of mutations on folding and binding
stabilities and experimentally validate the predictions by protein purification,
circular dichroism, and isothermal titration calorimetry. The use of
computational approach first to select biophysically viable mutants will lead to
efficient experimental efforts. Initial studies will focus on beta-lactamase and
the respiratory syncytial virus F protein because they represent two fundamental
interaction types: protein-substrate and protein-protein. The researchers will
investigate how single mutations and combinations of mutations affect
biophysical phenotypes and their mappings to higher-level phenotypes in these
two systems. Further studies will focus on larger, naturally occurring,
mutational combinations and the role of the environment in modifying the
genotype to phenotype mappings. The research will identify generic and system-
specific lessons about the mapping of genotypes to phenotypes, such as how often
biophysical and higher-level phenotypes show epistasis and how well models
predict deviations from additivity. Participating early career faculty will
receive mentoring for advancement in their academic career and will have
networking opportunities for professional development. The project will develop
a website to aid in the communication of project results, tools, and activities.
Students and scientists will work with animators and virtual technology and
design experts to produce interactive animations called "Geno-Pheno-Mations" to
disseminate the project results to the general public.