* 1737127
* Collaborative Proposal: Selection and Genetic Succession in the Intertidal - Population Genomics of Pisaster ochraceus During a Wasting Disease Outbreak and its Aftermath
* GEO,OCE
* 12/01/2017,11/30/2020
* Ian Hewson, Cornell University
* Standard Grant
* Daniel J. Thornhill
* 11/30/2020
* USD 173,863.00

This project seeks to understand the outcomes of predator-disease dynamics by
exploring a recent pandemic that decimated 90% of ochre sea stars (Pisaster
ochraceus) in the eastern North Pacific in 2013. The research team will explore
how recovery may depend upon often difficult-to-see processes such as the
interplay of migration and natural selection in marine species. While the
population of sea stars is currently rebounding due to several years of
unusually high recruitment, the sea star wasting disease continues to persist at
low levels. This project aims to determine the genetic consequences of the
pandemic and subsequent recovery. The team will determine whether the majority
of susceptible sea stars have died and identify possible refuges where
susceptible sea stars survived. They will examine the potential for heritable
variation in resistance to this disease in order to assess whether the new
recruits are tolerant or susceptible to wasting. Resolving these issues will
enable predictions about the trajectory of their recovery and the potential
responses to future large scale disease outbreaks. Research findings will be
shared with resource managers and scientists at a collaborative workshop that
will focus on state-of-the-art methods to advance research on marine diseases.
The public will have the opportunity to learn more about sea star wasting
disease through a partnership with the UCSC Seymour Marine Discovery Center and
can track the incidence of disease using an online interactive map available at
www.seastarwasting.org. Results will be incorporated into professional
development for teachers with CalTeach and adapted for teaching materials up to
college-level. This project will train diverse early career scientists -
undergraduates, graduates, and a postdoctoral scholar - in integration of
ecological and genomic methods.&lt;br/&gt;&lt;br/&gt;Understanding the
consequences of large-scale pandemics in the broader contexts of geographic
heterogeneity and chronic changes in ocean pH and temperature is an emerging
contemporary issue. This project employs long-term characterization of
population dynamics and genetic consequences of a sea star wasting disease
(SSWD) outbreak, which caused median 90% mortality in Pisaster ochraceus
populations in the northeastern Pacific, to estimate potential long-term
consequences for the species. While the largest recorded influx of new recruits
occurred in 2014-2016, it is unknown where they originated from, whether
recruits and surviving adults remained susceptible to the disease, which
persisted at low levels, and for how long these dynamics might continue. This
long-term dataset provides a unique opportunity for exploring the short and long
term repercussions of such large-scale disease outbreaks and the population
dynamics that they precipitate. This project builds on long-term field studies
of wild populations to describe host population dynamics, the disease, and
genomic diversity. The goal is to discover genetic variation associated with
SSWD and to dissociate that variation from population genomic effects
attributable to abiotic environmental variation. Objectives are: (1) Census P.
ochraceus at 24 sites throughout its range to describe population dynamics, the
prevalence of SSWD, and measure abiotic variables. (2) Conduct laboratory
experiments coupled with RNAseq analyses to determine loci differentially
regulated during exposure to SSWD, temperature, salinity, and pCO2 anomalies.
(3) Map ddRAD, RNAseq, and candidate loci under selection to a P. ochraceus
genome. (4) Conduct range-wide population genomic analyses for 3 years to assess
genetic (SNP) variation among wild-caught specimens with, versus without, SSWD
across a geographic mosaic of abiotic variation. (5) Explore links between SSWD
and candidate loci, such as EF1A. These analyses will describe the immediate
genomic consequences of the disease outbreak, the population dynamics that the
outbreak set in motion, and the interplay of factors and mechanisms - such as
disease, temperature, migration, selection - that affected these changes. The
results will advance understanding of general processes and interactions that
shape population genomic structure in coastal ecosystems, providing resources to
inform future research and applications in design of management strategies for
coastal living resources.