* 1742870
* EAGER:Collaborative: Metagenomic Analysis to ensure water quality and safety
* ENG,CBET
* 08/01/2017,12/31/2018
* Menu Leddy, Orange County Water District
* Standard Grant
* Karl Rockne
* 12/31/2018
* USD 68,404.00

1742869/ 1742870&lt;br/&gt;Huq; Leddy&lt;br/&gt;&lt;br/&gt;With increased water
scarcity, it is critical that safe water supplies be maintained, including
wastewater for environmental discharge or reuse. Currently microbial (bacterial
and viral) water quality is measured using standard microbiological and
biotechnological culture methods to establish safe water quality. This proposed
EAGER is transformative as the study aims to explore the use of a newly
developed detailed and sensitive metagenomic sequence analysis to understand and
identify microbial communities for accurate determination of biological water
quality in water, wastewater, biofilms, and water reuse
systems.&lt;br/&gt;&lt;br/&gt;With the proposed metagenomic analysis system,
microorganisms, including bacteria, viruses, fungi, and parasites, will be
accurately identified to species and strains. Additionally, antibiotic
resistance and pathogenicity properties are also identified to evaluate
diversity and abundance of the genes and microorganisms in-situ. By assessing
the microbial communities and comparing their composition, a better
understanding of the microbial communities and their ecology is developed to
provide improved methods for assessing microbial water quality for indirect
potable reuse and direct potable reuse. Culture-dependent methods and indicator
microorganism measurements have proven to be slow and biased because they do not
represent all species, often not even the dominant species present during
advanced water and wastewater treatment. Molecular techniques have provided
insights into microbial community structure; however, these methods are
seriously limited because they target single microorganisms that are often the
less dominant species. Therefore, with metagenomic analysis using next
generation sequencing, accurate resolution of the microbial communities and
their diversity is achieved, resulting in significant benefits for environmental
and public health applications. Metagenomic whole DNA sequencing involves
shearing all DNA in a sample that is then sequenced by next generation
sequencing. Next generation sequencing is non-discriminatory, identifying all
organisms, even the underrepresented, while identifying functional and resistant
genes, pathogenicity, and providing taxonomic structure based on phylogenetic
trees, all in a single analysis and within approximately one day. Therefore,
next generation sequencing provides a more complete identification, with greater
resolution of the microbial community and its ecology. The results from this
study have the potential of defining microbial communities in water systems at
each step of the process and will transform the ability to understand and design
better indicators, validate surrogates, and also define sustainable microbial
communities beneficial for drinking water and water reuse. The long term
societal impact will be paramount as the proposed project will yield a more
defined microbial community structure for water quality and safe water supplies
and will demonstrate the utility of this important, emerging approach for the
water and water reuse industry. During the course of this study, a doctoral
student will gain fundamental and applied knowledge in the area of
bioinformatics and water reuse that encompasses advanced treatment technologies
that are employed in water reuse. Student interns will have the opportunity to
learn field sampling techniques. This study will establish a good foundation to
achieve notable broader impacts for the environment.