* 1744001
* TRANSFORM-PGR: Whole genome assembly of the maize NAM founders
* BIO,IOS
* 01/15/2018,12/31/2021
* R. Kelly Dawe, University of Georgia Research Foundation Inc
* Continuing Grant
* Gerald Schoenknecht
* 12/31/2021
* USD 3,111,447.00

Non-Technical Abstract: Maize is not only an important crop but an important
study species for answering basic questions about how plants grow and adapt to
different environments. Genome assemblies, which are complete representations of
the genetic information in a plant variety, are critical resources for answering
these important questions. However, currently only a single type specimen is
used as the sequence reference for most of the genetic information in maize,
leaving unknown much of the highly valuable natural variation in maize. This
project will assemble the genomes of 26 additional maize lines, chosen to
represent a broad cross section of the maize lines used in modern breeding. The
sequence assemblies will be enhanced by adding information about the nature of
the genes and how the genomes differ from each other. All information will be
released on an accelerated schedule through public databases.
&lt;br/&gt;&lt;br/&gt;Technical Abstract: Maize is an important crop and model
organism for plant genetics. However, currently nearly all forms of sequence
analysis are referenced to the single B73 inbred. Beyond B73, the most
extensively researched maize lines are the core set of 25 inbreds known as the
NAM founder lines, which represent a broad cross section of modern maize
diversity. Prior data show that gene content can differ by more than 5% across
lines and that as much as half of the functional genetic information lies
outside of genes in highly variable intergenic spaces. To capture and utilize
this variation, the NAM founder inbreds and a twenty-sixth line containing
abnormal chromosome 10 will be sequenced and assembled using a mate-pair
strategy. Scaffolds will be validated by BioNano optical mapping, and ordered
and oriented using linkage data. RNA-seq data from multiple tissues will be used
to annotate each genome, and assemblies and annotations will be released with
genome browser support through MaizeGDB, NCBI, and Cyverse. Comparative genomic
tools will be used to identify and to catalog the maize pangenome, and to assess
the role of structural variation such as presence-absence variation and copy
number variation in the determination of agronomic traits. Results will be
disseminated through a project web site and a CyVerse/Gramene/MaizeCODE Workshop
at the 61st Annual Maize Meeting in March of 2019.