* 1753489
* SG: Inferring phylogenies under ancestral population structure
* BIO,DEB
* 03/01/2018,09/30/2019
* Michael DeGiorgio, Pennsylvania State Univ University Park
* Standard Grant
* Christopher Schneider
* 09/30/2019
* USD 200,000.00

Accurate estimation of population and species relationships from genetic data is
essential for understanding the evolutionary history of everything from
influenza viruses to humans. However, as genetic datasets rapidly grow in size
due to technological advances, a number of hurdles arise when trying to estimate
such relationships. Though many methods have been developed to address these
challenges, one source of error that is not accounted for by available methods
is non-random mating in ancient populations. Because individuals generally do
not mate randomly, and because population and species relationships are used to
answer diverse research questions from basic science to epidemiology, addressing
this source of error is critical. The primary goal of this project is to design
statistical methods for estimating population and species relationships that
account for non-random mating in ancient populations, thereby increasing the
accuracy of estimation. Moreover, it is of high priority that both the
scientific community and the public are engaged in the advances of this project.
To this end, the researchers will make all approaches developed during this
project freely available for use by the wider scientific community. Also, the
researchers will work with K-12 students in hands-on activities for learning why
and how to build population and species relationships through the Penn State
Science-U program. Finally, the researchers will engage indigenous peoples as
part of the Summer internship for INdigenous peoples in Genomics (SING) Workshop
which examines the use of genomic data in science and
society.&lt;br/&gt;&lt;br/&gt;Ancestral structure, which has been uncovered in
many diverse species, can skew gene tree frequencies, thereby hindering the
performance of methods for estimating species trees. This research seeks to
develop novel likelihood methods that can infer phylogenies under such
scenarios, and apply these methods to test evolutionary hypotheses about
ancestral structure and gene flow in several model and non-model organisms. The
model organisms considered will be mouse, yeast, and mosquito, for which
previous studies have observed skewed gene tree frequencies that were attributed
to gene flow through hybridization, but may instead be the result of ancestral
structure. The researchers will also apply these methods to a non-model coral
system, which is of particular interest because morphological and fossil data
provide evidence of hybridization, suggesting that this system may exhibit
skewed gene tree frequencies. Application to these systems will serve to
elucidate and refine knowledge of the events shaping the evolution of these
lineages.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.