* 1754337
* The Microbial Metagenome of the Termite Hindgut
* BIO,DEB
* 07/01/2018,06/30/2023
* Gillian Gile, Arizona State University
* Standard Grant
* Maureen Kearney
* 06/30/2023
* USD 232,954.00

Termites have the remarkable ability to survive on a diet of wood. From one
perspective, this can be a big problem, costing billions in damage repair to
man-made structures and termite eradication. On the other hand, it is also an
opportunity to understand how energy can be efficiently extracted from wood.
Termites rely on a complex community of microorganisms in their hindgut to break
down wood and provide energy for themselves. However, the identity of many of
these organisms, and the specific roles they each play in wood digestion, are
still unknown. This research focuses on a highly invasive and destructive
subterranean species of termite, and aims to identify the community diversity of
their associated microorganisms and to understand their specific roles in the
wood digestion process. This can be achieved in part through genome sequencing,
as each microorganism's metabolic capabilities can be predicted from its genome
sequence. This information is fundamentally important for science and society as
it may enable exploring alternative approaches to biological termite control as
well as biofuel and biomaterial production. In addition to providing research
training for students, this project also develops an interactive outreach module
for K-12 students to help spark appreciation for science, technology, and
engineering research.&lt;br/&gt;&lt;br/&gt;The Coptotermes formosanus hindgut
community includes large, complex eukaryotic microbes called hypermastigotes.
Their genomes are unexplored but are expected to be very large, necessitating
deep sequencing. The hindgut community also includes bacteria, archaea, and
viruses, at highly variable abundances. To adequately sequence microbes from
each of these "domains" of life, enrichment steps will be performed before
sequencing, namely isolation of hypermastigote nuclei, isolation of viral
particles, and rolling-circle amplification of lower abundance bacterial
genomes. Assembled microbial genomes will be identified by 16S or 18S ribosomal
RNA genes, so that carbohydrate active enzyme genes can be assigned to microbial
species. Nitrogen fixing capability will also be inferred by quantitative real-
time PCR of nifH genes. For the most abundant bacteria, their symbiotic status
(ectosymbiont, endosymbiont, or free swimming) will be determined through
fluorescent in-situ hybridization. By identifying and inferring the metabolic
capabilities of each microbe in the hindgut bioreactor, this project will lay
the groundwork for a deeper understanding of efficient wood
digestion.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.