* 1755761
* CRII: III: Computational Methods to Explore the Role of Post-transcriptional Regulation in Cancer
* CSE,IIS
* 07/01/2018,06/30/2021
* Wei Zhang, The University of Central Florida Board of Trustees
* Standard Grant
* Sylvia Spengler
* 06/30/2021
* USD 187,021.00

This project will develop computational methods to investigate the role of post-
transcriptional regulation in cancer patients using both high-throughput
sequencing data and molecular networks to improve the understanding of cancer
and assess the cause of the disease in a patient. Deregulation of gene
expression is a hallmark of the human tumor cells. Post-transcriptional
regulation is a pervasive mechanism in the relation of most human genes; its
implication in cancer is only beginning to be appreciated. The central
hypothesis underlying this project is that by considering post-transcriptional
regulation events, estimated protein expressions can provide more accurate
molecular signatures to detect complex disease mechanisms, compared to mRNA
expressions. This project will study post-transcriptional regulation, with the
goal of generating computational methods to predict the changes of protein
expression level without doing large-scale proteomics experiments. The outcomes
of the proposed research will lower the barriers for interacting with analyzing
high-dimensional genomic profiles and cut time and costs spent on biomedical
research. The new methods will enable biologists and biomedical researchers to
perform comprehensive analysis with high-throughput sequencing data and
biological networks together to investigate the impact of post-transcriptional
regulation in certain cancer types.&lt;br/&gt;&lt;br/&gt;This project to use a
variety of sequences modalities has three phases: 1) Develop advanced machine
learning methods to identify the genome-wide alternative polyadenylation (APA)
events in the 3' untranslated region (3'UTR) of the mRNAs between cancer
patients and matched normal samples with RNA-Seq data. 2) Develop biologically
motivated graph-based learning models and efficient scalable algorithms to
estimate the protein expression levels with microRNA-mRNA interaction networks
and 3'UTR-APA events. 3) Investigate the prognostic power of the identified APA
and estimated protein expressions compared to the canonical genomic features
such as mRNA expression, mutation, and copy number variations. The project not
only employs interpretable computational models which capture the causal
relations in post-transcriptional regulation of protein-coding genes to better
understand complex human diseases, but also develops unique machine learning
methods and optimization techniques for computer science
research.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.