* 1759940
* ABI Development:  Computational infrastructure for phylogenetic analysis in the R environment
* BIO,DBI
* 04/01/2018,03/31/2024
* Liam Revell, University of Massachusetts Boston
* Standard Grant
* Reed Beaman
* 03/31/2024
* USD 454,589.00

Over the past several decades, phylogenetic trees, the graphs that are used to
represent the ancestor-descendant relationships between gene sequences,
populations, or species, have become integral in scientific research across an
increasingly widening range of biological disciplines such as ecology,
evolutionary biology, molecular biology, comparative genomics, and development.
Phylogenies too have begun to see application outside of biology in such diverse
fields of study as comparative linguistics, anthropology, and forensic science.
Meanwhile, the computational environment known as 'R' has developed into one of
the leading platforms for scientific computing. The strength of R comes from the
literally thousands of contributed function libraries (called "packages")
developed and distributed for free by computational scientists and developers
that are not members of the R core development team. Because of these
contributions, R has grown into the preeminent software platform for
manipulating, visualizing, and analyzing phylogenetic trees. That said, the
capability to actually estimate phylogenetic trees from data in R remains
relatively limited. The major objective of this award is to substantially extend
the toolkit that is available to undertake phylogeny estimation in the R
scientific computing environment. This toolkit will be built using an open
development model and will be distributed as a free and open-source resource to
the national and international scientific communities. Among the major
advantages of developing software for scientific computing in R is that so doing
dramatically facilitates scientific reproducibility by allowing all stages of
the data analysis pipeline to be built into a single, fully reproducible
workflow from open source components.&lt;br/&gt;&lt;br/&gt;This research will
create new, open-source, and freely distributed software tools that will be of
broad utility throughout the phylogenetic scientific community and beyond. The
general goal is computational infrastructure development for phylogenetics in R.
These investigators will focus on four specific objectives in particular: (1)
the development and application of methods to compute distances between trees;
(2) the development of new consensus and average tree methods; (3) the
development of joint species-tree estimation methods based on likelihood; and,
finally, (4) the application of visualization tools for sets of trees and
phylogenetic networks. They focus specifically on these research objectives
because they interact in particularly interesting ways and because they reflect
areas in which substantial research is needed; however, research on these topics
will likely result in unanticipated discoveries in fundamentally new areas. The
award will also support postdoctoral and graduate training, the latter via an
intensive, phylogeny methods boot camp for young developers. This training
objective will be undertaken with the aim of helping to form the next generation
of computational phylogenetic biologists and will target for participation
graduate students from groups traditionally underrepresented in the sciences.
Phylogenetic tools from this project will be hosted on the CRAN at https://cran.
r-project.org/web/packages/phytools/index.html&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.