* 1759277
* Collaborative Research: ABI Development: The next stage in protein-protein docking
* BIO,DBI
* 06/01/2018,05/31/2022
* Dmytro Kozakov, SUNY at Stony Brook
* Standard Grant
* Reed Beaman
* 05/31/2022
* USD 312,500.00

Protein-protein interactions are integral to many mechanisms of cellular
control, including protein localization, allosteric and gene regulation and
signal transduction, and therefore their characterization is an important task
for both experimental and computational approaches to systems biology. Genome-
wide proteomics studies provide a growing list of putative protein-protein
interactions, and demonstrate that most if not all proteins have interacting
partners in the cell. However, these techniques can only identify whether two
proteins possibly interact. A full comprehension of how proteins bind and form
complexes can only come from high-resolution three-dimensional structures, since
they provide the atomic details necessary to understand how the interactions
occur and how the high degree of specificity can be achieved. While the most
complete structural characterization is provided by X-ray crystallography, many
biologically important interactions occur in weak, transient complexes that are
not amenable to direct experimental analysis, even when both proteins can be
isolated and their structures determined. The goal of this project is to further
improve the protein-protein docking server ClusPro that, starting from the
structures of component proteins, attempts to determine the structure of their
complexes with accuracy close to that provided by X-ray crystallography. The
server at https://cluspro.org already has close to 10,000 registered users,
although it can also be used without registration. Docked structures generated
by ClusPro have been reported in over 600 research papers. The public server is
important for biological and chemical scientists who may not have extensive
computational experience but still will be able to use state of the art docking
methods. The increased availability of protein complex structures will have
major impact in many areas of biology, biochemistry, and biotechnology.
&lt;br/&gt;&lt;br/&gt;The project has six aims focusing on problems of
increasing difficulty. First, a new opportunity to progress is provided by
earlier development of the manifold fast Fourier transform (FFT) correlation
algorithm, which increased the speed of sampling by orders of magnitude. An
additional advantage of the method is that increasing the number of correlation
function terms is computationally inexpensive. Therefore, Aim 1 is the
development and implementation of complex energy functions, including distance-
dependent and three-body potentials for scoring. Second, recently developed
methods have shown considerable promise in predicting contacts between residues
in proteins using evolutionary covariance information. The problem is that these
methods require large numbers of evolutionarily related sequences of interacting
proteins to robustly assess the extent of residue covariation. Aim 2 is
exploring how sequence information can be used for extracting inter-protein
contacts to improve docking results even with somewhat limited number of
sequences. Third, with the increase in the number of protein complex structures,
there is increasing need for integrating direct docking and template-based
prediction methods, and several strategies will be explored to implement such
integration. Fourth, the docking method will be modified to work optimally with
homology models rather than X-ray structures of interacting proteins. The fifth
aim is exploring new approaches to the docking of proteins with substantial
backbone conformational changes upon binding. The sixth and final aim is
implementing the newly developed algorithms in the ClusPro
server.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.