* 1759625
* ABI Innovation: Identification and correction of incorrect sequences in reference protein databases
* BIO,DBI
* 05/01/2018,06/30/2022
* William Pearson, University of Virginia Main Campus
* Standard Grant
* Peter McCartney
* 06/30/2022
* USD 825,826.00

With the sequencing of the Human genome and the dawn of the genome age,
physicians and scientists have a complete 'parts-list' - DNA and protein
sequences - for the machinery in the cells of thousands of different organisms,
from people to plants to bacteria. But, because we cannot look at the machines
or their parts directly, some of the details are incorrect. The methods produce
incorrect details in part because they were developed in the 'pre-genome' era,
when our parts lists were far less complete, and in part because we use the
parts-list from one organism (mouse) to help us identify the parts in another
organism (rat). But, if the first set of parts (protein sequences) has errors,
those errors can end up in the second set, and as more organisms are studied,
the errors multiply. Incorrect protein sequences make it more difficult to
identify mutations that may be associated with cancer and other diseases. Today,
protein sequence quality control largely relies on the accuracy of other protein
sequences. This proposal seeks to greatly expand the types of information used
to confirm that the details (sequences) of the cellular machinery are correct,
and to collaborate with the Uniprot protein sequence database to ensure that
incorrect sequences are corrected and so that future analyses do not repeat
mistakes. The proposed research will increase researchers' confidence that
exciting differences between phenotypes, such as diseased and normal cells, or
different strains of food crops, are genuine, and not the result of mistaken
protein sequences.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;The UniProt Reference protein
sets are a critical resource, providing a foundation for advances in genome
biology, structure/function prediction, and personalized medicine. But Reference
protein sets have characteristic errors that can be amplified in genome
annotation, reducing the accuracy of functional annotation and similarity search
sensitivity. This project is a collaboration with the Uniprot protein resource
that will develop novel strategies for identifying protein sequence errors;
these strategies will be incorporated into the Uniprot annotation pipelines to
remove erroneous sequences from Reference databases. Initially, errors will be
found by integrating exon-boundary and Pfam domain annotations into BLAST-based
similarity search results to identify currently recognized errors, such as
missing/additional exons and partial domains. In addition, strategies will be
developed to detect currently unrecognized errors, by clustering similar
proteins and looking for 'outliers'. Lists of 'suspect' protein sequences will
be examined by Uniprot annotators and classified as true or mistaken errors, and
successful error detection strategies will be refined and integrated into the
Uniprot annotation pipeline, reducing functional annotation
errors.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.