* 2308495
* Principled phylogenomic analysis without gene tree estimation
* MPS,DMS
* 08/01/2023,07/31/2026
* Sebastien Roch, University of Wisconsin-Madison
* Standard Grant
* Amina Eladdadi
* 07/31/2026
* USD 295,263.00

This project aims to improve the estimation of species trees from genomic
datasets. This estimation is challenging because different genomic regions
evolve under processes that make their evolutionary histories (i.e., gene trees)
discordant. This issue is exacerbated by widespread gene tree estimation errors
in modern phylogenomic analyses. To address this challenge, this project's
primary objective is to devise innovative mathematical, statistical, and
computational techniques to analyze phylogenomic datasets without relying on
gene tree estimation. This approach will produce more reliable species tree
estimates in the presence of confounding processes. Species trees provide an
evolutionary and comparative context in which many biological questions can be
addressed. They play a vital role in understanding gene evolution, estimating
divergence dates, detecting adaptation, studying trait evolution, etc. The
developed methods will enhance the precision of biological discoveries based on
species trees, advancing research that utilizes phylogenies. The project
includes interdisciplinary research training for graduate students as well as
the involvement of undergraduate students recruited through local initiatives.
New course materials based on the proposed research will be developed for
existing graduate courses and be made available through the PIâ€™s website. The
project will leverage connections to NSF-funded interdisciplinary institutes.

It is well established that different regions of a genome can evolve under
different gene trees, due to processes such as incomplete lineage sorting, gene
duplication and loss, and lateral gene transfer, complicating the estimation of
species trees. Many methods that first estimate gene trees and then combine this
information to estimate a species tree are known to have good theoretical
guarantees, under the assumption that the true gene trees are known. That
assumption is not satisfied in practice. Accounting theoretically for gene tree
estimation error has proved challenging and few results are available. Building
on prior work by the PI on the rigorous study of stochastic processes arising in
this phylogenomic context, the proposed research will establish much-needed
theoretical foundations for the analysis of multi-locus, multi-site datasets and
the estimation of species trees without gene trees, including the development of
novel estimators, the derivation of impossibility results and matching finite
sample bounds, and the investigation of the effect of intra-locus recombination.
This project will also enable the development of statistically rigorous,
scalable algorithms. This interdisciplinary research will involve a close
integration of applied probability, statistical theory, graph algorithms, and
evolutionary biology.

This proposal is jointly funded by the Mathematical Biology and Statistics
Programs at the Division of Mathematical Sciences.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.