* 2310355
* Collaborative Research: RESEARCH-PGR: Predicting Phenotype from Molecular Profiles with Deep Learning: Topological Data Analysis to Address a Grand Challenge in the Plant Sciences
* BIO,IOS
* 07/01/2023,06/30/2027
* Daniel Chitwood, Michigan State University
* Standard Grant
* Diane Jofuku Okamuro
* 06/30/2027
* USD 639,998.00

Organisms are a consequence of information embedded in their genome expressed
through molecular processes. Sequencing technologies allow biologists to extract
nearly all information content from the genome. However, measuring what an
organism is has not advanced as far as genomic sequencing: unlike the genome, it
is not yet possible to measure the totality of information embedded in the
organismal form. If all the information that is contained within organisms could
be extracted, a model could be developed that would address one of the Grand
Challenges in biology, the ability to predict what an organism is from its
genomic information. In this project, mathematical approaches that have not been
fully explored in biology will be used to extract information in data by
measuring its structure. This field of mathematics has a motto: that all shape
is data, and all data have shape. By measuring the shapes and gene expression
patterns of leaves, the project will treat them as data from which embedded
information can be extracted. Deep learning methods will then be used to predict
the shapes of leaves from their gene expression profiles. As part of the
connection between the project and its impact to society, students from both the
U.S. and México will help analyze the data through Plants&Python, a bilingual,
freely available curriculum initiated as a means to bring together plant
biologists who have never coded and data scientists new to plant science, with
groups that comprise U.S. agriculture. Using X-ray Computed Tomography (CT) to
measure plant morphology and transcriptome profiling (RNA-seq) to measure gene
expression, the project will use the Euler Characteristic Transform (ECT) and
the Mapper algorithm, two Topological Data Analysis (TDA) techniques, to extract
the total information embedded in the leaf morphology of Arabidopsis accessions
with contrasting developmental reproducibility. The ECT is mathematically proven
to distinguish any object from any other, and the Mapper algorithm is used to
visualize underlying data structures as a graph. Specific aims include: 1) using
the ECT to measure the total information embedded in leaf shape and benchmarking
against traditional methods to see how much “hidden” phenotypic information is
revealed when measured comprehensively; 2) generating RNA-Seq gene expression
profiles from identical leaves, visualizing the underlying data structure as a
Mapper graph; the same will be done for phenotypic data as measured by the ECT;
and, 3) predicting the precise leaf shape features associated with gene
expression signatures using deep learning. By converting underlying molecular
and phenotypic data structures into node embeddings, an encoder-decoder neural
network will align molecular and phenotypic Mapper graphs. The result will be a
mapping of gene expression profiles to features of leaf shape as predicted using
deep learning methods on underlying data structures. All project outcomes will
be made publicly available through long term data repositories.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.