* 2319757
* EDGE FGT: RNAi-based tools to unlock functional genomics of obligate oomycete plant pathogens
* BIO,MCB
* 09/15/2023,08/31/2025
* John McDowell, Virginia Polytechnic Institute and State University
* Standard Grant
* Manju Hingorani
* 08/31/2025
* USD 400,000.00

The goal of this project is to develop and optimize new techniques to
investigate the function of genes in a class of plant pathogens called “downy
mildews”. Many downy mildew species cause destructive diseases of crops that
include cruciferous vegetables, peas, cucumbers, and grapes. These pathogens
cannot be studied efficiently with conventional genetic and molecular
techniques, because of their “obligate” lifestyle in which the pathogens cannot
be cultured apart from their host plants. Therefore, little is known about how
downy mildews evolve resistance to fungicides and overcome resistance in their
plant hosts. The PIs of this project have developed novel techniques, based on
RNA interference, to inactivate specific downy mildew genes. This project will
optimize the efficiency and cost-effectiveness of these techniques and will
generalize their applicability for study of diverse downy mildew species. The
deliverables will enable the research community to overcome a major obstacle for
understanding the molecular mechanisms and evolution of plant diseases caused by
downy mildews. In addition, this research will lay important groundwork for the
long-term goal of developing RNAi-based biofungicides to control downy mildews
and other crop diseases. The project includes outreach to growers and the public
about RNAi-based biofungicides for plant disease control, framed in the topical
context of “RNA vaccines for plant diseases”. The project will also provide an
eclectic training experience for postdoctoral scholars and undergraduate
researchers.

The project will develop new functional genomic tools for understudied downy
mildew pathogens, building on two breakthroughs by the PIs: First, RNA
interference (RNAi) can be triggered against downy mildew (DM) genes by mixing
short, synthetic, double-stranded RNAs (SS-dsRNAs) in downy mildew spore
suspensions. Treated spores are analyzed in isolation or inoculated onto plants
to assess pathogen virulence. This approach is surprisingly simple but also
prohibitively expensive for large-scale functional genomics and for applications
in the field, due to high costs of dsRNA synthesis. Moreover, the approach needs
optimization (e.g., to protect dsRNA) and generalization to diverse DM species.
Accordingly, the second breakthrough is development of a one-step process for
production and encapsulation of dsRNA in anucleate “mini-cells” of E. coli. The
resultant minicell-encapsulated dsRNAs (ME-dsRNAs) are protected from
environmental degradation, can be shelved for long periods, and provide
effective resistance to fungal pathogens when applied as a spray to plants. This
protection exemplifies “Spray-Induced Gene Silencing” (SIGS) which has shown
promise as a tool for research and control of diseases caused by viruses, fungi,
nematodes, and insects. However, neither ME-dsRNAs nor SIGS have been tested on
DMs. Therefore, the aims of this proposal are to develop low-cost procedures for
production of “naked” and ME-dsRNA in E. coli and test the RNAi efficacy of
these formulations compared to SS-dsRNA. The resultant protocols will circumvent
a major bottleneck for genotype-phenotype research on DM-plant interactions, at
scales ranging from molecular to evolutionary.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.