* 2325149
* eMB: Mathematical analyses of multidimensional single cell transcriptional vector fields
* MPS,DMS
* 09/01/2023,08/31/2026
* Jianhua Xing, University of Pittsburgh
* Standard Grant
* Amina Eladdadi
* 08/31/2026
* USD 250,000.00

Cells can exist in different types such as liver cells and neurons. It is a
fundamental question in biology how a cell can develop into an organism composed
of different cell types. Furthermore, one can induce a cell to change from one
type (e.g., skin cells) to another type (e.g., neurons or heart muscle cells)
for curing various diseases such as Alzheimerâ€™s disease. Current advances of
single cell techniques and machine learning algorithms lead to development of
data-driven mathematical equations to describe how a large number of genes
regulate each other to regulate the cell types. The funded research is to apply
mathematical tools originally developed in other fields to analyze these
equations. Such mathematical analyses will provide mechanistic insights on
understanding how genes coordinately regulate cell types, and guide experimental
designs to effectively accelerate or slow down transitions between different
cell types, which are of high biomedical significance. The project also fosters
collaborations between mathematicians and cell biologists as well as train
students and researchers of the next generation in interdisciplinary research.
With advances of single cell genomics techniques, an emerging direction is to
learn dynamical models from the data. The research aims to address a new
challenge to expand the tools for downstream analyses of the dynamical equations
and apply tools developed in other contexts to single cell data analyses. In
mathematical biology researchers have toolkits to analyze the behaviors of a
biological system described by a set of dynamical equations. These tools, such
as phase-plane analyses and bifurcation analyses, are typically restricted to
either a system with a few degrees of freedom or a few selected parameters.
Thus, new tools are needed for analyzing dynamical equations with a large number
(e.g., > 20) of degrees of freedom. The investigators will use the recently
developed discrete graph representation of a vector field together with discrete
Hodge decomposition and other network analysis methods for characterizing multi-
dimensional vector field and system dynamics including cyclic transitions on
developmental and cell fate transition processes. The investigators will also
study nonlinear dynamical systems, including dynamical mode decomposition,
Koopman operator analysis, pseudospectral analyses, and spectral submanifold
analyses, and analyze the collective dynamical modes of cellular dynamics within
and between attractors.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.