* 2325172
* Collaborative Research: ML/AI-assisted environmental scale microbial nonlinear metabolic models
* MPS,DMS
* 09/01/2023,08/31/2026
* Karsten Zengler, University of California-San Diego
* Standard Grant
* Amina Eladdadi
* 08/31/2026
* USD 125,000.00

Our world is dominated in many respects by communities of single celled
organisms, e.g., bacteria, archaea and non-filamentous fungi. Such communities
are key players in all geochemical cycles; they are present in all multicellular
organisms, including humans, where in addition to possibly harmful effects, they
also have essential beneficial roles. They are ubiquitous in engineered systems
as well where, again, they can be beneficial (e.g., waste water treatment) or
harmful (e.g., drinking water distribution). In large part, microbial
communities interactions with their environments are metabolic through chemicals
they take in, products they make with these inputs, and byproducts they excrete,
so understanding these metabolic capabilities is essential for understanding how
microbes effect their surroundings. Advances in genetic sequencing are making it
easier and easier to determine microbial "machinery" (enzymes); researchers are
becoming increasingly adept in predicting how this "machinery" combines into
"assemblage lines" (metabolic pathways). Armed with this knowledge, the next
step is to understand how these "assembly lines" fit into their environment into
a sort of large scale "distribution system" that determines overall microbial
community function. At large scale, this becomes a challenging computational
problem, and current methods are not adequate. This project aims to accelerate
these computations by introducing machine learning tools into key bottlenecks in
the algorithms. The project is a collaboration between Temple University,
Montana State University, and the University of California, San Diego and offers
valuable educational, training, and outreach opportunities. Activity funded by
this proposal would center on development of computational methods, based on
machine learning and artificial intelligence assisted optimization, that are
sufficiently efficient so as to make it possible to embed complex cell-scaled
models of microbial behavior (metabolic and gene expression models, so-called ME
modes) into environmental scale PDE-based models of microbial community
activity. In support of this effort, ME models of several specific organisms (S.
aureus, S. epidermidis, B. subtilis) will be adapted for use in specific
environmental-scale models (basic biofilm communities and built-environment
subaerial communities). In complement, environmental-scale continuum-mechanics-
based (partial differential equation) models for these systems will be
constructed and adapted for use with the new computational methods. AI will also
be applied at this macroscale to attempt to identify key metabolic processes at
the large scale.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.