* 1501423
* DISSERTATION RESEARCH: A coalescent analysis of the diversity of standing genetic variation and adaptive evolution in the threespine stickleback
* BIO,DEB
* 06/01/2015,07/31/2017
* William Cresko, University of Oregon Eugene
* Standard Grant
* Leslie J. Rissler
* 07/31/2017
* USD 18,850.00

Evolution by natural selection is thought to often take thousands or millions of
years. The environments in which species live can change drastically on much
shorter timescales, however, and species must adapt quickly or risk extinction.
Along the west coast of the US and Canada, a small fish species, the threespine
stickleback, is known to adapt within decades to wildly different environments.
This study will examine how stickleback achieve this feat by focusing on the
influence a species' gene pool has on its ability to respond rapidly to a
changing environment. Specifically, the proposed research will examine (1) how
much of adaptive evolution is aided by genetic variation already existing in the
stickleback gene pool, and (2) how the stickleback's history of adaptation has
influenced the organization of genetic variation in its entire genome. This
understanding of adaptive potential is crucial to our ability to predict how
species and ecosystems will respond to future environmental shifts, and thus has
implications for conservation and game management efforts, climate change
mitigation, and agriculture. &lt;br/&gt;&lt;br/&gt;Standing genetic variation is
the pool of available genetic variants in a population available for
evolutionary change. Recent research, facilitated by next-generation DNA
sequencing technologies and new analytical techniques, has revitalized the
debate over the relative influence of standing genetic variation versus new
mutation on adaptive evolution and speciation. The research funded by this grant
will use approaches based in coalescent theory, a mathematically-rigorous
approach at the interface of phylogenetics and population genetics, to
understand the prevalence and influence of standing variation in natural
populations of threespine stickleback fish. By combining restriction site-
associated DNA sequencing, a powerful method for genome-wide inference, with new
long-read sequencing capabilities, the proposed research will generate data-rich
genealogies at tens of thousands of loci and provide unprecedented detail into
adaptation of natural populations.