* 1515153
* RUI: Catalytic regulation of ribosome processing factors:  Investigation of peripheral domain effects on the enzymatic capabilities of the DEAD-box protein Rok1p.
* BIO,MCB
* 08/01/2015,07/31/2020
* Ivelitza Garcia, Allegheny College
* Standard Grant
* Wilson Francisco
* 07/31/2020
* USD 174,038.00

Title: RUI: Catalytic regulation of ribosome processing factors: Investigation
of peripheral domain effects on the enzymatic capabilities of the DEAD-box
protein Rok1p.&lt;br/&gt;&lt;br/&gt;Ribonucleic acids, RNAs, play a critical
role in a variety of cellular processes. In order for RNAS to perform their
specific functions in cells they often need to be processed by protein factors.
Protein processing is generally associated with structural changes that are very
important to the processing protein function. This research project will develop
chemical-based methodologies to identify and characterize the protein structural
elements that can regulate the activity and initial structural state of
essential RNA processing proteins. The second objective of this research is to
provide an effective training environment for students interested in RNA-protein
research. Undergraduate students will perform all investigations, including the
synthesis and purification of all RNAs and proteins utilized in this study. To
cultivate a rich learning environment, comprehensive multi-approach mentoring
and Just-in-Time teaching of experimental research will be used. This mentoring
infrastructure is designed to advance problem solving and critical thinking
skills in experimental design and analysis, and it has been successful in the
past. Thus, this project will not only further the understanding of protein
regulation, but also provide a strong foundation for students in the sciences.
&lt;br/&gt;&lt;br/&gt;DEAD-box proteins constitute a large sub-class of putative
unwindases essential in many RNA metabolic pathways. For example, Ribosomal RNA
(rRNA) processing and assembly are highly regulated pathways requiring several
DEAD-box proteins. This family of proteins couples cellular function to protein
structural changes. In a subset of DEAD-box proteins, activity is mediated
through additional peripheral elements/domains. The goal of this research is to
understand the structural and thermodynamic nature of peripheral domain effects
on the activity and conformational state of a model protein. Specifically, a
DEAD-box protein model will be biochemically characterized and comparatively
analyzed to evaluate the catalytic consequence of domain deletion variants.
Equilibrium and kinetic methodologies will be developed to study the effects of
initial conformations on nucleotide binding. Lastly, solvent-accessible and
dynamic regions will be identified for the model system. Since the deregulation
of the model protein and its human homolog has critical cellular consequences,
the proposed research will further the understanding of (1) how DEAD-box
proteins can bind early during processing and function in later events, and (2)
how DEAD-box proteins and other RNA folding and processing accessory factors can
utilize various conformations to regulate activity. Thus, the characterization
of these states is important for any protein that utilizes ATP hydrolysis within
a dynamic multi-factor complex.