* 1515163
* Ancient American tuberculosis: origin(s), spread, and replacement
* SBE,BCS
* 06/15/2015,05/31/2019
* Michael Rosenberg, Arizona State University
* Continuing Grant
* Rebecca Ferrell
* 05/31/2019
* USD 333,450.00

The exchange of pathogens between humans and other animals has been
longstanding, and recent examples include HIV, SARS, flu viruses, and Yersinia
pestis (which causes the plague). Today, especially in the developing world,
humans and their domesticated animals continue to encroach upon wild animal
habitats, with opportunities for cross-species transmission (or "spillover")
expanding as exposure increases. This project will provide insight into the
process by which pathogens adapt to new hosts and the impact of human migration
and interaction on the spread of pathogens in the past. In particular, the
project will use ancient DNA and skeletal analyses to investigate how human
migration events and possible changes in virulence of M. tuberculosis strains
have facilitated the spread of tuberculosis (TB) during human evolutionary
history, with specific hypotheses about how tuberculosis may have been
transmitted from animals to humans, and from the Old World to the Americas. A
better understanding of the evolutionary history of mycobacteria such as those
causing TB and leprosy can provide insights for other researchers interested in
developing clinical treatments. Because human TB (and its descendants) has had a
profound impact on other species, it is thus also a conservation concern. During
this project, both undergraduate and graduate students will be trained in
molecular techniques and data analysis. The data generated will be deposited in
public databases such as GenBank, and the results of data analyses will be
published in scholarly articles as well as in formats accessible to the general
public.&lt;br/&gt;&lt;br/&gt;This project will identify, document, and sample
285 remains with skeletal lesions characteristic of TB that date to before and
after European contact in the Americas. Newly developed methods will then be
used to extract DNA from these samples, as well as to target and sequence the
ancient pathogen genomes. We have assembled a team of experts in ancient DNA,
bioarchaeology, bioinformatics, and population genetics to address two specific
aims. The first is to examine the geographic patterning of pathogenic
mycobacteria, particularly M. tuberculosis complex (MTBC), in the Americas
through time. Specifically, we will test whether prehistoric TB in Peru and
South America were the result of a single jump of M. pinnipedii into humans, if
prehistoric TB in North America was caused by MTBC strains that are most closely
related to M. pinnipedii strains found in ancient South Americans, and whether
strains found in specimens with atypical skeletal pathologies from pre-contact
Mexico are M. lepromatosis (a newly discovered strain of mycobacteria that
causes atypical leprosy). The second aim is to discern signatures of adaptation
to humans by mycobacteria through time. For this aim, we will test the
hypotheses that 1) the zoonotic M. pinnipedii strain(s) that "jumped" into
humans in South America shows signs of selection that indicate adaptation to
this new host and 2) local mycobacterial strains in the Americas were replaced
by more virulent strains at contact over a relatively short time period (ex:
following contact/increased trade in the New World). The genome data obtained
from the ancient samples along with comparative data from modern strains will be
used to construct phylogenies of these pathogens, to assess patterns of
diversity across the Americas (and through time), and to test for signals of
selection. The pattern of adaptation of the MTBC and other mycobacteria in
humans over time is of interest since it helps understand the success of these
organisms and their possible future trajectories.