* 1516839
* Recruitment and Dynamics of the Mediator Complex
* BIO,MCB
* 08/01/2015,07/31/2019
* Randall Morse, Health Research Incorporated/New York State Department of Health
* Continuing Grant
* Karen Cone
* 07/31/2019
* USD 645,000.00

This project will provide new insights into molecular interactions that govern
how genes are turned on and off in living cells. The project focuses on a large
protein complex called Mediator, which selectively turns on subsets of genes in
both yeast and mammalian cells. Two central questions will be addressed. First,
how does Mediator select and turn on its gene targets? Second, how does Mediator
action in yeast compare to its action in mammalian cells? The comparative
approach used in this project will have broad impact on fundamental
understanding of gene activity in "simple" yeast versus "complex" mammalian
cells. Some of the results will be in the form of "big data"--large data sets
that describe molecular interactions across an entire genome--and these will be
deposited in public archives where they can be freely accessed by scientists and
the public. In addition, the project will provide training for undergraduates, a
postdoctoral fellow, and two female graduate students, one of whom is a member
of an underrepresented minority.&lt;br/&gt;&lt;br/&gt;The Mediator protein
complex consists of over twenty interacting proteins and is found in all cells
with nuclei (eukaryotes). Mediator is known to play a critical gene activating
role via association with the machinery responsible for RNA synthesis
(transcription). However, how Mediator locates its gene targets and exactly what
it does when it finds them, are not completely understood. This project will
address these issues in three aims. The first will test the idea that proteins
belonging to the general transcription machinery help Mediator to find its
targets. The approach will be to compare Mediator association with its targets
in normal yeast cells and in mutants in which components of the general
transcription machinery are impaired, using a method called ChIP-seq that
identifies where proteins associate with DNA across an entire genome. The second
aim will follow up on previous results that suggested that Mediator association
with different targets is related to differential dynamics--how fast Mediator
finds its targets and how often and rapidly it leaves again. This idea will be
tested using variations of chromatin immunoprecipitation (ChIP, the basis of
ChIP-seq) that provide information on binding dynamics. These first two aims
will be done using the model system of baker's yeast (Saccharomyces cerevisiae),
which is easy to work with but is quite similar to mammalian cells in its
molecular makeup. In the third aim, the effect of impairing function of specific
subunits of Mediator in mammalian cells on gene transcription will be tested by
"knocking down" expression of those subunits and measuring the effect on genome-
wide transcription by high throughput sequencing. In yeast, this has been a
productive strategy for understanding how Mediator structure relates to its
function. Comparing the results obtained in mammalian cells will reveal
similarities and differences with yeast, and thereby reveal the value and
limitations of knowledge gained using the yeast system.