* 1516999
* Higher-Order Structure Of Chromatin
* BIO,MCB
* 08/15/2015,07/31/2020
* Sergei Grigoryev, Pennsylvania State Univ University Park
* Continuing Grant
* Manju Hingorani
* 07/31/2020
* USD 710,000.00

This project addresses the molecular mechanisms that enable several meters of
DNA (the genome) to be packaged into a cell nucleus which has a diameter that is
approximately 100,000 times smaller than this. This amazing degree of compaction
of the DNA is achieved by repeatedly coiling of the DNA double helix around
multiple spools, each of which is composed of protein (histone) cores. The DNA
is thus formed into a "beads-on-a-string" array of DNA/histone particles called
nucleosomes. The nucleosome arrays are then further folded into structures that
include multiple loops where the DNA is physically constrained. Beyond a basic
understanding of this structure, the architectural organization of the DNA-
protein complex in the cell nucleus (known as chromatin), remains enigmatic.
This research will use biochemical experiments and computational 3-dimensionsal
modeling to describe the structural interactions of DNA in living cells and to
predict how the structure impacts function. The project will promote education
and training for graduate students and for undergraduates, including those from
underrepresented groups and underserved areas of rural Pennsylvania who
typically have little opportunity to participate in fundamental research.

In most current textbooks, the higher-order structure of DNA is modeled based on
zigzag or solenoidal folding of nucleosomes forming linear chromatin fibers.
However, recent in vivo studies suggested a dynamic looping of flexible and
disordered nucleosome arrays rather than linear fibers in most cell types. This
project is centered on a hypothesis that different lengths and conformations of
the linker DNA connecting nucleosome beads impose distinct topological states
and orders of nucleosome packing that segregate the chromatin domains into
either flexible nucleosome chains facilitating functional interactions between
distant DNA elements or, alternatively, tightly packed nucleosomes that inhibit
interactions between distant chromosomal sites. This hypothesis will be tested
using a unique electron microscopy nucleosome interaction capture method, DNA
topology assays, and computational modeling to determine patterns of nucleosome
interactions and DNA topology of both circular arrays of precisely positioned
nucleosomes and circular minichromosomes isolated from yeast cells. The results
will provide a more detailed picture of chromatin organization as it is likely
to exist in the cell, and this knowledge will serve as a framework for better
understanding of how chromatin features regulate gene expression.

This award is co-funded by programs in Genetics Mechanism (Division of Molecular
and Cellular Biosciences) and the Physics of Living Systems (Division of
Physics).