* 1517081
* Mechanistic Analysis of How a Plant DNA Methylase Acts on Chromatin
* BIO,MCB
* 07/15/2015,06/30/2018
* Geeta Narlikar, University of California-San Francisco
* Standard Grant
* Karen Cone
* 06/30/2018
* USD 700,000.00

This project is funded jointly by the Genetic Mechanisms Cluster in the Division
of Molecular and Cellular Biosciences and the Plant Genome Research Program in
the Division of Integrative Organismal Systems in the Directorate for Biological
Sciences.&lt;br/&gt;&lt;br/&gt;Cell identity is determined and maintained by
selective expression of a group of genes. This project will help uncover new
molecular mechanisms for how large regions of the inherited genome are turned
off or left unread, forming different types of cells in organisms like plants.
To do this, structures called heterochromatin must be created and manipulated.
Inside heterochromatin are structures called nucleosomes, which are packages of
DNA wrapped proteins that help organize the DNA strands. The mechanism of an
important protein in maize (corn), which may reduce the availability of DNA to
protein interactions responsible for reading DNA will be investigated. This
program will attempt to find answers to two questions: (1) how this protein
chemically changes the DNA located in nucleosomes, and (2) how this protein
alters nucleosome packing to create structures that can turn off or silence
certain genes. Multi-disciplinary training will be given to the participating
students in the form of combining a variety of biophysical approaches, as well
as genetic approaches. Active participation of under-represented minorities
through a partnership with the principle investigator's laboratory and the San
Francisco State University's Masters Program, and leading participation by women
graduate students in this program will encourage these students to apply bio-
regulatory concepts to real problems in a team setting. The results from this
research will also build towards a better understanding of how heterochromatin
based silencing mechanisms interfere in the formation of crops with pest-
resistance and enhanced growth and nutritional content.&lt;br/&gt;&lt;br/&gt;In
plants and mammals heterochromatin is defined by the presence of two post-
translational modifications, histone H3 lysine 9 methylation (H3K9me) and DNA
cytosine methylation as well as specific protein factors that recognize these
marks. In both contexts two properties of heterochromatin are considered to be
essential for its function: (i) the ability to spread to adjacent genomic
regions from the sites of initiation, which enables action across large
stretches of the genome and (ii) the ability to condense the underlying
chromatin, which is thought to inhibit DNA access and make the chromatin
refractory to transcription. However the biochemical basis for how DNA and
histone methylation enable repressive chromatin structures is poorly understood.
The major cytosine DNA methyltransferase, CMT3 from Arabidopsis thaliana and its
maize homolog ZMET2, provide an opportunity to tackle this fundamental question.
CMT3 and ZMET2 have two accessory domains that recognize the H3K9 methyl mark
and this recognition is essential for DNA methylation in vivo. This project will
test the following hypotheses: (i) ZMET2 bridges H3K9me nucleosomes and compacts
the underlying chromatin and (ii) ZMET2 activity is controlled by defined
chromatin architectures. The project combines mechanistic enzymology with
methods such as multi-angle light scattering, FRET and cutting edge electron
cryo-microscopy. The models derived from the work on ZMET2 will be tested in
vivo in the context of the related CMT3 enzyme in Arabidopsis thaliana. The
mechanistic dissection here is expected to: (i) elucidate the biophysical basis
for how DNA methyltransferases collaborate with histone methylation to generate
heterochromatin and (ii) uncover new regulatable steps in heterochromatin
formation.