* 1517701
* Genomic signatures of reproductive isolation in a primate hybrid system
* SBE,BCS
* 06/15/2015,05/31/2019
* Liliana Cortes-Ortiz, Regents of the University of Michigan - Ann Arbor
* Continuing Grant
* Rebecca Ferrell
* 05/31/2019
* USD 303,924.00

This project will study two closely-related species of howler monkey in order to
improve our understanding of the genetics of speciation in a primate model. When
two closely-related species reproduce with one another, they produce hybrid
individuals that have a mixture of genetic characteristics of both species.
However, there are likely to be certain genetic regions in those hybrids that
are not randomly mixed and that may play be important for local adaptation,
reproductive isolation, and speciation. Studying genetic mechanisms that
underlie reproductive isolation in a primate hybrid system can serve as a living
model to understand the mechanisms involved in shaping our own genome, in the
face of recent evidence of the occurrence of hybridization between modern and
archaic human populations during hominin evolution. This project will also
increase diversity in science by supporting the researchers and students who are
underrepresented minorities and by promoting the conservation of natural
populations of primates through educational activities for school-age children
in local communities. &lt;br/&gt;&lt;br/&gt;Studies of patterns of introgression
(the movement of alleles from one gene pool to another through hybridization)
provide an outstanding opportunity to improve our understanding of the number,
distribution and function of differentiated genomic regions during the process
of speciation. To explore the genetic mechanisms that underlie reproductive
isolation in primates, the proposed work will analyze pure and admixed
populations of two species of howler monkeys that hybridize in Mexico. Analyses
of genome-wide data will be used to first determine the number and distribution
of genomic regions that have higher than expected levels of divergence between
non-hybridizing populations of parental species. In addition, genomic cline
analyses will be performed to determine whether or not these divergent loci in
allopatric populations also show reduced levels of introgression in the hybrid
zone (as expected for loci involved in reproductive isolation). Levels of
linkage disequilibrium will be estimated for loci with reduced introgression and
the loci will be mapped to publicly available reference genomes to determine
whether they are clumped or scattered throughout the genome. Sequences will also
be annotated through comparison with the NCBI non-redundant protein and
nucleotide databases to determine the association of these regions to genic
regions with known functions and/or to regulatory regions. Finally, the
proportion of non-introgressing loci located in the X chromosome relative to
loci located in autosomal regions will be estimated to test the hypothesis that
loci involved in reproductive isolation have accumulated in the X chromosome.