* 1517617
* Establishing the link between biomolecular dynamics and function from an atomistic perspective
* BIO,MCB
* 07/15/2015,06/30/2020
* Donald Hamelberg, Georgia State University Research Foundation, Inc.
* Continuing Grant
* Wilson Francisco
* 06/30/2020
* USD 841,874.00

Regulation of biological processes in the cell relies on physical interactions
between biomolecules and on the resulting changes in the dynamics and
conformations (shape) of the biomolecules. These processes in turn regulate
relocation of biomolecules within different regions of the cell, speed up
biochemical reactions, turn on and off gene expression, and aid the correct
folding of other proteins. The relationship between three-dimensional structures
and function of biomolecules alone is insufficient to fully explain the
molecular basis of these processes. Biomolecules are not static but inherently
flexible, and the role of biomolecular motions in the regulation of biological
functions has remained unclear, despite an increasing number of experimental and
theoretical studies. The inability to separate out the contributions of
different factors to overall function is at the heart of the problem. The goal
of this research project is to establish the molecular basis of the motions of
biomolecules involved in biological processes. Computational tools that are
developed will be publicly available and implemented into widely used open
computational software for anyone to use and improve upon. The research will
provide an excellent opportunity to attract and train the next generation of
scientists. An advantage of the research is that it cuts across many
disciplines, including chemistry, biology, physics, mathematics, and computer
science, and, therefore, provides an excellent opportunity to attract students
into the sciences and scientific research. &lt;br/&gt;&lt;br/&gt;The
relationship between structure and function in biomolecules is well established,
however, this information is not always adequate to provide a complete
understanding of the mechanism of biomolecular processes. The relationship
between structure, dynamics and function in allostery and enzyme mechanisms is
therefore far from being fully understood. Critical barriers to progress are the
lack of fundamental understanding of the role of dynamical motions in
biomolecular function and the inability to experimentally describe temporal
behaviors of biomolecules at atomic resolution. The goal of this project is to
establish the molecular basis of dynamical and allosteric regulation in
biological systems involving cooperative substrate binding and transient
protein-protein interactions in complex biomolecular assemblies. Specifically,
the research uses theory and computer simulations as complementary tools to a
variety of experiments, including NMR, X-ray crystallography and Surface Plasmon
Resonance, to understand allosteric regulation in a multi-domain enzyme, Pin1,
and a transcription factor co-regulator Pirin. The main objectives are to
identify dynamical networks of residue-residue interactions that regulate
function, to describe their characteristics on a quantitative level, and to
explore computational approaches to access long time scale dynamics, overcoming
limitations of traditional direct simulation methods. Computational
investigation of the above systems in atomic detail will allow understanding of
how modulation in dynamics in one region is communicated via a dynamical network
of amino acid residues to a spatially distal region. The research will enhance
the incorporation of biomolecular dynamics in interpreting experimental results.
The research will provide the fundamental knowledge needed to establish the
relationship between dynamics, structure, and function in sub-cellular
processes. &lt;br/&gt;&lt;br/&gt;This project is jointly funded by the Molecular
Biophysics Cluster in the Division of Molecular and Cellular Biosciences in the
Directorate for Biological Sciences and the Chemical Theory, Models, and
Computational Methods Program in the Division of Chemistry in the Directorate of
Mathematical and Physical Sciences.