* 1518130
* Regulatory Signaling Logic In Self-Assembled Microbial Communities During Oscillating Environmental Conditions
* BIO,MCB
* 08/01/2015,07/31/2020
* Daniel Noguera, University of Wisconsin-Madison
* Standard Grant
* Anthony Garza
* 07/31/2020
* USD 695,892.00

Scientists using new techniques based on DNA sequencing are now able to study
how bacteria thrive in many different conditions. This is important because
bacteria are much simpler and easier to study than other life forms such as
plants and animals, and much of what is learned using bacteria can be applied to
other types of organisms. However, most research involves studies of bacteria
living in isolation in the laboratory. Very few organisms live in isolation in
nature, and therefore it is important to develop research methods to study
bacteria living in communities. The research will show how genes and other
component parts of the bacteria work together in order to sustain life and
especially how these interactions help bacteria survive in harsh and rapidly
changing situations. The results of this research will help predict how
organisms can adapt and survive when their surroundings change to less
hospitable conditions. The project is anticipated to have applications in many
different areas including the study of bacteria that live in and on the human
body, the design and control of pollution clean-up systems, and acquisition of
fundamental knowledge of how organisms in aquatic environments survive and
thrive.This project will provide interdisciplinary training to students
(undergraduate and graduate) and postdoctoral fellows in engineering,
microbiology, biotechnology and computational biology, in the context of both
academic research and industrial applications. &lt;br/&gt;&lt;br/&gt;A primary
goal of systems biology is to develop a quantitative understanding of how
microorganisms respond to changing conditions. Therefore, it is important to
understand the molecular and regulatory "wiring diagrams" that enable
unculturable organisms to thrive in dynamic environments. One such group of
unculturable organisms,the biotechnologically-relevant group of bacteria
(Accumulibacter), combines metabolic and regulatory processes in different ways
to yield novel phenotypes in response to environmental stimuli. Accumulibacter
is a well-studied yet uncultivated microbe. It must constantly adapt its global
physiological response to changing environmental stresses, namely biphasic
cycles of anaerobic "feast" and aerobic "famine" conditions. These oscillating
conditions select strongly for the Accumulibacter phenotype, which includes the
ability to sequester massive amounts of carbon and phosphate intracellularly in
different phases of the cycle. However, the regulatory and metabolic network
dynamics that coordinate such precise transitions remain poorly understood.
Employing a variety of -omics techniques and a systems biology framework within
the context of a tractable and self-assembled microbial community, studies will:
1) identify the metabolic modules responsible for Accumulibacter's unique
physiology via comparative genomics and transcriptomics; 2) map the regulons
that coordinate Accumulibacter's storage response via ChIP-seq targeting
candidate regulatory elements; and 3) identify environmental drivers of
Accumulibacter's regulatory program.