* 1534246
* DMREF: Collaborative Research:  Helical Protein Assemblies by Design
* MPS,DMR
* 09/01/2015,08/31/2020
* Gevorg Grigoryan, Dartmouth College
* Standard Grant
* John Schlueter
* 08/31/2020
* USD 493,102.00

NON-TECHNICAL SUMMARY Molecular self-assembly is a fundamental principle of
life, with cells having mastered this process to encode incredible diversity of
function. Helical protein assemblies organize much of the intracellular and
extracellular structure, and direct all movement. The ability to emulate such
functions by designing synthetic protein assemblies would transform modern
molecular science, with far-reaching applications including locomotion,
controlled release, directional transport, dynamic switching, and shape-
selective catalysis. However, structurally ordered supramolecular materials on
the nanometer length-scale are the most challenging to rationally construct and
the most difficult to structurally analyze. The size and structural complexity
of these extended protein assemblies present a significant challenge to current
computational design methods. The rules that govern protein-protein interactions
are more complex and difficult to reliably predict than for DNA. In this
project, a novel intellectual framework for the targeted design of synthetic
protein assemblies at atomic-level accuracy will be established, validated, and
made available to the research community. Enabled by the combined expertise of
the three investigators involved, this approach will merge significant advances
in modeling and computational design with never-before-possible experimental
techniques for structural determination of protein assemblies at the atomic
level. On the way to developing this framework, fundamental questions of acute
significance to biology, chemistry, and materials science will be addressed:
from development of an understanding of the functional roles of native
biological assemblies to construction of synthetic assemblies for technological
applications. Students (graduate and undergraduate), postdoctorals, and faculty
involved in this project will gain experience in a variety of computational,
synthetic, and analytical methods in research areas of fundamental technological
interest that will prepare them well for future scientific careers. An exchange
program between the three academic institutions (Emory University, University of
Virginia, and Dartmouth College) will be established that will permit students
and postdoctorals to become involved in the different aspects of this research
project.

TECHNICAL SUMMARY Helical protein assemblies in biological systems exhibit a
rich portfolio of structure and function; capturing these within simpler and
more tractable synthetic materials would amount to a major leap in molecular
science. Recent technological advances in genome sequencing, bioinformatic
analysis, near-atomic resolution cryo-EM structural determination, and
computational protein design, in combination with extant synthetic and
analytical methods, present an unprecedented opportunity to engineer novel
protein assemblies that emulate and improve upon their native counter-parts.
This project will employ protein designability, estimated on the basis of native
structural representation, as a mechanism to promote and control association
between folded protein motifs with an aim to create protein-based materials of
defined structure and function. Designability in the context of protein
engineering refers to robustness of a protein fold in sequence space. A proxy
for designability is the frequency of occurrence of a structural motif within
the Protein Data Bank (PDB). This approach will be employed to search for
designable interfaces between protomers within the protein structural databank.
The ultimate objective of the proposed research will be to define sequences
based on simple secondary or tertiary structural elements that are competent for
self-assembly into nano-scale materials with extended helical symmetry.
Computational methods will be employed to interrogate the protein structural
databank to identify designable interfaces within robust structural motifs.
Suitable candidate sequences will be computationally optimized and synthesized.
Proven biophysical methods will be employed initially to identify sequences with
promising self-assembly behavior. State-of-the-art high-resolution structural
analyses will be performed on these assemblies using Iterative Helical Real-
Space Reconstruction (IHRSR) from cryo-EM images. Dramatic improvements in
imaging hardware, reconstruction algorithms, and computational methods of
structural refinement have provided rapid access to near-atomic resolution
structures of native and synthetic helical assemblies. These analyses will
inform future rounds of computational modeling and design, thus establishing a
dynamic feedback loop between theory, synthesis, and advanced methods of
structural analysis.