* 1541539
* Collaborative Research: Filling the largest void of the fungal genealogy of life (the Pezizomycotina) and integrating symbiotic, environmental and physiological data layers
* BIO,DEB
* 01/01/2016,06/30/2022
* Louise Lewis, University of Connecticut
* Continuing Grant
* Christopher Schneider
* 06/30/2022
* USD 208,230.00

Plants and fungi are interdependently connected in nature, through interactions
that are beneficial to both (mutualism, and decomposition of dead plant material
by fungi) or detrimental to one partner (parasitism). Therefore, fungal and
plant biology cannot be fully understood without studying them within the
context of their symbiotic interactions. With the advent of new DNA sequencing
technologies, it is becoming clear that the 100,000 known species of fungi
represent only a tiny fraction of the estimated 5.1 million species of fungi
present in nature. Amazingly, the largest fraction of this unknown fungal
diversity seems to be hidden in healthy leaves of plants and lichens, as
endophytic and endolichenic fungi. Endophytes are known to protect plants from
pathogens, and enhance their resistance to abiotic stresses such as droughts and
high temperatures. Most of these hyperdiverse endophytic and endolichenic fungi
are classified within a group of filamentous Ascomycota fungi called the
Pezizomycotina. The main aims of this research are to reconstruct a phylogeny
(genealogical relationships) for the Pezizomycotina and link this evolutionary
framework with a rich and diverse set of organismal data. The research will be
complemented by diverse outreach activities, engagement of high school students
in research, training of junior scientists in five states, and development of
tools with potential applications for studying the evolution of species
interactions across the life sciences. Accelerating the discovery of these
hidden fungi also promises to provide a rich source of secondary compounds for
new pharmaceutical and biocontrol products.&lt;br/&gt;&lt;br/&gt;The objectives
of this study are to fill the largest phylogenetic void in the fungal genealogy
of life (the Pezizomycotina) and integrate diverse data layers with the emergent
tree as a means to understand the evolution, ecology, and physiology of fungal-
plant symbioses. Culture-dependent and independent methods will generate the
largest sample of endophytic and endolichenic fungi to date, targeting diverse
plants, lichens, macroalgae, and co-occurring microalgae and cyanobacteria in
terrestrial, aquatic, and marine environments, with a focus on biotically rich
but understudied bioclimatic zones. Efficient collecting and analytical
pipelines will enable rapid integration of phylogenetic data from these new
symbiotic fungi into the fungal genealogy of life, while novel biodiversity
informatics tools will integrate existing multi-locus and phylogenomic
frameworks with short sequence reads obtained with next-generation sequencing.
Resulting evolutionary trees, greatly enriched by systematic sampling of
currently unknown biodiversity, will be integrated with host and trophic spectra
in three data layers (symbio-phylogenetic, eco-phylogenetic, and physio-
phylogenetic), enabling for the first time novel insights from geographic data,
abiotic and biotic factors, and the physiological traits that enable mutualistic
interactions with plants on which most communities depend.