* 1541529
* GoLife: Collaborative Research: Bringing the diverse microbial clade Stramenopila + Alveolata + Rhizaria (SAR) into a modern genomic context
* BIO,DEB
* 09/01/2015,08/31/2022
* Charles Delwiche, University of Maryland, College Park
* Continuing Grant
* Maureen Kearney
* 08/31/2022
* USD 604,419.00

Most of the biological diversity in eukaryotes (organisms with distinct nuclei)
is in the form of microscopic species and, when compared to other eukaryotes
(plants, animals and fungi), is less well-studied in terms of morphological
characterisitics, evolutionary relationships among species, and genetic
variation. This relative lack of information on microbial eukaryotes not only
has consequences for our understanding of all biodiversity on Earth, but also
how we interpret cellular and evolutionary biology in the broadest sense. This
research will advance our knowledge of a major evolutionary group (clade) of
microbial eukaryotes that comprise the Stramenopila, Alveolata and Rhizaria,
(SAR). SAR is a major clade of diverse microscopic eukaryotes that was recently
identified by evolutionary analyses and additional data have robustly supported
SAR as an independent evolutionary lineage. This discovery is forcing a re-
evaluation of the evolution of several eukaryotic traits, most notably
photosynthesis. Within SAR there are many diverse lineages but genetic data are
rare and concentrated in only a few lineages: Apicomplexa (e.g. malarial
parasites), omycetes (e.g. parasite water molds) and diatoms (e.g. ecologically
important phytoplankton). This project proposes to increase the number of SAR
species with genetic data as well as massively increase the amount of genetic
data collected per species. Organisms used for genomic data will be imaged and
will be integrated to the Encyclopedia of Life (EOL) to provide a comprehensive
resource on microbial eukaryotic diversity.&lt;br/&gt;&lt;br/&gt;Despite their
global ecological importance, fewer than 50% of all SAR clades are represented
by even a single genome in public databases. The work proposed here would add at
least 250 novel genomic-scale datasets (transcriptomes, draft genomes, single-
cell amplified genomes), focused primarily on capturing diversity within SAR.
These datasets will be made publicly available, substantially increasing our
understanding of the evolution of these organisms. The application of both
established phylogenomic and emerging similarity network methods will enable a
multi-layered analysis that will provide new information for evolutionary
analyses of these organisms. This will include: 1) diversity discovery using
targeted environmental 18S surveys coupled with high-throughput FlowCam imaging;
2) high-throughput transcriptomic sequencing; and 3) single cell genomics of
unculturable taxa. Phylogenetic analyses will capture the genetic mosaics that
underlie the early evolution within SAR. Beyond the increase in genome scale
resources from diverse members of the SAR clade, the proposed work will generate
image data layers and integrate these with other images from the literature in
EOL. Undergraduates, graduate students and postdoctoral fellows will be trained
in cutting edge techniques and will provide a workshop for teacher training in
microscopy. This project will increase diversity of participants by recruiting
students from underrepresented groups and develop of a high school curriculum on
the microbial tree of life.