* 1542703
* EAGER:The Evolutionary Impact of Methylation on Grass Genomes and Gene Expression
* BIO,IOS
* 07/01/2015,06/30/2019
* Brandon Gaut, University of California-Irvine
* Standard Grant
* Gerald Schoenknecht
* 06/30/2019
* USD 296,666.00

To meet the food needs of future generations in a sustainable manner, an
understanding of crop plant growth and development is essential. Information
about growth and development is encoded in a plant's DNA, which has four
fundamental chemical components, often referred to as A, C, G and T. However,
the C can also be modified by a process called methylation, which allows it to
function as a 5th information component in DNA. While it is apparent that
methylated Cs play an important role in plant growth and development, their role
has yet to be fully understood. In this proposal, the number and pattern of
methylated Cs among different crop plants will be compared, including barley,
einkorn wheat and sorghum. These comparisons will be used to investigate the
influence of methylated Cs on the function of genes. The overall goal is to
better understand the contribution of methylated Cs to plant growth and
development, so that eventually this information can be used for crop
improvement.&lt;br/&gt;&lt;br/&gt;DNA methylation is one of several epigenetic
modifications that are crucial to plant genome function. Recent work has also
shown that methylation has broad consequences for genome evolution, including
the composition of intergenic DNA, as a fundamental evolutionary property of
specific genes, and as a driver of compositional biases among genomic regions.
Given the importance of epigenetic mechanisms, a thorough understanding of the
evolution and function of plant genomes requires characterization of epigenetic
variation among plant species. Yet, such a characterization is presently
lacking. This project will generate and analyze whole genome bisulfite
sequencing (BSseq) and transcript (RNAseq) data for an array of grass species.
The data will be released to the Short Read Archive and Gene Expression Omnibus
databases prior to publication. The goal is to use the data to better understand
the relationship among DNA methylation, gene evolution and gene function, as
measured by gene expression. To achieve this end, a series of unique
evolutionary analyses will be developed that will address three specific
hypotheses: 1) that gene-body methylation is a conserved feature of functionally
important genes and maintained over evolutionary time by natural selection; 2)
that evolutionary shifts in gene body methylation (and/or the methylation of
flanking regions) drive shifts in gene expression between species; and 3) that
DNA methylation variation among tissues - specifically in genes and flanking
regions - contributes to tissue-specific gene expression.