* 1543963
* BREAD: Advancing the Cowpea Genome for Food Security
* BIO,IOS
* 04/01/2016,03/31/2021
* Philip Roberts, University of California-Riverside
* Continuing Grant
* Diane Jofuku Okamuro
* 03/31/2021
* USD 1,585,814.00

Cowpea, also known as black-eyed pea in the U.S.A., is a protein-rich grain-
producing plant native to Africa. When cowpea is combined with starchy grains
such as corn, sorghum or rice, the result is a complete source of major
nutrients for energy and growth. It also provides protein-rich leaves and is
used for animal feed. Cowpea is the primary source of protein for people in sub-
Saharan Africa (SSA) where it is presently produced on over 20 million acres
mainly by farmers with small land holdings. Its popularity is also due to its
resilience to low rainfall and heat while also enriching the soil. Despite its
relevance to food security in Africa and elsewhere, there has been relatively
little basic research on cowpea. Advances in next generation sequencing
technology have made it possible to "read" the genetic material of cowpea and
decipher the instructions (genes and genetic variation) that ultimately provide
for crop improvement. Producing an optimized reference genome sequence for
cowpea is one of the main goals of this project, a feat that has been
accomplished for human, mouse, cow and economically important crop plants such
as corn, soybean, rice, bean and poplar. A broader purpose is to provide genetic
resources to improve decision-making abilities within an existing network of
African cowpea breeders. Breeders develop new varieties by making crosses
between parents that carry favorable traits and selecting progeny containing
favorable characteristics of both parents. This project will provide new
materials for progeny selection in addition to knowledge that will facilitate
the development of new cowpea varieties for increased food
production.&lt;br/&gt; &lt;br/&gt;Cowpea (Vigna unguiculata) is in the family
Fabaceae, tribe Phaseoleae which includes other warm season legumes such as
soybean, common bean and pigeon pea, as well as Vigna species adzuki bean and
mung bean. Cowpea is diploid with 2n=22 chromosomes and a genome size of 620 Mb.
Unlike other legumes, cowpea lacks a published reference genome. A highly
fragmented draft genome sequence assembly has been available from previous work
since 2011 (harvest-blast.org; harvest-web.org) and is derived from 60x short-
reads combined with BAC-end and gene survey Sanger sequences. This project will
elevate the sequence of cowpea to state-of-the art by: (1) adding long-read
sequences and an optical map; (2) anchoring contigs to a recently developed
genetic map containing 37,372 SNPs; and (3) improving browser access for the
broader legume community. This will enable deeper utilization of cowpea
germplasm for gene discovery and marker development, while providing a stronger
foundation for the cowpea genetics and breeding community. Genetic intercrosses
will be conducted to produce new pre-breeding lines that will theoretically
carry optimal combinations of traits present within elite lines using a recently
developed multi-parent advanced generation intercross (MAGIC) population based
on eight parental lines. Marker-trait information, in the context of the genome
sequence and haplotypes, will be disseminated for broad adoption of marker-
informed breeding. This work is tightly linked to leading breeding programs in
Burkina Faso, Ghana, Mozambique, Nigeria and Senegal through USAID Feed the
Future and CGIAR Tropical Legumes projects.