* 1543412
* Methylmercury in Antarctic Krill Microbiomes.
* GEO,OPP
* 08/15/2017,11/30/2018
* Tamar Barkay, Rutgers University New Brunswick
* Standard Grant
* Jennifer Burns
* 11/30/2018
* USD 49,572.00

Marine food webs can concentrate monomethylmercury (MMHg), a neurotoxin in
mammals, in upper trophic level consumers. Despite their remoteness, coastal
Antarctic marine ecosystems accumulate&lt;br/&gt;and biomagnify MMHg to levels
observed at lower latitudes and in the Arctic. Marine sediments and other anoxic
habitats in the oceans are typical areas where methylation of mercury occurs and
these are likely places where MMHg is being produced. Krill, and more
specifically their digestive tracts, may be a previously unaccounted for site
where the production of MMHg may be occurring in the Antarctic. If
monomethylmercury production is occurring in krill, current views regarding
bioaccumulation in the food web and processes leading to the production and
accumulation of mercury in the Antarctic Ocean could be better informed, if not
transformed. This project will conduct a preliminary assessment of the krill gut
microbiomes, the microbiome's genomic content and potential for production of
monomethyl mercury by detecting the genes involved in mercury transformations.
By analyzing the krill gut microbiome, the project will provide insights
regarding animal-microbe interactions and their potential role in globally
important biogeochemical cycles. &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;This project
will conduct a preliminary assessment of the krill gut microbiomes, the
microbiomes genomic content and potential for production of monomethylmercury.
The diversity and metabolic profiles of microorganisms in krill digestive tracts
will be evaluated using massively parallel Illumina DNA sequencing technology to
produce 16S rRNA gene libraries and assembled whole metagenomes. The project
will also quantify the abundance and expression of Hg methylation genes, hgcAB,
and identify their taxonomic affiliations in the microbiome communities.
Environmental metagenomes, 16S rRNA gene inventories produced from this project
will provide the polar science community with valuable databases and
experimental tools with which to examine coastal Antarctic microbial ecology and
biogeochemistry. The project will seek to provide a wider window into the
diversity of extremophile microbial communities and the identification of
potentially unique and useful bioactive compounds. In addition to public
education and outreach. This project will train graduate students and provide
educational and outreach opportunities at the participating institutions