* 1545277
* Collaborative Research: Statistical Methodology for Network based Integrative Analysis of Omics Data
* MPS,DMS
* 01/01/2015,06/30/2017
* George Michailidis, University of Florida
* Standard Grant
* Junping Wang
* 06/30/2017
* USD 315,463.00

The overarching goal of this project is to delineate pathways-based on
coordinated activity of genes, transcripts, proteins and metabolites, that could
potentially serve as therapeutic targets, as well as create Omics based
biomarker panels for early detection and prognosis of disease. Specifically, we
address the problem of integration and analysis of multiple sources of high
dimensional biological data with network structure. It is a well documented fact
that correlation between different molecular compartments is relatively low,
while the information derived from a single compartment is often highly noisy or
even incomplete. Hence, there is a need to develop advanced models and
techniques for integrating multiple data sets from diverse Omics platforms,
while taking explicitly into consideration the available information of
interactions within and between compartments. Particular emphasis is placed on
pathway analysis and enrichment due to their role in complex diseases onset and
progression. The following directions will be pursued: (1) Development of
network based methods that integrate data from multiple Omics platforms for
pathway analysis and enrichment. (2) Development of fast computational
algorithms for estimating large scale network based integrative models.
Investigation of associated inference problems and study of properties of
proposed estimators together with their robustness to the noise levels of the
network information employed. (3) Introduction of novel hypergraph models for
assessing differential activity of pathways that utilize different degrees of
information about the structure and accuracy of the underlying network. (4)
Development of a novel scheme based on perturbed P-values for detecting active
members of pathways that would aid in biomarker discovery. (5) Implementation of
the propose methodology into an easy to use software
tool.&lt;br/&gt;&lt;br/&gt;The proposed research program will have a three-
pronged impact: methodological, scientific and educational. On the methodology
front, the research based on this project will lead (a) to a developing a
comprehensive framework for assessing differential activity of pathways based on
different models that integrate data from multiple Omics platforms and utilize
different degrees of information about the structure and accuracy of the
underlying network, (b) a systematicunderstanding of the computational issues
involved in large scale (generalized) mixed linear models;and (c) a novel scheme
based on perturbed P-values for identifying active members of pathways that
become potential targets for therapeutic drugs. The enhanced scientific
understanding will provide tangible impact at the level of applications. A
number of the proposed methods have already been used in the analysis of high
dimensional genomic, proteomic and metabolomic data with emphasis on identifying
active pathways (subnetworks) in different disease (primarily cancer) states.
Further, a number of new experiments are in the design stage that would utilize
some of the advanced models and techniques proposed in this project. Another key
aspect of this proposal is the development of an easy to use by practitioners
open source software, built within a domain independent workflow management
system. This allows users to enhance the software by adding their own
functionality and computational tools in an easy and transparent manner. The
novel methodological procedures ensuing from this research agenda will be
disseminated to the relevant scientific communities, both via inter-disciplinary
interaction and collaboration and through presentations at conferences and
specialized workshops. Finally, on the educational front, the material from the
project will provide research topics for doctoral students working under the
supervision of the PIs; it will therefore play an important role in the training
of future quantitative scientists.