* 1545805
* Collaborative Research: Rational Design of Anticancer Drug Combinations using Dynamic Multidimensional Theory
* MPS,PHY
* 06/15/2016,05/31/2020
* Raul Rabadan, Columbia University
* Continuing Grant
* Krastan Blagoev
* 05/31/2020
* USD 240,480.00

This award is part of the NSF effort to promote significant advances in the
fundamental understanding of cancer biology made possible through
multidisciplinary research that involves experts in theoretical physics, applied
mathematics, and computer science.&lt;br/&gt;&lt;br/&gt;Achieving durable
control of metastatic solid tumors will require high-order targeted therapeutic
combinations, because single-agent therapeutics eventually become thwarted by
the development of tumor drug resistance. However, design of combinatorial
regimens cannot be done by empirical trial and error in the clinical setting.
The goal of the project is to blend a systems biology network-based theoretical
framework with an integrated experimental and analytical program in order to
address the combinatorial regimen challenge in oncology. Based on areas of
exemplary clinical need, investigator expertise, and the availability of
patient-derived tumor tissue, the project will focus on BRAF-mutant melanoma and
PIK3CA-mutant, estrogen receptor positive (ER+) breast cancer as initial tumor
types in which to pilot the approach. In addition the project will offer
interdisciplinary training and research experience to postdoctoral and clinical
fellows, graduate students, and indirectly to all members of the groups who
participate. Professional development of all trainees will be enhanced by yearly
meetings of the whole project team which will include tutorials on modeling and
experimental methodologies. A symposium on the quantitative science of cancer
will be organized at the Dana Farber Cancer Institute during the third year of
this project. Team members are also committed to broadening the participation of
women and under-represented minorities in STEM fields by pro-active recruitment
and mentoring.&lt;br/&gt;&lt;br/&gt;The project will integrate dynamic modeling
of signal transduction pathways relevant to cell proliferation and apoptosis,
genomic and evolutionary analyses of tumor cells, and systematic cell death and
therapeutic resistance studies. The dynamic models will be informed, tested, and
iterated using experimental approaches applied to relevant cancer model systems.
The experiments leverage emerging technologies such as pooled genome-wide open
reading frame screens, dynamic BH3 profiling of cancer cells' closeness to the
apoptotic threshold, whole exome sequencing and single cell RNA-seq analysis.
The models will recapitulate steady state signaling network activation, acute
adaptive effects of treatment (e.g., feedback dysregulation) and the range of
drug-resistant states that may emerge following longer-term drug exposure. Tumor
cell heterogeneity will be represented by the implementation of different
initial configurations or state overrides of network components. Using newly
developed systems control methodologies, the models will be used to prioritize
drug combinations and dosing/scheduling principles for in vitro and in vivo
testing. The final result will be a theoretical and experimentally validated
approach that can be generalized across many other cancer types. This project
develops a new framework to address cancer as a deregulated complex dynamical
system and it will lead to an improved understanding of adaptive and acquired
drug resistance mechanisms. The project will make a significant contribution
toward a major goal of cancer precision medicine, namely the identification of
optimal high-order combinations for individual cancer patients. The project will
also establish new connections between evolutionary theory and dynamical systems
theory. The theoretical and methodological advances will be applicable or
adaptable to other cancers and diseases in general, leading to potentially
transformative impacts on human health. &lt;br/&gt;&lt;br/&gt;This proposal is
cofunded by the Physics of Living Systems Program in the Physics Division and
the Systems and Synthetic Biology Program in the Molecular and Cellular
Biosciences Division.