* 1546899
* RESEARCH-PGR: Discovery and Evaluation of Inbred-specific and Hybrid-specific Regulatory Modules
* BIO,IOS
* 09/01/2016,08/31/2021
* Steven Briggs, University of California-San Diego
* Continuing Grant
* Diane Jofuku Okamuro
* 08/31/2021
* USD 2,913,922.00

Hybrid vigor in agriculture is defined as the increase in yield of an offspring
over those of its inbred parents. Hybrid vigor is a critical component of the
high productivity of many crops, including maize. While hybrid vigor has been
efficiently employed in many crops, we lack a basic understanding of how it
works. A better understanding could provide novel avenues for the improvement of
hybrid vigor, and for extending its benefits to non-hybrid crops. This project
will develop new approaches for discovering sets of interacting genes from the
inbred parents. Knowing the identity of those genes will make it possible to use
them as predictive tools and as targets of manipulation for crop improvement.
The project will provide a model for using predictive gene network-based
approaches to understand and improve complex traits in crops. Interest in using
biological networks for crop research is growing, but access to the necessary
technical capabilities remains limited. Therefore, the project will also provide
plant scientists with free technical and analytical services with an emphasis on
crop genomics-enabled research. These services will range from consultation on
experimental design to data interpretation, ensuring that results are useful and
capacity is not wasted. Training investigators who wish to become plant
biological network experts will be the project's highest priority. Two to three
undergraduate students will provide the services. These students will be trained
in chemistry, biochemistry, computer science, and engineering so that they can
support users in sample preparation, data generation, and network
analysis.&lt;br/&gt;&lt;br/&gt;This project will create unsupervised gene
regulatory networks (GRNs) and protein kinase networks (PKNs) for two maize
inbreds (B73, Mo17) and their hybrid (SX19). GRN modules are comprised of a
transcription factor (TF) and its target genes. PKN modules are comprised of a
protein kinase and its substrates. The GRN s will be made using three different
proxies for TF activity: TF mRNA abundance, TF protein abundance, and level of
TF protein phosphorylation; network targets will be measured by the levels of
all mRNAs. The PKN will be made using phosphorylation of the activation-loop as
a proxy for kinase activity; network targets will be measured by protein
phosphorylation levels of all proteins. All measures will be made on 26
different tissues. GRNs will be constructed using the GENIE3 random forest
algorithm. The PKN will be made using our previously described correlative
method. We will measure the preservation and divergence of modules between the
inbreds and their hybrid. We will test whether inbred-specific or hybrid-
specific modules contribute to heterosis. Selected nearÂ­ isogenic lines (NILs)
derived from SX19 with introgressed segments containing module regulators will
be test-crossed to the donor parent. The Fl will be exactly like SX19 except for
the segment containing the regulator, which will be homozygous. Comparison of
the transcriptome and proteome of SX19 to the test-crossed NIL will reveal
whether the regulator is acting on its targets as predicted by the network. If
the module contributes significantly to heterosis then hybrid vigor will be
reduced in the test-crossed NIL.