* 1546781
* RESEARCH-PGR: Transcriptional Control of the Maize Genome
* BIO,IOS
* 08/15/2016,07/31/2022
* Jay Hollick, Ohio State University
* Standard Grant
* Gerald Schoenknecht
* 07/31/2022
* USD 993,904.00

This project defines the passages of information encoded in the maize genome
thus enabling the selection, deciphering, and editing, essential to current and
future corn improvement strategies. A novel technique is used to delimit the
beginning and ends of genome sequences that cellular machines read and
transcribe into transient instructions. Three types of these machines transcribe
fundamentally different passages in all organisms, but plants have two
additional types that transcribe enigmatic regions referred to as "dark matter".
In some manner, these "dark matter" instructions impact traits of economic
interest by helping guide the coordinate reading of the genetic blueprint
necessary for proper growth and optimal health. Although these "dark matter"
regions represent the single largest source of genetic variation in worldwide
plant varieties, their functions remains poorly understood. This project aims to
decipher some of these functions by evaluating the effects of interfering with
the transcription machinery dedicated to these "dark matter" regions in diverse
maize varieties. The publicly available information, analytical tools, and
materials generated by this project will significantly elevate the utility of
existing resources and provide an essential reference for novel discoveries in
plant biology and genetics. The activities will integrate the training of young
scientists at both undergraduate and graduate levels and, in collaboration with
the American Chemical Society, provide research experiences for high school
students of economic disadvantage. Additionally, educational materials for
teaching both basic and advanced genetic concepts will be generated and made
available through an existing outreach program of the Arabidopsis Biological
Resource Center.&lt;br/&gt;&lt;br/&gt;Gene expression is controlled at
transcriptional and post-transcriptional levels; yet the contribution of each to
RNA abundance is often unknown. In plants, heritable changes to these controls
can be impacted by action(s) of Pol II-related RNA polymerase (RNAP) complexes.
At least five of these RNAPs are found in grasses but their functional
significance(s) remains unclear. This project defines the nascent
transcriptional landscape of maize, identifies potential cases of co- or post-
transcriptional control, and uses mutant analyses to understand how these Pol
II-related RNAP complexes affect gene expression. Specifically, this project
will 1) optimize protocols for global run-on sequencing and generate reference
nascent transcriptome datasets to enable novel genome annotations; 2) use
specific mutants and comparative RNA profiling to differentiate gene regulation
due to RNA-directed DNA methylation versus RNAP IV competitions and to allow
individual RNA stabilities to be inferred; 3) identify features where regulation
is heritably altered in the absence of RNAP IV to catalog epialleles whose
transcriptional behaviors resemble those susceptible to paramutation; and 4) use
mutant analyses to evaluate the hypothesis that RNAP IV translates environmental
perception into changes in heritable transcriptional control. Project outcomes
will provide important community resources to understand the nuclear systems
that generate and maintain epigenome diversity in a crop species. All data
generated from this project will be made immediately available, integrated into
existing database frameworks, and promoted through training and outreach
opportunities to empower a greater understanding of eukaryotic genetics and the
genome biology of crop species.