* 1546625
* RESEARCH-PGR: Leveraging Natural Variation in Tomato to Identify, Characterize, and Deploy New Sources of Disease Resistance
* BIO,IOS
* 09/15/2016,08/31/2022
* Samuel Hutton, Boyce Thompson Institute Plant Research
* Standard Grant
* Gerald Schoenknecht
* 08/31/2022
* USD 4,278,165.00

PI: Gregory Martin (Boyce Thompson Institute for Plant
Research)&lt;br/&gt;&lt;br/&gt;CoPIs: Susan Strickler and Zhangjun Fei (Boyce
Thompson Institute for Plant Research); Alan Collmer and Magdalen Lindeberg
(Cornell University); Samuel Hutton (University of Florida); and, Dilip Panthee
(North Carolina State University)&lt;br/&gt;&lt;br/&gt;Nearly 30 years of
intensive research on the molecular basis of the plant immune system and
pathogen infection processes has generated a broad understanding of their
underlying mechanisms in certain model plant species. The field is now
positioned to use this knowledge to explore the natural genetic variation that
contributes to plant immunity and to apply it to crop improvement. This project
takes advantage of the interaction of tomato with a pathogen that causes
bacterial speck, an economically important disease that decreases the
marketability and yield of fresh-market and processing tomatoes. This
interaction is a powerful system for understanding disease resistance because
both the plant and the pathogen are experimentally tractable. In addition, a
vast source of natural genetic variation exists in wild relatives of tomato, all
of which can be crossed to cultivated tomato. The goal of this project is to
leverage this natural variation to identify genes that play a role in plant
immunity, characterize these genes using molecular methods, and introduce them
into fresh-market breeding lines that are important parents in two public tomato
improvement programs. An important aspect of the project is the improvement and
promotion of VEGEVADERS, a game to entertain and teach high school students and
the broader public about the science and importance of plant-pathogen
interactions. The enhancement of disease resistance in fresh-market and
processing tomatoes could lead to decreased use of pesticides thus saving
growers money, reducing the impact of disease control on the environment and
providing food for consumers with fewer pesticide residues.
&lt;br/&gt;&lt;br/&gt;Diseases of crop plants have major economic and
environmental impacts because they decrease yields and require extensive
pesticide applications. The overarching goal of this project is to take
advantage of the natural variation present in tomato and its wild relatives
along with the extensive genome sequence data available to discover new
genes/loci that contribute to pattern-triggered immunity (PTI). The molecular
functions of these genes will be investigated by using CRISPR-mediated mutations
and other approaches and selected genes/loci will be introgressed into two
foundational fresh-market tomato breeding lines. Specifically, the objectives of
the project are to: 1) screen 200 tomato heirlooms, breeding lines, and wild
species accessions for natural variation for PTI using pathogen elicitors and
engineered P. syringae strains with different effector repertoires; 2) identify
novel PTI-associated loci/genes and develop DNA markers for use in two tomato
breeding programs; 3) develop 150 tomato lines with CRISPR-mediated alterations
in candidate immunity-associated genes and characterize them for PTI responses;
4) introduce novel sources of disease resistance into two fresh-market tomato
breeding lines and investigate the molecular functions of PTI-associated
loci/genes; and 5) enhance VEGEVADERS, a game based on plant-microbe
interactions, expand functional genomics web resources, develop a plant breeding
workshop, and promote educational outreach. Undergraduates, graduate students,
and postdocs will be trained in bioinformatics, plant breeding, plant pathology,
and functional genomics. Knowledge generated in the project will both enhance
the understanding of the plant immune system and lead to tomato varieties with
improved disease resistance. All data and resources will be accessible to the
public upon request and through long-term data and germplasm repositories.