* 1546049
* EAGER: Developing a High Resolution Method for Mapping Regulatory QTLs
* BIO,DEB
* 03/01/2016,02/29/2020
* Robert Reed, Cornell University
* Continuing Grant
* Leslie J. Rissler
* 02/29/2020
* USD 299,995.00

A major of focus of many research groups is to characterize specific genomic
elements that control variation in quantitative traits (traits that are
determined by many genes and their interactions with the environment). Some
trait variation can be easily mapped to particular genes due to clear
associations with variation in encoded proteins. Much trait variation, however,
is difficult to characterize with precision because it is caused by differences
in DNA sequences that control gene expression. Variation in these so-called
regulatory elements has traditionally been much more difficult to characterize
than variation in protein-coding regions for technical reasons. The purpose of
this project is to develop new methods and computational resources to allow
biologists to identify regulatory elements that control quantitative variation
in natural and emerging model systems. The development of these methods will
transform the manner in which biologists link phenotype with genotype and thus
how they study the functional basis of evolutionary change. The methods will be
broadly used and likely impact fields as disparate as agriculture and
bioengineering.

Nucleotide polymorphism association studies, sometimes coupled with transcript
abundance data, have become a favorite approach for identifying genomic loci
underlying trait variation. This commonly adopted approach, however, often fails
to characterize specific causal genomic elements. Instead, it often settles for
neutral markers indicative of selection across regions spanning tens to hundreds
of thousands of nucleotides. In some cases this may be sufficient to identify
causal genes, especially when association intervals contain obvious changes in
protein coding regions. In many cases, however, it is difficult to improve
resolution because important functional changes in regulatory elements are
difficult to identify. For this project the investigators are working to develop
a novel, high-resolution approach to precisely map cis-regulatory quantitative
trait loci (creQTLs). Specifically, they aim to characterize creQTLs by
combining standard genotyping analysis with assays for chromatin structure and
active transcription. This approach would allow association studies to consider
both regulatory activity and nucleotide sequences, and could lead to resolution
of <150bp for QTLs mapping to regulatory elements. The overarching goal of this
project is to optimize this approach for use in emerging model systems and to
develop bioinformatic tools for annotation and statistical inference of creQTLs,
with an emphasis on utilizing readily available draft-quality genomes.