* 1546837
* RESEARCH-PGR: Dissecting the Genomic Architecture of Functional Redundancy to Modulate Meristem Homeostasis and Crop Yields
* BIO,IOS
* 09/01/2016,08/31/2021
* David Jackson, Cold Spring Harbor Laboratory
* Continuing Grant
* Gerald Schoenknecht
* 08/31/2021
* USD 4,616,755.00

Plant genome research over the past 20 years has provided a deep understanding
of genetic pathways that underlie economically important processes in crop
plants. However, as in most organisms, many plant genes have "backup" copies, or
duplicates representing genetic redundancy. Very little is known about the
effect of such redundancy on plant improvement efforts. This lack of knowledge
complicates the efficient use of genetic resources. This project will focus on a
known group of signaling genes to understand the basic principles that underlie
genetic redundancy in plants. It will therefore advance knowledge in a
fundamental area of plant genome biology. Outcomes from this project will have
the potential to bring improvements to US agriculture by providing new knowledge
and tools to develop high yielding crops. The project will also train a number
of young scientists at various levels, as well as promote outreach and education
in plant genomics. Project personnel will develop new teaching modules to
highlight the importance of plant genomics in crop domestication, and will
present these in schools and workshops that target female students and
underrepresented minorities. Outreach activities will also target rural farming
communities in the New York, Massachusetts and North Carolina areas, where open
house displays and lab visits will be used to educate these groups about the
importance of plant genomics research in agriculture.

This project asks how redundancy in signaling pathways has evolved across the
plant kingdom. It will develop a genome-level understanding to link genes and
pathways to complex phenotypes, by testing the hypothesis that genetic
redundancy in plants is controlled by Responsive Backup Circuits (RBCs). A
second hypothesis to be tested is that signaling network outputs can be
modulated and exploited using weak promoter alleles. Three species will be used,
the model system Arabidopsis, to rapidly test hypotheses, and tomato and maize,
divergent and economically important crop species. Genetic redundancy is a major
limitation to the ability to link genes to phenotypes in plants, and this
project will use a subset of Leucine Rich Repeat Receptor Like Kinases and their
predicted ligands as a model network. Signaling genes selected by phylogenetic
analysis will be targeted for knockouts using genome editing technologies
(CRISPR/Cas9). Genome-wide transcript profiling will then be used to deduce
redundancy mechanisms and reiteratively design new knockouts to address the
effect of disrupting redundant paralogs. At each stage, careful phenotyping will
be used to understand the effect of multiple gene knockouts at different
developmental stages relevant to crop productivity. Redundancy in gene
regulatory sequences (promoters) will also be addressed by developing a
generalizable CRISPR/Cas9 multiplex knockout strategy to make semi-random
mutations across gene regulatory sequence regions. These lines will be screened
en masse, and represent a new approach to mutagenesis in plants, with a
potential to generate new genetic diversity, and to recover weak alleles with
enhanced yield traits.