* 1546218
* RESEARCH-PGR: Systems Genomics of Rice Stress Adaptation
* BIO,IOS
* 09/01/2016,08/31/2023
* Georgina Vergara, New York University
* Continuing Grant
* Gerald Schoenknecht
* 08/31/2023
* USD 4,005,347.00

Adaptation of crop plants like rice to environmental stress is an essential
requirement to ensure high yields under those conditions. Salinity is among the
major environmental stresses that crop plants face, and it is currently
threatening rice cultivation due to increasing land pressure and climate change,
which are pushing agriculture to marginal lands. Rice is a major world crop
comprising two species: one from Asia (Oryza sativa), and one from Africa (O.
glaberrima). This project aims to identify genes associated with adaptation to
saline soils through comparative analyses of rice varieties that have
contrasting adaptation to salt stress, within and between the two species. The
discovery of more than one adaptive strategy will increase the options rice
breeders could implement in their programs. This project will use several
approaches to identify genes that have a uniquely coordinated pattern of
expression when the plant is exposed to elevated salt, and at the same time are
correlated with vigorous growth and productivity. This project will also
undertake public outreach programs using the New York City BioBus. This is a
transit bus equipped as a mobile educational laboratory that has been on the
road reaching 115,000 people at more than 420 schools and communities. Plant
biology laboratory modules will be developed for deployment across New York City
on the BioBus. These modules will primarily target schoolchildren in underserved
communities, and also the general public in street
fairs.&lt;br/&gt;&lt;br/&gt;Salinity is one of the growing stress challenges
crop plants face. In this systems genomics proposal, we will dissect the
adaptive response of O. sativa (Asian rice) and O. glaberrima (African rice) to
salt stress. First, we will infer the gene regulatory interaction network for
salt stress in Asian and African rice using time-series transcriptome and
chromatin accessibility data, coupled with state-of-the-art network inference
methods. Second, we will use phenotypic selection analysis on gene expression
levels to determine the strength and pattern of selection on stress response
genes. Finally, we will map expression genome-wide association (GWAS) loci
accompanying gene expression variation under salt stress, focusing on the genes
that have been identified in phenotypic selection analysis as undergoing
positive, stabilizing or disruptive selection. Moreover, we will also identity
GWAS loci for plant fitness and fitness-related traits under saline conditions
in both rice species, and integrate the results of these mapping analyses and
phenotypic selection analyses with evidence for selective sweeps in whole genome
sequence data from Asian and African rice. Our work will identify genes, genetic
networks and genomic variants that affect gene expression differences in key
loci in response to an environmental perturbation - salt stress -and explicitly
connect gene regulatory variation to plant fitness.