* 1547056
* Bilateral NSF/BIO-BBSRC: Signal encoding by transcription factor pulsing and its functional advantages
* BIO,MCB
* 08/15/2015,07/31/2019
* Michael Elowitz, California Institute of Technology
* Standard Grant
* Charles Cunningham
* 07/31/2019
* USD 480,000.00

Cellular signaling is fundamental to many biological processes, from the
development of embryos to the growth of cancerous cells. Signaling systems are
composed of a cluster of genes and their expression products, proteins, which
can interact with each other by controlling the expression of the corresponding
gene. Transcription factors are proteins that regulate the expression of a group
of genes within signaling systems and do so by binding to a specific DNA
sequence motif upstream of targeting genes. By measuring the amount of
transcription factors from a population of cells with traditional biochemistry
assays, biologists have shown that environmental inputs (i.e., stresses or
extracellular ligands) can alter the concentrations of transcription factors
that regulate gene expression. The objective of this project is to understand
how individual cells encode external inputs as time-based signals and how these
signals are transmitted to downstream genes, and to examine how the single cell
behavior differs from the behavior of the population of cells. The collaborative
team of investigators from the California Institute of Technology (US) and the
University of Edinburgh (UK) will use new single-cell measurements based on
fluorescent protein and time-lapse optical microscopy techniques can provide new
and distinct information about the signal processing capabilities of individual
cells. Through its broader impacts, this project will promote understanding of
the dynamic nature of cellular information processing. Furthermore, this project
will provide new techniques and tools to perturb and control cell
functions.&lt;br/&gt;&lt;br/&gt;Cellular signaling is fundamental to many
biological processes, from embryo development to disease progression. It is
often considered to be a continuous and concentration-based process, where
extracellular input is encoded in the concentrations of one or more
transcription factors (TFs) or other regulators, which are in turn decoded to
control gene expression in a continuous fashion. In contrast, recent single-cell
studies have revealed that many central TFs are activated in stochastic pulses
in response to inputs. Examples include nuclear factor-kappa-B and p53 in
mammals, and the general stress regulator Msn2 in yeast. In these systems,
inputs generate pulses of TF activity, which are then decoded by target
promoters in a dynamic fashion. The extent of signal encoding and decoding in
pulsing systems as well as the potential physiological functions of these
systems remain largely unexplored. This project will address these questions in
budding yeast through collaborative efforts between the Swain and the Elowitz
laboratories. The team will integrate single-cell time-lapse microscopy,
microfluidics, and techniques from information theory to measure the information
transfer within dynamic signaling systems by investigating the encoding and
decoding of TF dynamics in single cells. This project will contribute to the
understanding of three aspects of dynamic signaling: First, the team will
produce an extensive dataset of input- output relationship between stresses and
TF dynamics, providing a comprehensive understanding of dynamic cellular
responses. Second, information theoretical analysis of single-cell signaling
will provide a more comprehensive view of dynamics-based information
transduction and pulse-based signal encoding and decoding. Third, it will
explore the physiological roles of dynamic pulsing, to open up the possibility
of controlling cellular behaviors through the perturbation of TF dynamics.
Through its broader impacts, this analysis of pulse-based signaling in cells
should allow scientists to understand the design principles of natural signaling
systems.&lt;br/&gt;&lt;br/&gt;This collaborative US/UK project is supported by
the US National Science Foundation and the UK Biotechnology and Biological
Sciences Research Council.