* 1547796
* RESEARCH: Predicting Genotypic Variation in Growth and Yield under Abiotic Stress through Biophysical Process Modeling
* BIO,IOS
* 09/01/2016,08/31/2023
* C. Robertson McClung, University of Wyoming
* Continuing Grant
* Gerald Schoenknecht
* 08/31/2023
* USD 3,457,977.00

Rising demand for high quality food crops due to increasing world populations
along with more likely temperature and drought stress requires further crop
improvements from breeding programs. A major limitation to these programs is an
understanding of how the genetic information affects the characteristics of
plants that improve the amount of edible portions. Moreover, the predictive
understanding is even less if the plants are placed in new, stressful
environments like low rainfall or high temperature. A likely avenue of inquiry
to improve breeding is to better connect how information stored in genes becomes
traits of plants that combine their biology, such as photosynthetic rate or the
amount of resources allocated to an edible root, with the physical world, such
as the amount of water available or an excessive heat wave. These connections
are currently often made in a way that requires new data collection for every
new crop plant or environment such as a new soil, new temperature range or even
a new improved line that shows variation in the amount of resources the plant
allocates to an edible root. This continuous need for new information ultimately
slows down the breeding program and the ability of plant scientists to quickly
respond to the needs of society. This project will test a new approach that uses
large amounts of data to calculate the probability that a particular plant
characteristic will be displayed by a given plant line under various
environmental conditions. Specifically, the project will measure plant
performance continuously by sending electrical pulses through plants,
integrating the data generated with large data sets that show which genes are
active as well as the level of biologically relevant molecules that contribute
to major metabolic pathways within the plants at any given time. This new
approach requires high performance computing to test many times how the
probability of phenotypic improvement in the crop may occur. These high
performance computing approaches will become a core part of a modern,
competitive workforce. In this regard, the project will provide workshops for
high school teachers in the use of high performance yet open source computing
tools in their classrooms. In addition, the project will develop experimental
and computational modules in biological and quantitative learning for students
in grades 6-12 using the highly successful Wisconsin FastPlants system
(http://www.fastplants.org/). &lt;br/&gt;&lt;br/&gt;With increasing world
populations, genetic advances to improve crop growth, yield and resistance to
abiotic stress are a pressing need. Limiting the speed of crop improvement is a
crucial knowledge gap regarding biophysical processes that modulate the
relationship between the genome and phenome, hindering the ability to predict
the phenotype of novel genotypes in novel environments. As a first step towards
bridging this gap, a combination of high-throughput phenotyping and biophysical
process modeling will incorporate allelic variation at key genes affecting plant
carbon metabolism, hydraulics, and resource allocation, all of which are known
to impact drought- and heat-stress resistance in plants. Variable selective
pressures during crop diversification have caused extensive phenotypic variation
among B. rapa crops, making it an excellent study system to both connect organ-
level measures both down to the level of transcriptomic and metabolomic
phenotypes and up to yield and to test predictive process models. Process models
will be developed and refined using the mechanistic links that connect cell
processes and ultimately whole plant physiology to regulatory intermediates such
as metabolites and gene transcripts. If successful, the models developed will
enable prediction of whole-plant stress-response phenotypes in heterogeneous
genotypes and environments. The goals of the project are to: 1) deploy a novel
high-throughput and real-time phenotyping method to measure diel physiological
dynamics in eight B. rapa parental Nested Association Mapping (NAM) lines under
drought- and heat-stress conditions; 2) predict yield in a Recombinant Inbred
Line (RIL) population of B. rapa using a biophysical process model of carbon
metabolism, hydraulics and resource allocation to test systems-level links
between circadian, transcriptomic, metabolomic, and physiological QTL; and 3)
test the predictive ability of the biophysical process model under heat- and
drought-stress environments using the RIL population used in Aim 2. All data and
resources generated in this project will be made accessible to the public
through long-term open access repositories such as Project Github and the NCBI
Short Read Archive.