* 1547787
* TOOLS-PGR Alternative Splice Isoforms in Plant Genomes: Collection, Characterization and Evolutionary Relationships
* BIO,IOS
* 09/01/2016,08/31/2020
* William Barbazuk, University of Florida
* Standard Grant
* Gerald Schoenknecht
* 08/31/2020
* USD 457,671.00

Expressing a eukaryotic gene stimulates transcription, which produces an mRNA
transcript. Most often, the mRNA transcript is used as a template for
translation into protein molecules. The transcription process produces a pre-
messenger RNA, which is extensively processed to yield a mature mRNA ready for
translation. Because the structure of most genes in animals and plants is one
where the DNA sequence information used to specify protein (exons) is
interspersed with stretches of DNA sequences that are non-coding (introns), one
of the modifications that pre-mRNA undergo is the splicing out of intron
sequence and the joining of the exon sequences. In many cases, this splicing
process can be directed to purposefully skip exons, retain introns, and change
the exon boundaries. This process, called Alternative Splicing (AS), enables the
production of distinctly different messages from a single gene, and this can
increase the number of different proteins a gene is able to produce. The extent
and roles of AS in plants is not well understood; however, a thorough
understanding of AS, especially within crop plants that produce food, industrial
and fuel products, is necessary to enable prediction of phenotype from genotype
and utilize crop genetic resources to their full potential. The outcomes of this
project will be the genome-wide identification and public release of AS isoforms
within 42 plant species. Analysis of these data during this project will provide
a better understanding of the evolutionary history of AS in plants and provide
access to some of the genetic features involved in regulating AS. With regard to
outreach and training, this project will develop a course-based undergraduate
research experience (CURE) that provides hands-on training for undergraduate
life science majors in modern genomics, lab techniques, high performance
computing and "big data" OMICS analysis, skill sets that have become
increasingly important in research and medicine.

An understanding of the extent of AS in plants and the biological processes that
AS transcript isoforms impact are not well established. However, it is proposed
that alternatively spliced transcripts may play significant roles during growth,
development and stress response, and failure to identify these and understand
their significance hampers the understanding of gene to phenotype, and slows
crop improvement efforts. The long-term goal of this research is to understand
the function, regulation and evolution of alternative splicing (AS) in plants.
Specifically, the project will address the following questions: 1) What genes
produce alternatively spliced transcripts and what processes do they affect? 2)
How is AS in plants regulated? 3) What are the evolutionary histories of AS
isoforms in plants? To address these questions, this project will identify and
characterize AS events broadly from available public data and investigate their
ancestral states in plants. Specifically, AS isoforms will be identified across
and within 46 genomes representing 42 plant species broadly distributed across
the kingdom. Once identified, AS isoforms will be characterized with regard to
their conservation and evolutionary history. Finally, evolutionary conserved AS
events will be selected to investigate sequence and epigenetic features
associated with AS that may play a regulatory role. All data and resources
generated in this project will be made available to the public through the
project website, as well as long-term through the NCBI's SRA and CyVerse.