* 1550838
* EAGER-NEON: Genomic Plasticity in Response to Variable Environments
* BIO,DEB
* 01/01/2016,12/31/2019
* Michael Barker, University of Arizona
* Standard Grant
* Anne W. Sylvester
* 12/31/2019
* USD 300,000.00

Michael S. Barker, PI and Katrina M. Dlugosch, CoPI&lt;br/&gt; &lt;br/&gt;Plants
are rooted in place and cannot escape harsh environmental conditions. Instead,
many plant species can acclimate to varying environments by modifying their
structure or growth habit. In some cases, the acclimation is visible, as in the
example of plants that grow small leaves or flower earlier when the local
environment becomes water stressed. In other cases, the response to
environmental change is not visible because subtle biochemical or genetic modes
of survival are activated. This so-called "genomic plasticity" allows plants
such as invasive species to thrive in new or harsh niches compared with less
responsive plants. Similarly, plants with highly duplicated genomes, termed
polyploids, may acclimate better because they express more variation in diverse
traits in response to environmental change. Understanding how adaptive
plasticity occurs is critical to being able to predict plant responses to
environmental change. This project uses the novel environmental and biodiversity
data from NEON to investigate the genomic basis of plasticity. To study adaptive
plasticity of plants in the NEON footprint, new sequencing technologies will be
used to characterize all the genes expressed over several time periods in
selected polyploids and invasive species. These results will be compared against
the abundance and plant community structure of the species included in the NEON
biodiversity data. The project will thus establish the foundation for an
unprecedented view of how plant genomes, and their subsequent traits, change
over time. The study both leverages and expands NEON's long-term data to
determine how plasticity at the genome level may predict plant success.
&lt;br/&gt;&lt;br/&gt;Adaptive phenotypic plasticity is one of the primary means
by which species respond to climatic variation, but how plasticity functions in
plant genomes is not well understood. Plasticity is not easy to identify or
study because homeostasis in key traits can be maintained by plasticity at
underlying levels of organization. This research circumvents this problem by
using genomic measures of plasticity to track response to environmental
variation in plants across the NEON network. To measure and document plasticity,
the project combines transcriptome profiles from RNA sequencing of selected
plants at a NEON core site with NEON surveys of plant community composition. The
specific activities are to (1) identify ploidy levels in species at the Harvard
Forest NEON core site and select target polyploid, diploid and invasive species
for the study; (2) conduct RNA sequencing of 30 target species and (3) correlate
genome plasticity with species abundance. The project leverages NEON's long-term
study sites and data, while extending the power of NEON's plant diversity data
to the genomic level. Measurements of genomic plasticity will provide highly
relevant and comparable metrics of plastic responses to the environment across
taxa, space, and time. The data and analytical tools will be publicly available
through NCBI and the iPlant Collaborative, and will benefit society by providing
methods to predict how plants in uncertain environments will respond to change.