* 1553211
* CAREER: Integrated Annotation and Comparative Analysis of Metabolic Models
* BIO,DBI
* 04/01/2016,03/31/2022
* Ying Zhang, University of Rhode Island
* Continuing Grant
* Peter McCartney
* 03/31/2022
* USD 801,379.00

The new high-throughput sequencing technologies have led to a rapid accumulation
of genomic data across a great diversity of organisms. Those data can now be
used to build models that describe how genome variation is translated into
phenotypic differences. This research will build a new computational framework
to combine all of the genomic variants and simulate the resulting metabolic
phenotypes. This framework will be broadly applicable, so it can analyze any
group of organisms, and will include good documentation for the user-friendly
interface, designed to be immediately accessible instead of requiring extensive
training. The project will develop an online game based on the research
framework and data that will be used to educate students and interested members
of the public about the basic concepts of metabolic activities and metabolic
evolution. Graduate students who have developed applications in the advanced
framework will advise undergraduate students, who will be recruited from the
University of Rhode Island's Seed of Success (SOS) program for summer
internships, and who will develop and test the game interfaces. Finally, the
educational game will be used during the two-day Engineering Challenges in the
Science and Math Investigative Learning Experiences (SMILE) Program to provide
an interactive learning experience for high-school students. The computational
tools and the educational gaming system will be released through public web
servers and software repositories, and the research outcomes from this project
will be disseminated through open-source documentations, journal publications,
and conference presentations.&lt;br/&gt;&lt;br/&gt; The genome-scale models
(GEMs) of metabolic networks have broad applications in phenotype prediction,
evolutionary reconstruction, functional analysis, and metabolic engineering.
Despite the reconstruction of over 100 GEMs in public literature, significant
challenge remains in integrating the heterogeneous information about genes,
proteins, reactions and metabolites into a comprehensive understanding of the
associations between genome diversity and metabolic plasticity. To solve this
problem, this research will focus on constructing a new computational
infrastructure that combines the simulation of GEMs with the analysis of pan-
genomes (i.e. an ensemble of genomes from any given group of organisms). The
computational infrastructure will include a user-friendly interface that
supports the collaborative construction, annotation, and quality checking of
metabolic models. It will also enable the identification of genome-wide
metabolic variations by providing new algorithms for GEM comparison and pan-
genome analysis. Specifically, this project will focus on achieving three aims:
(1) implement a new data format that will integrate heterogeneous information
into the construction of metabolic models; (2) build a computational tool that
will support the comparative analysis of metabolic networks; (3) develop a new
algorithm that will improve the efficiency of ortholog mapping in pan-genome
analysis and permit the comparison of metabolic variants among different
organisms. More information about this project can be found at
https://zhanglab.github.io/psamm/.