* 1553680
* III: EAGER: Novel algorithms for de novo transcriptome assembly using RNA-seq data and for metagenome assembly
* CSE,IIS
* 09/01/2015,08/31/2018
* Xiuzhen Huang, Arkansas State University Main Campus
* Standard Grant
* Aidong Zhang
* 08/31/2018
* USD 115,280.00

While high-throughput sequencing technology provides an unprecedented
opportunity to reveal the complexity of transcriptomes or metagenomes, it poses
a significant challenge to accurately and efficiently assemble the huge amount
of short fragments into transcriptomes or genomes. A number of assemblers have
been developed, but they all have limitations that have hindered their
applications. This project will develop novel approaches, which would transform
the method design and development of the challenging transcriptome and
metagenome assembly. This project will also help educate students through
seminars and courses how effective computational models could make a difference
in addressing bioinformatics challenges.&lt;br/&gt;&lt;br/&gt;Based on the new
insights gained through the preliminary work on a heuristic approach-based de
novo assembler Bridger, recently published in Genome Biology, this project will
develop novel algorithms for de novo transcriptome assembly using RNA-seq data.
The novelty of the approach lies in (1) new graph models, different from the de
Bruijn graph or overlap graph of existing assemblers, and (2) the search
algorithms, both of which will integrate sequence coverage depth information and
paired-end reads into the procedure. It is anticipated that the new approach
will achieve significantly increased sensitivity and specificity, compared with
current de novo assemblers including Trinity or even current reference-based
assemblers such as Cufflinks or StringTie. Furthermore, based on the algorithmic
techniques developed for transcriptome assembly and the common features between
transcriptome and metagenome assemblies, this project will explore and develop
algorithms to assemble metagenomes, which is a computationally demanding task
due to the mixture of large collections of short DNA sequence fragments from
many different organisms, and the inconsistency of sequencing coverage of
different organisms. &lt;br/&gt;&lt;br/&gt;For further information see the
project web site at: http://bioinformatics.astate.edu/assembly/