* 1555769
* COLLABORATIVE RESEARCH: NSFDEB-BSF: Quantifying genomic porosity in non-model radiations
* BIO,DEB
* 03/01/2016,02/28/2019
* Adam Siepel, Cold Spring Harbor Laboratory
* Standard Grant
* David Cannatella
* 02/28/2019
* USD 197,334.00

Understanding how organisms are related to each other is a central goal of
biological research, yet much of the branching structure of the tree of life
remains imperfectly known. Comparisons of DNA sequences have been used to build
reliable evolutionary trees for many groups of organisms. Yet in cases where
species hybridize with one another (or have done so in the past), their
descendants become a mosaic of genetic backgrounds, and this genetic mixing
makes it more difficult to back-calculate their past relationships.
Hybridization is particularly common in groups of species that have undergone
rapid radiations - situations when large numbers of species evolve over a
relatively short period of time - while at the same time these rapid radiations
are models for many other kinds of evolutionary analysis. This project will
investigate the type of genomic regions that are exchanged among species during
rapid radiations. A central goal of the researchers is to develop a computer
program that leverages large-scale genomic data to better understand the
phylogenetic relationships among species, improving the ability to incorporate
the effects of past hybridization events. This tool will be made available to
the research community to help biologists understand evolution at the tips of
the tree of life. The project will add interactive multimedia-based modules on
phylogenetics and systematics to the highly successful All About Bird Biology
website. The project is a funded collaboratively by the NSF and the Binational
Science Foundation of Israel.

The overarching goals of this project are to test questions about the degree and
type of genomic regions exchanged among species in rapid radiations, and to
explore the implications for phylogenetic inference. These objectives are based
on the further development of a new bioinformatics tool (Generalized
Phylogenetic Coalescent Sampler - G-PhoCS) that will allow scientists studying
non-model organisms to test hypotheses about phylogenetic relationships and
generate detailed demographic inferences (e.g., quantifying gene flow, effective
population sizes and divergence times) across a phylogeny, using a variety of
genomic-scale datasets. As empirical test cases, the researchers will leverage
natural evolutionary experiments in two avian radiations in which pairs of
species can be classified as either introgression-prone or introgression-
resistant based on external criteria such as ecological overlap and
sympatry/allopatry. The research team brings together systematists and computer
scientists attempting to realize the full potential of newly emerging high
throughput sequence data by developing inference methods that make use of the
rich information they provide. This collaboration will more generally broaden
the understanding of the patterns and processes underlying genealogical
discordance among regions of the genome.