* 1557605
* QuBBD: Mathematical models for a molecular genetic understanding of population variation in risk of cardiovascular disease
* MPS,DMS
* 09/15/2015,08/31/2016
* Stephen Ramsey, Oregon State University
* Standard Grant
* Nandini Kannan
* 08/31/2016
* USD 99,579.00

Genome-wide association studies (GWAS) are a powerful approach for mapping the
regions of the genome containing single-nucleotide differences in the population
(single-nucleotide variants or SNVs) that are associated with trait variation.
Narrowing down from GWAS-identified genomic regions to the individual SNVs that
are responsible for trait variation is particularly challenging for regions of
the genome that are in-between genes. A second challenge in GWAS is that the
large number of SNVs necessitates a high level of statistical stringency, and
thus, many biologically relevant SNVs are missed. GWAS is used for many human
disease traits (such as coronary artery disease or CAD), and thus, addressing
these two challenges would have broad significance in biology and biomedical
research. This award supports initiation of a collaborative research project
that will address these two challenges by developing mathematical models that
integrate a variety of types of measurements and information derived from cells
and population studies, in order to pinpoint SNVs in-between genes that affect
trait variation, and to improve the statistical power of GWAS. CAD is a high-
significance application for improving GWAS because of CAD's prevalence (15
million in the U.S.).

The objectives of this project are to (1) create and evaluate an integrative
statistical model for improving power for GWAS analysis and for discovering
novel gene-trait associations and (2) create and evaluate a machine-learning
model for identifying regulatory variants within intergenic GWAS regions. The
models would incorporate features from large-scale datasets from the Framingham
Heart Study SHARe database, the ENCODE project, the GTEx project, and
CARDIOGRAMplusC4D. The models' performance would be benchmarked against
previously published models. The project's significant outcomes would be: (1)
the first analytic statistical model for integrative GWAS analysis that would
provide a readily interpretable significance score; (2) a quantitatively
validated and interpretable machine-learning model for combining genomic
information types to predict regulatory variants; (3) feature importance scores
for the genomic features that are integrated within the model; and (4)
identification of new GWAS loci for population variation in CAD risk, and, for
regulatory variants within the loci, the genes and transcription factors (and
ultimately, the gene functional annotations) with which they are associated.
Software implementations of the methods will be shared in an open-source
software repository. This award is supported by the National Institutes of
Health Big Data to Knowledge (BD2K) Initiative in partnership with the National
Science Foundation Division of Mathematical Sciences.