* 1564621
* ABI Innovation: A computational framework for integrating image informatics with transcriptomics for discovering spatiotemporally resolved regulatory gene networks in plants
* BIO,DBI
* 07/01/2016,06/30/2020
* Harkamal Walia, University of Nebraska-Lincoln
* Standard Grant
* Peter McCartney
* 06/30/2020
* USD 563,801.00

Biological processes at all scales from molecules to ecosystems are coordinated
through the encoding, exchange, and interpretation of information. Many of the
recent advances in the biological sciences collect very large sets of DNA
(genotype) and protein sequence: making sense of how their properties lead to
the appearance, responses and behaviors (phenotype) of organisms requires new
computational tools. Linking the genotype to the phenotype requires untangling
very complicated relationships. Collecting large amounts of data for large
numbers of samples means developing new tools for data processing and data
modeling. This project will develop computational tools to address both of these
challenges, while investigating the effects of salt tolerance in rice. Large
sets of expression data will be collected along with images of the developing
plants from several imaging technologies. The images will be used to produce 3D
reconstructions of the plant architecture that can be stored as digital
phenotypes. It is expected that these digital phenotypes will allow plant
breeders to identify traits that were not captured by standard observation
methods, and find new genotype-phenotype associations. The educational
activities are timely, providing training opportunities for plant biology and
computer science students. As the volumes of image-derived digital data surge,
image informatics is emerging as a cutting-edge discovery tool. Outreach and
training activities include a summer workshop for plant phenomics and
establishing a data visualization community at University of Nebraska. Project
resources will be released on CyVerse to ensure wide availability.
&lt;br/&gt;&lt;br/&gt;High-throughput sequencing technology- driven
transcriptome analyses are a commonly utilized approach for gaining molecular
insights on an organism's response to environmental changes. These
transcriptome-level responses and the underlying regulatory gene networks that
drive phenotypic responses to changing environment are highly dynamic. With the
advent of high-throughput image-based plant phenotyping, it is now possible to
capture the dynamics of growth and other digital-features responding to
environmental or genetic perturbation with increased temporal and spatial
resolution. This project will use temporal imaging to inform the collection of
time series transcriptome data with 3D-spatial sensitivity for discovering
dynamic regulatory co-expression networks. Outcomes of the project will be to
develop innovative algorithms to integrate heterogeneous datasets and implement
it in a prototype computational framework. The platform will enable interactive,
multidimensional visualization of dynamic regulatory networks with spatial and
temporal resolution. The project resources can be accessed at:
http://cropstressgenomics.org/phenomics.php