* 1564659
* ABI Development: Improved Tools for Population Genomics
* BIO,DBI
* 07/01/2016,11/30/2016
* Emanuel Hey, Temple University
* Standard Grant
* Peter McCartney
* 11/30/2016
* USD 794,836.00

The goal of this project is to develop computer software that allows scientists
to compare the genomes of natural populations of organisms. Population genomics
(the study of diversity in the genomic DNA of populations instead of
individuals) has important applications in agriculture, anthropology,
conservation, medicine, and several other fields of biology. Recent technology
advances have greatly lowered the cost of DNA sequencing so large amounts of
data are available. At the same time statisticians have developed new methods
for analyzing large collections of genome-sized data. Combining the data with
the new analysis approaches can let scientists answer fundamental questions
about the biology of species that have never before been accessible. However,
most population genomics studies fail to take advantage of the latest population
genomics statistical methods. This is because the genomic data requires a lot of
processing to prepare it for the analysis steps, and these operations are not
easy to learn. Lack of bioinformatics knowledge has led to erroneous
conclusions, irreproducible results, incomplete analyses, and slow turnaround
time of population genomics projects. This project aims to provide software that
can handle those tasks in a consistent and correct way and allow direct access
to the statistical tools, to bring about new insights in population biology.
&lt;br/&gt;&lt;br/&gt;This project will address the largest impediments to
model-based population genomics by building a user-friendly and intuitive
pipeline platform, the "Pop-gen Pipeline Platform" (PPP). This platform will
help researchers avoid the problems inherent in re-inventing data processing
scripts to feed data into their population genomics analyses, and considerably
reduce time and effort. PPP will include tested scripts and pipelines for data
preparation, and in addition the tools needed to build evolutionary models,
carry out analyses of data, and create visualization of results via the Ruffus
Computation Pipeline Library in Python. PPP will also be developed as a stand-
alone application with a graphical user interface (GUI) under the popular and
robust Galaxy Project (www.galaxyproject.org) for bioinformatics analyses. An
online server deploying Galaxy/PPP will be developed and maintained at Temple
University, Philadelphia (www.dev3.cst.temple.edu) for users to interactively
submit jobs on a first-come-first-served basis. Online forums will be
established for support, and to collect suggestions regarding what is desired
for further development under PPP. Tutorials and videos about example
applications of the PPP will be added to the Galaxy Project Wikipedia training
channels. Additionally, a significant part of this work will involve educating
Temple University seniors, Philadelphia area high school teachers, and their
students about the power of bioinformatics in modern biology through summer
workshops. Information about PPP, and associated developments can be accessed at
https://bio.cst.temple.edu/~hey/software/software.htm.