* 1565057
* Collaborative Research: ABI Development: Extensible, reproducible and documentation-driven microbiome data science
* BIO,DBI
* 05/01/2016,04/30/2019
* Rob Knight, University of California-San Diego
* Standard Grant
* Peter McCartney
* 04/30/2019
* USD 329,606.00

Single-cellular organisms (microbes) represent a vast component of the diversity
of life on Earth and perform an amazing array of biological functions. They
rarely live or act alone and instead exist in complex communities composed of
many interacting species that make up the microbiome. This award supports the
development of the next generation of Quantitative Insights Into Microbial
Ecology (QIIME, pronounced "chime"), a free and open source software platform
for analyzing microbiomes based on DNA sequencing data. Microbiome science is in
a transformation from being descriptive and technically challenging, to becoming
hypothesis-driven, actionable, and technically straight-forward, in part enabled
by QIIME. We now know that the traditional approach for studying microbial
communities, which relied on culturing microbes in the lab, is insufficient
because we don't know the conditions required for the growth of most microbes.
Recent advances link microbiomes to functional processes via 'culture
independent' techniques, such as sequencing fragments of microbial genomes, and
then using those fragments as 'molecular fingerprints' to profile the
microbiome. The bottleneck in microbiome analysis is not DNA sequencing, but in
interpreting the large quantities of sequence data generated. QIIME 2 will
advance knowledge of microbiomes by helping users derive insight through
interactive exploratory analysis capabilities, understand the underlying
methods, and report their results in ways accessible to end users from outside
of the field, including physicians, engineers and policymakers who urgently need
access to conclusions drawn from studies of complex microbial ecosystems.
Societal benefits range from global to personal (from understanding cycling of
biologically essential nutrients, such as carbon and nitrogen in the environment
to curing disease, including obesity and cancer). QIIME has been cited over
4,000 times and has active user and developer communities. Educational workshops
on QIIME are taught approximately monthly in the USA and around the
world.&lt;br/&gt;&lt;br/&gt;At its core, QIIME 2 will provide a stable
application programming interface (API) relying on existing community standards
for documentation, coding style, and testing. It will have a novel
'documentation-driven' graphical user interface that will make QIIME accessible
to users without requiring advanced computational skills. At the same time, it
will help users improve their computational skills through exposure to the
underlying bioinformatics methods. QIIME 2 will have fully integrated provenance
tracking, which will simplify reporting and the reproducibility of
bioinformatics workflows. A first-class plugin system will decentralize
development by allowing outside developers to add new methods to the QIIME 2
platform. The API will also support improved integration of QIIME as a component
of other widely used systems, such as Illumina BaseSpaceÂ® and Qiita, and an
automatically generated command line interface will be provided for power users.
QIIME 2 will have a completely redeveloped parallel framework, which will
support deployment on diverse high-performance computing resources, from locally
owned and operated computer clusters to commercially available cloud computing
platforms. All stages of QIIME 2 development will be driven by user community
input through the QIIME Forum (currently over 2500 active users) and our public
GitHub repository. Further details on this project are on the QIIME website
(www.qiime.org).