* 1565166
* ABI Innovation: A framework to predictably manipulate a microbial gene regulatory program
* BIO,DBI
* 05/01/2016,04/30/2021
* Nitin Baliga, Institute for Systems Biology
* Continuing Grant
* Peter McCartney
* 04/30/2021
* USD 1,541,850.00

Living organisms have to adjust to changes in their environment in order to
optimize their use of resources, minimize stress and maintain stability. This
proposal aims to uncover the basic principles that direct how organisms tailor
their physiological responses to environmental changes. Understanding why
certain responses occur requires a precise map showing which genes are regulated
in a given response and how they are coordinated. Using prior NSF ABI support,
the Baliga Laboratory at the Institute for Systems Biology developed an approach
to create precise maps of gene regulation for any microbial species. Using this
approach, they mapped gene regulation in a set of important organisms including
uranium-reducing Desulfovibrio vulgaris, lipid-accumulating Chlamydomonas
reinhardtii, yeast, diatoms, and Mycobacterium tuberculosis. The next goal is to
see how well the maps work as tools to predict the results when manipulating
complex behaviors. Effective tools have wide-ranging implications for
biotechnology, agriculture and medicine. Initially, new algorithms and software
will be developed to further refine the gene regulation maps and identify
factors affecting gene regulation during environmental changes. The focus will
be understanding how organisms switch on or off specific behaviors in response
to environmental cues. The work will be performed in two organisms - E. coli, a
well-known, widely studied bacterium, and Halobacterium salinarum, an
extremophile that thrives in high salt environments; it will be readily
applicable to all sequenced microorganisms that are of significant industrial,
agricultural, and medical importance. Part of the project is to develop and
disseminate new high school curriculum to introduce the importance of
computational modeling in solving real world problems such as food scarcity and
climate change. Diverse populations of students and teachers from a variety of
backgrounds, including those currently underrepresented in science, technology,
engineering and math (STEM), will receive training and sustained support as they
learn this interdisciplinary science. This curriculum and training is part of a
program called Systems Education Experiences (SEE) that reaches thousands of
students and teachers each month. SEE works toward cultivating systems thinkers
who can tackle problems, contribute to a STEM-literate citizenry, and help build
a more diverse population of STEM professionals.

The primary objective of this project is to develop a framework to elucidate and
predictably manipulate the gene regulatory program of any microbe. In previous
ABI-funded research, the Baliga lab developed a systems approach to reverse
engineer the environment and gene regulatory influence network v2.0 (EGRIN 2.0)
model directly from a compendium of transcriptome profiles. The EGRIN 2.0 model
elucidates mechanisms for environment-specific transcriptional regulation of all
genes with unprecedented nucleotide-level resolution, at canonical promoter
locations and even within coding sequences and inside operons. Here, an approach
will be developed to elucidate transcription factor interactions within EGRIN
2.0 and characterize how the topology of these interactions (i.e., 'network
motifs') generates genome-wide, temporally coordinated transcriptional
responses. These studies will be performed in the context of understanding how
two phylogenetically distant organisms --Escherichia coli (a bacterium) and
Halobacterium salinarum (an archaeaon)-- use distinct regulators to mediate
physiologically different yet phenotypically similar transitions from aerobic
growth to anaerobic quiescence. First, an approach will be developed to
precisely map conditional binding of transcription factors to sequence elements
within promoters of all genes in the genome (EGRIN 3.0). Next, EGRIN 3.0 will be
used to identify, characterize, and manipulate topologies of transcription
factor interactions (i.e., network motifs) to predictably alter oxygen
(O2)-responsive state transitions in H. salinarum and E. coli. In addition to
developing a generalized framework for manipulating a microbial gene regulatory
program within any organism, the activities will test the hypothesis that
similar environmental forcing drives convergent evolution of topologically
similar network motifs in phylogenetically distant organisms. The high-level
thinking and process used by this interdisciplinary group will be translated
into curriculum and training experiences in the form of real-word cases studies
for high school teachers and students. One of the goals will be for students to
use experimentation and modeling to better understand the influence of
environmental parameters (such as oxygen, nitrates, pH, light, etc.) on
productivity and stability of food systems, such as aquaponic systems. Students,
teachers, and STEM professionals will work together to iteratively develop and
test curriculum and experiences through a modified Dick and Carey Instructional
Design model. All curricula will be integrated with published national education
standards. All needed technology, software, lesson plans and learning aides will
be provided to teachers and students through multiple online sources, resource
centers, and in-person and online trainings. For further information about this
project and its products, visit the Baliga Laboratory's website at
http://baliga.systemsbiology.net and http://see.systemsbiology.net.