* 1565076
* ABI Innovation: Tools and databases for network-based plant systems biology with applications to understanding plant-virus interactions
* BIO,DBI
* 04/15/2016,03/31/2021
* Garry Sunter, University of Texas at San Antonio
* Standard Grant
* Peter McCartney
* 03/31/2021
* USD 683,839.00

This project aims to improve the tools for making models for networks of
interacting molecules in the small mustard plant, Arabidopsis. To demonstrate
the effectiveness of these methods, the resources developed in this research
will be used to study the plant's immunity to a virus infection under several
conditions. Understanding how and when the virus overcomes the plants defenses
gives us a handle on controlling an important pathogen of crop plants in the
Southern US, with benefits such as better yield and less pesticide use. While
biologists can measure the presence and amounts of tens of thousands of
molecules in a cell all at once, understanding how they are connected, the
'networks' or 'systems' that lead to function, is a much harder problem. The
huge amounts of measurements have to be properly managed so they are usable, and
additional information has to be correctly added: it is important to track how
amounts of one type of molecule change over time and not mix up different
things. It is also important to understand which parts of the cell affect one
another: those that belong to a functionally connected pathway (or network), and
which are independent of each other. For example, plants have complicated
mechanisms to defend themselves against biological and environmental stresses.
Signaling pathways cause the plant's response, and they are influenced by
internal genetic factors as well as the external ones that are more easily
observed. If every protein (or other molecule) inside the cell that plays a part
in carrying and interpreting the signal is known then very effective predictions
about the final response are possible. However, plant researchers don't know
nearly as much about the molecules in their organisms as is available to many
researchers studying animals, which means there are a lot of missing nodes. That
makes it hard to come up with a specific prediction that can be tested: this
research aims to overcome this problem for selected plant pathways, to showcase
what is possible when there is sufficient information. This project will
actively engage students in interdisciplinary research, with a particular focus
on recruiting underrepresented groups. The University of Texas at San Antonio
(UTSA) is a Hispanic-Serving Institution.

This project has the following four specific aims: 1) to construct genome-wide
transcriptional regulatory network in Arabidopsis with validation in immune-
responsive genes; 2) to improve the prediction of protein-protein interactions
and identification of defense subnetworks in Arabidopsis; 3) to perform network-
based analysis of Arabidopsis immune-responsive network in order to decipher the
role of plant viral RNA silencing suppressors in plant immunity; and 4) to
provide online databases and analytic tools for network-based plant systems
biology studies. This project promises to significantly improve network-based
analysis with several innovative ideas. First, the proposed approaches focus on
improving accuracy of predictions for individual genes by defining a network
neighborhood for each node and testing for enrichment in the neighborhood for
each node. This is in contrast to most existing approaches that make predictions
on gene modules (within- or cross-species) and therefore lack quality control on
an individual gene level. Second, combining protein-protein interaction network,
gene co-expression network, and sample-sample network, this research provides an
example to analyze such networks in a dynamic context automatically defined by
the global transcriptomic landscape; as such, this study is expected to provide
more specific predictions that can be experimentally tested. In addition,
integration of computational tools to characterize defense-related network
structure in this work will significantly improve the ability to study the role
of co-regulated networks of genes in any number of processes, including but not
limited to genes implicated in both plant and animal disease, cancer or stem
cell biology, or tissue specificity of gene expression. The results of the
project can be found at http://cs.utsa.edu/~jruan/plantnet/.