* 0346591
* Collaborative Research: Patterns and Mechanisms of Gene Expression in Recent Allopolyploids of Recurrent Origin
* BIO,MCB
* 02/15/2004,02/28/2006
* Z. Jeffrey Chen, Texas A&M Research Foundation
* Continuing Grant
* Susan Porter Ridley
* 02/28/2006
* USD 138,069.00

Polyploidy (genome duplication) represents a major force in plant evolution and
an important evolutionary process in other organisms, but the ecological,
physiological, genetic, and genomic consequences of genome doubling remain
mostly unknown. Most polyploids are formed via hybridization between two
species, and it is unclear whether the polyploids express all of the attributes
of both parents or are mosaics that combine some characters of each parent.
Recent studies of polyploid gene expression have focused on artificially
produced polyploids in Arabidopsis, cotton, and wheat rather than on naturally
occurring polyploids. In addition, although most polyploid species have formed
repeatedly in nature, nothing is known about the consequences of this recurrent
formation on gene expression. A central question yet to be addressed is whether
individuals of separate origin show similar or divergent gene expression
profiles. A model system for the study of recent and recurrent polyploidy in
natural populations is provided by Tragopogon (Asteraceae, the Sunflower
family). Two new polyploid species (T. mirus and T. miscellus) were formed
following the introduction of three diploid species (T. dubius, T. pratensis,
and T. porrifolius) from Europe to western North America during the early 1900s.
Tragopogon miscellus is particularly interesting because it has also formed
reciprocally (that is, once with T. dubius as the maternal parent and multiple
times with T. pratensis as the maternal parent). This project will involve
molecular and genomic approaches to address a series of questions that are
fundamental to an increased understanding of polyploidy. For example, does novel
gene expression occur in these recently formed polyploids? Do gene expression
patterns differ among polyploid populations of independent origin? Is gene
expression affected by maternal and paternal contributions? Do gene expression
patterns in F1 hybrids and newly produced polyploids differ from those of the
naturally occurring polyploids? What types of genes undergo differential
expression in the natural polyploids, and what types of mechanisms lead to the
observed differences in gene expression? Such questions cannot be readily
addressed in current crop systems or with resynthesized polyploids. This
research will provide the first rigorous assessment of gene expression in
natural polyploid populations, plus new insights into the rapid evolutionary
success of two new polyploid species. The collaboration among PIs D. Soltis, P.
Soltis, J. Tate (University of Florida) and Z. J. Chen (Texas A&amp;M
University) combines their respective expertise in evolutionary biology and
molecular genetics. Opportunities for training of undergraduate and graduate
students, post-docs, and K-12 teachers are integrated into the
project.&lt;br/&gt;