* 0211308
* Statistical Mechanics of Gene Regulation
* MPS,DMR
* 08/01/2002,07/31/2006
* Terence Hwa, University of California-San Diego
* Continuing Grant
* Daryl Hess
* 07/31/2006
* USD 225,000.00

This theoretical research is funded jointly by the Divisions of Materials
Research and Molecular and Cellular Biology. Rapid advances in genomic and
molecular technology have led to much excitement and opportunities in molecular
biology. A hugh amount of data have already been collected on genomic sequences,
gene expression profiles, and protein-protein interactions. System-level issues
involving gene, protein, and cellular networks have emerged as the new
challenges in quantitative understanding of molecular and cellular biology. This
research will study system-level issues in the context of gene regulation. An
explicit model will be used to describe the combinatoric control of gene
expression, and using simple generic interactions of transciption factors with
each other and with their DNA building sites. The model will be used to
investigate how the molecular components may be put together to implement
regulatory functions of increasing complexity, and how these regulatory systems
can in turn be coupled to each other to form genetic networks exhibiting robust
patterns of gene expression. A key feature of this model is the identification
of a subset of interactions that can be tuned by simple manipulation of DNA
sequences in the regulatory regions. They are referred to as "programmable"
parameters and can be individually adjusted by the system in order to implement
the imposed functional requirements. The model can be cast in the form of an
interacting spin system, and shares striking similarities with the "recurrent
networks" studied in neural networks. It can also be reduced to a more versatile
Boolean description in some cases. Analytical and numerical studies will be done
to characterize the properties of the networks, e.g., the complexity of the
regulatory functions the system can implement, and the capacity of the stable
expression patterns the network can maintain. In particular, a stochastic
"supervised learning rule" resembling the evolutionary dynamics of regulatory
sequences will be studied to search for the programmable parameters of the
system that best achieve the desired functional responses. This dynamics can
therefore be used also to study the evolvability of the regulatory control
systems and the gene networks. The evolution of the stability and complexity of
these regulatory and interactive gene systems will also be
studied.&lt;br/&gt;%%%&lt;br/&gt;This theoretical research is funded jointly by
the Divisions of Materials Research and Molecular and Cellular Biology. Rapid
advances in genomic and molecular technology have led to much excitement and
opportunities in molecular biology. A hugh amount of data have already been
collected on genomic sequences, gene expression profiles, and protein-protein
interactions. System-level issues involving gene, protein, and cellular networks
have emerged as the new challenges in quantitative understanding of molecular
and cellular biology. This research will study system-level issues in the
context of gene regulation.&lt;br/&gt;***&lt;br/&gt;