* 0215769
* Combinatorial Regulation of Differential Gene Expression in E. coli
* BIO,MCB
* 08/15/2002,07/31/2007
* Donald Senear, University of California-Irvine
* Continuing Grant
* Jo Ann Wise
* 07/31/2007
* USD 530,000.00

Coordinate regulation of unlinked genes is fundamental to genetic expression in
all organisms. Such regulation relies on the cooperative, site-specific assembly
of regulatory, protein-DNA complexes. Combinatorial strategies are common. These
involve both widely used transcription factors and also factors that are
specific either to the gene, the tissue or cell type, or the development stage.
Variable arrays of DNA sites, as are generally found in different genes,
assemble different protein-DNA complexes from the same factors. In this manner,
unlinked genes can be expressed differentially. Whole programs of gene
expression are easily controlled by modulating the activity of only the specific
factors. This project addresses the combinatorial mechanism of differential gene
expression in the E. coli CytR regulon. This gene family consists of nine
unlinked transcription units that are regulated coordinately by the interplay of
just two regulatory proteins. One is the cyclic-AMP receptor protein (CRP) a
widely used transcriptional activator in bacteria; the other, CytR, is a
regulon-specific bacterial repressor. CytR is also a member of the LacI family
of homologous repressors. A key feature of the regulation is that the various
cistrons differ from one another in extents of activation, repression and
induction. This differential regulation is achieved by nesting levels of local
repression, mediated by CytR, on the global regulation mediated by CRP. An
unusual feature of most CytR-regulated promoters is that they contain two CRP
sites. These activate transcription via different mechanisms. CytR and CRP bind
cooperatively to DNA, as a result of direct protein-protein interactions. The
importance of cooperativity is underscored by the fact that induction occurs
when CytR binds cytidine because the CytR-CRP cooperativity is lost and despite
no affect on intrinsic CytR binding to DNA. Unlike most bacterial repressors,
CytR does not repress transcription independently; instead it acts by modulating
the CRP-mediated activation. It does so by interacting with DNA-bound CRP to
control the interactions between activating regions on CRP and the bacterial RNA
Polymerase. The objectives of the research are: 1) to understand how the
different structures of CytR-regulated promoters mediate different patterns of
cooperative interactions between CRP and CytR; and 2) to understand the role of
CytR-CRP cooperativity in controlling activation and repression of RNAP at
different promoters, using both site occlusion and allosteric mechanisms. To
accomplish these goals a comparison of the interactions among CRP, CytR and the
four class III CytR-regulated promoters will be made using DNase footprinting
and gel mobility shift assays. The role of different binding modes of CytR and
different arrangements of CytR and CRP binding sites will be assessed
systematically using artificial promoters. These will allow independent controls
on the structure of the CytR operator and locations of regulatory sites. These
data will be combined with results of both steady state and pre-steady state
assays of transcription initiation to develop a comprehensive kinetic model of
differential gene regulation. CytR and CRP mutants that are defective either in
their mutual interaction or in their interaction with RNAP will be used to
assess the role of the interaction between CytR and CRP bound to CRP2 in
modulating the class II mediated activation.&lt;br/&gt;This project is supported
by the Biochemistry of Gene Expression Program and the Molecular Biophysics
Program in the Division of Molecular and Cellular Biosciences in the Directorate
for Biological Sciences.