* 0217594
* Genomics of Loblolly Pine Embryogenesis
* BIO,IOS
* 09/01/2002,11/30/2005
* Carol Buell, Institute of Paper Science and Technology
* Standard Grant
* TEST user
* 11/30/2005
* USD 1,380,910.00

The formation of embryos within the seeds of pine trees differs from seed
formation in traditional crop and vegetable plants. While in both cases an
embryo forms inside a seed, develops and finally germinates to form a plant, the
morphology, biochemistry and genetics of the processes in gymnosperms (e.g.
pine) and angiosperms (e.g. carrot) are distinctly different. Embryogenesis is
the result of a regulated program of gene activity and the elucidation of the
program for loblolly pine is the goal of this project. The multiplication of
pine embryos in a laboratory setting and the production of plantlets, has great
implications for reforestation

After corn, forest products are the second most valuable US agricultural
product. Loblolly pine (Pinus taeda L.) is the primary commercial species in
southern conifer forests covering 13.4 million hectares, and over 1.5 billion
loblolly pine seedlings are planted annually. Somatic embryogenesis, a tissue
culture technique for embryo multiplication, is an excellent experimental tool
for investigating embryogenesis in this tree. However, success of the technique
is genotype-dependent and remains inefficient. Knowledge of the program of gene
expression in natural and laboratory-cultivated embryos will provide insight
into varying metabolism of these embryos and a basis for protocol improvement.

A cDNA library will be created from loblolly pine zygotic (natural) and somatic
(laboratory-cultivated) embryos. Together, these will represent a combined set
of transcripts expressed during all stages of embryogenesis, thus giving a
'snapshot' of genes participating in the formation of the pine embryo. Randomly
selected cDNA clones will be 5' end-sequenced to generate 35,000 Expressed
Sequence Tags or ESTs. These ESTs will be clustered and assembled to construct a
non-redundant pine EST database (TIGR Gene Index).

The non-redundant cDNA clone set will be used to obtain gene expression profiles
during various stages of zygotic and somatic loblolly pine embryogenesis. This
approach should identify the genes active in embryogenesis allow determination
of their identities and temporal expression patterns. A comparison of profiles
from laboratory-grown and natural embryos will allow insights into embryogenesis
to be gained, and further development of improved protocols for loblolly pine
somatic embryogenesis. Together, these data will provide the first detailed
overview of genome-wide gene expression patterns in conifer embryo development.
In the long-term, this kind of information will be useful in developing
strategies to accelerate clonal propagation.