* 0223590
* [KIL-d]: A Novel Epigenetic Regulator of Viral Gene Expression in Yeast.
* BIO,MCB
* 09/01/2002,08/31/2006
* Michael Leibowitz, Rutgers, The State University of New Jersey-RBHS-Robert Wood
* Standard Grant
* Susan Porter Ridley
* 08/31/2006
* USD 387,012.00

This project aims to determine the genetic locus encoding [KIL-d], a
cytoplasmically-inherited genetic element in yeast that exerts epigenetic
regulatory effects on the M double-stranded (ds) RNA genomic segment of the
cytoplasmic killer virus of yeast. In wild-type yeast cells, M dsRNA confers on
infected cells the phenotypes of toxin production and toxin resistance, both
activities of processed products of the virus-encoded preprotoxin protein. In
the presence of [KIL-d], the expression of M dsRNA is normal in diploid cells,
but upon meiosis each haploid progeny clone shows a different mitotically stable
(variegated) defect in toxin production and/or resistance function. Upon mating
of the defective haploids, wild-type virus expression is restored (healing), and
upon meiosis variegated defective phenotypic expression in each haploid progeny
reappears (resetting). The investigator has previously shown that the effects of
[KIL-d] are exerted on M dsRNA but not on a cDNA clone derived from it. [KIL-d]
is transmissible to a new strain by cytoplasmic transfer, but it does not exert
its epigenetic effects on M dsRNA expression until the recipient haploid cells
have gone through a cycle of mating and meiosis, indicating that nuclear
functions are involved in the epigenetic regulation. [KIL-d] does not map on the
viral M or L-A dsRNA segments, mitochondrial DNA or the 2-micron DNA plasmid.
[KIL-d] has a relatively high rate of loss (like many plasmids), and an even
higher rate of reappearance with variegated phenotypic expression in clones that
have lost it. This behavior resembles the known prions of yeast, but not any
known nucleic acid plasmid. This research will quantify the epigenetic effects
of [KIL-d] on M dsRNA by determining whether it alters the size or quantity of
viral genomic or transcript RNA or of the protein products of the virus. In
order to determine the molecular nature of [KIL-d], a genomic library from a
primary isolate in which this element has appeared after mutagenesis (available
in this laboratory) will be used to generate a genomic DNA library (or a cDNA
library), that will be screened for genes that can generate [KIL-d] when
transferred to wild-type recipients. If [KIL-d] is a prion, then this screen
should identify the gene(s) encoding the protein that forms the prion. If the
prion model is incorrect, this screen should identify the genetic elements
involved in this virus-specific epigenetic regulatory process, which appears to
be the first such process to be subjected to molecular genetic analysis. Further
studies using the genetic methods available in yeast will allow characterization
of the genes identified in this library screening procedure. This research will
provide a training opportunity for a postdoctoral fellow, graduate student and
several undergraduate students, who will be trained in the methods of yeast
molecular genetics. It will also include a new group summer research experience
for a group of summer undergraduate students participating in two programs run
on this campus with support from the NSF and an Initiative for Minority Student
Development Award from the NIH. &lt;br/&gt;&lt;br/&gt; Misfolded proteins that
can cause normal proteins of the same type to misfold are termed prions. In
yeast, prions are cytoplasmically-inherited non-nucleic acid genetic elements.
[KIL-d] is an unusual genetic element in yeast that alters expression of the
genes of a viral RNA. This research will characterize this genetic element and
determine whether it is a prion. &lt;br/&gt;