* 0321450
* Prediction and Validation of Phospho-Regulatory Sites in Crop Plant Proteomes
* BIO,IOS
* 02/01/2004,06/30/2004
* Jeffrey Harper, The Scripps Research Institute
* Standard Grant
* TEST user
* 06/30/2004
* USD 691,501.00

The genomes of vascular plants encode more than 1000 protein kinases. The long-
term goal is to&lt;br/&gt;define all the regulatory phosphorylation sites in
crop plant proteomes. Such sites represent potential control points for all
aspects of plant growth, including responses to biotic and abiotic stress. The
focus is to use bioinformatics to predict a subset of potentially important
sites of phospho-regulation in the rice proteome, and then use a peptide-based
experimental strategy to start validating those predictions. The four Specific
Aims are:&lt;br/&gt;1. Improve bioinformatics tools to identify motifs involved
in phospho-signaling in plants.&lt;br/&gt;2. Conduct a clustering analysis of
all Ser and Thr containing sub-sequences in plant proteomes.&lt;br/&gt;3. Design
peptide array platform to profile kinase activities in vitro.&lt;br/&gt;4.
Identify peptides that can disrupt in vivo signaling pathways (more than 50 will
be tested).&lt;br/&gt;The uniqueness and strength of the proposed research rests
upon the underlying rationale that a&lt;br/&gt;computational comparison of two
plant proteomes can be used as a 'noise filter' to identify&lt;br/&gt;important
sequence motifs that are fundamental to signaling pathways conserved between
monocots and dicots. This filtering provides the critical foundation for
implementing 'peptide-disruptors' as a powerful new approach to understanding
and manipulating the dynamics of the plant cell phosphorylome. DNA constructs
for all genes found to encode a 'peptide disruptor' or function as a reporter
for a particular kinase activity will be deposited at the ABRC at Ohio State
University, and will be freely available.