* 0344286
* The RNA Degradosome: Functional Consequences of Enzyme Association on RNA Metabolism
* BIO,MCB
* 03/01/2004,04/30/2007
* Christopher Burns, AUGUSTA UNIVERSITY RESEARCH INSTITUTE, INC.
* Continuing Grant
* Karen Cone
* 04/30/2007
* USD 408,000.00

Fundamental understanding of gene expression requires knowing the life cycle of
messenger RNA transcripts that transmit information from the DNA genome.
Explaining the end of this life cycle is challenging because multiple cellular
enzymes may govern transcript disassembly through independent or combinatorial
actions. The overall project goal is to define consequences of enzyme
association on RNA transcript disassembly through analysis of a multi-enzyme RNA
degrading complex from E. coli called the "degradosome". Degradosomes are mainly
assemblies of endoribonuclease RNase E, exoribonuclease PNPase, and RNA helicase
RhlB, plus the glycolytic enzyme enolase, which has no known role in RNA
metabolism. A major obstacle to fully understanding degradosome function is
defining the role enolase plays in the complex. The first aim is to resolve
whether enolase contributes to transcript disassembly, possibly through
coordination of RNA and energy metabolism. The second aim is to determine the
impact of degradosome association on individual component enzymes. RhlB will be
employed as a model to describe structural and functional alterations helicase
enzymes can undergo upon substrate and protein binding. This will identify both
the conformational changes likely responsible for the strand separation
mechanism and the consequences of RhlB interaction with degradosomes. The final
aim is to discern cellular benefits of degradosome assembly by explaining the
molecular basis for a phenotypic difference observed between wild-type cells and
mutant cells unable to assemble degradosomes. Students from groups
underrepresented in science, or of disadvantaged background, will participate in
the project as active members of the research team.