* 0800272
* Collaborative Research: Mathematical Models for RNA
* MPS,DMS
* 06/01/2008,05/31/2013
* Ming-Ying Leung, University of Texas at El Paso
* Continuing Grant
* Mary Ann Horn
* 05/31/2013
* USD 415,632.00

Ribonucleic acid (RNA) molecules play important roles in many biological
processes, including gene expression and regulation. An RNA molecule is a linear
polymer that folds back on itself to form a three-dimensional functional
structure. While experimental determination of precise RNA structures is a time-
consuming and costly process, useful information about the molecule can be
gained from knowing its secondary structure, a collection of hydrogen-bonded
base pairs. Structural elements in RNA secondary structures can be separated
into two large categories: stem-loops and pseudoknots. Development of
mathematical models and prediction algorithms for simple stem-loop structures
has started in the 1980?s. However, the tremendous demand on computer memory and
time by pseudoknot prediction remains a computing challenge even today. The
recently developed grid computing technology can offer a possible solution to
this challenge. In this project, the investigators shall address some
mathematical problems associated with the grid computing approach to RNA
secondary structure prediction. To partition a large RNA molecule to smaller
segments assigned to different computers on the grid, a good cutting strategy is
necessary. The investigators propose to develop probabilistic models to study
the inversion distribution in RNA sequences and to combine the results with the
general theory of excursions to maximize the prediction accuracy using an
optimal RNA segment length. The mathematical results will be integrated into a
toolset for computational RNA structure analysis to test their applicability.
The toolset will be used to investigate the possible association of pseudoknot
types with functions using data in public domains and applied to the prediction
of secondary structures adopted by nodavirus genomes. The prediction results
will be compared with the secondary structures experimentally determined by
mutational studies. Studies on RNA secondary structures and functions contribute
significantly to the understanding and control of RNA viruses in plant and
animal diseases. Computational methods recently used in these RNA studies have
been shown to greatly reduce time and cost. Solving computationally intensive
problems is now feasible using grid computing technology that simulates high-
performance computing on a superstructure of networked computers. By combining
rigorous mathematical methods, current computing technology, and careful
experimental verification, this project develops an interactive investigative
approach in an interdisciplinary research endeavor. The validation of
computational prediction results by wet-lab experiments will encourage
experimental scientists to make use of computing technology with confidence to
assist their scientific pursuits. Such a collaborative effort will have
significant impacts on education and training of new scientists by promoting the
concept of interdisciplinary research designs, by enhancing diversity in the
student population from different geographical and economic areas, and by
encouraging the development of long-term collaborative research and educational
partnerships between minority-serving institutions and research universities.
The work of these investigators will result in a research product for viral RNA
genome analyses with mathematical concepts useful for studying many other
patterns in genetic sequences. This project represents a direct contribution of
mathematics through the advancement of computing technologies to the elucidation
of diverse biological functions of RNA.