* 0800631
* Statistical Approaches to Integration of Mass Spectral and Genomic Data of Yeast Histone Modifications
* MPS,DMS
* 06/15/2008,05/31/2013
* Neil Kelleher, University of Illinois at Urbana-Champaign
* Continuing Grant
* Mary Ann Horn
* 05/31/2013
* USD 595,000.00

New statistical and analytical methods will be developed to study regulatory
role of histone modifications in Saccharomyces cerevisiae. Gene activities in
eukaryotic cells are concertedly regulated by transcription factors and
chromatin structure. The basic repeating unit of chromatin is the nucleosome, an
octamer containing two copies each of four core histone proteins. While
nucleosome occupancy in promoter regions typically occludes transcription factor
binding, thereby repressing global gene expression, the role of histone
modification is more complex. Histone tails can be modified in various ways,
including acetylation, methylation, phosphorylation, and ubiquitination. Even
the regulatory role of histone acetylation, the best characterized modification
to date, is still not fully understood. Mass spectral and genome-wide microarray
data from Saccharomyces cerevisiae have offered new opportunities for
investigators to evaluate the regulatory effects of histone modifications. The
investigators will develop statistical methods for identifying target genes of
histone modifications and associated DNA sequence features of histone
modifications. The investigators will also develop computational and statistical
methods for predicting histone modifications and their interactions.

Experimental data are noisy and high dimensional, which renders many tradition
statistical methods ineffective. How to build prediction models with only a
small set of informative variables adds another layer of complexity. New
statistical methods will be developed to surmount the challenges. The proposed
methods lead to a statistical framework for integrating multiple types of
proteomic and genomic data. A complete framework for such integration has not
been developed and tested in the statistics and computational biology
literature. The proposed method can produce innovative methodologies for
analyzing very large amounts of heterogeneous data, suggest new lines of
quantitative investigations in systems biology, and offer opportunities for
students to participant in inter-disciplinary research.