* 0804721
* Modeling and Evolution of Biological Networks
* MPS,DMR
* 09/15/2008,08/31/2011
* Eric Siggia, Cornell University
* Continuing Grant
* Daryl Hess
* 08/31/2011
* USD 315,000.00

&lt;br/&gt;TECHNICAL SUMMARY:&lt;br/&gt;This award supports theoretical research
and education in modeling and evolution of biological networks. The research
undertaken addresses evolution of genetic patterns using theoretical models that
represent the genetic code and the changes that are possible while selecting
changes as favorable with a ''fitness function.'' The theoretical approach draws
on analogies with learning models in Computer Science and optimization in
Statistical Physics. This theoretical machinery predicts biological networks by
how rapidly they can be learned from the random examples provided by mutation
and selection. The approach assumes the networks in living things can be built
incrementally and grow continually by stepwise increases in fitness. They are
not necessarily global optimum. Network evolution will be modeled with
simplified differential equations for the central molecular components of
developmental pathways, e.g., transcription factors, ligand-receptors
interactions, protein-protein complexes, kinases and phosphorylated proteins
etc. Evolution requires a fitness function and rapid evolution is facilitated by
a smooth monotone function, not a jagged landscape. This is in accord with the
goal of evolving patterning networks common to all animals, not specific phyla.
A plausible fitness function quantifies how well embryonic position is related
to gene expression patterns. This research follows a preliminary application of
these ideas to periodic segmentation in animals (somitogenesis) where the
earliest versions of the approach found an encouraging degree of
success.&lt;br/&gt;&lt;br/&gt;The effort undertaken has broader impacts with
both scientific and educational consequences. The research, employing the
approaches from a condensed matter physics perspective, takes place at
Rockefeller University in an environment focused on biological studies. Graduate
students involved in the research gain a unique interdisciplinary education with
the foundations of theoretical physics immersed in the research environment of
biological sciences. The research makes contributions to the scientific
community beyond publishing and the usual forms of dissemination. The
bioinformatics tools employed in the modeling and analysis are packaged into web
sites for broad dissemination. The relevance of the tools developed to real
world medicine were illustrated in a paper where this group collaborated in
tracking the evolution of drug resistant Staphylococcus aureus (''Super bugs'')
in a human patient by whole genome
resequencing.&lt;br/&gt;&lt;br/&gt;NONTECHNICAL SUMMARY:&lt;br/&gt;This award
supports theoretical research and education in modeling and evolution of
biological networks. The research models evolution of genetic patterns using
theoretical models that represent the genetic code and the changes that are
possible while selecting changes as favorable with a ''fitness function.'' The
theoretical approach draws on analogies with learning models in Computer Science
and optimization in Statistical Physics. This theoretical machinery predicts
biological networks based on how rapidly they can be learned from the random
examples provided by mutation and selection. The approach assumes the networks
deployed in living things can be built incrementally and grow continually by
stepwise increases in fitness. They are not necessarily global optimum. Network
evolution will be modeled with simplified equations for the central molecular
components of developmental pathways. This research follows a preliminary
application of these ideas to periodic segmentation in animals (somitogenesis)
where the earliest versions of the approach found an encouraging degree of
success.&lt;br/&gt;&lt;br/&gt;The effort undertaken has broader impacts with
both scientific and educational consequences. The research, employing the
approaches from a condensed matter physics perspective, takes place at
Rockefeller University in an environment focused on biological studies. Graduate
students involved in the research gain a unique interdisciplinary education with
the foundations of theoretical physics immersed in the research environment of
biological sciences. The research makes contributions to the scientific
community beyond publishing and the usual forms of dissemination. The
bioinformatics tools employed in the modeling and analysis are packaged into web
sites for broad dissemination. The relevance of the tools developed to real
world medicine were illustrated in a paper where this group collaborated in
tracking the evolution of drug resistant Staphylococcus aureus (''Super bugs'')
in a human patient by whole genome resequencing.&lt;br/&gt;&lt;br/&gt;