* 0812246
* III-CXT-Small: Analysis of Mass Spectrometry Data for Biomarker Discovery
* CSE,IIS
* 09/01/2008,08/31/2012
* Radoslav Goldman, Georgetown University
* Continuing Grant
* Sylvia Spengler
* 08/31/2012
* USD 449,072.00

Label-free quantification of analytes is gaining recognition as a very good
strategy for biomarker discovery. However, such quantification is typically not
addressed adequately in the instrument specific software packages. Biological
variability and disease heterogeneity in human populations further complicate
the MS-based biomarker discovery. The specific research aims of this project are
the following: (1) Investigate data preprocessing algorithms for label-free
quantification of peptides in serum using liquid chromatography?mass
spectrometry (LC-MS) technologies. The initial focus will be on investigating
and developing normalization, alignment, and peak detection methods to
preprocess LC-MS spectra generated by two LC-MS instruments. (2) Investigate
feature selection algorithms that can isolate subgroup-specific biomarkers; this
process will account for biological variability and disease heterogeneity among
human populations. &lt;br/&gt; &lt;br/&gt;Spike-in study will be conducted to
obtain replicate LC-MS spectra with known peptide differences. The spectra from
this study will be used to optimize the proposed data preprocessing and feature
selection algorithms, and compare them with other existing solutions. The
optimized analytical tools will be applied to select the most useful peaks for
detecting hepatocellular carcinoma (HCC) at a treatable stage. Blood samples
collected from patients with cirrhosis, HCC cases, and healthy controls in
Egypt, United States, and Thailand will be used. The peptides that correspond to
the selected peaks will be identified by sequencing the peaks using MS/MS
instruments and complementary methods. The ability of these candidate peptide
markers to detect HCC will be validated using appropriate isotope dilution mass
spectrometric assays. &lt;br/&gt; &lt;br/&gt;Novel aspects of the proposed
biomarker discovery algorithms include the capability to (1) normalize and align
spectra to allow the discovery of candidate peptide biomarkers whose
performances are not influenced by non-disease related artifacts in the data,
(2) detect peaks that are consistent with the ones manually selected by MS
experts, (3) select biomarkers through an approach that accounts for biological
variability and disease heterogeneity among subjects, and (4) provide a
framework for a complete analytical pipeline beginning from sample preparation
to validation of candidate peptide biomarkers. &lt;br/&gt; &lt;br/&gt;The
project will establish new interdisciplinary research collaboration in the areas
of bioinformatics, mass spectrometry, and tumor biology, in which faculty and
student members of the project will be cross-trained to transcend traditional
disciplinary boundaries. It will provide research-enriched learning experiences
to students through two educational activities at Georgetown University Medical
Center. &lt;br/&gt; &lt;br/&gt;Further information on the project may be found
at the project web site: http://microarray.georgetown.edu/ressomlab&lt;br/&gt;
&lt;br/&gt; &lt;br/&gt;&lt;br/&gt;