* 0817593
* Swimming, Swarming, or Sticking in Vibrio Parahaemolyticus
* BIO,MCB
* 09/01/2008,11/30/2011
* Linda McCarter, University of Iowa
* Standard Grant
* Gregory W. Warr
* 11/30/2011
* USD 449,838.00

&lt;br/&gt;Intellectual Merit &lt;br/&gt; What regulatory processes govern
whether a bacterium chooses to swim in liquid, to induce the specific swarmer
cell program in order to move over and colonize surfaces, or to become less
mobile and more adhesive in order to form biofilms? The marine bacterium Vibrio
parahaemolyticus is an excellent model organism in which to address this
question because the distinctions between the swimming, swarming, and sticky
biofilm-proficient cells are very profound for this organism. Surface-associated
growth promotes global, genome-wide reprogramming of gene expression resulting
in an altered cell type and the induction of an entirely new motility system
that is not expressed by the swimmer cell. This alternate system allows
extremely rapid movement over surfaces, which is called swarming. Growth on
surfaces can also result in robust biofilm formation. Gene sets have been
identified that are reciprocally regulated in response to growth on surfaces;
specifically motility and adhesive genes as well as the regulatory genes that
govern the expression of these gene sets. These regulatory genes, named scr,
encode potential signal transduction proteins containing GGDEF and EAL signature
domains, which possess enzymatic activities causing synthesis and degradation of
the signaling nucleotide c-di-GMP. This project focuses on the role of c-di-GMP
in programming gene expression and behavior. It is hypothesized that a specific
array of Scr signal transducing proteins, capable of detecting and integrating
diverse environmental signals by modulating the level of c-di-GMP, influences
the decision to swarm or stick. A low concentration of c-di-GMP favors surface
mobility; whereas high levels of this nucleotide promote a more adhesive and
sessile, or biofilm-proficient, cell type. This project focuses on: 1, defining
the scope of the scr c-di-GMP network by using genetic screens to detect
potential scr sensors and fluorescent fusion proteins to determine protein
localization; and 2, establishing the molecular basis of c-di-GMP control of
gene expression by examining candidate transcriptional regulators of swarming
and sticking. Thus, this research is designed to map a complex regulatory
circuit controlling processes by which bacteria sense, integrate, and respond to
multiple signals in their environment, and will provide insight into how
bacteria make decisions.&lt;br/&gt;&lt;br/&gt;Broader Impact&lt;br/&gt; These
studies will reveal general principles with respect to the developmental program
that enables bacteria to successfully colonize surfaces, as well as indicate the
specific surface-adaptation strategies utilized by V. parahaemolyticus. This
project is specifically designed to integrate teaching and research by
implementing an undergraduate-driven discovery project to examine environmental
signal input and define new output targets. This discovery plan will allow the
direct participation of undergraduate students in the research process. Students
will learn to make observations, formulate hypotheses and test their ideas; they
will also develop their writing and public speaking skills. Importantly, the
student projects are formulated in such a manner that the students will also
gain knowledge of the broad arsenal of genetic, physiological, molecular
biological, and bioinformatics-based techniques that can be applied to unravel
programs of gene control and the underlying basis for bacterial behavior.
&lt;br/&gt;