* 0817900
* Gene Expression Networks and their Regulators in a Model Perennial Plant
* BIO,MCB
* 09/01/2008,08/31/2010
* Matias Kirst, University of Florida
* Standard Grant
* Karen Cone
* 08/31/2010
* USD 250,000.00

Intellectual merit: Transcription networks may be defined as the organization
and relationships of elements that control the expression of functionally
related genes. Previous studies have established the feasibility of
characterizing the genetic regulation of gene expression based on full genome
sequence and transcription profiling of families/pedigrees. However, few studies
have attempted to describe transcription networks of genes and their regulators,
much less to define the mechanisms of this regulation. This project represents
the first level of information required to link genome and genotype to phenotype
and, through this analysis, gain an understanding of the basis of morphological
and developmental diversity of organisms. Describing these networks and their
diversity also provides insight into the role of the regulation of gene
expression in the evolution of higher plants. In this project, a comprehensive
description of the genetic regulation of gene expression will be generated for
the model perennial plant Populus trichocarpa. Transcription networks will be
identified and genetic models for their control will be generated for different
plant tissues, creating the first description of their ontogeny (developmental
regulation within a given plant), diversity (number and complexity) and
conservation both within the species and in comparison to other species.
Candidates for the regulation of these transcription networks and the mechanisms
of regulation will also be pursued. To achieve these objectives, whole-genome
microarrays will be analyzed in combination with the P. trichocarpa genome
sequence. Genetic loci that regulate gene transcript abundance variation
(expression QTLs, or eQTL) will be identified. Gene expression networks will be
defined based on coordinated regulation of functionally related genes and shared
eQTLs. In addition to transcriptional regulatory circuits, these studies have
the potential to provide information about post-transcriptional control, which
has not been studied extensively in higher plants. The raw and processed
microarray data, as well as the eQTLs and networks discovered through the
research will be made available through a new web site
(http://forestgenomics.ifas.ufl.edu/KirstLab/PoplarTranscriptionalNetworks). In
addition, this information will be integrated with the genome sequence, genetic
maps, and the Gene Ontology/Kegg functional annotation system that is available
on the Poplar Genome Initiative Portal
(www.jgi.doe.gov/poplar).&lt;br/&gt;&lt;br/&gt;Broader impacts: This project
will provide training and support to four graduate students, and two
postdoctoral associates, using innovative approaches that integrate quantitative
genetics and genomics. It will also create opportunities for high-school
students from underrepresented groups to participate in research at the
University of Florida. For the plant scientific community, this research will
establish a framework for comparisons with the architecture of transcript
regulation in monocots, other dicots, and gymnosperms. The models of genetic
regulation of individual genes and metabolic and regulatory networks will
provide support for the analysis and modification of pathway products for
improvement of phenotypes and metabolic engineering. The population that will be
used in this study has also been extensively phenotyped for growth, plant
architecture, wood quality and other valuable properties. This phenotypic
information, in combination with the analysis of gene expression networks will
be used to identify genes that regulate variation of commercially important
traits for the agricultural and forestry industry. All the data will also be
integrated with genome sequence, genetic maps, and the Gene Ontology / Kegg
functional annotation based on a visual, searchable tool.