* 0820345
* Arabidopsis 2010:   Collaborative Research:   Evolution of gene position and function in Arabidopsis using outgroup genomes
* BIO,MCB
* 02/15/2009,01/31/2013
* FRANK FELTUS, Clemson University
* Standard Grant
* Karen Cone
* 01/31/2013
* USD 158,508.00

Last Modified Date: 02/04/09 Last Modified By: Susan Porter Ridley &lt;br/&gt;
&lt;br/&gt;Abstract &lt;br/&gt;Over time, the position of a gene in its genome
often changes, but the functional consequences of such gene movements are
largely unknown. To help fill this gap, these three investigators are not only
exploiting the wealth of research tools and resources available for the model
plant Arabidopsis, along with the genome sequences for papaya, grape, and
poplar, they are also creating new ones. Focusing on Arabidopsis's 26,500 non-
transposon, non-tandem genes, and using a new top-down, multi-alignment
algorithm to infer gene order in the last common ancestor of Arabidopsis and
papaya, grape, and poplar, the investigators have found evidence for an ancient
hexaploidy shared by all dicots and perhaps all plants. Additional preliminary
data show that about 40% of Arabidopsis genes are no longer ancestral, either
because they (1) have diverged rapidly; (2) exhibit a high birth-and-death
behavior in the phylogenetic tree; (3) are positioned in rearrangement-prone
regions of the chromosome; or (4) have transposed to new locations in the
genome. Interestingly, a "flanking gene test" has shown that certain gene
families contain genes that are almost always newly transposed. The
investigators also show that conserved noncoding sequences sometimes
subfunctionalize among homeologs. Over 4,000 deeply conserved noncoding
sequences (CNSs) are shared between papaya and Arabidopsis. These regulatory
sites will be sorted to gene, made into a database, and evaluated for
subfunctionalization at the level of sequence, gene expression, and fitness
(reproductive output) through the use of knockout mutations and phenotypic
analyses. &lt;br/&gt;In addition, using microarray data normalized by The
Arabidopsis Information Resource (TAIR), the investigators will test, in silico,
hypotheses concerning issues such as the relationship between genes' positions
and their coregulation. &lt;br/&gt;&lt;br/&gt;Broader Impacts of Project. This
project provides exciting training opportunities at both pre- and postgraduate
levels. The investigators will continue their long-established partnerships with
Chevron on the SEED project, and with Berkeley High School on improving biology
training for promising, underserved minority high school students. The Freeling
laboratory provides these students with paid summer research experiences
tailored to the needs of the students and their families. The University of
Georgia ranks sixth in the US in percentage of minorities, and research there is
closely tied to strong outreach programs. This project will contribute to the
development and implementation of research-based educational materials to
improve comprehension and mastery of scientific concepts by middle and high
school students through training of one teacher per year in the GIFT (Georgia
Internships for Teachers) program. Together with GIFT, the investigators seek to
nucleate a "teachers-teach-teachers" program that uses metaphorical examples and
dialogue to portray key scientific concepts without becoming ensnarled in the
debate on creationism, an issue prominent in many areas, including Georgia and
South Carolina. In addition, exchanges between the three institutions will
enhance the scientific and cultural experiences of these investigators students
and add further to the bonds formed by research collaboration. The investigators
on this project will communicate with the public and provide data and
applications for download at their NSF 2010 project-specific website,
http://www.plantgenome.uga.edu/outgroup.html .