* 0820005
* A Medicago Truncatula HapMap as a Platform for Exploring the Genetics of Legume Symbioses
* BIO,IOS
* 03/01/2009,02/28/2013
* Nevin Young, University of Minnesota-Twin Cities
* Continuing Grant
* Diane Jofuku Okamuro
* 02/28/2013
* USD 5,689,397.00

PI: Nevin D. Young (University of Minnesota)&lt;br/&gt;&lt;br/&gt;CoPIs: Michael
J. Sadowsky and Peter L. Tiffin (University of Minnesota), Maria J. Harrison
(Boyce Thompson Institute for Plant Research), Betsy M. Martinez-Vaz (Hamline
University), and Joann Mudge (National Center for Genome
Resources)&lt;br/&gt;&lt;br/&gt;Collaborators: Gregory D. May (National Center
for Genome Resources)&lt;br/&gt;&lt;br/&gt;Legumes, the third largest family of
flowering plants, are notable for their ability to form symbiotic relationships
with rhizobia bacteria. This symbiosis leads to massive amounts of biological
nitrogen worldwide, providing a major source of organic fertilizer and vegetable
protein for humans and animals. Medicago truncatula is a widely studied model
species for legume genomics and one important question focuses on the identity
of naturally occurring genes that control variation in symbiosis in legumes.
This project will use association mapping techniques to create a Medicago
"HapMap". In brief, 384 diverse genetic lines obtained from collaborators at
INRA-Montpellier, Ecole National Superieur Agronomique de Toulouse (ENSAT) and
the Noble Foundation will be resequenced using next generation sequencing
technology for sequence polymorphisms (SNPs) between the different Medicago
lines. SNP discovery through genome resequencing is possible because a reference
sequence for the gene-rich euchromatin of Medicago has already been created
through previous NSF funding. The massive database of SNPs between Medicago
lines enables the prediction of genome segments with shared ancestry
(haplotypes), which can then be associated statistically with trait variation in
symbiosis. Because of the exceedingly high level of SNP density, association
mapping can approach the resolution of a single gene.&lt;br/&gt;&lt;br/&gt;The
broader community of scientists will access the Medicago hapmap platform through
an intuitive web interface (to be established) that will be modeled on the human
hapmap website (www.hapmap.org). Seed for all lines included in the hapmap will
be available by contacting project personnel or directly through INRA-
Montpellier, ENSAT and the Noble Foundation (through the GRIN facility). With
access to the data and germplasms, scientists will be able to carry out their
own association mapping experiments, including traits other than symbiosis.
Because the development of a hapmap platform applied to the study of symbiosis
in legumes is a naturally cross-disciplinary project, high school, undergraduate
and graduate students will find numerous projects that integrate their knowledge
of plant biology, microbiology, population genetics and bioinformatics. This
project will partner with Hamline University to develop new undergraduate
curriculum and will also reach out to undergraduates coming from Hamline
University and the University of Puerto Rico, as well as high school students
coming from institutions throughout the state of New Mexico to participate as
summer interns in projects directly involved in the Medicago hapmap and
symbiosis project. These research interns will be complemented by an integrated
group of graduate students spanning multiple graduate programs who are co-
advised by a co-PI faculty in academic departments all the way from
biostatistics to plant pathology and soil science.