* 0820047
* Arabidopsis 2010:  Global Analysis of Translational Regulons
* BIO,MCB
* 03/01/2009,02/28/2013
* Frans Tax, University of California-Riverside
* Continuing Grant
* Karen Cone
* 02/28/2013
* USD 1,754,310.00

This project will make the first comprehensive effort, for any eukaryote, toward
defining genes whose expression is regulated by the general translation
initiation machinery. Evidence indicates that gene-specific regulation of
translation initiation is widespread, phylogenetically conserved, and occurs
under a variety of environmental and developmental conditions. Gene-specific
pathways of translational control are disrupted by mutations in individual
subunits of the eukaryotic initiation factor (eIF) complexes, but there is no
comprehensive information about the extent of translational regulation and its
dependence on specific eIFs. Building on our preliminary data and complementary
expertise, this project will define groups of genes whose mRNAs are co-dependent
on specific translation factors, i.e. "translational regulons". This work will
provide the first genome-wide overview of the network of translational
regulatory pathways through the pursuit of three specific
aims:&lt;br/&gt;&lt;br/&gt;1) Determine the developmental, physiological, and
stress-related phenotypes of eif mutants. The metabolic and developmental
pathways impacted by the loss of a specific translation factor will be assessed
at the level of gross morphology and development, responses to an abiotic stress
(i.e., heat), light, and photosynthesis.&lt;br/&gt;&lt;br/&gt;2) Identify
candidate client mRNAs whose translation is controlled by specific eIFs. DNA
microarray analysis on polysomal RNA from eif mutants will be performed to
identify client mRNAs for each eIF. Detailed bioinformatic analyses of the
client mRNAs will be used to define translational regulons and to predict mRNA
structural and/or sequence features that may influence translational
regulation.&lt;br/&gt;&lt;br/&gt;3) Validation of eIF and client mRNA
translational regulation. The requirement of an eIF in the translation of a
subset of client mRNAs will be validated using in vivo mRNA-reporter constructs.
An in vitro translation lysate will be developed from Arabidopsis to be used in
the analysis of the eIF-dependence of client mRNAs in a homologous in vitro
translation assay.&lt;br/&gt;&lt;br/&gt;With respect to broader impacts, this
project will dramatically increase our understanding of translational control in
plants and as the first genome-wide effort for any eukaryote and will make a
considerable contribution to plant science and to the greater scientific
community. Taking advantage of the fact that UC-Riverside, UT-Austin, UT-
Knoxville, and U-Arizona have substantial representation of minority students,
the project will involve a robust educational component emphasizing the
involvement of minority graduate and undergraduate students in scientific
research, acting to encourage minority students to pursue a career in scientific
research, education, or policy. Graduate and post-doctoral students will be
engaged in full-time research on the project. Undergraduate students will be
involved in cross-disciplinary training through the NSF-supported Freshman
Research Initiative (UT-Austin), which provides freshmen a significant research
experience, or the Undergraduate Biology Research Program (U-Arizona) that
places undergraduate students into research laboratories. The project will also
partner with federally-funded programs that promote science training for
minorities including the NSF-funded California Alliance for Minority
Participation in Science, Engineering and Mathematics (CAMP-UCR) that works to
double the number of minority students receiving a degree in science; the
Copernicus Project, a U.S. Department of Education program that seeks to
increase substantially the number, quality and diversity of science teachers
(UC-Riverside); and the NSF-funded Louise Stokes Alliance for Minority
Participation, which brings minority students from other U-Tennessee system
schools to do summer research. In addition, each laboratory will use summer REU
supplements to provide research experience to additional undergraduate students.
The proposed research includes research approaches that are suitable for the
involvement of students at every level (high school, undergraduate, intern,
graduate and post-doctoral), thus providing a positive research experience while
producing scientifically literate citizens as well as generating extensive
information about the role of translational machinery in regulating gene
expression. All project data and biological resources can be accessed through
FIAT, the project website (http://research.cm.utexas.edu/kbrowning/fiat/).
Mutants and associated phenotypic data and images will also be available through
the Arabidopsis Biological Resource Center while microarray data will be
deposited and available through the NCBI Gene Expression Omnibus (GEO,
http://www.ncbi.nlm.nih.gov/geo/). &lt;br/&gt;&lt;br/&gt;