* 0820729
* Genetic and Genomic Approaches to Understanding the Role of Auxin in Shoot Development
* BIO,IOS
* 10/01/2008,02/28/2011
* Simon Malcomber, Pennsylvania State Univ University Park
* Continuing Grant
* Diane Jofuku Okamuro
* 02/28/2011
* USD 2,823,793.00

PI: Paula McSteen (The Pennsylvania State University)&lt;br/&gt;CoPI: Simon
Malcomber (California State University - Long Beach)&lt;br/&gt;CoPI: Andrea
Gallavotti (University of California, San Diego)&lt;br/&gt;CoPI: Yunde Zhao
(University of California, San Diego. &lt;br/&gt;Senior Personnel: Naomi Altman
(The Pennsylvania State University)&lt;br/&gt;Senior Personnel: Reka Albert (The
Pennsylvania State University)&lt;br/&gt;Collaborator: Robert Schmidt
(University of California, San Diego)&lt;br/&gt;&lt;br/&gt;The hormone auxin
regulates almost every aspect of plant growth and development, including leaf
and flower formation, and stem and flower cluster (inflorescence) branching. The
long-term goal of this project is to understand how shoot (the above ground
portion of a plant) growth is regulated through pathways of auxin biosynthesis,
transport and signal transduction, and the extent to which these mechanisms and
interactions are conserved among the approximately 250,000 described species of
flowering plants. Genes that function in auxin mediated shoot development will
be identified and characterized using a combination of high throughput forward
genetics in maize and comparative expression profiling in both maize and
Arabidopsis. Reverse genetic analyses will be used to uncover the function of
genes in these pathways that may not be apparent from forward genetic approaches
alone. Gene function will be analyzed using transgenic reporter lines coupled
with phenotypic, expression and genetic analyses of characterized mutants to
assemble a network of genetic interactions. A gene regulatory network model will
be built to integrate the data from expression profiling together with genetic
and biochemical interaction studies. Phylogenetic characterization of identified
gene families using available genomic data and genes isolated during the course
of the research will provide a framework and series of testable hypotheses
concerning the conservation and diversification of auxin mechanisms in all
flowering plants. The findings from this project will lead to a better
understanding of how auxin controls shoot growth and development in
agronomically and economically important crops.&lt;br/&gt;&lt;br/&gt;Through a
series of training and outreach programs this project will provide opportunities
for high school students, undergraduates, graduate students, and postdoctoral
researchers to learn and apply a broad range of genetic, developmental,
evolutionary, genomic, statistical, mathematical and bioinformatic tools. Annual
summer workshops and exchange programs will augment inter-disciplinary plant
science training and dialog across fields. Results from the project will be
incorporated into the Botanical Society of America's Planting Science program
(http://www.plantingscience.org/) to create online open-ended, inquiry-based
educational materials focusing on the role of auxin and the interdisciplinary
nature of plant biology for high school students across the nation. Materials
from this project will provide new resources and tools to investigate the role
played by auxin in plant growth and development across flowering plants, and new
information and strategies for improving crop yield in diverse crop species.
&lt;br/&gt;&lt;br/&gt;Project data will be made available through the project
website (http://www.auxinevodevo.org). In addition, mutant stocks will be
available from the Maize Coop Stock Center (http://maizecoop.cropsci.uiuc.edu/),
phenotypic and genetic data including map locations will be deposited at
MaizeGDB (http://www.maizegdb.org/), expression profiling data will be deposited
at the Plant Expression Database (http://www.plexdb.org/), phylogenetic trees
will be deposited at TREEBASE (http://www.treebase.org) and sequences will be
deposited in Genbank(http://www.ncbi.nlm.nih.gov/) and Plant GDB
(http://www.plantgdb.org).