* 0821893
* Regulation of Histone H4-K20 Methylation
* BIO,MCB
* 09/01/2008,08/31/2012
* Craig Mizzen, University of Illinois at Urbana-Champaign
* Continuing Grant
* Karen Cone
* 08/31/2012
* USD 490,000.00

Project Summmary: If the DNA from all of the chromosomes in a single human cell
were joined end to end, the genome would span approximately 2 meters. In
eukaryotic cells, the DNA is tightly associated with proteins called histones
that allow it to be extensively folded upon itself to form a polymer called
chromatin. The compaction of DNA afforded by wrapping around a complex of
histones allows the entire genome to be packed into the microscopic confines of
the nucleus of each cell. However, for individual genes to be expressed, for
damaged DNA to be repaired, and for the genome to be replicated prior to cell
division, regions of chromatin must be unfolded so that the relevant cellular
machinery can gain access to the DNA. Conversely, chromatin must be condensed to
form chromosomes during cell division to ensure that daughter cells each receive
a complete copy of the genome. The addition and removal of small chemical groups
or "modifications" to histones are central to both chromatin folding and
unfolding. A wide variety of histone modifications have been described, but
little is known about the molecular mechanisms involved in mediating their
function or how the levels of individual modifications are controlled. This
project will investigate how the levels and genomic localization of histone
H4-lysine 20 methylation are regulated. Lysine 20 can be unmodified or modified
by the addition of one, two or three methyl groups. The enzyme PR-Set7 mediates
monomethylation, whereas Suv4-20 mediates both di- and trimethylation. Decreased
levels of PR-Set7 are associated with increased DNA damage and defective cell
division, whereas depletion of Suv4-20 alters cellular responses to DNA damage
and increases genomic instability. Both enzymes are activated during cell
division and it is likely that their activities are coordinately regulated by
dynamic modifications or interactions with other proteins. State-of-the-art
approaches will be used to identify proteins associated with PR-Set7 and
Suv4-20, characterize the modifications affecting the activities of these
enzymes and associated proteins, and ascertain the significance of the PI's
findings in human and Drosophila cells. Understanding the molecular regulation
of these enzymes will impact our understanding of the roles of chromatin
structure in cell cycle progression, detection and repair of damaged DNA,
mitosis, and other processes that are critical for normal cell function.
&lt;br/&gt;&lt;br/&gt;Broader Impacts: This work will enhance our understanding
of an epigenetic pathway involved in aspects of chromatin function that are
fundamental to normal cell biology and the development of multicellular
organisms. The cell lines, antisera and other reagents created during this
project will be useful to labs investigating other aspects of chromatin function
and will be made readily available. In addition, this work demonstrates how
proteomic methodologies can be used in conjunction with cell biology and
molecular biology approaches to make paradigm-shifting discoveries in the life
sciences. The unique training that graduate and undergraduate students receive
in this project will enable them to apply these powerful strategies in their own
careers.