* 0822383
* Efficient Identification of Induced Mutations in Crop Species by Ultra-High Throughput DNA Sequencing
* BIO,IOS
* 09/15/2008,08/31/2012
* Jorge Dubcovsky, University of California-Davis
* Standard Grant
* Diane Jofuku Okamuro
* 08/31/2012
* USD 1,966,061.00

PI: Luca Comai (University of California - Davis)&lt;br/&gt;CoPIs: Jorge
Dubcovsky (University of California - Davis) and Steven Henikoff (Fred
Hutchinson Cancer Research Center, Seattle)&lt;br/&gt;Collaborator: Robert Tran
and Dawei Lin (University of California - Davis). &lt;br/&gt;&lt;br/&gt;The
growing information on gene sequence and function stimulates interest in
pathways expected to have large effects on important agronomic and quality
traits. Nevertheless, while more and more genes are valuable inactivation
targets, approaches for crop gene modification that can flexibly target 50 to
3000 genes per crop species are limited. This project will apply ultra-high
throughput sequencing to TILLING (Targeting Induced Local Lesions IN Genomes), a
general method for the discovery of induced mutations in genes of interest.
Working with rice and wheat TILLING lines, it will employ PCR-mediated
amplification from pooled genomic templates to target dozens to hundreds of
genes. The research will test and implement several technical and computational
methods to resolve real mutations from errors in highly complex templates. To
target hundreds to thousands of genes, the project will use sequence barcoding
to create DNA pools of short adapter-ligated random fragments from hundreds or
thousands of existing TILLING lines of Arabidopsis, and later rice and wheat.
Aliquots from an amplified pool will be subjected to a capture procedure using
custom programmable arrays or long oligos attached to either microarrays or
beads, and the eluted fragments subjected to Solexa sequencing. The project will
build an economical and effective pipeline that will be sustainable as a user-
supported service, superseding more laborious and expensive nucleotide mismatch-
detection technology. Rice and wheat span the range of critical characteristics
and can therefore serve as models for all crops. They differ in ploidy, diploid
versus allopolyploid, sequence information, high versus low, genome size, small
versus large, and mutation density, moderate versus very high. While developing
a sequencing approach to TILLING, the project will use already existing and
tested populations to target a set of genes in each crop that will be useful to
the community of scientist and breeders. Thus, this research will facilitate
breeding of improved crop varieties by providing facile isolation and analysis
of variants. &lt;br/&gt;&lt;br/&gt;This research will provide methods for
efficient TILLING of crop plants. By doing so, it will stimulate both basic
discovery as well as targeted trait improvement. For example, in wheat
independent inactivation of homeologous gene sets followed by breeding can
address a host of important agronomic and quality traits. Because discovery of
mutations by direct sequencing will allow researchers to choose ab initio which
mutant to characterize, in silico collections of induced mutations will provide
powerful resources for breeding communities. The developed methods will not only
be useful for TILLING but also to discover natural variation, or to identify
allelic changes underlying quantitative trait loci. The connected outreach
program combines the undergraduate and professional training. It is based on
three approaches: an internship program that will pair students and teachers
from underrepresented populations with laboratory-to-field experimental genomic
pipeline, a database and methods web site, and a yearly workshop for training
crop scientists. Data and methods will be accessible to the public through the
project web site (http://tilling.ucdavis.edu) and through established genomics
databases (http://www.gramene.org/; http://www.arabidopsis.org/).