* 0827055
* High-Throughput Small RNA Sequencing in Archaea
* BIO,EF
* 09/01/2008,08/31/2012
* Todd Lowe, University of California-Santa Cruz
* Standard Grant
* Matthew Kane
* 08/31/2012
* USD 809,428.00

It was only 30 years ago that scientists discovered that some micoorganisms
constituted a new "Domain" of life, the Archaea. Today we know that they
represent as much as one third of the diversity of life on Earth. These microbes
appear indistinguishable from bacteria under the microscope, but at the
molecular level they are actually more closely related to plants and animals.
Diverse archaeal species are notable for being able to thrive in extremes of
temperature (boiling hot springs, within Antarctic ice), high pressure, acidic
pH, or Dead Sea-like high salt conditions. Other archaea inhabit the soil, human
and animal digestive tracts, and all depths of the ocean. In spite of their
major contributions to our biosphere, we currently have very little
understanding of how their genes are used and regulated. High throughput
sequencing of microbes has focused almost exclusively on decoding genomic DNA,
the genetic instructions for life. However, sequencing a related genetic
material from cells, RNA, gives a rich source of information for when genes are
used, and how they might be regulated. This approach has revolutionized our
understanding of biology through the discovery of "microRNAs" and "RNA
interference" and how they control all aspects of cell function. This project
seeks to apply the latest new gene sequencing technology to discover thousands
of unknown RNA genes across a broad sampling of 24 key species of Archaea. The
new data will also enable creation of new probabilistic models for new RNA gene
families, and vastly improve automated RNA gene annotation for newly decoded
species.

This project will catalyze many areas of archaeal RNA gene research, including
discovery of: novel structural and regulatory RNAs, the mechanism and evolution
of Clustered Regularly Interspaced Short Palindrome Repeat (CRISPR) cell
immunity, RNA gene processing motifs, and locating genes regulated by RNA
"antisense" transcripts. Nearly a dozen collaborating labs will be involved.
Graduate students in the lab will be trained in the integration of cutting edge,
high-throughput sequencing and bioinformatic approaches to RNA analysis.
Collection of new RNA gene data across 24 species, all within the Archaeal
Genome Browser comparative framework (developed in the same lab), will stimulate
pooling of knowledge, exchange of ideas between groups, and inspire many new
research directions in archaeal biology. Ultimately, a better understanding of
gene regulation in archaea will enable other groups to study their effect on the
biosphere, and potentially engineer useful new properties to address issues as
fundamental as global warming and remediation of environmental pollution.