* 0835494
* III-CXT: Learning from graph-structured data: new algorithms for modeling physical interactions in cellular networks
* CSE,IIS
* 08/15/2007,07/31/2011
* Christina Leslie, Sloan Kettering Institute For Cancer Research
* Continuing Grant
* Vasant G. Honavar
* 07/31/2011
* USD 788,325.00

III-CXT: Learning from graph-structured data: new algorithms for
&lt;br/&gt;modeling physical interactions in cellular
networks&lt;br/&gt;&lt;br/&gt;The complex behavior of the cell derives from an
intricate network of &lt;br/&gt;molecular interactions of thousands of genes and
their products. &lt;br/&gt;Understanding how this network operates and
predicting its behavior &lt;br/&gt;are primary goals of biology and have broad
implications for life &lt;br/&gt;science, medicine and
biotechnology.&lt;br/&gt;&lt;br/&gt;The genomic information revolution of the
last ten years has enabled &lt;br/&gt;new systems-level and data-driven
approaches for studying cellular &lt;br/&gt;networks. In particular, using
machine learning to model gene &lt;br/&gt;regulatory networks---the switching on
and off of genes by regulatory &lt;br/&gt;proteins that bind to non-coding DNA
---has emerged as a central &lt;br/&gt;problem in systems biology. Now, an
explosion of new high-throughput &lt;br/&gt;technologies for measuring physical
interactions between proteins and &lt;br/&gt;between protein and DNA provides a
new data integration challenge for &lt;br/&gt;computational modeling of gene
regulation. These new data can all be &lt;br/&gt;viewed as graph-structured
data, or physical interaction networks.&lt;br/&gt;&lt;br/&gt;The central
computational goal of this project is to develop new &lt;br/&gt;machine learning
learning algorithms for exploiting graph-structured &lt;br/&gt;data, including:
(1) boosting with efficient graph mining; (2) graph &lt;br/&gt;kernels based on
subgraph histogramming; and (3) information-based &lt;br/&gt;graph partitioning.
These new algorithms will be used to integrate &lt;br/&gt;physical interaction
network data into models of gene regulation in &lt;br/&gt;order to better
represent underlying biological mechanisms. The &lt;br/&gt;focus will be two
fundamental modeling problems: inferring signal &lt;br/&gt;transduction pathways
and modeling cis regulatory modules at the &lt;br/&gt;level of DNA sequence and
interacting regulatory proteins. The &lt;br/&gt;algorithms will be applied both
to publicly available data and to &lt;br/&gt;primary gene expression data
provided by one of the investigators to &lt;br/&gt;study the hypoxia in yeast
and the response to environmental toxins &lt;br/&gt;in mammalian neural
cells.&lt;br/&gt;&lt;br/&gt;This project will learn systems-level models that
lead to new insight &lt;br/&gt;into the underlying mechanisms of gene regulation
and open the way to &lt;br/&gt;broader biological discoveries. All data, results
and source code &lt;br/&gt;will be publicly available via the Web
(http://www.cs.columbia.edu/ &lt;br/&gt;compbio/cellular-networks) and
disseminated through courses and &lt;br/&gt;bioinformatics software packages.
The project will also create &lt;br/&gt;undergraduate research opportunities for
joint dry and wet lab &lt;br/&gt;projects and outreach activities to introduce
New York City public &lt;br/&gt;high school students to new interdisciplinary
areas of science.&lt;br/&gt;