* 0845381
* CAREER:   Computational Analysis and Prediction of Genome-Wide Protein Targeting Signals and Localization
* BIO,DBI
* 08/01/2009,07/31/2014
* Jianjun Hu, University South Carolina Research Foundation
* Standard Grant
* Anne Maglia
* 07/31/2014
* USD 579,818.00

(This award is funded through the American Recovery and Reinvestment Act of
2009: Public Law 111-5). &lt;br/&gt;&lt;br/&gt;This is a CAREER award to support
the research of Dr. Jianjun Hu, in the &lt;br/&gt;Department of Computer Science
and Engineering at University of South Carolina. Dr. Hu is a second-year,
tenure-track Assistant Professor.&lt;br/&gt;&lt;br/&gt;A typical cell has a size
of only 10 microns while it contains about a billion proteins. How these
proteins are transported from their synthesis sites to their target locations
within or outside of the cell is still not well understood. Experiments showed
that translocation of nascent proteins are usually guided by postal code-like
targeting signals encoded within the amino acid sequences of proteins. Genome-
wide identification and decoding of these so-called molecular zip codes are
fundamental to comprehensive understanding of the cell. Experimentally
identifying protein targeting signals is labor-intensive. Computational
prediction of targeting signals is still a big challenge due to their low
conservation at the amino acid level. Currently, no de novo discovery algorithm
is available for identifying new protein targeting signals. Also missing are
appropriate models and algorithms for comparing these signals. This grant is 1)
investigating novel computational algorithms for de novo discovery of new
protein targeting signals; 2) developing models and algorithms for representing,
detecting, and comparing targeting signals and 3) developing a protein
functional network-based integrative algorithms for protein localization
prediction. A transformative result of these studies will be a sequence encoding
scheme based on amino acid indexes. This scheme will convert protein sequences
into sequences of amino acid groups (AAGs) such that conserved patterns can be
represented, modeled and discovered. Finally, protein function networks will be
derived from models of protein localization prediction. With this research,
computational identification and decoding of genome-wide protein targeting
signals and precise protein localization predication will greatly enhance the
understanding of how proteins are assembled in a cell. Tools developed during
this project will be made available on the lab website:
http://mleg.cse.sc.edu&lt;br/&gt;&lt;br/&gt;As a part of his CAREER grant, Dr.
Hu will conduct short-term projects and student-run seminars to bring
undergraduates into the bioinformatics research. A special effort will be made
to change the perception that computer science is debugging code, as perceived
by many high-school students. A novel computer game will be employed to show how
bioinformatics addresses real-world problems. This will raise the public and
especially the awareness and interest of K-12 students in bioinformatics.
Students in the NSF STARTS Alliance program at the University of South Carolina
will be targeted for students. Mini programming problems with a bioinformatics
background will be developed for lower-level college students so that they will
be exposed to bioinformatics early in their introductory programming courses.
This project will also develop bioinformatics web services for de novo
discovery, comparison, and retrieval of protein targeting signals and precise
protein localization prediction.