* 0848085
* Fidelity of Transcription by RNA Polymerase II
* BIO,MCB
* 04/01/2009,03/31/2014
* David Peterson, Texas A&M Research Foundation
* Continuing Grant
* Karen Cone
* 03/31/2014
* USD 449,998.00

This project seeks to understand the ways that cells maintain the fidelity of
transcription, the first step in the utilization of genetic information.
Transcription of protein-coding genes is catalyzed by RNA polymerase II (pol
II), an enzyme that contains 12 different protein components in the model
organism Saccharomyces cerevisiae (baker's yeast). Maintaining transcriptional
fidelity is important, as even a small population of altered RNA transcripts
could have very negative consequences for the cell, such as production of
dominant negative proteins that could disrupt other cellular functions.
Preliminary results have demonstrated that Rpb9, a small, non-essential subunit
of pol II, is required for accurate transcription. The research is designed to
answer the following questions: How does Rpb9 contribute to transcriptional
fidelity? Is error-prone transcription an inherent property of Rpb9-deficient
pol II? What other proteins affect transcriptional fidelity?
&lt;br/&gt;&lt;br/&gt;These questions will be addressed with three approaches.
The first approach will examine the transcriptional properties of Rpb9-deficient
pol II. Transcriptional activity will be examined in vitro to directly assess
the rates of NTP misincorporation and extension after misincorporation, as well
as the rate of nucleolytic proofreading of a misincorporated nucleotide.
Potential effects of Rpb9 on the Km for incorporation of correct and incorrect
nucleotides will also be explored. Preliminary studies have established that
when Rpb9 is missing from pol II, the rate of misincorporation of UTP for a
templated CTP is increased, and, furthermore, that the misincorporated U at the
3'-end of the nascent RNA is less likely to be excised by proofreading. A second
experimental approach will identify amino acids or domains of Rpb9 that are
important for maintaining fidelity. Mutations will be engineered at sites
designed to disrupt function or alter interactions with other proteins, and an
alternative, unbiased strategy will be exploited to generate random mutations in
RPB9. Altered functions resulting from these mutations will be assessed both in
vivo and in vitro. The third approach will identify other genes important for
transcriptional fidelity based on synthetic lethality when they are deleted in
the context of a deletion in RPB9 or by their ability to suppress mutations in
RPB9. At least one gene, which encodes a subunit of the chromatin remodeling
coactivator SAGA, has been shown in preliminary work to have an effect on
transcriptional fidelity in vivo. &lt;br/&gt;&lt;br/&gt;Broader Impacts: From a
broader perspective, the proposed research will not only generate new knowledge,
but it will also serve to train graduate and undergraduate students, including
participants in an NSF-sponsored REU program. The research activities at large
public universities are their most notable and under-utilized asset for
providing learning experiences for students, and the PI has a long history of
training undergraduate and high school students in his lab.