* 0850214
* Stress-Induced DNA Duplex Destabilization in Eukaryotic Genomes
* BIO,DBI
* 09/01/2009,08/31/2015
* Craig Benham, University of California-Davis
* Standard Grant
* Jennifer Weller
* 08/31/2015
* USD 1,093,549.00

The University of California Davis is awarded a grant to study the propensity of
the DNA duplex to become destabilized under the superhelical (primarily
torsional) unwinding stresses that occur in living organisms. The project builds
on previous work which developed computational methods to predict the locations
of destabilized regions and their extents of destabilization within a
superhelically constrained DNA molecule of specified sequence. This approach has
been applied to prokaryotic and archaeal genomes, where superhelicity has long
been known to play key roles in the regulation of numerous normal and
pathological processes. This project will analyze the stress-induced
destabilization properties of the genomic sequences of model eukaryotes
including human, mouse, Drosophila, nematode and Arabidopsis, as well as partial
sequences from many other eukaryotic organisms. &lt;br/&gt;&lt;br/&gt;For each
organism the project will find all statistically significant associations
between easily destabilized sites and many types of regulatory regions including
promoters, terminators, enhancers, silencers, introns, exons, replication
origins, regulatory protein binding sites and others. The team will analyze the
stress-induced destabilization properties in putative gene regulatory regions,
including 5' and 3' gene flanks, untranslated regions, and first introns. The
researchers will compare the results found for genes that are related in various
ways - by biochemical function of their products, coordinately expressed genes,
cancer-associated genes, etc. to identify the processes in eukaryotes where
stress induced destabilization of DNA may play a role. The project will compare
destabilization properties with expression profile results (where available) to
determine which sets of genes may be regulated by destabilization-based
mechanisms. Available sequence data will determine how destabilization
properties are evolutionarily conserved among gene orthologs in different
strains of the same species, among different species in the same genus, and
among more widely diverged species. This information will be used to develop
predictors of each genomic feature found to have characteristic destabilization
properties. The project will develop and implement strategies for analyzing
destabilization induced by dynamically changing superhelicity, as occurs through
transcription. There is evidence that a second type of superhelically driven DNA
structural transition, to the so-called G4 quadruplex, also can occur in
eukaryotes. To model how this transition may compete with stress-induced
destabilization, equilibrium statistical mechanical methods will analyze the
competition between these transitions in DNA regions that contain sequences
susceptible to both forms of destabilization. Further information about this
project and access to SIDD data may be found at
http://genomics.ucdavis.edu/benham.