* 0850356
* Collaborative Research:  PHANTOME:  PHage ANnotation TOols and MEthods
* BIO,DBI
* 04/15/2009,03/31/2013
* Robert Edwards, San Diego State University Foundation
* Continuing Grant
* Julie Dickerson
* 03/31/2013
* USD 903,361.00

Viruses are the most abundant biological entities on the planet. Since the most
abundant living organisms on Earth are bacteria, the majority of these viruses
are phages, the viruses which infect bacteria. Through their diverse lifestyles
and gene products, phages play important roles in horizontal gene exchange, in
structuring natural microbial communities, and in global biogeochemical cycles.
Phages carry genes for some of the deadliest toxins known and can also carry
genes which confer adaptive advantages to the hosts they infect. Furthermore,
phage genes and the proteins they encode are the outcome of evolution over eons,
the products of which we would be able to exploit if only we could decode the
information in the phage DNA sequences. The number of available phage genome
sequences is increasing rapidly; on the other hand, they represent the largest
global reservoir of uncharacterized genetic material. Bioinformatic tools
necessary for interpreting this data has lagged behind the growth in genome
sequences. Grants to develop a platform and toolbox of computational tools for
phage genome analysis have been awarded to support collaborative research in the
laboratories of Drs. Robert Edwards, Department of Computer Sciences, San Diego
State University, Mya Breitbart, College of Marine Sciences, University of South
Florida, Jeffrey Elhai, Biology Department, Virginia Commonwealth University and
Matthew Sullivan, Department of Ecology and Evolutionary Biology, University of
Arizona. Dr. Elhai is an Associate Professor, the other three investigators are
Assistant Professors. This collaborative project is creating new computational
tools to establish a consistent nomenclature for phage genomes, to annotate
phage sequences, both from completely sequenced phage genomes and from
environmental phage metagenome sequences. Most importantly, this project will
engage a wide spectrum of researchers, regardless of their computational
background, to access the wealth of information contained in phage genomes
through familiar graphical interfaces. These collaborators have developed an
extensive and far-reaching education plan that targets high school students,
undergraduate students and graduate students. The students trained in the use of
the tools will rotate into trainer roles via user forums and workshops. The
postdocs will be working across all the labs and thereby gain an unparalleled
panoramic view of phage biology.

