* 0204674
* Statistical Problems in Hidden Markov Modeling for Biology and Chemistry
* MPS,DMS
* 07/01/2002,06/30/2006
* Jun Liu, Harvard University
* Continuing Grant
* Grace Yang
* 06/30/2006
* USD 325,771.00

Proposal ID: DMS-0204674&lt;br/&gt;PI: Jun S. Liu&lt;br/&gt;Title: Statistical
problems in hidden Markov modeling for biology and
chemistry&lt;br/&gt;&lt;br/&gt;Abstract&lt;br/&gt;&lt;br/&gt;With the completion
of genomes of many species and the advances of microarray technologies,
biological researchers begin to possess a tremendous amount of data --- but
these "raw products" are still far from usable. One of the most challenging
problems of this century is to decipher this huge amount of biological
information, turning the data into knowledge. Simultaneously, there also has
been a revolution in chemistry research: scientists can now use advanced
technology to make observations on single-molecule dynamics, which promises to
rewrite some fundamental laws in physics and chemistry derived from traditional
ensemble-averaged experiments. As the data concerning molecular movements are
inherently noisy, the development of advanced statistical tools for handling
such data is a pressing need. The past decade has witnessed the power of formal
statistical modeling, especially the use of hidden Markov models, in
revolutionizing the field of computational biology. &lt;br/&gt;&lt;br/&gt;It is
the investigators' belief that using proper statistical models to describe the
underlying chemical processes and to derive efficient inference methods can also
greatly strengthen the data analysis in single-molecule studies. For the
biological information analysis, the investigators describe a few problems
related to the statistical models used for finding motifs, whose solutions can
deepen the understanding of a few popular Bioinformatics algorithms for sequence
analysis. The investigators show that these algorithms are based on special
hidden Markov or semi-Markov models and can be generalized to accommodate more
detailed biology knowledge. For single-molecule data analysis, the investigators
outline an efficient likelihood-based approach for inferring quantities of
special interests in single-molecule studies. The form of the observed data
naturally calls for a data augmentation framework, which is a promising means
for solving the computational difficulty. In single-molecule studies, besides
the problem of model inference, model selection is also an important and
difficult task, as it is often the case that there are competing models
describing one chemical reaction, making it necessary to use the experimental
data to choose the appropriate model. The investigators, using a data
augmentation approach, propose a few generalized methods for choosing among
different chemical models.&lt;br/&gt;