* 0206374
* SGER: Strategy for Systematic Identification of Targets for a Novel Splicing Pathway
* BIO,MCB
* 02/15/2002,05/31/2003
* Suzanne Sandmeyer, University of California-Irvine
* Standard Grant
* Joanne Tornow
* 05/31/2003
* USD 99,999.00

This research focuses on a novel splicing system thought to be common among
eukaryotic cells but for which only a single example of a defined mRNA substrate
has thus far been identified. The primary goal in this work is to develop a
systematic means for identifying additional substrates for this unique splicing
system. This novel splicing mechanism was first identified as a component step
in a stress-response signaling pathway in the yeast Saccharomyces cerevisiae. A
key step in this pathway, termed the Unfolded Protein Response (UPR), is
regulated splicing of the mRNA encoding a transcriptional activator (termed
Hac1p). The result is to modulate levels of the Hac1p activator which, in turn,
controls expression of the cognate stress response genes. This splicing event is
not mediated by the conventional mRNA splicing apparatus. Instead, two distinct
gene products, Ire1p and Trl1p, act as splicing endonuclease and RNA ligase,
respectively, to carry out the splicing reaction. Remarkably, the Ire1p is a
transmembrane receptor that responds to changes in ligand binding by modulating
its own splicing activity. Thus, splicing is truly a component step in the
signaling cascade that constitutes this stress response pathway. Extensive
evidence exists for conservation of UPR function among eukaryotes including the
observations that yeast Hac1 mRNA can be accurately spliced in mammalian cells
and recombinant human Ire1p accurately cleaves yeast Hac1 RNA. Based on this
conservation of function, it is hypothesized that: (1) Regulated splicing may be
a component step in UPR signaling pathways in all eukaryotes. Thus, Hac1
homologues may be found in a wide variety of eukaryotic cells. (2) This splicing
system may serve a variety of gene regulatory functions other than the UPR
response. Thus, regulated splicing of a variety of mRNAs may be mediated by this
system. These hypotheses can be tested by the systematic identification and
characterization of splicing substrates from a variety of organisms. The use of
Trl1p as an affinity ligand in a selection/amplification approach will be tested
as a means for conducting such a systematic analysis. The experimental goals are
as follows. First, microarray analyses will be used as a comprehensive means for
characterizing populations of Saccharomyces mRNAs enriched in a
selection/amplification procedure. Second, application of this approach to
another organism and the utility of homologous versus heterologous Trl1 gene
products as affinity reagents will be assessed. Candida albicans has been chosen
for this purpose based on the availability of appropriate genomic resources, its
medial phylogenetic relationship to Saccharomyces, and on evidence for
conservation of the splicing machinery in this organism. The results of this
exploratory research will make it possible to evaluate the significance of this
splicing system as a fundamental stress response mechanism among eukaryotic
cells and to assess the potential for additional gene regulatory functions for
this novel splicing pathway.