* 0209809
* Arabidopsis 2010: Visual Informatics Tools to Interactively Link Arabidopsis Metabolic and Regulatory Network Maps with Genome-Wide Expression Data
* BIO,DBI
* 08/15/2002,07/31/2005
* Dianne Cook, Iowa State University
* Standard Grant
* Manfred D. Zorn
* 07/31/2005
* USD 312,500.00

Emerging molecular technologies are enabling the collection of massive data sets
that contain information about the full complement of mRNAs, proteins and
metabolites in an organism. These data sets detail the fluctuations in levels of
mRNAs, proteins, and metabolites in different organs, tissues and cell types,
over time, and as the organism is subjected to environmental stresses. The
development of Arabidopsis as a model plant system, and the ensuing emphasis on
functional genomics in the Arabidopsis 2010 program, is leading to the
generation of a multitude of such data sets. These data sets can compare wild-
type and genetically modified Arabidopsis to understand how a change in a single
gene can affect the function of the entire genome. The overall goal is to
understand the interactions occurring between all the genes in a simple plant
system, as the basis for developing a logical framework to improve the
composition and yield of agronomically important plant
species.&lt;br/&gt;Computational technologies are essential to integrate and
interpret these large data sets. The goal is to develop and implement powerful
software tools for analysis of genome-wide expression data. The software tools
will consist of three modules that will integrate the visualization of the mRNA,
protein, and metabolite data sets with our current understanding of the
Arabidopsis metabolic and regulatory network. These are: 1) A metabolic and
regulatory network mapping capability with an interactive graph display. 2) A
data integration capability, wherein global profiling data can be clustered,
visualized, superimposed on the metabolic and regulatory map. 3) A modeling
capability in which the metabolic or regulatory flow in the network is modeled
using fuzzy cognitive maps. These tools will be used by the biological community
to generate and test hypotheses about the regulation of metabolic pathways that
contribute to the final composition and yield of the plant. The 24-month
deliverable will provide a proof-of-concept for the software, using test data
sets from Arabidopsis.&lt;br/&gt;The proposed multidisciplinary research will
provide invaluable opportunities for the integration of genomics and
bioinformatics education into undergraduate and graduate curricula. Three-five
undergraduate students and two graduate students will be involved over the
course of two years. In addition, interations with local high school teachers
are planned to enable motivated high school students to participate in this
research. There is also a plan to integrate students from underrepresented
groups into the research project. The ability to reach young, computer savvy
students even in high school will be further enhanced by the high power computer
graphics used in the project. Development of this software will lead directly to
its use as a teaching tool in secondary education.&lt;br/&gt;This coordinated
approach to displaying multiple views of metabolic networks, cellular functions,
and datasets will put abstract data in the context in which it arises, and
facilitate studies of uncertainty and variation in metabolic networks. Although
the software will be directed specifically towards the biology of Arabidopsis,
it can be modified for use in any species. These tools, and the database will be
freely available to the scientific community for academic purposes
http://www.botany.iastate.edu/~mash/metnetex/metabolicnetex.html. They will
provide a novel framework to formulate testable hypotheses regarding the
function of specific genes and expand our holistic understanding of
metabolism.&lt;br/&gt;