* 0213832
* Computational Tools and A Database for the Analysis of Binding Sites in Enzymes
* NONE,NONE
* 08/15/2002,07/31/2006
* Charles DeLisi, Trustees of Boston University
* Continuing grant
* Manfred D. Zorn
* 07/31/2006
* USD 607,603.00

A major challenge in structural genomics is the elucidation of enzyme active
sites from molecular structure. The characterization of the ligand/substrate
binding site provides the basis for a number of applications, including the
design of inhibitors for the analysis of biochemical and signal transduction
pathways, drug design, and protein engineering aimed at altered specificity or
catalytic activity. The primary sources of information on specific molecular
interactions are the structures of the enzyme (or its homologues) co-
crystallized with various ligands (substrates, cofactors, inhibitors, products,
and transition state analogs). Although such structures are available for over
70% of the enzymes currently in the Protein Data Bank, collecting all binding
site information for a particular protein requires substantial efforts.

The goal of this proposal is to develop a computational resource and database,
called Predicted and Consensus Interaction Sites in Enzymes (PRECISE), which
will provide query and visualization tools for the comparative analyses of the
interactions extracted from all relevant structures. For each enzyme, the web-
based analysis tool will determine the consensus binding site, obtained by
aligning all homologous sequences, identifying the residue positions that are
important for the binding of any ligand, and assessing the roles of amino acids
at these positions. A dynamic retrieval system will help to examine individual
interactions, and to generate various statistics. In addition to the interaction
extracted from enzyme structures, the database will also store putative
interactions, based on mutation studies or theoretical predictions. Such
interactions can be submitted to the website and checked for consistency with
all the interactions extracted from structural data. The submissions will be
stored in a separate table, adding a predicted interaction component to the
database.

The database will also house the results of applying computational solvent
mapping, a novel binding site prediction method, to a large set of well-
characterized enzymes. The method moves molecular probes - small organic
molecules or functional groups - on the protein surface in order to identify the
positions that bind the highest number of different probes. It was shown that
such consensus binding occurs at major subsites of the enzyme binding site, and
the amino acid residues that interact with the probes also bind the specific
ligands of the enzyme. Thus, computational solvent mapping can be used for the
identification and characterization of enzyme binding sites. In addition to
providing solvent mapping results for a set of well-characterized enzymes, an
e-mail server will be set up, so that solvent mapping calculations can be
requested for any enzyme. It is expected that the PRECISE website will become a
central depository of enzyme binding site information, will have a major impact
on enzymology research due to the solvent mapping web-server, as well as provide
an important educational resource for studies in biochemistry and molecular
biology.