* 0228144
* Analysis of the Ribonucleases of Arabidopsis thaliana
* BIO,MCB
* 04/01/2002,02/28/2005
* Pamela Green, University of Delaware
* Standard Grant
* Joanne Tornow
* 02/28/2005
* USD 356,576.00

Gene expression is a multi-step process fundamental to all phases of plant
growth and development. The first step in the process, transcription, has thus
far received the most attention and therefore is the most well understood at the
molecular level. However, it is clear that post-transcriptional events can have
profound effects on gene expression. One prominent post-transcriptional event is
that of mRNA degradation. In plant and other eukaryotic cells, the number of
genes considered to be regulated at the level of mRNA stability is increasing
rapidly. Despite the importance of mRNA stability in gene expression, the
enzymes that catalyze mRNA decay in cells of plants and other multicellular
eukaryotes have yet to be identified. This project will address the function of
several potential mRNA degrading enzymes (mRNases) in Arabidopsis using
functional genomic and biochemical analysis. The hypothesis is that these
enzymes catalyze different steps in mRNA decay: AtPARN removes the poly(A) tail,
AtDCP1 with co-factor AtDCP2 removes the cap, and AtXRN4 degrades the remainder
of the RNA starting from one end. Although the DCPs and XRN1 facilitate the
latter two steps of most cytoplasmic mRNAs in yeast, their functions in
multicellular eukaryotes such as mammals and plants remain an open question
because plants and mammals exhibit differences in mRNA decay compared to that in
yeast. Nevertheless, the presence of homologous genes in plants and animals, as
well as other observations, suggests that yeast will provide some clues about
components common to all eukaryotes. Therefore, what is needed is a careful
analysis of the general mRNA decay machinery in a multicellular eukaryote to
elucidate the fundamentals of how mRNA stability is controlled in complex
systems versus yeast. Arabidopsis has been chosen as the model system because of
the experimental tools available and the importance of mRNA stability in
establishing plant gene expression levels. To further understanding of the mRNA
decay roles of AtPARN, AtDCP1, and AtXRN4, and their potential substrates
genome-wide, this project will 1) characterize the enzymes biochemically and
examine their intracellular locations, 2) obtain knockout or "conditional"
knockout mutants in the corresponding genes, and 3) analyze the most informative
of these mutants via DNA microarray analysis and other RNA analyses to identify
mRNA substrates. The roles and potential targets of these putative mRNases have
never been studied on a global scale nor have the corresponding genes been
inactivated in any multicellular eukaryote. Therefore, these experiments should
have broad significance. Moreover, for some of these enzymes, characteristics
unique to plants have been discovered; thus this work has the potential to
pinpoint aspects of mRNA decay that are novel and plant specific. &lt;br/&gt;