* 0236316
* Structural Properties of Initial States in Protein Folding from NMR Alignment Data
* BIO,MCB
* 02/15/2003,01/31/2008
* Andrei Alexandrescu, University of Connecticut
* Continuing Grant
* Kamal Shukla
* 01/31/2008
* USD 575,803.00

Recent work has shown that proteins under extreme denaturing conditions retain a
net alignment with respect to an NMR magnetic field when immersed in the
asymmetric environment of liquid crystals or gels. The asymmetric shapes of
these matrices impose weak steric restrictions on the molecular reorientation of
solutes. The resulting time- and ensemble-averaged alignment leads to incomplete
cancellation of NMR dipolar couplings. Dipolar couplings report on the
orientation of individual bond vectors with respect to the molecular alignment
frame. The objectives of this project are to further the understanding of the
structural basis for net alignment in denatured proteins and to use the
alignment information encoded in dipolar couplings towards a structural
characterization of denatured states. The objectives will be addressed with two
model systems. The first consists of the two proteins CspA and LysN, which share
a native five-stranded beta-barrel structural motif but no detectable sequence
similarity. Dipolar couplings will be used to investigate the extent to which
native-like structure is conserved between the denatured states of the two
proteins. The second model system is comprised of two polypeptides that adopt
regular extended alpha-helical structures: the S-peptide, which corresponds to
the first alpha-helix of the protein ribonuclease A; and the GCN4-p alpha-
helical coiled-coil dimer. Dipolar couplings will be used to determine the
persistence length of alpha-helical conformations in the denatured forms of the
two polypeptides.&lt;br/&gt;&lt;br/&gt;The principal motivation of this work is
to obtain an improved understanding of the structural basis of protein folding.
Protein folding remains one of the most difficult and intriguing problems in
biology today, and is essential for progress in structural genomics. The new
experimental window on structure in denatured states afforded by NMR alignment
data, will allow more accurate modeling of protein folding and of the factors
responsible for the stability of proteins to unfolding. Modeling of protein
folding requires information on denatured states in as much as structure in
these initial states can favor or oppose folding to the functional native state.
The structural characterization of denatured states additionally impacts
mechanisms for denatured protein aggregation reactions, which can occur through
illegitimate recombination of incompletely formed nascent structures. The
delineation of structural building blocks that persist in denatured proteins
provides insights into the mechanisms by which protein structures evolve,
information that in turn could be useful in protein design. This project offers
the opportunity for students to train in a rich, multidisciplinary approach to
protein folding that combines protein chemistry, state of the art NMR
experiments, and structure modeling. &lt;br/&gt;