* 0241160
* Statistical and Computational Approaches for Integrated Genomics and Proteomics Analysis and Their Applications to Modeling G1/S Transition During Yeast Cell Cycle
* MPS,DMS
* 01/01/2003,12/31/2006
* Michael Snyder, Yale University
* Continuing Grant
* Ashwani K. Kapila
* 12/31/2006
* USD 1,234,850.00

Advances in technologies are changing the field of biology to move beyond
genomes to transcriptomes, proteomes and metabolomes. It has become clear that
the combination of predictive modeling with systematic experimental verification
will be required to gain a deeper insight into living organisms, therapeutic
targeting and bioengineering. Although the importance of integrating various
types of biological data to address scientific questions is well recognized and
appreciated, the potential information carried in different types of data may
not be fully realized without a sound and comprehensive statistical framework to
integrate these data. In addition, close collaborations among statisticians,
biologists, bioinformaticians, and computer scientists are essential to ensure
that these statistical methods provide a reasonable description of the
biological processes studied and the validity of these methods should be
rigorously tested through biological experiments. In this project, a team of
researchers with expertise in statistics, genomics and proteomics,
bioinformatics, and computer science will develop an integrated approach to
reconstructing biological pathways. Statistical and computational methods will
be developed to better identify transcription factor targets, to integrate yeast
two-hybrid data, protein complex data, protein localization data, and gene
expression data to infer protein interaction networks, and to further integrate
DNA- protein binding data to reconstruct transcriptional regulatory networks.
This project focuses on the G1/S transition during the yeast cell cycle to
statistically model and experimentally validate inferred regulatory networks. In
addition, parallel computing methods will be developed to overcome the computing
bottleneck in the analysis of large-scale networks. The resources generated from
this project, both computer programs and network information will be made
available to the scientific community. It is anticipated that this project will
lead to a statistical framework that can be utilized to dissect biological
pathways and also will lead to an approach to integrating expertise from diverse
disciplines to address important scientific problems in the post-genome
era.&lt;br/&gt;With recent progresses in biotechnologies, it has become reality
to collect tens of thousands of gene expression and protein expression levels in
humans and other organisms. In addition, scientists now are able to monitor
interactions among proteins and interactions between proteins and DNA sequences,
to investigate the location that each gene is expressed, and to study the
overall effects on the whole organism of individual genes through large
collections of mutation strains. The availability of such data has led to a
revolution in biological and biomedical sciences. Although there is a great
potential and an enormous amount of information in these data, the major
challenge is how to best integrate, analyze, and interpret these data to
understand biological pathways. In this project, statistical and computational
methods will be developed to integrate various types of data in an effort to
reconstruct biological pathways with a focus on the understanding of gene
regulations in cell cycle. The statistical models to be developed will be
validated with biological experiments. Computer programs will be developed and
distributed to the scientific community after extensive testing to allow
biologists and medical researchers to use these tools to study other biological
pathways. This project will also develop high-performance computing approaches
to implementing the developed methods and will involve training activities in
the general area of computational biology and bioinformatics. This grant is made
under the Joint DMS/NIGMS Initiative to Support Research Grants in the Area of
Mathematical Biology. This is a joint competition sponsored by the Division of
Mathematical Sciences (DMS) at the National Science Foundation and the National
Institute of General Medical Sciences (NIGMS) at the National Institutes of
Health. &lt;br/&gt;