
* 0706989
* A New MCMC Framework with Applications to Protein Bioinformatics
* DMS,STATISTICS
* 07/01/2007,07/14/2011
* Jun Liu,MA,Harvard University
* Continuing Grant
* Gabor Szekely
* 06/30/2013
* USD 629,206.00

In the past two decades, statisticians and other quantitative researchers have
begun to appreciate the power of Monte Carlo integration and optimization
methods. This proposal focuses on the development of a novel Markov chain Monte
Carlo (MCMC) framework, which promises to greatly enhance our capability of and
flexibility in designing effective Monte Carlo algorithms. More precisely, the
investigator proposes a unified framework to generalize the standard Metropolis-
Hastings approach to design Markov chains and shows its deep relationship with a
few existing MCMC methods, such as multigrid Monte Carlo, configurational-bias
Monte Carlo, and orientational-bias Monte Carlo. The investigator will also
focus on one of the fastest growing application areas, protein bioinformatics
(encompassing multiple sequence alignments, protein function annotation, and
protein-protein interactions, and protein structural modeling, etc.), which
serves both as an important application and as a great source of significant
challenges to existing MCMC methods. On one hand, the investigator seeks to
apply the new MCMC framework to design novel protein structure and sequence
analysis tools; on the other hand, the challenging problems encountered during
such endeavors will motivate and steer the investigator to develop new MCMC
strategies.

With the ever growing need of quantitative (statistical) analysis of very large
datasets with complex structures (such as genomics data, consumer goods data,
internet data, etc.), the need for designing more efficient computational
methods to analyze these data and to make useful predictions is also strong.
This proposal has three inter-related themes: to develop a novel Monte Carlo
framework, which can be generally understood as a new way of utilizing computer-
generated random numbers to approximately solve an optimization or integration
problem, to develop novel statistical models for biological sequence and
proteinstructure analysis, and to apply these new computational methods and
statistical models to infer molecular mechanisms of protein functions and to
predict protein structures. The proposed research will not only significantly
advance the Monte Carlo methodology and computational statistics theory, which
are applicable to a wide range of optimization and simulation problems in
different application areas, but will also bring the power of these new methods
and theory to bear on one of the most important application areas, computational
biology. It will particularly advance the modeling, analysis, and computational
techniques in protein bioinformatics. It will help educators revise and generate
new courses on computational biology and Monte Carlo methodologies for both
undergraduate and graduate students. It will also provide interdisciplinary
research opportunities for such students, and will result in software and
methodologies that may be of interest to the pharmaceutical industry.
