
* 2113646
* Statistical Learning and Inference for Single-Cell RNA Sequencing
* DMS,Genetic Mechanisms, STATISTICS, MSPA-INTERDISCIPLINARY
* 08/01/2021,07/20/2021
* Jingshu Wang,IL,University of Chicago
* Standard Grant
* Yong Zeng
* 07/31/2024
* USD 230,000.00

Single-cell genomics is an emerging technique that has become an indispensable
tool for understanding cellular diversity and cell activities. Among the various
single-cell sequencing technologies, single-cell RNA sequencing (scRNA-seq),
simultaneously measuring tens of thousands of RNAs inside each individual cell,
is the most mature and widely used technology. This project aims to develop new
analytical tools for scRNA-seq with explicit and coherent statistical frameworks
to provide reliable uncertainty quantification and inference. At the same time,
the new tool will retain the scalability and user-friendly features in existing
algorithmic-based methods. The PI will focus on building probabilistic models
for machine learning frameworks such as deep learning and address new challenges
to account for biological randomness and technical noise in scRNA-seq. The PI
will develop open-source software for analyzing scRNA-seq data to help
scientists understand cell development, the mechanism of gene regulation, and
cell-type-specific features of common diseases. Because of the interdisciplinary
feature of this project, it will also train both graduate and undergraduate
students within and outside statistics to become future scientists in the fast-
evolving area of applied statistics and computational biology.

The PI plans to focus on three research problems that are unique to the analysis
of single-cell data: trajectory inference, cell type deconvolution, and gene-
gene co-expression / co-bursting. For trajectory inference, the PI will
incorporate a hierarchical mixture model into a deep neural network to infer
trajectories shared by cells from multiple sources. In the cell type
deconvolution problem where scRNA-seq data are used as references to estimate
cell type proportions in bulk samples, the PI will derive asymptotically valid
confidence intervals of the estimated cell type proportions without parametric
assumptions and account for three major uncertainty-inflation factors: the
technical noise, biological heterogeneity across individuals, and dependence
across genes. Finally, in the gene-gene co-expression / co-bursting analysis,
the PI will estimate the true gene-gene correlation and co-bursting pattern from
noisy observed data and design a scalable multiple testing framework that can
efficiently find gene pairs that are co-expressed or co-bursted. The PI also
aims to link the co-expression and co-bursting signals with the enhancer-
promoter contacts in the three-dimensional genome structure to understand causal
mechanisms of transcriptional regulation.

This award reflects NSF's statutory mission and has been deemed worthy of
support through evaluation using the Foundation's intellectual merit and broader
impacts review criteria.
