* 0318047
* RNA Processing Regulation of Immunoglobulin Gene Expression
* BIO,MCB
* 08/01/2003,07/31/2007
* Martha Peterson, University of Kentucky Research Foundation
* Standard Grant
* Jo Ann Wise
* 07/31/2007
* USD 463,048.00

The human genome has been found to encode a surprisingly small number of genes.
However, as many as 60% of all genes may be alternatively processed to generate
a much greater diversity of proteins. Thus, to fully understand gene expression,
how the complex process of alternative RNA processing is regulated must be
understood. This will require that well-established models for regulated
processing be used to decipher the mechanistic details of these processes. The
mouse immunoglobulin mu gene has been studied for many years as a model system
for regulated RNA processing during B lymphocyte development. This gene contains
a cleavage-polyadenylation site that is in competition with a splice reaction;
the relative use of these two processing options is modulated as B cells mature
to plasma cells. The arrangement and relative strengths of RNA processing
signals in the Ig mu pre-mRNA is more important than gene-specific cis-acting
regulatory sequences for the RNA processing regulation. This indicates that the
regulation between B cells and plasma cells must involve changes in components
of the general RNA processing machinery. Both cleavage-polyadenylation and
splicing activity have been shown to change as B cells mature to plasma cells.
In addition, other steps of RNA metabolism, including transcriptional
elongation, nuclear:cytoplasmic mRNA transport and RNA stability have been shown
to differ between B cells and plasma cells. The goal of this project is to
identify proteins involved in regulating alternative Ig mu mRNA processing to
better understand this regulatory mechanism. There is substantial evidence that
the terminal differentiation program of B cells, including changes in mu-s/mu-m
mRNA processing, is driven by a complex interplay of transcriptional regulators.
Microarray screening will be used to identify genes whose expression changes
during the developmental transition of B cells to plasma cells; some of these
differentially expressed genes will encode products relevant to RNA processing
regulation. Over-expression and inhibition of candidate proteins will determine
their role in RNA metabolism during B cell maturation. This research will
involve graduate, undergraduate, and, potentially, high school students.