* 0321756
* Coupling Expressed Sequences and Bacterial Artificial Chromosome Resources to Access the Barley Genome
* BIO,IOS
* 09/01/2003,01/31/2009
* Timothy Close, University of California-Riverside
* Continuing Grant
* Diane Jofuku Okamuro
* 01/31/2009
* USD 2,497,294.00

Several cereal crops of major economic and social importance, including barley,
wheat, rye and triticale, are members of the tribe Triticeae. These crops
account for over 30% of the world's grain production, with more than 35% of the
human population using them as a staple food. In the United States, Triticeae
crops are sown on approximately 81 million acres each year with an average value
over 9 billion dollars. Although products derived from the Triticeae crops are
diverse (bread, pastries, pasta, beer, malt whiskey, soup ingredients, animal
feed), the species are genetically closely related. The Triticeae genomes, each
with a size of about 5 x 109 base pairs per haploid genome, are too large to
seriously consider whole-genome sequencing at present. This is about 12 times
the size of rice. Yet, reliable methods can be immediately employed to gain
access to the majority of expressed genes in Triticeae genomes. During the past
several years, considerable progress has been made in the establishment of core
public resources in barley genomics. These core resources include genomic
"bacterial artificial chromosome" (BAC) libraries and cDNA libraries, several
widely used mapping populations, more than 360,000 partial cDNA sequences
("Expressed Sequence Tags; ESTs) representing about 75% of all genes in the
barley genome, and a microarray that makes it possible to simultaneously monitor
the expression of more than 21,000 barley genes.&lt;br/&gt;The purpose of this
project is to couple these resources to facilitate access of US and worldwide
researchers to the barley genome, thereby building a fuller understanding of
cereal plant traits that are pertinent to agriculture and biology. The unifying
objective is to accelerate a transition to comprehensive physical mapping and
the sequencing of all gene-containing regions of the barley genome. The project
is structured into six progressive objectives, each oriented toward gradual
extraction of the essence of the barley genome. The first objective will utilize
knowledge of unigene sequences to identify the majority of BAC clones that carry
expressed genes, expected to be about 20% of all BAC clones. The second
objective will use a genetic fingerprinting technique to create many thousands
of contiguous alignments ("contigs") of these BAC clones, and from them define a
minimal set representing all. The third and fourth objectives place special
emphasis on 1000 genes that are related to abiotic stress, including drought,
low temperature, heat, and salinity by adding these contigs and genes to the
barley genetic linkage map. The fifth and sixth objectives will create easy
access to this information through a database browser that will offer users a
choice of purely web-based functionality or, for those who prefer more speed,
hybrid software that operates principally on personal computers and relies on
the web only for supplemental
information.&lt;br/&gt;&lt;br/&gt;Deliverables&lt;br/&gt;Information on gene-
specific and popular overgo probes will be available during year #1. Sub-arrays
of the entire BAC library enriched for the gene-rich portion of the genome will
be available beginning at the end of year #2. Physical maps of BAC contigs
related to this portion of the genome will be available starting year #1, with
additions through year #2. BAC contigs and ESTs related to 1000 genes pertinent
to abiotic stress will be anchored to the genetic linkage through year #4. The
total compilation of progress will be freely available through the project web
site (http://harvest.ucr.edu) throughout and after the end of the
project.&lt;br/&gt;