* 0326029
* RUI:   Regulation of Gene Expression by the Nonsense-mediated mRNA Decay Pathway of Yeast
* BIO,MCB
* 06/01/2003,05/31/2006
* Jeffrey Dahlseid, Gustavus Adolphus College
* Continuing Grant
* Vicki B. Vance
* 05/31/2006
* USD 198,325.00

Nonsense-mediated mRNA decay (NMD) accelerates the degradation of mRNAs that
undergo premature translation termination due to a nonsense mutation. NMD exists
in all eukaryotes thus far examined, from yeast to man, and safeguards cells
against the accumulation of potentially deleterious protein fragments encoded by
so-called nonsense mRNAs. In budding yeast, the UPF1, UPF2, and UPF3 genes are
required for NMD. These genes also affect the accumulation of hundreds of wild-
type mRNAs, which suggests that NMD is an important part of the natural cellular
repertoire for regulating wild-type gene expression. The primary objective of
this research is to study the role of NMD in regulating the expression of wild-
type genes. NMD affects the mRNA accumulation of several wild-type genes that
encode proteins involved in chromosome transmission and stability. The mRNAs for
CTF13, which encodes an essential protein of the CBF3 kinetochore complex, two
additional kinetochore proteins, and five proteins that affect telomere function
are elevated in upf mutant yeast strains. The three specific aims of this
research are to determine if NMD directly affects the stability of these mRNAs,
to characterize the recognition and degradation of any that are wild-type mRNA
targets of NMD, and to investigate potential coordinate regulation of wild-type
gene expression by NMD. Analysis of expression from promoter-reporter gene
fusions and measurement of mRNA decay and transcription rates will be used to
determine whether NMD directly affects mRNA stability or exerts indirect
influence upon mRNA transcription. Decay of many wild-type yeast mRNAs involves
deadenylation followed by decapping and 5'-&gt;3' exonucleolysis, whereas
nonsense mRNAs bypass deadenylation but then undergo the same decay. Analysis of
deadenylation rates and susceptibility to decapping and 5'-&gt;3' exonucleolysis
will be used to determine whether NMD degrades wild-type mRNAs through
deadenylation-dependent or -independent decapping and 5'-&gt;3' exonucleolysis
or some other mechanism. Sequences or structures necessary for recognition of
wild-type mRNAs by NMD will be identified using deletion mutations and gene
fusions. The possibility that kinetochore- and/or telomere-related genes may be
coordinately regulated by NMD will be investigated through analysis of mRNA from
synchronized cell cultures and genetic approaches to identify putative
regulatory genes, which may encode mRNA targets of NMD. This project serves as
an important starting point to increase understanding of the cellular role of
NMD in regulating the expression of wild-type genes and, ultimately, the
mechanism for recognition and decay of specific wild-type mRNAs by NMD. In
summary, protein molecules are functional components of nearly all the molecular
processes in biological systems. Instructions for protein formation are stored
in the DNA of genes and are provided as a chemical information intermediate,
known as messenger RNA (mRNA), when genes are expressed. To achieve normal
growth and development, cells must regulate the expression of genes. Selectively
degrading the mRNA of a wild-type gene is an important mechanism for cells to
regulate its expression. This research aims to increase understanding of the
cellular role for specialized mRNA degradation pathways in regulating wild-type
gene expression and the mechanisms involved in recognizing specific wild-type
mRNAs for selective degradation.