* 0354771
* A Computational Capability for Fast and Reliable Characterization of Protein Complexes
* BIO,DBI
* 08/01/2003,05/31/2008
* Ying Xu, University of Georgia Research Foundation Inc
* Continuing Grant
* Peter McCartney
* 05/31/2008
* USD 817,034.00

Protein-protein interactions are at the heart of biological activities. They
constitute the basic components of many biological processes such as signal
transduction, cell-cycle control, metabolism, and general cellular machines.
Characterization of protein-protein interactions in protein complexes in a cell,
in a systematic manner, represents a highly challenging and important problem to
functional genomics and proteomics in the post-genome sequencing era. An
integrated computational capability for characterization of protein complexes in
a cell will be developed, through (a) analyzing mass spectrometry data and
chemical cross-linking information and (b) protein docking prediction under
geometric constraints derived from these experimental data. Initially this
capability will be tested and validated on a selected set of protein complexes
from yeast, as a proof of principle. When fully developed, this capability will
be used for genome-scale cataloging of protein complexes in yeast (and other
genomes in general). The specific aims of this proposed project are: (i)
development of improved computational methods for locating and identifying
cross-links, particularly inter-molecular ones, from mass spectrometry data;
(ii) development of new computational methods for identification of protein
complexes and their component proteins, through analysis of cross-linking and
other experimental data; (iii) development of new computational methods for
data-constrained docking of two proteins that will significantly improve the
existing docking methods in both prediction accuracy and application generality;
(iv) development of new computational methods for data-constrained multi-party
protein docking; and (v) applications of the developed methods to a selected set
of protein complexes for their complex structure characterization. The proposed
computational capability will significantly improve the ability to interrogate
protein-protein interactions in a way not possible before, and open new doors
for the emerging fields of functional genomics and
proteomics.&lt;br/&gt;Investigation of these computational capabilities for
characterization of protein complexes will provide a number of opportunities for
graduate students and postdoctoral trainees who are interested in moving into
the emerging field of computational biology/bioinformatics from other
disciplines. A number of postdoctoral trainees will be hired to implement some
of the key components of this research, and Ph.D. students will be recruited to
work on this project as part of their thesis projects. In addition, a
computational biology course will be offered to the graduate students of the
joint ORNL/UTK Graduate Program in Genome Science and Technology, in the Fall
semester of 2002. Some well-defined but challenging issues encountered in this
project will be used as student term projects through this course. The
challenging scientific problems to be solved in this project will expose the
graduate students and postdoctoral trainees to the core issues of
bioinformatics, i.e., intelligent and meaningful interpretation of massive
amount of biological data and computational modeling of biological
structures/processes, providing an excellent opportunity for hands-on training.
&lt;br/&gt;New scientific findings through the implementation of this project
will be made publicly accessible through journal publications and conference
presentations. Computer software to be developed in this project will be made
freely available to the academic and government organizations through the
Internet. All the biological data generated in this project will be organized as
databases, and also be made freely and publicly available through the
Internet.&lt;br/&gt;