* 2004952
* Analysis of a Model Developmental Gene Regulatory Network
* BIO,IOS
* 09/01/2020,08/31/2025
* Charles Ettensohn, Carnegie-Mellon University
* Continuing Grant
* Anna Allen
* 08/31/2025
* USD 952,047.00

Multicellular organisms arise from a single cell- a fertilized egg- through a
process known as embryonic development. During this process, repeated cell
divisions create an embryo composed of many cells. These cells gradually take on
distinct functions and organize themselves into tissues and organs.
Understanding development is of great scientific importance, as this is the
process by which new generations of multicellular organisms arise. Moreover,
during earth’s history, modifications to developmental programs have produced
the diverse animal morphologies we see in nature. Lastly, failures of human
development have profound societal impacts, and any appreciation of the causes
of such dysregulation must begin with an understanding of the mechanisms that
ordinarily operate during development. Remarkably, the entire blueprint for
embryonic development is contained in the DNA sequence of the genome. A central
challenge of modern biology is to explain how this genomic information is
“decoded” to control the process of embryogenesis. This research project
addresses that question. It exploits new methods (and undergraduate researchers)
for examining the physical structure of genomic DNA, for monitoring when and
where in the embryo various genes are working, and for dissecting interactions
among these genes. The work will shed light on the mechanisms by which the
information contained in the genome is deciphered during the transformation of a
single cell into an organism, and will form a foundation for development of a
digital atlas of echinoderm development. &lt;br/&gt; &lt;br/&gt;A hallmark of
animal development is the gradual emergence of anatomy, as represented by the
many different cell types of the mature organism and their organization into
distinct tissues and organs. The emergence of anatomy during embryonic
development (morphogenesis) requires specialized cell and tissue behaviors that
are driven by differential gene expression. Systems level approaches have
revealed that programs of differential gene expression are controlled by complex
networks of transcription factors and non-coding DNA sequences to which they
bind. Such gene regulatory networks (GRNs) are powerful tools for analyzing the
genetic control and evolution of development. To understand the genotype-to-
phenotype transformation it will therefore be important to link GRNs to
morphogenesis. The overarching goal of this work is to uncover the detailed
architecture of a model GRN and elucidate its control of crucial downstream
effectors that drive morphogenesis. It leverages a GRN that drives a suite of
spectacular behaviors by skeleton-forming primary mesenchyme cells (PMCs) of the
sea urchin embryo. The project combines “top-down” and “bottom-up” approaches.
The former uses gene knockdowns and gene expression profiling to elucidate the
regulatory topology of the network. The latter uses chromatin accessibility
profiling and eRNA analysis to identify cis-regulatory DNA sequences that
control morphogenetic effector genes, and then applies transcription factor
footprinting, mutagenesis, and transgenic reporter assays to dissect these
sequences and bootstrap into the upstream network circuitry. Lastly, the project
extends a successful gene knockdown screen to identify and characterize key
morphogenetic effectors. With respect to Broader Impacts, this project involves
undergraduate researchers, outreach to high school students and teachers, and
the development of a digital atlas of echinoderm
development.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.