* 2010807
* NSF Postdoctoral Fellowship in Biology FY 2020: Predicting Evolution Through  Quantitative Dissection of the Bacterial Proteome
* BIO,DBI
* 01/01/2021,12/31/2023
* Griffin Chure, Chure, Griffin Daniel
* Fellowship Award
* Daniel Marenda
* 12/31/2023
* USD 207,000.00

This action funds an NSF Postdoctoral Research Fellowship in Biology for FY
2020, Integrative Research Investigating the Rules of Life Governing
Interactions Between Genomes, Environment and Phenotypes. The fellowship
supports research and training of the fellow that will contribute to the area of
Rules of Life in innovative ways. This research will use modern technologies to
predict how bacteria will evolve in response to environmental pressures, an
ability necessary to understand the evolution of dangerous traits such as
antibiotic resistance. While an organism’s genetic sequence records its life
history, it is difficult to predict how the organism will evolve based on
sequence alone. This is due to the inability to predict how mutations can affect
the organism’s ability to reproduce, termed its fitness. The project will use
recently discovered biological laws to explore how mutations alter the cell’s
energy budget to make new proteins. This information will be used to produce a
mathematical model to understand how mutations are tied to fitness. The work
will develop an experimental system where bacterial evolution can be controlled
and predicted, which may have real world applications in medicine and
agriculture. This project will augment the fellow’s diverse background through
the application of new technology. To broaden the impact of the work, the fellow
will train undergraduate and graduate students in the sponsoring scientists’
labs.&lt;br/&gt;&lt;br/&gt;Recently developed bacterial “growth laws” predict a
strong correlation between the cellular growth rate and the proteomic fraction
dedicated to ribosomes. This implies that adaptive evolution should skew the
allocation of cellular resources towards maintaining ribosomes rather than
unnecessary proteins. The objective of this work is to dissect how adaptive
mutations modulate the allocation of resources to maximize growth rate,
culminating in a mathematical model to predict evolution. The fellow will rely
on novel sequencing-based experiments that permit time-resolved measurement of
the emergence of novel beneficial mutations in large microbial populations. The
effect of these beneficial mutations on the proteomic composition will be
monitored via mass spectrometry and RNA-sequencing, illuminating how beneficial
mutations alter the allocation of resources. Given a pairwise mapping of the
identity of a beneficial mutation and the gene expression profile, the fellow
aims to quantitatively map the fitness landscape. It then becomes possible to
predict how changes in gene expression lead to changes in fitness. The fellow’s
background in modeling and experimentation will be strengthened by the
development and application of sequencing-based experiments. The broader impacts
will include the training of undergraduate and graduate students in the design,
execution, and interpretation of project experiments.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.