* 2013998
* SCH: INT: Enabling real time surveillance of antimicrobial resistance
* CSE,IIS
* 01/01/2021,12/31/2024
* Jaime Ruiz, University of Florida
* Standard Grant
* Wendy Nilsen
* 12/31/2024
* USD 1,219,178.00

Antimicrobial resistance (AMR) refers to the ability of an organism to stop an
antimicrobial (e.g., antibiotic) from working against it and has become a
serious threat to public health since it causes antibiotics to be ineffective,
resulting in outbreaks becoming more frequent, widespread, and severe. It is
estimated that 2.8 million people per year in the United States are infected
with resistant bacteria, and more than 35,000 of these infections are lethal.
One manner to control these outbreaks is with real-time identification of AMR.
Currently, the most effective method for identification of AMR is to apply high-
throughput sequencing to a biological sample (e.g., nose swab or blood sample).
Advancements in sequencing technology have shrunken the size of the devices so
that they can fit into one hand, however the bioinformatics analysis â€“ requires
comparing millions or billions of DNA sequences -- has been limited to high
performance computers that have significant memory and disk space. This, in
turn, makes AMR identification limited in low-resource settings, such as rural
areas of the U.S. This project will overcome the challenge of detection of AMR
in rural areas by developing bioinformatics analysis methods for on-site, real-
time detection of AMR using portable computing devices (such as phones and
tablets). To realize this, the project will conceptualize and implement novel
algorithms and interfaces due to computing limitations created by using portable
computing devices. The outcome of this project will be a real-time portable
identification of AMR, which can be used to dramatically increase the efficiency
in which society can control and monitor outbreaks. In addition, these
techniques will also help realize identification of viral species (such as
COVID-19), which will assist in rapid diagnosis in areas with limited computing
and sequencing resources. Lastly, an immediate outcome of the work will be
research opportunities to under-served students through the Machen Florida
Opportunity Scholars program, an organization that aims to foster the success of
first-generation university scholars. For each year of the program, the
investigators will work with the coordinator of the Machen program to recruit a
student to be a research assistant and work hands-on the project with the
investigators and their trainees. &lt;br/&gt;&lt;br/&gt;The goal of this project
is to create mobile bioinformatics methods for on-site, real-time detection of
AMR using Nanopore technology. The expected methods will work on-device, meaning
they will only use the hardware (RAM, cache, hard disk, processors) on the
portable device. In particular, the project will aim to: (1) create on-device
methods to identify the bacteria in a biological samples; (2) create on-device
methods to identify the AMR genes in a biological sample; and lastly, (3)
evaluate the usability of the methods and prepare for their wide-spread
dissemination. This will be accomplished by combining the recent advancements in
cache-oblivious algorithms with that of space-efficient data structures.
Briefly, cache-oblivious algorithms divide the input of a problem into smaller
subsets so that each can be solved in cache and combined into a solution to the
original problem. This proposal further brings advancements that will have
impact beyond the stated application. Since portable devices pose significant
computational challenges, including smaller memory, cache, hard disk, this work
will result in novel algorithm and tool development that combine succinct data
structures with cache oblivious approaches. Next, this work will advance the
knowledge of AMR mechanisms. The use of antibiotics needs to be understood and
preserved in order to ensure it is judicious. This project will contribute to
acquiring such an understanding by detecting the drivers of AMR evolution,
persistence and dissemination in real-time. Lastly, it will further the use of
third sequencing technology that have broad application. One specific
application of this work is the real-time detection of COVID-19 in areas that
lack sequencing and computing facilities. Thus, this project will be the first
in creating a benchwork-to-bedside bioinformatic system for detection of AMR and
viral strains such as COVID. This will deepen the study of the technology,
highlight specific areas of improvement and expansion, and have significant
impact on public health.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.