* 2018296
* Pathways to Tertiary Structure Interfaces in RNA: An Integrated NMR, Biochemical, and Computational Approach
* BIO,MCB
* 07/01/2020,06/30/2024
* Charles Hoogstraten, Michigan State University
* Standard Grant
* Ishita Mukerji
* 06/30/2024
* USD 880,000.00

RNA, a close chemical cousin of DNA, is most familiar in its messenger role of
conveying information from the DNA genome to the cellular machinery that
synthesizes proteins encoded by that genome. In recent decades, however, a great
variety of noncoding roles for RNA molecules have been discovered, including in
catalyzing chemical reactions, regulating gene expression, and in specialized
functions such as CRISPR. To fulfill these roles, RNA takes up a wide variety of
specific three-dimensional shapes and forms. Relatively little is known,
however, about the formation of these three-dimensional RNA structures, leaving
large gaps in the search for a predictive framework for the design of functional
RNA molecules and of other molecules that target cellular RNAs. This project
will carry out an integrated program of spectroscopic, biophysical, biochemical,
and computational analyses in order to map out the energetics and folding
pathways for RNA structures. The work will provide a full atomic-level
description of the formation of an RNA three-dimensional structural interaction,
and a scientific understanding of such RNA-RNA interactions in noncoding systems
in general. This proposal will train underrepresented groups at the graduate and
undergraduate levels. The PIâ€™s efforts in mental-health awareness in the
scientific workforce at regional and national conferences will be
expanded.&lt;br/&gt;&lt;br/&gt;Although the factors underlying secondary
structure (helix formation) in DNA and RNA are relatively well understood, the
detailed study of the driving forces and mechanisms of formation of complex RNA
tertiary structures has not kept pace with the great recent advances in RNA
structural biology. The docking transition in the hairpin ribozyme is a rare
example of a tertiary-only interaction amenable to intermolecular biophysical
and spectroscopic investigation, with a wealth of existing structural and
functional data and multiple functional readouts. This project will use a
combination of NMR spectroscopy and molecular dynamics computations to examine
conformational sampling of the individual loops and use functional assays of
dynamically-quenched variants to determine the relevance of the observed
motional modes determined in each loop and derive internally-consistent models
of docking-competent states. An innovative protocol of NMR and Activity-
initialized Iterative Markov Analysis (NAIMA) will be used to integrate all of
the data in a computational determination of the pathway for the overall docking
process. In the course of this work, the derivation of a reference database of
unfolded RNA NMR chemical shifts for the general use of the field will be
completed. This project is supported by the Molecular Biophysics Cluster of the
Molecular and Cellular Biosciences Division in the Directorate for Biological
Sciences.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.