* 2020754
* Research PGR: Co-transcriptome networks to identify conserved and lineage specific plant resistance against a generalist pathogen
* BIO,IOS
* 08/01/2020,07/31/2025
* Daniel Kliebenstein, University of California-Davis
* Continuing Grant
* Gerald Schoenknecht
* 07/31/2025
* USD 994,239.00

To live and reproduce, any organism must prevent disease caused by pathogens
like viruses, bacteria and fungi. Understanding the mechanisms controlling this
battle between host and pathogens is necessary for understanding ecology and
evolution and efforts to improve productivity in agricultural systems. These
host-pathogens interactions span a wide range of systems from specialized where
one host faces off with one pathogen to more general systems where a pathogen
may attack a wide range of hosts. Most mechanistic knowledge is limited to the
specialized systems and this project will work to deepen our knowledge of
generalist host-pathogen systems. By using a fungal pathogen, Botrytis cinerea,
that causes disease on nearly any plant, this project will test the diversity of
plant defense mechanisms to a common attacker. Do plants defend themselves with
a common defense system, novel individualized mechanisms or a blend of common
and novel defense mechanisms? This information will improve our understanding of
when evolution generates novel solutions versus slight tweaks to existing
solutions to improve disease resistance. This knowledge can be used to
understand how and when evolution of defense has occurred, improve crops
resistance to disease and also help guide our efforts to engineer new biological
systems.&lt;br/&gt; &lt;br/&gt;Theories of plant-pathogen interactions are
molded by extensive research on host-specialist pathogen interactions controlled
by lineage-specific large-effect genes. Those theories shape how plants are bred
for disease resistance. However, large-effect models do not describe host-
generalist pathogen interactions, dominated by numerous small to moderate effect
genes distributed across diverse mechanisms. The polygenic nature of host-
generalist interactions complicates the translation of any mechanistic insights
from one host to another as mechanisms can be lineage specific or conserved.
Identifying these conserved mechanisms will improve the ability translate
mechanisms across species. This work will identify resistance mechanisms that
are lineage specific versus conserved using a collection of the generalist
pathogen Botrytis cinerea to measure lesion development across 16 dicot plant
species representing eight rosid and asterid lineages. Incorporating a co-
transcriptome approach, we will measure the host and pathogens transcriptome
responses and identify conserved or lineage-specific regulatory networks. By
developing rules to predict conserved and lineage-specific resistance
mechanisms, this project will increase the ability to predictively translate
scientific discoveries from one species to another. The identified conserved and
lineage specific resistance mechanisms in Tomato, Lettuce and Arabidopsis will
be validated for their influence on disease resistance via genome editing. This
project will also provide an integrative introduction to STEM for first
generation and transfer college students that are often overlooked in broader
outreach efforts. They will be trained in modern quantitative analysis of gene
networks to prepare them for future careers in industry or academics.
&lt;br/&gt;&lt;br/&gt;This award was co-funded by the Plant Genome Research
Program and the Plant Biotic Interactions Program in the Division of Integrative
Organismal Systems.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.