* 2021552
* URoL: Epigenetics 2: Learning the rules of dynamic epigenetic regulation
* BIO,EF
* 08/01/2020,07/31/2024
* Georg Seelig, University of Washington
* Standard Grant
* Manju Hingorani
* 07/31/2024
* USD 2,524,657.00

In the nucleus of a eukaryotic cell, the DNA genome can be chemically modified
to confer additional, so-called epigenetic, coding information. A central
unanswered question in regulatory biology is how epigenetic information is
decoded and used by cells as they grow and differentiate. This project aims to
address that question by first developing experimental and computational
strategies to predict how the genome and its epigenetic modifications are used
and then testing the validity of the predictions. The testing will be done
through experiments in simple cellular systems, as well as in cells programmed
to differentiate into components of the human immune system. The results are
expected to reveal regulatory ‘rules’ that can be used to engineer desirable
cellular outcomes. The project will have educational impact by partnering with
an existing program (Rainier Scholars, www.rainiercholars.org) to broaden
participation in STEM by offering a pathway to college graduation for low-income
students of color. In addition, research results from this project will be
incorporated in development of course modules to teach design principles of
epigenetic regulation at both participating institutions, the University of
Washington and California Institute of Technology.
&lt;br/&gt;&lt;br/&gt;Epigenetic regulation is central to all aspects of human
biology. However, despite its centrality and the wealth of molecular information
already collected, we still lack a predictive understanding of epigenetic
control. This project combines synthetic biology, developmental biology, and
computational machine learning approaches, to develop and validate predictive
models of how chromatin state, sequence context, and specific regulators
together establish gene expression state. These models will provide three key
benefits. First, they will provide fundamental biological insight into the
architecture of eukaryotic gene regulation. Second, they will permit engineering
of synthetic chromatin-based regulatory systems that provide useful
functionality, including the ability to establish stable gene expression states.
Such synthetic systems would enable cell-based therapies that utilize the
machinery of chromatin regulation to establish, maintain, and change cellular
states in response to environmental conditions, even in cells participating in
immune responses and other complex processes. Third, these models will help
researchers understand how eukaryotic organisms harness chromatin regulation to
control cell state transitions in their development. This knowledge not only
will enhance the ability to interface with the natural cell fate control
circuitry, but also will reveal design principles for optimizing the function of
synthetic chromatin regulatory systems.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.