* 2022182
* URoL: Epigenetics 2: Connecting cell fate and epigenome drift through physical models of chromatin structure and dynamics
* BIO,EF
* 07/15/2020,06/30/2025
* Andrew Spakowitz, University of California-Irvine
* Standard Grant
* Manju Hingorani
* 06/30/2025
* USD 3,000,000.00

When cells divide, not only the genetic sequence but also the chemical additions
to the DNA and its three-dimensional organization (the so-called epigenome) must
be replicated. The epigenome plays a central role in controlling the activity
levels of individual genes within a cell, and correct replication is critically
important for maintaining cellular function from one cell generation to the
next. Through a combination of genomics, microscopy, and computational modeling,
this project seeks to uncover the molecular and physical principles of epigenome
replication. The results are expected to open new opportunities for cellular
engineering. This project will broaden participation in research at the
crossroads of molecular biology, applied mathematics, and data-science for high
school and undergraduate students, with an emphasis on recruitment and training
of students from underrepresented groups. In addition, the project will expand
access to science education through a unique program for middle- and high-school
students in hospital settings.&lt;br/&gt;&lt;br/&gt;Temporal variation over the
cell cycle is an under-appreciated source of heterogeneity in the epigenetic
landscape and an under-utilized mechanism of control in epigenome engineering.
This project will characterize how sub-cell cycle dynamics in DNA and histone
methylation after replication direct the physical compaction and phase
separation of chromatin to ultimately drive both the activation of genes during
stem cell differentiation and the divergence of epigenetic patterns observed
across multiple cell generations. The project will establish whether temporal
heterogeneity in DNA and/or histone methylation operates as a critical feature
of mammalian life that either facilitates cell state-transitions, leads to error
accumulation and drift in the regulatory landscape of cells, or both. In
addition to advancing the fundamental understanding of epigenome regulation and
dynamics, the project will deliver: predictive models rooted in polymer physics
theory and biomolecular kinetics, which will unify mechanistic principles with
â€˜omic-scale datasets, and novel tools (biochemical, genetic, and optical) for
measuring and perturbing epigenome dynamics, thereby advancing the fields of
developmental biology, biotechnology, and biophysics/mathematical
biology.&lt;br/&gt;&lt;br/&gt;This project is funded by the Understanding the
Rules of Life: Epigenetics Program, administered as part of NSF's Ten Big Ideas
through the Division of Emerging Frontiers in the Directorate for Biological
Sciences. Co-funding is provided by the Genetic Mechanisms Program, Division of
Molecular and Cellular Biosciences, Directorate for Biological
Sciences.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.