* 2024073
* Locus specific manipulation of chromatin function
* BIO,MCB
* 01/01/2020,07/31/2021
* Mazhar Adli, Northwestern University at Chicago
* Standard Grant
* Arcady Mushegian
* 07/31/2021
* USD 118,849.00

In eukaryotic cells, the DNA genome is tightly packaged in the nucleus by close
physical association with small proteins called histones. Chemical modifications
of the histone proteins can loosen or strengthen their association with DNA.
Such chemical changes have been correlated indirectly with changes in how the
information in DNA is used, but to date, direct cause-and-effect tests of the
outcome of histone modifications on DNA function have not been feasible. This
project will develop and apply new cutting-edge tools to enable direct
measurements of how histone modifications impact the function of DNA. The
results will provide deeper understanding of normal regulatory mechanisms and
provide a framework for understanding how they contribute to proper growth and
development. In addition to the research goals, the project will also establish
educational outreach activities to attract younger generations to science
through summer research opportunities and seminar series for local high school
students as well as undergraduate students.

Regulation of gene expression is tightly linked to mechanisms that package DNA
inside the nucleus. DNA is wrapped around histone proteins to form chromatin
fiber. Small chemical modifications on histone proteins or DNA, which are called
epigenetic modifications, are associated with gene activation or repression.
Understanding the molecular details of how epigenetic modifications regulate
gene expression has immediate implications for better understanding of crucial
developmental processes in both animals and plants. However, due to technical
limitations, the causal relationship between the presence of an epigenetic mark
and gene function in the chromatin is not proven. This project will utilize
cutting edge CRISPR-Cas9-based genome editing tools to deposit specific
epigenetic marks at target genomic sites to study their functional roles over
time. By precisely manipulating temporal and spatial locus-specific epigenetic
marks, the project will address fundamental cause-and-effect mechanisms relating
epigenetic information to changes in gene expression.