* 2025293
* Collaborative Research: MTM 1: Decoding the genomic rules of denitrification in bacterial communities
* BIO,EF
* 10/01/2020,09/30/2023
* Seppe Kuehn, University of Chicago
* Standard Grant
* Mamta Rawat
* 09/30/2023
* USD 460,060.00

Microbes perform chemistry in the soils, oceans, and even our bodies. They
accomplish this collectively in communities composed of many distinct members,
each with their genetically-encoded capabilities, all participating in an
intricate metabolic trade network. The health and robustness of organisms and
ecosystems alike depend on the metabolism of microbes. A deep understanding of
the principles that predict the collective metabolism of microbial communities
would grant us the ability to control and engineer these communities for the
betterment of human health, and the health of our environment. DNA sequencing is
one of our most powerful tools for characterizing microbes. Unfortunately, it
remains challenging to relate the DNA sequences to the metabolic capabilities of
a microbial community. This problem is challenging given the complexity of
cellular physiology, and the many complex interactions between different
species. Using natural bacterial isolates that perform denitrification, a
process that makes up an important part of the nitrogen cycle, mathematical
modeling, and machine learning, the researchers will show that it is possible to
predict community metabolism from genomic sequences. The work is groundbreaking
because it means that by sequencing the DNA of complex communities one can now
"readout" their metabolic function. The insight will enable the design of
microbial communities with predefined metabolic function. Other broader impacts
of this research include training of undergraduate researchers in biochemical,
statistical, and computational research and outreach to parents and children
regarding microbiomes and denitrification. &lt;br/&gt;&lt;br/&gt;The collective
metabolic function of microbial communities emerges through a hierarchy of
genomically-encoded processes, from sub-cellular information processing and gene
expression to interactions mediated by extracellular metabolites, abiotic
factors, and collective phenomena. Understanding how this collective metabolic
function is encoded in the genomic structure of the consortium is a core
challenge for microbial ecology. This research proposed leverages
denitrification, a cascade of reactions that reduce oxidized nitrogen compounds
via anaerobic respiration, as a model metabolic function to build a bridge
between genomic structure and metabolic function in microbial communities. By
employing metabolite measurements on a library of sequenced denitrifying
bacterial isolates, the researchers will determine the predictive quantitative
relationship between the denitrification genes that each strain possesses and
metabolic dynamics. Community assembly experiments will then be used to predict
community metabolic dynamics from the dynamics of individual populations. In
cases where community dynamics are not predictable from the dynamics of
individual populations alone, the researchers will investigate genomic
predictors and molecular mechanisms underlying the behavior. Once the bridge
between genomic structure and metabolic function is built in one experimental
system, the researchers will attempt to do the same in a more complex system
that better approximates a natural context. The result will be a generalizable
mapping from genomic structure to metabolic function that enables the rational
design of microbial communities and provides a new quantitative interpretation
of functional gene content.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.