* 2025426
* MTM 2: Combining structural informatics and crosslinking mass spectrometry to predict the key protein-protein interactions shaping symbiotic microbial communities
* BIO,EF
* 12/01/2020,11/30/2024
* Vincent Young, Regents of the University of Michigan - Ann Arbor
* Standard Grant
* Mamta Rawat
* 11/30/2024
* USD 2,899,960.00

Microbial communities on and inside of animals play profound roles in biology,
affecting the behavior, nutritional status, and resilience of the host. Much of
the behavior of microbial communities is driven by interactions between proteins
of microbes and the host. These interactions are used by microbes to adhere to,
attack, or communicate with other cells in their surroundings. Because microbial
communities are tremendously complex, it has proven difficult to determine the
set of protein-protein interactions that define and stabilize a community or
affect other community members. Understanding the mechanics of those
interactions is essential to engineer or manipulate microbial communities, e.g.,
by encouraging the presence of beneficial microbes or driving out those that are
problematic. This project will unravel the rules governing how protein-protein
interactions establish and maintain microbial communities using a combination of
bioinformatics and biochemical approaches. The project will target the
interaction of two well-known probiotic (‘good’) bacteria with models of the
human intestine. The developed methods will also be applicable to a wide range
of other microbial communities, both those involving mammals (with applications
to agriculture and medicine) and those involving non-living surfaces (with
applications to food processing and infrastructure maintenance). Other broader
impacts include outreach activities at a museum and training of the next
generation scientists in microbiome research.&lt;br/&gt;&lt;br/&gt;This project
will develop a combination of computational and experimental approaches to
determine the identities and functional implications of host-microbe protein-
protein interactions (PPIs), in the context of synthetic microbial communities
grown on epithelial cell cultures and in human organoids. The studies will focus
on colonization by the well-known probiotic strains E. coli Nissle and
Lactobacillus rhamnosus GG. The use of a highly controllable, reductionist
system to study the roles of PPIs in the broader behavior of microbial
communities will permit the investigators to develop, test, and benchmark
experimental and computational tools to identify important PPIs and examine
their functional implications. The project will be anchored by the development
of advanced computational pipelines to enable large-scale high-quality
prediction of the structure and function of PPI networks. In parallel, for the
target organoid-microbe communities, the researchers will make use of recently
developed crosslinking mass spectrometry methods to experimentally examine the
landscape of PPIs in vivo, and transposon library profiling experiments to
identify the microbial genes that contribute substantially to host colonization
or microbe-microbe competition. This will be followed by the identification of
PPIs in multi-species communities using mouse fecal extracts and/or anaerobes
that play an important role in the gut microbiome. The research proposed here
will contribute to our knowledge base and technical capabilities both through
the enumeration of key trans-kingdom protein-protein interactions driving
bacterial host colonization (including identification of the interactions that
are functionally important), and through the development and refinement of a
computational framework for high throughput prediction of inter-organism
protein-protein interactions and their functional importance. The methods
developed here will be generalizable to permit rapid enumeration of key host-
microbe and microbe-microbe PPIs in complex microbial
communities.&lt;br/&gt;&lt;br/&gt;This project is funded by the Understanding
the Rules of Life: Microbiome Theory and Mechanisms Program, administered as
part of NSF's Ten Big Ideas and the Division of Emerging Frontiers in the
Directorate for Biological Sciences.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.