* 2025567
* MTM 1: Does replicon biochemistry define the infection dynamics of viruses within ecosystems?
* BIO,EF
* 01/01/2021,12/31/2023
* K. Wommack, University of Delaware
* Standard Grant
* Mamta Rawat
* 12/31/2023
* USD 425,000.00

Microbial communities, consisting of single celled organisms such as bacteria
and protists as well as virus particles, are ubiquitous throughout the
biosphere. These communities are the engines of nutrient cycles, processing
complex molecules into the simpler compounds essential for the growth of higher
organisms such as plants and algae. While we appreciate the biosphere-sustaining
role of microbes, our understanding of the ecological mechanisms supporting
nutrient cycles is rudimentary. In particular, little is known of how the
interactions between viruses and their microbial host cells influences nutrient
cycles. This project is exploring whether the biochemical characteristics of an
enzyme, DNA polymerase, which is responsible for a key step in viral
replication, can provide detailed insights on the nature of interactions between
viruses and their host cells. Connections between DNA polymerase biochemistry
and viral biology, will provide a framework for predicting the outcomes of viral
host interactions within microbial communities based on DNA sequence data
gathered from entire microbial communities (known as metagenomic sequence data).
Over the longer term, improved understanding of viral-host interactions within
ecosystems will provide one component of the foundational information needed for
future green technologies that will help in sustaining both natural and
engineered agri-ecosystems. This multidisciplinary project supports the
education of two PhD students in the fields of microbiology, biochemistry, and
bioinformatics. The investigators and students are mentoring undergraduate
students in laboratory research and provide educational outreach to K-12
students. Students are recruited from populations under-represented in the
scientific workforce when possible. &lt;br/&gt;&lt;br/&gt;Creating a theoretical
framework for predicting the infection phenotypes of unknown viruses based on
genes within the replication module (i.e., the replicon) is the overarching
objective of this interdisciplinary project. Using experimental and
computational approaches, the project is seeking to uncover hypothesized genome
to phenome linkages between the replicon and infection phenotypes of unknown
viruses. Experimental objectives include: 1) synthesis of Family A DNA
polymerase (PolA) enzymes representing a broad cross-section of PolA diversity
within viruses; 2) in vitro biochemical characterization of viral PolA
replicases (quantitative data on polymerase speed, strand displacement,
processivity, exonuclease activity, and fidelity); 3) in vivo assessment of how
changes in PolA impact phage infection dynamics; 4) development of a
classification scheme for viruses based on the phylogeny of PolA and the genetic
composition of the replicon; 5) development of genome to phenome rules that
predict the infection phenotypes of unknown viruses based on PolA replicon
classification groups; and 6) a comprehensive biogeographic study of phage
infection phenotypes within the global ocean based on existing virome data and
the application of predictive genome to phenome rules based on the PolA
replicon. The success of the research will rely on an existing collaborative
interdisciplinary team with expertise in enzyme biochemistry, phage biology,
bioinformatics, microbial oceanography and molecular
genetics.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.