* 2028902
* Folded state dynamics and hydration in transcriptional regulation
* BIO,MCB
* 09/01/2020,08/31/2024
* Gregory Poon, Georgia State University Research Foundation, Inc.
* Standard Grant
* Ishita Mukerji
* 08/31/2024
* USD 832,243.00

This research is aimed broadly at understanding fundamental principles that
govern genetic regulation along the evolutionary tree of life. Higher-order
organisms, such as animals, have complex body plans and well-defined
developmental stages. To meet these complicated physiologic demands, gene
expression in high order organisms is controlled by finely tuned molecular
mechanisms that are absent in single-cell counterparts. To date, many basic
details of these mechanisms remain unknown. The objective of this project is to
elucidate the role of dynamics in gene regulation. Dynamics represent a
fundamental property of biological molecules and are intimately related to their
interactions with their water-filled environment. The knowledge gained from this
project will advance our understanding of emerging concepts in gene regulation
that have thus far received little attention. While completing this project, the
PI will also develop novel methods of computation and data analysis that will
apply to tackling other scientific problems, thus benefiting the wider research
community. This project will introduce young students to topics and careers in
the physical sciences by a coordinated effort through Science ATL, a local
nonprofit organization that produces events and community-building activities
that enhance access to STEM opportunities. &lt;br/&gt;&lt;br/&gt;The folded
states of proteins, nucleic acids and their complexes exist as conformational
ensembles in dynamic fluctuation. Folded state dynamics play a major role in the
affinity and specificity of protein/DNA interactions. As binding in solution
also involves a redistribution of water-accessible surface area, hydration and
dynamics are coupled contributors in protein/DNA recognition. In contrast with
prokaryotic gene regulators, eukaryotic transcription factors form distinct
complexes with cognate DNA sequences that invoke divergent biological outcomes.
These complexes assume discrete structures that vary in a discontinuous, non-
continuum manner with binding affinity. This property, which we term sequence-
directed tuning, is central to the regulation of complex eukaryotic genomes. The
objective of this project is to unveil the dynamic and hydration contributions
to sequence-directed tuning. To achieve this objective, 1) the project will
establish a volumetric framework to analyze the hydration and dynamics
contributions to protein/DNA recognition. 2) A joint solution NMR and molecular
dynamics approach to dissect the transduction pathways and the relevant DNA
dynamics that initiate sequence-directed tuning, 3) the dynamic constraints that
drive the evolution of sequence-directed tuning in eukaryotic transcription
factors will be defined. Using ancient helix-turn-helix (HTH) motifs as models,
the project will query the dynamic requirements for non-continuum DNA binding in
characteristic HTH-based motifs that are conserved across the kingdoms of life.
This project is supported by the Molecular Biophysics Cluster of the Molecular
and Cellular Biosciences Division in the Biological Sciences
Directorate.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.