* 2029216
* EDGE FGT: Functional Genomics in Gar - Discovery Tools for Major Vertebrate Transitions
* BIO,IOS
* 11/01/2020,10/31/2025
* Allyse Ferrara, Michigan State University
* Standard Grant
* Theodore Morgan
* 10/31/2025
* USD 1,703,340.00

The biodiversity of vertebrate animals includes 30,000 land vertebrates
(tetrapods, including humans) as well as &gt;30,000 species of modern fishes
(teleosts). It remains challenging to compare the two groups directly because
they significantly changed from their common ancestor 450 million years ago:
tetrapod bodies underwent major transitions when they emerged from fishes and
began to live on land; teleosts drastically changed their genome organization
and gained thousands of extra genes. This NSF EDGE project develops the archaic
and uniquely informative garfishes (Lepisosteidae) as research organisms that
bridge the disparate worlds of tetrapods and teleosts. Their genome resembles
the tetrapod genome in its organization while gars develop very similar to
teleosts, making gars a “Rosetta Stone” to understand vertebrate evolution and
our own biological origins. The project will improve gar husbandry and spawning
techniques and develop novel methodologies for genetic manipulations such as
CRISPR genome editing to test gar gene functions. This will make gars available
as experimental species for comparative genomic, evolutionary, developmental,
physiological, behavioral, regeneration, and ecological research among the broad
community of vertebrate biologists. The project will train the next generation
of vertebrate biologists through laboratory experiences for K-12 and
undergraduate students, gar conferences, and workshops on vertebrate
evolutionary genetics. Public seminars, animal donations to aquaria and zoos,
and strong social media presence will strengthen scientific literacy of the
general public by emphasizing the importance of fishes for understanding
evolution, genetics, and development. Thereby, the project will raise awareness
for the ecological significance of declining gar
populations.&lt;br/&gt;&lt;br/&gt;The evolution of vertebrates has been
accompanied by lineage-specific gains and losses of genes and morphological
structures. Research organisms with informative genomes, development, and
phenotypic traits are thus essential to understand vertebrate origins, major
transitions, their biodiversity, and genome functions. This project will enable
the investigation of broad biological questions across vertebrate evolution by
developing genome-to-phenome technologies for the spotted gar (Lepisosteus
oculatus) that occupies a key phylogenetic position among bony vertebrates. Gars
serve both as an experimentally accessible, "unduplicated" outgroup to the
teleost fishes that went through a whole genome duplication event early in their
evolution as well as an outgroup to lobe-finned vertebrates including tetrapods.
The gar genome assembly has been used in 300+ genomic studies, but functional
genetic investigations that enable direct testing of genotype-to-phenotype
relations in gar are currently lacking. This project leverages ongoing research
efforts in order to facilitate the genome enablement of gar. Resources and
genetic tools to be developed include: 1.) improvements in gar spawning
techniques and infrastructure to perform molecular genetic experiments in gar;
2.) advances in CRISPR genome editing, targeted gene silencing, and transgenic
technologies for the gar model; and 3.) dissemination of project results through
a project-specific online database, two Gar Conferences, and two Workshops on
Functional Vertebrate Evo-Devo Genomics. The project thereby seeks to understand
the role of a select set of gar genes and regulatory elements as proof-of-
principle to unlock the full potential of gars as model organisms for broad
research questions in comparative vertebrate biology.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.