* 2029595
* RAPID: finding virulence genes as therapeutic targets in Covid-19
* BIO,DBI
* 04/15/2020,03/31/2022
* Jonathan Arnold, University of Georgia Research Foundation Inc
* Standard Grant
* Steven Ellis
* 03/31/2022
* USD 200,000.00

This RAPID award is made by the Division of Biological Infrastructure (DBI)
using funds from the Coronavirus Aid, Relief, and Economic Security (CARES) Act.
This award to the University of Georgia will leverage bioinformatics tools to
identify virulence genes that assisted SARS-COV-2 in jumping species boundaries.
Identifying shared genetic changes between human SARS-COV-2 and its relatives in
mammalian reservoirs in the bat, pangolin, and civit is important to
understanding the virus mode of action. It is especially critical to identify
which of the changes contribute to human virulence of SARS-COV-2. The
researchers will analyze 3200+ genomes from SARS-COV-2 and coronavirus genomes
from animal reservoirs to look at genetic changes associated with virulence and
COVID-19 symptoms. To accomplish this work, new software tools will be developed
to better leverage modern computer graphics processing units (GPUs) for high-
speed parallel computing. The project will engage graduate students and REU
participants in interdisciplinary research of critical national importance and
contribute useful tools for future bioinformatic
research.&lt;br/&gt;&lt;br/&gt;The objective of this project is to identify
virulence genes that enable SARS-COV-2 to jump species boundaries from
vertebrate reservoirs. The project has three main components: (1) building gene
trees within species trees of coronavirus by coalescent methods; (2) testing for
shared polymorphisms under selective sweep between natural reservoirs and human
SARS-COV-2; (3) validating candidates identified in task (2) independently by
comparing gene genealogies for selective sweep (under selection for virulence)
vs. incomplete lineage sorting. An interdisciplinary team from genetics,
bioinformatics, and infectious diseases will develop gene trees within a species
tree of 3,200+ published SARS-COV-2 genomes using a novel BEST coalescent method
developed for GPUs. Results from these studies will be rapidly disseminated and
published in peer-reviewed journals, presented at scientific conferences, and
made available through shared software repositories.&lt;br/&gt;&lt;br/&gt;This
award reflects NSF's statutory mission and has been deemed worthy of support
through evaluation using the Foundation's intellectual merit and broader impacts
review criteria.