* 2030377
* RAPID: Exploring Covid-19 RNA Viral Targets By Graph-Theory-Based Modeling
* MPS,DMS
* 05/15/2020,04/30/2022
* Tamar Schlick, New York University
* Standard Grant
* Zhilan Feng
* 04/30/2022
* USD 200,000.00

The urgently needed treatments and vaccines for COVID-19 rely on a fundamental
understanding of the complex viral apparatus. This project will determine the
structural properties and drug-binding potential of two regions of the viral RNA
essential for invasion and propagation of the COVID-19 genome in host cells:
genes responsible for making spike and fusion proteins. Specifically, the
project will develop new and efficient graph-theory based computational
algorithms for identifying subregions in the COVID-19 viral genome that alter
the RNA substructure when they are mutated. Identification of these subregions
will aid in the discovery of anti-viral inhibitor compounds. Graph theory tools
already developed in the PIâ€™s lab offer coarse-grained approaches for RNA
structural analysis and design. The PI will combine these tools with
biomolecular modeling to examine the therapeutic potential of anti-viral
inhibitors known from SARS, MERS, and other viruses. This project will produce
structural insights into the RNA viral regions and identify critical nucleotides
and candidate inhibitors that will help make progress against COVID-19. The
research has profound impact to COVID-19 as well as other coronaviruses that
could emerge in the future. The project offers unique interdisciplinary training
in mathematics, biology, chemistry, and scientific computing for young
scientists, including women and minorities. The research results will be shared
rapidly with the COVID-19 research community at large.&lt;br/&gt;&lt;br/&gt;RNA-
targeting approaches have therapeutic potential due to the high sequence and
structure conservation of the viral genomes and the rapid emergence of CRISPR
technology. They also present alternatives when protein-inhibiting compounds
lead to invasion of the RNA viral genome itself. Such compounds that alter the
RNA structure significantly are expected to inhibit viral invasion and
replication. Because the fusion-protein coding region contains a pseudoknot
(intertwined base-pair) substructure involved in a frame-shifting mechanism, the
determination of critical mutation regions and associated compounds that destroy
this pseudoknot will be invaluable. The project team has rich experience in
biomolecular modeling and simulation of nucleic acid complexes and has developed
a graph-theory framework for analyzing RNA motifs, predicting structures, and
designing novel RNA folds. The graph-theory framework will be extended and
applied in this project to address the COVID-19 pandemic by determining key
regions in the RNA COVID-19 viral genome and associated chemical inhibitors that
would interfere with viral fusion into and replication within host cells. With
this award, the Mathematical Biology Program in the Division of Mathematical
Sciences and the Chemistry of Life Processes Program in the Division of
Chemistry are funding Dr. Schlick from New York University to determine the
structural properties and drug-binding potential of the COVID-19 viral
RNA.&lt;br/&gt;&lt;br/&gt;This grant is being awarded using funds made available
by the Coronavirus Aid, Relief, and Economic Security (CARES) Act supplemental
funds allocated to MPS.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.