* 2031204
* RAPID: Analyses of polymorphism and divergence to illuminate molecular evolution permissive of zoonoses in SARS and COVID-19
* BIO,DEB
* 06/01/2020,05/31/2021
* Jeffrey Townsend, Yale University
* Standard Grant
* Katharina Dittmar
* 05/31/2021
* USD 122,716.00

COVID-19 disease has caused the third major outbreak in the past two decades
resulting from a spillover of an animal coronavirus to humans. Among them, it is
by far the most severe. Novel diseases typically appear because of such
spillovers. However, it is unknown what genetic changes in the virus causing
COVID-19 enabled it to infect humans. To identify those genetic changes, this
project will use newly developed computational methods to compare the genes in
coronaviruses infecting humans to the genes in coronaviruses infecting other
animal hosts. This comparison of viral genomes will provide insight into which
sites within coronavirus genomes enable the switching of host species as well as
which sites change following host switches. Awareness of these genetic
mechanisms is critical to determining if general rules underlie the ability of
coronaviruses to switch hosts, as well as to provide necessary historical
context for the changes coronavirus genomes have experienced following the onset
of human infections. The ensuing knowledge provides precise guidance on new
targets for ongoing decisions regarding vaccine design and drug development.
Results of this project will also be incorporated into multiple engaging and
educational exhibits at the North Carolina Museum of Natural Sciences.
&lt;br/&gt;&lt;br/&gt;This project will apply molecular evolutionary approaches
to reveal the rates of evolution of SARS-CoV-1, SARS-CoV-2, and of viruses
sequenced from reservoir hosts, revealing the strength of selection across sites
occurring proximate to zoonosis within the viral genomes. First, phylogenetic
comparisons of extant SARS-causing, SARS-like, and COVID-19-causing viral
sequences collected from infected humans and from animal reservoirs will be used
to reconstruct the history of coronavirus evolution. Second, the rate of change
of each nucleotide within each gene—and each amino acid site within each
protein—in viruses that were transmitting within the animal reservoir, in
viruses transmitted among humans during the SARS outbreak, and in viruses
transmitted among humans during the COVID-19 pandemic will quantified. Third,
virus gene sequences that bracketed (pre- and post-) the host transition events
for both SARS and COVID-19 will be estimated through phylogenetic ancestral
state reconstruction methods. Finally, application of a novel computational
approach comparing the divergence between reconstructed ancestors to the
polymorphism present during the outbreak will identify genomic sites under
selection that are associated with host transitions. This divergence will be
mapped to known protein domains and structures, illuminating sites important to
human infection and transmission, and thereby aiding molecularly targeted
vaccine and therapy development. This RAPID award is made by the Systematics and
Biodiversity Science Cluster in the Division of Environmental Biology, using
funds from the Coronavirus Aid, Relief, and Economic Security (CARES)
Act.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.