* 2032357
* Single-cell feedback, optogenetics, and deep learning to control gene expression in bacteria
* BIO,MCB
* 08/15/2020,07/31/2024
* Mary Dunlop, Trustees of Boston University
* Standard Grant
* Anthony Garza
* 07/31/2024
* USD 820,298.00

Heterogenous gene expression changes in bacterial communities are thought to
produce subpopulations with particular survival benefits such as antibiotic
tolerance. Currently, scientists have few tools to mimic and study these gene
expression events in a controlled manner. The overall goal of this project is to
develop light-based tools for precisely driving gene expression in targeted
bacterial cells. The tools have the potential to reveal how heterogenous changes
in gene expression impact the survival of subpopulations of bacterial cells. In
addition to these scientific advances, the research program is complemented by
educational and outreach activities. These include partnering with the Tumble
Science Podcast for Kids to create episodes targeted for young audiences (ages
6-12). The project also provides training opportunities for undergraduate
students and includes additional outreach efforts related to K-12 science
education. &lt;br/&gt;&lt;br/&gt;The technical goal of this project is to
develop methods to precisely drive gene expression in single cells, and to test
these techniques in the context of bacterial stress response. To achieve this,
the researchers will use a high-throughput approach that uses automated
microscopy, optogenetics, and microfluidics to implement real-time feedback
control. Cellular processes are noisy and nonlinear, and can involve delays and
differential responses between genetically identical cells. Thus, traditional
control algorithms are either not accurate enough or too computationally
expensive to be appropriate for large-scale analysis of gene regulatory network
dynamics. This proposal takes a new approach, taking advantage of emerging
control algorithms based on deep learning models. The researchers will conduct
high-throughput experiments where optogenetic light signals are applied to many
single cells to generate datasets for training a deep learning model. The deep
learning model will then be used to implement feedback control in single cells
using Deep Model Predictive Control to rapidly predict the optimal optogenetic
stimulus to apply. The ultimate utility of these tools is in their ability to
precisely control genes within biological networks. To test this potential, the
researchers will focus on the acid stress response network. By using feedback
control to impose dynamic profiles on network regulators, the researchers will
study how signals propagate within the network and how cells use gene expression
dynamics to hedge against the sudden appearance of
stress.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has
been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.