* 2034015
* EAGER: spRNA-seq: high-throughput transcriptome analysis of single plastids
* BIO,IOS
* 08/01/2020,07/31/2024
* Xuemei Chen, University of California-Riverside
* Standard Grant
* Kimberly Gallagher
* 07/31/2024
* USD 300,000.00

Plastids are ubiquitous and essential organelles in plant cells. They come in
diverse types, with chloroplasts being the most well-known plastid type. Through
photosynthesis, chloroplasts harvest solar energy and convert carbon dioxide
into plant biomass, and thus sustain almost all life on Earth. In fact, the
functions of plastids are not limited to photosynthesis â€“ other types of
plastids exist and perform a variety of essential functions in development,
metabolism, signaling, and immunity in plants. Plastids and their diverse
functions can be harnessed for the benefit of humans and the environment.
However, our knowledge of plastids is severely limited by current technologies.
Each plant cell harbors tens to hundreds of plastids and current approaches can
either study a few plastids at a time, such as by microscopy, or millions of
them in bulk, such as by molecular techniques. The former lacks efficiency while
the latter averages potentially distinct plastid types. The proposed experiments
are aimed at establishing a new technology to study plastids, namely single-
plastid RNA sequencing (spRNA-seq). This technology determines the molecular
signatures of hundreds of thousands of individual plastids in one experiment,
making it possible to efficiently and fully understand plastid diversity and how
they function. The technology is expected to revolutionize research on plastids
and will generate novel insights into how plastids affect plant biology,
ecology, and evolution. This knowledge can be used to harness the power of
plastids for the betterment of life on Earth.&lt;br/&gt;&lt;br/&gt;The PIs of
this collaborative project plan to use plastids from Arabidopsis thaliana and
Pisum sativum, as both technical controls as well as biologically comparative
models, to establish the spRNA-seq method. The project comprises three
integrative components. First, a genomics strategy is designed, optimized, and
implemented to achieve spRNA-seq that is reproducible, cost-effective, and user-
friendly. In particular, plastids will be isolated and a split-and-pool strategy
will be employed to enable in-plastid combinatorial barcoding during library
preparation, with each plastid being identified bioinformatically through the
unique combination of barcodes. Next, a suite of computational and statistical
approaches will be applied to evaluate the technical outcomes of spRNA-seq, and
more importantly, to glean insights into new plastid types through
transcriptomic heterogeneity. As the plastid transcriptome is heavily molded by
posttranscriptional mechanisms such as splicing, editing, and endonucleolytic
and exonucleolytic processing, potential inter-plastid heterogeneity with
respect to these posttranscriptional events will be interrogated. Finally,
single-molecule RNA in situ hybridization will be deployed for validation of
plastid heterogeneity. This complementary approach adds spatial resolution to
provide further insights into plastid heterogeneity within a cell or tissue.
Further, the project will train the next generation of scientists, including
underrepresented minority students at University of California Riverside, in
critical thinking, experimental design and execution, and scientific
communications. The research team will disseminate this empowering technology
through publications, conference presentations, and training videos to propel
scientific discovery.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory
mission and has been deemed worthy of support through evaluation using the
Foundation's intellectual merit and broader impacts review criteria.