* 2036532
* SBIR Phase II:  Reagent Development for a Rapid Enzymatic DNA Synthesis Platform
* TIP,TI
* 04/15/2021,03/31/2023
* Daniel Arlow, Ansa Biotechnologies, Inc.
* Cooperative Agreement
* Erik Pierstorff
* 03/31/2023
* USD 994,682.00

The broader impact of this Small Business Innovation Research (SBIR) Phase II
project is to optimize a system to rapidly synthesize long, user-defined
sequences of DNA as a commercial service. The process will be capable of
producing error-free DNA fragments, each long enough to encode multiple genes,
in 1-2 weeks, at a price that will be affordable even to university research
labs. Researchers currently can purchase short DNA fragments that must be
stitched together to make functional genes – which is labor-intensive and
sometimes impossible – or pay vendors to do it. The ability to buy an intact
group of genes on one piece of DNA will help deepen the understanding of all
biological systems, from animals and plants to the bacteria and viruses that
infect them. Easy access to long DNAs will also allow synthetic biologists to
build completely novel biological devices, such as bacteria that manufacture
vitamins or medicines, cells that detect and destroy cancer, or new sustainable
food ingredients and novel biomaterials.&lt;br/&gt;&lt;br/&gt;The proposed
project is focused on developing an enzymatic method for DNA synthesis that will
alleviate several problems inherent to chemical DNA synthesis, the only method
currently available commercially. Chemical synthesis works well for short DNA
fragments, but high-quality synthesis is limited to &lt;200 bases. Longer
fragments often are created by stitching together many short fragments, but this
process is unreliable for sequences that contain repeats or high AT or GC
content. In the proposed enzymatic DNA synthesis method, a single
deoxynucleoside triphosphate (dNTP) is conjugated to the template-independent
polymerase Terminal Deoxynucleotidyl Transferase (TdT) by a cleavable linker.
When presented with a DNA primer, the dNTP-TdT conjugate adds its tethered
nucleotide and remains attached, blocking further elongation by other
conjugates. Cleavage of the linker releases the TdT, to be washed away with
unreacted conjugates, and exposes the oligo for extension with the next dNTP-TdT
conjugate. These two steps of “extension” and “deprotection” are iterated to
synthesize a defined sequence. The accuracy of the resulting DNA strand depends
on the stability and cleavage efficiency of the linker. A variety of linkages
and cleavage enzymes will be tested to yield &gt;99.9% stability and cleavage in
rapid, 30-second reactions.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.