* 2039489
* Context-dependent activity of an essential transcription factor
* BIO,IOS
* 05/01/2021,10/31/2022
* Justin Walley, Ohio State University
* Continuing Grant
* J.D. Swanson
* 10/31/2022
* USD 692,244.00

Multicellular organisms acquire their complex shapes through carefully
controlled developmental processes. Key regulators of plant development include
a class of proteins called transcription factors, which coordinate the
specificity and function of plant cells and tissues. The action of some
transcription factors are limited to only a few cells or a subset of tissues,
but there are other factors that play key roles in a large number of developing
tissues. This raises an interesting biological question: how can the same
transcription factor generate different developmental outcomes? This project
answers the question by identifying how the same transcription factor can serve
different functions in various cells and tissues. The project combines
techniques of genetics and biochemistry, including advances in high-resolution
mapping of protein-protein interactions. Central to this proposal is the
inclusion of students traditionally excluded from research-intensive
insitituions. At The Ohio State University (OSU), this is manifested in Black
students and students-of-color being overrepresented at OSU regional campuses,
where research opportunities are limited. To mitigate this problem,
undergraduates will be brought to the OSU Columbus main campus, provided food
and lodging, paid a stipend, and enrolled in a &gt;100-student consortium
consolidating the various concurrent summer research programs. This pipeline
broadens participation of traditionally excluded groups by eliminating the
unfair choice between working a summer job and exploring research as a
career.&lt;br/&gt;&lt;br/&gt;Biological development results from tightly
controlled gene expression driven by the activity of transcription factors
(TFs). TFs have been repeatedly coopted throughout evolution to regulate key
developmental advances, in part because their outputs vary depending on
developmental context. Mechanisms that generate these context-dependent readouts
remain largely unknown, and this knowledge gap is particularly large in the
plant kingdom. This gap will be addressed using CLASS III HOMEODOMAIN-LEUCINE
ZIPPER (HD-ZIPIII) TFs as a model. HD-ZIPIII proteins are a particularly
attractive model as they regulate a broad range of essential developmental
processes, acting in tissues and organs ranging from meristems to roots to
lateral organs. In addition, HD-ZIPIII protein architecture includes a StAR-
related lipid transfer (START) domain. START domains are typically activated by
lipophilic ligands and often mediate protein-protein interactions, presenting
possible mechanisms to generate context-dependent activity. This proposal will
test the genetic consequences and functional partnerships of the representative
HD-ZIPIII PHABULOSA (PHB) in two functionally-distinct cell types: the
indeterminate cells of the shoot apical meristem and the determinate cells of
the adaxial side of leaves. Experiments including global genomic profiling and
proximity labeling will identify meristem- and leaf-specific PHB gene regulatory
networks and interactomes, and test whether the START domain plays a role in
generating context-dependent activity.&lt;br/&gt;&lt;br/&gt;This award reflects
NSF's statutory mission and has been deemed worthy of support through evaluation
using the Foundation's intellectual merit and broader impacts review criteria.