* 2040697
* Collaborative Research: RUI: A multiscale quantification of plasmid acquisition in Escherichia coli pathogens
* BIO,MCB
* 03/15/2021,02/29/2024
* Jonathan Snow, Barnard College
* Standard Grant
* David Rockcliffe
* 02/29/2024
* USD 371,643.00

The overall goal of this project is to better understand what enables some
bacteria to adapt more successfully than others. Bacterial pathogens often share
beneficial genetic material amongst themselves, enabling them to quickly adapt
to their surroundings. The process of sharing genetic material, known as
horizontal gene transfer (HGT), is the most common way that bacteria rapidly
adapt to diverse environments. While much focus has been placed on
characterizing pathogens that have already acquired beneficial traits via HGT,
relatively little is known about the immediate adaptation that cells must
undergo to successfully acquire such traits. This project investigates the
adaptation process following HGT to provide insights into why certain pathogens
are more successful (and therefore, prevalent) than others. The results and
insights generated are applicable to a wide variety of biological areas and open
questions and promotes ongoing and future collaborative efforts across the
microbiology research community. In addition to its scientific objectives, this
project facilitates significant educational opportunities for undergraduate
students, particularly women and those from backgrounds underrepresented in STEM
disciplines. Specifically, this project closely integrates research methods and
results into a newly developed Computational Biology major and associated upper-
level courses. Students are empowered with vital computational skills as well as
an understanding and appreciation of cutting-edge research techniques. This
project supports four students to pursue hands-on research year-round,
benefiting from close guidance from the principal
investigator.&lt;br/&gt;&lt;br/&gt;Horizontal gene transfer, (HGT), particularly
through the transfer of plasmids via direct cell-cell contact (termed
"conjugation"), is the most common way that bacterial pathogens adapt to
environmental stressors by acquiring catabolic, virulence, or antibiotic
resistance genes. Previous studies have primarily focused on the plasmid fitness
cost as a determinant of plasmid-strain success: strains bearing high-cost
plasmids are either out-competed or evolve compensatory mutations that
ameliorate the plasmid's metabolic burden. In addition to the fitness cost,
acquiring a plasmid introduces an immediate, but transient, disruption to
metabolism which also impacts population growth dynamics. The impacts of these
short-term effects remain an understudied feature of conjugation dynamics. This
project synergistically leverages longitudinal transcriptomics, computational
modeling, and whole-genome sequencing to investigate the mechanistic
underpinnings of this acquisition cost. In so doing, the project directly
connects gene expression patterns, arising as a consequence of plasmid
acquisition, to their population-level effects. The first objective elucidates
the mechanistic determinants that lead to plasmid acquisition cost for the
representative plasmid RP4, using a combination of transcriptomics,
genetic/biochemical validation, and metabolic network modeling. The second
objective determines how plasmid acquisition impacts population dynamics and
clonal dominance of Escherichia coli pathogens isolated from wastewater using
conjugation experiments, genomics, and mathematical modeling. By combining
multiple scales of data, this project elucidates the genetic determinants
underlying plasmid acquisition, and leverages this knowledge to predict the
short-term, long-term, and competitive dynamics of populations undergoing
conjugation.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission
and has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.