* 2041202
* RUI: Transcription Factor Discovery in the Extremophile Thermus Thermophilus
* BIO,MCB
* 07/01/2021,06/30/2024
* Michael Van Dyke, Kennesaw State University Research and Service Foundation
* Standard Grant
* Clifford Weil
* 06/30/2024
* USD 330,011.00

This award funds an investigation of transcription factors and their potential
biological roles in an extremophile bacterium that survives in very high
temperatures that would be lethal to plants and animals. Bacterial transcription
factors are proteins that bind to DNA and turn nearby genes on or off in
response to environmental or internal cues. Regulation is essential so that the
bacteria can survive stress and reproduce as efficiently as possible. Presently,
the DNA binding sites for most bacterial transcription factors are unknown. The
lab has developed a novel selection method that can be conducted in a test tube
and allows them to screen billions of DNA sequences to find ones that a
transcription factor can bind. Having identified those sequences it is possible
to find those sequences in the genome sequence of the bacterium and then infer a
likely biological role for the transcription factor. This project aims to
identify the DNA binding sites for transcription factors in the extremophile
Thermus thermophilus, a model organism whose proteins are widely used in
biotechnology and biomanufacturing. This project has implications for many
industries, ranging from agribusiness and medical testing to consumer products
and waste treatment. These applications could be improved through better
understanding gene regulation in such extremophiles. This project will also
train both undergraduate and masterâ€™s students in the biochemistry and
bioinformatics of protein-DNA interactions, thereby providing future scientists
with expertise in these fields. &lt;br/&gt;&lt;br/&gt;This research project
seeks to better understand transcriptional regulatory networks in the model
extremophile T. thermophilus HB8. The binding specificity of putative
transcription factors will be determined using the iterative selection method
Restriction Endonuclease Protection, Selection, and Amplification, followed by
massively parallel semiconductor sequencing and Multiple Expectation
Maximization for Motif Elicitation analysis. Potential binding sites will then
be mapped to the Thermus thermophilus HB8 genome using Find Individual Motif
Occurrences and functionally validated in vitro (electrophoretic mobility shift
assays, biolayer interferometry) and in vivo (quantitative Polymerase Chain
Reactions, microarray data). Information from this study will provide a better
understanding of regulons within a model bacterial extremophile. They will also
contribute to ongoing collaborative projects, for example, the Structural-
Biological Whole Cell Project, which seeks to understand the molecular
constituents of life at an atomic level. Finally, this project's methods should
broadly apply to characterizing orphan transcription factors and defining
regulons in many other organisms.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.