* 2041378
* Elucidation of translational regulatory mechanisms of plant immune responses
* BIO,IOS
* 07/01/2021,06/30/2025
* Xinnian Dong, Duke University
* Continuing Grant
* Courtney Jahn
* 06/30/2025
* USD 1,000,000.00

Since plants do not have the specialized immune cells present in animals,
activation of immunity in plants involves switching from producing growth-
promoting proteins to defense proteins. Even though the process of generating
templates (RNA transcripts) for defense proteins has been studied extensively,
how the templates are selectively translated into proteins remains largely
unknown. This major knowledge gap significantly impedes progress in developing
new strategies of engineering disease resistant crops in agriculture as well as
in bioengineering in general because controlled protein production is central to
most biological functions. The long-term goal of this project is to elucidate
the regulatory mechanisms by which plants produce defense proteins during immune
responses. To achieve this goal, new technologies have been developed or adapted
to examine the multiple steps connecting RNA transcripts to protein synthesis.
Using translational switches to tighten the regulation of defense protein
translation will minimize the yield penalty often associated with broad-spectrum
disease resistance and revolutionize the practice of controlling crop diseases,
which has mainly been focused on the disease-specific type of resistance. This
project may also benefit other industries and medicine by improving recombinant
protein production. The success of the project will also add to the pipeline of
scientists from high school students to postdoctoral researchers with strong
STEM training in performing frontier research.
&lt;br/&gt;&lt;br/&gt;Identification of genes with altered translation
efficiency during plant immune responses led to the discovery of novel
regulators. Surprisingly, immune-related translation reprogramming in plants has
been found to be distinct from the integrated stress response pathway found in
yeast and mammals, indicating that plants use alternative regulatory mechanisms
to control defense proteome translation. To elucidate these regulatory
mechanisms, whole genome translatome analyses were performed for both pattern-
triggered immune (PTI) and effector-triggered immunity (ETI) and a consensus
sequence consisting of mostly purines (“R-motif”) was discovered in the
5’-leader sequences of translationally induced mRNAs during PTI. The R-motif was
found to be not only necessary, but also sufficient for the PTI-induced
translation. Its regulation is through the interaction with poly(A)-binding
proteins (PABPs). Moreover, m6A-seq and SHAPE-MaP have been established to
examine mRNA modifications and in vivo mRNA structural dynamics, respectively.
With the establishment of these cutting-edge technologies, this project will
focus on two specific aims: Aim 1: Study the effects of upstream open reading
frames (uORF) on translation of the main ORF (mORF) in response PTI induction to
test the hypothesis that RNA modification and secondary structure are critical
determinants of ribosome accessibility to uORFs in controlling mORFs translation
during PTI. Aim 2: Elucidate the signaling pathway leading to translatomic
changes during PTI using both forward and reverse genetic approaches. The
ultimate goal is to connect the various translational cis- and trans-elements
with the currently known defense signaling network to fill the knowledge gap on
how plants reprogram the proteome to mount immune
responses.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.