* 2043478
* OPUS:  Genome doubling in an evolutionary model: Synthesis across biological and temporal scales
* BIO,DEB
* 06/01/2021,05/31/2024
* Douglas Soltis, University of Florida
* Standard Grant
* Matt Carling
* 05/31/2024
* USD 315,682.00

Genome doubling, which doubles all genetic information in a cell, is a major
evolutionary force that has produced much of the species diversity on Earth.
Even the human genome has duplicated segments that result from ancient genome
doubling in early vertebrates hundreds of millions of years ago. Although it
occurs in animals and fungi, genome doubling is particularly common in plants.
Organisms that have undergone genome doubling are often more successful than
their non-duplicated parents and have new traits; for example, plants with
doubled genomes are often larger than their parents and produce more fruits and
seeds. In fact, most agricultural crops and the worldâ€™s worst weeds have all
originated through this process. As a result, genome doubling is very important
both ecologically and economically, yet scientists do not yet understand how the
presence of duplicated genes produces new traits. Although the earliest stages
of this process are the most critical for understanding the connection between
genes and traits, it is precisely these early stages that are the most difficult
to study, mainly because nearly all genome doubling events occurred in the very
distant past, millions of years ago. In this project, the researchers will
investigate these early stages by focusing on two species of plants in the
sunflower family that formed in nature via genome doubling in the past 90 years.
These species are unique in that their time of origin is so recent; as a result,
they are perfect models for learning about how genome doubling generates new
traits and how this process can be used to address societal problems related to
food security, human health, and climate change. &lt;br/&gt;&lt;br/&gt;Whole-
genome duplication (WGD, or polyploidy) is a major evolutionary force, yet the
critical early stages of polyploid evolution remain poorly understood. This
project will promote novel understanding of WGD by providing new insights into
recurring polyploid formation and the consequences of genomic merger through
analysis of data compiled over the past 30 years for Tragopogon, a genus of
plants from the sunflower family that includes two recently (~90 years old) and
repeatedly formed natural allotetraploids (T. mirus, T. miscellus) and their
diploid parents (T. dubius, T. pratensis, T. porrifolius). New tools, including
genetic manipulation via CRISPR-Cas9, will be applied, and new long-read
sequence data for both genomes and transcriptomes will be acquired to maximize
the value of available data. New analyses to address fundamental questions
regarding WGD will focus on: 1) Distribution of natural populations of the
polyploids and their diploid parents; 2) Introductions of the diploid parents
from Europe to U.S.; 3) Crossability between tetraploids of separate origin; 4)
Genetic/genomic consequences of polyploid formation (transcriptomes; reference
genomes; proteomes; alternative splicing); 5) Properties of synthetic lines; and
6) Ecological comparisons of polyploids and their diploid parents (greenhouse
data; niche modeling). Emphasis will be placed on synthesizing information on
the early stages of WGD in Tragopogon and other systems, both natural and
synthetic. This broad synthesis will not only provide new insights into WGD, but
also set the stage for future investigations by diverse researchers to
investigate the patterns, tempo, mechanisms, and forces driving and shaping
post-WGD genome evolution.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's
statutory mission and has been deemed worthy of support through evaluation using
the Foundation's intellectual merit and broader impacts review criteria.