* 2046550
* CAREER: Machine Learning Approaches to Understanding Molecular Mechanisms Underlying Convergent Evolution of Vocal Learning Behavior
* BIO,DBI
* 06/01/2021,05/31/2026
* Andreas Pfenning, Carnegie-Mellon University
* Continuing Grant
* Jean Gao
* 05/31/2026
* USD 411,333.00

The ability to perform a variety of complex behaviors, like human speech, is
encoded in the billions of nucleotides that make up the genome of an organism.
Although speech itself is uniquely human, vocal learning, the ability to modify
vocal output as a result of experience, has evolved independently in multiple
mammals and birds, including songbirds, parrots, hummingbirds, bats, and whales,
as well as humans, uniquely among great apes. During the evolution of each of
these species, genome sequence mutations over millions of years have led to
differences in the molecular properties of cell types within their brains,
allowing for their vocalizations to be learned. This project leverages that
diversity across species to take a comparative genomic approach to understanding
how vocal learning evolved: what features do the genomes of vocal learning
species have in common relative to species without this ability? To answer that
question, this research will develop artificial intelligence methods to look for
common patterns of gene activity across dozens of brain cell types and common
genome sequence patterns across hundreds of mammals. In addition to linking
vocal learning behavior to specific cell types and genome sequence mutations,
the artificial intelligence methods that will be developed have the potential to
be applied to study the evolution of additional behaviors and other traits that
vary across species with available genomes. To help facilitate the adoption of
these methods, as well as other new artificial intelligence approaches, this
project seeks to train the next generation of interdisciplinary scientists, who
are experts in artificial intelligence, evolutionary biology, and neuroscience.
Undergraduate neuroscience and biology majors will get the opportunity to
participate in the research by applying the computational techniques to other
behaviors. More broadly, a video and guided tutorials that explain the research
and how to conduct it will be made publicly
available.&lt;br/&gt;&lt;br/&gt;Vocal learning demonstrates striking
similarities across several of the lineages at multiple levels including the
behavior itself, neural circuit features, and even shared gene expression
patterns. Despite this wealth of information, there is still no solid connection
between genotype and phenotype. This project seeks to make that connection by
linking vocal learning-associated gene expression patterns to specific neural
cell types that form the neural circuits for the production of vocalization and
by linking them to variation in genome sequence at candidate regulatory
elements. To uncover the cell type-specific gene expression features associated
with vocal learning behavior, a nested tree probabilistic graphical model-- a
machine learning approach that simultaneously models hierarchies of cell types
and species -- will be applied. Then, to trace the evolution of regulatory
elements associated with those gene expression patterns, convolutional neural
network models that predict differences in open chromatin from differences in
genome sequence will be applied. As a result of these analyses, this project
will produce hypotheses for how differences in genome sequence between vocal
learners and non-learners lead to differences in cell type-specific gene
expression and open chromatin. Beyond the contributions to the study of vocal
learning, the tools developed here fill a growing need for methods to study the
evolution of cell types and regulatory elements across a rapidly increasing set
of genomes. The results of the project will be found at
http://www.pfenninglab.org/project/vocal-2/.&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.