* 2047611
* CAREER: Statistical approaches and computational tools for analyzing spatially-resolved single-cell transcriptomics data
* BIO,DBI
* 07/15/2021,06/30/2026
* Jean Fan, Johns Hopkins University
* Continuing Grant
* Jean Gao
* 06/30/2026
* USD 491,408.00

Both healthy and diseased tissues are comprised of a multitude of interacting
cells representing many different cell-types and cell-states. Rapid advances in
sequencing and imaging technologies are making it possible to profile these
differences for hundreds to thousands of genes in hundreds to thousands of
individual cells and small groups of cells in a spatially resolved manner.
However, statistical methods and computational tools are still needed to model
and analyze these high-dimensional spatially resolved measurements in order to
extract relevant biological insights. This project will develop statistical
approaches for analyzing these spatially resolved transcriptomics datasets in
order to characterize spatial gene expression patterns and associate them with
underlying cellular phenotypes and functions, particularly within tissues. Such
characterization will help provide insights into how spatial context and
organization may impact cellular phenotype and function. The developed
approaches will also be made available to the scientific community as open-
source software and be applicable to other biological contexts. Finally,
instructional material including case study tutorials developed through this
project will be used in a hands-on-learning course to engage young girls in
computer science. &lt;br/&gt;&lt;br/&gt;On a technical level, to analyze
spatially resolved transcriptomics data of tissues, this project will use a
density agnostic encoding of spatial positional information using Voronoi
tessellation to accommodate variations in cell density common to tissues.
Spatial autocorrelation and cross-correlation analyses will be used to identify
genes with significant spatial patterns that may further be indicative of cell-
cell communication. Convolutional neural network-based approaches will be
applied to integrate the subcellular spatial organization of mRNAs in cell
segmentation within tissues and be combined with previous RNA velocity models in
order to infer temporal dynamics and delineate potential cellular migration
within tissues. To interrogate how gene expression variation may be associated
with cellular phenotype, neural networks will also be applied to predict
molecular features directly from histological staining images of tissues.
Finally, a client-side web application will be developed to enable such
exploration and analysis of spatially resolved transcriptomics data within the
browser. For more information regarding the ongoing results of this project,
please see: https://jef.works/NSF_CAREER&lt;br/&gt;&lt;br/&gt;This award
reflects NSF's statutory mission and has been deemed worthy of support through
evaluation using the Foundation's intellectual merit and broader impacts review
criteria.