* 2051283
* IIBR Informatics: Mixture model algorithms for inferring covariance structures and microbial associations from microbiome data
* BIO,DBI
* 06/01/2021,05/31/2024
* Shibu Yooseph, The University of Central Florida Board of Trustees
* Standard Grant
* Reed Beaman
* 05/31/2024
* USD 634,673.00

Microbial communities are found almost everywhere on earth and they play
important functional roles in the environments that they are found in. Microbes
in a community interact with each other as they compete for the food and energy
resources available in their environment. These direct and indirect interactions
between microbes, termed microbial associations, play a large role in
determining the structure, organization, and function of the community. This
project addresses the computational challenge of inferring microbial
associations from microbiome data generated using high-throughput DNA sequencing
technologies. The novel computational tools and resources developed by this
project will enable the advancement of knowledge in several disciplines,
including environmental sciences, medicine, and human health science. This
project will contribute to understanding the rules of life for microbial
ecosystems, and it will further our understanding of the important roles that
microbes play in biogeochemical processes in the environment and in the
progression of microbe-associated diseases. This project will provide
interdisciplinary training for graduate students, with an emphasis on training
under-represented groups (including women and minorities). This project will
also contribute to enabling an increased level of high school student
participation in STEM areas through the development of an education module that
will introduce high-school teachers, via workshops, to introductory topics in
genomics and bioinformatics. &lt;br/&gt;&lt;br/&gt;Microbial associations can be
inferred from the underlying covariance structure that is determined from
microbial taxa abundances. These abundances are often estimated from DNA
sequence data. However, sequence data are compositional in nature, in the sense
that they only provide relative abundance information for taxa, and this poses
challenges when determining microbial associations. Furthermore, associations
between groups of microbial taxa are not always fixed, and they can change when
factors such as resource availability and environmental characteristics vary.
This project will develop novel computational methods to determine the number of
covariance structures in large microbiome datasets and to reconstruct the sets
of microbial associations. These methods will be able to capture both positive
and negative microbial associations from sequence data while dealing with the
challenges posed by the compositional nature of sequence data. The overall
approach is based on a mixture model framework incorporating component
distributions that model microbial abundance data. This project will develop
variational approximation algorithms to determine the number of covariance
structures in a given microbiome dataset, fast numerical optimization algorithms
to estimate the parameters of the mixture model, and an integrated framework to
incorporate metadata in the analysis. The algorithms will also enable the
reconstruction of sparse models, thus handling the scenario when the number of
microbial associations in the community is small. The application of these
algorithms to analyze large microbiome datasets will generate new insights into
microbial ecology of three different environments (human, ocean, and soil). This
analysis will include an elucidation of microbial associations at the strain
level, the structures of the underlying microbial networks, and the identities
of the key taxa in these environments. The results of the project can be found
at https://github.com/syooseph/YoosephLab/tree/master/MixtureMicrobialNetworks.&
lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and has been
deemed worthy of support through evaluation using the Foundation's intellectual
merit and broader impacts review criteria.