* 2052653
* Scalable Population Genetic and Phylogenetic Inference Using Large Samples of Microbial Data
* MPS,DMS
* 04/01/2021,03/31/2024
* Jonathan Terhorst, Regents of the University of Michigan - Ann Arbor
* Standard Grant
* Amina Eladdadi
* 03/31/2024
* USD 270,000.00

This project will enable us to more effectively use large amounts of DNA and
other genomic data to study human history, natural selection, pathogen
evolution, and other topics that are important to science and human well-being.
Examples of the types of questions it addresses include: When did humans migrate
out of Africa? How did polar bears fare during the last global warming event?
Does being tall confer evolutionary advantages? What is the current status of
the COVID-19 pandemic, and when will it end? Although definitively answering
these questions is challenging, evolution furnishes clues about them in the form
of genetic variation. These clues can be decoded using mathematical models to
analyze DNA samples from current populations. The amount of available genetic
data has increased dramatically in recent years, and consequently faster and
more accurate analytical methods are needed to fully utilize these rich new
sources of information. This project will develop those methods. In addition, it
will facilitate the creation of new curriculum materials designed to educate
students about quantitative genetics and computational
biology.&lt;br/&gt;&lt;br/&gt;This project will develop new and scalable methods
for phylogenetic and population genetic inference, with a particular focus on
analyzing pathogen genetic data. Although these areas are already quite mature,
historically many methods designed to analyze genetic data made modeling
assumptions based on human biology and data availability. Such assumptions
complicate efforts to study the genetics of species whose biology is very
different from humans, even though these species can have important impacts on
human health. This project addresses these shortcomings through: the creation of
novel methods for phylogenetic inference which can adapt to the amount of
phylogenetic signal present in the data; faster likelihood-based phylogenetic
network inference methods which allow for horizontal gene transfer or other
reticulate events; variational methods for rapidly inferring epidemiological
parameters from pandemic data; and new applications of coalescent hidden Markov
models which are faster and have less bias than existing
methods.&lt;br/&gt;&lt;br/&gt;This award reflects NSF's statutory mission and
has been deemed worthy of support through evaluation using the Foundation's
intellectual merit and broader impacts review criteria.