* 1237993
* Expanding the Medicago truncatula Hapmap as a Platform for Exploring the Genetics of Legume Symbioses
* BIO,IOS
* 02/15/2013,01/31/2018
* Nevin Young, University of Minnesota-Twin Cities
* Continuing Grant
* timothy nelson
* 01/31/2018
* USD 4,996,854.00

PI: Nevin Young (University of Minnesota)&lt;br/&gt;&lt;br/&gt;Co-PIs: Michael
Sadowsky, Robert Stupar, and Peter Tiffin (University of Minnesota), Maria
Harrison (Boyce Thompson Institute for Plant Research), Betsy Martinez-Vaz
(Hamline University), Jason Rafe Miller (J. Craig Venter Institute), Joann Mudge
(National Center for Genome Resources) &lt;br/&gt;&lt;br/&gt;This project uses
functional assays, de novo genome assembly and bioinformatic data-mining to
characterize symbiosis genes in the model legume Medicago truncatula. Legumes
are noteworthy for the sophisticated symbioses they form with rhizobial bacteria
and arbuscular mycorrhizal (AM) fungi. However, existing knowledge about
symbioses comes primarily from knockout mutants, an approach that often misses
genes of subtle yet significant effect, especially genes likely to be important
in contemporary evolution. In earlier work, several strongly supported candidate
symbiosis loci were discovered through genome-wide association analysis (GWAS)
and support for candidate genes often included independent evidence like
expression profile, correlation with multiple traits or co-localization with
known symbiotic phenotypes. The current project will test ~100 of these
candidate loci through reverse genetic experiments involving Tnt1 insertion and
RNAi "knockdown" plant lines. Promising genes will be examined through
interaction assays involving previously defined Sinorhizobium and AM strains and
tested for "gene-for-gene" relationships using a panel of 48 sequenced
Sinorhizobium strains. Earlier GWAS mapping only targeted SNP variation, even
though structural variants (SVs) and copy number variants (CNVs) are known to
have major impacts on genome variation. This is especially relevant to the
genomics of symbiosis because the large gene families such as the NB-ARC domain-
containing genes and nodule cysteine rich peptides (NCRs) play critical roles in
symbiosis. This project will deeply sequence and de novo assemble 30 nodal M.
truncatula accessions, in order to discover SVs and CNVs. SVs and CNVs will be
imputed genome-wide, leading to a new round of GWAS to discover symbiotic loci
missed in the earlier phase of mapping. The primary outcomes of this project
will be the identification of genes associated with the contemporary evolution
of symbiosis as well as the architecture of M. truncatula genomic diversity.
&lt;br/&gt;&lt;br/&gt;This research effort will be extended by involving
undergraduates from Hamline University, a four year institution located near the
University of Minnesota, and from the University of Puerto Rico (UPR). Students
at Hamline will work throughout the academic year and then together with UPR
students during the summer as part of Minnesota's Life Sciences Summer
Undergraduate Research Program (LSSURP) program. Their work will target the
important but largely uncharacterized Sinorhizobial enzyme, ACC deaminase, and
students will also participate in the screening of reverse genetic mutants.
These experiences will provide the students the opportunity to develop their own
hypothesis-driven projects. Through joint mentoring by project PIs, student
training will also assist the undergraduate research program at Hamline to
become more competitive for its own future external research initiatives.
Genomic sequence resources will be available for public through
medicagohapmap.org, and the Short Read Archive, dbSNP and FTP sites at Genbank.
Importantly, the underlying Medicago Hapmap GWAS platform, available at
medicagohapmap.org, will provide a long-term resource for the broader research
community to discover legume genes controlling quantitative variation of
agricultural and biological interest, especially trait variation in alfalfa
(Medicago sativa), the fourth most widely cultivated crop in the U.S.