* 1238812
* Improving Dynamic Metabolic Flux Analysis for the Discovery of Molecular Determinants of Plant Phenotypes
* BIO,IOS
* 10/01/2012,09/30/2019
* Adrian Hegeman, University of Minnesota-Twin Cities
* Continuing Grant
* Gerald Schoenknecht
* 09/30/2019
* USD 3,440,109.00

PI: Adrian D. Hegeman (University of Minnesota - Twin
Cities)&lt;br/&gt;&lt;br/&gt;CoPI: Jerry D. Cohen (University of Minnesota -
Twin Cities)&lt;br/&gt;&lt;br/&gt;Key Collaborator: Aaron Rendahl (University of
Minnesota - Twin Cities)&lt;br/&gt;&lt;br/&gt;With the recent rapid expansion in
accessible genome and mRNA nucleotide sequence data there is an increasing need
for robust methods for connecting observable plant phenotypes with changes at
the molecular level. Measurement of metabolic flux (the flow of matter through
an organism's network of metabolic pathways) is a direct molecular indicator of
an organism's phenotype. Metabolic flux information describes how nutrient
resources and stores are used by which segments of a metabolic network to
generate and accumulate products required for growth, development, and
differentiation, integrated with responses to environmental conditions.
Correspondingly, metabolic flux analysis (MFA) provides a direct route for both
the discovery (and subsequent manipulation) of the molecular basis of any number
of complex plant behaviors. Unfortunately, most MFA is currently performed using
cultured cells, both to provide homogeneity and as an easy method for
introduction of isotope labeled nutrients. Several challenges need to be
addressed before dynamic-MFA can be applied practically to intact plants for
molecular phenotyping. This project will provide for 1) optimize methods for
measurement of dynamic metabolic fluxes in intact plant systems using timed
stable isotope labeled nutrient incorporation, mass spectral analysis, and
automated data extraction and calculation of dynamic fluxes; 2) develop a
procedure for finding additional connectivity and pathway components of
metabolic network models using dynamic flux information using a publicly
available metabolic network model for Arabidopsis with flux data collected from
Arabidopsis plants subjected to multiple stress conditions and 3) develop
microsampling approaches for dynamic flux analysis by examining crowding stress
in maize. Computational and bioinformatics resources for metabolic flux analysis
workflows will be created and distributed.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;In
this project, postdoctoral scientists and graduate students will be exposed to a
unique multidisciplinary approach to metabolomics. In addition, the project will
continue to offer a summer workshop for graduate students, postdoctoral
scientists and others interested in stable isotope approaches for understanding
metabolic flux measurement and how it might be used for biological research.
Short-term dissemination of data, processing algorithms and methods will be
accomplished by means of the project website
(http://plantmetabolomics.cfans.umn.edu/).