* 1242275
* EAGER: Improved Computational Tools for Plant Gene Assembly and Synteny Detection
* BIO,IOS
* 06/01/2012,05/31/2014
* David Nelson, The University of Tennessee, Memphis - The Health Science Center
* Standard Grant
* Diane Jofuku Okamuro
* 05/31/2014
* USD 211,212.00

The ever-increasing rate of genome sequence production will result in literally
thousands of genome sequences being determined. Current automated methods for
plant gene assembly have certain inherent weaknesses that result in many
incorrectly assembled genes due to inaccurate detection of true intron-exon
boundaries including GC boundaries, inability to detect N-terminal exons, and
inappropriate fusion of adjacent genes. Expert manual annotation can usually
overcome these limitations, but manual annotation is not scalable to many
thousands of genes. Genome projects generally are able to claim only 1000-3000
genes were manually checked. In plant genomes this may be only 3-10% of the
genes in the genome. This EAGER project seeks to improve automated plant gene
assembly and annotation with a focus on cytochrome P450 genes, which comprise
one of the largest gene families in plants. Successful implementation of
accurate gene prediction algorithms and the development of a novel tool for
identifying ohnologs (genes created by whole genome duplication events) in plant
genomes will have a transformative effect on genome
annotation.&lt;br/&gt;&lt;br/&gt;This project will provide international cross-
disciplinary training opportunities for one postdoctoral research associate. All
programs will be available as a set of executable files from an ftp site and
through the project website, both of which will be accessible via
http://www.uthsc.edu/molecular_sciences/directories/faculty/d_nelson.php. The
source code will also be made available at this location.