* 1243724
* The Drosophila Fourth Chromosome: Gene Expression in the Context of Repetitious DNA
* BIO,MCB
* 03/15/2013,02/29/2016
* Sarah Elgin, Washington University
* Continuing Grant
* Manju Hingorani
* 02/29/2016
* USD 244,800.00

Intellctual Mert: The DNA of the eukaryotic genome is packaged by association
with the histones and other chromosomal proteins into alternative forms: the
more condensed form, heterochromatin, inhibits gene expression while a more open
form, euchromatin, facilitates gene expression. Eukaryotic genomes have been
greatly expanded over evolutionary time by the retention of repetitious
sequences, primarily copies of transposable elements (DNA transposons and
retroviruses - TEs). Where these sequences are present at high density, they are
packaged as heterochromatin, inhibiting expression of these elements. The long-
term goal of this project is to understand how genes can be expressed from a
domain with high levels of silenced repetitious sequence. Work in the fruit fly
Drosophila melanogaster has shown that many genes normally resident in
euchromatin exhibit a variegating phenotype (due to silencing of the gene in
some of the cells where it should be active) when inserted into heterochromatin
by rearrangement or transposition. The small fourth chromosome (Muller F
element), which is largely heterochromatic, contains ~80 genes, both
housekeeping and developmentally regulated genes. In higher eukaryotes,
specifically mammals, the long chromosome arms that contain genes appear to be
structured like the F element, with ~30% TE repeats. Thus how genes can function
in this environment, while repetitious elements are silenced, is a general
issue. The goal here is to understand the combination of characteristics that
enables fourth chromosome genes to function in a chromatin context where
euchromatic genes variegate (are silenced). The investigators postulate the
presence of one or more key sequence features or motifs organizing and/or
protecting the fourth chromosome genes. To test this hypothesis, they will
identify 'landing pad' sites in the D. melanogaster fourth chromosome where a
fourth chromosome gene is appropriately expressed, but an hsp70-white gene gives
a variegating phenotype, and test whether this is determined by the 5' region of
the gene (the transcription start site with ~1 kb flanking sequence, both
upstream and downstream). Establishing this system will allow one to identify
the minimal features that can drive full expression of any reporter in a fourth
chromosome heterochromatic site. As a complement to this work, the investigators
plan to use an in silico approach to find sequences in other species that are
similar to the sequences identified in the D. melanogaster F element. This
comparative bioinformatics approach will be facilitated by carrying out ChIP-seq
experiments to find binding sites for RNA polymerase II (marking the start site
for transcription) in two Drosophila species which are at a suitable
evolutionary distance for finding sequence motifs, D biarmipes and D. elegans.
By testing the findings gained from one approach with the methodology of the
other approach, the investigators expect to gain a better understanding of the
global regulation of different chromosomal regions and of the sequence elements
needed to drive expression of genes in heterochromatin.

Broader Impacts: This project is a collaborative effort with the Genomics
Education Partnership (GEP), a group of over 80 faculty who are using Drosophila
genomics to bring students into the research community. The GEP undergraduate
students will be responsible for generating high quality annotations for the
genes on the Muller F elements of D. biarmipes and D. elegans. The student
annotations will facilitate the search for common features/motifs on the F
element using a comparative genomics approach. Undergraduates will also take
charge of looking at the impact of Su(var) and E(var) mutations on various
reporters, testing sensitivity of the "synthetic fourth chromosome gene"
constructs to mutations in appropriate chromosomal proteins. The postdoctoral
fellow supported here will become an integral member of GEP, and will become
very skilled at working with undergraduates in a research program.