* 1244146
* Testing Models of Centromere Drive
* BIO,MCB
* 04/01/2013,03/31/2017
* Rachel O'Neill, University of Connecticut
* Continuing Grant
* Karen Cone
* 03/31/2017
* USD 929,999.00

Intellectual Merit: Centromeres are the site on every chromosome of mammals
where kinetochore assembly and spindle attachment occur during cell division.
Faithful segregation of every chromosome relies on the proper functioning of
centromeres, and errors in the cascade of events prior to and after spindle
attachment lead to chromosome loss - a disastrous genetic fate. This research
aims to understand the functional modules found within mammalian centromeres. A
genetic process called molecular drive is thought to account for the observation
that the DNA found at centromeres across all individuals within one species is
highly similar, whereas a counter process called genetic conflict may be
responsible for the observation that vastly different centromere satellite
sequence suites are found between species. Accordingly, as satellite DNA arrays
expand on a chromosome, they can attract more microtubules during female meiosis
and lead to unequal transmission of one parental chromosome over the other. In a
recently proposed model, this process, called Centromere Drive, results in the
rapid evolution of centromere binding proteins selected to equalize the
transmission of each chromosome during meiosis, ensuring all chromosomes are
inherited equally in a population. Current models of centromere evolution and
predict that different, but closely related species within a given species group
would experience shifts and expansions of satellite sequences that would result
in species-specific satellite sequences at centromeres. The well-documented
evolutionary history of species of kangaroos and wallabies, along with the known
history of the evolution of their chromosome complements, provides an ideal
opportunity to test the Centromere Drive hypothesis directly, as opposed to the
inferential studies that currently support this theory. In doing so, the
functional components of centromeres that facilitate equal chromosome
segregation will be uncovered. This project will determine whether satellite DNA
interacting proteins evolve in concert with satellite sequence suites, as
predicted by centromere drive, or with species divergence, as predicted by
centromere drift and molecular drive. Inherent to this research will be efforts
to determine whether the proposed conflict driven evolution of these components
are responsible for hybrid incompatibilities and which components in the cell
are the subject of drive. &lt;br/&gt;&lt;br/&gt;Broader impacts: This research
involves a broad range of participants, including visiting faculty, post-
graduate, undergraduate and high school students from a variety of socioeconomic
backgrounds in the New York, Connecticut, Massachusetts and Rhode Island area.
Undergraduate independent study students and high school students enrolled in
the UConn Mentor Connection will be active participants throughout this
research. Moreover, this research will serve as the foundation for development
of three modular-format courses to provide training in utilization of massively
parallel sequencing technology, which has revolutionized genome biology, but
remains largely inaccessible to the individual scientist. Students will learn to
prepare a library for sequencing, perform sequencing on the SOLiD sequencing
platform and use bioinformatics to analyze the resulting data. The course will
be open to advanced undergraduates, graduate students, postdocs and visiting
high school teachers as part of the Professional Science Masters program in the
Center for Applied Genetics and Technology. The inclusion of high school
teachers as participants is part of the broader goals of this research in
empowering educators to train both future generations of scientists as well as
the future nonscientist members of the general public that will be directly
impacted by shifts in genomic technologies.