* 1248106
* Computing Regulatory DNA by Comparing Genomes
* BIO,IOS
* 11/01/2012,10/31/2015
* Michael Freeling, University of California-Berkeley
* Standard Grant
* Diane Jofuku Okamuro
* 10/31/2015
* USD 882,032.00

PI: Michael Freeling (University of California - Berkeley)

This two year project leverages previous work and uses computational tools,
along with published sequence and data on levels of mRNA, to hypothesize
specific function for noncoding sequences in higher plants. The project takes
advantage of the recent release of two important paleopolyploid genomes, maize
and Brassica rapa (Chinese cabbage). Having diverged approximately 12 million
years ago, each of these ancient polyploids has close relative genomes that can
serve as outgroups - sorghum and the other grasses for maize and Arabidopsis,
and several recently sequenced crucifers for the cabbages.

There are four objectives with the goal of developing a better understanding of
the function of enhancers and cis-acting modules, those sequences that regulate
gene expression in plants: 1) to develop a new application for CoGe that brings
motifs and transcription factor binding sites into view when comparing the
sequences of chromosomal segments. CoGe is a popular, public toolbox for
visualizing and comparing sequences; 2) to discover genes with new regulatory
capabilities by finding rare cases of new sequence corresponding to new patterns
of expression. Finding genes that switch their patterns of genome dominance
should relate to this aim; 3) to find the means by which plant regulatory DNA
evolves into non-detectability without losing its function, a process known as
"binding site turnover" in mammals; and, 4) to elucidate enhancer evaluation in
plants by using the natural process of fractionation mutagenesis to generate
specific sequences with specific functions using computational data only.
Hypotheses based on the predicted enhancers will be tested using transient and
stable expression of engineered reporter genes.

Broader aims of the project include a summer internship program for underserved,
local high school students, and continued development of the popular CoGe
website (http://genomevolution.org/CoGe/) containing applications that help find
meaning when genes and genomes are compared. This project will host one or two
students each summer. Additionally, this project involves improvements to the
CoGe suite of free, on-the fly applications now powered by iPlant
(http://www.iplantcollaborative.org/) that will provide broad access to the
broader scientific community.