* 1402101
* Alternative pathways in directed evolution
* ENG,CBET
* 06/01/2014,05/31/2018
* Marc Ostermeier, Johns Hopkins University
* Standard Grant
* Steven Peretti
* 05/31/2018
* USD 350,000.00

Proposal Number: 1402101&lt;br/&gt;Institution: Johns Hopkins
University&lt;br/&gt;P.I.: Ostermeier, Marc A. &lt;br/&gt;Title: Alternative
pathways in directed evolution&lt;br/&gt;&lt;br/&gt;The study of evolution is
important, not only because it is central to biology, but also because
researchers apply evolution in the laboratory to develop beneficial proteins and
cells for use in in many areas including human health, detection of chemical and
biological agents, and remediation of toxic compounds in the environment. This
proposal seeks to evaluate a fundamentally new approach for applying evolution
in the laboratory. Paradoxically, the goal is to first develop proteins that are
worse at performing the desired function and then re-engineering these to be
improved in function. The approach is based on a growing appreciation in
evolutionary biology of the complexity of proteins and that sometimes a set of
changes to a protein that have a negative effect on protein function in
isolation, can have a positive effect when combined. Success of this project
would impact the fields of evolutionary biology and biotechnology. Potentially,
this new approach could lead to proteins and cells with superior properties that
could not be readily created using existing methods. The project will also train
a pre-doctoral and undergraduate student in research.&lt;br/&gt;&lt;br/&gt;The
objectives are (1) to evaluate fundamentally different directed evolution
selection strategies for evolving proteins away from local fitness peaks towards
other distant fitness peaks and (2) to address shortcomings in our understanding
of evolutionary processes in fluctuating environments (i.e. evolution under
variable selection). The TEM-15 beta-lactamase allele will be subjected to
directed evolution under a variety of selective pressure schemes designed to
traverse the fitness landscape in different ways. These schemes include (1)
positive selection, (2) neutral drift, (3) retrogressive selection, (4)
oscillating selection, and (5) fluctuating antibiotic environments. This
evolution will exploit our band-pass gene circuit that allows the selection of
beta-lactamase alleles at any desired fitness level (e.g. away from genes with
lower or higher fitness). The progress and outcomes of evolution will be
monitored by deep sequencing, which will determine the identity and distribution
of mutations for each selection mechanism. The interpretation of mutational
trajectories and outcomes will be aided by the comprehensive determination of
the effect of individual mutations in TEM-15.&lt;br/&gt;&lt;br/&gt;Due to the
interdisciplinary nature of the project, this award by the Biotechnology,
Biochemical, and Biomass Engineering Program of the CBET Division is co-funded
by the Systems and Synthetic Biology Program of the Division of Molecular and
Cellular Biology.