* 1407028
* A reliable and scalable approach to causal inference for large-scale multivariate data
* MPS,DMS
* 08/15/2014,07/31/2017
* Garvesh Raskutti, University of Wisconsin-Madison
* Continuing Grant
* Gabor Szekely
* 07/31/2017
* USD 119,999.00

With masses of large-scale data being generated, a key challenge facing many
scientists is to infer relationships amongst variables of interest. In
particular, inferring causal or functional relationships amongst genes,
proteins, and other biological elements is of fundamental interest to
scientists. This project will develop methods for inferring causal or functional
relations between genetic, proteomic, and transcriptomic features both for the
ENCODE human genome project and data for mice with different susceptibility to
obesity and diabetes. For both types of data, this project will develop
frameworks that comprise: (1) domain knowledge that informs the choice of model
and algorithm; (2) fast, parallelizeable algorithms with provable run-time
guarantees; and (3) statistical consistency guarantees for the algorithms
developed under assumptions that are likely to be satisfied in
practice.&lt;br/&gt;&lt;br/&gt;Directed graphical models or Bayesian networks
provide a useful framework for representing causal or functional relationships.
A number of algorithms have been developed for inferring directed or Bayesian
networks from data. However prior approaches are either unreliable as they
require assumptions that are rarely satisfied in practice, or do not scale to
larger datasets. The proposed project will address this issue by developing
algorithms for inferring directed networks with both statistical consistency
guarantees and run-time guarantees. The new algorithms will involve exploiting
connections between techniques in numerical linear algebra for developing fast
solvers of linear systems and concepts in graph theory. Algorithms will be coded
in R and will exploit parallel processing. Evaluation will involve both small-
scale and large-scale synthetic graphical models with known network structure,
real datasets involving yeast data where some of the directions are known, and
new biochemistry data in which most of the directions are unknown. Theoretical
guarantees on run-time and statistical consistency will be provided using a
combination of tools from graph theory, numerical linear algebra, and
concentration of measure the PI has used and developed in prior work.