* 1408243
* Systematic Mapping of the Sequence Space Critical to Bacterial Signal Transduction
* BIO,MCB
* 07/15/2014,03/31/2017
* Michael Laub, Massachusetts Institute of Technology
* Standard Grant
* Devaki Bhaya
* 03/31/2017
* USD 300,000.00

Nontechnical description: Cells have a remarkable ability to sense and respond
to changes in their environment. This sensory capability often relies on
membrane-embedded protein kinases that can relay information about the
environment to intracellular effectors. In bacteria, such information transfer
frequently involves two-component signal transduction pathways that transfer the
signals from the outside to the interior of the cell. The signal transfer
activates a specific transcription factor, which, in turn, will activate a set
of genes that are necessary to respond to the specific environmental changes. In
his previous work, the investigator studied the signal transfer system on the
protein level and discovered four amino acid residues that are necessary for the
two-component signal transduction partners to interact and subsequently activate
the appropriate set of target genes. This is remarkable because bacterial cells
have many such highly homologous parallel signal transduction systems that are
dedicated to activate a highly diverse set of gene systems. This raises the
question of how these pathways maintain their specificity with apparent minimal
cross talk between them. The focus of this study is to assess how these four
specificity residues ensure that the signal-receiving component of the system
finds the cognate response regulator.

Technical description: Protein-protein interactions are critical to the
operation and functions of all cells. The specificity of these interactions is
often dictated at the level of molecular recognition, meaning proteins have an
intrinsic ability to discriminate cognate from non-cognate partners.
Understanding precisely how this discrimination is accomplished remains a major
problem in biology. In particular, delineating the set of amino acids that
contribute to a protein-protein interaction, and elucidating their individual
and collective contributions are significant experimental challenges. Directed
mutagenesis studies can help elucidate the function of individual residues, but
such efforts have traditionally been limited by the number of mutants
constructed and tested, thus precluding truly systematic studies of protein-
protein interactions. To tackle this challenge, the investigator will leverage
the power of next-generation sequencing to comprehensively map the sequence
space underlying a protein-protein interaction critical to signal transduction
in bacteria. Using the E. coli pathway PhoQ-PhoP as a model system, libraries of
possible amino acid combinations at four key specificity residues (separately in
either the kinase or regulator) will be developed. A high-throughput flow
cytometry-based screen and next-generation sequencing will identify all
combinations that support a functional PhoQ-PhoP interaction. The resulting
data, and subsequent bioinformatic analyses, will reveal the degeneracy of this
protein-protein interaction. Individual mutants identified as signal-responsive
will be characterized in detail using a battery of in vitro and in vivo assays
to assess how they impact the quantitative output and dynamics of the PhoQ-PhoP
signaling pathway. This project will include also a detailed quantitative,
differential equation-based model of two-component signaling pathways, which
will be used to dissect the data resulting form the high-throughput screen to
learn at each step of the pathway how autophosphorylation, phosphotransfer, and
dephosphorylation, contribute to the complex, nonlinear signaling dynamics
observed in vivo. The model will drive testable predictions of the behavior of
other two-component pathways, thereby broadening the scope and generalizability
of our findings to a wide range of organisms in bacteria, fungi, and plants, all
of which rely on two-component signaling pathways for survival in the wild.