* 1409321
* In-Sequence Coding of Stochastic Gene Expression Via Synonymous Mutations
* MPS,PHY
* 07/15/2014,06/30/2018
* Philippe Cluzel, Harvard University
* Continuing Grant
* Krastan Blagoev
* 06/30/2018
* USD 703,437.00

Because 18 of the 20 amino acids are encoded at the DNA level with multiple
synonymous codons, the genetic code used during protein synthesis is highly
redundant -- a codon is the sequence of three DNA base pairs coding an amino
acid. While it is well known that proteins can be encoded by different
synonymous codons, the role of this redundant code on the efficiency of protein
synthesis has remained unclear. Remarkably, the investigator found that under
nutrient starvation, synonymous codons played very distinct roles on both
synthesis levels and the variations from one cell to another of the quantity of
synthetized proteins. Some codons were found to be robust to the limitation of
certain amino acids, allowing for stronger protein expression, while others were
sensitive to this limitation yielding low levels of protein expression.
Importantly, cells are believed to starve during the formation of bacterial
biofilms or during cancerous proliferation for which synonymous codons should
play an important role in regulating protein synthesis. Therefore, the goal of
this project is to quantify and establish the role of synonymous codons in
protein synthesis during starvation of nutrients. The training of students with
scientific backgrounds in mathematics or biology is an important component of
this effort. Students will have the opportunity for hands-on participation in
dynamics simulation of gene expression extracted from data made possible by this
project. Over the last decade, there has been considerable interest from
physicists and biologists to develop a quantitative framework to predict the
stochastic behavior of gene expression at the single cell level. In bacteria, a
clear picture has emerged: noise in gene expression is mainly caused by
transcriptional bursts that are Poisson distributed and translation solely
amplifies these fluctuations. While this canonical model holds when bacteria
grow in rich media, it fails to explain the observed noise when bacteria are
exposed to environmental perturbations such as nutrient limitation. The outcome
of this project will be to create a novel physical framework to predict the
noise in gene expression governed by the in-sequence distribution of sensitive
synonymous codons under environmental perturbations. Here the investigator
proposes to characterize the noise associated with robust and sensitive codons
when used to encode the expression of the yellow fluorescent protein. Firstly,
the investigator will focus on quantifying the stochastic cell-to-cell behavior
in gene expression arising from synonymous codon choice. Secondly, work will be
done to identify how the dynamics associated with the translation of synonymous
sensitive codons exhibits near-critical behavior. Finally, the third aim
examines the contribution of the variations of key intracellular parameters as
control parameters of noise driven by synonymous codons under well-characterized
environmental perturbations.

This award is supported jointly by the Physics of Living Systems Program in the
Physics Division and the Cellular Dynamics and Function Program in the Division
of Molecular and Cellular Biosciences.