* 1413331
* Mechanisms of epigenetic gene regulation by the Drosophila COMPASS-like complex
* BIO,MCB
* 07/01/2014,06/30/2018
* Andrew Dingwall, Loyola University Stritch School of Medicine
* Continuing Grant
* Karen Cone
* 06/30/2018
* USD 708,613.00

The genetic code of every living organism is contained within chromosomes,
referred to collectively as the genome. In all eukaryotes, the genome is
compacted into the nucleus through the formation of condensed molecules known as
chromatin. The smallest individual units of compaction are known as nucleosomes
that are comprised of 147 base pairs of DNA wrapped around an octamer of small
basic histone proteins. In order for regulatory factors and enzymes to access
the genetic code and properly regulate processes essential for normal
development and cell survival, the nucleosomes need to be moved aside or
directly modified on specific amino acid residues in a process termed chromatin
remodeling. The enzymes and other proteins that carry out chromatin remodeling
are ancient and remarkably conserved, with increasing complexity from single
celled organisms up through vertebrates. The overall impact of this project will
be a better understanding of how chromatin remodeling controls expression of the
information that is encoded within the genome. In addition, high school,
undergraduate and graduate students will participate in mentored research. Women
and minorities will be highly represented. Multidisciplinary training and
education are vital to prepare for diverse science careers and this project
employs the unique tools currently available for Drosophila studies. Students
will be trained in molecular and developmental genetics, biochemistry, molecular
biology and bioinformatics, structural and developmental biology. The project
will enhance interactions among faculty scientists within the institution and
with students at multiple undergraduate institutions to explore the rapidly
emerging field of bioinformatics. Undergraduate students will be tasked with
learning new informatics tools and approaches using experimental data derived
from this project. &lt;br/&gt;&lt;br/&gt;In order to define the histone
recognition and binding properties of Cmi and related mammalian domains and
elucidate the potential mechanism of COMPASS-like complex targeting to gene
enhancers, collaborative structural studies that include X-ray crystallography
and nuclear magnetic resonance of the histone recognition domains and targeted
mutagenesis combined with in vitro measurements of binding affinities and in
vivo chromatin association will be employed. The project also uses next-
generation high throughput chromatin and RNA analyses (ChIP-Seq, RNA-Seq) and
target gene studies. ChIP-Seq using chromatin from cmi mutant animals will
enable correlation of epigenetic marks with Cmi function and align the RNA-Seq
gene expression data with chromatin binding to address key developmental
functions. Tissue-specific targeted removal of both cmi and trr and
overexpression of cmi using unique genetic tools developed for these studies
will allow for direct testing of COMPASS-like functions on target genes. Leading
edge chromatin technologies that examine physical connections between distant
gene regions will elucidate novel regulatory roles for the Drosophila COMPASS-
like complex in facilitating enhancer-promoter communication necessary for
proper gene control. These studies will help to reveal essential and
foundational properties of chromatin remodeling and modifying complexes and
provide critical insight into the mechanisms of developmental gene regulation.