* 1415670
* SBIR Phase I:  Rapid Pathogen Diagnostics and Biosurveillance using Multiplexed High-throughput Sequencing
* TIP,TI
* 07/01/2014,06/30/2015
* Eduardo Castro-Nallar, Aperiomics, Inc.
* Standard Grant
* Jesus Soriano Molla
* 06/30/2015
* USD 179,960.00

The broader impact/commercial potential of this Small Business Innovation
Research (SBIR) Phase I project is to improve our ability to fight infectious
diseases that negatively impact agricultural yields and reduce the efficiency of
global food production and distribution systems. This innovation will enhance
scientific and technological understanding by leveraging the power of high-
throughput sequencing and bioinformatics to provide a pathogen identification
and surveillance tool with demonstrated efficacy against known and unknown
infectious agents. This platform is fast, sensitive, and cost-effective, and can
be used for any animal sample to detect virtually all possible microbes ? even
microbes that have never before been characterized. Hundreds of samples can be
rapidly screened without relying upon known genetic/genomic data of microbes.
The global molecular diagnostics market is expected to grow at a compound annual
growth rate (CAGR) of over 14% from 2012 to 2017, with infectious disease
testing being the leading application at 26% share, therefore the commercial
opportunity of this project is vast.

The proposed project tackles problems arising from low-throughput targeted
detection technologies such as PCR, hybridization arrays, and culture. Current
diagnostic methods rely on what is already known about target microbe genetics,
and provide limited information in the form of presence/absence of a known
target sequence. The proposed research objectives are related to lowering the
technical risks associated with a high-throughput unbiased pathogen detection
platform based on DNA sequencing and Bayesian statistics. The ultimate goal is
to develop, standardize and validate our metagenomics pathogen identification
platform for use in agricultural detection and biosurveillance contexts, using
aquaculture related fish species and their infectious agents as a relevant
application. This project proposes to: 1) characterize relative performance of
sequencing platforms for pathogen identification; 2) validate and benchmark our
agricultural detection platform using fish samples and spike-ins diagnosed using
established methods; and 3) evaluate the use of host gene expression signatures
as supporting evidence for infection. The team will obtain metagenomic sequence
data from infected fish, compare their analysis results against current methods,
and establish the limit of detection using known quantities of pathogenic
material.