* 1419282
* CAREER:  Developing Biochemoinformatics Tools for Large-Scale Metabolomics Applications
* BIO,DBI
* 11/01/2013,12/31/2018
* Hunter Moseley, University of Kentucky Research Foundation
* Continuing Grant
* Jennifer Weller
* 12/31/2018
* USD 1,053,735.00

Research: Recent advances in stable isotope-resolved metabolomics (SIRM) are
enabling orders-of-magnitude increase in the number of observable metabolic
traits (a metabolic phenotype) for a given organism or community of organisms.
Analytical experiments that take only a few minutes to perform can detect stable
isotope-labeled variants of thousands of metabolites. Thus, unique metabolic
phenotypes may be observable for almost all significant biological states,
biological processes, and perturbations. Currently, the major bottleneck is the
lack of data analysis that can properly organize and interpret this mountain of
phenotypic data as insightful biochemical and biological information. The
research goals are to develop systems-level biochemical tools as part of an
integrated data analysis pipeline that will alleviate this limitation, enabling
a broad application of SIRM from the discovery of specific metabolic phenotypes
representing biological states of interest to a mechanism-based understanding of
a wide range of biological processes with particular metabolic phenotypes. The
major specific intellectual merits are developing:&lt;br/&gt;- Novel methods for
detection and identification of metabolites that utilize the combined advantages
from stable isotope labeling, chemoselective probes, ultra-high
resolution/accurate MS, and NMR. Since unidentified metabolites make up the
majority of detected features in current metabolomics datasets, identification
of metabolites is a key focus. &lt;br/&gt;- Key error analyses that allow: i)
rigorous quantitative evaluation of detected isotopologue intensities and their
errors; ii) evaluation of error propagation through subsequent analyses; and
iii) development of quality control measures derived from the detected errors.
&lt;br/&gt;- New algorithms for isotopic non-steady state conditions of SIRM
experiments, especially deconvolution methods that will aid relative flux
interpretation and metabolic flux analysis. &lt;br/&gt;- New methods that
integrate and cross-validate metabolomics with genomics, transcriptomics, and
proteomics via mutually-identified metabolic, gene expression, and signaling
pathways.&lt;br/&gt;Education: Simultaneous trends of declining student effort
and declining graduation rates in STEM disciplines do not bode well for the
successful education of the next generation of scientists. A more expedient
approach to improving student outcomes may be to increase the effectiveness of
students? effort. Using a design-based research approach, this project
integrates multiple advanced teaching-learning methods into content-rich college
science courses. Statistical analysis of these methods shows large effect sizes
for the use of scaffolded explicit revision to improve the effectiveness of
student effort and indicates a path for significant refinement of these methods,
which will be pursued and implemented. &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;The
proposed research will create computational tools that analyze and derive unique
mechanistic information from large datasets available from cutting-edge
metabolomics technologies which track atomic level changes in the production and
utilization of thousands of molecules (metabolites) inside the cells of
organisms. These novel computational tools will be tested and refined in the
Center for Regulatory and Environmental Analytical Metabolomics (CREAM), which
provides state-of-the-art analytical services and expertise for national and
international stable isotope-resolved metabolomics (SIRM) research efforts. Once
these computational tools reach production-quality, they will be disseminated to
the broader scientific community for a wide variety of scientific applications
involving biological processes with changes in cellular metabolism. Also, these
methods will integrate metabolomics datasets with genomics and other omics-level
datasets, allowing new systems-level metabolic insights into a wide range of
biological processes. During the execution of this proposed research,
significant numbers of high school, undergraduate, and graduate students from a
wide variety of STEM (Science, Technology, Engineering, &amp; Math) disciplines
will be exposed to and trained with multidisciplinary bioinformatics research
projects using interdisciplinary approaches to research. In addition, the
principal investigator has developed an integrated set of advanced teaching-
learning methods amenable to content-rich college science courses that have
statistically significant impacts on student effort and outcomes. These advanced
teaching-learning methods focus students' efforts at correcting and learning
from prior mistakes on assignments, quizzes, and exam questions via a series of
explicit revision steps that span different levels of learning.