* 1422591
* III: Small: Algorithmic Techniques for Determining Alterations in the Patterns of Chromosome Spatial Organization inside the Cell Nucleus
* CSE,IIS
* 08/01/2014,07/31/2018
* Ronald Berezney, SUNY at Buffalo
* Continuing Grant
* Sylvia Spengler
* 07/31/2018
* USD 499,968.00

Recent research in cell biology has led to the emerging view that the 3-D
arrangement of chromosome territories (CT) and the spatial positioning of genes
within these territories are linked to genomic function and regulation. Despite
this progress, it is not clear what is the spatial organization patterns of CTs
inside the cell nucleus and how such patterns dynamically change during the cell
cycle and at different stages of cell differentiation and cancer progression. To
facilitate more in-depth studies of these important biological problems, in this
project, the Pi will develop a set of efficient algorithmic techniques for
determining the patterns and their alterations of three chromosome organization
problems. First, how are chromosomes spatially positioned inside the nucleus?
Second, how are chromosomes neighboring or associating with each other? Third,
what is the internal structure of each individual chromosome? The core of this
project is to develop efficient algorithms for solving a set of challenging
computational problems which are essential for the chromosome organization
problems, thus addressing the emerging science at the interface of computing and
biology. This project will bring research and educational opportunities to both
graduate and undergraduate students. It will involve several Ph.D. students
(including 2 female students), and one or two undergraduate students, from both
the Computer Science and Engineering Department and Biological Sciences
Department. As an integral part of this project, a new course in biomedical
imaging will be developed that will enable the solving of biomedical problem
with a knowledge of computer science.

This project will yield a set of efficient algorithmic techniques for the
proposed problems, and will be used as automatic (or semi- automatic) tools to
accurately determine the patterns of chromosome spatial organization inside the
cell nucleus and how such patterns dynamically change during the normal cell
cycle, during differentiation of keratinocytes into skin cells and following
progression of normal breast cells to malignant cancer. This could provide new
insight into how the global arrangement of chromosomes in the cell nucleus is
related to the malignant state. The set of algorithms will be implemented and
tested using randomly generated data and real biological data. They will be
integrated into an Algorithmic Toolbox developed in our previous research on the
spatial positioning of the cell nucleus, and will be made available to the
research community. (3) Results from this projects are likely to be used in
other areas, and have a positive impact on them. For example, the problems of
k-prototype learning, chromatic median, chromatic k-median clustering, median
point-sets, and projective clustering are all fundamental problems in computer
science and have applications in many other areas, such as machine learning,
computer vision, and data mining. Some of the solutions can be used as
information integration tools.