* 1438660
* Developing a Novel Metatranscriptomic Approach for Identifying Biomarkers Directly from Mixed Microbial Communities
* ENG,CBET
* 09/01/2014,08/31/2019
* Susan De Long, Colorado State University
* Standard Grant
* Karl Rockne
* 08/31/2019
* USD 278,467.00

1438660 De Long

Developing a Novel Metatranscriptomic Approach for Identifying Biomarkers
Directly from Mixed Microbial Communities

Quantitative molecular biology assays targeting functional biomarker genes are
powerful tools that can be used for broad applications including to guide
development of novel environmental biotechnologies and to assess the response of
natural ecosystems to perturbations. However, for many systems, knowledge of
relevant functional biomarker genes is currently lacking. Compounding this
problem, currently available methods to identify genes that can serve as
biomarkers are not usually suitable for application to environmentally-relevant
mixed microbial communities because they require isolation of pure cultures.
Thus, a critical need exists to develop novel biomarker gene identification
tools that can be applied directly to mixed microbial communities. In addition
to supporting the development of environmental biotechnologies, educational
outreach will be carried out to develop new curriculum for elementary science
education to improve overall levels of science and engineering literacy. An
interdisciplinary problem-based learning module will be developed using
bioremediation and sustainability as a theme. Engineering graduate research
assistants will work with pre-service teachers to develop the curriculum under
the guidance of the PI with the support of an elementary science education
expert. The module will be tested at three Colorado elementary schools. The
problem-based learning module will be developed to enhance development of 21st
century skills including critical thinking, question-posing, problem solving,
and innovation/creativity. An engineering doctoral student researcher will be
trained via the proposed project. Additionally, the pre-service teachers will
receive hands-on training in curriculum development. The new curriculum will be
evaluated via analysis of pre- and post-test data on student knowledge,
comprehension, and abilities to apply and synthesize information from the unit.

The long-term goal in this research program is to develop functional gene-based
microbial ecology tools and apply these tools to guide development of
sustainable environmental biotechnologies. In pursuit of this long-term goal,
the PI, with colleagues, has recently adapted and validated a differential gene
expression-based technique (prokaryotic cDNA subtraction) for identifying and
sequencing candidate biomarker genes in bacteria; however, further research is
needed to extend this approach for application to mixed cultures. Therefore, the
objective in this application is to apply this original approach to identify
specific biodegradation genes from microbial communities using anaerobic
o-xylene-degrading methanogenic cultures as a prototype model system. The goal
of this research will be achieved by 1) extending differential gene expression-
based methodology cDNA Subtraction for identification of putative biomarkers
from mixed microbial communities, and 2) validating functional biomarker genes
identified via Objective #1 for the model system. To achieve Objective 1, the
approach will be to adapt and apply the differential gene expression-based
methodology already available in the PIs laboratory to identify genes that are
specifically expressed during anaerobic o-xylene degradation in a mixed
syntrophic culture. The PI will also develop and apply in silico cDNA
Subtraction methodology using comparative metatranscriptomics (RNA-seq). To
achieve Objective 2, the approach will be to identify genes for which expression
correlates with biodegradation performance in microcosm studies via quantitative
reverse-transcription PCR. The proposed work is expected to lead to the
development of the entire work flow associated with two novel approaches to
biomarker development termed Subtractive Community Metatranscriptomics (SCM) and
in silico SCM that will allow researchers to develop biomarkers directly from
mixed microbial communities, thus overcoming a longstanding barrier that has to
date, prevented efficient identification of biodegradation genes from mixed and
syntrophic microbial communities. This contribution is significant because it is
expected to have immediate and widespread fundamental implications regarding how
functional gene-based tools are developed and applied in the fields of microbial
ecology and environmental engineering.