* 1440458
* The Small-Number Limit of Biological Information Processing
* BIO,EF
* 03/01/2014,08/31/2014
* Simon DeDeo, Indiana University
* Standard Grant
* Saran Twombly
* 08/31/2014
* USD 55,663.00

Intellectual Merit: &lt;br/&gt;Living entities of all sizes, from single cells
to multi-cellular organisms, process energy, such as light or chemical energy,
for use in growth, development and repair. They also process information,
adapting their behavior to changing circumstances in their environment and to
changing internal states. For example, a cell must adjust its production of
specific proteins in response to changes in its environment, including signals
from nearby cells, as well as new instructions from its own DNA. These responses
amount to information processing by "biological computers," which suggests that
we can advance our understanding of complex biological systems using the tools
of information technology and computer science, as long as the distinctions
between human-designed computers and naturally-occurring biological systems are
respected. This research addresses three central points of difference. In direct
contrast to current engineered systems, which are designed to minimize noise,
are deployed with circuitry fixed at time of fabrication, and rely on a small
number of identical subunits (for example, transistors on a two-dimensional
chip), biological systems appear to: (1) use noise to their advantage; (2)
dynamically adjust their processing methods; and (3) exploit an unusually
diverse set of underlying mechanisms. Aided by computer simulations, the
investigators will generalize classical results in the theory of computation,
such as the Shannon-Lyupanov bounds for circuit size, to account for the
constraints and differences known to obtain in biology. In addition they will
bring into a mathematical framework the great variety of actual processing
mechanisms, such as the parallel epigenetic regulation of gene expression, that
are being continuously uncovered in large-scale laboratory investigations. Such
work will aid in applying the vast investment made in the theory of computation
over the last sixty years to the study of biological systems.
&lt;br/&gt;&lt;br/&gt;Broader Impacts:&lt;br/&gt;The project is expected to
produce results useful to a wide scientific and engineering audience. It is
anticipated that the new algorithms to be developed for the "reverse
engineering" of biological systems will be applicable in other domains;
therefore, a portion of the project resources is devoted to making techniques
developed for the extraction of logical structures available for use by the
broader community. Project resources will also be devoted to the training and
mentoring of undergraduate and graduate students at the Santa Fe Institute in
New Mexico, which provides a unique, research-focused, interdisciplinary
educational experience. Students, recruited through the Institute's NSF REU
(Research Experiences for Undergraduates) program and through the Investigators'
network of collaborators at graduate programs, will conduct research, and
publish their results in peer-reviewed literature, under the guidance of the
Investigators. Finally, the Investigators will carry out outreach activities to
public school teachers who teach in under-served rural areas in New Mexico and
reach large numbers of students from minority groups under-represented in STEM
fields. The outreach will be conducted in collaboration with Irene Lee, head of
the NSF GUTS (Growing Up Thinking Scientifically) program. At the GUTS Summer
Teacher Institutes in Socorro, New Mexico, the PI will introduce middle and high
school teachers to the concepts of information processing in biological systems
and work with them to develop ways to incorporate this content into lesson
plans, brainstorming activities and games for use in science classrooms. The
Investigators will also, with guidance from Lee, provide mentorship and coaching
to secondary students involved in the New Mexico Supercomputing Challenge
Program.