* 1444532
* Nuclease Profiling as an Integrative Resource for Maize Epigenomics
* BIO,IOS
* 08/01/2015,07/31/2021
* Oghenekome Onokpise, Florida State University
* Standard Grant
* Gerald Schoenknecht
* 07/31/2021
* USD 2,133,825.00

PI: Henry W. Bass (Florida State University-
Tallahassee)&lt;br/&gt;&lt;br/&gt;CoPIs: Jonathan H. Dennis, Jinfeng Zhang, and
Daniel L. Vera (Florida State University-Tallahassee), Oghenekome U. Onokpise
and Violeta Tsolova (Florida A &amp; M
University)&lt;br/&gt;&lt;br/&gt;Eukaryotic chromosomes consist of DNA-protein
complexes referred to as chromatin. The current understanding of the
relationship between chromatin structure and genome behavior is still relatively
underdeveloped. The goal of this project is to identify and map genome-wide
changes in chromatin structure in five diverse tissues of maize using a novel
method developed with previous NSF funding. It is expected that chromatin
mapping will reveal previously undetected and biochemically functional regions
of the maize genome, while integrating genetic information from maize and other
plant genome research projects. Together these efforts will provide basic
information about chromatin structure that could accelerate crop improvement
strategies to address current and future challenges in food and biofuel
production. With regard to outreach and training, the project will provide
research training for postdoctoral associates, graduate and undergraduate
students, particularly from underrepresented groups. The project will organize
public field displays and hold summer science camps for high school students to
engage a diverse group of people in several Science, Technology, Engineering,
and Mathematics (STEM) disciplines related to agriculture, genomics, and
bioinformatics.&lt;br/&gt;&lt;br/&gt;A novel assay for genome-wide nuclease
profiling using micrococcal nuclease (MNase) will be used to develop reference
nuclease profiles for chromatin from five core tissues of maize: stem cells from
root and shoot tips, the immature ear, developing nutritive seed tissue
(endosperm), and pollen sperm nuclei. Measuring the differential nuclease
sensitivity (DNS-seq) across the entire maize genome will uncover localized
sites coupled to evolutionarily conserved sequences, transcription factor
binding sites, and gene expression levels. This method uniquely maps specialized
and functionally important regions of the genome while greatly enhancing the
information content of nucleosome position data, providing new opportunities to
connect epigenomic data across disparate projects. All data will be made
accessible to the public through various long-term repositories and portals.
These include the NCBI's GEO and SRA as well as the UCSC Genomaize
(http://genomaize.org/), EPIC-CoGe (https://www.plant-epigenome.org/), and the
iPlant Collaborative (Data Store, Discovery Environment, Powered-by-iPlant
genome browsers).