* 1444575
* ABR-PG: Sequencing the Hexaploid Oat Genome Using Diploid References
* BIO,IOS
* 08/01/2015,07/31/2018
* Eric Jellen, University of North Carolina at Charlotte
* Standard Grant
* Clifford Weil
* 07/31/2018
* USD 1,488,244.00

PI: Jessica Schlueter (University of North Carolina-Charlotte) CoPIs: Cory
Brouwer (University of North Carolina-Charlotte), Eric Jellen and Jeff Maughan
(Brigham Young University) Senior Personnel: Jeremy Jay (University of North
Carolina-Charlotte) Key Collaborators: Tim Langdon (Aberystwyth University,
United Kingdom), Nicholas Tinker (Agriculture and Agri-Food Canada) and Eric
Jackson (General Mills, Manhattan, KS) Oat is a grain crop that is high in fiber
and used in a number of foods that carry an FDA label listing oats as being
capable of "reducing the risk of heart disease". Although oat has this important
health benefit, the understanding of how oat fiber develops is lacking. Genomic
resources for oats have significantly lagged behind that for other grains. The
fact that oat has a very large and complex polyploid genome compared to other
crop plants has contributed to the challenges in identifying the gene networks
that control fiber development and other economically important processes and
pathways in oats. The goal of this project is to generate a draft genome
sequence of cultivated oats by sequencing related wild oat genomes that are
smaller in size to that of cultivated oats to assist in sequence assembly and
annotation. This project will utilize high throughput sequencing strategies,
optical mapping and diploid genetic resources to facilitate the assembly and
annotation of a functional hexaploid oat (Avena sativa) genome sequence. The
specific objectives include: 1) finishing the sequencing of the A. ventricosa
diploid CC-genome as well as creating an A. eriantha diploid CC-genome assembly;
2) generating physical optical maps for six hexaploid Avena genomes, two CC-
diploid genomes, two AA-diploid genomes and a putative diploid DD-genome by
using cutting edge BioNano optical mapping technology; 3) increasing GBS marker
density across parental lines of a selected linkage mapping population to
provide additional anchor points for assembled contigs that will be combined
with short-read (HiSeq) and long-read (PacBio) sequencing to assemble and anchor
the hexaploid oat genome (AADDCC); and, 4) annotating the sequenced genomes by
generating RNA-seq data for each species in conjunction with ab initio
prediction methods. Importantly, annotations will be evaluated and manually
curated through user-contributed annotation as part of the Plant Pathway
Elucidation Project (P2EP; http://p2ep.org) which is a collaborative effort
between academia and industry established at the North Carolina Research Campus.
The goal of this program is to tie together educational opportunity, scientific
discovery and the development of a knowledge base to "advance scientific
research, create opportunities for industry and consumers, and enhance human
health". Students and postdoctoral fellows associated with this project will
participate in the P2EP summer sessions. All data generated in this project will
be disseminated widely through the P2EP website, OatGlobal
(http://oatglobal.org) and Gramene (http://gramene.org). All sequence
information will be accessible through the GenBank Short Read Archive (SRA) and
the Gene Expression Omnibus (GEO). Batch download datasets will also be
available at the project website.