* 1444522
* Evolutionary Genomics of Abiotic Stress Resistance in Wild and Cultivated Sunflowers
* BIO,IOS
* 08/15/2015,07/31/2023
* Loren Rieseberg, University of Georgia Research Foundation Inc
* Continuing Grant
* Diane Jofuku Okamuro
* 07/31/2023
* USD 4,172,290.00

PI: John M. Burke (University of Georgia-Athens)&lt;br/&gt;&lt;br/&gt;Co-PIs:
Lisa A. Donovan (University of Georgia-Athens), Loren H. Rieseberg (Indiana
University), Khaled M. Bali (University of California Division of Agriculture
&amp; Natural Resources), Pedro Andrade-Sanchez (University of Arizona), and
Jennifer M. Dechaine (Central Washington U.)&lt;br/&gt;&lt;br/&gt;Key
Collaborators: Jeffrey W. White (USDA-ARS/Maricopa) and Nicolas B. Langlade
(INRA, France)&lt;br/&gt;&lt;br/&gt;Wild and cultivated plants are regularly
challenged by a variety of abiotic stresses resulting from factors such as
drought, soil salinization, and low nutrient availability. These stresses affect
plant growth and development and reduce crop productivity. Though wild plant
populations have adapted to many of these challenges, crops are often less
resilient. This is likely due to the occurrence of tradeoffs between stress
resistance and overall growth/productivity, as well as the loss of genetic
diversity during the domestication and breeding of crop plants. To combat
stress-induced yield loss and improve food security, attention has turned to the
development of stress-resistant crops, but such efforts require knowledge. An
improved understanding of the mechanisms underlying abiotic stress resistance is
thus needed to develop crops capable of feeding a rapidly growing population in
the context of an increasingly variable climate, particularly as marginal lands
are brought into production. The research conducted in this project will have
important practical outcomes that will ultimately facilitate the production of
more resilient crop plants. These outcomes include the production of actionable
knowledge regarding the molecular and physiological mechanisms underlying
resistance to abiotic stresses that limit agricultural productivity, the
identification of favorable genetic variants that can be moved from wild
populations into downstream breeding programs, and the development and
distribution of valuable germplasm resources. This project will also provide
educational opportunities for students at multiple levels, and from diverse
backgrounds.&lt;br/&gt;&lt;br/&gt;This project involves a detailed investigation
of the genomic and physiological basis of drought, salt, and low nutrient stress
resistance in cultivated sunflower and its reproductively compatible, stress-
adapted wild species that are potential donors of beneficial alleles. Sunflower
is an ideal study system for this work because the productivity of cultivated
sunflower is clearly limited by such stresses, while related wild species are
adapted to life in a variety of extreme environments. The goals of this project
are to: (1) assess resistance to drought, salt, and low nutrient stress and
related traits in cultivated sunflower and its wild relatives using traditional
and high-throughput phenotyping approaches; (2) associate variation in abiotic
stress resistance and related traits in cultivated and wild sunflowers with
specific genes, regulatory networks, and/or causal variants; (3) determine the
mechanistic basis of stress resistance via in-depth physiological and
transcriptomic characterization of genotypes with divergent stress responses;
(4) identify suitable stress resistance alleles (i.e., those exhibiting
resistance in multiple genetic backgrounds, ideally with minimal tradeoffs) for
use in sunflower breeding programs; and (5) prepare students from diverse
backgrounds for careers in the plants sciences. All germplasm resources will be
made available via deposition in public repositories (e.g., USDA-GRIN) and the
resulting data will be disseminated via peer-reviewed publications, public
presentations, and deposition into relevant public databases including the
sunflower genome database (http://www.sunflowergenome.org/), the HeliaGene
sunflower database (http://www.heliagene.org/), GenBank
(http://www.ncbi.nlm.nih.gov/genbank/), Phytozome (http://www.phytozome.net/),
the Gene Expression Omnibus (http://ncbi.nlm.nih.gov/geo/), and Dryad
(http://datadryad.org/).