* 1444543
* Genome and Transcriptome Based Prediction and Regulator Inference of Molecular and Whole-Plant Phenotypes
* BIO,IOS
* 03/15/2015,08/31/2020
* Matias Kirst, University of Florida
* Continuing Grant
* Gerald Schoenknecht
* 08/31/2020
* USD 1,956,424.00

Right now, plant breeders have to wait to determine if their seedlings will
mature into useful plants or not. This leads to delays and increase in costs of
development of new varieties that are more productive, disease resistant and
tolerant to environmental stresses. In the case of forestry, this wait can be
decades-long, thus hampering tree improvement for the timber, paper and
bioenergy industries. Development of new cultivars could be accelerated if there
was a way to tell what useful traits a plant would have based on its DNA alone.
The ability to use DNA to predict the traits of a grown plant, from its
development to its productivity, remains one of the recognized grand challenges
of plant biology. Even more relevant and incomplete is the understanding of the
chain of events that lead from DNA to trait. A complete understanding of a
plant's biological system will allow the identification of the key regulatory
points that impact its traits of economic importance, while also allowing their
prediction. In this project novel approaches will be developed to predict the
development and the growth properties of plants, and their immediate regulators.
These approaches will integrate information beyond DNA to include gene
expression data, and create a more complete description and prediction of trait
regulation. Tools developed here will be disseminated to the community by the
release of analysis software packages, and workshops will be conducted to
enhance teaching of quantitative genetics and statistical methods in predictive
models to the scientific community. The project incorporates a training
component focused on high-school teachers and students. High-school teachers
will join the investigator laboratories to learn about real-world applications
of genomics and develop curriculum with support of the University of Florida
Center for Precollegiate Education and Training (CPET). Students from
underrepresented groups will be trained in science through CPET's Student
Science Training Program. The proposed research will also provide ample
opportunities to train graduate students at the intersection between
quantitative and statistical genetics, and genomics.
&lt;br/&gt;&lt;br/&gt;Genomic prediction of complex traits is a critical tool in
animal and plant breeding. However, the ability to fully explain a phenotypic
trait is still limited, in part because several dimensions of information are
either not available or not incorporated to prediction models. In this project,
genome sequence information will be used initially to predict gene expression,
and gene expression of the ensemble of genes will be applied to predict whole-
plant phenotypes. As a last step, genome sequence and transcriptome data will be
integrated for prediction of phenotypes and their regulators. Thus, a model for
prediction that incorporates multiple layers of information will serve as a tool
for prediction of plant molecular properties (e.g. gene expression), as well as
growth and developmental traits. This project will make use of a large Populus
deltoides association population for which abundant genome, transcriptome and
phenotypic resources are already available or will be developed here. Prediction
tools will also be applicable to other species for which similar type of
resources are available. The research methods to be applied to each aim will
consist of four components: (1) detection of large-effect factors (e.g.,
individual or groups of variants with large regulatory effects) using single-
factor regression methods, (2) development of prediction models that combine
whole-genome information from large and small-effect factors, (3) biological
validation of findings using genetic modification, and (4) statistical
validation using an independent population. Genotypic and gene expression data
will be publically available in NCBI's dbSNP and the sequence read archive/gene
expression omnibus databases (www.ncbi.nlm.nih.gov). Analysis packages will be
deposited in the R Archive (http://cran.r-project.org/), and project details and
plant resource access procedures will be described in the project web site
(http://sfrc.ufl.edu/forestgenomics/Kirstlab/).