* 1444561
* Identification of Translational Hormone-Response Gene Networks and cis-Regulatory Elements
* BIO,IOS
* 08/01/2015,07/31/2024
* Jose Alonso, North Carolina State University
* Continuing Grant
* Gerald Schoenknecht
* 07/31/2024
* USD 3,199,404.00

Co-PIs: Anna Stepanova (Plant and Microbial Biology, North Carolina State
University), Steffen Heber (Computer Science, North Carolina State University),
Cranos Williams (Electrical and Computer Engineering, North Carolina State
University)&lt;br/&gt;&lt;br/&gt;Unlike animals that can seek shelter to escape
from bad weather, plants cannot move and must spend their entire lives in one
place, rain or shine. Plants are able to adapt to challenging conditions by
adjusting how they grow and how much nutrients they take in from the
environment. Understanding these adjustments of plants in response to
environmental challenge is important for a basic understanding of how plants may
respond in rapidly changing environments, and will yield insights about ways to
increase agricultural production to support a fast growing human population.
Basic research suggests that plants, like animals, use a small set of signaling
chemicals, hormones, to sense the environment, and to control their metabolism
and growth. Since hormones play such a key role in regulating plant development
and responses to the environment, it is important to identify the main 'genetic
hubs', where environmental signals and hormonal changes are integrated in order
to control a plant's life. This basic scientific knowledge will improve our
understanding of how plants grow and react to stress and will help in the
development of new crop varieties that perform well under extreme conditions.
Traditionally scientists have studied how hormones influence gene activity by
examining gene expression by measuring the copying of DNA into RNA, the first
step in the transfer of genetic information into proteins, which play an
enormous variety of functional roles in cells. However, an increase in
expression does not always mean that more protein will be made because protein
production from encoded RNA molecules is a highly regulated process.
Fortunately, the recently developed Ribo-seq technology provides the capability
of measuring protein production by every active gene in the genome at once. By
using this technology, 'genetic hubs' that connect environmental sensing with
the control of plant growth can be uncovered. Understanding how these hubs work
at the molecular level will add new tools to the "biotechnological toolbox" that
scientists can then use to develop better and more resilient
crops.&lt;br/&gt;&lt;br/&gt;Genome-wide changes in translation activity in
response to ethylene have been quantified at codon resolution by taking
advantage of the recently developed ribosome footprinting technology, making it
possible to identify new translational regulatory elements in Arabidopsis.
Importantly, detailed characterization of one of the novel regulatory elements
indicates that regulation of translation is not miRNA dependent, and that the
element identified is also responsive to the plant hormone auxin, implicating
this element in the interaction between ethylene and auxin. These findings not
only confirm the basic biological importance of translational regulation and its
potential as a signal integration mechanism, but also open new avenues for
identifying, characterizing and utilizing additional regulatory modules of gene
expression in plant species of economic importance. Towards that goal, a plant-
optimized ribosome footprinting methodology will be deployed to examine the
translation landscape of two plant species, tomato and Arabidopsis, in response
to two plant hormones, ethylene and auxin. A time-course experiment will be
performed to maximize the detection sensitivity and diversity (early vs. late
activation) of translational regulatory elements. The resulting data, reflecting
temporal changes, will be used to generate hierarchical transcriptional and
translational interaction network models for the two hormones, and to identify
key regulatory nodes. Finally, the comparison between two plant species will
provide critical information about conservation of the regulatory elements,
thereby informing research on future practical applications. All the sequence
data will be made publicly available through the NCBI-SRA. All biological
materials and genomic data will be available through a dedicated project website
at brcwebportal.cos.ncsu.edu/plantriboprints). In addition, software and
contents of the genome browser will be deposited at iPlant to ensure long-term
public access. Arabidopsis seeds and DNA constructs will be deposited at the
Arabidopsis Biological Resource Center. A new outreach program to promote
interest among middle- and high-school kids in combining biology, computers, and
engineering will be developed. A web-based bilingual dissemination tool,
Plants4Teens, as well as live monthly demos at the North Carolina Museum of
Natural Sciences and local schools, will be implemented in this new outreach
program. Examples of demonstration modules will include comparisons between
simple electronic and genetic circuits.