* 1457832
* SG: Will phylogenomics resolve the most difficult parts of the tree of life: an empirical evaluation with the turtle genus Pseudemys
* BIO,DEB
* 04/01/2015,03/31/2018
* H. Bradley Shaffer, University of California-Los Angeles
* Standard Grant
* Simon Malcomber
* 03/31/2018
* USD 150,000.00

Species are the most important biological unit of scientific research,
conservation and management. As a society, we protect species under the
Endangered Species Act, we observe, hunt, and admire species in their native
habitats, and we manage national parks for species conservation. As new
molecular technologies and tools have become available, researchers can now use
the entire genome, including billions of pieces of information, to precisely
determine how many species occur in a taxonomic group or geographical region in
ways that were inconceivable a few years ago. As these extremely informative
data sets become available, those who study species and speciation are currently
facing the dilemma of what data to collect and how to analyze those data so that
species can be quickly, accurately, and inexpensively identified and delimited
across all of life. In this project, the research team will evaluate the three
most promising, cutting edge approaches that are currently in use to determine
which genomic method is optimal for species identification and delimitation in a
group of freshwater turtles from the southeastern United States. This group,
although widespread and well studied ecologically, has thwarted accurate species
delimitation for decades; it also contains two critically endangered taxa. The
work in this project will provide critical guidance for the large group of
biologists who seek to identify, delimit, and protect species using genomic
data.&lt;br/&gt;&lt;br/&gt;In this analysis, the turtle genus Pseudemys is used
as an empirical example of an extremely difficult group to determine species
delimitation and interrelationships. The project uses a comparative genomic
approach with extensive rangewide sampling of all nine potential taxa within
Pseudemys to evaluate the relative efficacy and power of three genomic
approaches: restriction-site associated DNA (RADseq), targeted enrichment where
known portions of the genome are sequenced using probes designed from existing
genomic resources, and low-coverage, whole genome resequencing. A novel
comparative approach is utilized to ask which of these data streams performs
optimally based on cost, ease and efficiency of lab work, and empirical
performance. In so doing, two fundamental issues in systematics will be
addressed. First, what are the prospects for identifying lineages, and for
reconstructing species trees for extremely difficult groups using large amounts
of genomic data? And second, given the current methods available for generating
and analyzing genomic data, what approach is optimal? Overall, the results of
this project will provide both general guidelines for analyses of difficult
groups and clear resolution of this empirical case study of an important faunal
component of the southeastern United States.