* 1458509
* ABI Development: Utilization of diverse data in exploring protein-protein interactions
* BIO,DBI
* 06/01/2015,05/31/2019
* Sandor Vajda, Trustees of Boston University
* Standard Grant
* Peter McCartney
* 05/31/2019
* USD 605,088.00

Each living cell is packed with proteins that continuously interact with each
other in response to envirtonmental and other signals to control the cell's
growth and eventual fate. The analysis of such interactions is crucial for
understanding signal transduction pathways that drive cell differentian during
development and throughout the life an an organism. These pathways are often
involed in disease progression, such as cancer, so a broader impact of
understanding these pathways could be the design of drugs to modulate such
pathways. Moreover, such an understanding of protein protein interactions may
aid the develpment of protein-based biomaterials. Current technologies exist for
providing a blueprint for large protein interaction networks, but a deeper
understanding requires detailed structures of the complexes formed by protein
pairs or partners. Structural information is frequently difficult or even
impossible to obtain by experimental tools, emphasizing the need for
computational approaches. Vajda and Kozakov at Boston University have developed
the web based server ClusPro for predicting the three dimensional structures of
protein-protein complexes. According to the worldwide experiment CAPRI (Critical
Assessment of Predicted Interactions), ClusPro consistently has been the best
protein-protein docking server. It has over 5000 registered users and performs
around 3500 docking calculations each month. Structures generated by the server
have been reported in over 350 research papers. The goal of this project is to
develop the next generation of ClusPro that will be able to optimally utilize
the vast amount of data accumulated in public bioinformatics databases in order
to improve the reliability and accuracy of the predicted structures. Since the
server is used primarily by biological and chemical scientists who may not have
expertise in the use of current bioinformatics tools, the new version will
provide convenient access to state-of-art methods. Integration of bioinformatics
and computational biophysics approaches, in combination with experimental
validation, will result in a unique and powerful research tool. The increased
availability of protein complex structures will have a major impact in many
areas of biology, biochemistry, and biotechnology. All software developed will
be released free of charge for academic and governmental use. In addition, the
project will be used to train a new generation of graduate students, who will be
able to optimally combine data from a variety of experimental and bioinformatics
techniques with high performance computing. The use of the server will also be
incorporated into undergraduate courses to teach aspects of bioinformatics and
biophysical principles of molecular recognition.

The current version of ClusPro systematically samples the conformational space
of a target protein complex, and scores the structures using physics based
energy functions. The major shortcoming of this approach is that it performs
docking without consideration of the large body of interaction, sequence,
structural, and experimental information available in public databases. Rather
than simply generating and scoring docked structures of the starting proteins,
the new approach will use their sequences to collect all orthologs to identify
the globular domains and inter-domain regions that may contain short linear
motifs (SLIMs). Domains that occur in a number of interacting protein pairs are
likely to mediate the interaction, enabling the identification of the specific
domains that interact with domains or SLIMs of the partner protein.
Superimposing the domains from different orthologs identifies the conserved
regions. Since the interface in complexes is sequentially and structurally more
conserved relative to the rest of the protein, for the case of domain-domain
interactions docking just the key segments generally yields near-native docked
structures. Docking key segments from different orthologs and selecting
consensus models improves the reliability of predicted structures. This approach
substantially improves the docking of proteins that have flexible loops, and
will be extended to the docking of homology models. If the interface includes a
SLIM, docking structures of peptide fragments that contain the motif extends the
method to proteins with interactions mediated by flexible or unstructured
regions. The algorithms to be developed will be implemented in ClusPro, which
will provide substantial new information on protein-protein complexes. ClustPro
software can be accessed at http://cluspro.bu.edu.