* 1458347
* ABI Innovation: A new computational framework for the prediction of microbiome dynamics
* BIO,DBI
* 09/01/2015,08/31/2019
* Vanni Bucci, University of Massachusetts, Dartmouth
* Standard Grant
* Peter McCartney
* 08/31/2019
* USD 491,682.00

The dynamics of microbial communities play a fundamental role in the functioning
of many natural, engineered and host-associated systems. Even though the
application of DNA sequencing technologies has allowed profiling the response of
these communities to external perturbations, the important knowledge resulting
from this approach stems from descriptive and correlation-based analysis of
these data. This strongly limits the understanding of the ecology (e.g. how the
microbes interact) of these systems and, more importantly, hinders the ability
to make quantitative predictions. This project will deliver new theoretical
methods and related computational algorithms that, for the first time, allow
forecasting microbiome dynamics that are typically constrained with sequencing
surveys. This will benefit researchers working on host-associated and
environmental microbiomes as it will enable them to computationally explore
scenarios that are difficult to set-up experimentally. The tools developed in
this project will be delivered as an open-source, freely downloadable and
upgradable package, and will encourage and enable end-user contributions with
the novel scripts and algorithms provided. Multiple graduate and undergraduate
students will be included in the project and will benefit from interdisciplinary
hands-on training in mathematical and computational biology, statistics, and
microbial genetics. As the proposed methods combine concept from multi-linear
regression and solution of large systems of differential equations, they will
perfectly integrate with coursework in Biostatistics and Theoretical Biology at
UMass Dartmouth. &lt;br/&gt;&lt;br/&gt;This research will deliver the first
computational suite that allows for simulating and predicting microbiome
dynamics consistent with metagenomics observations. This will be achieved by: 1)
the development of new time-reverse engineering inference methods solved by a
combination of regularized regression and quadratic programming for the
estimation of an optimal set of model parameters, and 2) the application of
computational tools for metagenome reconstruction based on modeling predictions.
This research will also include the development of explicit and implicit
numerical methods for the solution of large systems of differential equations to
predict microbiome transient dynamics and the use of linear stability analysis
to determine all possible microbiome predicted configuration states in response
to different sets of perturbations. Method testing and validation against data
from simple in silico and in vitro microbial ecosystem of known ecological
structure will allow accuracy testing of the proposed approaches both for
predicting temporal dynamics and in recovering the correct microbial interaction
network in response to external perturbation. Method application (and
validation) on diverse datasets will allow testing of fundamental hypotheses
about the role of the intestinal microbiome in resisting colonization by foreign
bacteria and in shaping host immunity. More information about this project can
be found at: http://www.vannibucci.org/research-interests.html