* 1462629
* Algebraic Geometric Approaches to Biological Complexity
* MPS,DMS
* 09/15/2015,08/31/2021
* Jeremy Gunawardena, Harvard University
* Continuing Grant
* Junping Wang
* 08/31/2021
* USD 1,447,148.00

The Human Genome project and subsequent developments have given us unprecedented
knowledge of the genes and proteins within us. There is much anticipation that
this knowledge will translate into substantial improvements in human health and
well-being. A central challenge, however, is the enormous complexity that we
encounter at the molecular level. It is not merely that we have 20,000 genes but
that the machinery that regulates each gene is itself extraordinarily complex
with, in the case of some genes, billions of potential states. Moreover, when
the gene is expressed as a protein, each protein may itself become modified so
that it too can exist in many different states, which may have profoundly
different consequences. This project will use two mathematical formalisms
developed in previous work -- the "linear framework" and the method of
"invariants" -- that have been shown to be capable of rising above the
complexity and eliciting general principles. The investigators will use these
formalisms to analyze two focus problems in which the underlying complexity has
proved intractable: (1) "epigenetic" mechanisms in gene regulation, which play a
central role in how the environment influences gene expression, and (2)
decision-making switches based on protein modification states, which play a
central role in controlling cellular behavior. A key aspect of the project is
the involvement of undergraduate students from the mathematical and physical
sciences, especially students from under-represented groups. The project will
act as pipeline to motivate such students towards research in the biological and
bio-medical sciences, a strategy that contributed substantially to the success
of previous work. &lt;br/&gt;&lt;br/&gt;The biochemical reactions within cells
are generally assumed to follow the principle of mass action -- the rate of a
reaction is the product of the concentrations of the reactants -- so that their
behavior can be represented by a polynomial dynamical system. The steady states
of a network of reactions are therefore solutions to a set of polynomial
equations and form an algebraic variety. The investigators have pioneered the
use of methods from algebraic geometry in the study of biochemical networks,
leading to the introduction of the "linear framework" and the method of
"invariants," both of which can distill biological principles in the face of
molecular complexity. This project will further develop these approaches by
attacking two major focus problems. First, the investigators will analyze the
behavior of protein modification switches, which are central motifs in
regulating cellular signaling pathways and cell behavior, and will characterize
them in terms of their invariants. Second, the role of energy-dissipating,
epigenetic mechanisms in gene regulation will be analyzed. In previous work, the
investigators showed that energy dissipation leads to a profound increase in
complexity through the phenomenon of history-dependence. Methods for
interpreting that complexity and identifying how energy-dissipation contributes
to environmental modulation of gene expression will be developed.