* 9723098
* Regulation of Plasmid Stability by pAD1 par
* BIO,MCB
* 10/01/1997,09/30/2001
* Keith Weaver, University of South Dakota Main Campus
* Standard Grant
* Philip Harriman
* 09/30/2001
* USD 300,000.00

Weaver 9723098 A small determinant, no larger than 404 bp, has been identified
on the bacterium Enterococcus faecalis plasmid pAD1 that is essential for stable
inheritance of the plasmid. This determinant, designated par, stabilizes the
plasmid via a post-segregational killing mechanism. Stability determinants of
this class stabilize plasmid inheritance in a bacterial population by killing
host cells that lose the plasmid. Such systems are common on plasmids native to
Gram-negative bacteria and generally encode a stable toxin and an unstable
antidote. The antidote prohibits either toxin activity or translation. Par is
the first such system identified on a plasmid native to a Gram-positive
bacterium. Preliminary evidence suggests that par is regulated by a unique
antisense RNA mechanism. The par "antidote" is a small RNA that contains
segments complementary to both the 3' and 5' ends of the toxin-encoding RNA. The
5' complementary segments would be expected to sequester the translational start
site for the proposed toxin of the system, but the 3' end may be necessary for
initial interactions between the RNAs since it encodes the only complementary
stem-loop structure present in both RNAs. Such stem-loops have been shown to be
important for RNA-RNA interaction in other antisense systems. The putative par
toxin is a small peptide of 33 amino acids. Evidence indicates that the toxin
target is involved in chromosomal separation and/or cell division. The
objectives of this project are to i) determine the molecular mechanisms of
regulation of par function, ii) determine the nature of the toxin, and iii)
identify the molecular target of the toxin. Three experimental approaches will
be taken to address the first objective. First, par will be cloned on a
conditional replicon and its effects on host viability assessed after a switch
to non-permissive conditions. Plasmid loss should lead to host cell death. This
experiment will verify the post-segregational nature of par-mediated killing.
Second, relative stabilities of the antisense antidote and the toxin message
will be determined in rifampicin-exposed cells. If par is regulated like other
post-segregational systems, the antisense antidote should be less stable than
the toxin message. Processing intermediates of the toxin message may also be
identified by this approach. Third, the interaction between the par RNAs will be
reproduced in vitro, where the specific elements involved in the interaction
will be determined. RNAs will be produced in vitro using templates produced by
PCR and linked to T7 RNA polymerase promoters. Interactions will be assessed by
fractionation on various polyacrylamide gel formulations. Mutations of elements
suspected of being required for interaction will be introduced by PCR-mediated
mutagenesis. The second objective will be approached from both genetic and
biochemical directions. In the genetic approach, the effects of mutations in the
proposed toxin reading frame but not affecting the RNA structure and vice versa
will be assessed. If the peptide is the toxin, mutations affecting its coding
frame should result in loss of function regardless of effect on RNA structure.
In the biochemical approach, production of the peptide will be assessed in vitro
in the presence and absence of the antidote. In addition, antibody will be
produced to peptides related to the putative toxin. These peptides will be
produced either synthetically or using translational fusion vectors. The
antibodies will be used to visualize the toxin in cell extracts by Western blot
and may also be used for protein purification. The third objective will be
addressed by isolating mutations in E. faecalis host cells resistant to the
over-production of toxin. The affected gene will be identified by
complementation analysis. Possible effects of toxin over-production on other
bacterial hosts, such as Escherichia coli and Bacillus spp. will also be
determined. This study should make a significant contribution to the understandi
ng of the maintenance of extra-chromosomal genetic elements, particularly in
Gram-positive hosts. Furthermore, the unique nature of the complementarity
between the two par RNAs suggests that the study of their interaction may
provide information on the variety of ways in which RNAs can interact,
information that would be useful in fields ranging from evolution to the
development of antisense therapeutics. Finally, the par toxin may provide a
useful tool for the study of chromosomal separation and/or cell division in E.
faecalis, a Gram-positive coccus. Very little data on these basic life processes
are available for such organisms.