* 9723129
* Interactive Regulatory Networks Controlling Metabolic       Adjustments in de novo Pyrimidine Biosynthesis
* BIO,MCB
* 09/01/1997,08/31/2001
* James Wild, Texas A&M AgriLife Research
* Continuing Grant
* Hector E. Flores
* 08/31/2001
* USD 324,060.00

9723129 Wild The management of intracellular pyrimidine pools via the regulation
of de novo biosynthesis involves one of the most delicate physiological
balancing systems in cellular metabolism. While it is necessary to maintain
specific nucleotide levels in order to sustain RNA synthetic capacity, all of
the nucleoside triphosphate pools are interdependent and energetically costly to
maintain. In response to these basic needs, there is an intricate, synergistic
network of controls which have developed at the level of gene expression and
allosteric modification of individual enzymatic steps. However, as has been
observed with other metabolic pathways, the detailed description of individual
regulatory components does not adequately describe metabolic/growth responses at
the pathway (much less organismal) level. The classical concept of a primary
regulatory control point, which for the pyrimidine pathway has traditionally
been considered to be the aspartate transcarbamylase (ATCase), does not
adequately predict pathway responses to metabolic perturbations. However, the
wealth of information and the variety of genetic constructs which are now
available for most of the individual steps of the pyrimidine pathway have
prepared the way for development of a system-level understanding of its
metabolic role. This project addresses four specific aspects of the regulation
of pyrimidine nucleotide metabolism in Escherichia coli: 1) Evaluation of the
genetic and metabolic regulatory features of arginine/pyrimidine metabolism in
mediating responses to environmental/nutritional shifts. 2) Integration of the
various control processes, genetic and biochemical, through in vivo studies with
genetic constructs which selectively modify individual components; leading to 3)
Pathway modeling in the analysis of the biological regulatory network. Given the
central role that pyrimidine biosynthesis plays in cellular metabolism, and the
regulatory relationship of this pathway with other central pathways, an
appropriate un derstanding of the regulatory logic at an integrated
physiological level rather than an isolated, mechanistic one is important.
Nucleic acids are the "informational molecules" of living organisms and as such
contain the information necessary for cell growth and division. Nucleic acids
are constructed from a set of smaller components (nucleotides) which are
identical in all living species. These nucleotides fall into two classes:
purines and pyrimidines. This project studies the biochemistry and metabolism of
the pyrimidine nucleotides, the details of which are critical for an
understanding of DNA and RNA synthesis and regulation of gene expression and
cell growth. The wealth of knowledge available on the genetics and regulation of
the individual steps in pyrimidine metabolism in the bacterium E. coli makes it
a model system for studying these complex issues. As knowledge is gained from
the simpler, bacterial systems, application to the more complex and inter-woven
pathways of eukaryotic systems becomes possible.