* 9727630
* RNA-Protein Interactions in a Double-Stranded RNA Virus of  Saccharomyces Cerevisiae
* BIO,MCB
* 05/15/1998,04/30/2002
* Jeremy Bruenn, SUNY at Buffalo
* Continuing grant
* Parag R. Chitnis
* 04/30/2002
* USD 180,000.00

Jeremy A Bruenn MCB 9727630 RNA-Protein Interactions in a Double-Stranded RNA
Virus of Sacchromyces Cerevisiae 1. Technical Abstract - Many of the double-
stranded RNA viruses of lower eucaryotes are non-infectious viruses that persist
indefinitely by replicating in tandem with their host cells, their spread
mediated entirely by sexual and asexual reproduction. They have adapted to this
lifestyle in some cases by packaging extra RNAs inessential to the virus but
encoding products of selective value to their host cells. These products are
cellular toxins that kill cells not harboring the virus (the killer phenotype).
Several such systems are known among the fungal dsRNA viruses, the best studied
of which is the Saccharomyces cerevisiae virus (ScV). A number of
characteristics make this system very attractive for studying nucleic acid-
protein interactions. First, the host cells are readily amenable to molecular
and classical genetics. This makes it possible to use the killer phenotype for
screening and selection of viral mutants. Second, the virus separately
encapsidates its RNAs, making many experiments simpler than those in the dsRNA
viruses of higher eucaryotes, many of which have as many as 12 unique dsRNAs
encapsidated in each viral particle. Third, extensive analysis of the single
essential large viral dsRNA (4580 bp) has mapped many viral functions. There are
only two viral proteins, one the major capsid polypeptide and product of the cap
gene, and the second a multifunctional protein the result of frameshifting
between the two open reading frames (cap and pol) to produce a Cap-Pol fusion
protein. The Cap-Pol fusion protein has the enzymatic and packaging activities
of the virion. Among the cis-acting sites is the viral binding site (packaging
site) which has (for RNA) a very high affinity for its recognition protein (the
Cap-Pol fusion protein), with a Kd of less than 1 nM. The present proposal study
seeks to understand the nature of the interaction between Cap-Pol and this b
inding site. This will involve further in vitro selection of binding sites,
cross-linking and mutagenesis experiments to map active site residues in the
Cap-Pol protein, structural determination of the 20-base RNA binding site, and
optimization of the Cap-Pol binding domain. This is one of the few systems in
which both in vivo and in vitro systems for selection and analysis of altered
binding sites and binding protein exist. It promises to contribute significantly
to understanding RNA-protein interactions. 2. Non-technical Abstract - There is
a group of small viruses of fungi and protozoans in which the genome is a single
large double-stranded RNA. These viruses express their genomes by transcribing
the RNA in the viral particles and extruding one strand (the plus strand) into
the cytoplasm. Their RNAs may either be translated by the cell?s machinery or
packaged in newly assembled viral particles. Consequently, the viral particles
must choose from among all the RNAs in the cell the one RNA that is viral for
packaging. This is accomplished in the yeast virus by the specific recognition
of a small sequence in the RNA by a domain of 75-150 amino acids in the viral
polymerase. This interaction is examined in this study by a number of molecular
biological, biochemical, and biophysical techniques. The very high affinity of
the RNA for its viral protein and the very small RNA sequence recognized only 20
bases) and the simplicity of the packaging process in this group of viruses
makes this system an attractive one for understanding how RNA viruses construct
their particles.