* 9728277
* Studies on RNA Recombination In Vivo and In Vitro
* NONE,NONE
* 03/01/1998,10/31/2000
* Clifford Carpenter, University of Massachusetts Amherst
* Continuing grant
* Susan Porter Ridley
* 10/31/2000
* USD 375,000.00

9728277 Simon The TCV RdRp, after synthesizing full-length or near full-length
plus-strands of the associated RNA sat-RNA D, switches to using minus-strands of
either a second associated RNA, sat-RNA C, or the TCV genomic RNA with
transcription reinitiating just 3' of stable hairpins. This system has been
reconstructed for in vitro studies by creating a chimeric RNA consisting of the
recombination hot-spot from sat-RNA D plus-strands at the 3' end connected by a
short loop to sat-RNA C minus-strands including the sat-RNA C recombination hot-
spot hairpin (motif1-hairpin). Using partially purified TCV RdRp, recombination
in vitro is detected as an intramolecular extension product from the 3' end of
the sat-RNA D sequence using the sat-RNA C-derived region as template. The
motif1-hairpin and a short intrastrand base-paired region just upstream of the
motif1-hairpin are important for primer extension both in vitro and in vivo.
Recombination between sat-RNA D and TCV genomic RNA in vivo does not apparently
involve a base-paired region 3' of the TCV motif3-hairpin recombination hot-
spot. In addition, transcription start sites for subgenomic (sg) RNA promoters
are hot-spots for recombination in vivo. This research will: ( 1 ) establish an
in vitro primer extension system for the TCV motif3-hairpin region and (2) sgRNA
promoter region analogous to the system established for the sat-RNA C
motif1-hairpin. Using a mutagenesis strategy, the roles of the hairpins and
single-stranded regions surrounding the motif'3-hairpin or sgRNA promoter
hairpin will be determined. In addition, the in vivo effects of adding sequences
capable of forming heteroduplexes with sat-RNA D plus-strands upstream and
downstream of the TCV motif3-hairpin and sgRNA promoter will be established. (3)
The mechanism of RdRp processing of the 3' end of the nascent strand will be
determined using the in vitro primer extension constructs labeled at the 3' end.
By examining small labeled products of the primer-extension reaction, a determ
ination of whether processing involves endonucleolytic or exonucleolytic
activity and also the length of the 3' end sequence that is removed will be
made. RNA recombination is one of the major processes involved in the evolution
of RNA virus genomes. In addition, analyses of sequences and structures that are
important for recombination have provided major insights into a poorly
characterized enzyme, the enzyme encoded by all RNA viruses that replicate
through RNA intermediates, called RNA-dependent RNA polymerase (RdRp). The
turnip crinkle virus (TCV) system is an unusually powerful system for studying
RNA recombination since recombination can be studied using the whole organism
(plant), cell culture (protoplast) and in vitro systems. In addition,
recombination in the TCV system is between subviral (satellite) RNAs (such as
sat-RNAs C and D) that are non-coding and are substantially smaller than the
viral genomic RNA, or between sat-RNAs and the genomic RNA. This research will
establish the sequences and structures on the RNA molecules that are important
in the recombination event. In addition, the ability of the RdRp to repair
truncations at the ends of virus RNAs using a process similar to recombination
will be analyzed.