* 9728230
* Genomic Molecular Evolution in E. coli
* BIO,MCB
* 06/01/1998,05/31/2002
* Roger Milkman, University of Iowa
* Continuing Grant
* Karen Cone
* 05/31/2002
* USD 343,000.00

Milkman 9728230 The intent of this project is to relate several molecular and
population properties to the diversity and evolution of the E. coli species
genome. Evidence is increasing that a high proportion of the E. coli genome
consists of genes imported over tens of millions of years from phylogenetically
distant (but physically close) sources. Since the genome size must be limited,
some existing genes must be eliminated from each chromosome -- but not
necessarily the same genes from every chromosome. For each gene added relatively
quickly to the species, 100 genes (for example) might decline in frequency by
1%. This would lead to abundant presence/absence variation (in addition to
allelic variation, already well known). The complete E. coli strain MG1655 DNA
sequence has revealed many genes whose specific function is not known. Some (not
all) of these may have reduced value and disappear from individual lines without
leaving the species altogether. In a diverse set of strains, PCR and Southern
hybridization will test for presence/absence polymorphism. Now in the two
hypervariable regions, replacements are selected which carry novel variants
(O-antigens or restriction-modification systems) until they are no longer rare.
This results in a downward gradient of replacement rates outward from the
selected common anchor point, since replacements vary in length. Using 11
strains relatively recently diverged from a common ancestor, a place near the
well-defined hypervariable rfb region is chosen in which each individual strain
acquires, during the period since divergence, about one distinct replacement
from more distant strains. Sequencing defines the end of the replacement.
Finally, since bacterial chromosomes are dividing progressively though the
entire cell cycle, fragments of incoming DNA can be incorporated into one or
another nascent chromosome and detected by the restriction analysis of PCR
fragments amplified from segregating daughter colonies identified by standard
markers. This project exploits the completion of one E. coli genome, which
defines in ultimate detail the entire genetic content of a representative member
of the species. As a biological species cannot be described fully without a
complete organism, a species genome cannot be fully described from a fragmentary
set of genes. Now, given the available analytic tools, including biochemical
methods and population genetic theory, the structural diversity (DNA sequence
variation, including presence/absence), mechanisms of change (transfer of DNA to
an E. coli cell from a different member of the species or from outside the
species; production of several variant types after a single transfer; various
forms of natural selection), and the dynamics of change (rates of DNA transfer
as a function of chromosomal region , clonal sweeps -- the multiplication and
spread of genome-wide clones, as well as smaller regions of the chromosome) can
be considered and addressed experimentally. This leads to a clearer
understanding of the specific genetic properties of various sorts of E. coli as
well as the genetic properties common to the entire species. Many of the results
should be applicable to higher organisms.