* 9728281
* Function & Regulation of the Bradyrhizobium japonicum nolA  Gene
* BIO,MCB
* 04/01/1998,03/31/2001
* Gary Stacey, University of Tennessee Knoxville
* Continuing grant
* Philip Harriman
* 03/31/2001
* USD 325,600.00

Stacey 9728281 Recent research has identified several large families of
bacterial global regulators. Although such global regulators are being
extensively studied, relatively little is known about how such regulatory
systems may interact to integrate the cellular response to the environment, and
few model systems have been identified to study such integrated responses.
Nodulation (nod) gene expression in Bradyrhizobium japonicum is regulated by
members of the LysR, two-component, and MerR families of transcriptional
regulatory proteins. Therefore, it is possible that regulation of nod gene
expression in B. japonicum represents an interesting and useful system in which
to explore the interactions between these global regulatory systems. In B.
japonicum, a LysR homologue, NodD1, is required for nod gene activation in the
presence of plant produced isoflavones. NodV and NodW make up a second,
independent system that activates nod gene expression upon addition of the
isoflavone inducer. Finally, NoIA, a member of the MerR family, acts indirectly
to repress nod gene expression and likely activates the expression of other
genes. This research focuses primarily on the regulation and function of the
nolA gene. Surprisingly, recent results indicate that the nolA gene encodes
three separate proteins (termed NolAL, NolAM, and NolAs ), at least two of which
are synthesized from separate messenger RNAs. Moreover, nolA expression appears
to be activated by a unique plant signal molecule. Site directed mutations have
been constructed that, in each case, result in the expression of only one form
of NoIA. Using these constructs, the independent regulation of the three NolA
proteins is being investigated. Which of the three forms of NolA is important in
determining symbiotic host range among genotypes of soybean will be determined.
The nolA gene is expressed from at least two independent transcriptional start
sites (P1 and P2). Expression from P1 appears to require a novel plant signal
molecule, while expressi on from P2 requires a functional NoIA. Mutations in the
5' region of both P1 and P2 will be constructed to test between competing models
of how their independent regulation may be mediated. Expression of NoIAM and
NoIAS may be due to a third, unidentified transcriptional start site, or due to
a posttranscriptional regulatory event. In addition, the possibility that
secondary structure in the vicinity of ATG2 (initiation of NolAM) may be
important in determining the relative expression of NolAM and NolAS will be
determined. Finally, purification of the unique chemical from soybean plants
that induces nolA expression will be done. These studies should clarify the role
of NolA in nod gene expression. The results obtained, when added to other on-
going research, should give a more complete picture of nod gene expression in B.
japonicum and may serve as a paradigm for other complex regulatory systems where
global regulatory networks impinge on the expression of a single operon. The B.
japonicum-soybean symbiosis is an important system for exploring the cell
signaling pathways between host and symbiont. The ultimate goal of this work is
a clearer understanding of the integration of regulatory mechanisms involved in
the global regulation of cellular metabolism.