* 9729599
* A Rapid and Inexpensive Whole-Genome Mapping Method for     Arabidopsis Using Single Nucleotide Amplified Polymorphisms (SNAPS), Co-Dominant Ecotype-Specific PCR-based Markers
* BIO,MCB
* 02/01/1998,01/31/2001
* Frederick Ausubel, Massachusetts General Hospital
* Continuing Grant
* DeLill Nasser
* 01/31/2001
* USD 467,900.00

Ausubel 9729599 Experiments are described which are designed to develop and map
350 PCR-based DNA mapping markers for Arabidopsis thaliana. Each marker will
correspond to a mapped single nucleotide polymorphism (SNP) from the Columbia
and Landsberg erecta ecotypes. These SNPs will be detected using specially
designed allele-specific PCR primers that will generate allele specific patterns
(i.e., detectable amplification product only from one ecotype) that can be
rapidly and reliably scored by a variety of analytical methods. The acronym SNAP
has been coined to designate single nucleotide amplified polymorphism's. The
proposed experiments involve the synthesis of 350 sets of SNAP primers (each
consisting of two primer pairs, one for each ecotype), each of which
preferentially amplifies a single, mapped DNA sequence from Columbia or
Landsberg erecta DNA. Using these co-dominant SNAPs, it will be possible to map
any Arabidopsis gene with only a few hours of labor to an accuracy of
approximately 2cM, using DNA isolated from a single leaf, harvested from a
limited number of F2 (Columbia X Landsberg erecta) progeny. The specific aims
are: 1) To use 350 previously identified sequences that contain single
nucleotide polymorphism's between the Columbia and Landsberg erecta ecotypes of
Arabidopsis to generate SNAP primers that will preferentially amplify either the
Columbia or the Landsberg erect allele 2)To map the SNAP markers generated using
DNA from 96 recombinant inbred lines from a Columbia X Landsberg erecta cross 3)
To develop methods that will allow many genes to be mapped in an efficient
manner using SNAPs. This will include the re-design of the reverse primers to
adjust the sizes of the products to allow amplification of several markers in
the same amplification reaction. The usefulness of SNAP markers for detecting
polymorphisms in DNA pooled from several individual F2 mutants will also be
investigated. )4 To test the feasibility of adapting SNAP mapping to automation
using DNA micro array analysis by modeling a DNA microarray using a traditional
dot blot hybridization procedure. The proposed approach will be the fastest,
simplest and potentially the least expensive of all the whole-genome mapping
methods developed to date. The sequences of the primers, as they are designed
and mapped, will be provided to the Arabidopsis community. Additionally,
agreements with a commercial firm will be arranged to synthesize sets of primers
as inexpensive "mapping kits".