* 9302889
* Functional Inactivation of a DNA Repair Protein Complex
* BIO,MCB
* 08/01/1993,01/31/1997
* Martin Marinus, University of Massachusetts Medical School
* Continuing Grant
* Philip Harriman
* 01/31/1997
* USD 285,000.00

9302889 Marinus Errors of DNA replication are potentially a major source of
spontaneous and drug induced mutation. Repair systems exist to minimize such
damage. The best understood is DNA adenine methylation (Dam) directed mismatch
repair in Escherichia coli. Among the proteins involved in this repair system,
three (MutH, MutL and MutS) are of particular importance. These three proteins
appear to form a complex in vivo with DNA containing a replication error. MutL
and MutS protein homologs have been detected in other bacteria, yeast, mouse and
man indicating that the same basic repair mechanism is used. This work is
defining, in E. coli, how the MutL protein interacts with the other Mut proteins
(MutS and MutH) using genetic methods coupled with biochemical assays. Dominant
negative mutations in the mutL gene will be selected by a newfangled procedure
and will be mapped by DNA sequencing. The mutants will be sorted into different
classes by their ability to be complemented in trans by MutS, MutH and MutL.
Partially purified MutL from representative mutants will be assayed in vitro for
ability to assemble and form a complex (MutH, L, S,) capable of incising DNA.
This will test for the ability to associate with and activate MutH endonuclease
activity. We will also test for the ability to interact with MutS using DNAse I
protection assays ("footprinting"). These biochemical assays will then allow
correlations to be made between location of the mutation in the gene and
functional activity of protein domains. %%% This research represents a way not
only to increase our knowledge about how mutations are prevented but also to
understand in general how proteins assemble into functional complexes, ***