* 9315948
* Mechanism of Recombination Between RNAs Associated With     Turnip Crinkle Virus
* NONE,NONE
* 01/15/1994,06/30/1998
* Clifford Carpenter, University of Massachusetts Amherst
* Continuing grant
* DeLill Nasser
* 06/30/1998
* USD 559,968.00

Simon MCB-9315948 Abstract Turnip crinkle virus is an ideal system to study the
mechanism of RNA recombination. The genome is among the smallest viral RNA
genomes and is associated with satellite RNAs and numerous small, subviral RNAs
(Sat-RNA) which are the products of recombination events within the genomic RNA
or between the genomic and subviral RNAs. The development of an in vitro
recombination as well as a protoplast recombination system has allowed a more
complete characterization of the elements and factors which mediate
recombination in this system. A model of recombination has resulted from our
work which suggests that coat protein helps to mediate the recombination event
by possibly positioning the templates in proximity. In order to test this model
and to continue characterizing the event, we will: 1. Establish protoplast and
in vitro assays for recombination. 2. Conduct structural studies on motif I (the
sequences found at the right sides of all junctions of recombinant RNA) using
both mutagenesis and chemical and enzymatic probes. 3. Determine the role of
coat protein in recombination between sat-RNA D and Sat-RNA C using gel-shift
assays and 4. Assay for binding of the RNA-dependent RNA polymerase to motif I.
%%% Viruses which depend on RNA-dependent RNA polymerases for replication of
their genomic and sub-genomic RNAs are subject to high inherent mis-
incorporation of nucleotides that can lead to heterogeneous mixtures of related
species. Thus, RNA recombination is now thought to have been, and perhaps still
is, a major factor in the evolution of viruses. The discovery that the turnip
crinkle virus contained nearly the entire sequence of a satellite RNA joined to
two regions of genomic RNA by this laboratory suggested that non-homologous
recombination occurs. That fact, as well as small genome size and the generation
of recombinant molecules in a non-random manner suggested that a detailed
examination of the sequences, structures and mechanisms inv olved in the
recombinational event would be productive. It is critical in many ways to have a
better understanding of the mechanisms used by viruses in the evolutionary
process and the information resulting from the specific goals pursued in this
work will enhance that understanding.