* 9600544
* Primer Synthesis Kinetics by E. coli Primase
* BIO,MCB
* 08/01/1996,07/31/2000
* Mark Griep, University of Nebraska-Lincoln
* Continuing Grant
* Thomas E. Smith
* 07/31/2000
* USD 247,529.00

9600544 Griep This is a study of the nucleotide insertion and incorporation
kinetics of primase. When errors are incorporated into the DNA during
replication, a process during which primase plays a key role, the result is
genetic mutation and possibly cancer. Recent studies indicate that primase is
the most error-prone enzyme of DNA replication and may very well be the enzyme
which misincorporates the most errors into the chromosome. Primase is the
single-stranded DNA-dependent RNA polymerase that synthesizes a short (11+1
nucleotide) RNA polymer that serves to initiate DNA synthesis. Primase is
required because DNA polymerases are great at elongating DNA but cannot initiate
polymers. The primase from E. coli will be studied because it has an especially
high specificity during initiation, it prefers to initiate complementary to
d(CTG) in the template. This specificity allows one to monitor the length- and
sequence-dependence of the primers that are made. Other than this feature
though, eukaryotic and bacterial primases exhibit very similar kinetic,
structural and binding properties. Primases, like all RNA polymerases, carry out
polymer synthesis in three distinct stages, polymer initiation, elongation and
termination. Among the nucleic acid polymerases, primases (from bacterial to
human) exhibit the highest nucleotide bypass efficiency and the greatest ability
to incorporate NTP sugar analogs such as dNTP. Nucleotide bypass is when a
polymerase inserts an incorrect nucleotide opposite the template strand and then
incorporates that misinsertion by adding the next correct nucleotide. If this
were to happen in vivo with anywhere near the efficiency that it occurs in vivo,
it might lead to mutation of the genetic material. Current evidence from the
proposer's lab suggests that the efficiency of nucleotide bypass changes during
the different stages of polymer synthesis. Primase has very high template
sequence specificity during initiation but not during elongation or termination.
For instance , primase is unlikely to initiate synthesis with a dNTP but will
insert a dNMP just as efficiently as a NMP during termination. One possible role
for this ability during elongation may be to create a RNA polymer with a
3'-terminal dNMP which may be recognized more effectively by the DNA polymerase.
On the other hand, if dNMPs are incorporated more efficiently into the middle of
the "RNA primer", then the primer may serve as a better (or worse) target of
RNase H or the 5'-3' exonuclease of DNA polymerase I. This study will determine
the stage at which primase most efficiently incorporates the correct versus the
incorrect dNMPs. The information gained will be used to hypothesize a role for
these (mis)incorporated deoxyribonucleotides. %%% The two DNA strands are
replicated separately and coordinately. Helicase and DNA polymerase are required
for continuous synthesis of the "leading strand" while the "lagging strand"
additionally requires primase so that it may be synthesized as numerous short
fragments. Each of these fragments begin with an 11+1 nucleotide RNA synthesized
by primase that is later excised by other enzymes. Using only primase, its
nucleotide substrates, and a DNA template, we have developed a simple assay
system to measure template sequence-specific primer synthesis. It was possible
to develop this assay system because initiation takes place at a known sequence,
d(CTG), and the length of the synthesized primer is 12 nucleotides and greater.
In the test tube, pure primase is the most error-prone DNA replication enzyme.
We would like to determine when during primer synthesis the most errors occur so
that we can hypothesize whether the errors stimulate the excision enzymes and
what might be their biological relevance. We will measure the rate of primase
errors and determine the effect of distance from the initiation site, of one or
several missing sites on the template, of different initiation sequences, and of
DNAB helicase on all of these. ***