* 9603567
* Structure Function Analysis of Bacterial DNA                Methyltransferase
* BIO,MCB
* 05/01/1997,04/30/2000
* Norbert Reich, University of California-Santa Barbara
* Continuing Grant
* Thomas E. Smith
* 04/30/2000
* USD 294,163.00

9603567 Reich Part 1-Technical The proposed work focuses on the bacterial DNA
methyltransferases, M.EcoRI and M.BamHI, which methylate the underlined N6 and
N4 cytosine in GAATTC and GGATCC, respectively. M.EcoRI was recently shown to
bend DNA by 50 degrees and to flip out the adenine prior to methylation. Base-
flipping was detected by a novel, dynamic assay which uses the fluorescent base
analog, 2-aminopurine. The first goal uses M.EcoRI and various methods to
investigate base-flipping and DNA bending processes. Presteady state
fluorescence experiments will be used to determine if base-flipping is enzyme-
assisted or if the enzyme simple stabilizes the extrahelical base. Time-resolved
fluorescence measurements will be used to determine the orientation and mobility
of the extrahelical base within the enzyme-DNA complex. The combined DNA
fluorescence measurements will be compared to similar changes in protein
(tryptophan) fluorescence so that a complete kinetic description of early steps
in the catalytic cycle should be provided. This analysis will provide detailed
kinetic information about the enzyme-mediated deformation of DNA conformation.
Amino acid residues implicated in catalysis and adenine stabilization by
structural homology to the related M. TaqI enzyme will investigated to assess
their functional role (s). The second goal is to determine the structures of
M.EcoRI and M. BamHI using X-ray diffraction methods. Small crystals of an
M.EcoRI-DNA complex and crystals of M.BamHI which diffract to 2.7 angstrom have
been obtained. Further crystallization trials are proposed. Crystallization
trials of various M.EcoRI mutants are proposed. Part 2-Non technical Many
enzymes which alter the ahemical structure of DNA, do so by stabilizing large
scale conformational changes within the DNA molecule. These include significant
DNA "bends" of up to 100 degrees and movement of individual bases outside of the
normal double-helix of DNA. How these enzyme-mediated changes in DNA conf
ormation contribute to the enzyme's function is not understood. This proposal
will measure changes in DNA structure involving bending and "base flipping"
using the bacterial DNA methyltransferase, M.EcoRI. This enzyme transfers a
methyl group to the second adenine in GAATTC. We have developed a novel
fluorescence-based assay to measure base-flipping in this enzyme, and propose to
extend these experiments to measure how this occurs over time. Related
experiments will be done to measure how the enzyme changes the flexibility and
orientation of the base once it is removed from the double helix. The proposal
also seeks to identify which amino acids within the protein are responsible for
these conformational changes. A second long term goal of the proposal is to
determine the three-dimensional structure of the enzyme using X-ray
crystallographic methods.