* 9631137
* How are Restriction Systems Controlled, and How Do They     Recognize DNA Sequences?
* NONE,NONE
* 08/01/1996,01/31/2000
* Robert Blumenthal, University of Toledo Health Science Campus
* Continuing grant
* Susan Porter Ridley
* 01/31/2000
* USD 274,285.00

9631137 Blumenthal Restriction endonucleases (REases) are enzymes that cleave
both strands of double-stranded DNA at specific nucleotide sequences. Bacteria
that produce REases must protect their own DNA, and this is generally done by
producing a methyltransferase (MTase) that transfers a methyl group from
S-adenosyl-L-methionine to the DNA: the REases do not cleave DNA sequences that
have been properly methylated. Two questions stand out. First, the genes for
REases and their cognate MTases are often located on mobile genetic elements
such as plasmids. When such a plasmid is transferred into a new host cell, how
are the genes and enzymes controlled to ensure that the new host's DNA is
methylated before REase activity appears? This question has not been fully
answered for any REase / MTase pair. One major goal of this project is to answer
this question for the PvuII REase / MTase pair. The proposing laboratory has
discovered three features affecting PvuII a predicted hairpin in the mRNA for
the PvuII REase which may modulate initiation of translation; an apparent
transcriptional activator that is required for expression of the PvuII REase
gene and a small protein that appears to prevent the two identical subunits of
PvuII REase from associating with one another. The second major question
regarding REases and MTases is how they recognize their substrate nucleotide
sequence amongst all of the other DNA that is present. REases and MTases must
not only bind to a specific DNA sequence, but must then bring the relevant
portions of the DNA molecule into contact with an active site or sites. Does
this impose particular constraints on how the DNA sequence is recognized? One
way to approach this is to compare several proteins that recognize the same
sequence (or closely related sequences). This project seeks to characterize the
DNA sequence recognition by the following proteins "^" indicates cleavage, 5mC
and N4mC indicate methylation of the cytosine at the indicated position: PvuII
REase CAG^CTG PvuII MTase CAGN4mCTG AluI Ease AG^CT AluI MTase AG5mCT These
proteins will be compared to those basic region helix-loop-helix transcription
factors, studied by others, which recognize the sequence CAGCTG but do not
catalyze any reaction. %%% Certain bacteria produce specialized DNA modifying
enzymes, restriction endonucleases (REase), that destroy DNA from other bacteria
or viruses. Protection of the organism's DNA is done by a different type of DNA
modifying enzymes, methytransferases, that change the chemical nature of the DNA
so that the REase does not destroy the DNA of the bacteria that produces the
REase. REases are utilized in biological research to cut DNA at precise
locations like microscopic lasers to dissect genes. This project will provide
insight into the regulation, structure, and function of these modifying enzymes
providing information for the eventual design of new molecular biological tools,
and the ability to devise mechanisms to manipulate genetic transfer in bacteria.
***