* 1101859
* Modified Uridines, Contributors of Novel Chemistries to Functional RNA
* BIO,MCB
* 05/15/2010,12/31/2013
* Paul Agris, SUNY at Albany
* Continuing Grant
* Michele McGuirl
* 12/31/2013
* USD 223,162.00

Decoding of genomic information, so basic to the biology of protein synthesis in
all cells and seemingly so simple, now is complicated by the finding that it
requires a diverse set of chemical additions to the RNA molecules that are
involved, transfer RNAs (tRNAs). tRNAs are responsible for translating the
genetic information for a protein into the amino acid sequence of that protein.
Many tRNAs devoid of these chemical modifications, particularly at two different
positions (the anticodon wobble position-34 and/or the purine nucleoside-37),
will not recognize the amino acid code incorporated into messenger RNA. The
chemical modifications at these two positions in the tRNAs rescue tRNAs' ability
to read the amino acid codes, codons. The tRNAs for amino acids that have two
codons have to be restricted in their reading so that they do not read the
codons of other amino acids. The tRNAs for amino acids that have four codons
have to have the ability to read the four codons. Significantly, these two sets
of tRNAs are differently modified. Thus, the proposal's long-term objectives are
to a) Establish general principles for tRNA's modification-dependent decoding of
codons; b) Educate and excite young scientists to the chemistries of RNA by
integrating education with science; c) educate the scientific community as to
the importance of modifications tRNA function; and d) apply an understanding of
modification structure/function relationships to designing RNA tools in
molecular and cellular biology. The research is focused on the physical chemical
mechanism(s) by which modifications may control tRNA's recognition of two versus
four codons. Do distinctly different modifications contribute to tRNA's ability
to decode two versus four-fold codons by restricting codon recognition in the
former and expanding wobble in the latter? Does codon recognition require that
the protein synthesizing machinery, the ribosome, induce a correct fit
architecture to tRNA? Do modifications pre-form the tRNA structure for protein
synthesis? To answer the first question modifications that restrict tRNAs
binding to two codons will be compared to the binding of differently modified
tRNAs to four codons on the ribosome. To answer the second and third questions,
the affects of the various modifications tRNA stability, and structure in
solution, and structure on the ribosome will be related to their ability to
decode. Solutions structures from nuclear magnetic resonance spectroscopy will
be compared to each other and to the structures bound to codon on the ribosome.
These experiments will determine whether modifications pre-structure the tRNA
prior to codon binding, and whether the ribosome induces architecture for codon
binding.

An understanding of the modification-dependent mechanism of tRNA's reading of
the Genetic Code is fundamental to biochemistry, cell biology and to the
education of young scientists. A new structural-based model of wobble decoding
on the ribosome with modification-dependent, novel base pairing geometries will
be generated. The model will include either a modification-dependent pre-formed
or ribosome induced-fit anticodon architecture.

The broader impact of the project is focused on the educational objective to
instill scientific curiosity in young minds, and research experience in young
hands by exciting their involvement in answering questions about RNA biology. By
integrating research with education undergraduate and graduate students at North
Carolina State University and North Carolina Agricultural and Technical State
University, a historically black college, will learn scientific methodology from
peers and mentors. To the benefit of society underrepresented minorities and
women will be mentored in physical biochemistry; they represent a significant
proportion (12%) of the Department's present majors. NCSU's research
infrastructure will be enhanced by purchase of a microcalorimeter for the
physiochemical study of RNA. The collective housing of a number of instruments
will establish a shared educational infrastructure and creates a supportive
environment for interactions and mentoring among Triangle Universities dedicated
to RNA research.