* 1109989
* BREAD: Novel Biomarkers that Rapidly Identify Insect Populations that are Efficient Vectors of Circulative Plant Viruses
* BIO,IOS
* 07/01/2011,06/30/2015
* Stewart Gray, Cornell University
* Continuing Grant
* Diane Jofuku Okamuro
* 06/30/2015
* USD 924,624.00

PI: Stewart Gray (USDA-ARS/Cornell University)

Co-PI Michael MacCoss (University of Washington) and Lava Kumar [International
Institute of Tropical Agriculture (IITA), Nigeria]

Senior Personnel: Michelle Cilia [USDA-ARS, Ithaca, NY], Michael Bereman
(University of Washington), Alvin Simmons [USDA-ARS, Charleston, SC], and Rachid
Hanna [International Institute of Tropical Agriculture (IITA), Cameroon]

Control of insect vectors of plant and animal viruses is arguably one of the
biggest challenges to agriculture and human health and is one of the seven goals
outlined by the Bill and Melinda Gates Foundation in its Grand Challenges in
Global Health initiative. Plant viruses in the families, Luteoviridae,
Nanoviridae, and Geminiviridae, cause diseases with serious economic
consequences in many staple food crops in Sub-Saharan Africa and a majority are
transmitted aphids and whiteflies in a circulative, persistent manner. Cultural
and chemical control strategies, aimed at reducing or avoiding infection, can be
effective; however, their success is dependent upon knowledge of the population
dynamics of the insect vectors, including their efficiency to transmit viruses.
This is challenging because populations of insects, even within the same
species, vary widely in their efficiency of virus transmission making targeted
management strategies nearly impossible. Recently several protein biomarkers
were identified that distinguish aphid populations capable of transmitting these
circulating viruses from those that are incapable of transmitting the virus.
This project will determine if these biomarkers can be used to identify vector
competent populations of other aphid and whitefly species that transmit similar
circulating viruses affecting a wide range staple food crops in sub-Saharan
Africa and elsewhere in the world. If common or species-specific biomarkers are
identified, they can be used to rapidly identify those insect populations that
are efficient virus vectors and target them for application of virus disease
management strategies. Targeted vector control will reduce the unnecessary
application of pesticides and other crop inputs. In many cases the insects that
are not vectors of viruses are not injurious to overall crop health.

This targeted approach will help to minimize virus transmission within and
between cropping systems, and will disproportionately benefit the resource poor,
food insecure nations. The judicious use of crop inputs based on risk
information rather than continual prophylactic applications has economic,
environmental and health benefits. Indeed, a targeted approach to integrated
pest management has the potential to provide farmers with the best form of plant
virus protection: early and fast detection of vector species and avoidance of
infection. The project will focus on solving the key technology components
necessary to give smallholder farmers access to advanced approaches for the
control of insects that transmit some of the most economically devastating plant
viruses. Outreach and education components are focused on implementing a
pipeline for the eventual deployment of a field-based assay in Africa to target
the most important insect vector species. All of the biological information
generated from virus transmission assays and gene sequencing will be made
available to the scientific community through publications, long-term
repositories (NCBI, USDA-ARS) and project websites and directly to integrated
pest management (IPM) and agricultural stakeholders at conferences. Data will be
compiled into a handbook for smallholder farmers for rapid dissemination
throughout Africa via IITA extension specialists. All raw mass spectrometry data
files, Skyline documents, MS/MS spectrum libraries, and refined targeted
proteomics instrument methods generated using Skyline will be available at
http://proteome.gs.washington.edu.