* 1110080
* BREAD: Determining the Pan-African Sweet Potato Virome: Understanding Virus Diversity, Distribution and Evolution and their Impacts on Sweet Potato Production in Africa
* BIO,IOS
* 07/01/2011,06/30/2015
* Jan Kreuze, Boyce Thompson Institute Plant Research
* Continuing Grant
* Diane Jofuku Okamuro
* 06/30/2015
* USD 700,196.00

PI: Zhangjun Fei (Boyce Thompson Institute for Plant
Research)&lt;br/&gt;&lt;br/&gt;Co-PIs: Jan Kreuze [International Potato Center
(CIP), Peru]&lt;br/&gt;&lt;br/&gt;Key Collaborators: Martin Chiona (National
Agricultural Research Organization, Zambia), Paul Demo (CIP-Malawi), Britta
Kowalski (CIP-Angola), Douglas Miano (Kenya Agricultural Research Institute,
Kenya), Settumba Mukasa (Makerere University, Uganda), Joseph Ndunguru
(Mikocheni Agricultural Research Institute, Tanzania), Ian Robertson and
Elizabeth Ngadze (University of Zimbabwe), Steffen Schulz (CIP-Ethiopia) and
Martine Tachin (Université d'Abomey-Calavi, Rép.
Bénin&lt;br/&gt;&lt;br/&gt;Sweet potato is among the most important food crops
in the world and an extremely important food crop for subsistence farmers in
sub-Saharan Africa (SSA). It is grown throughout the African continent and
currently around 34.5% of global sweet potato area is in Africa. One major
limitation in sweet potato production is cultivar decline, mostly due to the
cumulative effect of virus infection on this vegetatively propagated crop. Thus,
viral diseases are considered a major limiting factor in sweet potato production
worldwide, and particularly in SSA. The objective of this project is to evaluate
a novel approach, deep sequencing of small RNAs from field-grown sweet potato
samples collected throughout Africa, to systematically and efficiently identify
known and novel virus genome sequences. A total of around 1750 geo-referenced
field-grown samples of sweet potato will be collected from more than ten
countries in Africa. Small RNA populations of these samples will be prepared and
sequenced using high throughput next-generation sequencing technology, and then
assembled to identify known and novel sweet
potato&lt;br/&gt;viruses.&lt;br/&gt;&lt;br/&gt;Broader Impacts: Food security
remains a huge challenge for millions of Africans, particularly for those in
sub-Saharan regions, who depend on agriculture for their subsistence. Emerging
and reemerging pathogens, including many viruses, continue to cause devastating
losses of food production in Africa. Current knowledge of crop viruses in Africa
is limited and sporadic at best. Novel virus genome identification technology
through deep sequencing of small RNA population is potentially applicable to
continental surveys as its efficiency in virus identification has been proven
with both greenhouse and field grown samples. This technology can then be
applied to systematically determine the total virus genome sequences (virome) on
a number of other major crop plants in Africa such as maize, rice, tomato, bean,
and cassava. Public availability of such information will provide the scientific
community and government unprecedented possibilities to understand crop virus
distribution in Africa, guide phytosanitary requirements, predict risks of
future epidemics, and suggest regional disease management strategies. Data
generated under this project will be disseminated through the project website
(accessible via http://bti.cornell.edu/ZhangjunFei.php) and raw and processed
small RNA sequences will also be available through EBI Sequence Read Archive and
NCBI Gene Expression Omnibus, respectively. Novel virus genome sequences will be
deposited into GenBank.