* 1127027
* Novel Quantitative Proteomic Methods to Discover and Localize Endogenous Protein Complexes
* BIO,IOS
* 12/15/2011,11/30/2017
* Mark Hall, Purdue University
* Standard Grant
* Anne W. Sylvester
* 11/30/2017
* USD 1,709,240.00

PI: Daniel Szymanski (Purdue University)

CoPIs: Mark Hall, Daisuke Kihara, Jun Xie (Purdue University)

There is a strong need for systems-level data sets that enable a more efficient
translation of basic knowledge into improved crop traits. Information on protein
complex composition is one such example, but resources for crop plants are
lacking. This project proposes a novel mass spectrometry-based method to solve
endogenous protein complexes in soybean. The results will serve as hypothesis-
generating machines for the reverse-genetic analysis of a major crop. The
approach couples multiple chromatography separations of cytosolic extracts with
quantitative label-free mass spectrometry of the column fractions. The relative
abundance of the proteins across the purification scheme will be used to cluster
co-purifying proteins into groups that reveal protein complex composition.
Unlike other large-scale, protein interaction techniques, this new approach is
simple to experimentally validate using existing mutant collections. This
inexpensive technique does not require gene replacement technology, large-scale
gene cloning, or targeted purification strategies. To establish the
effectiveness of this method, the investigators will pursue three research
objectives: 1) to predict and validate the composition of 100 endogenous protein
complexes isolated from leaf cytosol; 2) to use bioinformatics and existing
"omics" data sets to predict and evaluate models for protein complex
composition; and 3) to determine if the compositions of cytosolic protein
complexes are conserved in Glycine max.

Although mass spectrometry has been used extensively in proteomics research, to
the investigators' knowledge the proposed strategy for large-scale protein
complex prediction has not been done. Successful completion of this work will
provide broadly useful results, databases, and reagents that will reveal the
composition of endogenous protein complexes. This deep analysis of the cytosolic
proteome will provide an important data set that enables crop scientists to
further modify proteins and pathways with the goal to improve crop productivity.

The Purdue Initiative on Plant Protein Interactions (PIPPI) will provide broad
impact by reaching diverse audiences through such activities as research
opportunities for undergraduate and graduate students and postdoctoral fellows;
and a website providing data on plant protein complexes for researchers.
Students will gain skills collaborating in diverse university research
environments, be exposed to cutting-edge research in plant proteomics, and
develop leadership and organizational abilities by assisting with project
management, research design, and data release. By partnering with various campus
diversity programs and offices, all aspects of the project will involve
underrepresented undergraduate and graduate students in STEM disciplines.
Proteomic data sets will be available to the public at
https://proteomecommons.org. It is expected that this novel project will help
prepare the next generation of plant geneticists that utilize diverse types of
data to drive crop improvement strategies.