* 1129734
* Collaborative research: Diversity and dynamics of planktonic ciliates - what can next-generation sequencing  technologies tell us?
* GEO,OCE
* 09/01/2011,08/31/2015
* Laura Katz, Smith College
* Standard Grant
* David Garrison
* 08/31/2015
* USD 421,638.00

The Ocean's biomass and diversity are predominantly microbial, yet this aspect
of diversity remains underexplored. Efforts in recent years have begun to
document microbial diversity in marine systems, and to elucidate the processes
that structure assemblages across space and time. This project focuses on two
important sister clades of microbial eukaryotes, the oligotrich and choreotrich
ciliates. These organisms comprise a major component of planktonic food webs as
they graze on phytoplankton, and are in turn eaten by zooplankton and larval
fish.&lt;br/&gt;&lt;br/&gt;Earlier molecular work on ciliate diversity relied on
light microscopy, construction of clone libraries and Sanger sequencing. This
revealed a high degree of cryptic diversity (similar species that are
genetically distinct), which is surprising, given the long-held idea that all
microbes are globally distributed and that few species exist, at least as
compared to animals and plants. This past work also showed that ciliate
assemblages contain a few highly abundant forms and many rare ones, consistent
with the concept of a "rare biosphere". However, these methods are limited by
high costs of both labor and materials, so that efforts to sample any local
assemblage comprehensively usually resulted in undersaturation (repeated
sampling continued to uncover new species). Next generation approaches are
needed to truly assess the depths of biodiversity in planktonic
ciliates.&lt;br/&gt;&lt;br/&gt;This project brings together investigators with
strengths in ecology, taxonomy and oceanography (PI McManus) and in molecular
evolution, systematics and bioinformatics (PI Katz). Pyrosequencing will be used
to sample the oligotrichs and choreotrichs "to exhaustion" in coastal
environments. Denaturing gradient gel electrophoresis (DGGE), a technique that
generates a fingerprint of the diversity in a sample, will be used to pre-select
samples for pyrosequencing based on where strong gradients are observed in the
composition of assemblages in relation to environmental factors (density fronts,
thermoclines, etc.). Using these approaches, combined with an informatics
pipeline already in place, this project addresses three specific
objectives:&lt;br/&gt;&lt;br/&gt;Objective 1. Determine the spatial scale of
variability in ciliate diversity by measuring how ciliate assemblages change
over meter, kilometer, 100 km, and basin scales.&lt;br/&gt;Objective 2. Assess
the contributions of different size classes of ciliates to overall assemblage
diversity.&lt;br/&gt;Objective 3. Experimentally evaluate factors that control
the temporal shift of individual species from rarity to commonness in a natural
assemblage, and vice versa.&lt;br/&gt;&lt;br/&gt;BROADER IMPACTS: A program to
involve K-12 teachers in research, initiated under a previous award, will be
continued. For the past three years, teachers have been hosted at the UConn
marine labs, where they conducted independent research, learned to sample and
enumerate plankton, and developed teaching resources for subsequent use in the
classroom. In this project the program will be adapted to accommodate the
shorter, more intense experiences most teachers prefer, and to incorporate an
instructional component in broader marine sciences. In addition to the teacher
program, this project will involve training of postdoctoral fellows and
undergraduates. This will include cross-disciplinary work through exchanges
between the PI laboratories, with a focus on experimental design, data
collection/analysis, and manuscript preparation. Both PIs are committed to
involving students from underrepresented groups in their research, and PI Katz
maintains intensive recruiting efforts in this area. Hence, the project will
further increase the diversity of researchers studying biological oceanography.