* 1137089
* EFRI-MIKS: Control of Signaling and Function by Design with Spatially Pre-Structured Microbial Communities
* ENG,EFMA
* 09/01/2011,08/31/2017
* Rustem Ismagilov, California Institute of Technology
* Standard Grant
* Aleksandr Simonian
* 08/31/2017
* USD 1,998,143.00

1137089&lt;br/&gt;Ismagilov&lt;br/&gt;This project aims to develop a rigorous
engineering framework to enhance understanding of interspecies&lt;br/&gt;and
interkingdom signaling in order to enable building of "microbiome in a pill"
particles precisely&lt;br/&gt;engineered, spatially structured microbial
communities encapsulated in a biocompatible material to&lt;br/&gt;ultimately
prevent and treat human diseases associated with dysbiosis (microbial
imbalance). This&lt;br/&gt;engineering framework will enable prediction and
control of interspecies signaling and interactions both in&lt;br/&gt;vitro and
in vivo by solving four challenges: Goal 1 (MIKS1a): Use computational methods
to predict, for&lt;br/&gt;a given desired function, which specific microbes are
needed and which spatial structures are required to&lt;br/&gt;achieve that
function. Goal 2 (MIKS1b and MIKS2): Develop in vitro microfluidic technologies
and&lt;br/&gt;associated design principles to understand signaling by rapidly
characterizing and measuring the impact&lt;br/&gt;of spatial structure on the
complex molecular interactions that underlie signaling, and host cells?
response&lt;br/&gt;to signaling. Goal 3 (MIKS2 and MIKS3): Develop and test
design principles for the scale up of spatially&lt;br/&gt;pre-structured
microbial communities, discovered and characterized in Goals 1 and 2, to
enable&lt;br/&gt;production of engineered functional communities in gram
quantities needed for in vivo testing. Goal 4&lt;br/&gt;(MIKS2 and MIKS3): By
using communities produced in Goal 3, and by gavaging them into germ-
free&lt;br/&gt;and gnotobiotic mice, measure in vivo the function of the
community and the host's response, and iterate&lt;br/&gt;with Goals 1-3 to
develop a set of design rules that enable rational engineering of spatially pre-
structured&lt;br/&gt;"microbiome in a pill" particles that predictably function
in vivo in a mammalian gut.&lt;br/&gt;Intellectual Merit: The importance of
multi-species and inter-kingdom systems and their signaling is&lt;br/&gt;widely
appreciated but general engineering approaches for control of such systems are
urgently needed.&lt;br/&gt;This project proposes that signaling and interactions
can be precisely engineered via a biomimetic&lt;br/&gt;approach control of
spatial structure. This is a potentially transformative shift in understanding
how&lt;br/&gt;signaling, metabolic exchange, and function of communities are
controlled. The mammalian microbiome&lt;br/&gt;is an excellent test system
because it contains a diverse number of species interacting with one
another&lt;br/&gt;and the host. This work will lay the foundation for a new
engineering paradigm, a "microbiome in a pill",&lt;br/&gt;and establish design
principles for controlling signaling and metabolic exchange within the
community,&lt;br/&gt;between the community and competitors, and between the
community and the host. Engineering&lt;br/&gt;communities with spatial structure
may close important gaps in understanding the mechanisms of&lt;br/&gt;dysbiosis
and manipulating communities affected by dysbiosis, ultimately leading to more
effective&lt;br/&gt;prophylactics and treatments for human disease. The
assembled team is well-qualified to carry out the&lt;br/&gt;proposed work: The
PI has expertise in microfluidic technology development, and analysis and
modeling&lt;br/&gt;of complex networks in biological contexts. Co-PIs Meyer and
Henry are experts in automated annotation&lt;br/&gt;of metagenomic datasets and
in using these annotations to create and analyze genome-scale
metabolic&lt;br/&gt;models. Co-PIs Chervonsky and Mazmanian are experts in the
use of germ-free and gnotobiotic animal&lt;br/&gt;models to understand the
impact of microbiota in the mammalian gut on the host immune
system.&lt;br/&gt;Broader Impact: Ultimately, the simple yet precisely
engineered "microbiome in a pill" communities&lt;br/&gt;designed in this work
could dramatically improve quality of life and reduce healthcare costs by
modulating&lt;br/&gt;dysbiosis in the context of conditions affecting millions
of Americans: IBD, infections, diabetes,&lt;br/&gt;autoimmune conditions, and
obesity. This approach may pioneer new solutions for problems
where&lt;br/&gt;genetically modified microorganisms are not acceptable or
feasible. The design rules for building&lt;br/&gt;functional microbial
communities developed here should have broad impact outside of human health,
with&lt;br/&gt;the potential to address numerous issues facing society today,
including environmental contaminants,&lt;br/&gt;climate, and food and fuel
production, which are all strongly impacted by the function of
microbial&lt;br/&gt;communities. The project will provide education, training,
and mentoring for graduate and undergraduate&lt;br/&gt;students and postdoctoral
scholars, enhanced by the PI's excellent track record in mentoring
students.&lt;br/&gt;This project will also provide researchers with numerous
tools to study and use microbial communities for&lt;br/&gt;specific functions.
Co-PIs Meyer and Henry have extensive track records in disseminating
computational&lt;br/&gt;methods and tools, and the PI has an excellent track
record of disseminating knowledge via high-impact&lt;br/&gt;publications and
partnerships with industry. This project will support public outreach through
REEL&lt;br/&gt;Science, hands-on science for K-12 curriculum development through
CalTech Classroom Connection,&lt;br/&gt;and research training for teachers and
high school students through the Summer Research Connection.