* 1138368
* EAGER: Harnessing the Power of Short-read Technology to Investigate Unexplored Microbial Communities in the Deep Euphotic Zone
* GEO,OCE
* 07/15/2011,06/30/2015
* Gabrielle Rocap, University of Washington
* Standard Grant
* David Garrison
* 06/30/2015
* USD 299,383.00

Recent advances in genome sequencing have transformed marine microbial ecology.
Metagenomic datasets have led to the discovery of new protein families,
underscored the importance of the rare biosphere and even allowed the study of
intra-population diversity of dominant members of the community such as
Prochlorococcus. Although open ocean microbial communities have been repeatedly
demonstrated to vary significantly with depth, in terms of both phylogenetic
diversity and biogeochemical function, the vast majority of sequence available
in public databases is from surface waters. Much less is known about microbial
communities below the surface of the ocean, even those at 50-200m. Next
generation sequencing technologies offer a dramatic reduction in cost per base
compared to Sanger or even pyrosequencing approaches. The very short read
lengths from these platforms have not yet been successfully applied to mixed
communities. This project is a metagenomic study of the microbial communities
throughout the water column in the South Atlantic gyre using the ABI SOLiD
platform with the following objectives:&lt;br/&gt;&lt;br/&gt;- Construct and
sequence paired-end metagenomic libraries from the surface, chlorophyll maximum,
deep euphotic zone and twilight zone in the South Atlantic gyre using ABI SOLiD
technology.&lt;br/&gt;- Apply information theory based software methods to
assemble the short read sequences and reconstruct the phylogenetic diversity and
metabolic potential of these communities. &lt;br/&gt;- Compare reconstructed
sequences to existing metagenomic data from other parts of the world's oceans
and to metaproteomic data from the the same location to inform sampling
strategies and methodological choices for a proposed global molecular oceanic
survey (Biogeotraces)&lt;br/&gt;&lt;br/&gt;This project is appropriate as an
EAGER because the genomic potential of deep euphotic zone and twilight zone
communities will be very different from those in surface waters. Thus, this
dataset will reshape our understanding of oligotrophic community function.
Second, the massive advance in data generated at a low cost by short-read
sequencing platforms has the potential to transform microbial oceanography by
allowing metagenomic analyses to be routinely employed in hypothesis-driven
experiments. However, new interdisciplinary tools are required to assemble and
interpret this data. Until these next generation bioinformatic methods are
established, proposals that take advantage of these technological breakthroughs
remain high risk.&lt;br/&gt;&lt;br/&gt;Broader Impacts: Metagenomic analyses of
microbial communities collected on a GEOTRACES compliant cruise in the South
Atlantic will provide a pilot dataset demonstrating the potential of short-read
sequencing technologies for a future sectional molecular survey program. A post-
doctoral researcher and a graduate student will be trained in the
interdisciplinary perspectives required to analyze this sequence data and
interpret it in a biogeochemical context.