* 1146926
* Collaborative:EAGER: A Model Based System for the Automated Design of Synthetic Genetic Circuits by Mathematical Optimization
* CSE,CCF
* 09/01/2011,08/31/2014
* Matthias Koeppe, University of California-Davis
* Standard Grant
* Mitra Basu
* 08/31/2014
* USD 287,100.00

Synthetic Biology is a nascent field with applications that range from bio-
fabrication to alternative energy. Despite its significance, engineering of
biological circuits still relies on trial-and-error tinkering techniques, with
limited computational support. If Synthetic Biology is to advance to more
complex synthetic systems that go beyond a handful of interacting parts, a
scalable, integrative, methodological approach is necessary. In an analogy to
integrated circuits, when it comes to circuit engineering, the role of detailed
computer models, optimization methods, simulators and design tools is
paramount.&lt;br/&gt;&lt;br/&gt;Intellectual Merit: This project aims to pave
the way towards an optimization-based, automated design framework for synthetic
gene circuits that adhere to user-defined constraints. A synthetic gene circuit
is a collection of one or more genes, together with elements (promoters,
ribosome binding sites, etc.) that influence gene expression. The wiring, i.e.
the order and position of every element, within a synthetic gene circuit
determines the gene expression pattern, and overall behavior of the circuit.
These circuits are introduced, usually as part of a plasmid(s), in a host
organism that can be readily manipulated in order to achieve a desired outcome
(e.g. specific temporal behavior, or production of an enzyme).
&lt;br/&gt;&lt;br/&gt;To facilitate faster time-to-market solutions and more
robust, predictable designs, PIs will develop a design and optimization tool
prototype. To that end, PIs propose a new optimization formulation that
encompasses multiple biological models relevant to synthetic genetic circuit
design. In addition, they propose a hybrid optimization-simulation technique to
capture additional effects related to cell division, noise, and evolutionary
processes. The investigation will focus on how state-of-the-art techniques from
combinatorial optimization can be applied to find the optimal circuit for a
specific task. Since the tool will need a library of well-characterized
components to operate, PIs will create a mutant library of three widely-used
regulators, then quantitatively characterize them, and store this information in
a publicly available database. As a proof-of-concept experiment, they will
assess their integrative approach by constructing an automatically-designed
synthetic circuit, measuring its output and deviation from the desired goal, and
then comparing it to other similar designs that have been already available in
literature. &lt;br/&gt;&lt;br/&gt;Broader Impact: An optimization-based, design
tool for synthetic biology has the potential to provide a service to the
academic community by reducing drastically the time-to-market aspect of
synthetic designs, and providing insight on biological function, thus
accelerating research in an exponentially growing field. All components and
characterized libraries that will be developed as part of this award will be
publicly available, deposited in the synthetic biology community?s standard
Parts Registry. Furthermore, this award will partially support the work and
training of the UC Davis IGEM team, a synthetic biology undergraduate team who
competes in the annual IGEM competition. Knowledge from this project will be
directly transferred into classrooms through the course ECS 289K "Computational
Challenges in Systems and Synthetic Biology" (UC Davis), and the course CSC
450/550 "Algorithms for Bioinformatics" (U. Arizona).