* 1148017
* EAGER: Collaborative Research: Exploratory application of single-molecule real time (SMRT) DNA sequencing in microbial ecology research
* GEO,OCE
* 09/15/2011,08/31/2013
* Ramunas Stepanauskas, Bigelow Laboratory for Ocean Sciences
* Standard Grant
* David Garrison
* 08/31/2013
* USD 98,918.00

Microbial communities are the primary drivers of the global biogeochemical
cycles that maintain the nutrient balance of ecosystems and ultimately shape
overall ecosystem function. Today, we appreciate the critical role microbes play
in the biochemical processing of nutrient elements; yet our understanding of how
the structure of microbial communities influences the suite of biogeochemical
processes within a given nutrient cycle is somewhat rudimentary. Limited
understanding of structure-function relationships between microbial communities
and biogeochemical cycles is due in large part to technological limitations in
characterizing the ecology of microbial communities. Until recently, microbial
ecologists simply had no means of unambiguously characterizing the richness and
evenness of species within a microbial community and the prevailing biochemical
capabilities of constituent microbial populations. Using high-throughput DNA
sequencing and three methodological approaches,shotgun metagenomics, PCR
amplicon sequencing, and genomics microbial ecologists are beginning to unveil
the inner workings of microbial communities and connect genetic details with
biogeochemical processes. While this work holds great promise for the
advancement of microbial ecology, currently-available high-throughput sequencing
technologies are not ideally suited to the high-sample throughput demands of
ecosystem science, the small genome size of bacteria and viruses, and their
genetic novelty. Moreover, in some cases the failure to adequately ground-truth
application of next-generation DNA sequencers to environmental DNA samples has
resulted in biased data and erroneous scientific conclusions. This "high-risk;
high-reward" research seeks to explore and test the use of a new, next-
generation DNA sequencer, the PacBio RS, which has several attributes that may
make it better suited to the specific needs of microbial ecology research and
has the potential to be highly transformative to this geoscience discipline. A
series of controlled and carefully replicated experiments will be conducted that
will test the use of PacBio sequencing for shotgun metagenomics, 16S PCR
amplicon sequencing, and single cell genome sequencing. This project will
leverage existing datasets from other high-throughput sequencing platforms
(e.g., Illumina and 454) to directly compare the performance of PacBio in each
of these application areas. Through a NSF Major Research Instrumentation award
to the University of Delaware, the PIs will have access to one of the few PacBio
RS instruments available at an academic institution. Ultimately, these
investigations will constrain the experimental error within PacBio sequencing
and serve as an initial demonstration of the utility of the instrument for
microbial ecology research. The Broader Impacts of this proposal includes an
effort to understand and constrain the sources of error and other biases within
PacBio sequencing, and make technical recommendations that will shape the
optimal use of the instrument within microbial science. In the course of this
work, the PIs will mentor a Ph.D. graduate student and a post-doctoral
researcher, and provide open access to all project data and findings.