* 1158056
* Evolution of Regulatory Sequence Motifs
* BIO,MCB
* 05/15/2012,04/30/2017
* Ivan Erill, University of Maryland Baltimore County
* Continuing Grant
* Devaki Bhaya
* 04/30/2017
* USD 421,645.00

Transcription factors are proteins that regulate the expression of genes by
binding to DNA at specific locations. In spite of years of research, little is
known about the evolutionary constraints faced by transcription factors and
their binding sites in the genome. As a result, it is difficult to improve on
the decades-old models of transcription factor binding that are still in use
today. The advent of high-throughput technologies, the widespread use of
comparative genomics techniques and the rapid emergence of synthetic biology
both facilitate and make more pressing the need for understanding how these
genetic elements evolve and interact, since this will enable the development of
models capable of establishing a much needed link between the encoding of
transcription on DNA sequence and its effects on the regulatory networks it
spawns. This project aims at validating and exploiting a robust computational
framework for the co-evolution of transcription factors and their binding motifs
in a realistic genome environment. The research will capitalize on the
integration of a genetic algorithm backbone with innovative modeling of
transcription factors in order to analyze how transcription factors and their
binding motifs evolve when confronted with specific genomic selective pressures,
regulatory needs and genome sizes. Most importantly, this research introduces a
reverse engineering approach aimed at infering basic principles on the behavior
of these genetic elements. The project will systematically analyze and quantify
for the first time key constraints in the evolution of binding sites and thus
has the potential to reshape the way we think about and model these genomic
elements. Specific emphasis will be made in compiling published data to validate
the predictions made by the system and in testing the efficacy of inferred model
constraints at improving computational discovery of transcription factor binding
sites.&lt;br/&gt;&lt;br/&gt;Broader Impacts&lt;br/&gt;This project will advance
our understanding of transcriptional regulation and its evolution, leading to
improved models of transcription factor binding motifs. It will also
substantiate a progressive movement towards in-silico simulations in biology,
and to the analysis of theoretical models using a bottom-up approach. The
research will provide a scalable tool for the exploration of transcriptional
evolution, as well as an up-to-date curated database of transcription factor
binding sites. Both items will be made freely available to the public,
facilitating the development and benchmarking of new transcription factor
models. Training and involvement of undergraduates in research is an essential
component of this project and will be centralized on the adoption of a peer-
mentored system for database curation. The project also defines participative
methods for integrating research results into courses taught by the PI and
others through the use of evolutionary videos, and the use of these elements in
specific events targeting K-12 students and in popularizing evolutionary theory.
The PI will exploit the leverage provided by UMBC programs to mentor and involve
minority students in research, with particular emphasis on the growing Hispanic
community. The pro-active outreach and dissemination components of the project
will also capitalize on previous efforts to exploit evolutionary simulations in
order to foster an informed debate on evolutionary questions across a broad
range of audiences.